

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Finding Single Copy Genes Out of Sequenced Genomes for Multilocus Phylogenetics in Non-Model Fungi</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0018803"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0018803" />

    <meta name="citation_title" content="Finding Single Copy Genes Out of Sequenced Genomes for Multilocus Phylogenetics in Non-Model Fungi"/>
    <meta itemprop="name" content="Finding Single Copy Genes Out of Sequenced Genomes for Multilocus Phylogenetics in Non-Model Fungi"/>

      <meta name="citation_author" content="Nicolas Feau"/>
            <meta name="citation_author_institution" content="INRA, UMR1202, BIOGECO (Biodiversité Gènes et Communautés), Cestas, France"/>
      <meta name="citation_author" content="Thibaut Decourcelle"/>
            <meta name="citation_author_institution" content="INRA, UMR1202, BIOGECO (Biodiversité Gènes et Communautés), Cestas, France"/>
      <meta name="citation_author" content="Claude Husson"/>
            <meta name="citation_author_institution" content="INRA, Nancy Université, UMR1136 Interactions Arbres/Microorganismes (IFR110), Champenoux, France"/>
      <meta name="citation_author" content="Marie-Laure Desprez-Loustau"/>
            <meta name="citation_author_institution" content="INRA, UMR1202, BIOGECO (Biodiversité Gènes et Communautés), Cestas, France"/>
      <meta name="citation_author" content="Cyril Dutech"/>
            <meta name="citation_author_institution" content="INRA, UMR1202, BIOGECO (Biodiversité Gènes et Communautés), Cestas, France"/>

    <meta name="citation_date" content="2011/4/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0018803.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e18803"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="6"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Multilocus phylogeography and phylogenetics using sequence-based markers.; citation_author=PH Brito; citation_author=SV Edwards; citation_journal_title=Genetica; citation_volume=135; citation_number=1; citation_pages=439-455; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic species recognition and species concepts in Fungi.; citation_author=JW Taylor; citation_author=DJ Jacobson; citation_author=S Kroken; citation_author=T Kasuga; citation_author=DM Geiser; citation_journal_title=Fungal Genetics and Biology; citation_volume=31; citation_number=2; citation_pages=21-32; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Rapid development of multiple nuclear loci for phylogenetic analysis using genomic resources: An example from squamate reptiles.; citation_author=TM Townsend; citation_author=RE Alegre; citation_author=ST Kelley; citation_author=JJ Wiens; citation_author=TW Reeder; citation_journal_title=Molecular Phylogenetics and Evolution; citation_volume=47; citation_number=3; citation_pages=129-142; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Assessing the performance of single-copy genes for recovering robust phylogenies.; citation_author=G Aguileta; citation_author=S Marthey; citation_author=H Chiapello; citation_author=MH Lebrun; citation_author=F Rodolphe; citation_journal_title=Systematic Biology; citation_volume=57; citation_number=4; citation_pages=613-627; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Evaluation of mitochondrial genes as DNA barcode for Basidiomycota.; citation_author=A Vialle; citation_author=N Feau; citation_author=M Allaire; citation_author=M Didukh; citation_author=F Martin; citation_journal_title=Molecular Ecology Resources; citation_volume=9; citation_number=5; citation_pages=99-113; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=The impact of molecular data in fungal systematics.; citation_author=PD Bridge; citation_author=BM Spooner; citation_author=PJ Roberts; citation_journal_title=Advances in Botanical Research; citation_volume=42; citation_number=6; citation_pages=34-67; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=FUNYBASE: a FUNgal phYlogenomic dataBASE.; citation_author=S Marthey; citation_author=G Aguileta; citation_author=F Rodolphe; citation_author=A Gendrault; citation_author=T Giraud; citation_journal_title=BMC Bioinformatics; citation_volume=9; citation_number=7; citation_pages=456; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Assembling the fungal tree of life: progress, classification, and evolution of subcellular traits.; citation_author=F Lutzoni; citation_author=F Kauff; citation_author=CJ Cox; citation_author=D McLaughlin; citation_author=G Celio; citation_journal_title=American Journal of Botany; citation_volume=91; citation_number=8; citation_pages=1446-1480; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=New primers for promising single-copy genes in fungal phylogenetics and systematics.; citation_author=I Schmitt; citation_author=A Crespo; citation_author=P Divakar; citation_author=J Fankhauser; citation_author=E Herman-Sackett; citation_journal_title=Persoonia; citation_volume=23; citation_number=9; citation_pages=35-40; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=A method for designing primer sets for speciation studies in filamentous Ascomycetes.; citation_author=I Carbone; citation_author=LM Kohn; citation_journal_title=Mycologia; citation_volume=91; citation_number=10; citation_pages=553-556; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Oligonucleotide primers for the universal amplification of beta-tubulin genes facilitate phylogenetic analyses in the regnum Fungi.; citation_author=E Einax; citation_author=K Voigt; citation_journal_title=Organisms Diversity Evolution; citation_volume=3; citation_number=11; citation_pages=185-194; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic relationships among yeasts of the ‘Saccharomyces complex' determined from multigene sequence analyses.; citation_author=CP Kurtzman; citation_author=CJ Robnett; citation_journal_title=FEMS Yeast Research; citation_volume=3; citation_number=12; citation_pages=417-432; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of Zygomycota based on EF-1alpha and RPB1 sequences: limitations and utility of alternative markers to rDNA.; citation_author=Y Tanabe; citation_author=M Saikawa; citation_author=MM Watanabe; citation_author=J Sugiyama; citation_journal_title=Molecular Phylogenetics and Evolution; citation_volume=30; citation_number=13; citation_pages=438-449; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Neolecta: A Fungal Dinosaur? Evidence from β-tubulin amino acid sequences.; citation_author=S Landvik; citation_author=OE Eriksson; citation_author=ML Berbee; citation_journal_title=Mycologia; citation_volume=93; citation_number=14; citation_pages=1151-1163; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Are Microsporidia really related to Fungi: a reappraisal based on additional gene sequences from basal fungi.; citation_author=Y Tanabe; citation_author=MM Watanabe; citation_author=J Sugiyama; citation_journal_title=Mycological Research; citation_volume=106; citation_number=15; citation_pages=1380-1391; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Characterisation of a β-tubulin gene from Melampsora lini and comparison of fungal β-tubulin genes.; citation_author=MA Ayliffe; citation_author=PN Dodds; citation_author=GJ Lawrence; citation_journal_title=Mycological Research; citation_volume=105; citation_number=16; citation_pages=818-826; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolutionary relationships among β-tubulin gene sequences of basidiomycetous fungi.; citation_author=D Begerow; citation_author=J Beate; citation_author=F Oberwinkler; citation_journal_title=Mycological Research; citation_volume=108; citation_number=17; citation_pages=1257-1263; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A fungal phylogeny based on 42 complete genomes derived from supertree and combined gene analysis.; citation_author=D Fitzpatrick; citation_author=M Logue; citation_author=J Stajich; citation_author=B G; citation_journal_title=BMC Evolutionary Biology; citation_volume=6; citation_number=18; citation_pages=99; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Fifty-three polymorphic microsatellite loci in the chestnut blight fungus, Cryphonectria parasitica.; citation_author=TL Kubisiak; citation_author=C Dutech; citation_author=MG Milgroom; citation_journal_title=Molecular Ecology Notes; citation_volume=7; citation_number=19; citation_pages=428-432; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Characterization of microsatellite loci in the fungus, Grosmannia clavigera, a pine pathogen associated with the mountain pine beetle.; citation_author=CKM Tsui; citation_author=N Feau; citation_author=CE Ritland; citation_author=S Massoumi-Alamouti; citation_author=S DiGuistini; citation_journal_title=Molecular Ecology Resources; citation_volume=9; citation_number=20; citation_pages=1500-1503; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=A comparison of the nature and abundance of microsatellites in 14 fungal genomes.; citation_author=S Lim; citation_author=L Notley-McRobb; citation_author=M Lim; citation_author=DA Carter; citation_journal_title=Fungal Genetics and Biology; citation_volume=41; citation_number=21; citation_pages=1025-1036; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Characterization of 41 microsatellite loci developed from the genome sequence of the poplar rust fungus, Melampsora larici-populina.; citation_author=C Xhaard; citation_author=A Andrieux; citation_author=F Halkett; citation_author=P Frey; citation_journal_title=Conservation Genetics Resources; citation_volume=1; citation_number=22; citation_pages=21-25; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=A fungal phylogeny based on 82 complete genomes using the composition vector method.; citation_author=H Wang; citation_author=Z Xu; citation_author=L Gao; citation_author=B Hao; citation_journal_title=BMC Evolutionary Biology; citation_volume=9; citation_number=23; citation_pages=195; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Phylogenomics reveal a robust fungal tree of life.; citation_author=EE Kuramae; citation_author=V Robert; citation_author=B Snel; citation_author=M Weiβ; citation_author=T Boekhout; citation_journal_title=FEMS Yeast Research; citation_volume=6; citation_number=24; citation_pages=1213-1220; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The tree versus the forest: The Fungal Tree of Life and the topological diversity within the yeast phylome.; citation_author=M Marcet-Houben; citation_author=T Gabaldón; citation_journal_title=PLoS One; citation_volume=4; citation_number=25; citation_pages=e4357; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Using the incomplete genome of the ectomycorrhizal fungus Amanita bisporigera to identify molecular polymorphisms in the related Amanita phalloides.; citation_author=RI Adams; citation_author=HE Hallen; citation_author=A Pringle; citation_journal_title=Molecular Ecology Notes; citation_volume=6; citation_number=26; citation_pages=218-220; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The ectomycorrhizal fungus Amanita phalloides was introduced and is expanding its range on the west coast of North America.; citation_author=A Pringle; citation_author=R Adams; citation_author=H Cross; citation_author=T Bruns; citation_journal_title=Molecular Ecology; citation_volume=18; citation_number=27; citation_pages=817-833; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Sex in Penicillium: Combined phylogenetic and experimental approaches.; citation_author=M López-Villavicencio; citation_author=G Aguileta; citation_author=T Giraud; citation_author=DM de Vienne; citation_author=S Lacoste; citation_journal_title=Fungal Genetics and Biology; citation_volume=47; citation_number=28; citation_pages=693-706; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Patterns of evolution of mitochondrial cytochrome c oxidase I and II DNA and implications for DNA barcoding.; citation_author=AD Roe; citation_author=FAH Sperling; citation_journal_title=Molecular Phylogenetics and Evolution; citation_volume=44; citation_number=29; citation_pages=325-345; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mutation hot spots in mammalian mitochondrial DNA.; citation_author=N Galtier; citation_author=D Enard; citation_author=Y Radondy; citation_author=E Bazin; citation_author=K Belkhir; citation_journal_title=Genome Research; citation_volume=16; citation_number=30; citation_pages=215-222; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular Evolutionary Genetics.; citation_author=M Nei; citation_number=31; citation_date=1987; citation_publisher=Columbia University Press; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity in Ampelomyces isolates, hyperparasites of powdery mildew fungi, inferred from RFLP analysis of the rDNA ITS region.; citation_author=L Kiss; citation_journal_title=Mycological Research; citation_volume=101; citation_number=32; citation_pages=1073-1080; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Ribosomal ITS sequences and plant phylogenetic inference.; citation_author=I Álvarez; citation_author=JF Wendel; citation_journal_title=Molecular Phylogenetics and Evolution; citation_volume=29; citation_number=33; citation_pages=417-434; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Rate and mode differences between nuclear and mitochondrial small-subunit rRNA genes in mushrooms.; citation_author=TD Bruns; citation_author=TM Szaro; citation_journal_title=Molecular Biology and Evolution; citation_volume=9; citation_number=34; citation_pages=836-855; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Use of atp6 in fungal phylogenetics: An example from the Boletales.; citation_author=AM Kretzer; citation_author=TD Bruns; citation_journal_title=Molecular Phylogenetics and Evolution; citation_volume=13; citation_number=35; citation_pages=483-492; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The evolution of base composition and phylogenetic inference.; citation_author=AØ Mooers; citation_author=EC Holmes; citation_journal_title=Trends in Ecology and Evolution; citation_volume=15; citation_number=36; citation_pages=365-368; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Multi-gene phylogenies define Ceratocystiopsis and Grosmannia distinct from Ophiostoma.; citation_author=RD Zipfel; citation_author=ZW de Beer; citation_author=K Jacobs; citation_author=BD Wingfield; citation_author=MJ Wingfield; citation_journal_title=Studies in Mycology; citation_volume=55; citation_number=37; citation_pages=75-97; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome-enabled development of DNA markers for ecology, evolution and conservation.; citation_author=RC Thomson; citation_author=IJ Wang; citation_author=JR Johnson; citation_journal_title=Molecular Ecology; citation_volume=19; citation_number=38; citation_pages=2184-2195; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Getting started in gene orthology and functional analysis.; citation_author=G Fang; citation_author=N Bhardwaj; citation_author=R Robilotto; citation_author=MB Gerstein; citation_journal_title=PLoS Computational Biology; citation_volume=6; citation_number=39; citation_pages=e1000703; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Large-scale assignment of orthology: back to phylogenetics?; citation_author=T Gabaldón; citation_journal_title=Genome Biology; citation_volume=9; citation_number=40; citation_pages=235; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=The quest for orthologs: finding the corresponding gene across genomes.; citation_author=A Kuzniar; citation_author=RC van Ham; citation_author=S Pongor; citation_author=JA Leunissen; citation_journal_title=Trends in Genetics; citation_volume=24; citation_number=41; citation_pages=539-551; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=OrthoMCL-DB: Querying a comprehensive multi-species collection of ortholog groups.; citation_author=F Chen; citation_author=AJ Mackey; citation_author=CJJ Stoeckert; citation_author=DS Roos; citation_journal_title=Nucleic Acids Research; citation_volume=34; citation_number=42; citation_pages=D363-368; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An efficient algorithm for large-scale detection of protein families.; citation_author=AJ Enright; citation_author=S Van Dongen; citation_author=CA Ouzounis; citation_journal_title=Nucleic Acids Research; citation_volume=30; citation_number=43; citation_pages=1575-1584; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=OrthoMCL: Identification of ortholog groups for eukaryotic genomes.; citation_author=L Li; citation_author=CJJ Stoeckert; citation_author=DS Ross; citation_journal_title=Genome Research 13:; citation_volume=2178; citation_number=44; citation_pages=2189; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Assessing performance of orthology detection strategies applied to eukaryotic Q3 genomes.; citation_author=F Chen; citation_author=AJ Mackey; citation_author=JK Vermunt; citation_author=DS Roos; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=45; citation_pages=e383; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Ainsworth and Bisby's dictionary of the fungi: 9th edition.; citation_author=PM Kirk; citation_author=PF Cannon; citation_author=JC David; citation_author=JA Stalpers; citation_number=46; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Fungal Genome Initiative and Lessons Learned from Genome Sequencing.; citation_author=CA Cuomo; citation_author=BW Birren; citation_author=W Jonathan; citation_author=Christine Guthrie; citation_author=RF Gerald; citation_number=47; citation_pages=833-855; citation_date=2010; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=The next generation becomes the now generation.; citation_author=DA Martinez; citation_author=MA Nelson; citation_journal_title=PLoS Genetics; citation_volume=6; citation_number=48; citation_pages=e1000906; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Designing and optimizing comparative anchor primers for comparative gene mapping and phylogenetic inference.; citation_author=WJ Murphy; citation_author=SJ O'Brien; citation_journal_title=Nature Protocols; citation_volume=2; citation_number=49; citation_pages=3022-3030; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Nucleic acids II: The polymerase chain reaction.; citation_author=S Palumbi; citation_number=50; citation_date=1996; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=On the best evolutionary rate for phylogenetic analysis.; citation_author=Z Yang; citation_journal_title=Systematic Biology; citation_volume=47; citation_number=51; citation_pages=125-133; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=An evaluation of phylogenetic informativeness profiles and the molecular phylogeny of Diplazontinae (Hymenoptera, Ichneumonidae).; citation_author=S Klopfstein; citation_author=C Kropf; citation_author=DLJ Quicke; citation_journal_title=Systematic Biology; citation_volume=59; citation_number=52; citation_pages=226-241; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Profiling phylogenetic informativeness.; citation_author=JP Townsend; citation_journal_title=Systematic Biology; citation_volume=56; citation_number=53; citation_pages=222-231; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Heterotachy and long-branch attraction in phylogenetics.; citation_author=H Philippe; citation_author=Y Zhou; citation_author=H Brinkmann; citation_author=N Rodrigue; citation_author=F Delsuc; citation_journal_title=BMC Evolutionary Biology; citation_volume=5; citation_number=54; citation_pages=50; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Noise.; citation_author=JW Wenzel; citation_author=ME Siddall; citation_journal_title=Cladistics; citation_volume=15; citation_number=55; citation_pages=51-64; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evaluating the phylogenetic utility of genes: a search for genes informative about deep divergences among vertebrates.; citation_author=A Graybeal; citation_journal_title=Systematic Biology; citation_volume=43; citation_number=56; citation_pages=174-193; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=SlowFaster, a user-friendly program for slow-fast analysis and its application on phylogeny of Blastocystis.; citation_author=M Kostka; citation_author=M Uzlikova; citation_author=I Cepicka; citation_author=J Flegr; citation_journal_title=BMC Bioinformatics; citation_volume=9; citation_number=57; citation_pages=341; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Suppression of long-branch attraction artefacts in the animal phylogeny using a site-heterogeneous model.; citation_author=N Lartillot; citation_author=H Brinkmann; citation_author=H Philippe; citation_journal_title=BMC Evolutionary Biology; citation_volume=7; citation_number=58; citation_pages=S4; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The root of the mammalian tree inferred from whole mitochondrial genomes.; citation_author=MJ Phillips; citation_author=D Penny; citation_journal_title=Molecular Phylogenetics and Evolution; citation_volume=28; citation_number=59; citation_pages=171-185; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Nuclear intron sequences for phylogenetics of closely related mammals: an example using the phylogeny of Mus.; citation_author=R DeBry; citation_author=S Seshadri; citation_journal_title=Journal of Mammalogy; citation_volume=82; citation_number=60; citation_pages=280-288; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Contrasting population structure from nuclear intron sequences and mtDNA of humpback whales.; citation_author=SR Palumbi; citation_author= Baker; citation_author= CS; citation_journal_title=Molecular Biology and Evolution; citation_volume=11; citation_number=61; citation_pages=426-435; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Screening for intron-length polymorphisms in penaeid shrimps using exon-primed intron-crossing (EPIC)-PCR.; citation_author=N Bierne; citation_author=SA Lehnert; citation_author=E Bédier; citation_author=F Bonhomme; citation_author=SS Moore; citation_journal_title=Molecular Ecology; citation_volume=9; citation_number=62; citation_pages=233-235; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Patterns of intron gain and loss in Fungi.; citation_author=CB Nielsen; citation_author=B Friedman; citation_author=B Birren; citation_author=CB Burge; citation_author=JE Galagan; citation_journal_title=PLoS Biology; citation_volume=2; citation_number=63; citation_pages=e422; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Remarkable interkingdom conservation of intron positions and massive, lineage-specific intron loss and gain in eukaryotic evolution.; citation_author=IB Rogozin; citation_author=YI Wolf; citation_author=AV Sorokin; citation_author=BG Mirkin; citation_author=EV Koonin; citation_journal_title=Current Biology; citation_volume=13; citation_number=64; citation_pages=1512-1517; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evidence of mRNA-mediated intron loss in the human-pathogenic fungus Cryptococcus neoformans.; citation_author=JE Stajich; citation_author=FS Dietrich; citation_journal_title=Eukaryotic Cell; citation_volume=5; citation_number=65; citation_pages=789-793; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cryptic speciation in Hymenoscyphus albidus.; citation_author=V Queloz; citation_author=CR Grünig; citation_author=R Berndt; citation_author=T Kowalski; citation_author=TN Sieber; citation_number=66; citation_date=2010; citation_publisher=Forest Pathology; " />
      <meta name="citation_reference" content="citation_title=Chalara fraxinea is an invasive pathogen in France.; citation_author=C Husson; citation_author=B Scala; citation_author=O Caël; citation_author=P Frey; citation_author=N Feau; citation_number=67; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Promotion and regulation of ribosomal transcription in eukaryotes by RNA polymerase I.; citation_author=T Moss; citation_author=V Stefanovsky; citation_journal_title=Progress in Nucleic Acid Research Molecular Biology; citation_volume=50; citation_number=68; citation_pages=25-66; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Gene sequencing: The Race for the $1000 Genome.; citation_author=RF Service; citation_journal_title=Science; citation_volume=311; citation_number=69; citation_pages=1544-1546; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequencing's new race.; citation_author=E Podolak; citation_journal_title=BioTechniques; citation_volume=48; citation_number=70; citation_pages=105-111; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput.; citation_author=R Edgar; citation_journal_title=Nucleic Acids Research; citation_volume=32; citation_number=71; citation_pages=1792-1797; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CODEHOP (COnsensus-DEgenerate hybrid oligonucleotide primer) PCR primer design.; citation_author=TM Rose; citation_author=JG Henikoff; citation_author=S Henikoff; citation_journal_title=Nucleic Acids Research; citation_volume=31; citation_number=72; citation_pages=3763-3766; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=New insights into the identity and origin of the causal agent of oak powdery mildew in Europe.; citation_author=A Mougou; citation_author=C Dutech; citation_author=M-L Desprez-Loustau; citation_journal_title=Forest Pathology; citation_volume=38; citation_number=73; citation_pages=275-287; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Interspecific and intraspecific diversity in oak powdery mildews in Europe: coevolution history and adaptation to their hosts.; citation_author=M-L Desprez-Loustau; citation_author=N Feau; citation_author=A Mougou-Hamdane; citation_author=C Dutech; citation_number=74; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Comparing the influences of ecological and evolutionary factors on the successful invasion of a fungal forest pathogen.; citation_author=M Garbelotto; citation_author=R Linzer; citation_author=G Nicolotti; citation_author=P Gonthier; citation_journal_title=Biological Invasions; citation_volume=12; citation_number=75; citation_pages=943-957; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Genetic structure of an expanding Armillaria root rot fungus (Armillaria ostoyae) population in a managed pine forest in southwestern France.; citation_author=S Prospero; citation_author=B Lung-Escarmant; citation_author=C Dutech; citation_journal_title=Molecular Ecology; citation_volume=17; citation_number=76; citation_pages=3366-3378; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Fungi evolution revisited: application of the penalized likelihood method to a Bayesian fungal phylogeny provides a new perspective on phylogenetic relationships and divergence dates of Ascomycota groups.; citation_author=A Carolina; citation_author=B Padovan; citation_author=G Sanson; citation_author=A Brunstein; citation_author=M Briones; citation_journal_title=Journal of Molecular Evolution; citation_volume=60; citation_number=77; citation_pages=726-735; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparative evolutionary histories of the fungal chitinase gene family reveal non-random size expansions and contractions due to adaptive natural selection.; citation_author=M Karlsson; citation_author=J Stenlid; citation_journal_title=Evolutionary Bioinformatics; citation_volume=4; citation_number=78; citation_pages=47-60; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Molecular taxonomy, origins and evolution of freshwater ascomycetes.; citation_author=D Vijaykrishna; citation_author=R Jeewon; citation_author=K Hyde; citation_journal_title=Fungal Diversity; citation_volume=23; citation_number=79; citation_pages=351-390; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Inheritance of DNA methylation in Coprinus cinereus.; citation_author=ME Zolan; citation_author=PJ Pukkila; citation_journal_title=Molecular and Cellular Biology; citation_volume=6; citation_number=80; citation_pages=195-200; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Primer sets developed to amplify conserved genes from filamentous Ascomycetes are useful in differentiating Fusarium species associated with conifers.; citation_author=NL Glass; citation_author=GC Donaldson; citation_journal_title=Applied and Environmental Microbiology; citation_volume=61; citation_number=81; citation_pages=1331-1340; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=MetaPIGA v2.0: maximum likelihood large phylogeny estimation using the metapopulation genetic algorithm and other stochastic heuristics.; citation_author=R Helaers; citation_author=MC Milinkovitch; citation_journal_title=BMC Bioinformatics; citation_volume=11; citation_number=82; citation_pages=379; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Multiple comparisons of Log-likelihoods with applications to phylogenetic inference.; citation_author=H Shimodaira; citation_author=M Hasegawa; citation_journal_title=Molecular Biology and Evolution; citation_volume=16; citation_number=83; citation_pages=1114-1116; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Variability of nuclear SSU-rDNA group introns within Septoria species: incongruence with host sequence phylogenies.; citation_author=N Feau; citation_author=RC Hamelin; citation_author=L Bernier; citation_journal_title=Journal of Molecular Evolution; citation_volume=64; citation_number=84; citation_pages=489-499; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0018803" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Finding Single Copy Genes Out of Sequenced Genomes for Multilocus Phylogenetics in Non-Model Fungi"/>
    <meta name="twitter:description" content="
        Historically, fungal multigene phylogenies have been reconstructed based on a small number of commonly used genes. The availability of complete fungal genomes has given rise to a new wave of model organisms that provide large number of genes potentially useful for building robust gene genealogies. Unfortunately, cross-utilization of these resources to study phylogenetic relationships in the vast majority of non-model fungi (i.e. &ldquo;orphan&rdquo; species) remains an unexamined question. To address this problem, we developed a method coupled with a program named &ldquo;PHYLORPH&rdquo; (PHYLogenetic markers for ORPHans). The method screens fungal genomic databases (107 fungal genomes fully sequenced) for single copy genes that might be easily transferable and well suited for studies at low taxonomic levels (for example, in species complexes) in non-model fungal species. To maximize the chance to target genes with informative regions, PHYLORPH displays a graphical evaluation system based on the estimation of nucleotide divergence relative to substitution type. The usefulness of this approach was tested by developing markers in four non-model groups of fungal pathogens. For each pathogen considered, 7 to 40% of the 10&ndash;15 best candidate genes proposed by PHYLORPH yielded sequencing success. Levels of polymorphism of these genes were compared with those obtained for some genes traditionally used to build fungal phylogenies (e.g. nuclear rDNA, &beta;-tubulin, &gamma;-actin, Elongation factor EF-1&alpha;). These genes were ranked among the best-performing ones and resolved accurately taxa relationships in each of the four non-model groups of fungi considered. We envision that PHYLORPH will constitute a useful tool for obtaining new and accurate phylogenetic markers to resolve relationships between closely related non-model fungal species.
      "/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0018803.g006"/>

  <meta property="og:title" content="Finding Single Copy Genes Out of Sequenced Genomes for Multilocus Phylogenetics in Non-Model Fungi" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0018803" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3327'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=256'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9493&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0018803">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0018803" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0018803&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Finding Single Copy Genes Out of Sequenced Genomes for Multilocus Phylogenetics in Non-Model Fungi
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nicolas Feau
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:nfeau@bordeaux.inra.fr">nfeau@bordeaux.inra.fr</a></p>

                <p>Affiliation:
                  INRA, UMR1202, BIOGECO (Biodiversité Gènes et Communautés), Cestas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thibaut Decourcelle, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  INRA, UMR1202, BIOGECO (Biodiversité Gènes et Communautés), Cestas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claude Husson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  INRA, Nancy Université, UMR1136 Interactions Arbres/Microorganismes (IFR110), Champenoux, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marie-Laure Desprez-Loustau, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  INRA, UMR1202, BIOGECO (Biodiversité Gènes et Communautés), Cestas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Cyril Dutech
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  INRA, UMR1202, BIOGECO (Biodiversité Gènes et Communautés), Cestas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 13, 2011</li>
    <li>DOI: 10.1371/journal.pone.0018803</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0018803" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0018803" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0018803" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0018803" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0018803-g001" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0018803-g002" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0018803-t001" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0018803-t002" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0018803-g003" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0018803-t003" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0018803-g004" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0018803-g005" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0018803-g006" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0018803">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0018803" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Historically, fungal multigene phylogenies have been reconstructed based on a small number of commonly used genes. The availability of complete fungal genomes has given rise to a new wave of model organisms that provide large number of genes potentially useful for building robust gene genealogies. Unfortunately, cross-utilization of these resources to study phylogenetic relationships in the vast majority of non-model fungi (i.e. “orphan” species) remains an unexamined question. To address this problem, we developed a method coupled with a program named “PHYLORPH” (PHYLogenetic markers for ORPHans). The method screens fungal genomic databases (107 fungal genomes fully sequenced) for single copy genes that might be easily transferable and well suited for studies at low taxonomic levels (for example, in species complexes) in non-model fungal species. To maximize the chance to target genes with informative regions, PHYLORPH displays a graphical evaluation system based on the estimation of nucleotide divergence relative to substitution type. The usefulness of this approach was tested by developing markers in four non-model groups of fungal pathogens. For each pathogen considered, 7 to 40% of the 10–15 best candidate genes proposed by PHYLORPH yielded sequencing success. Levels of polymorphism of these genes were compared with those obtained for some genes traditionally used to build fungal phylogenies (e.g. nuclear rDNA, β-tubulin, γ-actin, Elongation factor EF-1α). These genes were ranked among the best-performing ones and resolved accurately taxa relationships in each of the four non-model groups of fungi considered. We envision that PHYLORPH will constitute a useful tool for obtaining new and accurate phylogenetic markers to resolve relationships between closely related non-model fungal species.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Feau N, Decourcelle T, Husson C, Desprez-Loustau M-L, Dutech C (2011) Finding Single Copy Genes Out of Sequenced Genomes for Multilocus Phylogenetics in Non-Model Fungi. PLoS ONE 6(4):
          e18803.
            doi:10.1371/journal.pone.0018803</p><p><strong>Editor: </strong>Geraldine Butler, University College Dublin, Ireland</p><p><strong>Received:</strong> November 10, 2010; <strong>Accepted:</strong> March 18, 2011; <strong>Published:</strong> April 13, 2011</p><p><strong>Copyright:</strong> © 2011 Feau et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported by the French National Agency for Research project DRYADE project (Vulnérabilité, Milieux et Climat programme) and from the Institut National de la Recherche Agronomique [Projet innovant du département EFPA 2010]. PCR products purifications and sequencing were performed at the genotyping and sequencing facility of Bordeaux (grants from the Conseil Regional d'Aquitaine n°20030304002FA and 20040305003FA and from the European Union FEDER n°2003227). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Accurate reconstruction of timing and order of species filiations critically depends on obtaining suitable characters for phylogenetic analyses <a href="#pone.0018803-Brito1">[1]</a>. Transition from single to multi-locus phylogenies has successfully provided novel insights into this field. For example, topological congruence between multigene phylogenies is now commonly used to accurately define phylogenetic species in fungi <a href="#pone.0018803-Taylor1">[2]</a>. Multigene datasets are also useful to address questions in population history, demography and speciation. As theoretically anticipated, different genes can evolve in radically different ways resulting in gene/species trees inconsistencies. Using few genes which contain accurate information can allow the construction of robust species trees <a href="#pone.0018803-Brito1">[1]</a>, <a href="#pone.0018803-Townsend1">[3]</a>, <a href="#pone.0018803-Aguileta1">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>To date, low-level phylogenetic relationships in non-model fungi have essentially been inferred using a small number of phylogenetic markers: nuclear and mitochondrial rDNA (including the internal transcribed spacer [ITS] region of the nuclear ribosomal repeat unit largely used for taxa discrimination and subgeneric phylogenetic inference <a href="#pone.0018803-Vialle1">[5]</a>, <a href="#pone.0018803-Bridge1">[6]</a>), β-tubulin, γ-actin, Chitin synthase I and Elongation factor EF-1α <a href="#pone.0018803-Marthey1">[7]</a>, <a href="#pone.0018803-Lutzoni1">[8]</a>, <a href="#pone.0018803-Schmitt1">[9]</a>, <a href="#pone.0018803-Carbone1">[10]</a>, <a href="#pone.0018803-Einax1">[11]</a>. The popularity of these genes may be explained by historical and/or practical criteria, e.g. publication of the first “universal primers”, provided for a large range of fungal species (<a href="#pone-0018803-g001">Figure 1</a>). However, the rationale behind the selection of these genes is not always clear and using such freely available primers has already resulted in amplification and analysis of gene regions which yielded inaccurate phylogenetic signal for the subset of the desired taxonomic range (see, for example <a href="#pone.0018803-Kurtzman1">[12]</a>, <a href="#pone.0018803-Tanabe1">[13]</a>). Some of these genes may also be duplicated in several specific taxa and thus, have the potential of being phylogenetically discrepant <a href="#pone.0018803-Tanabe1">[13]</a>, <a href="#pone.0018803-Landvik1">[14]</a>, <a href="#pone.0018803-Tanabe2">[15]</a>, <a href="#pone.0018803-Ayliffe1">[16]</a>, <a href="#pone.0018803-Begerow1">[17]</a>, <a href="#pone.0018803-Fitzpatrick1">[18]</a>.</p>
<div class="figure" id="pone-0018803-g001"><div class="img"><a name="pone-0018803-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Genes most commonly used in fungal multigene phylogenies.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Data were compiled from 196 multigene phylogenies published between 1996 and 2009 retrieved in ISI Thompson Reuters using the following keywords: “fungi” OR “fungal” AND “multigene” or “multilocus” AND “phylogeny”. A, distribution of genes/number of publications; B, occurrence of the ten most commonly gene combination used among the 196 studies.</p>
<span>doi:10.1371/journal.pone.0018803.g001</span></div><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The increase in complete fungal genome sequencing has given rise to a new wave of model organisms, empowered by well-established genomic resources. Usefulness of these genomic resources for SNP or microsatellite markers development has largely been proven <a href="#pone.0018803-Kubisiak1">[19]</a>, <a href="#pone.0018803-Tsui1">[20]</a>, <a href="#pone.0018803-Lim1">[21]</a>, <a href="#pone.0018803-Xhaard1">[22]</a>. Similarly, these resources have also been successfully used to infer kingdom-wide fungal phylogenies <a href="#pone.0018803-Fitzpatrick1">[18]</a>, <a href="#pone.0018803-Wang1">[23]</a>, <a href="#pone.0018803-Kuramae1">[24]</a>, <a href="#pone.0018803-MarcetHouben1">[25]</a> and occasionally to study relationships between close species (e.g. 69 markers available across 12 Saccharomycotina species; <a href="#pone.0018803-MarcetHouben1">[25]</a>). <em>Ad-hoc</em> phylogenetic informative markers from protein-coding loci can be obtained with these resources. Recently, a bioinformatic method named FUNYBASE was developed to identify and assess the performance of single-copy protein-coding loci for phylogenetic analyses in fungi <a href="#pone.0018803-Aguileta1">[4]</a>. This method involved automated BLAST comparisons of whole genome sequences to identify single copy protein-coding homologs (SCPCH) among 21 fungal genomes and assessed their informative value for phylogenetic reconstruction i.e. their performance in recovering a reference species tree reconstructed from concatenated protein alignments <a href="#pone.0018803-Aguileta1">[4]</a>, <a href="#pone.0018803-Marthey1">[7]</a>. Several of these genes proved to be successful to reconstruct species trees in specific fungal genera, including <em>Botrytis</em> and <em>Penicillium</em> <a href="#pone.0018803-Marthey1">[7]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The utility of these genomic resources for making phylogenetic inferences in the vast majority of non-model fungal species remains a largely unexamined question. Relatively few studies have investigated (with success) selection approaches of sequences homologous to those found in the genome sequences of fungal model species to develop PCR primer-based markers in non-model-ones <a href="#pone.0018803-Schmitt1">[9]</a>, <a href="#pone.0018803-Adams1">[26]</a>, <a href="#pone.0018803-Pringle1">[27]</a>, <a href="#pone.0018803-LpezVillavicencio1">[28]</a>. Cross utilization of genomic tools to infer phylogenies in non-model organisms requires resolution of a fundamental conflict between the need to identify genomic sequences that are conserved (largely or wholly) across many divergent taxa, and the need to identify DNA-level differences that reflect the evolutionary history at the desired taxonomic level. Identifying particular loci with such desirable properties among thousands of potential loci available in model organisms may appear somewhat impractical or at least daunting.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>We introduce a bioinformatic approach to automatically screen genomic databases (about 107 fungal genomes fully completed or in the draft assembly stage on February 2011 [see <a href="http://fungalgenomes.org/genome">http://fungalgenomes.org/genome</a>]) for SCPCH that might be well suited for phylogenetic studies, specifically at lower taxonomic level in non-model fungal species (e.g. closely related species or even within species). Ideally, good phylogenetic SCPCH will be chosen to maximize the three following criteria: i. to use single copy nuclear genes; ii. to obtain successful and reliable PCR amplifications (and subsequent DNA sequencing) for all target taxa; and iii. to maximize the number of DNA-level polymorphisms in the amplified fragment. To achieve this last point, we developed a graphical evaluation system based on the estimation of nucleotide divergence (measured by nucleotidic diversity, π) relatively to substitution type (transition/transversion ratio [Ti/Tv]) <a href="#pone.0018803-Roe1">[29]</a>. We postulated that gene regions highly sensitive to substitutions (i.e. highly polymorphic “mutation hot-spots” <a href="#pone.0018803-Galtier1">[30]</a>) could be potentially informative to resolve relationships between closely related taxa. We expect that a low %Ti (relatively to Tv) combined with a high divergence level observed between distantly related sequences would be an accurate indicator of such “mutation hot-spots”, since Tv gradually outnumbers Ti as two sequences diverge from a common ancestor.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>We implemented the bioinformatics approach in a single executable application named “PHYLORPH” (PHYLogenetic markers for ORPHan species), developed in Python language (ver. 2.6) and freely available. As test case, we applied it to the development of phylogenetic SCPCH sets in four groups of non-model fungal plant pathogens. We experimentally verified the efficiency and success of the method by i. amplifying and sequencing some of the SCPCH proposed by PHYLORPH and ii. ascertaining that the selected markers exhibited enough variation to resolve relationships within the groups of non-model fungi considered. The amplification success and levels of polymorphism in these SCPCH was compared with those obtained for a panel of genes traditionally used to build fungal phylogenies (e.g. nuclear rDNA, β-tubulin, γ-actin, Elongation factor EF-1α, Chitin-synthase). In addition, the phylogenetic performance of some of these SCPCH was ascertained by sequencing additional taxa and comparing phylogenies with ITS reference trees.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Computational testing</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The first step in the PHYLORPH method is the identification of single copy protein-coding homologs (SCPCH) by performing a blast search with a protein dataset retrieved from FUNYBASE, PHYLOME-T60 or OrthoMCL-DB against several fungal genomes (among the 107 available with full sequence; <a href="#pone.0018803.s003">Table S1</a>), close to the non-model taxa under investigation. In order to test the universality of our method, we performed 40 PHYLORPH runs, with one species dataset out of the 21 available in FUNYBASE blasted against two to five whole genomes in each run (<a href="#pone.0018803.s004">Table S2</a>).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The first objective of this test was to examine the variation in the number of SCPCH obtained relatively to the number of genomes considered. PHYLORPH proposed 152 (±30) single-copy orthologs on average over the 40 requests. This value did not significantly decrease when new genomic resources were gradually added, suggesting that as long as less than 6 genomic resources are considered, the number of SCPCH found between genomes strive for a constant value (<a href="#pone-0018803-g002">Figure 2A</a>). Interestingly, 119 SCPCH out of the 246 of FUNYBASE were systematically retained among 30 of the 40 searches. In contrast, 21 SCPCH out of the 246 were not retained in any of the 40 searches, indicating that these SCPCH were either absent or found as multiple copies in the genome sequences examined (<a href="#pone.0018803.s001">Figure S1</a>).</p>
<div class="figure" id="pone-0018803-g002"><div class="img"><a name="pone-0018803-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Number of candidate SCPCH obtained in 40 different PHYLORPH runs.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The distributions are represented with boxplot depending on the number of fungal genomic resources used (3 to 6 i.e. one species dataset from FUNYBASE and 2–5 full genome sequence; see <a href="#pone.0018803.s004">Table S2</a>) in the different runs. Boxplot representations include the median (central bar), the mean (dotted red line), the position of the upper and lower quartiles (called Q1 and Q3; central box), extremes of the data (‘‘whiskers’’) and very extreme points that can be considered as outliers (dots; points that exceed Q1+1.5IQR and Q1-1.5IQR for the upper and lower parts of the distribution, respectively, where IQR = Q3-Q1). Letters above boxplots indicate significant differences between groups as determined by a Tukey’s HSD test after significant one-way ANOVA (F and P-values on the right). “Accepted” are candidates SCPCH which contain both «mutation hot-spot» and «priming site» regions; « Rejected – no primers » are those which contain only « mutation hot-spot » region(s); « Rejected » are those with no «mutation hot-spot» nor « priming site » regions; “Total” includes all the SCPCH (“Accepted” + « Rejected – no primers » + « Rejected ») retrieved in the searches.</p>
<span>doi:10.1371/journal.pone.0018803.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The second objective aimed to test the accuracy of the sliding window analyses. Whatever the request made and the number of genome sequences used, PHYLORPH proposed a large panel of “Accepted” SCPCH i.e. candidate SCPCH which contain at least a “mutation hot-spot” region flanked by two “priming site” regions i.e. nucleotide regions sufficiently conserved for primer design (<a href="#pone-0018803-g002">Figure 2B</a>). The average number of “Accepted” SCPCH proposed was 110 (±37) and the values ranged from 26 for a request made with the 246 FUNYBASE proteins of <em>Ustilago maydis</em> on the genomes of <em>Puccinia graminis</em>, <em>P. triticina</em>, <em>Melampsora larici-populina</em>, <em>Sporobolomyces roseus</em> and <em>Cryptococcus neoformans</em> to 169 for the 246 proteins of <em>Phanerochaete chrysosporium</em> requested on <em>Coprinopsis cinereus</em> and <em>Pleurotus ostreatus</em>. We observed a significant decrease in the mean number of “Accepted” SCPCH when the number of genomic resources considered in a PHYLORPH run was increased (<a href="#pone-0018803-g002">Figure 2B</a>).</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>In addition, the relatedness between the genomes used in the request seemed to influence the number of “Accepted” SCPCH. Mean genetic diversity (measured by π; <a href="#pone.0018803-Nei1">[31]</a>) observed between the genome sequences was negatively correlated with the number of “Accepted” SCPCH (r = -0.61, P&lt;0.0001) and positively correlated with the number of SCPCH categorized into the “Rejected-no primers” folder (r = 0,74 P&lt;0.0001; data not shown). The observed decrease in the number of “Accepted” SCPCH was thus likely linked in a large part to the growing difficulty to identify “priming regions” when an increased number of (distantly related) genomic resources was considered (<a href="#pone-0018803-g002">Figure 2C</a>). In counterpart, the identification of regions of maximum nucleotidic variation (“mutation hot-spots”), did not seem to be significantly impacted by the increase of the number of genomic resources (<a href="#pone-0018803-g002">Figure 2D</a>). These “mutation hot-spot” regions were well represented, since they represented 19.4% on average of the total length of all “Accepted” SCPCH (data not shown).</p>


<h4>Experimental validation</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To assess the reliability of our method, we carried out an experimental validation by amplifying and sequencing a sample of proposed markers in four different groups of non-model fungi: <em>Erysiphe alphitoides</em> and <em>E. quercicola</em> (oak powdery mildews), North American and Eurasian lineages of <em>Heterobasidion annosum</em>, individuals of <em>Armillaria ostoyae</em> and cryptic species of the <em>Hymenoscyphus albidus</em>/<em>H. pseudoalbidus</em> (anamorph <em>Chalara fraxinea</em>) complex.</p>
<h5>PCR and sequencing success.</h5><a id="article1.body1.sec2.sec2.sec1.p1" name="article1.body1.sec2.sec2.sec1.p1"></a><p>For each group of non-model species tested, the primer pairs for 10 to 14 of the SCPCH proposed by PHYLORPH were tested for their capacity to amplify and sequence the expected DNA fragment. Based on observed sequence conservation/variation between <em>Blumeria graminis, Sclerotinia sclerotiorum</em> and <em>Botrytis cinerea</em>, PHYLORPH proposed 133 “Accepted” SCPCH for oak powdery mildews. We performed experimental validation on a panel of 12 SCPCH. After “classical” PCR with annealing at 55°C, two candidate SCPCH yielded amplification products of the expected size in <em>E. alphitoides</em>/<em>E. quercicola</em> (FUNYBASE ID MS447 and MS294). We then attempted to sequence additional SCPCH that yielded multiple bands at the first PCR stage by using PCR touch-downs or gel extraction to isolate the appropriate size band. PCR touch-downs resulted in two new amplification products among which only one (MS550) yielded readable sequencing chromatographs without mixed peaks (<a href="#pone-0018803-t001">Table 1</a>). In all except one (MS393) of the three gel extraction attempts, we were mostly unsuccessful since sequencing yielded either poor quality sequences (FG1020) or good sequence but unexpected fragment homologies (FG534). In this last case, we suspected that cross-amplification with DNA from the host had occurred since the sequencing reaction yielded products of strong homology with plants. PCR amplification efficiencies obtained for the candidates SCPCH tested on oak powdery mildews were compared with those obtained for universal primer pairs for genes commonly used in fungal phylogenies (EF-1α, Calmodulin, Chitin synthase I, γ-actin, Histone-3 and -4 and β-tubulin). Globally, the universal primer pairs did not perform better than the PHYLORPH genes, with only three out of ten yielding a single PCR fragment (<a href="#pone-0018803-t002">Table 2</a>). Morever, direct sequencing of these PCR products resulted either in jammed sequencing chromatographs (primer pairs CHS-79F/CHS-354R and Bt1a/BtMycR_Ery) or in one single readable sequencing chromatograph which exhibited high sequence similarity with a plant β-tubulin gene (primer pair Bt1a/Bt1b, best blastn hit: 89% identity with a <em>Populus trichocarpa</em> tubulin beta chain, Ac. No. XM_002330576). To assess the possibility of multiple copies of the β-tubulin gene in oak powdery mildew genomes, we performed a direct cloning of the PCR products obtained with primers Bt1a/BtMycR_ery for one <em>E. alphitoides</em> and one <em>E. quercicola</em> strain (data not show). Selective amplification and sequencing of ten clones in each strain resulted in only one allele for the <em>E. quercicola</em> strain whereas two alleles (81% identity) were obtained for <em>E. alphitoides</em>, indicating the possible occurrence of at least two divergent copies for the β-tubulin gene in this last species or contamination with material from other fungi as oak powdery mildew hyperparasites <a href="#pone.0018803-Kiss1">[32]</a>.</p>
<div class="figure" id="pone-0018803-t001"><div class="img"><a name="pone-0018803-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0018803.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0018803.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>PCR and sequencing results for the SCPCH candidates tested across the four non-model systems used as test cases.</span></strong></p><span>doi:10.1371/journal.pone.0018803.t001</span></div><div class="figure" id="pone-0018803-t002"><div class="img"><a name="pone-0018803-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0018803.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0018803.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>PCR and sequencing efficiencies on oak powdery mildews, <em>Erysiphe alphitoides</em> and <em>E. quercicola</em>, obtained with fungal primers developed in Carbone and Kohn (1999), Glass and Donaldson (1995) and Einax and Voigt (2003).</span></strong></p><span>doi:10.1371/journal.pone.0018803.t002</span></div><a id="article1.body1.sec2.sec2.sec1.p2" name="article1.body1.sec2.sec2.sec1.p2"></a><p>For <em>H. annosum</em> and <em>Armillaria ostoyae</em>, PHYLORPH supplied 31 “Accepted” SCPCH among which 10 were experimentally tested (<a href="#pone-0018803-t001">Table 1</a>). For <em>H. annosum</em>, four candidates (FG821, FG909, FG975 and FG533) yielded PCR and sequencing success. Sequencing of one candidate (FG543) was correctly achieved for <em>A. ostoyae</em> although three had resulted in PCR success, each with one single product amplified with the PCR touch-down protocol. The PCR fragment lengths and the sequences obtained for the two other candidates (FG649 and FG756) were not homologous to the expected SCPCHs.</p>
<a id="article1.body1.sec2.sec2.sec1.p3" name="article1.body1.sec2.sec2.sec1.p3"></a><p>Finally, for <em>H. albidus</em>/<em>H. pseudoalbidus</em>, the search made with the 246 protein sequences of <em>S. sclerotiorum</em> on the genomic sequences of five fungi included within Leotiomycetes or Sordariomycetes classes, resulted in 76 “Accepted” SCPCH. Only one candidate was validated out of the 14 tested with 21 primer pairs (<a href="#pone-0018803-t001">Table 1</a>).</p>

<h5>Polymorphism levels.</h5><a id="article1.body1.sec2.sec2.sec2.p1" name="article1.body1.sec2.sec2.sec2.p1"></a><p>The DNA sequences obtained were then evaluated for polymorphism and their ability to resolve taxa relationships in the four non-model species groups was studied. For each of the four non-model systems considered, the SCPCH sequences contained enough variation to resolve taxa relationships. In the <em>E. alphitoides</em>/<em>E. quercicola, H. albidus</em>/<em>H. pseudoalbidus</em> and <em>H. annosum</em> groups, inter-specific (or inter-types for <em>H. annosum</em>) polymorphisms ranged from 0.8 to 5.2%, depending on the SCPCH considered (<a href="#pone-0018803-t003">Table 3</a>). In <em>A. ostoyae</em> an intra-specific polymorphism level of 2.6% (eight SNPs) was found for the gene FG543 (<a href="#pone-0018803-t003">Table 3</a>). To compare these levels with those obtained for loci commonly used in multigene phylogenies, we retrieved 14 sequences datasets from the NCBI database (<a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) representing eight different loci (IGS1, Elongation factor EF-1α, γ-actin, ITS, RPBII, GPDH, ATP6 and MR1; <a href="#pone-0018803-g003">Figure 3</a>). These datasets were manually curated to retain only sequences that corresponded to the divergence levels considered in the four experimental validations (i.e. <em>E. alphitoides</em>/<em>E. quercicola</em> in different populations in France, <em>H. albidus</em>/<em>H. pseudoalbidus</em> for different populations from France, NA vs. EU types in <em>H. annosum</em> and at a regional scale from different populations in <em>A. ostoyae</em>). In each non-model system, the PHYLORPH genes were generally ranked among the best-performing ones in term of nucleotidic diversity. Systematically, these SCPCH showed higher nucleotidic diversity than the loci expected to show extensive nucleotide variation as the ITS regions <a href="#pone.0018803-lvarez1">[33]</a> or mitochondrial genes (rDNA and ATP6) for <em>H. annosum</em> <a href="#pone.0018803-Bruns1">[34]</a>, <a href="#pone.0018803-Kretzer1">[35]</a> (<a href="#pone-0018803-g003">Figure 3</a>). In all except one case (FG543 in <em>A. ostoyae</em>), polymorphism levels were on average one and a half higher in predicted “mutation hot-spot” regions than in the full SCPCH length (<a href="#pone-0018803-t003">Table 3</a>). An average polymorphism value of 5.7% was found in “hot-spot” regions, whereas this value was 3.75% on average for the full SCPCH sequence. Remarkably, 29.4% (ten out of 34) of the SNPs found in MS294 for oak powdery mildews were located in “mutation hot-spots” even though these regions represented only 15.2% (100 nt) of the SCPCH sequenced part.</p>
<div class="figure" id="pone-0018803-g003"><div class="img"><a name="pone-0018803-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Nucleotidic diversity levels obtained for the PHYLORPH genes.</span></strong></p><a id="article1.body1.sec2.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.sec2.fig1.caption1.p1"></a><p>The nucleotidic diversity levels obtained for the PHYLORPH genes (identified on the abscissa with their FUNYBASE ID) in the four non-model system considered are compared with those obtained for loci commonly used in multigene phylogenies for the same systems (retrieved from the NCBI database: Actin, γ-actin gene; ATP6, mitochondrial ATP synthase subunit 6 gene; Cal, Calmodulin gene; EF-1α, Elongation Factor-α1 gene; GPDH, Glyceraldehyde-3-phosphate dehydrogenase; IGS, Intergenic spacers I of the nuclear rDNA; ITS, internal transcribed spacer of the nuclear rDNA; MR1, MRI, portion of two non-homologous mitochondrial rDNA insertions in the ML5–ML6 region; RPBII, largets subunit of the RNA polymerase II gene). In plain grey, PHYLORPH genes; In dashed grey, loci commonly used in multigene phylogenies. Letters above columns indicate significant differences between nucleotidic diversity means as determined by a Tukey’s HSD test after significant one-way ANOVA.</p>
<span>doi:10.1371/journal.pone.0018803.g003</span></div><div class="figure" id="pone-0018803-t003"><div class="img"><a name="pone-0018803-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0018803.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0018803.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Polymorphism results for the SCPCH candidates tested across the fungi panel.</span></strong></p><span>doi:10.1371/journal.pone.0018803.t003</span></div><a id="article1.body1.sec2.sec2.sec2.p2" name="article1.body1.sec2.sec2.sec2.p2"></a><p>A total of 11 putative introns were predicted in the five SCPCH successfully amplified for <em>A. ostoyae</em> and <em>H. annosum.</em> After sequencing, ten of the predicted introns were retrieved; one out of the three introns predicted in FG533 was not retrieved in sequences obtained for <em>H. annosum</em> at this locus. As expected for protein-coding sequences, intron regions generally contained more polymorphisms than exons; indel polymorphisms (one to four nucleotides long) were exclusively found in these regions (<a href="#pone-0018803-t003">Table 3</a>).</p>

<h5>Phylogenetic performance of SCPCH.</h5><a id="article1.body1.sec2.sec2.sec3.p1" name="article1.body1.sec2.sec2.sec3.p1"></a><p>In this third test, we verified the phylogenetic performance of the SCPCH obtained for oak powdery mildews and <em>H. albidus</em>/<em>H. pseudoalbidus</em> by sequencing these genes in additional related species, reconstructing phylogenies and assessing their congruence and resolution with respect to ITS reference trees.</p>
<a id="article1.body1.sec2.sec2.sec3.p2" name="article1.body1.sec2.sec2.sec3.p2"></a><p>The four SCPCH obtained for oak powdery mildews were amplified/sequenced in seven additional powdery mildew species (<a href="#pone-0018803-g004">Figure 4</a>). The four genes were successfully amplified for the different species, except MS393 in <em>Oïdium neolycopersici.</em> Although these genes and ITS evolved at similar rates (1.22 to 1.38 steps/site vs. 1.39 steps/site respectively), and exhibited equal levels of homoplasy (CI = 0.81 to 0.91 for the PHYLORPH genes vs. CI = 0.82 for ITS), the phylogenetic content of SCPCH appeared slightly higher than ITS. MP (maximum of parsimony) heuristic searches resulted in one or two trees, whereas four equal trees were found for ITS. The majority rule consensus tree obtained in MP and ML (maximum of likelihood) for ITS appeared unresolved and weakly supported with values of node confidence which never exceeded 70%. In contrast, an enhanced phylogenetic resolution was observed for the trees obtained with the PHYLORPH SCPCH which showed three (MS550) to seven (MS447) nodes supported with bootstrap values ≥70%. Interestingly, each gene resolved a monophyletic group for the oak powdery mildew species complex (bootstrap support varying from 79 to 100%), which was weakly supported in the ITS tree (61% in the MP tree). Only one topology incongruence implicating the <em>Erysiphe convolvuli</em> and <em>E. cruciferarum</em> taxa was found between ITS/MS550 and MS294/MS393/MS447 trees. WSR-tests in MP and SH-tests in ML of alternative constraint hypotheses (i.e. <em>E. convolvuli</em>/<em>E. cruciferarum</em> clustered in an exclusive group for MS294, MS393 and MS447 and <em>E. convolvuli</em>/<em>E. cruciferarum</em> not clustered for ITS and MS550) indicated that this conflict was not significant. The combined ML analysis of the ITS, MS294, MS393, MS447 and MS550 supported the resolution of one monophyletic clade for <em>E. convolvuli</em>/<em>E. cruciferarum</em> (Bootstrap support = 80%).</p>
<div class="figure" id="pone-0018803-g004"><div class="img"><a name="pone-0018803-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Phylogenetic comparisons between the ITS and the PHYLORPH genes obtained for oak powdery mildews and five related species.</span></strong></p><a id="article1.body1.sec2.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec2.sec3.fig1.caption1.p1"></a><p>(A) Bootstrap values ≥60% are given above each node as follows: MP bootstrap value/ML bootstrap value. Boxes with arrows indicate the constraint nodes applied to the different datasets and results of both WSR- and SH-tests following constrained and unconstrained tree comparisons. Table (B) contains alignment lengths, summary tree statistics issued from MP analyses, and ML models; PI, Parsimony informative sites; CI, consistency index; RC, Rescaled consistency index.</p>
<span>doi:10.1371/journal.pone.0018803.g004</span></div><a id="article1.body1.sec2.sec2.sec3.p3" name="article1.body1.sec2.sec2.sec3.p3"></a><p>The FG740 SCPCH was sequenced in four species related to <em>H. albidus</em>/<em>H. pseudoalbidus</em> (<a href="#pone-0018803-g005">Figure 5</a>). Both the FG740 MP and ML trees were fully resolved (five nodes vs. four for ITS) with strong node confidence values up to 85% (<a href="#pone-0018803-g005">Figure 5</a>). A topological conflict implicating <em>Grosmannia clavigera</em> was detected between the ITS and FG740 trees. Monophyly of <em>G. clavigera</em>/<em>C. platani</em>/<em>C. paradoxa</em> was not significantly rejected by the ITS dataset. In contrast, the basal position of <em>G. clavigera</em> was rejected in MP by the FG740 dataset (P = 0.008). Interestingly, the ITS dataset exhibited significant heterogeneity in base composition among taxa (Chi-2 = 70.29, P&lt;0.0001). The high GC frequencies observed in ITS for G. <em>clavigera</em> (GC content = 64%) and <em>M. oryzae</em> (54%) relatively to other taxa (43 to 51%) have likely tended to group these unrelated species, explaining the unexpected basal placement of <em>G. clavigera</em> <a href="#pone.0018803-Mooers1">[36]</a>. Finaly, the topology of the FG740 gene was consistent with classification of <em>G. clavigera</em>, <em>C. platani</em> and <em>C. paradoxa</em> within the Ophiostomatales order <a href="#pone.0018803-Zipfel1">[37]</a>.</p>
<div class="figure" id="pone-0018803-g005"><div class="img"><a name="pone-0018803-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Phylogenetic comparison between the ITS and the PHYLORPH gene (FG740) obtained for <em>H. albidus</em>/<em>H. pseudoalbidus</em> and 6 related species.</span></strong></p><a id="article1.body1.sec2.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec2.sec3.fig2.caption1.p1"></a><p>(A) Bootstrap values ≥60% are given above each node as follows: MP bootstrap value/ML bootstrap value. Boxes with arrows indicate the constraint nodes applied to the different datasets and results of both WSR- and SH-tests following constrained and unconstrained tree comparisons. Table (B) contains alignment lengths, summary tree statistics issued from MP analyses, and ML models; PI, Parsimony informative sites; CI, consistency index; RC, Rescaled consistency index. The asterisk indicates a significant difference in base frequencies across taxa (Chi-2 = 70.29, df = 18, P&lt;0.0001).</p>
<span>doi:10.1371/journal.pone.0018803.g005</span></div>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our results indicate that the isolation of numerous SCPCH in several fungal species with or without a fully sequenced genome should be rapid and efficient in nearly any Ascomycota and Basidiomycota fungus. The large number of new candidates extracted using this method should help to resolve the inherent problems associated with phylogenetic inferences made from single gene trees and limitations of the most commonly loci sequenced to reconstruct multigene phylogenies <a href="#pone.0018803-Brito1">[1]</a>. Few studies have previously examined similar approaches, by comparing genomic resources from related species to develop phylogenetically informative markers in non-model species (reviewed in <a href="#pone.0018803-Thomson1">[38]</a>). For the fungal kingdom, Aguileta et al. <a href="#pone.0018803-Aguileta1">[4]</a> initially evaluated the phylogenetic performance of single copy protein-coding homologs (SCPCH) retrieved among 21 genomes with the goal of providing a reliable resource of orthologous gene families (246 SCPCH regrouped in the FUNYBASE database <a href="#pone.0018803-Marthey1">[7]</a>) to perform phylogenetic analyses. With PHYLORPH, we aimed to help researchers not specifically interested in genomics and bioinformatics to gain genetic information on their studied species or group of species. Indeed, PHYLORPH increases the interest which previously aroused the identification of the 246 FUNYBASE SCPCH by routinely expanding their use to a total of 107 fully sequenced fungal species and by demonstrating that some of these genes are relevant to resolve relationships between closely taxa. Using the 246 FUNYBASE SCPCH avoids to systematically perform the cumbersome work needed to re-extract single-copy orthologous genes among the large amount of gene models available in each fungal genome and allows to directly target well-annotated and phylogenetically informative genes <a href="#pone.0018803-Aguileta1">[4]</a>, <a href="#pone.0018803-Marthey1">[7]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The establishment of correct orthology and paralogy is crucial to infer accurate evolutionary relationships. If paralogs are inadvertently amplified, the resulting gene tree may not reflect the true species histories <a href="#pone.0018803-Townsend1">[3]</a>, <a href="#pone.0018803-Aguileta1">[4]</a>. Several orthology-prediction methods (and ensuing ortholog-databases) have emerged in recent years, among which the “pairwise comparison” and “phylogenetic” approaches remain the most popular <a href="#pone.0018803-Fang1">[39]</a>, <a href="#pone.0018803-Gabaldn1">[40]</a>, <a href="#pone.0018803-Kuzniar1">[41]</a>. These methods can be influenced by biases related to the lack of accuracy of gene predictions along genomes and the presence of multidomain proteins which can possibly generate artificial clusters of orthologs. To allow users to have access to an extended range of orthology repositories (and thus to consider a larger choice of potential markers), PHYLORPH can use the fungal proteins from the OrthoMCL-DB and PHYLOME-T60 ortholog-databases <a href="#pone.0018803-MarcetHouben1">[25]</a>, <a href="#pone.0018803-Chen1">[42]</a> as templates instead of those provided in FUNYBASE. These three ortholog databases differ in the methods followed to obtain the clusters of putatively orthologous genes. For instance, to identify the 246 single-copy orthologs of the FUNYBASE database, pairs of genes were automatically identified from complete fungal genomes using a “pairwise comparison” method (i.e. with the Bi-directional Best Hit method; BBH) and clusterized with the Tribe-MCL algorithm <a href="#pone.0018803-Enright1">[43]</a>. The BBH strategy was also used for OrthoMCL-DB but automatic detection of ortholog clusters was performed with the OrthoMCL algorithm <a href="#pone.0018803-Li1">[44]</a>, which is well-known to perform better than Tribe-MCL <a href="#pone.0018803-Chen2">[45]</a>. One-to-one orthologs regrouped into the PHYLOME-T60 database were inferred using a large scale phylogenetic approach. Such phylogeny-based method is considered more accurate than standard BBH strategies, since less prone to false positive ortholog linkages <a href="#pone.0018803-Fang1">[39]</a>, <a href="#pone.0018803-Gabaldn1">[40]</a>. Cross comparisons of these three different ortholog resources showed significant differences in the protein sets, indicating low redundancy between the databases. We expected that the reliability of databases on orthology is likely differ mainly because of the differences between the orthology predictions methods but also to differences in fungal species content and possibly asynchronous updates of the data resources <a href="#pone.0018803-Fang1">[39]</a>. Finally, these three ortholog databases represent a potential of 329 different markers (out of a 376) available among 62 fungal species (35 genera, 18 orders) (<a href="#pone.0018803.s002">Figure S2</a>).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The method of selection of single-copy genes implemented in PHYLORPH is also based on a BBH strategy, and thus could suffer of potential pitfalls of “pairwise comparison” methods. False positives are generated with BBH if the true ortholog was lost in the queried species, and the best blast hit is a paralog (i.e. mistaken for ortholog). In such a case, primers issued from PHYLORPH could either target the mistaken paralog resulting in gene/species tree discrepancies or multiple paralogous copies that will complicate amplification, sequencing and alignment. For these reasons however, primer pairs amplifying paralogs can easily be identified and discarded. Furthermore, integrating more accurate methods of orthologs prediction as those based on phylogeny will require reconstructing trees for each PHYLORPH request, and could drastically increase computation needs.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Globally, when integrating the three ortholog databases (FUNYBASE, OrthoMCL-DB and PHYLOME-T60) and the 107 full genome sequences available to date, the taxonomic coverage provided by PHYLORPH represents a potential of marker development for about 2247 known genera within the fungal kingdom (2079 within the dikarya subkingdom) <a href="#pone.0018803-Kirk1">[46]</a>. In few rare cases however, the searches conducted with our program were limited by the inability to obtain genomic alignments for a set of two or more species representatives of the studied target group. For example, the Lecanoromycetes (estimated to be &gt;13,500 species; about 90% of all described lichen-forming Ascomycota, <a href="#pone.0018803-Kirk1">[46]</a>) cannot yet be used in PHYLORPH, since no model species is fully sequenced in this class. However, we expect that fungal taxonomic coverage will shortly be increased as long as new fungal genomes are sequenced, specifically with the thrust of new sequencing technologies applied to microbial full genome sequencing <a href="#pone.0018803-Cuomo1">[47]</a>, <a href="#pone.0018803-Martinez1">[48]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Selecting efficient primers from sequenced model organisms to transfer in orphan species appears as one of the complicated step of the approach presented here. The substantial diversity in fully sequenced fungal genomes (and their availability in PHYLORPH; discussed above) provides the possibility to easily consider several genomic resources for a single search, promoting the identification of a maximum of mutations at each site (particularly, in conserved regions allowable for primer design) and thus increasing the confidence in primer design. Our computational tests however indicated, that increasing the number of genomic resources in a search can hastily reduce the number of candidate SCPCH proposed by PHYLORPH, due to the impracticability to automatically identify “primer regions” (with the default criteria implemented in the program). This problem can be increased if distantly related genomic resources are selected for the search, given that sufficiently conserved regions for primer design tend to rapidly decline. From a practical point of view, two general approaches are conceivable for designing oligonucleotide primers that are potentially useful across a broad selection of species <a href="#pone.0018803-Murphy1">[49]</a>: a non-degenerate primers strategy that take into account nucleotide mismatches that form weak or partial bonds <a href="#pone.0018803-Palumbi1">[50]</a> or alternatively, the design of degenerate primers <a href="#pone.0018803-Murphy1">[49]</a>. We considered this second approach in our experimental validation by designing partially degenerate primers in few candidates SCPCH (&lt;15) obtained from model species which diverged from the test non-model species about 200–300 Million of years ago (Mya). According to the analyses of <a href="#pone.0018803-Thomson1">[38]</a>, PCR success obtained for marker transfers are correlated with the divergence times observed between the targeted non-model species and the reference model species. The PCR success rates obtained in our experimental validation varied from 7 to 40%, accurately corroborating the PCR efficiency values expected for divergence time greater than 200 Million years <a href="#pone.0018803-Thomson1">[38]</a>. One of the main drawback of degenerate primer pairs however, is that only a fraction of the primer pool will contain perfect matches to the target sequences, resulting in low PCR amplification yields and sometimes cross amplifications of non-target DNA. In few cases of our experimental validation, the degenerated nature of the primers used, coupled with poorly-stringent PCR conditions likely resulted in such non-specific cross-amplifications. The occurrence of non-specific cross-amplifications may furthermore be enhanced in the specific case of biotrophic fungi (as powdery mildews), since fungal material obtained from these unculturable organisms can occasionally be contaminated with plant material (see for example, the two cases of our experimental validation [FG534 and β-tubulin] where DNA from the host plant was likely amplified and sequenced). Further optimisation of primer and PCR protocols could help to increase the proportion of markers that amplify the correct product across the species of interest (i.e. efficiency of the method) by avoiding these non-target amplifications <a href="#pone.0018803-Murphy1">[49]</a>. However, defining and proposing alternatives for primer design and conditions to maximize PCR amplification specificity and success was beyond the scope of the experimental validation proposed in this work. Based on the fact that each PHYLORPH run usually provides many suitable SCPCH, we rather suggest adopting a marker development strategy based on testing several primer pairs with a standard PCR protocol on a large panel of candidate SCPCH. We expect that the few polymorphisms encountered in the alignment regions dedicated to primer design by PHYLORPH will be sufficiently representative of the group of species considered in the analysis. With many candidate SCPCH available for marker design, PCR and sequencing efficiencies are no longer a limiting factor since researchers can easily remove unsuitable candidates and move-on <a href="#pone.0018803-Thomson1">[38]</a>. Furthermore, we rather recommend restricting searches made with PHYLORPH to a maximum of six genomic resources: one reference set extracted from FUNYBASE used to search the complete sequence of five related species (from the same class, as far as possible). These searches can be restricted in some cases depending on the number of whole genomes available in vicinity of the non-model species of interest. A minimum knowledge on the taxonomical position of the non-model species targeted is thus needed to allow an efficient selection of the genomic resources used in a marker search conduced with PHYLORPH. Interestingly, with these parameters, the primers obtained performed globally well when sequencing of the candidate SCPCH was extended to a larger number of taxa (test cases on species related to powdery mildews and <em>Hymenoscyphus</em> spp.). Thus, as expected, a substantial number of closely related taxa can be targeted using the primer regions provided by PHYLORPH, without requiring additional technical improvement. If required, a redefinition of primers based on alignment of the DNA sequences obtained and those used for the PHYLORPH search could help to extend this sequencing to more distantly related species.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>A second challenge for PHYLORPH was to provide phylogenetic markers (candidate SCPCH) able to resolve phylogenetic relationships at low divergence levels. To evaluate the phylogenetic potential of each SCPCH proposed, we developed a graphical system which examines patterns of nucleotide substitutions using a sliding window analysis. We assumed that combined comparison of nucleotide divergence and substitution type observed between divergent species should help to localize “mutation hot-spots” (as measured by nucleotidic diversity [π] upper to %Ti). As we expected, the evolution rate obtained in these regions appeared to be sufficiently consistent to accurately discriminate closely related taxa. The identification of such “mutation hot-spot” regions was thus an attractive approach to get the maximum information for resolution in the case of recent divergences as those observed in species complexes. It remains however important to bear in mind that these rapidly evolving nucleotide sites could quickly be submitted to multiple hits, resulting in the loss of informative characters (i.e. homoplasy or saturation). The method we proposed aims to obtain phylogenetic markers for the resolution of recent divergences, between cryptic species in complexes or within genera. At these phylogenetic levels, limited saturation effects are expected <a href="#pone.0018803-Yang1">[51]</a>. Furthermore, gene phylogenetic informativeness can be considered as a tradeoff between optimal rate of evolution and effects of saturation <a href="#pone.0018803-Yang1">[51]</a>, <a href="#pone.0018803-Klopfstein1">[52]</a>. This relationship has been extensively discussed on theoretical grounds <a href="#pone.0018803-Townsend2">[53]</a>, <a href="#pone.0018803-Philippe1">[54]</a>, <a href="#pone.0018803-Wenzel1">[55]</a> and through empirical examples <a href="#pone.0018803-Roe1">[29]</a>, <a href="#pone.0018803-Klopfstein1">[52]</a>, <a href="#pone.0018803-Graybeal1">[56]</a>. The work presented here does not intend to feed this debate, but rather to focus on practical and simple issues involved in obtaining the raw data for the desired phylogenetic analysis. Several methods including for example, character partitioning and exclusion have been proposed as means to enhance phylogenetic signal and eventually, accommodate with nucleotidic saturation <a href="#pone.0018803-Kostka1">[57]</a>, <a href="#pone.0018803-Lartillot1">[58]</a>, <a href="#pone.0018803-Phillips1">[59]</a>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Intron regions constitute an additional source of variation that can be considered when using PHYLORPH, since protein-coding sequences are targeted. Levels of variation found in introns appear adequate for use in both phylogenetic (e.g. <a href="#pone.0018803-DeBry1">[60]</a>) and population genetic (e.g. <a href="#pone.0018803-Palumbi2">[61]</a>; <a href="#pone.0018803-Bierne1">[62]</a>) studies. One of the main issues concerns the conservation of these regions (and subsequent transfer) over different taxa. Indeed, fungal species can vary in their intron distributions, from on average far less than one intron per gene for the yeasts <em>Schizosaccharomyces pombe</em> and <em>Saccharomyces cerevisiae</em> to an intron content among the highest known in eukaryotes in Zygomycota and Basidiomycota (4 to 6 per gene) <a href="#pone.0018803-Nielsen1">[63]</a>, <a href="#pone.0018803-Rogozin1">[64]</a>. Specifically, although gene structure is known to be a slowly evolving phylogenetic character in fungi, intron composition can vary between closely related species as reported among <em>Cryptococcus</em> spp. <a href="#pone.0018803-Stajich1">[65]</a>. Despite the experimental evidence that intron positions have remained conserved among the group of fungi tested in this study, correct predictions on their conservation among fungal lineage can remain hard to establish.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Whatever the sources of variation considered (mutation “hot-spots” or intron regions), enough polymorphisms were found to resolve phylogenetic relationships in the four non-model species groups studied. Three of these groups (<em>E. alphitoides</em>/<em>E. quercicola</em>, <em>H. albidus</em>/<em>H. pseudoalbidus</em> and <em>H. annosum</em>) were complexes of sibling and closely related species initially differentiated on the basis of polymorphisms found in their ITS sequences <a href="#pone.0018803-Queloz1">[66]</a>, <a href="#pone.0018803-Husson1">[67]</a>. The nine SCPCH obtained with PHYLORPH brought an additional deal of clarity in the taxonomical resolution of these complexes by providing polymorphism levels equivalent or greater than those found in the ITS region. For example, phylogenetic content of the FG740 gene was identical to those from three genes (including ITS) to recognize the existence of two distinct species, <em>H. albidus</em> and <em>H. pseudoalbidus</em>, the last one being the teleomorphic state of <em>Chalara fraxinea</em> <a href="#pone.0018803-Husson1">[67]</a>. Sequencing of this gene was extended to four additional species representing three genera (<em>Hymenoscyphus</em>, <em>Ceratocystis</em> and <em>Grosmannia</em>) and two orders (Helotiales and Ophiostomatales). The resulting dataset did not suffer from the base compositional bias observed in the ITS dataset and resulted in a well resolved tree, indicating the accuracy of this gene for phylogenetic reconstruction. In a similar way, the four SCPCH obtained for the oak powdery mildews will help to resolve phylogenetic relationships among cryptic species in this complex. Applying this criterion should be particularly useful in delimiting the sympatric lineages included within this complex <a href="#pone.0018803-Taylor1">[2]</a>. When integrated in a larger phylogeny which included seven related powdery mildew species, these four SCPCH clearly delimited the oak powdery mildews species complex and accurately improved the initial phylogeny obtained with ITS sequences. In a fourth validation case, we attempted to obtain resolution at its most elementary-level by targeting intra-specific polymorphisms within the <em>A. ostoyae</em> species. One SCPCH was successfully amplified into the four isolates tested and contained intra-specific polymorphisms located either in predicted “mutation hot-spots” or intron regions. The polymorphism level obtained with this gene was upper than those found into the IGS regions known to be highly variable among (and even within) individuals of the same species (reviewed by <a href="#pone.0018803-Moss1">[68]</a>). PCR amplification and sequencing of additional isolates will help to determine if the sequence variation obtained with this marker is reliable for making population genetic studies.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Herein, we developed and evaluated an automated approach to mine fungal genomic resources and guide the selection of useful phylogenetic markers for clades of fungi in which only a few model organism genomes have been sequenced. Although we targeted a particular group of organisms (Ascomycota and Basidiomycota), our methods should be applicable to almost any group of organisms for which one or more complete nuclear genomes are available. This method thus successfully improves the toolbox of scientists interested in molecular phylogeny, phylogeography and taxonomy. Though faster and less expensive whole genome sequencing will be increasingly available in the near future <a href="#pone.0018803-Thomson1">[38]</a>, <a href="#pone.0018803-Martinez1">[48]</a>, <a href="#pone.0018803-Service1">[69]</a>, <a href="#pone.0018803-Podolak1">[70]</a>, cross-species primers as those obtained with PHYLORPH should still be useful for species for which full genome sequencing is not of interest or for which the available molecular techniques are found to be convenient to obtain the sample (biotrophic and endophytic fungi, for example). Furthermore, our marker development method only relies on standard molecular biology tools and on a simple and customized bioinformatic pipeline and is thus expected to be easily and routinely useable in any laboratory interested in a fast and cheap marker development procedure.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Overview of the method</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The procedure can be divided into three steps: Step 1 is the identification of single copy protein-coding homologs (SCPCH) by performing a tblastn search with a protein dataset retrieved from FUNYBASE against different fungal genomes, close to the non-model taxa under investigation. FUNYBASE provides a subset of 21 genomes representing the major fungal taxonomic groups across a large phylogenetic scale, among which 246 clusters of single copy proteins were identified (<a href="#pone.0018803.s003">Table S1</a>). For each of these clusters, the FUNYBASE also contains information regarding the amino-acid model that best fit the data, an annotation (when available), the mean identity percentage of the species-sequences in the cluster, the number of the variable sites and a topological score i.e. a measure for performance of each gene in estimating the phylogeny of the species included in the cluster <a href="#pone.0018803-Aguileta1">[4]</a>, <a href="#pone.0018803-Marthey1">[7]</a>. To explore other fungal orthology repositories than FUNYBASE and offer a larger choice of potential phylogenetic markers, two additional protein datasets (PHYLOME-T60 and OrthoMCL-DB) are available for performing the identification of SCPCH throught tblastn searches. PHYLOME-T60 and OrthoMCL-DB contains 69 and 61 one-to-one protein orthologs across a range of 61 and 18 fungal species, respectively (<a href="#pone.0018803.s003">Table S1</a>). Step 2 consists in recovering and aligning nucleotidic sequences of the SCPCH selected in step 1. In step 3, a sliding window analysis is carried out on each alignment to identify the best candidates i.e. with i. appropriate evolutionary rate for being potentially useful in phylogenetic analyses among close taxa and ii. conserved sequence regions to allow development of degenerate primers. Candidate SCPCHs are categorized according to their content in priming sites flanking phylogenetic informative regions. These three steps are automated in a Python (ver. 2.6) application named PHYLORPH and controlled with a graphical user interface (GUI) developed using the WxPython module. <a href="#pone-0018803-g006">Figure 6</a> gives a schematic overview and an explanatory example of the entire nucleotide protein coding loci discovery procedure. The executable version (2.0) of PHYLORPH can be downloaded at the following address: <a href="https://www4.bordeaux-aquitaine.inra.fr/biogeco_eng/Resources/Softwares/PHYLORPH">https://www4.bordeaux-aquitaine.inra.fr/​biogeco_eng/Resources/Softwares/PHYLORPH</a>.</p>
<div class="figure" id="pone-0018803-g006"><div class="img"><a name="pone-0018803-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0018803" data-uri="info:doi/10.1371/journal.pone.0018803.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0018803.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0018803.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0018803.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0018803.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Overview of the bioinformatic procedure implemented in PHYLORPH.</span></strong></p><a id="article1.body1.sec4.sec1.fig1.caption1.p1" name="article1.body1.sec4.sec1.fig1.caption1.p1"></a><p>An explanatory example is presented with the request made for <em>Armillaria ostoyae</em> and <em>Heterobasidion annosum</em>.</p>
<span>doi:10.1371/journal.pone.0018803.g006</span></div><h5>Step 1: identification of single-copy orthologs.</h5><a id="article1.body1.sec4.sec1.sec1.p1" name="article1.body1.sec4.sec1.sec1.p1"></a><p>In order to increase the probability to obtain suitable DNA amplification of the targeted marker, the philosophy underlying PHYLORPH is to target the most closely related genomic resources available (same class or order, as possible) to the non-model species targeted. The PHYLORPH GUI allows the user to select one species protein dataset out of those available in FUNYBASE, OrthoMCL-DB or PHYLOME-T60 to BLAST (tblastn) against 2–5 genome datasets among a panel of 107 fully sequenced genomes (representing 28 orders among 14 classes within the fungal tree of life; <a href="#pone.0018803.s003">Table S1</a>). Upon user request (selection of one protein dataset and at least two full genome sequences located in the same class or order than the non-model species targeted), genomes are automatically downloaded from various sources (NCBI, JGI, BROAD and others; <a href="#pone.0018803.s003">Table S1</a>), formatted for tblastn searches and interrogated for similarities with the protein sequences of the species dataset selected. Tblastn results were considered non-spurious if they met three criteria (identical to those employed in FUNYBASE): i. coverage of at least 70% of the query protein, ii. identity of at least 30% and iii. e-value cutoff of 1e-20. Furthermore, these tblastn results are filtered-out for hits with potential paralogous copies. This step is automatically accomplished by simply discarding any SCPCH that significantly matched more than one genome location.</p>

<h5>Step 2: alignment of nucleotide sequences.</h5><a id="article1.body1.sec4.sec1.sec2.p1" name="article1.body1.sec4.sec1.sec2.p1"></a><p>Phase 2 aims to provide a nucleotidic alignment for each SCPCH retained up to this point. First, the nucleotide-level records associated with the protein sequences of each SCPCH are automatically extracted from the previously selected genomes. Second, the files obtained are automatically aligned using the MUSCLE (version 3.7) algorithm <a href="#pone.0018803-Edgar1">[71]</a>. Since the tblastn searches (i.e. search translated nucleotide database using a protein query) performed in phase 1 to identify SCPCH (Step 1) were against translated genomic sequences, any continuous run of protein sequence that scored a significant match had to represent an individual exon or portion thereof. This information coupled together with MUSCLE gap-opening parameter set at −1000, allowed accurate alignments of intron-exon boundaries.</p>

<h5>Step 3: sliding windows.</h5><a id="article1.body1.sec4.sec1.sec3.p1" name="article1.body1.sec4.sec1.sec3.p1"></a><p>Patterns of nucleotide substitutions are then examined across each SCPCH alignment using a sliding window analysis. A 20 bp window with a 5 bp-step was chosen (default parameters), which improved visualization of nucleotide changes along the alignment independently of alignment length. Mean nucleotidic diversity (π; <a href="#pone.0018803-Nei1">[31]</a>) and percent transitions (%Ti) are graphed together to identify potential “mutation hot-spots”. These “mutation hot-spots” i.e. optimal regions for phylogenetic analysis among closely related taxa levels are identified as alignment regions which have a nucleotidic diversity greater than the %Ti <a href="#pone.0018803-Roe1">[29]</a>. We used %Ti rather than the more standard Ti/Tv ratio, to eliminate the problem of undefined values when transversions equal zero. In an attempt to compensate for possible differences in base compositions among genes (and allow %Ti to be comparable), each mutation type is divided by the count of its original base (e.g. a substitution from ‘A’ to ‘T’ divided by the number of ‘A’). This normalization of the Ti/Tv ratio is proposed as a default option of PHYLORPH. When at least one “mutation hot-spot” is detected, conserved alignment blocks suitable for primer development are identified using a method inspired from the strategy implemented in CODEHOP <a href="#pone.0018803-Rose1">[72]</a>: the DNA alignment is scanned in the 3’–5’ direction to identify a region of five nucleotides which starts on an invariant 3’ position and which has at most one SNP, followed by a region of five nucleotides with at most two SNPs and finally, a region of eight nucleotides with at most three SNPs. Distance parameter between forward and reverse priming sites can be adjusted by the user (default: 400 nt). Finally, SCPCH alignments (and graphs) that do not contain at least one “mutation hot-spot” region are automatically moved in a folder named “Rejected”. A second folder named “Rejected – no primers” holds those which contain at least one hot spot region but no conserved alignment blocks suitable for primer development. All the other alignments which contain at least one “hot spot” region flanked by two conserved alignment blocks are moved in a third folder named “Accepted”. Candidate SCPCH enclosed into the “Accepted” folder are listed onto an html window and saved in a tabulation-delimited file. These results are provided in a tabular view: the columns headers of the table are ID of the gene candidates, links towards fasta format alignment and divergence profile, alignment length, GC content, introns number and functional annotation of the candidate.</p>



<h4>Computational testing</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We performed a total of 40 different PHYLORPH runs, including those made for the experimental validation as a test case. For each run, the approach previously described was strictly respected: one species dataset out of the 21 available in FUNYBASE was randomly selected and used to interrogate two to five (depending on the run; see <a href="#pone.0018803.s004">Table S2</a>) closely related genomic resources available in PHYLORPH (same class or order, as possible). We limited these tests to a maximum of six genomic resources (i.e. one dataset from FUNYBASE and five full genome sequences) to save computational time and avoid making searches between increasingly distantly related species which could increase potential for sequence mis-alignements. For each run, the number of SCPCH found following searches of the 246 FUNYBASE proteins on the selected genome sequences (Step 1) was systematically registered in a log file. Similarly, the numbers of candidate SCPCH moved either in the i. “Accepted” or ii. “Rejected – no primers” or iii. “Rejected” folders (see the criteria previously described in Step 3) were also automatically registered in the same file. Finally, all these different results were compared relatively to the number of genomic resources considered in the different PHYLORPH runs.</p>


<h4>Experimental validation</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The accuracy of PHYLORPH to find polymorphic markers was tested with a set of four groups of non-model fungi representing 4 different orders among Ascomycota and Basidiomycota. We tested the efficiency of the programme to successfully provide candidate SCPCH that could be readily amplified and sequenced and that showed substantial informative variation across i. two cryptic ITS lineages related to <em>Erysiphe alphitoides</em> and <em>E. quercicola</em> and included within the oak powdery mildew complex <a href="#pone.0018803-Mougou1">[73]</a>, <a href="#pone.0018803-DesprezLoustau1">[74]</a>, ii. the North American and Eurasian lineages of the root- and heart-rot forest pathogen <em>Heterobasidion annosum</em> <a href="#pone.0018803-Garbelotto1">[75]</a>, iii. individuals of <em>Armillaria ostoyae</em> causing a root rot disease on Pines <a href="#pone.0018803-Prospero1">[76]</a> and, iv. cryptic species of the <em>Hymenoscyphus albidus</em>/<em>H. pseudoalbidus</em> (anamorph <em>Chalara fraxinea</em>) complex, responsible of an ash decline in Europe <a href="#pone.0018803-Queloz1">[66]</a>, <a href="#pone.0018803-Husson1">[67]</a>.</p>
<h5>PHYLORPH runs.</h5><a id="article1.body1.sec4.sec3.sec1.p1" name="article1.body1.sec4.sec3.sec1.p1"></a><p>For oak powdery mildews, we requested PHYLORPH to search the complete genomes sequences of <em>Blumeria graminis</em> and <em>Botrytis cinerea</em> for SCPCH homologous to the 246 protein sequences of <em>Sclerotinia sclerotiorum</em> i.e. the closest model species to <em>Erysiphe</em> spp. available in FUNYBASE (divergence time about 300 Million of years [My] between Erysiphales (<em>Erysiphe</em> spp. and <em>B. graminis</em>) and Helotiales [<em>S. sclerotiorum</em> and <em>B. cinerea</em>]; <a href="#pone.0018803-Carolina1">[77]</a>). Similarly, the 246 protein sequences of <em>Phanerochaete chrysosporium</em> were used to search for homologues in the full genome sequences of <em>Coprinopsis cinereus</em>, <em>Laccaria bicolor</em>, <em>Pleurotus ostreatus, Postia placenta</em> and <em>Schizophyllum commune,</em> to propose a set of candidate SCPCH for <em>H. annosum.</em> This request had been made few months before the first public release of the full genome sequence of this fungus in June 2009 (see at <a href="http://genome.jgi-psf.org/Hetan1/Hetan1.info.html">http://genome.jgi-psf.org/Hetan1/Hetan1.​info.html</a>). The same six genomic resources were also used to run a search for <em>A. ostoyae</em> (divergence time about 230 to 240 My between Polyporales [<em>P. chrysosporium</em> and <em>P. placenta</em>] and Agaricales [others, including <em>A. ostoyae</em> and <em>H. annosum</em>]; <a href="#pone.0018803-Karlsson1">[78]</a>). Finally, to find SCPCH candidates for the <em>H. albidus</em>/<em>H. pseudoalbidus</em> complex, we used the single copy genes set from <em>S. sclerotiorum</em> and the genomic sequences of <em>B. graminis</em>, <em>B. cinerea</em>, <em>Magnaporthe grisea</em>, <em>Neurospora crassa</em> and <em>Fusarium graminearum</em> (divergence time about 350–500 My between Leotiomycetes [<em>Hymenoscyphus</em> spp., <em>S. sclerotiorum</em>, <em>B. cinerea</em> and <em>B. graminis</em>] and Sordariomycetes <a href="#pone.0018803-Vijaykrishna1">[79]</a>). Following each PHYLORPH run, 10 to 14 SCPCH retained into the “Accepted” folder (see Step 3) were selected for primer designs (one or two pairs) based on the topological scores available from the FUNYBASE database. In an attempt to maximize polymorphism levels for <em>A. ostoyae</em> and <em>H. annosum</em>, we preferentially targeted SCPCH which included primer regions flanking the predicted locations of introns. PHYLORPH indeed provides an automatic prediction of intron/exons boundaries for each SCPCH when the spliced transcript sequence issued from the protein sequence is aligned to its closest homolog from the full genome sequences.</p>

<h5>Strains tested and DNA extractions.</h5><a id="article1.body1.sec4.sec3.sec2.p1" name="article1.body1.sec4.sec3.sec2.p1"></a><p>Twenty <em>E. alphitoides</em>/<em>E. quercicola</em> strains (10/10) were obtained from infected leaves collected in two oak regenerations (mainly <em>Q. robur</em>/<em>Q. petraea</em>) in France. Total DNA was extracted and diluted following the CTAB protocol described in <a href="#pone.0018803-Mougou1">[73]</a>. Each strain was identified as related to the <em>E. alphitoides</em> or the <em>E. quercicola</em> lineage by sequencing the internal transcribed spacers (ITS) region of the nuclear ribosomal repeat unit <a href="#pone.0018803-Mougou1">[73]</a>. For <em>H. albidus</em>/<em>H. pseudoalbidus</em> we used DNA extracted from 30 isolates collected in France described in <a href="#pone.0018803-Husson1">[67]</a>. DNA from eighteen <em>H. annosum</em> isolates collected on <em>Pinus</em> spp. for the European (15 isolates collected in France) and North-American types (one isolate from Italy) <a href="#pone.0018803-Garbelotto1">[75]</a> was extracted using the protocol of <a href="#pone.0018803-Zolan1">[80]</a>. For <em>A. ostoyae</em>, we extracted DNA as described in <a href="#pone.0018803-Prospero1">[76]</a> from four isolates collected in four <em>Pinus sylvestris</em> populations from the south-west region in France.</p>

<h5>PCR and sequencing assays.</h5><a id="article1.body1.sec4.sec3.sec3.p1" name="article1.body1.sec4.sec3.sec3.p1"></a><p>The primer pairs designed for oak powdery mildews and for <em>A. ostoyae</em> and <em>H. annosum</em> were tested with the following PCR thermal cycling protocols: initial denaturation of 94°C for 3 min, followed by 40 cycles of denaturation at 94°C (45 s), annealing at 55°C (45 s), extension at 72°C (1 min 10 s), and a final extension of 72°C (7 min). If necessary (i.e. when multiband profiles or no PCR product were obtained), PCR cycling was further optimized by assessing a 55-50°C touchdown (45 s) for 40 cycles. In both cases, PCR reactions (1x PCR buffer, 2 mM of MgCl<sub>2,</sub> 0.8 µM total dNTPs, 0.2 µM primers, 0.75U of Taq Polymerase [Sigma-Aldrich]) were carried out in a 20 µl volume. Amplified products were separated on 2% agarose gels and visualized by ethidium bromide staining. Finally, if the PCR assays yielded discrete bands of the appropriate size, PCR products were excised and cleaned using Qiaquick columns (QIAGen). Experimental validations of the candidate genes for <em>H. albidus</em>/<em>H. pseudoalbidus</em> were carried out with the same conditions that those described above, except that two different PCR cycles were used: i. a 55–50°C touchdown identical to the one previously described above and ii. an initial denaturation at 94°C for 3 min followed by 35 cycles at 95°C (30 s), a gradient at 50–60°C (30 s), 72°C (1 min 10 s), and a final extension at 72°C (7 min). In all cases, sequencing was then performed using dye labelled terminators (ABI PRISM<sup>TM</sup> BigDye<sup>TM</sup> Terminator Cycle Sequencing Ready Reaction Kit) and run on an ABI 3730 automated DNA sequencer. Sequences were edited and manually aligned using BioEdit v.7.0.5 (Hall 1999).</p>
<a id="article1.body1.sec4.sec3.sec3.p2" name="article1.body1.sec4.sec3.sec3.p2"></a><p>In addition, the amplification efficiencies obtained for the candidates SCPCH tested on oak powdery mildews were compared with those obtained for several genes commonly used to construct fungal phylogenies. The universal primer pairs (listed in <a href="#pone-0018803-t002">Table 2</a>) previously designed for the amplification of the Elongation factor EF-1α, Calmodulin, Chitin synthase I, γ-actin, Histone-3 and -4 and β-tubulin genes <a href="#pone.0018803-Carbone1">[10]</a>, <a href="#pone.0018803-Einax1">[11]</a>, <a href="#pone.0018803-Glass1">[81]</a> were tested on the <em>E. alphitoides</em>/<em>E. quercicola</em> strains using the “classical” PCR protocol described above (40 cycles with annealing temperature at 55°C).</p>

<h5>Phylogenetic performance of SCPCH.</h5><a id="article1.body1.sec4.sec3.sec4.p1" name="article1.body1.sec4.sec3.sec4.p1"></a><p>In order to determine if the SCPCH obtained also performed well in phylogenetic analyses at low evolutionary levels, we used an enlarged taxa sampling for the oak powdery mildews and the <em>H. albidus</em>/<em>H. pseudoalbidus</em> groups. The SCPCH previously obtained were amplified and sequenced in seven additional related species (<em>Erysiphe convolvuli</em>, <em>Erysiphe cruciferarum</em>, <em>Erysiphe trifolii</em>, <em>Erysiphe elevata</em>, <em>Erysiphe platani</em> and <em>Oïdium neolycopersici</em>) and 4 species (<em>Ceratocystis platani</em>, <em>Ceratocystis fructigena</em>, <em>Ceratocystis paradoxa</em> and <em>Grosmania clavigera</em>) for the oak powdery mildew and the <em>H. albidus</em>/<em>H. pseudoalbidus</em> groups, respectively. The ortholog sequence of <em>M. grisea</em> was obtained from the full genome sequence (NCBI Reference Sequence: NZ_AACU00000000.2) and used to root the trees reconstructed for <em>Hymenoscyphus</em> spp. and related species. For each taxa set, we used the PCR and sequencing protocols described above. Maximum parsimony (MP) and maximum likelihood (ML) trees were reconstructed among each genes using PAUP ver. 4.0b10 (Swofford 2003). All nucleotide sites were included in the analyses but gaps were treated as missing characters. MP trees were identified by heuristic searches and 100 random additions of sequences (RAS). Confidence was examined using bootstrapping (BS) with 1000 replicates and the heuristic option with 100 RAS per pseudo replicate. Heuristic ML searches were performed with 100 replicates of random sequence addition and tree bisection reconnection (TBR) branch swapping. The best-fit model of sequence evolution for each gene was determined using the Akaike information criterion (AIC) implemented in JModelTest 0.1.1 (Posada 2008). Branch support was evaluated using 100 bootstrap replicates and 100 RAS per pseudo replicate. In MP and ML analyses, monophylies supported by both BS ≥60% were considered as significant. ML combined analyses (e.g. different gene datasets concatenated) were executed through the metapopulation genetic algorithm implemented in MetaPIGA version 2.0 <a href="#pone.0018803-Helaers1">[82]</a> using a mixed-model approach with each search replicated 10,000 times. Topological incongruences detected between trees were statistically evaluated by using the Wilcoxon sign-ranks test under MP and the Shimodaira-Hasegawa multiple comparison test in ML (SH-test; <a href="#pone.0018803-Shimodaira1">[83]</a> implemented in PAUP, as described in <a href="#pone.0018803-Feau1">[84]</a>.</p>


</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0018803"></div><a name="pone.0018803.s001" id="pone.0018803.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0018803.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI"><strong>Distribution of the SCPCH found among the 40 PHYLORPH runs performed for the computational testing.</strong> For example, five SCPCH (FUNYBASE ID FG1010, FG543, FG635, FG646 and FG909) are systematically found in 39 out of the 40 searches made.</p>
<p class="siDoi">doi:10.1371/journal.pone.0018803.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0018803.s002" id="pone.0018803.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0018803.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI"><strong>Cross comparison of the three protein datasets (FUNYBASE, PHYLOME-T60 and OrthoMCL-DB) used as the initial step of PHYLORPH.</strong> (A) Number of orthologs shared between two or three databases (showed at the intersections of the Venn diagram) or exclusive to one database (into the circles); (B) Number of fungal species exclusive to one database or shared between two or three databases.</p>
<p class="siDoi">doi:10.1371/journal.pone.0018803.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0018803.s003" id="pone.0018803.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0018803.s003">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">
            <strong>Fungal protein and genome sources used in PHYLORPH.</strong>
          </p>
<p class="siDoi">doi:10.1371/journal.pone.0018803.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0018803.s004" id="pone.0018803.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0018803.s004">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI"><strong>List of the 40 different PHYLORPH runs performed for the computational testing.</strong> Those destined to the experimental validation are indicated in bold.</p>
<p class="siDoi">doi:10.1371/journal.pone.0018803.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors wish to acknowledge O. Fabreguettes and S. Monlor for technical assistance in laboratory experiments, R. Ioos for providing <em>Ceratocystis</em> strains and F. Delmotte and two anonymous reviewers for helpful comments on this manuscript. Part of the experiments (DNA sequencing) presented in the present publication were performed at the genotyping and sequencing facility of Bordeaux. The authors are also grateful to the different institutes listed in <a href="#pone.0018803.s003">Table S1</a> for publicly providing the data of the fungal genome sequencing projects.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: NF. Performed the experiments: NF TD CH. Analyzed the data: NF TD CH. Contributed reagents/materials/analysis tools: NF M-LD-L CD. Wrote the paper: NF. Supervised the project and improved the manuscript: MLDL CD. Designed and tested the program: NF. </p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0018803-Brito1" id="pone.0018803-Brito1"></a>Brito PH, Edwards SV (2009) Multilocus phylogeography and phylogenetics using sequence-based markers. Genetica  135: 439–455.  <ul class="find" data-citedArticleID="2657316" data-doi="10.1007/s10709-008-9293-3"><li><a href="http://dx.doi.org/10.1007/s10709-008-9293-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multilocus+phylogeography+and+phylogenetics+using+sequence-based+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multilocus+phylogeography+and+phylogenetics+using+sequence-based+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0018803-Taylor1" id="pone.0018803-Taylor1"></a>Taylor JW, Jacobson DJ, Kroken S, Kasuga T, Geiser DM, et al.  (2000) Phylogenetic species recognition and species concepts in Fungi. Fungal Genetics and Biology  31: 21–32.  <ul class="find" data-citedArticleID="2657356" data-doi="10.1006/fgbi.2000.1228"><li><a href="http://dx.doi.org/10.1006/fgbi.2000.1228" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+species+recognition+and+species+concepts+in+Fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+species+recognition+and+species+concepts+in+Fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0018803-Townsend1" id="pone.0018803-Townsend1"></a>Townsend TM, Alegre RE, Kelley ST, Wiens JJ, Reeder TW (2008) Rapid development of multiple nuclear loci for phylogenetic analysis using genomic resources: An example from squamate reptiles. Molecular Phylogenetics and Evolution  47: 129–142.  <ul class="find" data-citedArticleID="2657344" data-doi="10.1016/j.ympev.2008.01.008"><li><a href="http://dx.doi.org/10.1016/j.ympev.2008.01.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+development+of+multiple+nuclear+loci+for+phylogenetic+analysis+using+genomic+resources%3A+An+example+from+squamate+reptiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+development+of+multiple+nuclear+loci+for+phylogenetic+analysis+using+genomic+resources%3A+An+example+from+squamate+reptiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0018803-Aguileta1" id="pone.0018803-Aguileta1"></a>Aguileta G, Marthey S, Chiapello H, Lebrun MH, Rodolphe F, et al.  (2008) Assessing the performance of single-copy genes for recovering robust phylogenies. Systematic Biology  57: 613–627.  <ul class="find" data-citedArticleID="2657374"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Assessing+the+performance+of+single-copy+genes+for+recovering+robust+phylogenies.&amp;auth=&amp;atitle=Assessing+the+performance+of+single-copy+genes+for+recovering+robust+phylogenies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+the+performance+of+single-copy+genes+for+recovering+robust+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+the+performance+of+single-copy+genes+for+recovering+robust+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0018803-Vialle1" id="pone.0018803-Vialle1"></a>Vialle A, Feau N, Allaire M, Didukh M, Martin F, et al.  (2009) Evaluation of mitochondrial genes as DNA barcode for Basidiomycota. Molecular Ecology Resources  9: 99–113.  <ul class="find" data-citedArticleID="2657284" data-doi="10.1111/j.1755-0998.2009.02637.x"><li><a href="http://dx.doi.org/10.1111/j.1755-0998.2009.02637.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+mitochondrial+genes+as+DNA+barcode+for+Basidiomycota." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+mitochondrial+genes+as+DNA+barcode+for+Basidiomycota.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0018803-Bridge1" id="pone.0018803-Bridge1"></a>Bridge PD, Spooner BM, Roberts PJ (2005) The impact of molecular data in fungal systematics. Advances in Botanical Research  42: 34–67.  <ul class="find" data-citedArticleID="2657290" data-doi="10.1016/s0065-2296(05)42002-9"><li><a href="http://dx.doi.org/10.1016/s0065-2296(05)42002-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+impact+of+molecular+data+in+fungal+systematics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+impact+of+molecular+data+in+fungal+systematics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0018803-Marthey1" id="pone.0018803-Marthey1"></a>Marthey S, Aguileta G, Rodolphe F, Gendrault A, Giraud T, et al.  (2008) FUNYBASE: a FUNgal phYlogenomic dataBASE. BMC Bioinformatics  9: 456.  <ul class="find" data-citedArticleID="2657228" data-doi="10.1186/1471-2105-9-456"><li><a href="http://dx.doi.org/10.1186/1471-2105-9-456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FUNYBASE%3A+a+FUNgal+phYlogenomic+dataBASE." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FUNYBASE%3A+a+FUNgal+phYlogenomic+dataBASE.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0018803-Lutzoni1" id="pone.0018803-Lutzoni1"></a>Lutzoni F, Kauff F, Cox CJ, McLaughlin D, Celio G, et al.  (2004) Assembling the fungal tree of life: progress, classification, and evolution of subcellular traits. American Journal of Botany  91: 1446–1480.  <ul class="find" data-citedArticleID="2657286" data-doi="10.3732/ajb.91.10.1446"><li><a href="http://dx.doi.org/10.3732/ajb.91.10.1446" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assembling+the+fungal+tree+of+life%3A+progress%2C+classification%2C+and+evolution+of+subcellular+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assembling+the+fungal+tree+of+life%3A+progress%2C+classification%2C+and+evolution+of+subcellular+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0018803-Schmitt1" id="pone.0018803-Schmitt1"></a>Schmitt I, Crespo A, Divakar P, Fankhauser J, Herman-Sackett E, et al.  (2009) New primers for promising single-copy genes in fungal phylogenetics and systematics. Persoonia  23: 35–40.  <ul class="find" data-citedArticleID="2657260" data-doi="10.3767/003158509x470602"><li><a href="http://dx.doi.org/10.3767/003158509x470602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+primers+for+promising+single-copy+genes+in+fungal+phylogenetics+and+systematics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+primers+for+promising+single-copy+genes+in+fungal+phylogenetics+and+systematics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0018803-Carbone1" id="pone.0018803-Carbone1"></a>Carbone I, Kohn LM (1999) A method for designing primer sets for speciation studies in filamentous Ascomycetes. Mycologia  91: 553–556.  <ul class="find" data-citedArticleID="2657224" data-doi="10.2307/3761358"><li><a href="http://dx.doi.org/10.2307/3761358" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+method+for+designing+primer+sets+for+speciation+studies+in+filamentous+Ascomycetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+method+for+designing+primer+sets+for+speciation+studies+in+filamentous+Ascomycetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0018803-Einax1" id="pone.0018803-Einax1"></a>Einax E, Voigt K (2003) Oligonucleotide primers for the universal amplification of beta-tubulin genes facilitate phylogenetic analyses in the regnum Fungi. Organisms Diversity Evolution  3: 185–194.  <ul class="find" data-citedArticleID="2657270" data-doi="10.1078/1439-6092-00069"><li><a href="http://dx.doi.org/10.1078/1439-6092-00069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oligonucleotide+primers+for+the+universal+amplification+of+beta-tubulin+genes+facilitate+phylogenetic+analyses+in+the+regnum+Fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oligonucleotide+primers+for+the+universal+amplification+of+beta-tubulin+genes+facilitate+phylogenetic+analyses+in+the+regnum+Fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0018803-Kurtzman1" id="pone.0018803-Kurtzman1"></a>Kurtzman CP, Robnett CJ (2003) Phylogenetic relationships among yeasts of the ‘<em>Saccharomyces</em> complex' determined from multigene sequence analyses. FEMS Yeast Research  3: 417–432.  <ul class="find" data-citedArticleID="2657342" data-doi="10.1016/s1567-1356(03)00012-6"><li><a href="http://dx.doi.org/10.1016/s1567-1356(03)00012-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+relationships+among+yeasts+of+the+%E2%80%98Saccharomyces+complex%27+determined+from+multigene+sequence+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+relationships+among+yeasts+of+the+%E2%80%98Saccharomyces+complex%27+determined+from+multigene+sequence+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0018803-Tanabe1" id="pone.0018803-Tanabe1"></a>Tanabe Y, Saikawa M, Watanabe MM, Sugiyama J (2004) Molecular phylogeny of Zygomycota based on EF-1alpha and RPB1 sequences: limitations and utility of alternative markers to rDNA. Molecular Phylogenetics and Evolution  30: 438–449.  <ul class="find" data-citedArticleID="2657258" data-doi="10.1016/s1055-7903(03)00185-4"><li><a href="http://dx.doi.org/10.1016/s1055-7903(03)00185-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+of+Zygomycota+based+on+EF-1alpha+and+RPB1+sequences%3A+limitations+and+utility+of+alternative+markers+to+rDNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+of+Zygomycota+based+on+EF-1alpha+and+RPB1+sequences%3A+limitations+and+utility+of+alternative+markers+to+rDNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0018803-Landvik1" id="pone.0018803-Landvik1"></a>Landvik S, Eriksson OE, Berbee ML (2001) Neolecta: A Fungal Dinosaur? Evidence from β-tubulin amino acid sequences. Mycologia  93: 1151–1163.  <ul class="find" data-citedArticleID="2657332" data-doi="10.2307/3761675"><li><a href="http://dx.doi.org/10.2307/3761675" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neolecta%3A+A+Fungal+Dinosaur%3F+Evidence+from+%CE%B2-tubulin+amino+acid+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neolecta%3A+A+Fungal+Dinosaur%3F+Evidence+from+%CE%B2-tubulin+amino+acid+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0018803-Tanabe2" id="pone.0018803-Tanabe2"></a>Tanabe Y, Watanabe MM, Sugiyama J (2002) Are Microsporidia really related to Fungi: a reappraisal based on additional gene sequences from basal fungi. Mycological Research  106: 1380–1391.  <ul class="find" data-citedArticleID="2657318" data-doi="10.1017/s095375620200686x"><li><a href="http://dx.doi.org/10.1017/s095375620200686x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+Microsporidia+really+related+to+Fungi%3A+a+reappraisal+based+on+additional+gene+sequences+from+basal+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+Microsporidia+really+related+to+Fungi%3A+a+reappraisal+based+on+additional+gene+sequences+from+basal+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0018803-Ayliffe1" id="pone.0018803-Ayliffe1"></a>Ayliffe MA, Dodds PN, Lawrence GJ (2001) Characterisation of a β-tubulin gene from <em>Melampsora lini</em> and comparison of fungal β-tubulin genes. Mycological Research  105: 818–826.  <ul class="find" data-citedArticleID="2657304" data-doi="10.1017/s0953756201004245"><li><a href="http://dx.doi.org/10.1017/s0953756201004245" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterisation+of+a+%CE%B2-tubulin+gene+from+Melampsora+lini+and+comparison+of+fungal+%CE%B2-tubulin+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterisation+of+a+%CE%B2-tubulin+gene+from+Melampsora+lini+and+comparison+of+fungal+%CE%B2-tubulin+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0018803-Begerow1" id="pone.0018803-Begerow1"></a>Begerow D, Beate J, Oberwinkler F (2004) Evolutionary relationships among β-tubulin gene sequences of basidiomycetous fungi. Mycological Research  108: 1257–1263.  <ul class="find" data-citedArticleID="2657240" data-doi="10.1017/s0953756204001066"><li><a href="http://dx.doi.org/10.1017/s0953756204001066" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+relationships+among+%CE%B2-tubulin+gene+sequences+of+basidiomycetous+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+relationships+among+%CE%B2-tubulin+gene+sequences+of+basidiomycetous+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0018803-Fitzpatrick1" id="pone.0018803-Fitzpatrick1"></a>Fitzpatrick D, Logue M, Stajich J, G B (2006) A fungal phylogeny based on 42 complete genomes derived from supertree and combined gene analysis. BMC Evolutionary Biology  6: 99.  <ul class="find" data-citedArticleID="2657302"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+fungal+phylogeny+based+on+42+complete+genomes+derived+from+supertree+and+combined+gene+analysis.&amp;auth=&amp;atitle=A+fungal+phylogeny+based+on+42+complete+genomes+derived+from+supertree+and+combined+gene+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+fungal+phylogeny+based+on+42+complete+genomes+derived+from+supertree+and+combined+gene+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+fungal+phylogeny+based+on+42+complete+genomes+derived+from+supertree+and+combined+gene+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0018803-Kubisiak1" id="pone.0018803-Kubisiak1"></a>Kubisiak TL, Dutech C, Milgroom MG (2007) Fifty-three polymorphic microsatellite loci in the chestnut blight fungus, <em>Cryphonectria parasitica</em>. Molecular Ecology Notes  7: 428–432.  <ul class="find" data-citedArticleID="2657382" data-doi="10.1111/j.1471-8286.2006.01608.x"><li><a href="http://dx.doi.org/10.1111/j.1471-8286.2006.01608.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fifty-three+polymorphic+microsatellite+loci+in+the+chestnut+blight+fungus%2C+Cryphonectria+parasitica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fifty-three+polymorphic+microsatellite+loci+in+the+chestnut+blight+fungus%2C+Cryphonectria+parasitica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0018803-Tsui1" id="pone.0018803-Tsui1"></a>Tsui CKM, Feau N, Ritland CE, Massoumi-Alamouti S, DiGuistini S, et al.  (2009) Characterization of microsatellite loci in the fungus, <em>Grosmannia clavigera</em>, a pine pathogen associated with the mountain pine beetle. Molecular Ecology Resources  9: 1500–1503.  <ul class="find" data-citedArticleID="2657292" data-doi="10.1111/j.1755-0998.2009.02717.x"><li><a href="http://dx.doi.org/10.1111/j.1755-0998.2009.02717.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+microsatellite+loci+in+the+fungus%2C+Grosmannia+clavigera%2C+a+pine+pathogen+associated+with+the+mountain+pine+beetle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+microsatellite+loci+in+the+fungus%2C+Grosmannia+clavigera%2C+a+pine+pathogen+associated+with+the+mountain+pine+beetle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0018803-Lim1" id="pone.0018803-Lim1"></a>Lim S, Notley-McRobb L, Lim M, Carter DA (2004) A comparison of the nature and abundance of microsatellites in 14 fungal genomes. Fungal Genetics and Biology  41: 1025–1036.  <ul class="find" data-citedArticleID="2657364" data-doi="10.1016/j.fgb.2004.08.004"><li><a href="http://dx.doi.org/10.1016/j.fgb.2004.08.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+the+nature+and+abundance+of+microsatellites+in+14+fungal+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+the+nature+and+abundance+of+microsatellites+in+14+fungal+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0018803-Xhaard1" id="pone.0018803-Xhaard1"></a>Xhaard C, Andrieux A, Halkett F, Frey P (2009) Characterization of 41 microsatellite loci developed from the genome sequence of the poplar rust fungus, <em>Melampsora larici-populina</em>. Conservation Genetics Resources  1: 21–25.  <ul class="find" data-citedArticleID="2657232" data-doi="10.1007/s12686-009-9005-z"><li><a href="http://dx.doi.org/10.1007/s12686-009-9005-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+41+microsatellite+loci+developed+from+the+genome+sequence+of+the+poplar+rust+fungus%2C+Melampsora+larici-populina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+41+microsatellite+loci+developed+from+the+genome+sequence+of+the+poplar+rust+fungus%2C+Melampsora+larici-populina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0018803-Wang1" id="pone.0018803-Wang1"></a>Wang H, Xu Z, Gao L, Hao B (2009) A fungal phylogeny based on 82 complete genomes using the composition vector method. BMC Evolutionary Biology  9: 195.  <ul class="find" data-citedArticleID="2657370" data-doi="10.1186/1471-2148-9-195"><li><a href="http://dx.doi.org/10.1186/1471-2148-9-195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+fungal+phylogeny+based+on+82+complete+genomes+using+the+composition+vector+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+fungal+phylogeny+based+on+82+complete+genomes+using+the+composition+vector+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0018803-Kuramae1" id="pone.0018803-Kuramae1"></a>Kuramae EE, Robert V, Snel B, Weiβ M, Boekhout T (2006) Phylogenomics reveal a robust fungal tree of life. FEMS Yeast Research  6: 1213–1220.  <ul class="find" data-citedArticleID="2657262" data-doi="10.1111/j.1567-1364.2006.00119.x"><li><a href="http://dx.doi.org/10.1111/j.1567-1364.2006.00119.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenomics+reveal+a+robust+fungal+tree+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenomics+reveal+a+robust+fungal+tree+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0018803-MarcetHouben1" id="pone.0018803-MarcetHouben1"></a>Marcet-Houben M, Gabaldón T (2009) The tree versus the forest: The Fungal Tree of Life and the topological diversity within the yeast phylome. PLoS One  4: e4357.  <ul class="find" data-citedArticleID="2657280" data-doi="10.1371/journal.pone.0004357"><li><a href="http://dx.doi.org/10.1371/journal.pone.0004357" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tree+versus+the+forest%3A+The+Fungal+Tree+of+Life+and+the+topological+diversity+within+the+yeast+phylome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tree+versus+the+forest%3A+The+Fungal+Tree+of+Life+and+the+topological+diversity+within+the+yeast+phylome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0018803-Adams1" id="pone.0018803-Adams1"></a>Adams RI, Hallen HE, Pringle A (2006) Using the incomplete genome of the ectomycorrhizal fungus <em>Amanita bisporigera</em> to identify molecular polymorphisms in the related <em>Amanita phalloides</em>. Molecular Ecology Notes  6: 218–220.  <ul class="find" data-citedArticleID="2657324" data-doi="10.1111/j.1471-8286.2005.01198.x"><li><a href="http://dx.doi.org/10.1111/j.1471-8286.2005.01198.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+the+incomplete+genome+of+the+ectomycorrhizal+fungus+Amanita+bisporigera+to+identify+molecular+polymorphisms+in+the+related+Amanita+phalloides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+the+incomplete+genome+of+the+ectomycorrhizal+fungus+Amanita+bisporigera+to+identify+molecular+polymorphisms+in+the+related+Amanita+phalloides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0018803-Pringle1" id="pone.0018803-Pringle1"></a>Pringle A, Adams R, Cross H, Bruns T (2009) The ectomycorrhizal fungus <em>Amanita phalloides</em> was introduced and is expanding its range on the west coast of North America. Molecular Ecology  18: 817–833.  <ul class="find" data-citedArticleID="2657360" data-doi="10.1111/j.1365-294x.2008.04030.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2008.04030.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ectomycorrhizal+fungus+Amanita+phalloides+was+introduced+and+is+expanding+its+range+on+the+west+coast+of+North+America." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ectomycorrhizal+fungus+Amanita+phalloides+was+introduced+and+is+expanding+its+range+on+the+west+coast+of+North+America.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0018803-LpezVillavicencio1" id="pone.0018803-LpezVillavicencio1"></a>López-Villavicencio M, Aguileta G, Giraud T, de Vienne DM, Lacoste S, et al.  (2010) Sex in Penicillium: Combined phylogenetic and experimental approaches. Fungal Genetics and Biology  47: 693–706.  <ul class="find" data-citedArticleID="2657306" data-doi="10.1016/j.fgb.2010.05.002"><li><a href="http://dx.doi.org/10.1016/j.fgb.2010.05.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sex+in+Penicillium%3A+Combined+phylogenetic+and+experimental+approaches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sex+in+Penicillium%3A+Combined+phylogenetic+and+experimental+approaches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0018803-Roe1" id="pone.0018803-Roe1"></a>Roe AD, Sperling FAH (2007) Patterns of evolution of mitochondrial cytochrome c oxidase I and II DNA and implications for DNA barcoding. Molecular Phylogenetics and Evolution  44: 325–345.  <ul class="find" data-citedArticleID="2657226" data-doi="10.1016/j.ympev.2006.12.005"><li><a href="http://dx.doi.org/10.1016/j.ympev.2006.12.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+evolution+of+mitochondrial+cytochrome+c+oxidase+I+and+II+DNA+and+implications+for+DNA+barcoding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+evolution+of+mitochondrial+cytochrome+c+oxidase+I+and+II+DNA+and+implications+for+DNA+barcoding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0018803-Galtier1" id="pone.0018803-Galtier1"></a>Galtier N, Enard D, Radondy Y, Bazin E, Belkhir K (2006) Mutation hot spots in mammalian mitochondrial DNA. Genome Research  16: 215–222.  <ul class="find" data-citedArticleID="2657350" data-doi="10.1101/gr.4305906"><li><a href="http://dx.doi.org/10.1101/gr.4305906" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+hot+spots+in+mammalian+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+hot+spots+in+mammalian+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0018803-Nei1" id="pone.0018803-Nei1"></a>Nei M (1987) Molecular Evolutionary Genetics. New York, , NY: Columbia University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0018803-Kiss1" id="pone.0018803-Kiss1"></a>Kiss L (1997) Genetic diversity in <em>Ampelomyces</em> isolates, hyperparasites of powdery mildew fungi, inferred from RFLP analysis of the rDNA ITS region. Mycological Research  101: 1073–1080.  <ul class="find" data-citedArticleID="2657230" data-doi="10.1017/s0953756297003705"><li><a href="http://dx.doi.org/10.1017/s0953756297003705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+in+Ampelomyces+isolates%2C+hyperparasites+of+powdery+mildew+fungi%2C+inferred+from+RFLP+analysis+of+the+rDNA+ITS+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+in+Ampelomyces+isolates%2C+hyperparasites+of+powdery+mildew+fungi%2C+inferred+from+RFLP+analysis+of+the+rDNA+ITS+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0018803-lvarez1" id="pone.0018803-lvarez1"></a>Álvarez I, Wendel JF (2003) Ribosomal ITS sequences and plant phylogenetic inference. Molecular Phylogenetics and Evolution  29: 417–434.  <ul class="find" data-citedArticleID="2657354" data-doi="10.1016/s1055-7903(03)00208-2"><li><a href="http://dx.doi.org/10.1016/s1055-7903(03)00208-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+ITS+sequences+and+plant+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+ITS+sequences+and+plant+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0018803-Bruns1" id="pone.0018803-Bruns1"></a>Bruns TD, Szaro TM (1992) Rate and mode differences between nuclear and mitochondrial small-subunit rRNA genes in mushrooms. Molecular Biology and Evolution  9: 836–855.  <ul class="find" data-citedArticleID="2657234"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rate+and+mode+differences+between+nuclear+and+mitochondrial+small-subunit+rRNA+genes+in+mushrooms.&amp;auth=&amp;atitle=Rate+and+mode+differences+between+nuclear+and+mitochondrial+small-subunit+rRNA+genes+in+mushrooms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rate+and+mode+differences+between+nuclear+and+mitochondrial+small-subunit+rRNA+genes+in+mushrooms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rate+and+mode+differences+between+nuclear+and+mitochondrial+small-subunit+rRNA+genes+in+mushrooms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0018803-Kretzer1" id="pone.0018803-Kretzer1"></a>Kretzer AM, Bruns TD (1999) Use of atp6 in fungal phylogenetics: An example from the Boletales. Molecular Phylogenetics and Evolution  13: 483–492.  <ul class="find" data-citedArticleID="2657320" data-doi="10.1006/mpev.1999.0680"><li><a href="http://dx.doi.org/10.1006/mpev.1999.0680" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+atp6+in+fungal+phylogenetics%3A+An+example+from+the+Boletales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+atp6+in+fungal+phylogenetics%3A+An+example+from+the+Boletales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0018803-Mooers1" id="pone.0018803-Mooers1"></a>Mooers AØ, Holmes EC (2002) The evolution of base composition and phylogenetic inference. Trends in Ecology and Evolution  15: 365–368.  <ul class="find" data-citedArticleID="2657274" data-doi="10.1016/s0169-5347(00)01934-0"><li><a href="http://dx.doi.org/10.1016/s0169-5347(00)01934-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+base+composition+and+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+base+composition+and+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0018803-Zipfel1" id="pone.0018803-Zipfel1"></a>Zipfel RD, de Beer ZW, Jacobs K, Wingfield BD, Wingfield MJ (2006) Multi-gene phylogenies define <em>Ceratocystiopsis</em> and <em>Grosmannia</em> distinct from <em>Ophiostoma</em>. Studies in Mycology  55: 75–97.  <ul class="find" data-citedArticleID="2657250" data-doi="10.3114/sim.55.1.75"><li><a href="http://dx.doi.org/10.3114/sim.55.1.75" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multi-gene+phylogenies+define+Ceratocystiopsis+and+Grosmannia+distinct+from+Ophiostoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multi-gene+phylogenies+define+Ceratocystiopsis+and+Grosmannia+distinct+from+Ophiostoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0018803-Thomson1" id="pone.0018803-Thomson1"></a>Thomson RC, Wang IJ, Johnson JR (2010) Genome-enabled development of DNA markers for ecology, evolution and conservation. Molecular Ecology  19: 2184–2195.  <ul class="find" data-citedArticleID="2657278" data-doi="10.1111/j.1365-294x.2010.04650.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2010.04650.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-enabled+development+of+DNA+markers+for+ecology%2C+evolution+and+conservation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-enabled+development+of+DNA+markers+for+ecology%2C+evolution+and+conservation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0018803-Fang1" id="pone.0018803-Fang1"></a>Fang G, Bhardwaj N, Robilotto R, Gerstein MB (2010) Getting started in gene orthology and functional analysis. PLoS Computational Biology  6: e1000703.  <ul class="find" data-citedArticleID="2657256" data-doi="10.1371/journal.pcbi.1000703"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.1000703" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Getting+started+in+gene+orthology+and+functional+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Getting+started+in+gene+orthology+and+functional+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0018803-Gabaldn1" id="pone.0018803-Gabaldn1"></a>Gabaldón T (2008) Large-scale assignment of orthology: back to phylogenetics? Genome Biology  9: 235.  <ul class="find" data-citedArticleID="2657346" data-doi="10.1186/gb-2008-9-10-235"><li><a href="http://dx.doi.org/10.1186/gb-2008-9-10-235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+assignment+of+orthology%3A+back+to+phylogenetics%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+assignment+of+orthology%3A+back+to+phylogenetics%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0018803-Kuzniar1" id="pone.0018803-Kuzniar1"></a>Kuzniar A, van Ham RC, Pongor S, Leunissen JA (2008) The quest for orthologs: finding the corresponding gene across genomes. Trends in Genetics  24: 539–551.  <ul class="find" data-citedArticleID="2657358" data-doi="10.1016/j.tig.2008.08.009"><li><a href="http://dx.doi.org/10.1016/j.tig.2008.08.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+quest+for+orthologs%3A+finding+the+corresponding+gene+across+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+quest+for+orthologs%3A+finding+the+corresponding+gene+across+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0018803-Chen1" id="pone.0018803-Chen1"></a>Chen F, Mackey AJ, Stoeckert CJJ, Roos DS (2006) OrthoMCL-DB: Querying a comprehensive multi-species collection of ortholog groups. Nucleic Acids Research  34: D363–368.  <ul class="find" data-citedArticleID="2657254" data-doi="10.1093/nar/gkj123"><li><a href="http://dx.doi.org/10.1093/nar/gkj123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OrthoMCL-DB%3A+Querying+a+comprehensive+multi-species+collection+of+ortholog+groups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OrthoMCL-DB%3A+Querying+a+comprehensive+multi-species+collection+of+ortholog+groups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0018803-Enright1" id="pone.0018803-Enright1"></a>Enright AJ, Van Dongen S, Ouzounis CA (2002) An efficient algorithm for large-scale detection of protein families. Nucleic Acids Research  30: 1575–1584.  <ul class="find" data-citedArticleID="2657330" data-doi="10.1093/nar/30.7.1575"><li><a href="http://dx.doi.org/10.1093/nar/30.7.1575" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+efficient+algorithm+for+large-scale+detection+of+protein+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+efficient+algorithm+for+large-scale+detection+of+protein+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0018803-Li1" id="pone.0018803-Li1"></a>Li L, Stoeckert CJJ, Ross DS (2003) OrthoMCL: Identification of ortholog groups for eukaryotic genomes. Genome Research 13:  2178: 2189.  <ul class="find" data-citedArticleID="2657368" data-doi="10.1101/gr.1224503"><li><a href="http://dx.doi.org/10.1101/gr.1224503" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OrthoMCL%3A+Identification+of+ortholog+groups+for+eukaryotic+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OrthoMCL%3A+Identification+of+ortholog+groups+for+eukaryotic+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0018803-Chen2" id="pone.0018803-Chen2"></a>Chen F, Mackey AJ, Vermunt JK, Roos DS (2007) Assessing performance of orthology detection strategies applied to eukaryotic Q3 genomes. PLoS ONE  2: e383.  <ul class="find" data-citedArticleID="2657246" data-doi="10.1371/journal.pone.0000383"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000383" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+performance+of+orthology+detection+strategies+applied+to+eukaryotic+Q3+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+performance+of+orthology+detection+strategies+applied+to+eukaryotic+Q3+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0018803-Kirk1" id="pone.0018803-Kirk1"></a>Kirk PM, Cannon PF, David JC, Stalpers JA (2001) Ainsworth and Bisby's dictionary of the fungi: 9th edition.   Ainsworth and Bisby's dictionary of the fungi: 9th edition. Wallingford UK: CABI Publishing. xi + 655 p.  <ul class="find-nolinks"></ul></li><li><span class="label">47.
              </span><a name="pone.0018803-Cuomo1" id="pone.0018803-Cuomo1"></a>Cuomo CA, Birren BW, Jonathan W, Guthrie Christine, Gerald RF (2010) The Fungal Genome Initiative and Lessons Learned from Genome Sequencing. Methods in Enzymology: Academic Press. pp. 833–855.  <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0018803-Martinez1" id="pone.0018803-Martinez1"></a>Martinez DA, Nelson MA (2010) The next generation becomes the now generation. PLoS Genetics  6: e1000906.  <ul class="find" data-citedArticleID="2657340" data-doi="10.1371/journal.pgen.1000906"><li><a href="http://dx.doi.org/10.1371/journal.pgen.1000906" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+next+generation+becomes+the+now+generation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+next+generation+becomes+the+now+generation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0018803-Murphy1" id="pone.0018803-Murphy1"></a>Murphy WJ, O'Brien SJ (2007) Designing and optimizing comparative anchor primers for comparative gene mapping and phylogenetic inference. Nature Protocols  2: 3022–3030.  <ul class="find" data-citedArticleID="2657338" data-doi="10.1038/nprot.2007.429"><li><a href="http://dx.doi.org/10.1038/nprot.2007.429" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Designing+and+optimizing+comparative+anchor+primers+for+comparative+gene+mapping+and+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Designing+and+optimizing+comparative+anchor+primers+for+comparative+gene+mapping+and+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0018803-Palumbi1" id="pone.0018803-Palumbi1"></a>Palumbi S (1996) Nucleic acids II: The polymerase chain reaction. Molecular Systematics 2nd edn. Sundlerland, MA: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">51.
              </span><a name="pone.0018803-Yang1" id="pone.0018803-Yang1"></a>Yang Z (1998) On the best evolutionary rate for phylogenetic analysis. Systematic Biology  47: 125–133.  <ul class="find" data-citedArticleID="2657372"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+the+best+evolutionary+rate+for+phylogenetic+analysis.&amp;auth=&amp;atitle=On+the+best+evolutionary+rate+for+phylogenetic+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+best+evolutionary+rate+for+phylogenetic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+best+evolutionary+rate+for+phylogenetic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0018803-Klopfstein1" id="pone.0018803-Klopfstein1"></a>Klopfstein S, Kropf C, Quicke DLJ (2010) An evaluation of phylogenetic informativeness profiles and the molecular phylogeny of Diplazontinae (Hymenoptera, Ichneumonidae). Systematic Biology  59: 226–241.  <ul class="find" data-citedArticleID="2657220" data-doi="10.1111/j.1463-6409.2011.00481.x"><li><a href="http://dx.doi.org/10.1111/j.1463-6409.2011.00481.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evaluation+of+phylogenetic+informativeness+profiles+and+the+molecular+phylogeny+of+Diplazontinae+%28Hymenoptera%2C+Ichneumonidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evaluation+of+phylogenetic+informativeness+profiles+and+the+molecular+phylogeny+of+Diplazontinae+%28Hymenoptera%2C+Ichneumonidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0018803-Townsend2" id="pone.0018803-Townsend2"></a>Townsend JP (2007) Profiling phylogenetic informativeness. Systematic Biology  56: 222–231.  <ul class="find" data-citedArticleID="2657348"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Profiling+phylogenetic+informativeness.&amp;auth=&amp;atitle=Profiling+phylogenetic+informativeness." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Profiling+phylogenetic+informativeness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Profiling+phylogenetic+informativeness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0018803-Philippe1" id="pone.0018803-Philippe1"></a>Philippe H, Zhou Y, Brinkmann H, Rodrigue N, Delsuc F (2005) Heterotachy and long-branch attraction in phylogenetics. BMC Evolutionary Biology  5: 50.  <ul class="find" data-citedArticleID="2657310" data-doi="10.1186/1471-2148-5-50"><li><a href="http://dx.doi.org/10.1186/1471-2148-5-50" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterotachy+and+long-branch+attraction+in+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterotachy+and+long-branch+attraction+in+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0018803-Wenzel1" id="pone.0018803-Wenzel1"></a>Wenzel JW, Siddall ME (1999) Noise. Cladistics  15: 51–64.  <ul class="find" data-citedArticleID="2657242" data-doi="10.1111/j.1096-0031.1999.tb00394.x"><li><a href="http://dx.doi.org/10.1111/j.1096-0031.1999.tb00394.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Noise." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Noise.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0018803-Graybeal1" id="pone.0018803-Graybeal1"></a>Graybeal A (1994) Evaluating the phylogenetic utility of genes: a search for genes informative about deep divergences among vertebrates. Systematic Biology  43: 174–193.  <ul class="find" data-citedArticleID="2657322"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evaluating+the+phylogenetic+utility+of+genes%3A+a+search+for+genes+informative+about+deep+divergences+among+vertebrates.&amp;auth=&amp;atitle=Evaluating+the+phylogenetic+utility+of+genes%3A+a+search+for+genes+informative+about+deep+divergences+among+vertebrates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluating+the+phylogenetic+utility+of+genes%3A+a+search+for+genes+informative+about+deep+divergences+among+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluating+the+phylogenetic+utility+of+genes%3A+a+search+for+genes+informative+about+deep+divergences+among+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0018803-Kostka1" id="pone.0018803-Kostka1"></a>Kostka M, Uzlikova M, Cepicka I, Flegr J (2008) SlowFaster, a user-friendly program for slow-fast analysis and its application on phylogeny of Blastocystis. BMC Bioinformatics  9: 341.  <ul class="find" data-citedArticleID="2657266" data-doi="10.1186/1471-2105-9-341"><li><a href="http://dx.doi.org/10.1186/1471-2105-9-341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SlowFaster%2C+a+user-friendly+program+for+slow-fast+analysis+and+its+application+on+phylogeny+of+Blastocystis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SlowFaster%2C+a+user-friendly+program+for+slow-fast+analysis+and+its+application+on+phylogeny+of+Blastocystis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0018803-Lartillot1" id="pone.0018803-Lartillot1"></a>Lartillot N, Brinkmann H, Philippe H (2007) Suppression of long-branch attraction artefacts in the animal phylogeny using a site-heterogeneous model. BMC Evolutionary Biology  7: S4.  <ul class="find" data-citedArticleID="2657336" data-doi="10.1186/1471-2148-7-s1-s4"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-s1-s4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppression+of+long-branch+attraction+artefacts+in+the+animal+phylogeny+using+a+site-heterogeneous+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppression+of+long-branch+attraction+artefacts+in+the+animal+phylogeny+using+a+site-heterogeneous+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0018803-Phillips1" id="pone.0018803-Phillips1"></a>Phillips MJ, Penny D (2003) The root of the mammalian tree inferred from whole mitochondrial genomes. Molecular Phylogenetics and Evolution  28: 171–185.  <ul class="find" data-citedArticleID="2657248" data-doi="10.1016/s1055-7903(03)00057-5"><li><a href="http://dx.doi.org/10.1016/s1055-7903(03)00057-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+root+of+the+mammalian+tree+inferred+from+whole+mitochondrial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+root+of+the+mammalian+tree+inferred+from+whole+mitochondrial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0018803-DeBry1" id="pone.0018803-DeBry1"></a>DeBry R, Seshadri S (2001) Nuclear intron sequences for phylogenetics of closely related mammals: an example using the phylogeny of Mus. Journal of Mammalogy  82: 280–288.  <ul class="find" data-citedArticleID="2657294" data-doi="10.1644/1545-1542(2001)082<0280:nisfpo&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1644/1545-1542(2001)082<0280:nisfpo&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+intron+sequences+for+phylogenetics+of+closely+related+mammals%3A+an+example+using+the+phylogeny+of+Mus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+intron+sequences+for+phylogenetics+of+closely+related+mammals%3A+an+example+using+the+phylogeny+of+Mus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0018803-Palumbi2" id="pone.0018803-Palumbi2"></a>Palumbi SR, Baker , CS  (1994) Contrasting population structure from nuclear intron sequences and mtDNA of humpback whales. Molecular Biology and Evolution  11: 426–435.  <ul class="find" data-citedArticleID="2657272"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Contrasting+population+structure+from+nuclear+intron+sequences+and+mtDNA+of+humpback+whales.&amp;auth=&amp;atitle=Contrasting+population+structure+from+nuclear+intron+sequences+and+mtDNA+of+humpback+whales." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contrasting+population+structure+from+nuclear+intron+sequences+and+mtDNA+of+humpback+whales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contrasting+population+structure+from+nuclear+intron+sequences+and+mtDNA+of+humpback+whales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0018803-Bierne1" id="pone.0018803-Bierne1"></a>Bierne N, Lehnert SA, Bédier E, Bonhomme F, Moore SS (2000) Screening for intron-length polymorphisms in penaeid shrimps using exon-primed intron-crossing (EPIC)-PCR. Molecular Ecology  9: 233–235.  <ul class="find" data-citedArticleID="2657378" data-doi="10.1046/j.1365-294x.2000.00842.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2000.00842.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Screening+for+intron-length+polymorphisms+in+penaeid+shrimps+using+exon-primed+intron-crossing+%28EPIC%29-PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Screening+for+intron-length+polymorphisms+in+penaeid+shrimps+using+exon-primed+intron-crossing+%28EPIC%29-PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0018803-Nielsen1" id="pone.0018803-Nielsen1"></a>Nielsen CB, Friedman B, Birren B, Burge CB, Galagan JE (2004) Patterns of intron gain and loss in Fungi. PLoS Biology  2: e422.  <ul class="find" data-citedArticleID="2657268" data-doi="10.1371/journal.pbio.0020422"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020422" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+intron+gain+and+loss+in+Fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+intron+gain+and+loss+in+Fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0018803-Rogozin1" id="pone.0018803-Rogozin1"></a>Rogozin IB, Wolf YI, Sorokin AV, Mirkin BG, Koonin EV (2003) Remarkable interkingdom conservation of intron positions and massive, lineage-specific intron loss and gain in eukaryotic evolution. Current Biology  13: 1512–1517.  <ul class="find" data-citedArticleID="2657328" data-doi="10.1016/s0960-9822(03)00558-x"><li><a href="http://dx.doi.org/10.1016/s0960-9822(03)00558-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Remarkable+interkingdom+conservation+of+intron+positions+and+massive%2C+lineage-specific+intron+loss+and+gain+in+eukaryotic+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Remarkable+interkingdom+conservation+of+intron+positions+and+massive%2C+lineage-specific+intron+loss+and+gain+in+eukaryotic+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0018803-Stajich1" id="pone.0018803-Stajich1"></a>Stajich JE, Dietrich FS (2006) Evidence of mRNA-mediated intron loss in the human-pathogenic fungus <em>Cryptococcus neoformans</em>. Eukaryotic Cell  5: 789–793.  <ul class="find" data-citedArticleID="2657216" data-doi="10.1128/ec.5.5.789-793.2006"><li><a href="http://dx.doi.org/10.1128/ec.5.5.789-793.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+of+mRNA-mediated+intron+loss+in+the+human-pathogenic+fungus+Cryptococcus+neoformans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+of+mRNA-mediated+intron+loss+in+the+human-pathogenic+fungus+Cryptococcus+neoformans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0018803-Queloz1" id="pone.0018803-Queloz1"></a>Queloz V, Grünig CR, Berndt R, Kowalski T, Sieber TN, et al.  (2010) Cryptic speciation in <em>Hymenoscyphus albidus</em>. Forest Pathology.   DOI:10.1111/j.1439-0329.2010.00645.x.  <ul class="find-nolinks"></ul></li><li><span class="label">67.
              </span><a name="pone.0018803-Husson1" id="pone.0018803-Husson1"></a>Husson C, Scala B, Caël O, Frey P, Feau N, et al.  (2010) <em>Chalara fraxinea</em> is an invasive pathogen in France.   DOI:10.1007/s10658-011-9755-9.  <ul class="find-nolinks"></ul></li><li><span class="label">68.
              </span><a name="pone.0018803-Moss1" id="pone.0018803-Moss1"></a>Moss T, Stefanovsky V (1995) Promotion and regulation of ribosomal transcription in eukaryotes by RNA polymerase I. Progress in Nucleic Acid Research Molecular Biology  50: 25–66.  <ul class="find" data-citedArticleID="2657288" data-doi="10.1016/s0079-6603(08)60810-7"><li><a href="http://dx.doi.org/10.1016/s0079-6603(08)60810-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Promotion+and+regulation+of+ribosomal+transcription+in+eukaryotes+by+RNA+polymerase+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Promotion+and+regulation+of+ribosomal+transcription+in+eukaryotes+by+RNA+polymerase+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0018803-Service1" id="pone.0018803-Service1"></a>Service RF (2006) Gene sequencing: The Race for the $1000 Genome. Science  311: 1544–1546.  <ul class="find" data-citedArticleID="2657236" data-doi="10.1126/science.311.5767.1544"><li><a href="http://dx.doi.org/10.1126/science.311.5767.1544" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+sequencing%3A+The+Race+for+the+%241000+Genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+sequencing%3A+The+Race+for+the+%241000+Genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0018803-Podolak1" id="pone.0018803-Podolak1"></a>Podolak E (2010) Sequencing's new race. BioTechniques  48: 105–111.  <ul class="find" data-citedArticleID="2657244" data-doi="10.2144/000113371"><li><a href="http://dx.doi.org/10.2144/000113371" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing%27s+new+race." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing%27s+new+race.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0018803-Edgar1" id="pone.0018803-Edgar1"></a>Edgar R (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Research  32: 1792–1797.  <ul class="find" data-citedArticleID="2657264" data-doi="10.1093/nar/gkh340"><li><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0018803-Rose1" id="pone.0018803-Rose1"></a>Rose TM, Henikoff JG, Henikoff S (2003) CODEHOP (COnsensus-DEgenerate hybrid oligonucleotide primer) PCR primer design. Nucleic Acids Research  31: 3763–3766.  <ul class="find" data-citedArticleID="2657252" data-doi="10.1093/nar/gkg524"><li><a href="http://dx.doi.org/10.1093/nar/gkg524" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CODEHOP+%28COnsensus-DEgenerate+hybrid+oligonucleotide+primer%29+PCR+primer+design." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CODEHOP+%28COnsensus-DEgenerate+hybrid+oligonucleotide+primer%29+PCR+primer+design.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0018803-Mougou1" id="pone.0018803-Mougou1"></a>Mougou A, Dutech C, Desprez-Loustau M-L (2008) New insights into the identity and origin of the causal agent of oak powdery mildew in Europe. Forest Pathology  38: 275–287.  <ul class="find" data-citedArticleID="2657296" data-doi="10.1111/j.1439-0329.2008.00544.x"><li><a href="http://dx.doi.org/10.1111/j.1439-0329.2008.00544.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+insights+into+the+identity+and+origin+of+the+causal+agent+of+oak+powdery+mildew+in+Europe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+insights+into+the+identity+and+origin+of+the+causal+agent+of+oak+powdery+mildew+in+Europe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0018803-DesprezLoustau1" id="pone.0018803-DesprezLoustau1"></a>Desprez-Loustau M-L, Feau N, Mougou-Hamdane A, Dutech C (2011) Interspecific and intraspecific diversity in oak powdery mildews in Europe: coevolution history and adaptation to their hosts.   DOI:10.1007/s10267-010-0100-5.  <ul class="find-nolinks"></ul></li><li><span class="label">75.
              </span><a name="pone.0018803-Garbelotto1" id="pone.0018803-Garbelotto1"></a>Garbelotto M, Linzer R, Nicolotti G, Gonthier P (2010) Comparing the influences of ecological and evolutionary factors on the successful invasion of a fungal forest pathogen. Biological Invasions  12: 943–957.  <ul class="find" data-citedArticleID="2657276" data-doi="10.1007/s10530-009-9514-4"><li><a href="http://dx.doi.org/10.1007/s10530-009-9514-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparing+the+influences+of+ecological+and+evolutionary+factors+on+the+successful+invasion+of+a+fungal+forest+pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparing+the+influences+of+ecological+and+evolutionary+factors+on+the+successful+invasion+of+a+fungal+forest+pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0018803-Prospero1" id="pone.0018803-Prospero1"></a>Prospero S, Lung-Escarmant B, Dutech C (2008) Genetic structure of an expanding Armillaria root rot fungus (<em>Armillaria ostoyae</em>) population in a managed pine forest in southwestern France. Molecular Ecology  17: 3366–3378.  <ul class="find" data-citedArticleID="2657298" data-doi="10.1111/j.1365-294x.2007.03829.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2007.03829.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+structure+of+an+expanding+Armillaria+root+rot+fungus+%28Armillaria+ostoyae%29+population+in+a+managed+pine+forest+in+southwestern+France." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+structure+of+an+expanding+Armillaria+root+rot+fungus+%28Armillaria+ostoyae%29+population+in+a+managed+pine+forest+in+southwestern+France.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0018803-Carolina1" id="pone.0018803-Carolina1"></a>Carolina A, Padovan B, Sanson G, Brunstein A, Briones M (2005) Fungi evolution revisited: application of the penalized likelihood method to a Bayesian fungal phylogeny provides a new perspective on phylogenetic relationships and divergence dates of Ascomycota groups. Journal of Molecular Evolution  60: 726–735.  <ul class="find" data-citedArticleID="2657314" data-doi="10.1007/s00239-004-0164-y"><li><a href="http://dx.doi.org/10.1007/s00239-004-0164-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fungi+evolution+revisited%3A+application+of+the+penalized+likelihood+method+to+a+Bayesian+fungal+phylogeny+provides+a+new+perspective+on+phylogenetic+relationships+and+divergence+dates+of+Ascomycota+groups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fungi+evolution+revisited%3A+application+of+the+penalized+likelihood+method+to+a+Bayesian+fungal+phylogeny+provides+a+new+perspective+on+phylogenetic+relationships+and+divergence+dates+of+Ascomycota+groups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0018803-Karlsson1" id="pone.0018803-Karlsson1"></a>Karlsson M, Stenlid J (2008) Comparative evolutionary histories of the fungal chitinase gene family reveal non-random size expansions and contractions due to adaptive natural selection. Evolutionary Bioinformatics  4: 47–60.  <ul class="find" data-citedArticleID="2657326"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+evolutionary+histories+of+the+fungal+chitinase+gene+family+reveal+non-random+size+expansions+and+contractions+due+to+adaptive+natural+selection.&amp;auth=&amp;atitle=Comparative+evolutionary+histories+of+the+fungal+chitinase+gene+family+reveal+non-random+size+expansions+and+contractions+due+to+adaptive+natural+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+evolutionary+histories+of+the+fungal+chitinase+gene+family+reveal+non-random+size+expansions+and+contractions+due+to+adaptive+natural+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+evolutionary+histories+of+the+fungal+chitinase+gene+family+reveal+non-random+size+expansions+and+contractions+due+to+adaptive+natural+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0018803-Vijaykrishna1" id="pone.0018803-Vijaykrishna1"></a>Vijaykrishna D, Jeewon R, Hyde K (2006) Molecular taxonomy, origins and evolution of freshwater ascomycetes. Fungal Diversity  23: 351–390.  <ul class="find" data-citedArticleID="2657376"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+taxonomy%2C+origins+and+evolution+of+freshwater+ascomycetes.&amp;auth=&amp;atitle=Molecular+taxonomy%2C+origins+and+evolution+of+freshwater+ascomycetes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+taxonomy%2C+origins+and+evolution+of+freshwater+ascomycetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+taxonomy%2C+origins+and+evolution+of+freshwater+ascomycetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0018803-Zolan1" id="pone.0018803-Zolan1"></a>Zolan ME, Pukkila PJ (1986) Inheritance of DNA methylation in <em>Coprinus cinereus</em>. Molecular and Cellular Biology  6: 195–200.  <ul class="find" data-citedArticleID="2657218" data-doi="10.1007/978-1-60761-103-5_8"><li><a href="http://dx.doi.org/10.1007/978-1-60761-103-5_8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inheritance+of+DNA+methylation+in+Coprinus+cinereus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inheritance+of+DNA+methylation+in+Coprinus+cinereus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0018803-Glass1" id="pone.0018803-Glass1"></a>Glass NL, Donaldson GC (1995) Primer sets developed to amplify conserved genes from filamentous Ascomycetes are useful in differentiating <em>Fusarium</em> species associated with conifers. Applied and Environmental Microbiology  61: 1331–1340.  <ul class="find" data-citedArticleID="2657308"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Primer+sets+developed+to+amplify+conserved+genes+from+filamentous+Ascomycetes+are+useful+in+differentiating+Fusarium+species+associated+with+conifers.&amp;auth=&amp;atitle=Primer+sets+developed+to+amplify+conserved+genes+from+filamentous+Ascomycetes+are+useful+in+differentiating+Fusarium+species+associated+with+conifers." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Primer+sets+developed+to+amplify+conserved+genes+from+filamentous+Ascomycetes+are+useful+in+differentiating+Fusarium+species+associated+with+conifers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Primer+sets+developed+to+amplify+conserved+genes+from+filamentous+Ascomycetes+are+useful+in+differentiating+Fusarium+species+associated+with+conifers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0018803-Helaers1" id="pone.0018803-Helaers1"></a>Helaers R, Milinkovitch MC (2010) MetaPIGA v2.0: maximum likelihood large phylogeny estimation using the metapopulation genetic algorithm and other stochastic heuristics. BMC Bioinformatics  11: 379.  <ul class="find" data-citedArticleID="2657222" data-doi="10.1186/1471-2105-11-379"><li><a href="http://dx.doi.org/10.1186/1471-2105-11-379" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MetaPIGA+v2.0%3A+maximum+likelihood+large+phylogeny+estimation+using+the+metapopulation+genetic+algorithm+and+other+stochastic+heuristics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MetaPIGA+v2.0%3A+maximum+likelihood+large+phylogeny+estimation+using+the+metapopulation+genetic+algorithm+and+other+stochastic+heuristics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0018803-Shimodaira1" id="pone.0018803-Shimodaira1"></a>Shimodaira H, Hasegawa M (1999) Multiple comparisons of Log-likelihoods with applications to phylogenetic inference. Molecular Biology and Evolution  16: 1114–1116.  <ul class="find" data-citedArticleID="2657300" data-doi="10.1093/oxfordjournals.molbev.a026201"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+comparisons+of+Log-likelihoods+with+applications+to+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+comparisons+of+Log-likelihoods+with+applications+to+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0018803-Feau1" id="pone.0018803-Feau1"></a>Feau N, Hamelin RC, Bernier L (2007) Variability of nuclear SSU-rDNA group introns within <em>Septoria</em> species: incongruence with host sequence phylogenies. Journal of Molecular Evolution  64: 489–499.  <ul class="find" data-citedArticleID="2657282" data-doi="10.1007/s00239-005-0309-7"><li><a href="http://dx.doi.org/10.1007/s00239-005-0309-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variability+of+nuclear+SSU-rDNA+group+introns+within+Septoria+species%3A+incongruence+with+host+sequence+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variability+of+nuclear+SSU-rDNA+group+introns+within+Septoria+species%3A+incongruence+with+host+sequence+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.XML" value="191167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.PDF" value="1137937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g001.PNG_L" value="262157"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g001.PNG_M" value="102408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g001.PNG_S" value="11493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g001.TIF" value="460618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g001.PNG_I" value="32483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g002.PNG_L" value="758095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g002.PNG_M" value="319845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g002.PNG_S" value="18587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g002.TIF" value="1714316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g002.PNG_I" value="42359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t001.PNG_L" value="71612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t001.PNG_M" value="54706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t001.PNG_S" value="4999"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t001.TIF" value="310578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t001.PNG_I" value="10244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t002.PNG_L" value="65970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t002.PNG_M" value="48457"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t002.PNG_S" value="4468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t002.TIF" value="297685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t002.PNG_I" value="9024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g003.PNG_L" value="125545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g003.PNG_M" value="95309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g003.PNG_S" value="6973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g003.TIF" value="208478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g003.PNG_I" value="16186"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t003.PNG_L" value="132466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t003.PNG_M" value="99900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t003.PNG_S" value="6051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t003.TIF" value="442190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.t003.PNG_I" value="16628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g004.PNG_L" value="681564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g004.PNG_M" value="412153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g004.PNG_S" value="28727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g004.TIF" value="1364408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g004.PNG_I" value="107880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g005.PNG_L" value="239212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g005.PNG_M" value="306618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g005.PNG_S" value="17144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g005.TIF" value="731632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g005.PNG_I" value="48313"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g006.PNG_L" value="4245807"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g006.PNG_M" value="696458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g006.PNG_S" value="28929"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g006.TIF" value="6774050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.g006.PNG_I" value="124646"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.s001.TIF" value="722311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.s002.TIF" value="830066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.s003.DOC" value="157184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0018803.s004.DOC" value="89600"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0018803&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0018803" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0018803&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0018803&volume=&issue=&title=Finding Single Copy Genes Out of Sequenced Genomes for Multilocus Phylogenetics in Non-Model Fungi&author_name=Nicolas%20Feau%2C%20Thibaut%20Decourcelle%2C%20Claude%20Husson%2C%20Marie-Laure%20Desprez-Loustau%2C%20Cyril%20Dutech&start_page=1&end_page=17" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0018803" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0018803" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0018803" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0018803&amp;t=Finding%20Single%20Copy%20Genes%20Out%20of%20Sequenced%20Genomes%20for%20Multilocus%20Phylogenetics%20in%20Non-Model%20Fungi" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0018803&title=Finding%20Single%20Copy%20Genes%20Out%20of%20Sequenced%20Genomes%20for%20Multilocus%20Phylogenetics%20in%20Non-Model%20Fungi&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0018803&amp;title=Finding%20Single%20Copy%20Genes%20Out%20of%20Sequenced%20Genomes%20for%20Multilocus%20Phylogenetics%20in%20Non-Model%20Fungi" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0018803" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0018803&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Finding Single Copy Genes Out of Sequenced Genomes for Multilocus Phylogenetics in Non-Model Fungi';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0018803';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Finding%20Single%20Copy%20Genes%20Out%20of%20Sequenced%20Genomes%20for%20Multilocus%20Phylogenetics%20in%20Non-Model%20Fungi http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0018803" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0018803" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">




















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="60078"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+genetics%22" title="Search for articles in the subject area:'Fungal genetics'"><div class="flagText">Fungal genetics</div></a>
              <div data-categoryid="18483" data-articleid="60078"
                   data-categoryname="Fungal genetics"
                   class="flagImage" title="Flag 'Fungal genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+genomics%22" title="Search for articles in the subject area:'Fungal genomics'"><div class="flagText">Fungal genomics</div></a>
              <div data-categoryid="33097" data-articleid="60078"
                   data-categoryname="Fungal genomics"
                   class="flagImage" title="Flag 'Fungal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="60078"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="60078"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="60078"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="60078"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="60078"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9461'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6827'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5787&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
