

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Amplification of cox2 (~620 bp) from 2 mg of Up to 129 Years Old Herbarium Specimens, Comparing 19 Extraction Methods and 15 Polymerases</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0003584"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0003584" />

    <meta name="citation_title" content="Amplification of cox2 (~620 bp) from 2 mg of Up to 129 Years Old Herbarium Specimens, Comparing 19 Extraction Methods and 15 Polymerases"/>
    <meta itemprop="name" content="Amplification of cox2 (~620 bp) from 2 mg of Up to 129 Years Old Herbarium Specimens, Comparing 19 Extraction Methods and 15 Polymerases"/>

      <meta name="citation_author" content="Sabine Telle"/>
            <meta name="citation_author_institution" content="University of Hohenheim, Institute of Botany 210, Stuttgart, Germany"/>
      <meta name="citation_author" content="Marco Thines"/>
            <meta name="citation_author_institution" content="University of Hohenheim, Institute of Botany 210, Stuttgart, Germany"/>

    <meta name="citation_date" content="2008/10/31"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0003584.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e3584"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Specific enzymatic amplification of DNA in vitro: the polymerase chain reaction 1986.; citation_author=K Mullis; citation_author=F Faloona; citation_author=S Scharf; citation_author=R Saiki; citation_author=G Horn; citation_journal_title=Biotechnology; citation_volume=24; citation_number=1; citation_pages=17-27; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=DNA sequencing with chain-terminating inhibitors.; citation_author=F Sanger; citation_author=S Nicklen; citation_author=AR Coulson; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=74; citation_number=2; citation_pages=5463-5467; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Amplification and Sequencing of DNA from Fungal Herbarium Specimens.; citation_author=TD Bruns; citation_author=R Fogel; citation_author=JW Taylor; citation_journal_title=Mycologia; citation_volume=82; citation_number=3; citation_pages=175-184; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Instability and decay of the primary structure of DNA.; citation_author=T Lindahl; citation_journal_title=Nature; citation_volume=362; citation_number=4; citation_pages=709-715; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Genetic analyses from ancient DNA.; citation_author=S Pääbo; citation_author=H Poinar; citation_author=D Serre; citation_author=V Jaenicke-Després; citation_author=J Hebler; citation_journal_title=Ann Rev Genet; citation_volume=38; citation_number=5; citation_pages=645-79; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The use of herbarium specimens in DNA phylogenetics: Evaluation and improvement.; citation_author=V Savolainen; citation_author=P Cuénoud; citation_author=R Spichiger; citation_author=MDP Martinez; citation_author=M Crèvecoeur; citation_author=J-F Manen; citation_journal_title=Pl Systematics Evol; citation_volume=197; citation_number=6; citation_pages=87-98; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=DNA extraction and PCR amplification method suitable for fresh, herbarium-stored, lichenized, and other fungi.; citation_author=OF Cubero; citation_author=A Crespo; citation_author=J Fatehi; citation_author=PD Bridge; citation_journal_title=Pl Systematics Evol; citation_volume=216; citation_number=7; citation_pages=243-249; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Comparison of seven DNA extraction and amplification protocols in historical herbarium specimens of Juncaceae.; citation_author=L Drábková; citation_author=J Kirschner; citation_author=Č Vlček; citation_journal_title=Pl Mol Biol Rep; citation_volume=20; citation_number=8; citation_pages=161-175; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Amplification of DNA of Xanthomonas axonopodis pv. citri from historic citrus canker herbarium specimens.; citation_author=W Li; citation_author=RH Brlansky; citation_author=JS Hartung; citation_journal_title=J Microbiological Methods; citation_volume=65; citation_number=9; citation_pages=237-246; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA extraction from frozen fieldcollected and dehydrated herbarium fungal basidiomata: performance of SDS and CTAB-based methods.; citation_author=A Góes-Neto; citation_author=C Loguercio-Leite; citation_author=RT Rosa Trinidad Guerrero; citation_journal_title=Biotemas; citation_volume=18; citation_number=10; citation_pages=19-32; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Isolation of DNA for sequence analysis from herbarium material of some lichen specimens.; citation_author=S Aras; citation_author=D Cansaran; citation_journal_title=Turkish J Bot; citation_volume=30; citation_number=11; citation_pages=449-453; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of different DNA extraction protocols in fresh and herbarium specimens of the genus Dalbergia.; citation_author=RA Ribeiro; citation_author=MB Lovato; citation_journal_title=Genetic Mol Res; citation_volume=6; citation_number=12; citation_pages=173-187; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Successful extraction of DNA from 100-year-old herbarium specimens of the liverwort Bazzania trilobata.; citation_author=K Jankowiak; citation_author=K Buczkowska; citation_author=Z Szweykowska-Kulinska; citation_journal_title=Taxon; citation_volume=54; citation_number=13; citation_pages=335-336; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The utility of aged seeds in DNA banks.; citation_author=C Walters; citation_author=AA Reilley; citation_author=PA Reeves; citation_author=J Baszczak; citation_author=CM Richards; citation_journal_title=Seed Sci Res; citation_volume=16; citation_number=14; citation_pages=169-178; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identity of the mt DNA haplotype(s) of Phytophthora infestans in historical specimens from Irish Potato Famine.; citation_author=KJ May; citation_author=JB Ristaino; citation_journal_title=Mycological Res; citation_volume=108; citation_number=15; citation_pages=471-479; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning of ancient Egyptian mummy DNA.; citation_author=S Pääbo; citation_journal_title=Nature; citation_volume=314; citation_number=16; citation_pages=644-645; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Multiplex amplification of the mammoth mitochondrial genome and the evolution of Elephantidae.; citation_author=J Krause; citation_author=P Dear; citation_author=J Pollack; citation_author=M Slatkin; citation_author=H Spriggs; citation_journal_title=Nature; citation_volume=439; citation_number=17; citation_pages=724-727; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular analysis of Neanderthal DNA from the northern Caucasus.; citation_author=IV Ovchinnikov; citation_author=A Götherström; citation_author=GP Romanova; citation_author=VM Kharitonova; citation_author=K Liden; citation_journal_title=Nature; citation_volume=404; citation_number=18; citation_pages=490-49; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=DNA sequence from Cretaceous period bone fragments.; citation_author=SR Woodward; citation_author=NJ Weyand; citation_author=M Brunnell; citation_journal_title=Science; citation_volume=266; citation_number=19; citation_pages=1229-1232; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Detecting dinosaur DNA.; citation_author=SB Hedges; citation_author=MH Schweitzer; citation_author=S Henikoff; citation_author=MW Allard; citation_author=D Young; citation_journal_title=Science; citation_volume=268; citation_number=20; citation_pages=1191-1193; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Nuclear gene sequences from a late Pleistocene sloth coprolite.; citation_author=H Poinar; citation_author=M Kuch; citation_author=G McDonald; citation_author=P Martin; citation_author=S Pääbo; citation_journal_title=Current Biol; citation_volume=13; citation_number=21; citation_pages=1150-1152; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Ancient olive DNA in pits: preservation, amplification and sequence analysis.; citation_author=R Elbaum; citation_author=C Melamed-Bessudo; citation_author=E Boaretto; citation_author=E Galili; citation_author= Lev-Yadun; citation_journal_title=J Archaeol Sci; citation_volume=33; citation_number=22; citation_pages=77-88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular coproscopy: dung and diet of the extinct ground sploth Nothrotheriops shastensis.; citation_author=HN Poinar; citation_author=M Hofreiter; citation_author=WG Spaulding; citation_author=PS Martin; citation_author=BA Stankiewicz; citation_journal_title=Science; citation_volume=281; citation_number=23; citation_pages=402-406; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=DNA damage and DNA sequence retrieval from ancient tissues.; citation_author=M Höss; citation_author=P Jagura; citation_author=TH Zastawny; citation_author=M Dizdaroglu; citation_author=S Pääbo; citation_journal_title=Nucleic Acids Res; citation_volume=24; citation_number=24; citation_pages=1304-1307; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Instability and decay of the primary structure of DNA.; citation_author=T Lindahl; citation_journal_title=Nature; citation_volume=362; citation_number=25; citation_pages=709-715; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=DNA-extraction from processed wood: A case study for the identification of an endangered timber species (Gonystylus bancanus).; citation_author=J Asif M; citation_author=H Cannon C; citation_journal_title=Pl Mol Biol Reporter; citation_volume=23; citation_number=26; citation_pages=1-8; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Rapid detection of Phytophthora infestans in late blight infected potato and tomato using PCR.; citation_author=CL Trout; citation_author=JB Ristaino; citation_author=M Madritch; citation_author=T Wangsomboondee; citation_journal_title=Pl Dis; citation_volume=81; citation_number=27; citation_pages=1042-1048; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity of citrus bacterial canker pathogens preserved in herbarium specimens.; citation_author=W Li; citation_author=Q Song; citation_author=RH Brlansky; citation_author=JS Hartung; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=104; citation_number=28; citation_pages=18427-18432; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=PCR amplification of the Irish potato famine pathogen from historic specimens.; citation_author=JB Ristaino; citation_author=CT Groves; citation_author=GR Parra; citation_journal_title=Nature; citation_volume=411; citation_number=29; citation_pages=695-697; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Extraction and amplification of nuclear DNA from herbarium specimens of emmer wheat: a method for assessing DNA preservation by maximum amplicon length recovery.; citation_author=DL Lister; citation_author=MA Bower; citation_author=CJ Howe; citation_author=MK Jones; citation_journal_title=Taxon; citation_volume=57; citation_number=30; citation_pages=254-258; citation_date=2008; " />
      <meta name="citation_reference" content="citation_author=SB Lee; citation_author=JW Taylor; citation_title=PCR protocols: a guide to methods and applications; citation_number=31; citation_date=1990; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=Chelex® 100 as a medium for simple extraction of DNA for PCR-based typing from forensic material.; citation_author=PS Walsh; citation_author=DA Metzger; citation_author=R Higuchi; citation_journal_title=Biotechniques; citation_volume=10; citation_number=32; citation_pages=506-513; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Evaluation of an alkaline lysis method for the extraction of DNA from whole blood and forensic strains for STR analyses.; citation_author=M Klintschar; citation_author=F Neuhuber; citation_journal_title=J Forensic Sci; citation_volume=45; citation_number=33; citation_pages=669-678; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=An Asian origin for a 10,000-year old domesticated plant in the Americas.; citation_author=DL Erickson; citation_author=BD Smith; citation_author=AC Clarke; citation_author=DH Sandweiss; citation_author=D Tuross; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=20; citation_number=34; citation_pages=18315-18320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A cox2 molecular phylogeny of the peronosporomycetes.; citation_author=DS Hudspeth; citation_author=SA Nadler; citation_author=ME Hudspeth; citation_journal_title=Mycologia; citation_volume=92; citation_number=35; citation_pages=674-684; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0003584" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Amplification of cox2 (~620 bp) from 2 mg of Up to 129 Years Old Herbarium Specimens, Comparing 19 Extraction Methods and 15 Polymerases"/>
    <meta name="twitter:description" content="
During the past years an increasing number of studies have focussed on the use of herbarium specimens for molecular phylogenetic investigations and several comparative studies have been published. However, in the studies reported so far usually rather large amounts of material (typically around 100 mg) were sampled for DNA extraction. This equals an amount roughly equivalent to 8 cm2 of a medium thick leaf. For investigating the phylogeny of plant pathogens, such large amounts of tissue are usually not available or would irretrievably damage the specimens. Through systematic comparison of 19 DNA extraction protocols applied to only 2 mg of infected leaf tissue and testing 15 different DNA polymerases, we could successfully amplify a mitochondrial DNA region (cox2; ~620 bp) from herbarium specimens well over a hundred years old. We conclude that DNA extraction and the choice of DNA polymerase are crucial factors for successful PCR amplification from small samples of historic herbarium specimens. Through a combination of suitable DNA extraction protocols and DNA polymerases, only a fraction of the preserved plant material commonly used is necessary for successful PCR amplification. This facilitates the potential use of a far larger number of preserved specimens for molecular phylogenetic investigation and provides access to a wealth of genetic information in preserved in specimens deposited in herbaria around the world without reducing their scientific or historical value.
"/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0003584.g003"/>

  <meta property="og:title" content="Amplification of cox2 (~620 bp) from 2 mg of Up to 129 Years Old Herbarium Specimens, Comparing 19 Extraction Methods and 15 Polymerases" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0003584" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4921'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7843'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9969&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003584">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0003584" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0003584&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Amplification of <i>cox2</i> (~620 bp) from 2 mg of Up to 129 Years Old Herbarium Specimens, Comparing 19 Extraction Methods and 15 Polymerases
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sabine Telle, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  University of Hohenheim, Institute of Botany 210, Stuttgart, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marco Thines
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:thines@uni-hohenheim.de">thines@uni-hohenheim.de</a></p>

                <p>Affiliation:
                  University of Hohenheim, Institute of Botany 210, Stuttgart, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 31, 2008</li>
    <li>DOI: 10.1371/journal.pone.0003584</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0003584" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0003584" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0003584" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0003584" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0003584-g001" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0003584-t001" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0003584-g002" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0003584-g003" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0003584-t002" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0003584-t003" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0003584">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0003584" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>During the past years an increasing number of studies have focussed on the use of herbarium specimens for molecular phylogenetic investigations and several comparative studies have been published. However, in the studies reported so far usually rather large amounts of material (typically around 100 mg) were sampled for DNA extraction. This equals an amount roughly equivalent to 8 cm<sup>2</sup> of a medium thick leaf. For investigating the phylogeny of plant pathogens, such large amounts of tissue are usually not available or would irretrievably damage the specimens. Through systematic comparison of 19 DNA extraction protocols applied to only 2 mg of infected leaf tissue and testing 15 different DNA polymerases, we could successfully amplify a mitochondrial DNA region (<em>cox2</em>; ~620 bp) from herbarium specimens well over a hundred years old. We conclude that DNA extraction and the choice of DNA polymerase are crucial factors for successful PCR amplification from small samples of historic herbarium specimens. Through a combination of suitable DNA extraction protocols and DNA polymerases, only a fraction of the preserved plant material commonly used is necessary for successful PCR amplification. This facilitates the potential use of a far larger number of preserved specimens for molecular phylogenetic investigation and provides access to a wealth of genetic information in preserved in specimens deposited in herbaria around the world without reducing their scientific or historical value.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Telle S, Thines M (2008) Amplification of <em>cox2</em> (~620 bp) from 2 mg of Up to 129 Years Old Herbarium Specimens, Comparing 19 Extraction Methods and 15 Polymerases. PLoS ONE 3(10):
          e3584.
            doi:10.1371/journal.pone.0003584</p><p><strong>Editor: </strong>Robert DeSalle, American Museum of Natural History, United States of America</p><p><strong>Received:</strong> July 31, 2008; <strong>Accepted:</strong> October 7, 2008; <strong>Published:</strong> October 31, 2008</p><p><strong>Copyright:</strong> © 2008 Telle, Thines. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was funded by a research grant of the German Science Foundation (DFG) granted to MT. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Millions of herbarium specimens are deposited in herbaria around the world. The primary aim of these institutions is the permanent conservation of a diverse sample of plants, algae and fungi for documentation and comparative investigations, particularly of a taxonomic nature. With the advent of PCR <a href="#pone.0003584-Mullis1">[1]</a> and cheap sequencing techniques <a href="#pone.0003584-Sanger1">[2]</a>, molecular phylogenetic investigations have been the most important source of phylogenetic data and an important touchstone for morphology based taxonomies. The wealth of herbarium specimens preserved in international herbaria has, however, scarcely been exploited in this context, although already Bruns et al. <a href="#pone.0003584-Bruns1">[3]</a> addressed the topic of extracting DNA from fungal herbarium specimens. The main reason for this has been major difficulties in extracting sufficient DNA from older herbarium specimens that is of good enough quality for use in PCR amplification of the target sequences. Unfortunately the DNA from herbarium specimens is often highly degraded, depending on the conditions of drying and storage <a href="#pone.0003584-Lindahl1">[4]</a>. During storage, numerous alterations of the DNA take place, a topic extensively reviewed by Pääbo et al. <a href="#pone.0003584-Pbo1">[5]</a>. The second obstacle with respect to the use of herbarium material has been that many of the specimens collected are dating back to the 19<sup>th</sup> and early 20<sup>th</sup> century, thousands of samples were collected for morphological and taxonomic comparison. Many of these specimens are to be considered historic artefacts of great value, and this is especially true for type specimens. Therefore, only minute amounts of material could be taken from the specimens without doing irreparable damage to them. As an example, the Wageningen herbarium generally allows 50 mg (maximum 5%) of a non-type herbarium specimen to be removed (<a href="http://www.bis.wur.nl/UK/Links/DNAProtocol/">http://www.bis.wur.nl/UK/Links/DNAProtoc​ol/</a>). This equates roughly to a 4 cm<sup>2</sup> area of a medium thick leaf. During the past ten years, the extraction from herbarium samples has been comparatively investigated several times <a href="#pone.0003584-Savolainen1">[6]</a>–<a href="#pone.0003584-Ribeiro1">[12]</a> and Jankoviak et al. <a href="#pone.0003584-Jankowiak1">[13]</a> and Walters et al. <a href="#pone.0003584-Walters1">[14]</a> have reported amplification of up to 500 bp fragments from preserved 100 year old herbarium specimens and seeds respectively. However, in most comparative studies, roughly 100 mg of the specimens were used. With respect to herbarium vouchers of small plants or fungi, and especially phytopathogenic Oomycota and Eumycota, taking this amount of material would often result in the complete destruction of the specimens. For phytopathogenic species it is not uncommon that only minor parts of the specimens are affected by the disease and often herbarium specimen consist only of a single leaf. Therefore it was the aim of this study to test a variety of DNA extraction protocols (19 protocols, including modifications) and DNA polymerases (15 tested) to evaluate the best-suited method to extract and to amplify DNA from 2 mg of up to 130 years old preserved leaf tissue infected with oomycete pathogens.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The 19 different DNA extraction methods yielded highly diverging amounts of DNA, as presented in <a href="#pone-0003584-t001">Table 1</a>. In all cases, the DNA was highly fragmented and gave a smear on agarose gels, revealing mostly fragment sizes below 500 bp (data not shown). DNA extracts were colourless to brownish, depending on the method used. All DNA-amplicons sequenced revealed the target sequence and no contamination was observed. In an initial test, an amount of DNA extracted which equalled 0.05 mg of dried plant material was used in the PCR reactions (<a href="#pone-0003584-g001">Fig. 1</a>). Only 13 DNA extraction protocols gave PCR amplicons for one or more specimens. From these, 3 DNA extracts (gained by the methods AnaP, EznF, EriP) gave a 350 bp amplification product from a 129 year old sample.</p>
<div class="figure" id="pone-0003584-g001"><div class="img"><a name="pone-0003584-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0003584.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0003584.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Results of the PCR amplification of cox2 fragments from up to 129 years old herbarium specimens (leaves infected by biotrophic oomycete pathogens), using an amount of DNA equalling 0.05 mg of starting material.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Black fields: amplicon amount &gt;90 ng, dark grey: amplicon amount 30–90 ng, grey: amplicon amount 10–30 ng, light grey: amplicon amount &lt;10 ng, white: no amplicon detectable, asterisks: very faint band visible (≪10 ng). Upper half of each row: ~620 bp fragment, lower half: ~350 bp fragment.</p>
<span>doi:10.1371/journal.pone.0003584.g001</span></div><div class="figure" id="pone-0003584-t001"><div class="img"><a name="pone-0003584-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0003584.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0003584.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Total DNA amount, OD<span class="xref"><sup>*</sup></span> values at 260 nm and standard quality scores per sample and extraction method.</span></strong></p><span>doi:10.1371/journal.pone.0003584.t001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>When using 10 ng of extracted DNA per reaction (<a href="#pone-0003584-g002">Fig. 2</a>), eight protocols gave only PCR amplicons of the <em>cox2</em> region for one or two of the specimens, and with none of these protocols (AncB, RisN, RisNC, RisP, Ris PC, FerG, CheX, NaOH), amplification of the target sequence was obtained for samples more than 100 years old. Only the DNA extracts from five protocols gave 350 bp amplification products for the 129 year old sample (SigP, AnaP, EznF, EznP, EriP) in either absolute (i.e. extract amount) or relative (i.e. specific amount in ng) PCR. The larger fragment (~620 bp) could only be amplified from the samples less than 100 years old, when using the Fermentas Taq DNA polymerase. It was observed that with increasing age of the samples PCR amplification performance decreased.</p>
<div class="figure" id="pone-0003584-g002"><div class="img"><a name="pone-0003584-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0003584.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0003584.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Results of the PCR amplification of cox2 fragments from up to 129 years old herbarium specimens (leaves infected by biotrophic oomycete pathogens), from 10 ng of extracted DNA.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Black fields: amplicon amount &gt;90 ng, dark grey: amplicon amount 30–90 ng, grey: amplicon amount 10–30 ng, light grey: amplicon amount &lt;10 ng, white: no amplicon detectable, asterisks: very faint band visible (≪10 ng). Upper half of each row: ~620 bp fragment, lower half: ~350 bp fragment.</p>
<span>doi:10.1371/journal.pone.0003584.g002</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>Of the DNA extraction methods applied in this study, four gave consistent <em>cox2</em> amplification throughout all samples tested (SigP, AnaP, EznF, EriP). For subsequent experiments, three of these were selected, representing different methodologies (AnaP, EznF, EriP). To these, the extraction method of May and Ristaino (15: RisNC), which was reported to be suitable for amplification from minute amounts of a more than 150 year old herbarium specimen, a commercial DNA extraction kit especially designed for extraction of ancient DNA (AncA), and a simple genomic DNA extraction kit (Fermentas, FerG) were added. The DNA obtained by these protocols from the three oldest samples (82, 101, 129 years) was used to test the amplification performance of 15 DNA polymerases, encompassing 5 conventional Taq polymerases, one hot-start Taq polymerase, a DNA- Repair Kit, 3 Taq polymerase based blends, 8 proofreading enzymes, and a genetically engineered DNA polymerase. All polymerases tested gave amplification in the positive control, which was represented by a herbarium specimen less than three years old. For the herbarium specimens tested, the DNA polymerases tested showed highly different amplification performances (<a href="#pone-0003584-g003">Fig. 3</a>). Only four of the polymerases tested yielded the ~620 bp fragment from at least one of the DNA extracts obtained by at least one DNA protocol (BioTaqRed, Mango-Taq, peqGoldTaq, BIO-X-ACT short). In addition, also the PreCR-mix gave amplification of the ~620 bp fragment for one of the DNA extracts tested. Only two polymerases were able to amplify the ~620 bp fragment from the oldest sample (BioTaq Red, Mango-Taq). The best performance was exhibited by the Mango-Taq DNA polymerase, which was the only polymerase which was able to amplify the ~620 bp amplification product from the 102 year old sample.</p>
<div class="figure" id="pone-0003584-g003"><div class="img"><a name="pone-0003584-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0003584.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0003584.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comparison of the performance of various DNA polymerases.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>Square fields indicate amplification of the ~350 bp fragment with different intensity. Black fields: amplicon amount &gt;90 ng, dark grey: amplicon amount 30–90 ng, grey: amplicon amount 10–30 ng, light grey: amplicon amount &lt;10 ng, white: no amplicon detectable, asterisks: very faint band visible (≪10 ng); white dots indicate additional amplification of the ~620 bp fragment. +: positive control, −: negative (water) control.</p>
<span>doi:10.1371/journal.pone.0003584.g003</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>A cross comparison of DNA extraction methods and DNA polymerase used revealed that the DNA extracts (10 ng of DNA per PCR reaction) gained from the Analytikjena DNA extraction kit (AnaP) were suitable for the use of most polymerases, except for two enzymes. Also the DNA extracted by means of the E.Z.N.A. forensic kit (EznF) was amplifiable by a wide range of polymerases, including proofreading enzymes. The extraction method using N-phenacylthiazolium bromide (EriP) did not yield DNA consistently amplifiable by a wide range of polymerases, but when using Mango-Taq (Bioline) or BioTaq Red (Bioline), the ~620 bp fragment could be obtained from the oldest herbarium sample. DNA obtained from the May and Ristaino protocol (15: RisNC) did not yield optimal results, and dilution resulted in a decrease of PCR product (data not shown). In general, AnaP and EriP, in combination with using Mango-Taq were giving best results.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The use of ancient and historic DNA is becoming ever more popular since the first major breakthroughs in amplifying DNA from Egyptian mummies <a href="#pone.0003584-Pbo2">[16]</a>, mammoths <a href="#pone.0003584-Krause1">[17]</a> and Neanderthals <a href="#pone.0003584-Ovchinnikov1">[18]</a>. For a time, it seemed like there would be virtually be no age limit for the survival of DNA <a href="#pone.0003584-Woodward1">[19]</a>. However, more often than not it later turned out that reports of antediluvian DNA were the result of contamination with more recent DNA (e.g. 20). However, regarding the success with animal remains <a href="#pone.0003584-Poinar1">[21]</a>, archeological olive pits <a href="#pone.0003584-Elbaum1">[22]</a> and a well preserved coprolite from the Miocene <a href="#pone.0003584-Poinar2">[23]</a>, it seems odd that it still is considered a major challenge to obtain DNA from herbarium samples only a few decades old. The reason for this is probably the suboptimal drying and storage conditions for many historical herbarium specimens, which results in DNA modification and degradation <a href="#pone.0003584-Hss1">[24]</a>, <a href="#pone.0003584-Lindahl2">[25]</a>. But as herbarium specimens constitute an invaluable resource for the study of genetic diversity of plants, algae and fungi, several comparative studies have addressed the problem of extracting DNA from herbarium specimens. In most cases, only few different DNA extraction protocols were applied <a href="#pone.0003584-Drbkov1">[8]</a>, <a href="#pone.0003584-GesNeto1">[10]</a>, <a href="#pone.0003584-Aras1">[11]</a>, <a href="#pone.0003584-Ribeiro1">[12]</a>, <a href="#pone.0003584-AsifM1">[26]</a>, with the notable exception of Li et al. <a href="#pone.0003584-Li1">[9]</a>. This is regrettable, considering the importance of the subject. The most successfully applied methods were either CTAB-based methods <a href="#pone.0003584-Savolainen1">[6]</a>, <a href="#pone.0003584-Trout1">[27]</a>, or commercially available DNA extraction kits <a href="#pone.0003584-Drbkov1">[8]</a>. Especially the DNeasy plant extraction kit (DNeasy Plant Mini Kit, Qiagen) in particular has proved to be the best for the extraction of DNA from historic specimens <a href="#pone.0003584-Drbkov1">[8]</a>, <a href="#pone.0003584-Elbaum1">[22]</a>. Also in our study, this kit was amongst the best of the commercially available kits tested, with only two other kits performing slightly better (AnaP, SigP) on the oldest samples.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In most cases, roughly 100 mg of plant material were used. Also Li et al. <a href="#pone.0003584-Li1">[9]</a> used relatively high amounts of plant tissue and there was no standardized starting material for each of the protocols they tested. In their latest study <a href="#pone.0003584-Li2">[28]</a>, 30 mg of plant material were used, which although a great improvement, still constitutes major loss of valuable tissue, especially in smaller specimens. Therefore it was an important aim of this study to test DNA extraction methods on smaller amounts of plant tissue, and we chose to extract DNA from 2 mg dry weight of starting material, which is 15 times less than the amount used by Li et al. <a href="#pone.0003584-Li2">[28]</a>. As Ristaino et al. <a href="#pone.0003584-Ristaino1">[29]</a> had previously reported the successful amplification from a more than 150 year old specimen from the Irish Potato Famine, from only few square millimetres of infected leaf tissue, this method was tested on the specimens used by us, also with several modifications. Our investigations did not yield similarly promising results with this method, and although the smaller <em>cox2</em>-fragment could be obtained when using the best-suited polymerase, other DNA extraction methods performed significantly better. Also further dilution of the extracts, as done by May and Ristaino <a href="#pone.0003584-May1">[15]</a>, did not improve the results but resulted in decreasing amplicon amounts until complete loss. Therefore, it might be appropriate to resample the specimens investigated by Ristaino <a href="#pone.0003584-Li1">[9]</a> and May and Ristaino <a href="#pone.0003584-May1">[15]</a>, using additional phylogenetic markers in order to track historic epidemics of <em>Phytophthora infestans</em> in more detail.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Very recently, Lister et al. <a href="#pone.0003584-Lister1">[30]</a>, reported that the combination of PTB and a commercial DNA purification kit yielded results superior to previous methods and was successfully applied to amplify a 350 bp fragment from an a specimen that was over a 100 years old. The amount of material used was 10–40 mg, but amplification could only be obtained from grains and not from leaf blades, most likely due to inhibitory substances in the DNA extracts. Nonetheless, the progress reported by Lister et al. <a href="#pone.0003584-Lister1">[30]</a> in comparison to previous studies is obvious and it seems likely that this success was mainly due to the application of PTB, which has been successfully used in studies investigating Miocene ground sloth coprolite <a href="#pone.0003584-Poinar2">[23]</a> and processed wood <a href="#pone.0003584-AsifM1">[26]</a>. Adding PTB to the lysis buffer of the Analyticjena plant DNA extraction kit (final concentration of 2.5 mM) for other samples than those processed for this comparative study, provided a quick and reliable kit-based extraction protocol for the recovery of amplifiable ancient DNA. This protocol has resulted in the amplification of the ~620 bp fragment for samples more than 130 years old (Thines et al., in preparation).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In none of the studies focussing on improving the extraction of DNA from herbarium specimens, were the resulting DNA extracts tested with a variety of DNA polymerases. This study clearly demonstrates that not only the DNA extraction protocol used but also the choice of polymerase greatly influences the chances of getting successful PCR amplification. Apart from different enzyme performance, this is most likely due to the purity of the enzymes and the chemistry of the polymerase buffers supplied along with the enzymes. Considering that it was only possible to test a fraction of the plethora of DNA polymerases that are currently promoted by numerous companies, it is obvious that the full potential of DNA polymerase comparison for amplification from historic herbarium specimens is far from being fully exploited.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>With respect to the historical, cultural and scientific value of herbarium specimens, additional studies are required to further minimize the amount of material necessary for reliable PCR-amplification. This would potentially enable scientists to exploit the ‘untapped genetic treasure trove’ that is currently in thousands of herbaria worldwide. This is especially crucial for plant parasites, which are easily distributed because of the global trade in seeds and plants and pose a major thread for horticulture and agriculture.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Preparation of the plant material and DNA extraction</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>In total, 50–100 mg of infected plant tissue from herbarium voucher specimens (<a href="#pone-0003584-t002">Table 2</a>) were taken and disrupted in a mixer mill (MM2, Retsch, Germany), using six iron balls of 3 mm diameter per sample. The powdered tissue was than portioned to 2 mg portions in 2 ml reaction tubes (Sarstedt, Germany) using a precision gauge (R200D, Sartorius, Germany). 19 reaction tubes could be prepared for each of the samples and were subjected to DNA extraction. Sample preparation and DNA extractions were carried out in a contaminant free environment. No recent samples of the organisms investigated were processed in the laboratory before or during the experiments reported here.</p>
<div class="figure" id="pone-0003584-t002"><div class="img"><a name="pone-0003584-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0003584.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0003584.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Herbarium vouchers investigated.</span></strong></p><span>doi:10.1371/journal.pone.0003584.t002</span></div><a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The DNA extraction protocols used are summarized in <a href="#pone-0003584-t003">Table 3</a>, with the manufacturers given there. All DNA extractions were carried out exactly as given in the respective references or according to the manufacturer's instructions in case of commercially available DNA extraction kits. The resulting DNA was eluted in sterile water (DEPC-treated, Carl Roth, Germany).</p>
<div class="figure" id="pone-0003584-t003"><div class="img"><a name="pone-0003584-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0003584" data-uri="info:doi/10.1371/journal.pone.0003584.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0003584.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0003584.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0003584.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0003584.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0003584.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>List of protocols used for DNA extraction.</span></strong></p><span>doi:10.1371/journal.pone.0003584.t003</span></div><a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>The kit for ancient DNA extraction was carried out three times exactly as given in the manual with the three different, supplied lysis solutions (AncA, AncA2, AncB, respectively), and DNA was eluted in 100 µl water. In the SigP extraction method the longer drying variant was chosen. The lysis incubation step was carried out for 25 min in the EznP method. In the genomic DNA kit (Fermentas, Germany) the note for small amounts of DNA was followed, including precipitation for 20 hours and dissolving the DNA obtained in 20 µl water. The RisN method <a href="#pone.0003584-Ristaino1">[29]</a> was preformed with doubled extraction volume. In a modified procedure, 5% polyvinylpyrolidone (PVP) was added to the nuclei lysis buffer (RisP). Both procedures were also modified according to May and Ristaino (15: RisNC, RisPC) with the column based purification and concentration step reported there.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>DNA concentration was measured using a spectrophotometer for small volumes (NanoDrop ND-1000, NanoDrop Technologies, USA).</p>


<h4>PCR and sequencing</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For PCR-amplification of the mitochondrial cytochrome oxidase subunit II (<em>cox2</em>) gene region modified primers and PCR conditions of Hudspeth et al. <a href="#pone.0003584-Hudspeth1">[35]</a> were used. For all experiments, a single tube semi-nested approach was chosen, using forward (<span class="gene">3′-GGCAAATGGGTTTTCAAGATC-5′</span>), reverse (<span class="gene">3′-CCATGATTAATACCACAAATTT-5′</span>) and the forward nested (<span class="gene">3′-GGTAGTCAATGGTATTGG-5′</span>) primer in a single reaction. All of these primers are oomycete specific and would not amplify human, plant, fungal or bacterial DNA. Negative and positive PCR controls were done for all experiments reported in this study. To ensure maximum comparability, the PCR data presented is based on the same chemical background where possible, i.e. a single PCR mastermix was used were applicable. In addition, PCR reactions were carried out in only two PCR runs on the same thermal cycler (Primus 96, Peqlab, Germany).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>For the initial comparisons, the standard recombinant Taq DNA polymerase from Fermentas (Fermentas, Germany) was used with magnesium chloride and ammonium sulphate 10× PCR buffer supplied by the manufacturer. Apart from PCR-buffer, PCR reactions contained 0.2 mM dNTPs (Fermentas, Germany), 2.0 mM MgCl<sub>2</sub>, 1.0 µM of each primer (Sigma-Aldrich, Germany) and 1.2 units of Taq DNA polymerase per 30 µl. PCR cycling conditions were as follows: 4 min at 94°C initial denaturation; 36 cycles of 40 s at 94°C, 40 s at 51°C, 1 min at 72°C; and a final extension of 4 min at 72°C.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>For assessing the best suited DNA polymerase to amplify the <em>cox2</em> region of the pathogens from the oldest herbarium specimens, the 15 polymerases listed in <a href="#pone-0003584-g003">Fig. 3</a> were applied according to the manufacturer's instructions, using 1 unit of polymerase and the specific polymerase buffers supplied in 30 µl reactions with the same cycling conditions as mentioned before except for an additional 15 min, 94°C activation step in the hot-start DNA polymerase tested.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>Amplicon confirmation by sequencing was carried out by a commercial sequencing company (GATC, Germany) using the modified cox2-R primer.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The curators of the herbaria BR (National Botanic Garden of Belgium) and BP (Hungarian Natural History Museum) are gratefully acknowledged for allowing us to investigate the specimens in their keeping. Without their cooperation, this study had not been possible. We are indebted to Gordon Beakes for critically proofreading the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MT. Performed the experiments: ST. Analyzed the data: ST. Wrote the paper: MT.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0003584-Mullis1" id="pone.0003584-Mullis1"></a>Mullis K, Faloona F, Scharf S, Saiki R, Horn G, et al.  (1992) Specific enzymatic amplification of DNA in vitro: the polymerase chain reaction 1986. Biotechnology  24: 17–27.  <ul class="find" data-citedArticleID="1239268" data-doi="10.1101/sqb.1986.051.01.032"><li><a href="http://dx.doi.org/10.1101/sqb.1986.051.01.032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+enzymatic+amplification+of+DNA+in+vitro%3A+the+polymerase+chain+reaction+1986." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+enzymatic+amplification+of+DNA+in+vitro%3A+the+polymerase+chain+reaction+1986.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0003584-Sanger1" id="pone.0003584-Sanger1"></a>Sanger F, Nicklen S, Coulson AR (1977) DNA sequencing with chain-terminating inhibitors. Proc Natl Acad Sci USA  74: 5463–5467.  <ul class="find" data-citedArticleID="1239248" data-doi="10.1073/pnas.74.12.5463"><li><a href="http://dx.doi.org/10.1073/pnas.74.12.5463" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+sequencing+with+chain-terminating+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+sequencing+with+chain-terminating+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0003584-Bruns1" id="pone.0003584-Bruns1"></a>Bruns TD, Fogel R, Taylor JW (1990) Amplification and Sequencing of DNA from Fungal Herbarium Specimens. Mycologia  82: 175–184.  <ul class="find" data-citedArticleID="1239262" data-doi="10.2307/3759846"><li><a href="http://dx.doi.org/10.2307/3759846" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amplification+and+Sequencing+of+DNA+from+Fungal+Herbarium+Specimens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amplification+and+Sequencing+of+DNA+from+Fungal+Herbarium+Specimens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0003584-Lindahl1" id="pone.0003584-Lindahl1"></a>Lindahl T (1993) Instability and decay of the primary structure of DNA. Nature  362: 709–715.  <ul class="find" data-citedArticleID="1239266" data-doi="10.1038/362709a0"><li><a href="http://dx.doi.org/10.1038/362709a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Instability+and+decay+of+the+primary+structure+of+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Instability+and+decay+of+the+primary+structure+of+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0003584-Pbo1" id="pone.0003584-Pbo1"></a>Pääbo S, Poinar H, Serre D, Jaenicke-Després V, Hebler J, et al.  (2004) Genetic analyses from ancient DNA. Ann Rev Genet  38: 645–79.  <ul class="find" data-citedArticleID="1239270" data-doi="10.1146/annurev.genet.37.110801.143214"><li><a href="http://dx.doi.org/10.1146/annurev.genet.37.110801.143214" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+analyses+from+ancient+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+analyses+from+ancient+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0003584-Savolainen1" id="pone.0003584-Savolainen1"></a>Savolainen V, Cuénoud P, Spichiger R, Martinez MDP, Crèvecoeur M, Manen J-F (1995) The use of herbarium specimens in DNA phylogenetics: Evaluation and improvement. Pl Systematics Evol  197: 87–98.  <ul class="find" data-citedArticleID="1239250" data-doi="10.1007/bf00984634"><li><a href="http://dx.doi.org/10.1007/bf00984634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+herbarium+specimens+in+DNA+phylogenetics%3A+Evaluation+and+improvement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+herbarium+specimens+in+DNA+phylogenetics%3A+Evaluation+and+improvement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0003584-Cubero1" id="pone.0003584-Cubero1"></a>Cubero OF, Crespo A, Fatehi J, Bridge PD (1998) DNA extraction and PCR amplification method suitable for fresh, herbarium-stored, lichenized, and other fungi. Pl Systematics Evol  216: 243–249.  <ul class="find" data-citedArticleID="1239260" data-doi="10.1007/bf01084401"><li><a href="http://dx.doi.org/10.1007/bf01084401" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+extraction+and+PCR+amplification+method+suitable+for+fresh%2C+herbarium-stored%2C+lichenized%2C+and+other+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+extraction+and+PCR+amplification+method+suitable+for+fresh%2C+herbarium-stored%2C+lichenized%2C+and+other+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0003584-Drbkov1" id="pone.0003584-Drbkov1"></a>Drábková L, Kirschner J, Vlček Č (2002) Comparison of seven DNA extraction and amplification protocols in historical herbarium specimens of Juncaceae. Pl Mol Biol Rep  20: 161–175.  <ul class="find" data-citedArticleID="1239240" data-doi="10.1007/bf02799431"><li><a href="http://dx.doi.org/10.1007/bf02799431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+seven+DNA+extraction+and+amplification+protocols+in+historical+herbarium+specimens+of+Juncaceae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+seven+DNA+extraction+and+amplification+protocols+in+historical+herbarium+specimens+of+Juncaceae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0003584-Li1" id="pone.0003584-Li1"></a>Li W, Brlansky RH, Hartung JS (2006) Amplification of DNA of <em>Xanthomonas axonopodis</em> pv. <em>citri</em> from historic citrus canker herbarium specimens. J Microbiological Methods  65: 237–246.  <ul class="find" data-citedArticleID="1239234" data-doi="10.1016/j.mimet.2005.07.014"><li><a href="http://dx.doi.org/10.1016/j.mimet.2005.07.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amplification+of+DNA+of+Xanthomonas+axonopodis+pv.+citri+from+historic+citrus+canker+herbarium+specimens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amplification+of+DNA+of+Xanthomonas+axonopodis+pv.+citri+from+historic+citrus+canker+herbarium+specimens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0003584-GesNeto1" id="pone.0003584-GesNeto1"></a>Góes-Neto A, Loguercio-Leite C, Rosa Trinidad Guerrero RT (2005) DNA extraction from frozen fieldcollected and dehydrated herbarium fungal basidiomata: performance of SDS and CTAB-based methods. Biotemas  18: 19–32.  <ul class="find" data-citedArticleID="1239214"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+extraction+from+frozen+fieldcollected+and+dehydrated+herbarium+fungal+basidiomata%3A+performance+of+SDS+and+CTAB-based+methods.&amp;auth=&amp;atitle=DNA+extraction+from+frozen+fieldcollected+and+dehydrated+herbarium+fungal+basidiomata%3A+performance+of+SDS+and+CTAB-based+methods." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+extraction+from+frozen+fieldcollected+and+dehydrated+herbarium+fungal+basidiomata%3A+performance+of+SDS+and+CTAB-based+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+extraction+from+frozen+fieldcollected+and+dehydrated+herbarium+fungal+basidiomata%3A+performance+of+SDS+and+CTAB-based+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0003584-Aras1" id="pone.0003584-Aras1"></a>Aras S, Cansaran D (2006) Isolation of DNA for sequence analysis from herbarium material of some lichen specimens. Turkish J Bot  30: 449–453.  <ul class="find" data-citedArticleID="1239236"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation+of+DNA+for+sequence+analysis+from+herbarium+material+of+some+lichen+specimens.&amp;auth=&amp;atitle=Isolation+of+DNA+for+sequence+analysis+from+herbarium+material+of+some+lichen+specimens." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+DNA+for+sequence+analysis+from+herbarium+material+of+some+lichen+specimens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+DNA+for+sequence+analysis+from+herbarium+material+of+some+lichen+specimens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0003584-Ribeiro1" id="pone.0003584-Ribeiro1"></a>Ribeiro RA, Lovato MB (2007) Comparative analysis of different DNA extraction protocols in fresh and herbarium specimens of the genus <em>Dalbergia</em>. Genetic Mol Res  6: 173–187.  <ul class="find" data-citedArticleID="1239256"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+analysis+of+different+DNA+extraction+protocols+in+fresh+and+herbarium+specimens+of+the+genus+Dalbergia.&amp;auth=&amp;atitle=Comparative+analysis+of+different+DNA+extraction+protocols+in+fresh+and+herbarium+specimens+of+the+genus+Dalbergia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+different+DNA+extraction+protocols+in+fresh+and+herbarium+specimens+of+the+genus+Dalbergia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+different+DNA+extraction+protocols+in+fresh+and+herbarium+specimens+of+the+genus+Dalbergia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0003584-Jankowiak1" id="pone.0003584-Jankowiak1"></a>Jankowiak K, Buczkowska K, Szweykowska-Kulinska Z (2005) Successful extraction of DNA from 100-year-old herbarium specimens of the liverwort <em>Bazzania trilobata</em>. Taxon  54: 335–336.  <ul class="find" data-citedArticleID="1239244" data-doi="10.2307/25065361"><li><a href="http://dx.doi.org/10.2307/25065361" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Successful+extraction+of+DNA+from+100-year-old+herbarium+specimens+of+the+liverwort+Bazzania+trilobata." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Successful+extraction+of+DNA+from+100-year-old+herbarium+specimens+of+the+liverwort+Bazzania+trilobata.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0003584-Walters1" id="pone.0003584-Walters1"></a>Walters C, Reilley AA, Reeves PA, Baszczak J, Richards CM (2006) The utility of aged seeds in DNA banks. Seed Sci Res  16: 169–178.  <ul class="find" data-citedArticleID="1239272" data-doi="10.1079/ssr2006246"><li><a href="http://dx.doi.org/10.1079/ssr2006246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+utility+of+aged+seeds+in+DNA+banks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+utility+of+aged+seeds+in+DNA+banks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0003584-May1" id="pone.0003584-May1"></a>May KJ, Ristaino JB (2004) Identity of the mt DNA haplotype(s) of <em>Phytophthora infestans</em> in historical specimens from Irish Potato Famine. Mycological Res  108: 471–479.  <ul class="find" data-citedArticleID="1239280" data-doi="10.1017/s0953756204009876"><li><a href="http://dx.doi.org/10.1017/s0953756204009876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identity+of+the+mt+DNA+haplotype%28s%29+of+Phytophthora+infestans+in+historical+specimens+from+Irish+Potato+Famine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identity+of+the+mt+DNA+haplotype%28s%29+of+Phytophthora+infestans+in+historical+specimens+from+Irish+Potato+Famine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0003584-Pbo2" id="pone.0003584-Pbo2"></a>Pääbo S (1985) Molecular cloning of ancient Egyptian mummy DNA. Nature  314: 644–645.  <ul class="find" data-citedArticleID="1239232" data-doi="10.1038/314644a0"><li><a href="http://dx.doi.org/10.1038/314644a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+of+ancient+Egyptian+mummy+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+of+ancient+Egyptian+mummy+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0003584-Krause1" id="pone.0003584-Krause1"></a>Krause J, Dear P, Pollack J, Slatkin M, Spriggs H, et al.  (2006) Multiplex amplification of the mammoth mitochondrial genome and the evolution of Elephantidae. Nature  439: 724–727.  <ul class="find" data-citedArticleID="1239278" data-doi="10.1038/nature04432"><li><a href="http://dx.doi.org/10.1038/nature04432" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+amplification+of+the+mammoth+mitochondrial+genome+and+the+evolution+of+Elephantidae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+amplification+of+the+mammoth+mitochondrial+genome+and+the+evolution+of+Elephantidae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0003584-Ovchinnikov1" id="pone.0003584-Ovchinnikov1"></a>Ovchinnikov IV, Götherström A, Romanova GP, Kharitonova VM, Liden K, et al.  (2000) Molecular analysis of Neanderthal DNA from the northern Caucasus. Nature  404: 490–49.  <ul class="find" data-citedArticleID="1239220" data-doi="10.1038/35006625"><li><a href="http://dx.doi.org/10.1038/35006625" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+analysis+of+Neanderthal+DNA+from+the+northern+Caucasus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+analysis+of+Neanderthal+DNA+from+the+northern+Caucasus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0003584-Woodward1" id="pone.0003584-Woodward1"></a>Woodward SR, Weyand NJ, Brunnell M (1994) DNA sequence from Cretaceous period bone fragments. Science  266: 1229–1232.  <ul class="find" data-citedArticleID="1239246" data-doi="10.1126/science.7973705"><li><a href="http://dx.doi.org/10.1126/science.7973705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+sequence+from+Cretaceous+period+bone+fragments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+sequence+from+Cretaceous+period+bone+fragments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0003584-Hedges1" id="pone.0003584-Hedges1"></a>Hedges SB, Schweitzer MH, Henikoff S, Allard MW, Young D, et al.  (1995) Detecting dinosaur DNA. Science  268: 1191–1193.  <ul class="find" data-citedArticleID="1239258" data-doi="10.1126/science.7761839"><li><a href="http://dx.doi.org/10.1126/science.7761839" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+dinosaur+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+dinosaur+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0003584-Poinar1" id="pone.0003584-Poinar1"></a>Poinar H, Kuch M, McDonald G, Martin P, Pääbo S, et al.  (2003) Nuclear gene sequences from a late Pleistocene sloth coprolite. Current Biol  13: 1150–1152.  <ul class="find" data-citedArticleID="1239252" data-doi="10.1016/s0960-9822(03)00450-0"><li><a href="http://dx.doi.org/10.1016/s0960-9822(03)00450-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+gene+sequences+from+a+late+Pleistocene+sloth+coprolite." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+gene+sequences+from+a+late+Pleistocene+sloth+coprolite.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0003584-Elbaum1" id="pone.0003584-Elbaum1"></a>Elbaum R, Melamed-Bessudo C, Boaretto E, Galili E, Lev-Yadun , et al.  (2006) Ancient olive DNA in pits: preservation, amplification and sequence analysis. J Archaeol Sci  33: 77–88.  <ul class="find" data-citedArticleID="1239282" data-doi="10.1016/j.jas.2005.06.011"><li><a href="http://dx.doi.org/10.1016/j.jas.2005.06.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+olive+DNA+in+pits%3A+preservation%2C+amplification+and+sequence+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+olive+DNA+in+pits%3A+preservation%2C+amplification+and+sequence+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0003584-Poinar2" id="pone.0003584-Poinar2"></a>Poinar HN, Hofreiter M, Spaulding WG, Martin PS, Stankiewicz BA, et al.  (1998) Molecular coproscopy: dung and diet of the extinct ground sploth <em>Nothrotheriops shastensis</em>. Science  281: 402–406.  <ul class="find" data-citedArticleID="1239276" data-doi="10.1126/science.281.5375.402"><li><a href="http://dx.doi.org/10.1126/science.281.5375.402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+coproscopy%3A+dung+and+diet+of+the+extinct+ground+sploth+Nothrotheriops+shastensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+coproscopy%3A+dung+and+diet+of+the+extinct+ground+sploth+Nothrotheriops+shastensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0003584-Hss1" id="pone.0003584-Hss1"></a>Höss M, Jagura P, Zastawny TH, Dizdaroglu M, Pääbo S (1996) DNA damage and DNA sequence retrieval from ancient tissues. Nucleic Acids Res  24: 1304–1307.  <ul class="find" data-citedArticleID="1239242" data-doi="10.1093/nar/24.7.1304"><li><a href="http://dx.doi.org/10.1093/nar/24.7.1304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+damage+and+DNA+sequence+retrieval+from+ancient+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+damage+and+DNA+sequence+retrieval+from+ancient+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0003584-Lindahl2" id="pone.0003584-Lindahl2"></a>Lindahl T (1993) Instability and decay of the primary structure of DNA. Nature  362: 709–715.  <ul class="find" data-citedArticleID="1239224" data-doi="10.1038/362709a0"><li><a href="http://dx.doi.org/10.1038/362709a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Instability+and+decay+of+the+primary+structure+of+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Instability+and+decay+of+the+primary+structure+of+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0003584-AsifM1" id="pone.0003584-AsifM1"></a>Asif M J, Cannon C H (2005) DNA-extraction from processed wood: A case study for the identification of an endangered timber species (<em>Gonystylus bancanus</em>). Pl Mol Biol Reporter  23: 1–8.  <ul class="find" data-citedArticleID="1239230" data-doi="10.1007/bf02772709"><li><a href="http://dx.doi.org/10.1007/bf02772709" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA-extraction+from+processed+wood%3A+A+case+study+for+the+identification+of+an+endangered+timber+species+%28Gonystylus+bancanus%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA-extraction+from+processed+wood%3A+A+case+study+for+the+identification+of+an+endangered+timber+species+%28Gonystylus+bancanus%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0003584-Trout1" id="pone.0003584-Trout1"></a>Trout CL, Ristaino JB, Madritch M, Wangsomboondee T (1996) Rapid detection of Phytophthora infestans in late blight infected potato and tomato using PCR. Pl Dis  81: 1042–1048.  <ul class="find" data-citedArticleID="1239216" data-doi="10.1094/pdis.1997.81.9.1042"><li><a href="http://dx.doi.org/10.1094/pdis.1997.81.9.1042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+detection+of+Phytophthora+infestans+in+late+blight+infected+potato+and+tomato+using+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+detection+of+Phytophthora+infestans+in+late+blight+infected+potato+and+tomato+using+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0003584-Li2" id="pone.0003584-Li2"></a>Li W, Song Q, Brlansky RH, Hartung JS (2007) Genetic diversity of citrus bacterial canker pathogens preserved in herbarium specimens. Proc Natl Acad Sci USA  104: 18427–18432.  <ul class="find" data-citedArticleID="1239218" data-doi="10.1073/pnas.0705590104"><li><a href="http://dx.doi.org/10.1073/pnas.0705590104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+of+citrus+bacterial+canker+pathogens+preserved+in+herbarium+specimens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+of+citrus+bacterial+canker+pathogens+preserved+in+herbarium+specimens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0003584-Ristaino1" id="pone.0003584-Ristaino1"></a>Ristaino JB, Groves CT, Parra GR (2001) PCR amplification of the Irish potato famine pathogen from historic specimens. Nature  411: 695–697.  <ul class="find" data-citedArticleID="1239226"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=PCR+amplification+of+the+Irish+potato+famine+pathogen+from+historic+specimens.&amp;auth=&amp;atitle=PCR+amplification+of+the+Irish+potato+famine+pathogen+from+historic+specimens." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PCR+amplification+of+the+Irish+potato+famine+pathogen+from+historic+specimens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PCR+amplification+of+the+Irish+potato+famine+pathogen+from+historic+specimens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0003584-Lister1" id="pone.0003584-Lister1"></a>Lister DL, Bower MA, Howe CJ, Jones MK (2008) Extraction and amplification of nuclear DNA from herbarium specimens of emmer wheat: a method for assessing DNA preservation by maximum amplicon length recovery. Taxon  57: 254–258.  <ul class="find" data-citedArticleID="1239228"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Extraction+and+amplification+of+nuclear+DNA+from+herbarium+specimens+of+emmer+wheat%3A+a+method+for+assessing+DNA+preservation+by+maximum+amplicon+length+recovery.&amp;auth=&amp;atitle=Extraction+and+amplification+of+nuclear+DNA+from+herbarium+specimens+of+emmer+wheat%3A+a+method+for+assessing+DNA+preservation+by+maximum+amplicon+length+recovery." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extraction+and+amplification+of+nuclear+DNA+from+herbarium+specimens+of+emmer+wheat%3A+a+method+for+assessing+DNA+preservation+by+maximum+amplicon+length+recovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extraction+and+amplification+of+nuclear+DNA+from+herbarium+specimens+of+emmer+wheat%3A+a+method+for+assessing+DNA+preservation+by+maximum+amplicon+length+recovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0003584-Lee1" id="pone.0003584-Lee1"></a>Lee SB, Taylor JW (1990) PCR protocols: a guide to methods and applications.  Innes MA, Gelfand DH, Sninsky JJ, White TJ, editors. San Diego: Academic Press.   <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0003584-Walsh1" id="pone.0003584-Walsh1"></a>Walsh PS, Metzger DA, Higuchi R (1991) Chelex® 100 as a medium for simple extraction of DNA for PCR-based typing from forensic material. Biotechniques  10: 506–513.  <ul class="find" data-citedArticleID="1239254"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chelex%AE+100+as+a+medium+for+simple+extraction+of+DNA+for+PCR-based+typing+from+forensic+material.&amp;auth=&amp;atitle=Chelex%AE+100+as+a+medium+for+simple+extraction+of+DNA+for+PCR-based+typing+from+forensic+material." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chelex%C2%AE+100+as+a+medium+for+simple+extraction+of+DNA+for+PCR-based+typing+from+forensic+material." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chelex%C2%AE+100+as+a+medium+for+simple+extraction+of+DNA+for+PCR-based+typing+from+forensic+material.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0003584-Klintschar1" id="pone.0003584-Klintschar1"></a>Klintschar M, Neuhuber F (2000) Evaluation of an alkaline lysis method for the extraction of DNA from whole blood and forensic strains for STR analyses. J Forensic Sci  45: 669–678.  <ul class="find" data-citedArticleID="1239274"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evaluation+of+an+alkaline+lysis+method+for+the+extraction+of+DNA+from+whole+blood+and+forensic+strains+for+STR+analyses.&amp;auth=&amp;atitle=Evaluation+of+an+alkaline+lysis+method+for+the+extraction+of+DNA+from+whole+blood+and+forensic+strains+for+STR+analyses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+an+alkaline+lysis+method+for+the+extraction+of+DNA+from+whole+blood+and+forensic+strains+for+STR+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+an+alkaline+lysis+method+for+the+extraction+of+DNA+from+whole+blood+and+forensic+strains+for+STR+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0003584-Erickson1" id="pone.0003584-Erickson1"></a>Erickson DL, Smith BD, Clarke AC, Sandweiss DH, Tuross D (2005) An Asian origin for a 10,000-year old domesticated plant in the Americas. Proc Natl Acad Sci USA  20: 18315–18320.  <ul class="find" data-citedArticleID="1239264" data-doi="10.1073/pnas.0509279102"><li><a href="http://dx.doi.org/10.1073/pnas.0509279102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Asian+origin+for+a+10%2C000-year+old+domesticated+plant+in+the+Americas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Asian+origin+for+a+10%2C000-year+old+domesticated+plant+in+the+Americas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0003584-Hudspeth1" id="pone.0003584-Hudspeth1"></a>Hudspeth DS, Nadler SA, Hudspeth ME (2000) A cox2 molecular phylogeny of the peronosporomycetes. Mycologia  92: 674–684.  <ul class="find" data-citedArticleID="1239222" data-doi="10.2307/3761425"><li><a href="http://dx.doi.org/10.2307/3761425" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cox2+molecular+phylogeny+of+the+peronosporomycetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cox2+molecular+phylogeny+of+the+peronosporomycetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.XML" value="95746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.PDF" value="268762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g001.PNG_L" value="37215"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g001.PNG_M" value="94789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g001.PNG_S" value="26714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g001.TIF" value="202218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g001.PNG_I" value="40047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t001.PNG_L" value="364752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t001.PNG_M" value="398991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t001.PNG_S" value="22965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t001.TIF" value="1436928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t001.PNG_I" value="47466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e001.PNG" value="7649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e001.TIF" value="22264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e002.PNG" value="7659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e002.TIF" value="22280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e003.PNG" value="7649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e003.TIF" value="22264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e004.PNG" value="7659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e004.TIF" value="22280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e005.PNG" value="7649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e005.TIF" value="22264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e006.PNG" value="7659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e006.TIF" value="22280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e007.PNG" value="7649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e007.TIF" value="22264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e008.PNG" value="7659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e008.TIF" value="22280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e009.PNG" value="7649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e009.TIF" value="22264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e010.PNG" value="7659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.e010.TIF" value="22280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g002.PNG_L" value="40561"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g002.PNG_M" value="113077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g002.PNG_S" value="36116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g002.TIF" value="217716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g002.PNG_I" value="45206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g003.PNG_L" value="81400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g003.PNG_M" value="244853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g003.PNG_S" value="25832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g003.TIF" value="412100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.g003.PNG_I" value="34515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t002.PNG_L" value="104794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t002.PNG_M" value="139931"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t002.PNG_S" value="14101"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t002.TIF" value="438932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t002.PNG_I" value="18812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t003.PNG_L" value="202485"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t003.PNG_M" value="273744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t003.PNG_S" value="19913"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t003.TIF" value="926146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0003584.t003.PNG_I" value="36472"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0003584&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0003584" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0003584&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0003584&volume=&issue=&title=Amplification of cox2 (~620 bp) from 2 mg of Up to 129 Years Old Herbarium Specimens, Comparing 19 Extraction Methods and 15 Polymerases&author_name=Sabine%20Telle%2C%20Marco%20Thines&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0003584" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0003584" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0003584" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0003584&amp;t=Amplification%20of%20cox2%20(~620%20bp)%20from%202%20mg%20of%20Up%20to%20129%20Years%20Old%20Herbarium%20Specimens%2C%20Comparing%2019%20Extraction%20Methods%20and%2015%20Polymerases" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0003584&title=Amplification%20of%20cox2%20(~620%20bp)%20from%202%20mg%20of%20Up%20to%20129%20Years%20Old%20Herbarium%20Specimens%2C%20Comparing%2019%20Extraction%20Methods%20and%2015%20Polymerases&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0003584&amp;title=Amplification%20of%20cox2%20(~620%20bp)%20from%202%20mg%20of%20Up%20to%20129%20Years%20Old%20Herbarium%20Specimens%2C%20Comparing%2019%20Extraction%20Methods%20and%2015%20Polymerases" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0003584" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0003584&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Amplification of cox2 (~620 bp) from 2 mg of Up to 129 Years Old Herbarium Specimens, Comparing 19 Extraction Methods and 15 Polymerases';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0003584';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Amplification%20of%20cox2%20(~620%20bp)%20from%202%20mg%20of%20Up%20to%20129%20Years%20Old%20Herbarium%20Specimens%2C%20Comparing%2019%20Extraction%20Methods%20and%2015%20Polymerases http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0003584" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0003584" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+extraction%22" title="Search for articles in the subject area:'DNA extraction'"><div class="flagText">DNA extraction</div></a>
              <div data-categoryid="19965" data-articleid="29988"
                   data-categoryname="DNA extraction"
                   class="flagImage" title="Flag 'DNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+polymerase%22" title="Search for articles in the subject area:'DNA polymerase'"><div class="flagText">DNA polymerase</div></a>
              <div data-categoryid="40357" data-articleid="29988"
                   data-categoryname="DNA polymerase"
                   class="flagImage" title="Flag 'DNA polymerase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Leaves%22" title="Search for articles in the subject area:'Leaves'"><div class="flagText">Leaves</div></a>
              <div data-categoryid="17987" data-articleid="29988"
                   data-categoryname="Leaves"
                   class="flagImage" title="Flag 'Leaves' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleogenetics%22" title="Search for articles in the subject area:'Paleogenetics'"><div class="flagText">Paleogenetics</div></a>
              <div data-categoryid="33815" data-articleid="29988"
                   data-categoryname="Paleogenetics"
                   class="flagImage" title="Flag 'Paleogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="29988"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genetics%22" title="Search for articles in the subject area:'Plant genetics'"><div class="flagText">Plant genetics</div></a>
              <div data-categoryid="17433" data-articleid="29988"
                   data-categoryname="Plant genetics"
                   class="flagImage" title="Flag 'Plant genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="29988"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerases%22" title="Search for articles in the subject area:'Polymerases'"><div class="flagText">Polymerases</div></a>
              <div data-categoryid="17983" data-articleid="29988"
                   data-categoryname="Polymerases"
                   class="flagImage" title="Flag 'Polymerases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2892'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4197'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9248&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
