

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Episodic Evolution and Adaptation of Chloroplast Genomes in Ancestral Grasses</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0005297"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0005297" />

    <meta name="citation_title" content="Episodic Evolution and Adaptation of Chloroplast Genomes in Ancestral Grasses"/>
    <meta itemprop="name" content="Episodic Evolution and Adaptation of Chloroplast Genomes in Ancestral Grasses"/>

      <meta name="citation_author" content="Bojian Zhong"/>
            <meta name="citation_author_institution" content="School of Life Sciences, Fudan University, Shanghai, China"/>
      <meta name="citation_author" content="Takahiro Yonezawa"/>
            <meta name="citation_author_institution" content="School of Life Sciences, Fudan University, Shanghai, China"/>
      <meta name="citation_author" content="Yang Zhong"/>
            <meta name="citation_author_institution" content="School of Life Sciences, Fudan University, Shanghai, China"/>
      <meta name="citation_author" content="Masami Hasegawa"/>
            <meta name="citation_author_institution" content="School of Life Sciences, Fudan University, Shanghai, China"/>

    <meta name="citation_date" content="2009/4/24"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0005297.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e5297"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="4"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The origin of grass-dominated ecosystems.; citation_author=BF Jacobs; citation_author=JD Kingston; citation_author=LL Jacobs; citation_journal_title=Ann Mo Bot Gard; citation_volume=86; citation_number=1; citation_pages=590-643; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Relative rates of nucleotide substitution at the rbcL locus of monocotyledonous plants.; citation_author=BS Gaut; citation_author=SV Muse; citation_author=WD Clark; citation_author=MT Clegg; citation_journal_title=J Mol Evol; citation_volume=35; citation_number=2; citation_pages=292-303; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Extensive variation in evolutionary rate of rbcL gene sequences among seed plants.; citation_author=J Bousquet; citation_author=SH Strauss; citation_author=AH Doerksen; citation_author=RA Price; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=89; citation_number=3; citation_pages=7844-7848; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Interlocus comparisons of the nucleotide substitution process in the chloroplast genome.; citation_author=SV Muse; citation_author=BS Gaut; citation_journal_title=Genetics; citation_volume=146; citation_number=4; citation_pages=393-399; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Dating the monocot–dicot divergence and the origin of core eudicots using whole chloroplast genomes.; citation_author=S-M Chaw; citation_author=C-C Chang; citation_author=H-L Chen; citation_author=W-H Li; citation_journal_title=J Mol Evol; citation_volume=58; citation_number=5; citation_pages=424-441; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analysis of 81 genes from 64 plastid genomes resolves relationships in angiosperms and identifies genome-scale evolutionary patterns.; citation_author=RK Jansen; citation_author=Z Cai; citation_author=LA Raubeson; citation_author=H Daniell; citation_author=CW dePamphilis; citation_journal_title=Proc Natl Acad Sci; citation_volume=104; citation_number=6; citation_pages=19369-19374; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Using plastid genome-scale data to resolve enigmatic relationships among basal angiosperms.; citation_author=MJ Moore; citation_author=CD Bell; citation_author=PS Soltis; citation_author=DE Soltis; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=49; citation_number=7; citation_pages=19363-19368; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Chloroplast genome phylogenetics: why we need independent approaches to plant molecular evolution.; citation_author=W Martin; citation_author=O Deusch; citation_author=N Stawski; citation_author=N Grunheit; citation_author=V Goremykin; citation_journal_title=TRENDS in Plant Science; citation_volume=10; citation_number=8; citation_pages=203-209; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identifying the basal angiosperm node in chloroplast genome phylogenies: sampling one's way out of the Felsenstein zone.; citation_author=J Leebens-Mack; citation_author=LA Raubeson; citation_author=L Cui; citation_author=JV Kuehl; citation_author=MH Fourcade; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=9; citation_pages=1948-1963; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dinosaur coprolites and the early evolution of grasses and grazers.; citation_author=V Prasad; citation_author=CAE Stroemberg; citation_author=H Alimohammadian; citation_author=A Sahni; citation_journal_title=Science; citation_volume=310; citation_number=10; citation_pages=1177-1180; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dinosaurs dined on grass.; citation_author=DR Piperno; citation_author=HD Sues; citation_journal_title=Science; citation_volume=310; citation_number=11; citation_pages=1126-1128; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Estimating the rate of evolution of the rate of molecular evolution.; citation_author=JL Thorne; citation_author=H Kishino; citation_author=IS Painter; citation_journal_title=Mol Biol Evol; citation_volume=15; citation_number=12; citation_pages=1647-1657; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Performance of a divergence time estimation method under a probabilistic model of rate evolution.; citation_author=H Kishino; citation_author=JL Thorne; citation_author=WJ Bruno; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=13; citation_pages=352-361; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A nonparametric approach to estimating divergence times in the absence of rate constancy.; citation_author=MJ Sanderson; citation_journal_title=Mol Biol Evol; citation_volume=14; citation_number=14; citation_pages=1218-1232; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Inferring speciation times under an episodic molecular clock.; citation_author=B Rannala; citation_author=Z Yang; citation_journal_title=Syst Biol; citation_volume=56; citation_number=15; citation_pages=453-466; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A compound Poisson process for relaxing the molecular clock.; citation_author=JP Huelsenbeck; citation_author=B Larget; citation_author=D Swofford; citation_journal_title=Genetics; citation_volume=154; citation_number=16; citation_pages=1879-1892; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Relaxed phylogenetics and dating with confidence.; citation_author=AJ Drummond; citation_author=SYW Ho; citation_author=MJ Phillips; citation_author=A Rambaut; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=17; citation_pages=e88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Date of the monocot-dicot divergence estimated from chloroplast chloroplast DNA sequence data.; citation_author=KH Wolfe; citation_author=MY Gouy; citation_author=W Yang; citation_author=PM Sharp; citation_author=W-H Li; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=86; citation_number=18; citation_pages=6201-6205; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=PAML 4: Phylogenetic Analysis by Maximum Likelihood.; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=19; citation_pages=1586-1591; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The age of the grasses and clusters of origins of C4 photosynthesis.; citation_author=A Vicentini; citation_author=JC Barber; citation_author=SS Aliscioni; citation_author=LM Giussani; citation_author=EA Kellogg; citation_journal_title=Global Change Biol; citation_volume=14; citation_number=20; citation_pages=2963-2977; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=A codon-based model of nucleotide substitution for protein-coding DNA sequences.; citation_author=N Goldman; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=11; citation_number=21; citation_pages=725-736; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A likelihood approach for comparing synonymous and nonsynonymous nucleotide substitution rates, with application to the chloroplast genome.; citation_author=SV Muse; citation_author=BS Gaut; citation_journal_title=Mol Biol Evol; citation_volume=11; citation_number=22; citation_pages=715-724; citation_date=1994; " />
      <meta name="citation_reference" content="citation_author=Z Yang; citation_title=Computational Molecular Evolution; citation_number=23; citation_date=2006; citation_publisher=Oxford Univ Press; " />
      <meta name="citation_reference" content="citation_title=Information theory and an extension of the maximum likelihood principle.; citation_author=H Akaike; citation_title=Second International Symposium on Information Theory; citation_number=24; citation_pages=267-281; citation_date=1973; citation_publisher=Akademiai Kiado; " />
      <meta name="citation_reference" content="citation_title=Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level.; citation_author=J Zhang; citation_author=R Nielsen; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=25; citation_pages=1-8; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bayes empirical Bayes inference of amino acids sites under positive selection.; citation_author=Z Yang; citation_author=WSW Wong; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=26; citation_pages=1107-1118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Whole chloroplast genome comparison of rice, maize, and wheat: implications for chloroplast gene diversification and phylogeny of cereals.; citation_author=Y Matsuoka; citation_author=Y Yamazaki; citation_author=Y Ogihara; citation_author=K Tsunewaki; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=27; citation_pages=2084-2091; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gondwanan evolution of the grass alliance of families (Poales).; citation_author=K Bremer; citation_journal_title=Evolution; citation_volume=56; citation_number=28; citation_pages=1374-1387; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular evidence on plant divergence times.; citation_author=MJ Sanderson; citation_author=JL Thorne; citation_author=N Wikstroem; citation_author=K Bremer; citation_journal_title=Amer J Bot; citation_volume=91; citation_number=29; citation_pages=1656-1665; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bayesian models of episodic evolution support a late Precambrian explosive diversification of the Metazoa.; citation_author=S Aris-Brosou; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=30; citation_pages=1947-1954; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Retroposon analysis of major ceatcean lineages: The monophyly of toothed whales and the paraphyly of river dolphins.; citation_author=M Nikaido; citation_author=F Matsuno; citation_author=H Hamilton; citation_author=RL Brownell; citation_author=Y Cao; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=31; citation_pages=7384-7389; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Time scale of eutherian evolution estimated without assuming a constant rate of molecular evolution.; citation_author=M Hasegawa; citation_author=J Thorne; citation_author=H Kishino; citation_journal_title=Genes Genet Syst; citation_volume=78; citation_number=32; citation_pages=267-283; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Divergence dates for Malagasy lemurs estimated from multiple gene loci: geological and evolutionary context.; citation_author=AD Yoder; citation_author=Z Yang; citation_journal_title=Mol Ecol; citation_volume=13; citation_number=33; citation_pages=757-773; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A general comparison of relaxed molecular clock models.; citation_author=T Lepage; citation_author=D Bryant; citation_author=H Philippe; citation_author=N Lartillot; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=34; citation_pages=2669-2680; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Robust time estimation reconciles views of the antiquity of placental mammals.; citation_author=Y Kitazoe; citation_author=H Kishino; citation_author=PJ Waddell; citation_author=N Nakajima; citation_author=T Okabayashi; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=35; citation_pages=e384; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Strong mitochondrial DNA support for a Cretaceous origin of modern avian lineages.; citation_author=JW Brown; citation_author=JS Rest; citation_author=J Garcia-Moreno; citation_author=MD Sorenson; citation_author=DP Mindell; citation_journal_title=BMC Biology; citation_volume=2008, 6; citation_number=36; citation_pages=6; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Rooting and dating maples (Acer) with an uncorrelated-rates molecular clock: Implications for North American/Asian disjunctions.; citation_author=SS Renner; citation_author=GW Grimm; citation_author=GM Schneeweiss; citation_author=TF Stuessy; citation_author=RE Ricklefs; citation_journal_title=Syst Biol; citation_volume=57; citation_number=37; citation_pages=795-808; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Whole-gene positive selection, elevated synonymous substitution rates, duplication, and indel evolution of the chloroplast clpP1 gene.; citation_author=P Erixon; citation_author=B Oxelman; citation_journal_title=PLoS ONE; citation_volume=3; citation_number=38; citation_pages=e1386; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analyses of Geraniaceae plastid DNA reveal unprecedented patterns of increased nucleotide substitutions.; citation_author=MM Guisinger; citation_author=JV Kuehl; citation_author=JL Boore; citation_author=RK Jansen; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=105; citation_number=39; citation_pages=18424-18429; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Rates of molecular evolution are linked to life history in flowering plants.; citation_author=SA Smith; citation_author=MJ Donoghue; citation_journal_title=Science; citation_volume=322; citation_number=40; citation_pages=86-89; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=The fossil history of the monocotyledons.; citation_author=PS Herendeen; citation_author=PR Crane; citation_title=Monocotyledon: Systematics and Evolution; citation_number=41; citation_pages=1-21; citation_date=1995; citation_publisher=Royal Botanic Gardens, Kew; " />
      <meta name="citation_reference" content="citation_title=Evolutionary history of Poales.; citation_author=HP Linder; citation_author=PJ Rudall; citation_journal_title=Annu Rev Ecol Evol Syst; citation_volume=36; citation_number=42; citation_pages=107-124; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bayesian estimation of species divergence times under a molecular clock using multiple fossil calibrations with soft bounds.; citation_author=Z Yang; citation_author=B Rannala; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=43; citation_pages=212-226; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Statistical methods for detecting molecular adaptation.; citation_author=Z Yang; citation_author=B Bielawski; citation_journal_title=TREE; citation_volume=15; citation_number=44; citation_pages=496-503; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Complete sequence of the maize chloroplast genome: Gene content, hotspots of divergence and fine tuning of genetic information by transcript editing.; citation_author=RM Maier; citation_author=K Neckermann; citation_author=GL Igloi; citation_author=H Kossel; citation_journal_title=J Mol Biol; citation_volume=251; citation_number=45; citation_pages=614-628; citation_date=1995; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0005297" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Episodic Evolution and Adaptation of Chloroplast Genomes in Ancestral Grasses"/>
    <meta name="twitter:description" content="
Background
It has been suggested that the chloroplast genomes of the grass family, Poaceae, have undergone an elevated evolutionary rate compared to most other angiosperms, yet the details of this phenomenon have remained obscure. To know how the rate change occurred during evolution, estimation of the time-scale with reliable calibrations is needed. The recent finding of 65 Ma grass phytoliths in Cretaceous dinosaur coprolites places the diversification of the grasses to the Cretaceous period, and provides a reliable calibration in studying the tempo and mode of grass chloroplast evolution.

Methodology/Principal Findings
By using chloroplast genome data from angiosperms and by taking account of new paleontological evidence, we now show that episodic rate acceleration both in terms of non-synonymous and synonymous substitutions occurred in the common ancestral branch of the core Poaceae (a group formed by rice, wheat, maize, and their allies) accompanied by adaptive evolution in several chloroplast proteins, while the rate reverted to the slow rate typical of most monocot species in the terminal branches.

Conclusions/Significance
Our finding of episodic rate acceleration in the ancestral grasses accompanied by adaptive molecular evolution has a profound bearing on the evolution of grasses, which form a highly successful group of plants. The widely used model for estimating divergence times was based on the assumption of correlated rates between ancestral and descendant lineages. However, the assumption is proved to be inadequate in approximating the episodic rate acceleration in the ancestral grasses, and the assumption of independent rates is more appropriate. This finding has implications for studies of molecular evolutionary rates and time-scale of evolution in other groups of organisms.
"/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0005297.g003"/>

  <meta property="og:title" content="Episodic Evolution and Adaptation of Chloroplast Genomes in Ancestral Grasses" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0005297" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4536'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9899'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1751&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005297">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0005297" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0005297&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Episodic Evolution and Adaptation of Chloroplast Genomes in Ancestral Grasses
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bojian Zhong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Life Sciences, Fudan University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takahiro Yonezawa, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Life Sciences, Fudan University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yang Zhong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Life Sciences, Fudan University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Masami Hasegawa
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:hasegawa@fudan.edu.cn">hasegawa@fudan.edu.cn</a></p>

                <p>Affiliation:
                  School of Life Sciences, Fudan University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 24, 2009</li>
    <li>DOI: 10.1371/journal.pone.0005297</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0005297" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0005297" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0005297" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0005297" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0005297-g001" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0005297-g002" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0005297-g003" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0005297-t001" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0005297-t002" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0005297-t003" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0005297-t004" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0005297-t005" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0005297-t006" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0005297">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0005297" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>It has been suggested that the chloroplast genomes of the grass family, Poaceae, have undergone an elevated evolutionary rate compared to most other angiosperms, yet the details of this phenomenon have remained obscure. To know how the rate change occurred during evolution, estimation of the time-scale with reliable calibrations is needed. The recent finding of 65 Ma grass phytoliths in Cretaceous dinosaur coprolites places the diversification of the grasses to the Cretaceous period, and provides a reliable calibration in studying the tempo and mode of grass chloroplast evolution.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>By using chloroplast genome data from angiosperms and by taking account of new paleontological evidence, we now show that episodic rate acceleration both in terms of non-synonymous and synonymous substitutions occurred in the common ancestral branch of the core Poaceae (a group formed by rice, wheat, maize, and their allies) accompanied by adaptive evolution in several chloroplast proteins, while the rate reverted to the slow rate typical of most monocot species in the terminal branches.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our finding of episodic rate acceleration in the ancestral grasses accompanied by adaptive molecular evolution has a profound bearing on the evolution of grasses, which form a highly successful group of plants. The widely used model for estimating divergence times was based on the assumption of correlated rates between ancestral and descendant lineages. However, the assumption is proved to be inadequate in approximating the episodic rate acceleration in the ancestral grasses, and the assumption of independent rates is more appropriate. This finding has implications for studies of molecular evolutionary rates and time-scale of evolution in other groups of organisms.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zhong B, Yonezawa T, Zhong Y, Hasegawa M (2009) Episodic Evolution and Adaptation of Chloroplast Genomes in Ancestral Grasses. PLoS ONE 4(4):
          e5297.
            doi:10.1371/journal.pone.0005297</p><p><strong>Editor: </strong>Simon Joly, McGill University, Canada</p><p><strong>Received:</strong> January 29, 2009; <strong>Accepted:</strong> March 19, 2009; <strong>Published:</strong> April 24, 2009</p><p><strong>Copyright:</strong> © 2009 Zhong et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The study was financially supported by Shanghai Leading Academic Discipline Project (B111) and National Infrastructure of Natural Resources for Science and Technology (2005DKA21403) to Yang Zhong. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The grass family, Poaceae, is one of the largest plant families, comprising about 10,000 species including the most important agricultural plants, rice, wheat and maize, and grass-dominated ecosystems comprise about one-third of Earth's vegetative cover and support a vast number of animals <a href="#pone.0005297-Jacobs1">[1]</a>. It has long been suggested that the chloroplast (chl) genomes of the grass family have undergone an elevated evolutionary rate compared to other angiosperms <a href="#pone.0005297-Gaut1">[2]</a>–<a href="#pone.0005297-Chaw1">[5]</a>, yet little is known when, why and how this rate change occurred.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>To examine how the rate change occurred during evolution, it is prerequisite to know the time-scale of evolution. It has become increasingly feasible to estimate the phylogenetic tree and the time-scale of Angiosperm evolution by using chl genome sequences <a href="#pone.0005297-Chaw1">[5]</a>–<a href="#pone.0005297-LeebensMack1">[9]</a>. A reliable calibration is necessary to obtain reliable time estimates, but lack of good fossil evidence of the ancestral grasses has prevented us from addressing this issue. The recent finding of 65 Ma grass phytoliths in Cretaceous dinosaur coprolites <a href="#pone.0005297-Prasad1">[10]</a>, <a href="#pone.0005297-Piperno1">[11]</a> places the diversification of the grasses to the Cretaceous period, and provides a reliable calibration in studying the tempo and mode of grass chl evolution. By using this calibration, we here find that episodic rate acceleration occurred in the common ancestral branch of the core Poaceae (a clade formed by rice <em>Oryza</em>, wheat <em>Triticum</em>, maize <em>Zea</em>, and their allies) accompanied by adaptive evolution in several chl proteins, while the rate reverted to the slow rate typical of most monocot species in the terminal branches. We also find that the widely used method for estimating divergence times based on the assumption of correlated rates between ancestral and descendant lineages <a href="#pone.0005297-Thorne1">[12]</a>–<a href="#pone.0005297-Sanderson1">[14]</a> proved to be inadequate in approximating the process of grass chl evolution, and the assumption of independent rates <a href="#pone.0005297-Rannala1">[15]</a> is more appropriate to studies of rate change over time. These results have implications for studies of molecular evolutionary rates and time-scale of evolution in other groups of organisms.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Estimation of time-scale and pattern of rate change</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p><a href="#pone-0005297-g001">Fig. 1</a> shows the ML phylogenetic tree of Angiosperm chl with Gymnosperm as an outgroup. The elongated branches of Poaceae are in accord with their widely accepted rate acceleration <a href="#pone.0005297-Gaut1">[2]</a>–<a href="#pone.0005297-Chaw1">[5]</a>. The global clock model in Poales (including Poaceae and <em>Typha</em>)+<em>Musa</em> was rejected when compared with the 2-local-clocks model (Poaceae lineages have a different rate from basal lineages such as <em>Typha</em> and <em>Musa</em>) by the likelihood ratio test (LRT) (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.e001&amp;representation=PNG" class="inline-graphic"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.e002&amp;representation=PNG" class="inline-graphic"></span> with the codon-substitution model). Moreover, longer distances of the Poaceae species from <em>Musa</em> than the <em>Typha/Musa</em> distance both in terms of non-synonymous and synonymous substitutions (<a href="#pone-0005297-g001">Fig. 1</a>) indicate that both types of substitutions have undergone rate acceleration along the line leading to Poaceae.</p>
<div class="figure" id="pone-0005297-g001"><div class="img"><a name="pone-0005297-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0005297.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0005297.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The phylogenetic tree of chloroplast genomes for the 31 species.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The tree topology in <a href="#pone-0005297-g003">Fig. 3</a> of ref.<a href="#pone.0005297-Jansen1">[6]</a> was used, and the branch lengths were estimated by the ML with the codon-substitution model <a href="#pone.0005297-Goldman1">[21]</a>, <a href="#pone.0005297-Muse2">[22]</a> (CODEML in PAML <a href="#pone.0005297-Yang1">[19]</a>). The root was arbitrarily placed between Gymnosperm and Angiosperm. Non-synonymous (<em>d</em><sub>N</sub>) and synonymous (<em>d</em><sub>S</sub>) distances of Poales from <em>Musa</em> were estimated by CODEML.</p>
<span>doi:10.1371/journal.pone.0005297.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To explore the pattern of rate change during the course of grass evolution in more detail, we estimated the time-scale of Angiosperm phylogeny, particularly focusing on monocots. Although several powerful methods have been developed for molecular time estimation allowing the rate change (a relaxed clock) <a href="#pone.0005297-Thorne1">[12]</a>–<a href="#pone.0005297-Drummond1">[17]</a>, the poor quality of the fossil record for early grasses has prevented us from addressing this issue. Previously, the divergence among major groups of Poaceae was thought to have occurred in early Cenozoic, and the 60 Ma <a href="#pone.0005297-Wolfe1">[18]</a> and 50–60 Ma <a href="#pone.0005297-Chaw1">[5]</a> date calibrations for the maize/wheat divergence were used in estimating the monocots/eudicots divergence time with chl DNA sequences. However, recent findings of grass phytoliths in Cretaceous dinosaur coprolites <a href="#pone.0005297-Prasad1">[10]</a>, <a href="#pone.0005297-Piperno1">[11]</a> provided evidence that the major groups of core Poaceae had already diversified before Cretaceous/Tertiary (K/T) boundary of 65 Ma. <a href="#pone-0005297-g002">Fig. 2A</a> gives time estimates of the monocots evolution (<a href="#pone-0005297-g003">Fig. 3</a> and <a href="#pone-0005297-t001">Table 1</a> for the whole angiosperms) by a relaxed clock based on the Bayesian method implemented in MCMCTREE (in PAML <a href="#pone.0005297-Yang1">[19]</a>) with a constraint of &gt;65 Ma for the <em>Zea/Oryza</em> divergence and with the independent-rates model for the rate change along lineages <a href="#pone.0005297-Rannala1">[15]</a>, <a href="#pone.0005297-Drummond1">[17]</a>. In order to illustrate the rate change during evolution, a single instance of estimated rates along the lineage from the root to <em>Oryza</em> is also shown in <a href="#pone-0005297-g002">Fig. 2A</a>, where elevation of the rate only occurred on the common ancestral branch of Poaceae after they diverged from <em>Typha</em>.</p>
<div class="figure" id="pone-0005297-g002"><div class="img"><a name="pone-0005297-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0005297.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0005297.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Posterior estimates of divergence times and rate change during evolution.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Estimations were done by using MCMCTREE in PAML <a href="#pone.0005297-Yang1">[19]</a> with the IR model <a href="#pone.0005297-Rannala1">[15]</a> for the rate change along lineages. Shape and scale parameters, α and β, in the gamma prior for parameter σ<sup>2</sup> were 1.0 and 10.0, respectively. Only Poales+<em>Musa</em> part of the whole tree is shown, and a numbering of a node follows that of the whole tree in <a href="#pone-0005297-g003">Fig. 3</a>. The upper lines of the colored area trace the estimated rates along the lineage from the root to <em>Oryza</em> (the lineage indicated by colored lines in the tree) where 95% highest posterior density (HPD) is shown by a vertical line segment with two short horizontal line segments at boundaries. (A) &gt;65 Ma constraint and (B) no constraint to the <em>Zea/Oryza</em> separation (for other calibrations, see <a href="#s4">Materials and Methods</a>).</p>
<span>doi:10.1371/journal.pone.0005297.g002</span></div><div class="figure" id="pone-0005297-g003"><div class="img"><a name="pone-0005297-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0005297.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0005297.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Posterior estimates of divergence times of a whole Angiosperm tree.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Estimations were done by using MCMCTREE <a href="#pone.0005297-Yang1">[19]</a> with the IR model <a href="#pone.0005297-Rannala1">[15]</a>. The &gt;65 Ma constraint to the <em>Zea/Oryza</em> separation was applied.</p>
<span>doi:10.1371/journal.pone.0005297.g003</span></div><div class="figure" id="pone-0005297-t001"><div class="img"><a name="pone-0005297-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Posterior estimates of divergence times with the &gt;65 Ma constraint to the <em>Zea/Oryza</em> separation.</span></strong></p><span>doi:10.1371/journal.pone.0005297.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Although the fossil evidence for the &gt;65 Ma constraint of the <em>Zea/Oryza</em> divergence is important in demonstrating the rate acceleration in ancestral grasses with subsequent slow-down, it is not a prerequisite. Even when the constraint was removed, almost the same pattern of rate change as that with the &gt;65 Ma constraint was obtained when the IR model was used (<a href="#pone-0005297-g002">Fig. 2B</a> and <a href="#pone-0005297-t002">Table 2</a>), although the time estimate of the <em>Zea/Oryza</em> separation became younger (55.0 Ma). This time estimate is consistent with a conservative date of &gt;50 Ma presented in refs. <a href="#pone.0005297-Chaw1">[5]</a>, <a href="#pone.0005297-Vicentini1">[20]</a>, and our conclusion of the reverted slow rate in contemporary Poaceae can be regarded as robust to the calibration points used.</p>
<div class="figure" id="pone-0005297-t002"><div class="img"><a name="pone-0005297-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Posterior estimates of divergence times without constraint to the <em>Zea/Oryza</em> separation.</span></strong></p><span>doi:10.1371/journal.pone.0005297.t002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>In the first model of the relaxed clock implemented by Thorne and colleagues <a href="#pone.0005297-Thorne1">[12]</a>, <a href="#pone.0005297-Kishino1">[13]</a>, rates are auto-correlated between ancestral and descendant lineages on the tree, and the model is called the correlated-rates (CR) model. Sanderson's method of nonparametric rate-smoothing <a href="#pone.0005297-Sanderson1">[14]</a> was also based on the same idea. Later, an alternative model named the independent-rates (IR) model with no auto-correlation was developed <a href="#pone.0005297-Rannala1">[15]</a>, <a href="#pone.0005297-Drummond1">[17]</a>. In <a href="#pone-0005297-t001">Table 1</a>, estimates of divergence times with the &gt;65 Ma constraint for the <em>Zea/Oryza</em> separation are compared between the IR and CR models. The CR model tends to give older estimates for the nodes preceding the <em>Zea/Oryza</em> separation than the IR model. For example, the monocots/eudicots divergence time estimate was 239.1 Ma with the CR model, while the estimate with the IR model was 212.5 Ma which is more in accord with the recently published estimate of 140–150 Ma <a href="#pone.0005297-Chaw1">[5]</a> even though it is still older. Without the &gt;65 Ma constraint, the time estimate of the <em>Zea/Oryza</em> separation became too young (36.9 Ma) from the CR model to be compatible with the suggestion of &gt;50 Ma from the previous works <a href="#pone.0005297-Chaw1">[5]</a>, <a href="#pone.0005297-Vicentini1">[20]</a>, while the IR model gave compatible estimate of 55.0 Ma as mentioned before.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>In order to examine the impact of including rapidly evolving Poaceae in the analysis, a comparison between the two models was carried out excluding Poaceae (<a href="#pone-0005297-t003">Table 3</a>). The time estimates were similar between the two models, and were similar to those from the IR model including Poaceae. For the monocots/eudicots separation, the IR model gave almost consistent estimates of 212.5, 207.7, and 216.5 Ma, respectively, with the &gt;65 Ma constraint for the <em>Zea/Oryza</em> separation, without the constraint, and excluding Poaceae, while the CR model gave more diverged estimates of 239.1, 220.4, and 223.2 Ma, respectively. Interestingly, the estimates for this separation were very close between the two models when Poaceae species were excluded. This suggests that the episodic rate acceleration in ancestral Poaceae causes biased estimates, which the CR model cannot accommodate.</p>
<div class="figure" id="pone-0005297-t003"><div class="img"><a name="pone-0005297-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Posterior estimates of divergence times excluding Poaceae.</span></strong></p><span>doi:10.1371/journal.pone.0005297.t003</span></div><a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>In the above mentioned analyses, before fixing the shape and scale parameters (α and β) in the gamma prior for parameter σ<sup>2</sup>, which specifies how variable the rates are across branches, impact of priors on these parameters to posterior time and rate estimates was examined in detail (<a href="#pone.0005297.s001">Tables S1</a>, <a href="#pone.0005297.s002">S2</a>, <a href="#pone.0005297.s003">S3</a>, <a href="#pone.0005297.s004">S4</a>). Posterior time estimate for the <em>Zea/Oryza</em> separation depended less on the choice of the gamma prior for parameter σ<sup>2</sup> with the IR model (<a href="#pone.0005297.s001">Tables S1</a> and <a href="#pone.0005297.s003">S3</a>) than with the CR model (<a href="#pone.0005297.s002">Tables S2</a> and <a href="#pone.0005297.s004">S4</a>), and therefore α and β in the gamma prior for parameter σ<sup>2</sup> were arbitrarily chosen to be 1.0 and 10.0, respectively, in the analyses of <a href="#pone-0005297-t001">Tables 1</a>–<a href="#pone-0005297-t002"></a><a href="#pone-0005297-t003">3</a> and <a href="#pone-0005297-g002">Figs. 2</a> and <a href="#pone-0005297-g003">3</a>.</p>
<a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>In order to further check the robustness of the time estimation on the choice of the substitution model, additional analyses based on a more realistic model of codon-substitution <a href="#pone.0005297-Goldman1">[21]</a>, <a href="#pone.0005297-Muse2">[22]</a> were carried out (<a href="#pone.0005297.s005">Tables S5</a> and <a href="#pone.0005297.s006">S6</a> with and without the &gt;65 Ma constraint to the <em>Zea/Oryza</em> separation, and <a href="#pone.0005297.s007">Table S7</a> excluding Poaceae). Comparisons of <a href="#pone.0005297.s005">Tables S5</a> vs 1, <a href="#pone.0005297.s006">Tables S6</a> vs 2, and <a href="#pone.0005297.s007">Tables S7</a> vs 3 indicate that the estimated times are very similar between the two models, and that the estimated times are robust to the choice of a substitution model.</p>


<h4>Adaptive evolution</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Non-synonymous/synonymous rate ratio (ω = d<sub>N</sub>/d<sub>S</sub>) is widely used as an indicator of adaptive evolution or positive selection <a href="#pone.0005297-Yang2">[23]</a>. <a href="#pone-0005297-t004">Table 4</a> compares ω ratios along the branches estimated by different models. The minimum AIC <a href="#pone.0005297-Akaike1">[24]</a> model shows that a pronounced increase of ω ratio occurred in the common ancestral lineage of Poaceae after they diverged from <em>Typha</em>, followed by reversion in the terminal branches to the lower level typical of basal lineages. The elevation of the ω ratio can be due either by adaptive evolution or by relaxation of selective constraints. A higher ω value than 1 is usually regarded as an evidence of adaptive evolution, but since the analysis shown in the table averages over the entire genomes, we would not get such a high value even if positive selection operated in some parts of some proteins. Therefore, the branch-site model <a href="#pone.0005297-Zhang1">[25]</a>, <a href="#pone.0005297-Yang3">[26]</a> was applied.</p>
<div class="figure" id="pone-0005297-t004"><div class="img"><a name="pone-0005297-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Estimation of non-synonymous/synonymous rate ratio (ω) under different models by using CODEML in PAML <a href="#pone.0005297-Yang1">[19]</a>.</span></strong></p><span>doi:10.1371/journal.pone.0005297.t004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To identify positively selected sites, among 61 protein-encoding “genes”, we at first selected 16 “genes”, for which the reverted 2ω-model (with the rate ratio ω<sub>1</sub> of the common ancestral branch of Poaceae larger than the rate ratio ω<sub>0</sub> of other branches) is significantly better than the 1ω-model (<em>P</em>&lt;0.05) (<a href="#pone-0005297-t005">Table 5</a>), and by using the branch-site model <a href="#pone.0005297-Zhang1">[25]</a>, <a href="#pone.0005297-Yang3">[26]</a>, we identified 5 genes (<em>atp</em>E, <em>cem</em>A, <em>clp</em>P, <em>rpo</em>B, and <em>rps</em>11) which have <em>P</em> value of the branch-site LRT less than 0.05 and contain positively selected sites (<a href="#pone-0005297-t006">Table 6</a>).</p>
<div class="figure" id="pone-0005297-t005"><div class="img"><a name="pone-0005297-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>LRT of 1ω-model vs reverted 2ω-model for individual genes.</span></strong></p><span>doi:10.1371/journal.pone.0005297.t005</span></div><div class="figure" id="pone-0005297-t006"><div class="img"><a name="pone-0005297-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0005297" data-uri="info:doi/10.1371/journal.pone.0005297.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0005297.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0005297.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0005297.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Branch-site test of positive selection.</span></strong></p><span>doi:10.1371/journal.pone.0005297.t006</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Among the 16 genes with significantly higher ω<sub>1</sub> than ω<sub>0</sub> in <a href="#pone-0005297-t005">Table 5</a> and among the 5 genes with positively selected sites in <a href="#pone-0005297-t006">Table 6</a>, only <em>atp</em>E is among the 14 genes with significant heterogeneity of nucleotide substitution rates for maize vs. rice, maize vs. wheat, or rice vs. wheat comparisons listed in <a href="#pone-0005297-t005">Table 5</a> of ref. <a href="#pone.0005297-Matsuoka1">[27]</a>. Four “genes”, <em>psa</em>C, <em>rbc</em>L, <em>rpl</em>6, and PS13, have significantly lower ω<sub>1</sub> than ω<sub>0</sub> (stronger purifying selection in the ancestral branch of Poaceae than in other branches). In <a href="#pone-0005297-t005">Table 5</a>, we carried out multiple tests for 61 “genes”. The Bonferroni correction is a safeguard against multiple tests falsely giving the appearance of significance, since 1 out of every 20 hypotheses tests is expected to be significant at the 5% level purely by chance. After performing the Bonferroni correction, 7 genes with * in <a href="#pone-0005297-t005">Table 5</a> remained significant, that means, all the genes listed in <a href="#pone-0005297-t006">Table 6</a> remained significant even by the conservative test of Bonferroni. On the other hand, the 4 “genes” with lower ω<sub>1</sub> than ω<sub>0</sub> in <a href="#pone-0005297-t005">Table 5</a> were not significant after the Bonferroni correction.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In our study, the IR model gives more consistent results than the CR model, which has been widely used in estimating divergence times <a href="#pone.0005297-LeebensMack1">[9]</a>, <a href="#pone.0005297-Thorne1">[12]</a>–<a href="#pone.0005297-Sanderson1">[14]</a>, <a href="#pone.0005297-Bremer1">[28]</a>–<a href="#pone.0005297-Lepage1">[34]</a>. A basic assumption of the CR model is that rates change gradually over the tree. Our results suggest that the magnitude of the rate acceleration is underestimated by the CR model and that the IR model is more appropriate in approximating the rate change in the grass chl evolution. Although there exists a case in which the CR model outperforms the IR model <a href="#pone.0005297-Lepage1">[34]</a>, a number of authors have recently begun to notice that the IR model is superior to the CR model in approximating the evolution of evolutionary rates in several cases <a href="#pone.0005297-Drummond1">[17]</a>, <a href="#pone.0005297-Kitazoe1">[35]</a>–<a href="#pone.0005297-Renner1">[37]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Our analysis has revealed an episodic acceleration of the evolutionary rate of chl genomes during the emergence of core Poaceae, accompanied by adaptive evolution in several protein-encoding genes. Because the elevation of the rate occurred not only in non-synonymous substitutions but also in synonymous substitutions and because the elevated substitution rates were accompanied also by an elevated rate of insertions/deletions of nucleotides <a href="#pone.0005297-LeebensMack1">[9]</a>, the elevation of the mutation rate of chl genomes might have acted as a trigger of the adaptive evolution in the ancestral grasses, which might have facilitated the successful radiation and diversification of their descendants.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Suggested positive selection of <em>clp</em>P in <em>Oenothera</em> and <em>Sileneae</em> accompanied by elevated synonymous rate <a href="#pone.0005297-Erixon1">[38]</a> might be related to our finding of rate acceleration in ancestral grasses both in terms of synonymous and non-synonymous substitutions. A more extensive study of chl genomes showed highly accelerated non-synonymous rates of ribosomal protein and RNA polymerase genes in Geraniaceae accompanied with the elevation of the ω ratio <a href="#pone.0005297-Guisinger1">[39]</a>. Interestingly, the 4 genes (<em>atp</em>E, <em>cem</em>A, <em>rpo</em>B, and <em>rps</em>11) detected to have positively-selected sites in our analysis (<a href="#pone-0005297-t006">Table 6</a>) are included in the gene group with significantly high ω ratio in Geraniaceae relative to other angiosperms (<em>clp</em>P was not analyzed in ref. <a href="#pone.0005297-Guisinger1">[39]</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Recently, Smith and Donoghue <a href="#pone.0005297-Smith1">[40]</a> tested evolutionary rates across five groups of angiosperms, and found that the rates are generally low in trees/shrubs compared to related herbs. This is an interesting finding which links life history of plants to their rates of molecular evolution, and their conclusion generally holds in five different groups of Angiosperm. What we have shown in this work, however, is that the pattern of rate change during evolution is more complicated than has previously been anticipated. Our finding highlights the need for paying attention to rates of internal branches rather than averaging along a lineage in addressing the rate heterogeneity problem.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Since our main interest was on grass evolution, we used all the monocot genera (13 species) and selected 18 species from outside monocots (31 species in total) among the 64 species in ref. <a href="#pone.0005297-Jansen1">[6]</a>. We used 75 chl genes among 77 protein-encoding genes in ref. <a href="#pone.0005297-Jansen1">[6]</a>, excluding <em>inf</em>A and <em>ycf</em>2 because of missing data.</p>

<h4>Estimation of divergence times</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The concatenated 75 gene sequences of chl from 31 species (from ref. <a href="#pone.0005297-Jansen1">[6]</a>) and the tree topology in ref. <a href="#pone.0005297-Jansen1">[6]</a> were used. To estimate divergence times and molecular evolutionary rates, a Bayesian method implemented in MCMCTREE (in PAML <a href="#pone.0005297-Yang1">[19]</a>) was applied either with the CR model <a href="#pone.0005297-Thorne1">[12]</a>, <a href="#pone.0005297-Kishino1">[13]</a> or with the IR model <a href="#pone.0005297-Rannala1">[15]</a> (using the GTR model with a discrete gamma distribution with five rate categories (Γ<sub>5</sub>) for nucleotide substitutions), and multiple calibrations were incorporated through the time prior. The Gymnosperm/Angiosperm divergence time was set at 280–310 Ma <a href="#pone.0005297-Chaw1">[5]</a>, <a href="#pone.0005297-Moore1">[7]</a>. Three nodes were constrained with minimum ages as follows; (1) the minimum age of the <em>Zea/Oryza</em> divergence was set either to 65 Ma <a href="#pone.0005297-Prasad1">[10]</a>, <a href="#pone.0005297-Piperno1">[11]</a> or without this constraint, (2) &gt;115 Ma constraint to the divergence of Poales from other monocots based on the earliest fossils of Poales <a href="#pone.0005297-Herendeen1">[41]</a>, <a href="#pone.0005297-Linder1">[42]</a>, and (3) &gt;125 Ma for the most basal divergence in eudicots <a href="#pone.0005297-Moore1">[7]</a>. In order to check the robustness of the time estimation on the choice of the substitution model, the codon-substitution <a href="#pone.0005297-Goldman1">[21]</a>, <a href="#pone.0005297-Muse2">[22]</a>+Γ<sub>5</sub> model was also used. The program adopts soft bounds, so that the probability that the true divergence time is outside the bounds is small but not zero <a href="#pone.0005297-Yang4">[43]</a>. In the Bayesian framework, priors are assigned not only on times, but also on the overall substitution rate parameter μ and on the rate-drift parameter σ<sup>2</sup>. So we roughly estimated the prior mean of the overall rate μ using the strict molecular clock with 295 Ma constraint to Gymnosperm/Angiosperm divergence time, and assigned the gamma prior G(4, 80) and G(4, 22) for this prior parameter in applying the nucleotide and codon substitution models, respectively. We next examined the impact of the rate-drift parameter σ<sup>2</sup> by giving various priors for σ<sup>2</sup> in applying the nucleotide substitution model. Posterior distributions of parameters were approximated using two independent MCMC analyses of 10<sup>7</sup> steps each, following a discarded burn-in of 10<sup>6</sup> steps. All the analyses were repeated with different inseed values to check for convergence of the MCMC chain.</p>


<h4>Non-synonymous/synonymous rate ratio</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>To the concatenated sequences of 75 protein-encoding genes of chl from 6 Poaceae species (<em>Oryza</em>, <em>Triticum</em>, <em>Hordeum</em>, <em>Zea</em>, <em>Saccharum</em>, and <em>Sorghum</em>), <em>Typha</em>, and <em>Musa</em>, we applied the codon-based likelihood models that allow for variable ω ratios among different lineages <a href="#pone.0005297-Yang5">[44]</a>. We used the likelihood ratio test (LRT) to compare the likelihood of one-ω ratio model, which assumes the same ω for all branch in the tree, with the two-ω ratio models, which assumes two different ω ratios. One of the two-ratio models (named “Simple 2ω-model”) assumes that Poaceae (including the common ancestral branch) has different ω from other parts of the tree as is represented by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.e010&amp;representation=PNG" class="inline-graphic"></span><br>while the other (named “Reverted 2ω-model”) assumes that only the ancestral branch of Poceae has a different ω ratio than all the other branches in the tree as is represented by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0005297.e011&amp;representation=PNG" class="inline-graphic"></span><br>All the analyses were carried out with the CODEML program in PAML <a href="#pone.0005297-Yang1">[19]</a> using the codon-substitution model with the F61 codon frequency.</p>


<h4>Branch-site test of positive selection</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The branch-site test was applied to the dataset of 11 monocot species in our dataset excluding the two most basal monocots; <em>i.e.</em>, <em>Dioscorea</em> and <em>Acorus</em>. The branch preceding the common ancestor of the core Poaceae was specified as a foreground branch, and all the others as background branches. LRT is constructed to compare an alternative model that allows for some codons under positive selection on the foreground branch with a null model that does not. The null model restricts codons on the foreground lineage to be undergoing neutral evolution (ω = 1). The specific codons which evolved under positive selection were identified on the foreground branch using a Bayes empirical Bayes procedure <a href="#pone.0005297-Zhang1">[25]</a>, <a href="#pone.0005297-Yang3">[26]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0005297"></div><a name="pone.0005297.s001" id="pone.0005297.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0005297.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Impact of the shape and scale parameters (α and β) in the gamma prior for parameter σ<sup>2</sup> using IR model with the &gt;65 Ma constraint to the <em>Zea/Oryza</em> separation. 95% HPD is shown in parentheses. Times and rates are represented in 100 Ma (10<sup>8</sup> years ago) and 10<sup>−8</sup> substitutions/site/years, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0005297.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0005297.s002" id="pone.0005297.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0005297.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Impact of the shape and scale parameters (α and β) in the gamma prior for parameter σ<sup>2</sup> using CR model with the &gt;65 Ma constraint to the <em>Zea/Oryza</em> separation. 95% HPD is shown in parentheses. Times and rates are represented in 100 Ma (10<sup>8</sup> years ago) and 10<sup>−8</sup> substitutions/site/years, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0005297.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0005297.s003" id="pone.0005297.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0005297.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Impact of the shape and scale parameters (α and β) in the gamma prior for parameter σ<sup>2</sup> using IR model without the constraint to the <em>Zea/Oryza</em> separation. 95% HPD is shown in parentheses. Times and rates are represented in 100 Ma (10<sup>8</sup> years ago) and 10<sup>−8</sup> substitutions/site/years, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0005297.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0005297.s004" id="pone.0005297.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0005297.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Impact of the shape and scale parameters (α and β) in the gamma prior for parameter σ<sup>2</sup> using CR model without the constraint to the <em>Zea/Oryza</em> separation. 95% HPD is shown in parentheses. Times and rates are represented in 100 Ma (10<sup>8</sup> years ago) and 10<sup>−8</sup> substitutions/site/years, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0005297.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0005297.s005" id="pone.0005297.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0005297.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Posterior estimates of divergence times by MCMCTREE in PAML <a href="#pone.0005297-Yang1">[19]</a> using the codon-substitution+Γ<sub>5</sub> model with the &gt;65 Ma constraint to the <em>Zea/Oryza</em> separation. Shape and scale parameters, α and β, in the gamma prior for parameter σ<sup>2</sup> were 1.0 and 10.0, respectively. 95% HPD is shown in parentheses. Rate (10<sup>−8</sup> substitutions/codon/year) refers to the rate of the branch preceding the node. Node numbers refer to those in <a href="#pone-0005297-g003">Fig. 3</a>, and taxa in parentheses refer to those branched off from the lineage leading to <em>Oryza</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0005297.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0005297.s006" id="pone.0005297.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0005297.s006">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Posterior estimates of divergence times by MCMCTREE in PAML <a href="#pone.0005297-Yang1">[19]</a> using the codon-substitution+Γ<sub>5</sub> model without constraint to the <em>Zea/Oryza</em> separation. Shape and scale parameters, α and β, in the gamma prior for parameter σ<sup>2</sup> were 1.0 and 10.0, respectively. 95% HPD is shown in parentheses. Rate (10<sup>−8</sup> substitutions/codon/year) refers to the rate of the branch preceding the node. Node numbers refer to those in <a href="#pone-0005297-g003">Fig. 3</a>, and taxa in parentheses refer to those branched off from the lineage leading to <em>Oryza</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0005297.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0005297.s007" id="pone.0005297.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0005297.s007">Table S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Posterior estimates of divergence times by MCMCTREE in PAML <a href="#pone.0005297-Yang1">[19]</a> using the codon-substitution+Γ<sub>5</sub> model excluding Poaceae. Shape and scale parameters, α and β, in the gamma prior for parameter σ<sup>2</sup> were 1.0 and 10.0, respectively. 95% HPD is shown in parentheses. Rate (10<sup>−8</sup> substitutions/codon/year) refers to the rate of the branch preceding the node. Node numbers refer to those in <a href="#pone-0005297-g003">Fig. 3</a>, and taxa in parentheses refer to those branched off from the lineage leading to <em>Oryza</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0005297.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank James Crabbe for improving the manuscript. Simon Joly provided thoughtful comments that helped improve the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BZ MH. Analyzed the data: BZ TY. Wrote the paper: BZ TY YZ MH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0005297-Jacobs1" id="pone.0005297-Jacobs1"></a>Jacobs BF, Kingston JD, Jacobs LL (1999) The origin of grass-dominated ecosystems. Ann Mo Bot Gard  86: 590–643.  <ul class="find" data-citedArticleID="1395064" data-doi="10.2307/2666186"><li><a href="http://dx.doi.org/10.2307/2666186" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+origin+of+grass-dominated+ecosystems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+origin+of+grass-dominated+ecosystems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0005297-Gaut1" id="pone.0005297-Gaut1"></a>Gaut BS, Muse SV, Clark WD, Clegg MT (1992) Relative rates of nucleotide substitution at the <em>rbc</em>L locus of monocotyledonous plants. J Mol Evol  35: 292–303.  <ul class="find" data-citedArticleID="1395112" data-doi="10.1007/bf00161167"><li><a href="http://dx.doi.org/10.1007/bf00161167" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relative+rates+of+nucleotide+substitution+at+the+rbcL+locus+of+monocotyledonous+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relative+rates+of+nucleotide+substitution+at+the+rbcL+locus+of+monocotyledonous+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0005297-Bousquet1" id="pone.0005297-Bousquet1"></a>Bousquet J, Strauss SH, Doerksen AH, Price RA (1992) Extensive variation in evolutionary rate of <em>rbc</em>L gene sequences among seed plants. Proc Natl Acad Sci USA  89: 7844–7848.  <ul class="find" data-citedArticleID="1395044" data-doi="10.1073/pnas.89.16.7844"><li><a href="http://dx.doi.org/10.1073/pnas.89.16.7844" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+variation+in+evolutionary+rate+of+rbcL+gene+sequences+among+seed+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+variation+in+evolutionary+rate+of+rbcL+gene+sequences+among+seed+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0005297-Muse1" id="pone.0005297-Muse1"></a>Muse SV, Gaut BS (1997) Interlocus comparisons of the nucleotide substitution process in the chloroplast genome. Genetics  146: 393–399.  <ul class="find" data-citedArticleID="1395058"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Interlocus+comparisons+of+the+nucleotide+substitution+process+in+the+chloroplast+genome.&amp;auth=&amp;atitle=Interlocus+comparisons+of+the+nucleotide+substitution+process+in+the+chloroplast+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interlocus+comparisons+of+the+nucleotide+substitution+process+in+the+chloroplast+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interlocus+comparisons+of+the+nucleotide+substitution+process+in+the+chloroplast+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0005297-Chaw1" id="pone.0005297-Chaw1"></a>Chaw S-M, Chang C-C, Chen H-L, Li W-H (2004) Dating the monocot–dicot divergence and the origin of core eudicots using whole chloroplast genomes. J Mol Evol  58: 424–441.  <ul class="find" data-citedArticleID="1395072" data-doi="10.1007/s00239-003-2564-9"><li><a href="http://dx.doi.org/10.1007/s00239-003-2564-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dating+the+monocot%E2%80%93dicot+divergence+and+the+origin+of+core+eudicots+using+whole+chloroplast+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dating+the+monocot%E2%80%93dicot+divergence+and+the+origin+of+core+eudicots+using+whole+chloroplast+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0005297-Jansen1" id="pone.0005297-Jansen1"></a>Jansen RK, Cai Z, Raubeson LA, Daniell H, dePamphilis CW, et al.  (2007) Analysis of 81 genes from 64 plastid genomes resolves relationships in angiosperms and identifies genome-scale evolutionary patterns. Proc Natl Acad Sci  104: 19369–19374.  <ul class="find" data-citedArticleID="1395060" data-doi="10.1073/pnas.0709121104"><li><a href="http://dx.doi.org/10.1073/pnas.0709121104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+81+genes+from+64+plastid+genomes+resolves+relationships+in+angiosperms+and+identifies+genome-scale+evolutionary+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+81+genes+from+64+plastid+genomes+resolves+relationships+in+angiosperms+and+identifies+genome-scale+evolutionary+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0005297-Moore1" id="pone.0005297-Moore1"></a>Moore MJ, Bell CD, Soltis PS, Soltis DE (2007) Using plastid genome-scale data to resolve enigmatic relationships among basal angiosperms. Proc Natl Acad Sci USA  49: 19363–19368.  <ul class="find" data-citedArticleID="1395048" data-doi="10.1073/pnas.0708072104"><li><a href="http://dx.doi.org/10.1073/pnas.0708072104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+plastid+genome-scale+data+to+resolve+enigmatic+relationships+among+basal+angiosperms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+plastid+genome-scale+data+to+resolve+enigmatic+relationships+among+basal+angiosperms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0005297-Martin1" id="pone.0005297-Martin1"></a>Martin W, Deusch O, Stawski N, Grunheit N, Goremykin V (2005) Chloroplast genome phylogenetics: why we need independent approaches to plant molecular evolution. TRENDS in Plant Science  10: 203–209.  <ul class="find" data-citedArticleID="1395088" data-doi="10.1016/j.tplants.2005.03.007"><li><a href="http://dx.doi.org/10.1016/j.tplants.2005.03.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chloroplast+genome+phylogenetics%3A+why+we+need+independent+approaches+to+plant+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chloroplast+genome+phylogenetics%3A+why+we+need+independent+approaches+to+plant+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0005297-LeebensMack1" id="pone.0005297-LeebensMack1"></a>Leebens-Mack J, Raubeson LA, Cui L, Kuehl JV, Fourcade MH, et al.  (2005) Identifying the basal angiosperm node in chloroplast genome phylogenies: sampling one's way out of the Felsenstein zone. Mol Biol Evol  22: 1948–1963.  <ul class="find" data-citedArticleID="1395086" data-doi="10.1093/molbev/msi191"><li><a href="http://dx.doi.org/10.1093/molbev/msi191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+the+basal+angiosperm+node+in+chloroplast+genome+phylogenies%3A+sampling+one%27s+way+out+of+the+Felsenstein+zone." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+the+basal+angiosperm+node+in+chloroplast+genome+phylogenies%3A+sampling+one%27s+way+out+of+the+Felsenstein+zone.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0005297-Prasad1" id="pone.0005297-Prasad1"></a>Prasad V, Stroemberg CAE, Alimohammadian H, Sahni A (2005) Dinosaur coprolites and the early evolution of grasses and grazers. Science  310: 1177–1180.  <ul class="find" data-citedArticleID="1395090" data-doi="10.1126/science.1118806"><li><a href="http://dx.doi.org/10.1126/science.1118806" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dinosaur+coprolites+and+the+early+evolution+of+grasses+and+grazers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dinosaur+coprolites+and+the+early+evolution+of+grasses+and+grazers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0005297-Piperno1" id="pone.0005297-Piperno1"></a>Piperno DR, Sues HD (2005) Dinosaurs dined on grass. Science  310: 1126–1128.  <ul class="find" data-citedArticleID="1395042" data-doi="10.1126/science.1121020"><li><a href="http://dx.doi.org/10.1126/science.1121020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dinosaurs+dined+on+grass." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dinosaurs+dined+on+grass.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0005297-Thorne1" id="pone.0005297-Thorne1"></a>Thorne JL, Kishino H, Painter IS (1998) Estimating the rate of evolution of the rate of molecular evolution. Mol Biol Evol  15: 1647–1657.  <ul class="find" data-citedArticleID="1395032" data-doi="10.1093/oxfordjournals.molbev.a025892"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025892" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+the+rate+of+evolution+of+the+rate+of+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+the+rate+of+evolution+of+the+rate+of+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0005297-Kishino1" id="pone.0005297-Kishino1"></a>Kishino H, Thorne JL, Bruno WJ (2001) Performance of a divergence time estimation method under a probabilistic model of rate evolution. Mol Biol Evol  18: 352–361.  <ul class="find" data-citedArticleID="1395036" data-doi="10.1093/oxfordjournals.molbev.a003811"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Performance+of+a+divergence+time+estimation+method+under+a+probabilistic+model+of+rate+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Performance+of+a+divergence+time+estimation+method+under+a+probabilistic+model+of+rate+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0005297-Sanderson1" id="pone.0005297-Sanderson1"></a>Sanderson MJ (1997) A nonparametric approach to estimating divergence times in the absence of rate constancy. Mol Biol Evol  14: 1218–1232.  <ul class="find" data-citedArticleID="1395110" data-doi="10.1093/oxfordjournals.molbev.a025731"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025731" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+nonparametric+approach+to+estimating+divergence+times+in+the+absence+of+rate+constancy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+nonparametric+approach+to+estimating+divergence+times+in+the+absence+of+rate+constancy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0005297-Rannala1" id="pone.0005297-Rannala1"></a>Rannala B, Yang Z (2007) Inferring speciation times under an episodic molecular clock. Syst Biol  56: 453–466.  <ul class="find" data-citedArticleID="1395094"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inferring+speciation+times+under+an+episodic+molecular+clock.&amp;auth=&amp;atitle=Inferring+speciation+times+under+an+episodic+molecular+clock." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+speciation+times+under+an+episodic+molecular+clock." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+speciation+times+under+an+episodic+molecular+clock.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0005297-Huelsenbeck1" id="pone.0005297-Huelsenbeck1"></a>Huelsenbeck JP, Larget B, Swofford D (2000) A compound Poisson process for relaxing the molecular clock. Genetics  154: 1879–1892.  <ul class="find" data-citedArticleID="1395052"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+compound+Poisson+process+for+relaxing+the+molecular+clock.&amp;auth=&amp;atitle=A+compound+Poisson+process+for+relaxing+the+molecular+clock." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+compound+Poisson+process+for+relaxing+the+molecular+clock." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+compound+Poisson+process+for+relaxing+the+molecular+clock.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0005297-Drummond1" id="pone.0005297-Drummond1"></a>Drummond AJ, Ho SYW, Phillips MJ, Rambaut A (2006) Relaxed phylogenetics and dating with confidence. PLoS Biol  4: e88.  <ul class="find" data-citedArticleID="1395098" data-doi="10.1371/journal.pbio.0040088"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relaxed+phylogenetics+and+dating+with+confidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relaxed+phylogenetics+and+dating+with+confidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0005297-Wolfe1" id="pone.0005297-Wolfe1"></a>Wolfe KH, Gouy MY, Yang W, Sharp PM, Li W-H (1989) Date of the monocot-dicot divergence estimated from chloroplast chloroplast DNA sequence data. Proc Natl Acad Sci USA  86: 6201–6205.  <ul class="find" data-citedArticleID="1395054" data-doi="10.1073/pnas.86.16.6201"><li><a href="http://dx.doi.org/10.1073/pnas.86.16.6201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Date+of+the+monocot-dicot+divergence+estimated+from+chloroplast+chloroplast+DNA+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Date+of+the+monocot-dicot+divergence+estimated+from+chloroplast+chloroplast+DNA+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0005297-Yang1" id="pone.0005297-Yang1"></a>Yang Z (2007) PAML 4: Phylogenetic Analysis by Maximum Likelihood. Mol Biol Evol  24: 1586–1591.  <ul class="find" data-citedArticleID="1395056" data-doi="10.1093/molbev/msm088"><li><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML+4%3A+Phylogenetic+Analysis+by+Maximum+Likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML+4%3A+Phylogenetic+Analysis+by+Maximum+Likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0005297-Vicentini1" id="pone.0005297-Vicentini1"></a>Vicentini A, Barber JC, Aliscioni SS, Giussani LM, Kellogg EA (2008) The age of the grasses and clusters of origins of C<sub>4</sub> photosynthesis. Global Change Biol  14: 2963–2977.  <ul class="find" data-citedArticleID="1395106" data-doi="10.1111/j.1365-2486.2008.01688.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2486.2008.01688.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+age+of+the+grasses+and+clusters+of+origins+of+C4+photosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+age+of+the+grasses+and+clusters+of+origins+of+C4+photosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0005297-Goldman1" id="pone.0005297-Goldman1"></a>Goldman N, Yang Z (1994) A codon-based model of nucleotide substitution for protein-coding DNA sequences. Mol Biol Evol  11: 725–736.  <ul class="find" data-citedArticleID="1395050"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences.&amp;auth=&amp;atitle=A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0005297-Muse2" id="pone.0005297-Muse2"></a>Muse SV, Gaut BS (1994) A likelihood approach for comparing synonymous and nonsynonymous nucleotide substitution rates, with application to the chloroplast genome. Mol Biol Evol  11: 715–724.  <ul class="find" data-citedArticleID="1395040"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome.&amp;auth=&amp;atitle=A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0005297-Yang2" id="pone.0005297-Yang2"></a>Yang Z (2006) Computational Molecular Evolution. Oxford: Oxford Univ Press.   <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0005297-Akaike1" id="pone.0005297-Akaike1"></a>Akaike H (1973)  Information theory and an extension of the maximum likelihood principle. In: Petrov BN, Csaki F, editors. Second International Symposium on Information Theory. Budapest: Akademiai Kiado. pp. 267–281.  <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0005297-Zhang1" id="pone.0005297-Zhang1"></a>Zhang J, Nielsen R, Yang Z (2005) Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level. Mol Biol Evol  22: 1–8.  <ul class="find" data-citedArticleID="1395062" data-doi="10.1093/molbev/msi237"><li><a href="http://dx.doi.org/10.1093/molbev/msi237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0005297-Yang3" id="pone.0005297-Yang3"></a>Yang Z, Wong WSW, Nielsen R (2005) Bayes empirical Bayes inference of amino acids sites under positive selection. Mol Biol Evol  22: 1107–1118.  <ul class="find" data-citedArticleID="1395118" data-doi="10.1093/molbev/msi097"><li><a href="http://dx.doi.org/10.1093/molbev/msi097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayes+empirical+Bayes+inference+of+amino+acids+sites+under+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayes+empirical+Bayes+inference+of+amino+acids+sites+under+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0005297-Matsuoka1" id="pone.0005297-Matsuoka1"></a>Matsuoka Y, Yamazaki Y, Ogihara Y, Tsunewaki K (2002) Whole chloroplast genome comparison of rice, maize, and wheat: implications for chloroplast gene diversification and phylogeny of cereals. Mol Biol Evol  19: 2084–2091.  <ul class="find" data-citedArticleID="1395114" data-doi="10.1093/oxfordjournals.molbev.a004033"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole+chloroplast+genome+comparison+of+rice%2C+maize%2C+and+wheat%3A+implications+for+chloroplast+gene+diversification+and+phylogeny+of+cereals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole+chloroplast+genome+comparison+of+rice%2C+maize%2C+and+wheat%3A+implications+for+chloroplast+gene+diversification+and+phylogeny+of+cereals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0005297-Bremer1" id="pone.0005297-Bremer1"></a>Bremer K (2002) Gondwanan evolution of the grass alliance of families (Poales). Evolution  56: 1374–1387.  <ul class="find" data-citedArticleID="1395080" data-doi="10.1111/j.0014-3820.2002.tb01451.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2002.tb01451.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gondwanan+evolution+of+the+grass+alliance+of+families+%28Poales%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gondwanan+evolution+of+the+grass+alliance+of+families+%28Poales%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0005297-Sanderson2" id="pone.0005297-Sanderson2"></a>Sanderson MJ, Thorne JL, Wikstroem N, Bremer K (2004) Molecular evidence on plant divergence times. Amer J Bot  91: 1656–1665.  <ul class="find" data-citedArticleID="1395116" data-doi="10.3732/ajb.91.10.1656"><li><a href="http://dx.doi.org/10.3732/ajb.91.10.1656" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evidence+on+plant+divergence+times." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evidence+on+plant+divergence+times.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0005297-ArisBrosou1" id="pone.0005297-ArisBrosou1"></a>Aris-Brosou S, Yang Z (2003) Bayesian models of episodic evolution support a late Precambrian explosive diversification of the Metazoa. Mol Biol Evol  20: 1947–1954.  <ul class="find" data-citedArticleID="1395038" data-doi="10.1093/molbev/msg226"><li><a href="http://dx.doi.org/10.1093/molbev/msg226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+models+of+episodic+evolution+support+a+late+Precambrian+explosive+diversification+of+the+Metazoa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+models+of+episodic+evolution+support+a+late+Precambrian+explosive+diversification+of+the+Metazoa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0005297-Nikaido1" id="pone.0005297-Nikaido1"></a>Nikaido M, Matsuno F, Hamilton H, Brownell RL Jr, Cao Y, et al.  (2001) Retroposon analysis of major ceatcean lineages: The monophyly of toothed whales and the paraphyly of river dolphins. Proc Natl Acad Sci USA  98: 7384–7389.  <ul class="find" data-citedArticleID="1395082" data-doi="10.1073/pnas.121139198"><li><a href="http://dx.doi.org/10.1073/pnas.121139198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroposon+analysis+of+major+ceatcean+lineages%3A+The+monophyly+of+toothed+whales+and+the+paraphyly+of+river+dolphins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroposon+analysis+of+major+ceatcean+lineages%3A+The+monophyly+of+toothed+whales+and+the+paraphyly+of+river+dolphins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0005297-Hasegawa1" id="pone.0005297-Hasegawa1"></a>Hasegawa M, Thorne J, Kishino H (2003) Time scale of eutherian evolution estimated without assuming a constant rate of molecular evolution. Genes Genet Syst  78: 267–283.  <ul class="find" data-citedArticleID="1395096" data-doi="10.1266/ggs.78.267"><li><a href="http://dx.doi.org/10.1266/ggs.78.267" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Time+scale+of+eutherian+evolution+estimated+without+assuming+a+constant+rate+of+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Time+scale+of+eutherian+evolution+estimated+without+assuming+a+constant+rate+of+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0005297-Yoder1" id="pone.0005297-Yoder1"></a>Yoder AD, Yang Z (2004) Divergence dates for Malagasy lemurs estimated from multiple gene loci: geological and evolutionary context. Mol Ecol  13: 757–773.  <ul class="find" data-citedArticleID="1395100" data-doi="10.1046/j.1365-294x.2004.02106.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2004.02106.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergence+dates+for+Malagasy+lemurs+estimated+from+multiple+gene+loci%3A+geological+and+evolutionary+context." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergence+dates+for+Malagasy+lemurs+estimated+from+multiple+gene+loci%3A+geological+and+evolutionary+context.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0005297-Lepage1" id="pone.0005297-Lepage1"></a>Lepage T, Bryant D, Philippe H, Lartillot N (2007) A general comparison of relaxed molecular clock models. Mol Biol Evol  24: 2669–2680.  <ul class="find" data-citedArticleID="1395066" data-doi="10.1093/molbev/msm193"><li><a href="http://dx.doi.org/10.1093/molbev/msm193" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+comparison+of+relaxed+molecular+clock+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+comparison+of+relaxed+molecular+clock+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0005297-Kitazoe1" id="pone.0005297-Kitazoe1"></a>Kitazoe Y, Kishino H, Waddell PJ, Nakajima N, Okabayashi T, et al.  (2007) Robust time estimation reconciles views of the antiquity of placental mammals. PLoS ONE  2: e384.  <ul class="find" data-citedArticleID="1395074" data-doi="10.1371/journal.pone.0000384"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000384" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robust+time+estimation+reconciles+views+of+the+antiquity+of+placental+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robust+time+estimation+reconciles+views+of+the+antiquity+of+placental+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0005297-Brown1" id="pone.0005297-Brown1"></a>Brown JW, Rest JS, Garcia-Moreno J, Sorenson MD, Mindell DP (2008) Strong mitochondrial DNA support for a Cretaceous origin of modern avian lineages. BMC Biology  2008, 6: 6.  <ul class="find" data-citedArticleID="1395068" data-doi="10.1186/1741-7007-6-6"><li><a href="http://dx.doi.org/10.1186/1741-7007-6-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+mitochondrial+DNA+support+for+a+Cretaceous+origin+of+modern+avian+lineages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+mitochondrial+DNA+support+for+a+Cretaceous+origin+of+modern+avian+lineages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0005297-Renner1" id="pone.0005297-Renner1"></a>Renner SS, Grimm GW, Schneeweiss GM, Stuessy TF, Ricklefs RE (2008) Rooting and dating maples (<em>Acer</em>) with an uncorrelated-rates molecular clock: Implications for North American/Asian disjunctions. Syst Biol  57: 795–808.  <ul class="find" data-citedArticleID="1395104"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rooting+and+dating+maples+%28Acer%29+with+an+uncorrelated-rates+molecular+clock%3A+Implications+for+North+American%2FAsian+disjunctions.&amp;auth=&amp;atitle=Rooting+and+dating+maples+%28Acer%29+with+an+uncorrelated-rates+molecular+clock%3A+Implications+for+North+American%2FAsian+disjunctions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rooting+and+dating+maples+%28Acer%29+with+an+uncorrelated-rates+molecular+clock%3A+Implications+for+North+American%2FAsian+disjunctions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rooting+and+dating+maples+%28Acer%29+with+an+uncorrelated-rates+molecular+clock%3A+Implications+for+North+American%2FAsian+disjunctions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0005297-Erixon1" id="pone.0005297-Erixon1"></a>Erixon P, Oxelman B (2008) Whole-gene positive selection, elevated synonymous substitution rates, duplication, and indel evolution of the chloroplast <em>clpP1</em> gene. PLoS ONE  3: e1386.  <ul class="find" data-citedArticleID="1395034" data-doi="10.1371/journal.pone.0001386"><li><a href="http://dx.doi.org/10.1371/journal.pone.0001386" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-gene+positive+selection%2C+elevated+synonymous+substitution+rates%2C+duplication%2C+and+indel+evolution+of+the+chloroplast+clpP1+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-gene+positive+selection%2C+elevated+synonymous+substitution+rates%2C+duplication%2C+and+indel+evolution+of+the+chloroplast+clpP1+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0005297-Guisinger1" id="pone.0005297-Guisinger1"></a>Guisinger MM, Kuehl JV, Boore JL, Jansen RK (2008) Genome-wide analyses of Geraniaceae plastid DNA reveal unprecedented patterns of increased nucleotide substitutions. Proc Natl Acad Sci USA  105: 18424–18429.  <ul class="find" data-citedArticleID="1395078" data-doi="10.1073/pnas.0806759105"><li><a href="http://dx.doi.org/10.1073/pnas.0806759105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analyses+of+Geraniaceae+plastid+DNA+reveal+unprecedented+patterns+of+increased+nucleotide+substitutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analyses+of+Geraniaceae+plastid+DNA+reveal+unprecedented+patterns+of+increased+nucleotide+substitutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0005297-Smith1" id="pone.0005297-Smith1"></a>Smith SA, Donoghue MJ (2008) Rates of molecular evolution are linked to life history in flowering plants. Science  322: 86–89.  <ul class="find" data-citedArticleID="1395084" data-doi="10.1126/science.1163197"><li><a href="http://dx.doi.org/10.1126/science.1163197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+molecular+evolution+are+linked+to+life+history+in+flowering+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+molecular+evolution+are+linked+to+life+history+in+flowering+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0005297-Herendeen1" id="pone.0005297-Herendeen1"></a>Herendeen PS, Crane PR (1995)  The fossil history of the monocotyledons. In: Rudall PJ, Cribb PJ, Cutler DF, Humphries CJ, editors. Monocotyledon: Systematics and Evolution. London: Royal Botanic Gardens, Kew. pp. 1–21.  <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0005297-Linder1" id="pone.0005297-Linder1"></a>Linder HP, Rudall PJ (2005) Evolutionary history of Poales. Annu Rev Ecol Evol Syst  36: 107–124.  <ul class="find" data-citedArticleID="1395070" data-doi="10.1146/annurev.ecolsys.36.102403.135635"><li><a href="http://dx.doi.org/10.1146/annurev.ecolsys.36.102403.135635" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+history+of+Poales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+history+of+Poales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0005297-Yang4" id="pone.0005297-Yang4"></a>Yang Z, Rannala B (2006) Bayesian estimation of species divergence times under a molecular clock using multiple fossil calibrations with soft bounds. Mol Biol Evol  23: 212–226.  <ul class="find" data-citedArticleID="1395046" data-doi="10.1093/molbev/msj024"><li><a href="http://dx.doi.org/10.1093/molbev/msj024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+estimation+of+species+divergence+times+under+a+molecular+clock+using+multiple+fossil+calibrations+with+soft+bounds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+estimation+of+species+divergence+times+under+a+molecular+clock+using+multiple+fossil+calibrations+with+soft+bounds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0005297-Yang5" id="pone.0005297-Yang5"></a>Yang Z, Bielawski B (2000) Statistical methods for detecting molecular adaptation. TREE  15: 496–503.  <ul class="find" data-citedArticleID="1395108" data-doi="10.1016/s0169-5347(00)01994-7"><li><a href="http://dx.doi.org/10.1016/s0169-5347(00)01994-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+methods+for+detecting+molecular+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+methods+for+detecting+molecular+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0005297-Maier1" id="pone.0005297-Maier1"></a>Maier RM, Neckermann K, Igloi GL, Kossel H (1995) Complete sequence of the maize chloroplast genome: Gene content, hotspots of divergence and fine tuning of genetic information by transcript editing. J Mol Biol  251: 614–628.  <ul class="find" data-citedArticleID="1395076" data-doi="10.1006/jmbi.1995.0460"><li><a href="http://dx.doi.org/10.1006/jmbi.1995.0460" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+sequence+of+the+maize+chloroplast+genome%3A+Gene+content%2C+hotspots+of+divergence+and+fine+tuning+of+genetic+information+by+transcript+editing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+sequence+of+the+maize+chloroplast+genome%3A+Gene+content%2C+hotspots+of+divergence+and+fine+tuning+of+genetic+information+by+transcript+editing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.XML" value="119074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.PDF" value="410774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e001.PNG" value="8407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e001.TIF" value="22948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e002.PNG" value="7673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e002.TIF" value="22224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g001.PNG_L" value="130033"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g001.PNG_M" value="64034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g001.PNG_S" value="11000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g001.TIF" value="242976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g001.PNG_I" value="9931"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g002.PNG_L" value="334128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g002.PNG_M" value="188690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g002.PNG_S" value="16769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g002.TIF" value="495456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g002.PNG_I" value="24471"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g003.PNG_L" value="351289"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g003.PNG_M" value="218401"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g003.PNG_S" value="19498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g003.TIF" value="614576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.g003.PNG_I" value="33409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t001.PNG_L" value="280481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t001.PNG_M" value="435860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t001.PNG_S" value="25295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t001.TIF" value="1176830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t001.PNG_I" value="58159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t002.PNG_L" value="276680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t002.PNG_M" value="428305"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t002.PNG_S" value="25133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t002.TIF" value="1166886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t002.PNG_I" value="57110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t003.PNG_L" value="237267"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t003.PNG_M" value="355734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t003.PNG_S" value="22207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t003.TIF" value="979580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t003.PNG_I" value="46973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t004.PNG_L" value="123005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t004.PNG_M" value="156649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t004.PNG_S" value="14929"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t004.TIF" value="517158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t004.PNG_I" value="22503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t005.PNG_L" value="271386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t005.PNG_M" value="380285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t005.PNG_S" value="26062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t005.TIF" value="1204232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t005.PNG_I" value="52481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e003.PNG" value="6213"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e003.TIF" value="20780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e004.PNG" value="6281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e004.TIF" value="20816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e005.PNG" value="6367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e005.TIF" value="20940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e006.PNG" value="6375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e006.TIF" value="20956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e007.PNG" value="7459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e007.TIF" value="22000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e008.PNG" value="6224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e008.TIF" value="20776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e009.PNG" value="6251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e009.TIF" value="20800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t006.PNG_L" value="100172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t006.PNG_M" value="334825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t006.PNG_S" value="26612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t006.TIF" value="391374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.t006.PNG_I" value="51220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e010.PNG" value="12008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e010.TIF" value="27104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e011.PNG" value="12084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.e011.TIF" value="27156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.s001.DOC" value="39936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.s002.DOC" value="39424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.s003.DOC" value="40448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.s004.DOC" value="40448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.s005.DOC" value="44032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.s006.DOC" value="44544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0005297.s007.DOC" value="36864"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0005297&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0005297" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0005297&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0005297&volume=&issue=&title=Episodic Evolution and Adaptation of Chloroplast Genomes in Ancestral Grasses&author_name=Bojian%20Zhong%2C%20Takahiro%20Yonezawa%2C%20Yang%20Zhong%2C%20Masami%20Hasegawa&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0005297" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0005297" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0005297" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0005297&amp;t=Episodic%20Evolution%20and%20Adaptation%20of%20Chloroplast%20Genomes%20in%20Ancestral%20Grasses" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0005297&title=Episodic%20Evolution%20and%20Adaptation%20of%20Chloroplast%20Genomes%20in%20Ancestral%20Grasses&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0005297&amp;title=Episodic%20Evolution%20and%20Adaptation%20of%20Chloroplast%20Genomes%20in%20Ancestral%20Grasses" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0005297" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0005297&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Episodic Evolution and Adaptation of Chloroplast Genomes in Ancestral Grasses';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0005297';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Episodic%20Evolution%20and%20Adaptation%20of%20Chloroplast%20Genomes%20in%20Ancestral%20Grasses http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0005297" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0005297" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+adaptation%22" title="Search for articles in the subject area:'Evolutionary adaptation'"><div class="flagText">Evolutionary adaptation</div></a>
              <div data-categoryid="17795" data-articleid="33408"
                   data-categoryname="Evolutionary adaptation"
                   class="flagImage" title="Flag 'Evolutionary adaptation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Flowering+plants%22" title="Search for articles in the subject area:'Flowering plants'"><div class="flagText">Flowering plants</div></a>
              <div data-categoryid="43015" data-articleid="33408"
                   data-categoryname="Flowering plants"
                   class="flagImage" title="Flag 'Flowering plants' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="33408"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Grasses%22" title="Search for articles in the subject area:'Grasses'"><div class="flagText">Grasses</div></a>
              <div data-categoryid="43049" data-articleid="33408"
                   data-categoryname="Grasses"
                   class="flagImage" title="Flag 'Grasses' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oryza%22" title="Search for articles in the subject area:'Oryza'"><div class="flagText">Oryza</div></a>
              <div data-categoryid="44241" data-articleid="33408"
                   data-categoryname="Oryza"
                   class="flagImage" title="Flag 'Oryza' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+evolution%22" title="Search for articles in the subject area:'Plant evolution'"><div class="flagText">Plant evolution</div></a>
              <div data-categoryid="18531" data-articleid="33408"
                   data-categoryname="Plant evolution"
                   class="flagImage" title="Flag 'Plant evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Poaceae%22" title="Search for articles in the subject area:'Poaceae'"><div class="flagText">Poaceae</div></a>
              <div data-categoryid="44811" data-articleid="33408"
                   data-categoryname="Poaceae"
                   class="flagImage" title="Flag 'Poaceae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Zea%22" title="Search for articles in the subject area:'Zea'"><div class="flagText">Zea</div></a>
              <div data-categoryid="44007" data-articleid="33408"
                   data-categoryname="Zea"
                   class="flagImage" title="Flag 'Zea' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5604'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5754'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8814&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=7375">Welcome to any comment for our paper</a><br>Posted by bjzhong</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
