

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Molecular and Paleontological Evidence for a Post-Cretaceous Origin of Rodents</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0046445"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0046445" />

    <meta name="citation_title" content="Molecular and Paleontological Evidence for a Post-Cretaceous Origin of Rodents"/>
    <meta itemprop="name" content="Molecular and Paleontological Evidence for a Post-Cretaceous Origin of Rodents"/>

      <meta name="citation_author" content="Shaoyuan Wu"/>
            <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology and Museum of Comparative Zoology, Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Wenyu Wu"/>
            <meta name="citation_author_institution" content="Institute of Vertebrate Paleontology and Paleoanthropology, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Fuchun Zhang"/>
            <meta name="citation_author_institution" content="Xinjiang Key Laboratory of Biological Resources and Genetic Engineering, Xinjiang University, Urumqi, Xinjiang, China"/>
      <meta name="citation_author" content="Jie Ye"/>
            <meta name="citation_author_institution" content="Institute of Vertebrate Paleontology and Paleoanthropology, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Xijun Ni"/>
            <meta name="citation_author_institution" content="Institute of Vertebrate Paleontology and Paleoanthropology, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Jimin Sun"/>
            <meta name="citation_author_institution" content="Key Lab of Cenozoic Geology and Environment, Institute of Geology and Geophysics, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Scott V. Edwards"/>
            <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology and Museum of Comparative Zoology, Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Jin Meng"/>
            <meta name="citation_author_institution" content="Division of Paleontology, American Museum of Natural History, New York, New York, United States of America"/>
      <meta name="citation_author" content="Chris L. Organ"/>
            <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology and Museum of Comparative Zoology, Harvard University, Cambridge, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2012/10/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0046445.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e46445"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="7"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The modern molecular clock; citation_volume=4; citation_number=1; citation_pages=216-224; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Dating the time of origin of major clades: Molecular clocks and the fossil record; citation_volume=30; citation_number=2; citation_pages=65-88; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Local molecular clocks in three nuclear genes: divergence times for rodents and other mammals and incompatibility among fossil calibrations; citation_volume=57; citation_number=3; citation_pages=S201-S213; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Robust time estimation reconciles views of the antiquity of placental mammals; citation_volume=2; citation_number=4; citation_pages=e384; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mammalian evolution may not be strictly bifurcating; citation_volume=27; citation_number=5; citation_pages=2804-2816; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Phylogenomic datasets provide both precision and accuracy in estimating the timescale of placental mammal phylogeny; citation_number=6; citation_date=2012; " />
      <meta name="citation_reference" content="citation_number=7; " />
      <meta name="citation_reference" content="citation_title=New Late Paleocene rodents (Mammalia) from Big Multi Quarry, Washakie Basin, Wyoming; citation_volume=25; citation_number=8; citation_pages=301-321; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The osteology of Rhombomylus (Mammalia, Glires): Implications for phylogeny and evolution of Glires; citation_volume=275; citation_number=9; citation_pages=1-247; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Variance of molecular datings, evolution of rodents and the phylogenetic affinities between Ctenodactylidae and Hystricognathi; citation_volume=267; citation_number=10; citation_pages=393-402; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Multiple molecular evidences for a living mammalian fossil; citation_volume=104; citation_number=11; citation_pages=7495-7499; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Placental mammal diversification and the Cretaceous-Tertiary boundary; citation_volume=100; citation_number=12; citation_pages=1056-1061; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A molecular timescale for vertebrate evolution; citation_volume=392; citation_number=13; citation_pages=917-920; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The delayed rise of present-day mammals; citation_volume=446; citation_number=14; citation_pages=507-512; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Impacts of the Cretaceous terrestial revolution and KPg extinction on mammal diversification; citation_volume=334; citation_number=15; citation_pages=521-524; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny and divergence time estimates for major rodent groups: Evidence from multiple genes; citation_volume=18; citation_number=16; citation_pages=777-791; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular dating of phylogenetic trees: a brief review of current methods that estimate divergence times; citation_volume=12; citation_number=17; citation_pages=35-48; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Note on four species of dipodids (Dipodidae, Rodentia) from the Late Miocene Bahe Formation, Lantian, Shaanxi; citation_volume=43; citation_number=18; citation_pages=283-296; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Erlianomys combinatus, a primitive myodont rodent from the Eocene Arshanto Formation, Nuhetingboerhe, Nei Mongol, China; citation_volume=48; citation_number=19; citation_pages=133-144; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Tracing the origin and diversification of Dipodoidea (Order: Rodentia): Evidence from fossil record and molecular phylogeny; citation_number=20; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Middle Eocene small mammals from Liguanqiao Basin of Henan Province and Yuanqu Basin of Shanxi Province, central China; citation_volume=26; citation_number=21; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Papporicetodon, a pre-Oligocene cricetid genus (Rodentia) from central China; citation_volume=30; citation_number=22; citation_pages=1-16; citation_date=1992; " />
      <meta name="citation_reference" content="citation_number=23; " />
      <meta name="citation_reference" content="citation_number=24; " />
      <meta name="citation_reference" content="citation_title=South America's earliest rodent and recognition of a new interval of mammalian evolution; citation_volume=365; citation_number=25; citation_pages=434-437; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A New Mammaliaform from the Early Jurassic and Evolution of Mammalian Characteristics; citation_volume=292; citation_number=26; citation_pages=1535-1540; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A Jurassic eutherian mammal and the divergence of marsupials and placentals; citation_volume=476; citation_number=27; citation_pages=442-445; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=A new basal Caniform (Mammalia: Carnivora) from the middle Eocene of North America and remarks on the phylogeny of early carnivorans; citation_volume=6; citation_number=28; citation_pages=e24146; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=BEAST: Bayesian evolutionary analysis by sampling trees; citation_volume=7; citation_number=29; citation_pages=214; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Bayesian phylogenetic inference under mixed models; citation_volume=19; citation_number=30; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A simple, fast and accurate algorithm to estimate large phylogenies by maximum likelihood; citation_volume=52; citation_number=31; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock; citation_volume=19; citation_number=32; citation_pages=301-302; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Stem Lagomorpha and the Antiquity of Glires; citation_volume=307; citation_number=33; citation_pages=1091-1094; citation_date=2005; " />
      <meta name="citation_reference" content="citation_number=34; " />
      <meta name="citation_reference" content="citation_title=A review of the Eocene rodents of Pakistan; citation_volume=26; citation_number=35; citation_pages=19-35; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Fossil and molecular evidence constain scenarios for the early evolutionary and biogeographic history of hystricognathous rodents; citation_volume=106; citation_number=36; citation_pages=16722-16727; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Correcting parsimonious trees for unseen nucleotide substitutions: The effect of dense branching as exemplified by ribnuclease; citation_volume=7; citation_number=37; citation_pages=438-443; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=The evolution of prokaryotic ferredoxins-with a general method correcting for unobserved substitutions in less branched lineages; citation_volume=4; citation_number=38; citation_pages=381-394; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Detecting the node-sensity artifact in phylogeny reconstruction; citation_volume=55; citation_number=39; citation_pages=637-643; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The earliest known eutherian mammal; citation_number=40; citation_pages=816-822; citation_date=2002; " />
      <meta name="citation_reference" content="citation_number=41; " />
      <meta name="citation_reference" content="citation_title=High-level phylogeny of early Tertiary rodents: dental evidence; citation_volume=142; citation_number=42; citation_pages=105-134; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rodent phylogeny and a timescale for the evolution of Glires: Evidence from an extensive taxon sampling using three nuclear genes; citation_volume=19; citation_number=43; citation_pages=1053-1065; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Rodent phylogeny revised: analysis of six nuclear genes from all major rodent clades; citation_volume=9; citation_number=44; citation_pages=71; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Cretaceous eutherians and Laurasian origin for placental mammals near the K/T boundary; citation_volume=447; citation_number=45; citation_pages=1003-1006; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Fossil and phylogenies: integrating multiple lines of evidence to investigate the origin of early major metazoan lineages; citation_volume=47; citation_number=46; citation_pages=744-751; citation_date=2007; " />
      <meta name="citation_reference" content="citation_number=47; " />
      <meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution; citation_volume=14; citation_number=48; citation_pages=817; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models; citation_volume=19; citation_number=49; citation_pages=1572; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Relaxed phylogenetics and dating with confidence; citation_volume=4; citation_number=50; citation_pages=e88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Magnetostratigraphy of the Early Paleogene in the Erlian Basin; citation_volume=33; citation_number=51; citation_pages=62-68; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Late Oligocene-Miocene mid-latitude aridification and wind patterns in Asian interior; citation_volume=38; citation_number=52; citation_pages=515-518; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=The Neogene Dingshanyanchi Formation in northern Junggar basin of Xinjiang and its stratigraphic implications; citation_volume=46; citation_number=53; citation_pages=90-110; citation_date=2008; " />
      <meta name="citation_reference" content="citation_number=54; " />
      <meta name="citation_reference" content="citation_title=The Miocene mammals from Dingshanyanchi Formation of northern Junggar Basin, Xinjiang; citation_volume=47; citation_number=55; citation_pages=208; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=The history of the Dipodoidea (Rodentia, Mammalia) in the Miocene of Asia: 3. Allactaginae; citation_volume=34; citation_number=56; citation_pages=553-565; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0046445" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Molecular and Paleontological Evidence for a Post-Cretaceous Origin of Rodents"/>
    <meta name="twitter:description" content="The timing of the origin and diversification of rodents remains controversial, due to conflicting results from molecular clocks and paleontological data. The fossil record tends to support an early Cenozoic origin of crown-group rodents. In contrast, most molecular studies place the origin and initial diversification of crown-Rodentia deep in the Cretaceous, although some molecular analyses have recovered estimated divergence times that are more compatible with the fossil record. Here we attempt to resolve this conflict by carrying out a molecular clock investigation based on a nine-gene sequence dataset and a novel set of seven fossil constraints, including two new rodent records (the earliest known representatives of Cardiocraniinae and Dipodinae). Our results indicate that rodents originated around 61.7&ndash;62.4 Ma, shortly after the Cretaceous/Paleogene (K/Pg) boundary, and diversified at the intraordinal level around 57.7&ndash;58.9 Ma. These estimates are broadly consistent with the paleontological record, but challenge previous molecular studies that place the origin and early diversification of rodents in the Cretaceous. This study demonstrates that, with reliable fossil constraints, the incompatibility between paleontological and molecular estimates of rodent divergence times can be eliminated using currently available tools and genetic markers. Similar conflicts between molecular and paleontological evidence bedevil attempts to establish the origination times of other placental groups. The example of the present study suggests that more reliable fossil calibration points may represent the key to resolving these controversies."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0046445.g003"/>

  <meta property="og:title" content="Molecular and Paleontological Evidence for a Post-Cretaceous Origin of Rodents" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0046445" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8125'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6166'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6848&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046445">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0046445" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0046445&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Molecular and Paleontological Evidence for a Post-Cretaceous Origin of Rodents
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shaoyuan Wu
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:shaoyuan5@gmail.com ">shaoyuan5@gmail.com </a> (SW); <a href="mailto:jmeng@amnh.org ">jmeng@amnh.org </a> (JM)</p>

                <p>Affiliation:
                  Department of Organismic and Evolutionary Biology and Museum of Comparative Zoology, Harvard University, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wenyu Wu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Vertebrate Paleontology and Paleoanthropology, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fuchun Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Xinjiang Key Laboratory of Biological Resources and Genetic Engineering, Xinjiang University, Urumqi, Xinjiang, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jie Ye, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Vertebrate Paleontology and Paleoanthropology, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xijun Ni, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Vertebrate Paleontology and Paleoanthropology, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jimin Sun, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Lab of Cenozoic Geology and Environment, Institute of Geology and Geophysics, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Scott V. Edwards, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Organismic and Evolutionary Biology and Museum of Comparative Zoology, Harvard University, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jin Meng
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:shaoyuan5@gmail.com ">shaoyuan5@gmail.com </a> (SW); <a href="mailto:jmeng@amnh.org ">jmeng@amnh.org </a> (JM)</p>

                <p>Affiliation:
                  Division of Paleontology, American Museum of Natural History, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chris L. Organ
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Organismic and Evolutionary Biology and Museum of Comparative Zoology, Harvard University, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 05, 2012</li>
    <li>DOI: 10.1371/journal.pone.0046445</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0046445" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0046445" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0046445" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0046445" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0046445-g001" data-doi="info:doi/10.1371/journal.pone.0046445" data-uri="info:doi/10.1371/journal.pone.0046445.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0046445-g002" data-doi="info:doi/10.1371/journal.pone.0046445" data-uri="info:doi/10.1371/journal.pone.0046445.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0046445-t001" data-doi="info:doi/10.1371/journal.pone.0046445" data-uri="info:doi/10.1371/journal.pone.0046445.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0046445-t002" data-doi="info:doi/10.1371/journal.pone.0046445" data-uri="info:doi/10.1371/journal.pone.0046445.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0046445-g003" data-doi="info:doi/10.1371/journal.pone.0046445" data-uri="info:doi/10.1371/journal.pone.0046445.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0046445">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0046445" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The timing of the origin and diversification of rodents remains controversial, due to conflicting results from molecular clocks and paleontological data. The fossil record tends to support an early Cenozoic origin of crown-group rodents. In contrast, most molecular studies place the origin and initial diversification of crown-Rodentia deep in the Cretaceous, although some molecular analyses have recovered estimated divergence times that are more compatible with the fossil record. Here we attempt to resolve this conflict by carrying out a molecular clock investigation based on a nine-gene sequence dataset and a novel set of seven fossil constraints, including two new rodent records (the earliest known representatives of Cardiocraniinae and Dipodinae). Our results indicate that rodents originated around 61.7–62.4 Ma, shortly after the Cretaceous/Paleogene (K/Pg) boundary, and diversified at the intraordinal level around 57.7–58.9 Ma. These estimates are broadly consistent with the paleontological record, but challenge previous molecular studies that place the origin and early diversification of rodents in the Cretaceous. This study demonstrates that, with reliable fossil constraints, the incompatibility between paleontological and molecular estimates of rodent divergence times can be eliminated using currently available tools and genetic markers. Similar conflicts between molecular and paleontological evidence bedevil attempts to establish the origination times of other placental groups. The example of the present study suggests that more reliable fossil calibration points may represent the key to resolving these controversies.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wu S, Wu W, Zhang F, Ye J, Ni X, et al.  (2012) Molecular and Paleontological Evidence for a Post-Cretaceous Origin of Rodents. PLoS ONE 7(10):
          e46445.
            doi:10.1371/journal.pone.0046445</p><p><strong>Editor: </strong>Alistair Robert Evans, 
        Monash University, Australia
      </p><p><strong>Received:</strong> May 2, 2012; <strong>Accepted:</strong> August 31, 2012; <strong>Published:</strong> October 5, 2012</p><p><strong>Copyright:</strong> ©  Wu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Author SW was supported by the Putnam Expedition Grants and the Robert G. Goelet Research Fund from the Museum of Comparative Zoology of Harvard University. Authors WW, JY, XN, and JM were supported by the National Natural Science Foundation of China (408720320). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Molecular clocks and fossil record are the two major approaches to date evolutionary divergence times, which are crucial for using the Tree of Life to understand evolutionary processes and mechanisms. In the case of major divergences among groups of placental mammals, the general tendency has been for paleontological studies to suggest that these events took place in the Paleocene, while molecular ones place them deep in the Cretaceous <a href="#pone.0046445-Bromham1">[1]</a>, <a href="#pone.0046445-Smith1">[2]</a>, although some molecular studies have recovered estimated divergence times that are more compatible with the fossil record <a href="#pone.0046445-Douzery1">[3]</a>, <a href="#pone.0046445-Kitazoe1">[4]</a>, <a href="#pone.0046445-Hallstrom1">[5]</a>, <a href="#pone.0046445-dosReis1">[6]</a>. The general pattern of disagreement has confounded our ability to discern the influence of the K/Pg extinction event on the radiation of extant mammals. This problem is particularly evident in rodents, a group that accounts for approximate 42% of extant mammalian diversity <a href="#pone.0046445-Wilson1">[7]</a>. The oldest known fossil that can be clearly identified as a member of the rodent lineage, the fragmentary possible crown-rodent <em>Acritoparamys</em>, has a Late Paleocene age of about 57 million years (Ma) <a href="#pone.0046445-Dawson1">[8]</a>, <a href="#pone.0046445-Meng1">[9]</a>. However, early molecular studies almost unanimously supported a Cretaceous radiation of rodents <a href="#pone.0046445-Hallstrom1">[5]</a>, <a href="#pone.0046445-dosReis1">[6]</a>, <a href="#pone.0046445-Huchon1">[10]</a>, <a href="#pone.0046445-Huchon2">[11]</a>, <a href="#pone.0046445-Springer1">[12]</a>, <a href="#pone.0046445-Kumar1">[13]</a>, <a href="#pone.0046445-BinindaEmonds1">[14]</a>, <a href="#pone.0046445-Meredith1">[15]</a>, <a href="#pone.0046445-Adkins1">[16]</a>, although Douzery et al. <a href="#pone.0046445-Douzery1">[3]</a> and Kitazoe et al. <a href="#pone.0046445-Kitazoe1">[4]</a> obtained molecular results that placed the earliest divergences within crown-Rodentia in the Paleocene and was therefore compatible with the fossil record. Despite these exceptions, a strong discrepancy still persists between the fossil record and the preponderance of results from molecular clock studies. Resolving this discrepancy is therefore critical not only for understanding the evolutionary history and dynamics of rodents, but also for assessing the reliability of molecular clocks and fossils to accurately estimate divergence times.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Molecular clocks attempt to pinpoint divergence events whereas the fossil record alone can yield minimum estimates given by the first known fossil occurrence for a given group <a href="#pone.0046445-Bromham1">[1]</a>, <a href="#pone.0046445-Smith1">[2]</a>. The problem of molecular rate heterogeneity, a major source undermining the accuracy of molecular clock estimates, has been addressed by applying relaxed molecular clocks across sequences <a href="#pone.0046445-Rutschmann1">[17]</a>. The availability of reliable fossils that can be used as calibration points, therefore, may hold the key to obtaining an accurate time estimate of the origin and radiation of rodents. In this study, we employed five rodent calibrations based on recent fossil discoveries, including two new fossils that are chronologically constrained with palaeomagnetic chrons. The earliest known fossils of Dipodinae (three-toed jerboas) is dated to 10.5 Ma from the beginning of the Late Miocene of China (<a href="#pone.0046445.s001">Fig. S1</a> and <a href="#pone.0046445.s002">Fig. S2</a>), providing an upper (more recent) bound for the divergence time between Dipodinae and Allactaginae (five-toed jerboas). The earliest known fossils of the two extant genera of Cardiocraniinae (dwarf jerboas), <em>Cardiocranius</em> and <em>Salpingotus</em>, are dated to 9 Ma <a href="#pone.0046445-Li1">[18]</a>. The lower (more ancient) bound of each of these divergence events is 13 Ma, in the middle part of the Middle Miocene, based on recent biostratigraphic and paleomagnetic data. The earliest known myodont, <em>Erlianomys</em>, was discovered from the Early Eocene (54 Ma) <a href="#pone.0046445-Li2">[19]</a>, providing a reliable lowest known bound for the divergence time between the two primary myodont groups, Dipodoidea and Muroidea (<a href="#pone.0046445.s003">Fig. S3</a>) <a href="#pone.0046445-Zhang1">[20]</a>. The upper bound of this divergence can be constrained to 43 Ma, based on the earliest known dipodoids <a href="#pone.0046445-Tong1">[21]</a> and muroids <a href="#pone.0046445-Tong2">[22]</a> from China. We therefore use the split between mice and rats <a href="#pone.0046445-Jacobs1">[23]</a> as well as between octodontids and erethizontids <a href="#pone.0046445-Walton1">[24]</a>, <a href="#pone.0046445-Wyss1">[25]</a> as calibration points, because of their improved fossil record and stratigraphic data. In order to achieve a balanced distribution of calibration points within the phylogeny, we also applied two well-defined non-rodent calibrations in successive sister lineages to rodents, including the split of marsupials and placentals <a href="#pone.0046445-Luo1">[26]</a>, <a href="#pone.0046445-Luo2">[27]</a>, and feliforms and caniforms <a href="#pone.0046445-Tomiya1">[28]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We use the fossils noted above to create seven fossil calibrations with a nine-gene sequence dataset to re-evaluate the timing of rodent origin and diversification. For taxon sampling, we included major lineage across rodents, and sampled comprehensively within Dipodoidea to include all six subfamilies, an approach we referred to as “bottom-up” taxon sampling (i.e. building up an analytic model from a foundation of many individual data samples, versus the “top-down” approach of inferring an analytic model from relatively few data points). This sampling approach allowed us to accurately incorporate these new calibration points based on Chinese dipodoid fossils that are comparatively recent in geological time. Our analysis implements a relaxed molecular clock model using Bayesian and maximum likelihood approaches. Our results suggest that rodents originated and diversified after the K/Pg boundary at the beginning of the Cenozoic, a finding consistent with patterns found in the fossil record.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Test of molecular rate heterogeneity</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The BEAST <a href="#pone.0046445-Drummond1">[29]</a> analysis shows that substantial rate variation across the data set was only found in the CNR1 locus, with an ucld.stdev parameter of 1.788 (95% confidence interval (CI) 1.35–2.19). The ucld.stdev values of all other loci lie between 0 and 1, indicating that these loci have a moderate level of rate variation (see <a href="#pone.0046445.s006">Table S1</a>). The ucld.stdev parameter of the concatenated partitioned data set is 0.57 (95% CI 0.45–0.71), which is lower than that of all other individual genes except IRBP. These results demonstrate the necessity of utilizing a relaxed molecular clock mode on the DNA dataset for the molecular dating estimates.</p>


<h4>Molecular dating with complete fossil constraints</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We inferred a time-calibrated phylogenetic tree for 41 mammal species focusing on the superfamily Dipodoidea (jerboas and relatives) for which we sampled 18 species, representing all six subfamilies. Nine unlinked nuclear genes were used to construct the tree using Bayesian <a href="#pone.0046445-Ronquist1">[30]</a> and likelihood <a href="#pone.0046445-Guindon1">[31]</a> criteria. Topologically, the Bayesian and maximum likelihood approaches gave identical, highly resolved phylogeny, supporting Hystricomorpha as the most basal rodent clade (bootstrap support = 71, posterior probability = 0.98, <a href="#pone-0046445-g001">Fig. 1</a>). For time-calibrated trees, we estimated divergence times using Bayesian <a href="#pone.0046445-Drummond1">[29]</a> (<a href="#pone-0046445-g002">Fig. 2</a>) and likelihood <a href="#pone.0046445-Sanderson1">[32]</a> approaches, which returned similar results (<a href="#pone-0046445-t001">Table 1</a>). Our estimated divergence times are much younger than those estimated in previous molecular analyses, and are more congruent with the fossil record.</p>
<div class="figure" id="pone-0046445-g001"><div class="img"><a name="pone-0046445-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0046445" data-uri="info:doi/10.1371/journal.pone.0046445.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0046445.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0046445.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Phylogenetic relationships for rodents and outgroups estimated using Bayesian and maximum likelihood algorithms.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The Bayesian posterior probability and the maximum likelihood bootstrap values for each of the nodes are provided from the left to the right of the slash, respectively. Support scores are not shown for nodes that receive a full support of both posterior probability and bootstrap value.</p>
<span>doi:10.1371/journal.pone.0046445.g001</span></div><div class="figure" id="pone-0046445-g002"><div class="img"><a name="pone-0046445-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0046445" data-uri="info:doi/10.1371/journal.pone.0046445.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0046445.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0046445.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Molecular time scale for the orders of Rodentia, Lagomorpha, Primates, Carnivora and Perissodactyla obtained from the Bayesian estimates based on seven fossil calibration points and relaxed molecular clock model.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Fossil constraints are indicated by circle A to G on the corresponding nodes: <strong>A</strong>. 160–190 Ma for the split between Placentalia and Marsupialia; <strong>B</strong>. 38–61.7 Ma for the split between Caniformia and Feliformia; <strong>C</strong>. 28.5–37 Ma for the split between <em>Octodontomys</em> and <em>Erethizon</em>; <strong>D</strong>. 43–54 Ma for the basal split of Myodonta; <strong>E</strong>. 7.3–12.2 Ma for the split between mice and rats; <strong>F</strong>. 9–13 Ma for the split between <em>Cardiocranius</em> and <em>Salpingotus</em>; <strong>G</strong>. 10.5–13 Ma for the split between Dipodinae and Allactaginae.</p>
<span>doi:10.1371/journal.pone.0046445.g002</span></div><div class="figure" id="pone-0046445-t001"><div class="img"><a name="pone-0046445-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0046445" data-uri="info:doi/10.1371/journal.pone.0046445.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0046445.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0046445.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Support values of divergence dates for Bayesian and penalized likelihood estimates using seven fossil calibration points.</span></strong></p><span>doi:10.1371/journal.pone.0046445.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We estimate that the divergence between rodents and lagomorphs occurred about 61.7 Ma (Bayesian, Bayesian credibility interval (BCI) 52.8–71) or 62.4 Ma (likelihood). The basal divergence of rodents was estimated to be 57.7 Ma (Bayesian, BCI 50.1–66) or 58.9 Ma (likelihood). These estimates are roughly consistent with recent interpretations of the early gliran fossil record. The oldest known Glires, <em>Heomys</em> and <em>Mimotona</em>, may both fall on the lagomorph stem <a href="#pone.0046445-Asher1">[33]</a>, although Meng et al. <a href="#pone.0046445-Meng1">[9]</a> recovered <em>Heomys</em> as a stem-rodent rather than a stem-lagomorph. However, both analyses agree that <em>Heomys</em> and <em>Mimotona</em> belong to the gliran lineage. Both genera are placed in the early Late Paleocene <a href="#pone.0046445-Meng1">[9]</a>, <a href="#pone.0046445-Li3">[34]</a>, an age close to the rodent-lagomorph divergence time estimated by our study. The appearance of sciuromorphs was estimated at 55.6 Ma (Bayesian, BCI 48.4–63.2) or 57 Ma (likelihood). The divergence between castorimorphs and myodonts was estimated at 52.9 Ma (Bayesian, BCI 46.5–59.9) or 54.6 Ma (likelihood). In addition, we estimate the basal divergence of hystricomorphs to be 50.2 Ma (Bayesian, BCI 41.6–58.7) or 52.3 Ma (likelihood), a result consistent with the oldest hystricomorph fossils, which date to ~50 Ma <a href="#pone.0046445-Hartenberger1">[35]</a>. We estimate the origin of crown Hystricognathi to be 36.9 Ma (Bayesian, BCI 31.6–43) or 35.2 Ma (likelihood), compatible with the occurrence of the oldest hystricid fossils at ~34 Ma <a href="#pone.0046445-Sallam1">[36]</a>.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We tested for the Node Density Effect (NDE) <a href="#pone.0046445-Fitch1">[37]</a>, <a href="#pone.0046445-Fitch2">[38]</a> using the online utility available at (<a href="http://www.evolution.reading.ac.uk/pe/index.html">http://www.evolution.reading.ac.uk/pe/in​dex.html</a>), which is based on the method outlined by Venditti et al. (2006) <a href="#pone.0046445-Venditti1">[39]</a>. The results of this test indicate that the NDE is present in our tree (β significantly greater than zero). By successively pruning clades in the tree, we find that the NDE is present because of the correlation between nodes and branch lengths associated with the increasing sampling of Dipodinae and Allactaginae in relation to the rest of the tree. Reducing the numbers of taxa within Dipodinae and Allactaginae or pruning out Dipodinae removes the NDE. The NDE may have the effect of causing our estimates of divergence time to appear more recent, because unsampled lineages could downwardly bias the molecular divergences. To test this, we re-analyzed our data set using BEAST with reduced taxon sampling in the subfamilies Dipodinae and Allactaginae. For these two subfamilies, we sampled two taxa for each, including <em>Dipus sagitta</em> and <em>Jaculus blanfordi</em> for Dipodinae, and <em>Allactaga elater</em> and <em>Allactodipus bobrinskii</em> for Allactaginae, because such taxon sampling removes the NDE from our tree. Our results show that the BEAST analysis based on the reduced tree produced divergence times for major nodes that are similar to those for the full taxa (<a href="#pone.0046445.s011">Table S6</a>), and statistically we found no significant difference between these two estimates (t-test, p-value = 0.954). These results suggest that the impact of the NDE on the estimated divergence times for the rest of the tree is limited.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Although the confidence intervals attached to our Bayesian estimates are relatively wide, as is often the cases for studies like this one that employ a limited number of genes <a href="#pone.0046445-dosReis1">[6]</a>, the fact that our Bayesian and maximum likelihood estimates generally agree with one another reinforces the results of both analyses and suggests that the dates are reliable.</p>


<h4>Sensitivity test of fossil age constraints</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We tested the sensitivity of applying different fossil constraints for their impact on the estimated times of divergence within rodents.</p>


<h4>Bayesian</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Compared with the estimated dates using all constraints, omitting constraint D for the divergence between Dipodoidea and Muroidea has limited impact on the estimated times for all nodes. The estimated date of 44 Ma (BCI 33.9–54.9) for Dipodoidea-Muroidea divergence is close to 45.4 Ma (BCI 43–49.3) when this fossil constraint was employed (<a href="#pone-0046445-t002">Table 2</a>). When constraints on tip rodent nodes of calibration point E, F and G were relaxed, the estimates for most nodes increased remarkably, pushing the basal divergence of Glires into the Cretaceous at 68.4 Ma (BCI 55.6–82.6) (<a href="#pone-0046445-t002">Table 2</a>). When only using constraint E, the most commonly used rodent calibration for mouse-rat divergence, and the two non-rodent constraints A and B, as expected <a href="#pone.0046445-Springer1">[12]</a>, estimates for all nodes in Euarchontoglires incleased dramatically: 84.3 Ma (BCI 57.6–116.5) for the base of Boereoeutheria, 79 Ma (BCI 53.2–108.6) for the base of Euarchontoglires, 66.4 Ma (BCI 41–95.7) for the base of Primates, 74.7 Ma (BCI 50.6–103) for the base of Glires, and 69.7 Ma (BCI 46.5–96.1) for the base of Rodentia (<a href="#pone-0046445-t002">Table 2</a>). In addition, we test an alternative constraining of a minimum age of 124.6 Ma and a maximum age of 138.4 Ma for the split between placentals and marsupials to assess the sensitivity of descendant nodes to this date. The minimum age assignments were based on the Early Cretaceous <em>Eomaia</em> <a href="#pone.0046445-Ji1">[40]</a>, which was previously regarded as the oldest known eutherian before the recent discovery of <em>Juramaia sinensis</em> <a href="#pone.0046445-Luo2">[27]</a> from the Late Jurassic. The maximum age was based on the basal therian <em>Vincelestes</em> <a href="#pone.0046445-KielanJaworowska1">[41]</a>. The analysis shows that the change of this age resulted in slightly younger estimates of divergence times for deep nodes, but had little impact on recent nodes (<a href="#pone-0046445-t002">Table 2</a>). One major uncertainty in the rodent phylogeny is the position of the root of Rodentia. Based on dental morphology, Marivaux et al. <a href="#pone.0046445-Marivaux1">[42]</a> places Hystricomorpha as the basalmost clade of rodents. However, molecular phylogenetic studies support either Sciuromorpha, Hystricomorpha or the clade formed by Sciuromorpha and Hystricomorpha as the most basal clade of rodents, but all of these placements received relatively weak statistical support <a href="#pone.0046445-Huchon2">[11]</a>, <a href="#pone.0046445-Meredith1">[15]</a>, <a href="#pone.0046445-Huchon3">[43]</a>, <a href="#pone.0046445-BlangaKanfi1">[44]</a>. To test whether acceptance of the alternative topologies would strongly affect our dating estimates, we conducted two alternative BEAST analyses by changing the root of Rodentia from Hystricomorpha to (1) Sciuromorpha and (2) the clade formed by Sciuromorpha and Hystricomorpha. The results showed that neither permutation had an important impact on the results, so our major conclusions remain unchanged (<a href="#pone.0046445.s004">Figs. S4</a>, <a href="#pone.0046445.s005">S5</a>).</p>
<div class="figure" id="pone-0046445-t002"><div class="img"><a name="pone-0046445-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0046445" data-uri="info:doi/10.1371/journal.pone.0046445.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0046445.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0046445.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Summary of results of sensitivity test of fossil age constraints.</span></strong></p><span>doi:10.1371/journal.pone.0046445.t002</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The above analyses demonstrate that changes of the age of non-rodent constraints have limited impact on the estimated divergence times of nodes in the rodent tree. By contrast, removal of all three recent rodent constraints results in dramatic increase of ages for other nodes, reverting to the older divergence times estimated by earlier studies. Moreover, these results indicate that the use of a single mouse-rat constraint for the divergence time estimates for rodents can result in overestimates for all nodes in Glires.</p>


<h4>Likelihood</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>When constaints on tip rodent nodes E, F and G were relaxed, the estimated dates for all nodes change little (<a href="#pone-0046445-t002">Table 2</a>). This is not surprising, since age constraints on the root and deep nodes have a much bigger influence on the divergence estimates of other nodes in the program r8s <a href="#pone.0046445-Sanderson1">[32]</a>. Setting the age of the root to 180 Ma only caused a slight increase of the date estimates for nodes, including 64.8 Ma for the basal split of Glires, and 61.1 Ma for the basal diversification of Rodentia (<a href="#pone-0046445-t002">Table 2</a>). These analyses show that the influence of the root's age on the estimated dates of major rodent lineages is limited.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Three hypotheses have been proposed to characterize the evolutionary radiation of placental mammals: the Explosive Model puts the origin of placental orders and their intraordinal diversification shortly after the K/Pg boundary, whereas the Short Fuse Model places the origin of placental orders and intraordinal diversification in the Cretaceous, and the Long Fuse Model posits Cretaceous origins of placental orders but intraordinal diversification after the K/Pg boundary. Paleontological evidence favors the Explosive Model, suggesting that the origin and diversification of placental mammals occurred following the K/Pg extinction event that wiped out the non-avian dinosaurs and opened up many ecological niches. By contrast, recent molecular studies support either the Short Fuse or the Long Fuse Models, which suggests that continental breakup in the Late Cretaceous contributed to the origin and/or diversification of placental mammals, rather than the opening of ecological niches by differential extinction among groups. For rodents, most previous molecular studies consistently support a Short Fuse Model for them, making rodents one of the oldest placental orders which originated and diversified in the Cretaceous <a href="#pone.0046445-Huchon2">[11]</a>, <a href="#pone.0046445-Springer1">[12]</a>, <a href="#pone.0046445-Kumar1">[13]</a>, <a href="#pone.0046445-BinindaEmonds1">[14]</a>, <a href="#pone.0046445-Meredith1">[15]</a>. Because rodents lack a Cretaceous fossil record, however, there is no evidence to indicate whether their postulated diversification in the Cretaceous would have been driven by tectonic events or by other factors.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Recent phylogenetic studies, based on extensive sampling of fossil mammals, have placed all Cretaceous eutherians outside the placental crown groups <a href="#pone.0046445-Meng1">[9]</a>, <a href="#pone.0046445-Asher1">[33]</a>, <a href="#pone.0046445-Wible1">[45]</a>. Our estimated divergence times are consistent with a rapid radiation of major rodent lineages during the Paleocene. Therefore, our results agree with those of a few other recent molecular studies <a href="#pone.0046445-Douzery1">[3]</a>, <a href="#pone.0046445-Kitazoe1">[4]</a> in supporting the hypothesis that the origin and intraordinal diversification of rodents occurred after the K/Pg Boundary about 65 Ma, following the extinction of non-avian dinosaurs. These diversification patterns are consistent with the Explosive Model as applied to mammalian orders generally, rather than with the Short Fuse Models for radiation within the orders of Rodentia.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Our study is methodologically similar to others that employed the relaxed molecular clock and multiple fossil calibration points <a href="#pone.0046445-dosReis1">[6]</a>, <a href="#pone.0046445-Huchon2">[11]</a>, <a href="#pone.0046445-Springer1">[12]</a>, <a href="#pone.0046445-BinindaEmonds1">[14]</a>, <a href="#pone.0046445-Meredith1">[15]</a>, <a href="#pone.0046445-Zhang1">[20]</a> (<a href="#pone-0046445-g003">Fig. 3a</a>). However, the divergence times we obtained, particularly those for rodents, are significantly younger than those in some recent studies that considerably predate the K/Pg boundary (<a href="#pone-0046445-g003">Fig. 3b</a>). Compared to the results of previous studies, our younger estimates are probably attributable to the fact that we employed multiple, internal rodent fossil constraints, which are well documented stratigraphically in a continuous sequence dated with convincing paleomagnetic chrons (see <a href="#pone.0046445.s002">Figs. S2</a>, <a href="#pone.0046445.s003">S3</a>). The study of Springer et al. (2003) <a href="#pone.0046445-Springer1">[12]</a> applied one rodent constraint, the split of mouse and rat, with a minimum age of 12 Ma (<a href="#pone-0046445-g003">Fig. 3b</a>). But recent increased resolution of the fossil record has decreased the minimum age constraint for mouse-rat to be around 7.3 Ma <a href="#pone.0046445-Jacobs1">[23]</a>. Additionally, our sensitivity test shows that the use of a single rodent calibration point can result in overestimates for all nodes in rodents. Several recent studies employed multiple rodent fossil constraints <a href="#pone.0046445-dosReis1">[6]</a>, <a href="#pone.0046445-Huchon2">[11]</a>, <a href="#pone.0046445-Meredith1">[15]</a>, <a href="#pone.0046445-Zhang1">[20]</a>. However, their estimated divergence times for rodents are still similar to that of Springer et al. (2003), supporting the Short Fuse Model of rodent diversification (<a href="#pone-0046445-g003">Fig. 3b</a>).</p>
<div class="figure" id="pone-0046445-g003"><div class="img"><a name="pone-0046445-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0046445" data-uri="info:doi/10.1371/journal.pone.0046445.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0046445.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0046445.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0046445.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0046445.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0046445.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comparisons among rodent fossil constraints used and divergence times estimated.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p><strong>A</strong>. The temporal distribution of rodent fossil constraints used. The circles and squares represent the minimum and maximum age constraints for each of the fossil calibration points, respectively. The bar connecting the circle and square shows the range between the minimum and maximum age constraints. Note that most calibration points used by Huchon et al. 2007 and Meredith et al. 2011 have much larger gap between the minimum and maximum age constraints, compared to that used by this study. <strong>B</strong>. Divergence time estimates obtained for major boreoeutherian lineages. The estimates of Springer et al. 2003, Huchon et al. 2007 and Meredith et al. 2011 are close to each other, and predate the K/Pg boundary. However, the divergence estimates obtained by the present study are much younger and very close to the K/Pg boundary. Numbers from 1 to 5 indicate: <strong>1</strong>. Base of Rodentia; <strong>2</strong>. Base of Glires (rodents and lagomorphs); <strong>3</strong>. Base of Primates; <strong>4</strong>. Base of Euarchontoglires; <strong>5</strong>. Base of Boreoeutheria.</p>
<span>doi:10.1371/journal.pone.0046445.g003</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The major difference between the fossil constraints used in this study and those used by Huchon et al. (2007) <a href="#pone.0046445-Huchon2">[11]</a> and Meredith et al. (2011) <a href="#pone.0046445-Meredith1">[15]</a> is that the minimum and maximum age constraints for many rodent calibration points used in both the latter studies are farther apart than the constraints for points used in our study (<a href="#pone-0046445-g003">Fig. 3a</a>), because of uncertainty in the paleontological and stratigraphic data associated with the fossils in question. For example, the youngest rodent fossil constraint used by Meredith et al. (2011), the split between Ctenomyidae and Ocodontidae, has a minimum age of 9.07 Ma and a maximum age of 34 Ma, for a range of 24.93 Ma. Our fossil calibration points are well constrained, with smaller amounts of time between the minimum and maximum ages (<a href="#pone-0046445-g003">Fig. 3a</a>). By contrast, the Zhang et al. (2012) <a href="#pone.0046445-Zhang1">[20]</a> study was similar to ours in that dipodoids were densely sampled, but differed from ours in that sampling of major rodent lineages outside Dipodoidea was very limited. This lack of extensive sampling of rodent lineages may account for the fact that Zhang et al. (2012) recovered a comparatively early divergence time for the rodent-lagomorph node, consistent with earlier studies rather than with our results. Conversely, dos Reis et al. (2012) <a href="#pone.0046445-dosReis1">[6]</a> carried out a study with extensive genomic sampling and broad taxonomic sampling across Placentalia, but sampled relatively few rodents of any kind. dos Reis et al. (2012) estimated the rodent-lagomorph divergence to have occurred at 70.8 Ma (Confidence Interval: 69.9–71.8), considerably earlier than the estimate recovered by our study, and the discrepancy may arise from either the lack of dense sampling within Rodentia by dos Reis et al. (2012) or the lack of extensive genomic sampling and/or broad sampling outside Rodentia in our analysis. The strengths and weaknesses of these different sampling approaches remain to be fully explored.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Our results show that reconciliation between estimates of divergence times based on molecular clock and paleontological data is possible with standard tools and genetic markers, at least for the Rodentia, the most speciose of all mammalian orders. Achieving this consistency requires a reasonable number of reliable fossil calibration points supported by a well-constrained paleontological and stratigraphic record. The consistency between our results and the paleontological record suggests that similar controversies regarding the origin and diversification of other major biological groups, including the post-K/Pg diversification of various orders of modern mammals and birds <a href="#pone.0046445-Smith1">[2]</a> and the “Cambrian Explosion” of animal phyla <a href="#pone.0046445-Cartwright1">[46]</a> are potentially resolvable given adequate and reliable fossil calibration points.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Taxon and genomic sampling</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>This study includes 33 rodent species across major rodent lineages and eight outgroup taxa. The taxa examined and their classification are provided in <a href="#pone.0046445.s007">Table S2</a>. Portions of nine unlinked the nuclear genes were sampled, including alpha 2B adrenergic receptor (A2AB), cannabinoid receptor 1 (CNR1), growth hormone receptor (GHR), interphotoreceptor retinoid binding protein (IRBP), breast cancer susceptibility (BRCA1), von Willebrand factor (vWF), ATPase, Cu++ transporting, alpha polypeptide (ATP7A), 3′-UTR region of cAMP responsive element modulator (Crem), recombination activating gene 2 (RAG2). Detailed information of these loci is provided in <a href="#pone.0046445.s008">Table S3</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Genomic DNA was prepared from either muscle or liver tissue samples using DNeasy Tissue Kit (Qiagen, inc). PCR reactions were undertaken in 25-µL volumes with the following conditions: 94°C (5–10 min); 35 cycles of 94°C (45 s); 55°C (45 s); 72°C (40–60 s); 72°C (5–10 min). Sequence data were collected on ABI 3730 DNA Analyzers for both directions subsequent to Big Dye chemistry. The primers for both PCR and sequencing reactions are identified in <a href="#pone.0046445.s009">Table S4</a>.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>DNA sequences were aligned using Kalign as implemented in the program eBioX (<a href="http://www.ebioinformatics.org">http://www.ebioinformatics.org</a>) under default conditions, and refined manually using MacClade 4.06 <a href="#pone.0046445-Maddison1">[47]</a>. Ambiguous sites including potentially heterozygous sites were encoded based on IUPAC Ambiguity protocol. The GenBank accession numbers for each of the sequence data are provided in <a href="#pone.0046445.s010">Table S5</a>.</p>


<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Phylogenetic trees were constructed using maximum likelihood and Bayesian criteria. The Akaike information criterion was used to determine the best substitution models of sequence evolution based on the results from MODELTEST 3.07 <a href="#pone.0046445-Posada1">[48]</a> (<a href="#pone.0046445.s008">Table S3</a>). The maximum likelihood estimates for the best tree were performed with the program PhyML 3.0 <a href="#pone.0046445-Guindon1">[31]</a> for the concatenated dataset with 100 bootstrap replicates.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Bayesian inference of phylogeny was performed with the program MrBayes 3.1.2 <a href="#pone.0046445-Ronquist2">[49]</a>. We performed four independent runs under identical conditions with partitions defined for each of the nine loci evolving with independent model parameters. For each analysis, one run was performed with four chains, and was sampled every 2000 generations for fifty million generations after a burn-in cycle of 5000 trees. The convergence of each run was examined with the program Tracer 1.5 <a href="#pone.0046445-Drummond1">[29]</a>.</p>


<h4>Test of molecular rate heterogeneity</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The levels of molecular rate heterogeneity for the concatenated dataset and for each of the loci were examined in the program BEAST 1.5.4 <a href="#pone.0046445-Drummond1">[29]</a>. When running under the uncorrelated relaxed lognormal clock model, BEAST can measure the ucld.stdev parameter, which can determine how clock-like the DNA dataset is. The dataset is strictly clock-like if the ucld.stdev parameter is 0, and the dataset has substantial rate heterogeneity among lineages if the parameter is greater than 1. The dataset has a moderate level of heterogeneity if the ucld.stdev parameter lies between 0 and 1 <a href="#pone.0046445-Drummond1">[29]</a>. For each individual locus, the test incorporated in BEAST was performed using five million generations sampled every 500 generations. For the concatenated dataset, BEAST run was performed using 40 million generations, sampled every 500 generations for two independent runs.</p>


<h4>Molecular estimates of divergence times</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We employed two approaches to estimates divergence times of each node: a Bayesian method as implemented in the program BEAST 1.5.4, and a penalized likelihood method in the program r8s 1.71 <a href="#pone.0046445-Sanderson1">[32]</a>.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Bayesian analyses of molecular dating were estimated for the combined dataset with the substitution models for each gene partition. The relaxed molecular clock model was chosen for all BEAST analyses <a href="#pone.0046445-Drummond2">[50]</a>, since the estimated value of ucld.parameter is 0.57 for the concatenated DNA dataset. Each run of BEAST analyses comprised forty million generations, sampled either every 500 generations or 1000 generations for two independent runs. The output files of the two independent analyses were combined using LogCombiner 1.5.4 <a href="#pone.0046445-Drummond1">[29]</a> to produce the final results. Each run was examined with the program Tracer 1.5 <a href="#pone.0046445-Drummond1">[29]</a> for convergence.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>The program r8s <a href="#pone.0046445-Sanderson1">[32]</a> was used to compare maximum likelihood results with those obtained from BEAST. The r8s analyses were performed using the best maximum likelihood tree calculated in PhyML. We used the penalized likelihood model, the log penalty function and the truncated Newton algorithm based on the recommendation of the developer <a href="#pone.0046445-Sanderson1">[32]</a>. The optimal smoothing parameter was determined by a cross-validation run with r8s. For the nodes used as fossil age constraints, the check-gradient function was conducted to determine if the estimated values of divergence timings of these nodes fall beyond the age constraints. The date of root (marsupial-placental) was fixed at 160 and 180 Ma in order to assess the sensitivity of descendant nodes to the age of the root. Since the use of the two different root ages do not have significant impact on the estimated ages of other nodes (<a href="#pone-0046445-t002">Table 2</a>), this study used 160 as the age of the root for all r8s analyses.</p>


<h4>Fossil constraints</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>We applied seven fossil age constraints for the molecular dating analyses in this study. Minimum age constraints were based on the earliest known fossil record of a member of one of the divergent lineages. Where possible, the maximum age constraints are based on the age of the youngest well-sampled horizon that does not contain any members of the divergent lineages, in a stratigraphic sequence in which members of these lineages subsequently appear. When a stratigraphic sequence suitable for setting a particular upper bound by this method was not available, the age of the oldest member of the stem lineage leading up to the divergence was used as the upper bound. For the program BEAST, these calibration points were set as soft constraints with upper and lower bounds that allow for a 2.5% chance of lying beyond each user-input bound. The r8s program only allows the fossil calibrations to be set as a hard bound. Fossil constraints are as follows (<a href="#pone-0046445-g002">Fig. 2</a>):</p>


<ol class="alpha-upper">

<li>we assigned a minimum age of 160 Ma and a maximum age of 190 Ma for the divergence between marsupials and placentals, based on the earliest known placental mammal <em>Juramaia sinensis</em> <a href="#pone.0046445-Luo2">[27]</a> and the basal mammal <em>Hadrocodium</em> <a href="#pone.0046445-Luo1">[26]</a>.</li>

<li>We assigned a minimum age of 38 Ma and a maximum age of 61.7 Ma for the divergence between caniforms and feliforms, based on the oldest known crown carnivoran <em>Hesperocyon</em> and <em>Daphoenus</em> from the Late Eocene <a href="#pone.0046445-Tomiya1">[28]</a> and the oldest stem carnivore <em>Protictis schaffi</em> from the early Paleocene <a href="#pone.0046445-Tomiya1">[28]</a>, respectively.</li>

<li>We assigned a minimum age of 28.5 Ma and a maximum age of 37 Ma for the divergence between <em>Octodontomys</em> and the clade formed by <em>Cavia</em> and <em>Erethizon</em>, based on the oldest fossil record of Caviomorpha from the Late Eocene <a href="#pone.0046445-Wyss1">[25]</a> and the oldest fossil erethizontid (<em>Steiromys</em> sp.) from the mid-Oligocene <a href="#pone.0046445-Walton1">[24]</a>.</li>

<li>The earliest muroid is <em>Pappocricetodon</em> <a href="#pone.0046445-Tong2">[22]</a>, and the earliest dipodoid is <em>Primisminthus</em> and <em>Banyuesminthus</em> <a href="#pone.0046445-Tong1">[21]</a>. All the above species emerge in the middle part of the Eocene of China. <em>Erlianomys</em>, which is from the lower part of the Arshanto Formation in Nuhetingboerhe of Inner Mongolia, China, represents the earliest fossil record of myodonts <a href="#pone.0046445-Li2">[19]</a>. Based on recent magneto-stratigraphic analyses, the Nuhetingboerhe section was dated to the early part of the Early Eocene <a href="#pone.0046445-Sun1">[51]</a> (<a href="#pone.0046445.s003">Fig. S3</a>). Consistent with these fossil and stratigraphic results, we assigned a minimum age of 43 Ma and a maximum age of 54 Ma for the divergence between muroids and dipodoids.</li>

<li>We assigned a minimum age of 7.3 Ma and a maximum age of 12.2 Ma for the divergence between mice and rats, based on the occurence of the earliest known mouse <em>Mus</em> sp. and the earliest known <em>Prognomys</em> from the late Miocene of Pakistan <a href="#pone.0046445-Jacobs1">[23]</a>.</li>

<li>We assigned a minimum age of 9 Ma for the divergence between the two cardiocraniine genera <em>Cardiocranius</em> and <em>Salpingotus</em>, based on the occurence of the earliest known <em>Cardiocranius</em> (<em>C. pussillus</em>) and the earliest known <em>Salpingotus</em> (<em>S. primitivus</em>) from the Late Miocene of China <a href="#pone.0046445-Li1">[18]</a>.</li>

<li>We assigned a minimum age of 10.5 Ma for the divergence between the two dipodoid subfamilies Dipodinae and Allactaginae, based on the occurence of the earliest known dental fossils of Dipodinae from the middle bed of the Dingshanyanchi Formation, Xinjiang, China (<a href="#pone.0046445.s001">Fig. S1</a>). The cheek teeth of the Dingshanyanchi species lack the mesoloph and mesocone on the upper molars, and have no mesolophid and mesoconid on the lower molars. Cusps on the labial and lingual sides of each molar show an alternating, rather than opposite, arrangement. The anteroloph of M2 and ectolophid of the lower molars are oriented oblique to the longitudinal axis. These dental features represent synapomorphies of dipodine molars. According to stratigraphic and palaeomagnetic results, the middle bed of the Dingshanyanchi Formation falls toward the base of the long normal magnetic chron C5n.2n, and thus dates to the earliest part of the Late Miocene <a href="#pone.0046445-Sun2">[52]</a> (<a href="#pone.0046445.s002">Fig. S2</a>).</li>

</ol><a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>We set a maximum age of 13 Ma for the divergence between Dipodinae and Allactaginae and the divergence between <em>Salpingotus</em> and <em>Cardiocranius</em>. The Middle Miocene deposits are well exposed and have been extensively sampled in northern China and surrounding areas <a href="#pone.0046445-Meng2">[53]</a>, <a href="#pone.0046445-Qiu1">[54]</a>, <a href="#pone.0046445-Wu1">[55]</a>, <a href="#pone.0046445-Zazhigin1">[56]</a>. The only Dipodidae that can be found during the Middle Miocene is <em>Protalactaga</em>, a primitive genus. These deposits produce no species of extant dipodid subfamilies – Dipodinae, Allactaginae, Euchoreutinae and Cardiocraniinae – not even in the northern Junggar Basin of Xinjiang, which has a continuous geological sequence ranging from the Late Oligocene to the Late Miocene and where screen washing has been applied for decades <a href="#pone.0046445-Sun2">[52]</a>, <a href="#pone.0046445-Meng2">[53]</a>, <a href="#pone.0046445-Wu1">[55]</a>. On this basis, the maximal boundary for these two divergence events was placed in the middle part of the Middle Miocene.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0046445"></div><a name="pone.0046445.s001" id="pone.0046445.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI"><strong>Occlusal view of molars of the earliest fossil representative of Dipodinae from the Dingshanyanchi Formation.</strong> A. right m1 (IVPP V 16905.2). B. right m2 (IVPP V 16905.3). C. right M2 (IVPP V 16905.1). D. left m3 (IVPP V 16905.5). E. right m3 (IVPP V 16905.4).</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0046445.s002" id="pone.0046445.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI"><strong>Magneto-stratigaphic sequence of the Dingshanyanchi Formation, Xinjiang, China.</strong> This figure was based on and modified from Sun et al. 2010 <a href="#pone.0046445-Sun2">[52]</a>. Blue arrow indicates the layer that produced the earliest fossil representative of Dipodinae showing in <a href="#pone.0046445.s001">Figure S1</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0046445.s003" id="pone.0046445.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI"><strong>Magneto-stratigraphic sequence of the Arshanto Formation.</strong> This figure was based on and modified from Sun et al. 2009 <a href="#pone.0046445-Sun1">[51]</a>. Blue arrow indicates the layer that produced the earliest known myodont fossil, <em>Erlianomys</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0046445.s004" id="pone.0046445.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI"><strong>Divergence times obtained from Bayesian estimates based on the alternative topology with Sciuromorpha as the basal clade of Rodentia.</strong> Note that the estimated divergence times support a post-Cretaceous origin and diversification of Rodentia.</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0046445.s005" id="pone.0046445.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI"><strong>Divergence times obtained from Bayesian estimates based on the alternative topology with the clade of Sciuromorpha and Hystricomorpha as the basal clade of Rodentia.</strong> Note that the estimated divergence times support a post-Cretaceous origin and diversification of Rodentia.</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0046445.s006" id="pone.0046445.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s006">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Results of the test of molecular rate heterogeneity. The ucld.stdev parameters for each locus and the concatenated, partitioned data-set estimated by BEAST. Abbreviations: 95% C. I. = 95% Confidence Interval; ESS = Effective Sample Size.</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0046445.s007" id="pone.0046445.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s007">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">List of taxon sampling for this study. Abbreviations: MVZ = Museum of Vertebrate Zoology, University of California, Berkeley; MCZ = Museum of Comparative Zoology, Harvard University; AMNH = American Museum of Natural History; NMNH = National Museum of Natural History, Smithsonian Institution.</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0046445.s008" id="pone.0046445.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s008">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Characteristics of genes included show the AIC weights supporting the best model for each entry.</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0046445.s009" id="pone.0046445.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s009">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Primer.</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0046445.s010" id="pone.0046445.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s010">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">List of GenBank accession numbers.</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0046445.s011" id="pone.0046445.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0046445.s011">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Comparisons of divergence times for major nodes estimated using BEAST with full taxa and with a tree that is free of NDE by reducing taxon sampling in the subfamilies Dipodinae and Allactaginae. Values in parentheses are the 95% Bayesian credibility intervals. Note that these two analyses produced similar estimates of divergence times for major nodes. Statistical test shows that there is no significant difference between these two time estimates (t-test, p-value = 0.954).</p>
<p class="siDoi">doi:10.1371/journal.pone.0046445.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to D. Kramerov of the Russian Academy of Sciences, and A. Shahin of the Minia University of Egypt for donating DNA and tissue samples. We would like to thank F. Jenkins, L. Flynn and C. Sullivan for comments. We thank the Museum of Comparative Zoology (MCZ), Harvard University, the Museum of Vertebrate Zoology, University of California at Berkeley, the American Museum of Natural History, New York, and the National Museum of Natural History, Smithsonian Institution for the loan of tissues.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SW SE JM. Performed the experiments: SW. Analyzed the data: SW JM CO. Contributed reagents/materials/analysis tools: WW FZ JY XN JS. Wrote the paper: SW CO.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0046445-Bromham1" id="pone.0046445-Bromham1"></a>Bromham L, Penny D (2003) The modern molecular clock. Nat Rev Genet 4: 216–224.
        doi: 10.1038/nrg1020 <ul class="find" data-citedArticleID="5827281" data-doi="10.1038/nrg1020"><li><a href="http://dx.doi.org/10.1038/nrg1020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bromham%5Bauthor%5D+AND+The+modern+molecular+clock" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABromham+%22The+modern+molecular+clock%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0046445-Smith1" id="pone.0046445-Smith1"></a>Smith AB, Peterson KL (2002) Dating the time of origin of major clades: Molecular clocks and the fossil record. Ann Rev Earth Planet Sci 30: 65–88. <ul class="find" data-citedArticleID="5827283"><li><a href="http://www.crossref.org/guestquery/?auth2=Smith&amp;atitle2=Dating+the+time+of+origin+of+major+clades%3A+Molecular+clocks+and+the+fossil+record&amp;auth=Smith&amp;atitle=Dating+the+time+of+origin+of+major+clades%3A+Molecular+clocks+and+the+fossil+record" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Smith%5Bauthor%5D+AND+Dating+the+time+of+origin+of+major+clades%3A+Molecular+clocks+and+the+fossil+record" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASmith+%22Dating+the+time+of+origin+of+major+clades%3A+Molecular+clocks+and+the+fossil+record%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0046445-Douzery1" id="pone.0046445-Douzery1"></a>Douzery EJP, Delsuc F, Stanhope MJ, Huchon D (2003) Local molecular clocks in three nuclear genes: divergence times for rodents and other mammals and incompatibility among fossil calibrations. J Mol Evol 57: S201–S213.
        doi: 10.1007/s00239-003-0028-x <ul class="find" data-citedArticleID="5827285" data-doi="10.1007/s00239-003-0028-x"><li><a href="http://dx.doi.org/10.1007/s00239-003-0028-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Douzery%5Bauthor%5D+AND+Local+molecular+clocks+in+three+nuclear+genes%3A+divergence+times+for+rodents+and+other+mammals+and+incompatibility+among+fossil+calibrations" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADouzery+%22Local+molecular+clocks+in+three+nuclear+genes%3A+divergence+times+for+rodents+and+other+mammals+and+incompatibility+among+fossil+calibrations%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0046445-Kitazoe1" id="pone.0046445-Kitazoe1"></a>Kitazoe Y, Kishino H, Waddell PJ, Nakajima N, Okabayashi T, et al. (2007) Robust time estimation reconciles views of the antiquity of placental mammals. PLos one 2: e384.
        doi: 10.1371/journal.pone.0000384 <ul class="find" data-citedArticleID="5827287" data-doi="10.1371/journal.pone.0000384"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000384" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kitazoe%5Bauthor%5D+AND+Robust+time+estimation+reconciles+views+of+the+antiquity+of+placental+mammals" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKitazoe+%22Robust+time+estimation+reconciles+views+of+the+antiquity+of+placental+mammals%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0046445-Hallstrom1" id="pone.0046445-Hallstrom1"></a>Hallstrom BM, Janke A (2010) Mammalian evolution may not be strictly bifurcating. Mol Biol Evol 27: 2804–2816.
        doi: 10.1093/molbev/msq166 <ul class="find" data-citedArticleID="5827289" data-doi="10.1093/molbev/msq166"><li><a href="http://dx.doi.org/10.1093/molbev/msq166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hallstrom%5Bauthor%5D+AND+Mammalian+evolution+may+not+be+strictly+bifurcating" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHallstrom+%22Mammalian+evolution+may+not+be+strictly+bifurcating%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0046445-dosReis1" id="pone.0046445-dosReis1"></a>dos Reis M, Inoue J, Hasegawa M, Asher RJ, Donoghue MJ, et al. (2012) Phylogenomic datasets provide both precision and accuracy in estimating the timescale of placental mammal phylogeny. Proc R Soc B doi:10.1098/rspb.2012.0683. <ul class="find" data-citedArticleID="5827291" data-doi="10.1098/rspb.2012.0683"><li><a href="http://dx.doi.org/10.1098/rspb.2012.0683" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=dos+Reis%5Bauthor%5D+AND+Phylogenomic+datasets+provide+both+precision+and+accuracy+in+estimating+the+timescale+of+placental+mammal+phylogeny" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3Ados+Reis+%22Phylogenomic+datasets+provide+both+precision+and+accuracy+in+estimating+the+timescale+of+placental+mammal+phylogeny%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0046445-Wilson1" id="pone.0046445-Wilson1"></a>Wilson DE, Reeder DM (2005) Mammal Species of the World: A Taxonomic and Geographic Reference. Baltimore, MD: Johns Hopkins University Press. <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0046445-Dawson1" id="pone.0046445-Dawson1"></a>Dawson MR, Beard CK (1996) New Late Paleocene rodents (Mammalia) from Big Multi Quarry, Washakie Basin, Wyoming. Palaeovertebrata 25: 301–321.
        doi: 10.2992/007.080.0204 <ul class="find" data-citedArticleID="5827295" data-doi="10.2992/007.080.0204"><li><a href="http://dx.doi.org/10.2992/007.080.0204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dawson%5Bauthor%5D+AND+New+Late+Paleocene+rodents+%28Mammalia%29+from+Big+Multi+Quarry%2C+Washakie+Basin%2C+Wyoming" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADawson+%22New+Late+Paleocene+rodents+%28Mammalia%29+from+Big+Multi+Quarry%2C+Washakie+Basin%2C+Wyoming%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0046445-Meng1" id="pone.0046445-Meng1"></a>Meng J, Hu Y-M, Li C-K (2003) The osteology of <em>Rhombomylus</em> (Mammalia, Glires): Implications for phylogeny and evolution of Glires. Bull Amer Mus Natl Hist 275: 1–247.
        doi: 10.1206/0003-0090(2003)275&lt;0001:toormg&gt;2.0.co;2 <ul class="find" data-citedArticleID="5827297" data-doi="10.1206/0003-0090(2003)275<0001:toormg&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1206/0003-0090(2003)275<0001:toormg&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Meng%5Bauthor%5D+AND+The+osteology+of+Rhombomylus+%28Mammalia%2C+Glires%29%3A+Implications+for+phylogeny+and+evolution+of+Glires" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMeng+%22The+osteology+of+Rhombomylus+%28Mammalia%2C+Glires%29%3A+Implications+for+phylogeny+and+evolution+of+Glires%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0046445-Huchon1" id="pone.0046445-Huchon1"></a>Huchon D, Catzeflis FM, Douzery EJP (2000) Variance of molecular datings, evolution of rodents and the phylogenetic affinities between Ctenodactylidae and Hystricognathi. Proc Zool Soc London B 267: 393–402.
        doi: 10.1098/rspb.2000.1014 <ul class="find" data-citedArticleID="5827299" data-doi="10.1098/rspb.2000.1014"><li><a href="http://dx.doi.org/10.1098/rspb.2000.1014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Huchon%5Bauthor%5D+AND+Variance+of+molecular+datings%2C+evolution+of+rodents+and+the+phylogenetic+affinities+between+Ctenodactylidae+and+Hystricognathi" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHuchon+%22Variance+of+molecular+datings%2C+evolution+of+rodents+and+the+phylogenetic+affinities+between+Ctenodactylidae+and+Hystricognathi%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0046445-Huchon2" id="pone.0046445-Huchon2"></a>Huchon D, Chevret P, Jordan U, Kilpatrick CW, Ranwez V, et al. (2007) Multiple molecular evidences for a living mammalian fossil. Proc Natl Acad Sci 104: 7495–7499.
        doi: 10.1073/pnas.0701289104 <ul class="find" data-citedArticleID="5827301" data-doi="10.1073/pnas.0701289104"><li><a href="http://dx.doi.org/10.1073/pnas.0701289104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Huchon%5Bauthor%5D+AND+Multiple+molecular+evidences+for+a+living+mammalian+fossil" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHuchon+%22Multiple+molecular+evidences+for+a+living+mammalian+fossil%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0046445-Springer1" id="pone.0046445-Springer1"></a>Springer MS, Murphy WJ, Eizirik E, O'Brien SJ (2003) Placental mammal diversification and the Cretaceous-Tertiary boundary. Proc Natl Acad Sci 100: 1056–1061.
        doi: 10.1073/pnas.0334222100 <ul class="find" data-citedArticleID="5827303" data-doi="10.1073/pnas.0334222100"><li><a href="http://dx.doi.org/10.1073/pnas.0334222100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Springer%5Bauthor%5D+AND+Placental+mammal+diversification+and+the+Cretaceous-Tertiary+boundary" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASpringer+%22Placental+mammal+diversification+and+the+Cretaceous-Tertiary+boundary%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0046445-Kumar1" id="pone.0046445-Kumar1"></a>Kumar S, Hedges SB (1998) A molecular timescale for vertebrate evolution. Nature 392: 917–920.
        doi: 10.1038/31927 <ul class="find" data-citedArticleID="5827305" data-doi="10.1038/31927"><li><a href="http://dx.doi.org/10.1038/31927" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kumar%5Bauthor%5D+AND+A+molecular+timescale+for+vertebrate+evolution" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKumar+%22A+molecular+timescale+for+vertebrate+evolution%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0046445-BinindaEmonds1" id="pone.0046445-BinindaEmonds1"></a>Bininda-Emonds ORP, Cardillo M, Jones KE, MacPhee RDE, Beck RMD, et al. (2007) The delayed rise of present-day mammals. Nature 446: 507–512.
        doi: 10.1038/nature05634 <ul class="find" data-citedArticleID="5827307" data-doi="10.1038/nature05634"><li><a href="http://dx.doi.org/10.1038/nature05634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bininda-Emonds%5Bauthor%5D+AND+The+delayed+rise+of+present-day+mammals" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABininda-Emonds+%22The+delayed+rise+of+present-day+mammals%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0046445-Meredith1" id="pone.0046445-Meredith1"></a>Meredith RW, Janecka JE, Gatesy J, Ryder OA, Fisher CA, et al. (2011) Impacts of the Cretaceous terrestial revolution and KPg extinction on mammal diversification. Science 334: 521–524.
        doi: 10.1126/science.1211028 <ul class="find" data-citedArticleID="5827309" data-doi="10.1126/science.1211028"><li><a href="http://dx.doi.org/10.1126/science.1211028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Meredith%5Bauthor%5D+AND+Impacts+of+the+Cretaceous+terrestial+revolution+and+KPg+extinction+on+mammal+diversification" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMeredith+%22Impacts+of+the+Cretaceous+terrestial+revolution+and+KPg+extinction+on+mammal+diversification%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0046445-Adkins1" id="pone.0046445-Adkins1"></a>Adkins RM, Gelke EL, Rowe D, Honeycutt RL (2001) Molecular phylogeny and divergence time estimates for major rodent groups: Evidence from multiple genes. Mol Biol Evol 18: 777–791.
        doi: 10.1093/oxfordjournals.molbev.a003860 <ul class="find" data-citedArticleID="5827311" data-doi="10.1093/oxfordjournals.molbev.a003860"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003860" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adkins%5Bauthor%5D+AND+Molecular+phylogeny+and+divergence+time+estimates+for+major+rodent+groups%3A+Evidence+from+multiple+genes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAdkins+%22Molecular+phylogeny+and+divergence+time+estimates+for+major+rodent+groups%3A+Evidence+from+multiple+genes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0046445-Rutschmann1" id="pone.0046445-Rutschmann1"></a>Rutschmann F (2006) Molecular dating of phylogenetic trees: a brief review of current methods that estimate divergence times. Divers Distrib 12: 35–48.
        doi: 10.1111/j.1366-9516.2006.00210.x <ul class="find" data-citedArticleID="5827313" data-doi="10.1111/j.1366-9516.2006.00210.x"><li><a href="http://dx.doi.org/10.1111/j.1366-9516.2006.00210.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rutschmann%5Bauthor%5D+AND+Molecular+dating+of+phylogenetic+trees%3A+a+brief+review+of+current+methods+that+estimate+divergence+times" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARutschmann+%22Molecular+dating+of+phylogenetic+trees%3A+a+brief+review+of+current+methods+that+estimate+divergence+times%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0046445-Li1" id="pone.0046445-Li1"></a>Li Q, Zheng S-H (2005) Note on four species of dipodids (Dipodidae, Rodentia) from the Late Miocene Bahe Formation, Lantian, Shaanxi. Vert PalAsiat 43: 283–296. <ul class="find" data-citedArticleID="5827315"><li><a href="http://www.crossref.org/guestquery/?auth2=Li&amp;atitle2=Note+on+four+species+of+dipodids+%28Dipodidae%2C+Rodentia%29+from+the+Late+Miocene+Bahe+Formation%2C+Lantian%2C+Shaanxi&amp;auth=Li&amp;atitle=Note+on+four+species+of+dipodids+%28Dipodidae%2C+Rodentia%29+from+the+Late+Miocene+Bahe+Formation%2C+Lantian%2C+Shaanxi" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Li%5Bauthor%5D+AND+Note+on+four+species+of+dipodids+%28Dipodidae%2C+Rodentia%29+from+the+Late+Miocene+Bahe+Formation%2C+Lantian%2C+Shaanxi" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALi+%22Note+on+four+species+of+dipodids+%28Dipodidae%2C+Rodentia%29+from+the+Late+Miocene+Bahe+Formation%2C+Lantian%2C+Shaanxi%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0046445-Li2" id="pone.0046445-Li2"></a>Li Q, Meng J (2010) <em>Erlianomys combinatus</em>, a primitive myodont rodent from the Eocene Arshanto Formation, Nuhetingboerhe, Nei Mongol, China. Vert PalAsiat 48: 133–144. <ul class="find" data-citedArticleID="5827317"><li><a href="http://www.crossref.org/guestquery/?auth2=Li&amp;atitle2=Erlianomys+combinatus%2C+a+primitive+myodont+rodent+from+the+Eocene+Arshanto+Formation%2C+Nuhetingboerhe%2C+Nei+Mongol%2C+China&amp;auth=Li&amp;atitle=Erlianomys+combinatus%2C+a+primitive+myodont+rodent+from+the+Eocene+Arshanto+Formation%2C+Nuhetingboerhe%2C+Nei+Mongol%2C+China" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Li%5Bauthor%5D+AND+Erlianomys+combinatus%2C+a+primitive+myodont+rodent+from+the+Eocene+Arshanto+Formation%2C+Nuhetingboerhe%2C+Nei+Mongol%2C+China" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALi+%22Erlianomys+combinatus%2C+a+primitive+myodont+rodent+from+the+Eocene+Arshanto+Formation%2C+Nuhetingboerhe%2C+Nei+Mongol%2C+China%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0046445-Zhang1" id="pone.0046445-Zhang1"></a>Zhang Q, Xia L, Kimura Y, Shenbrot G, Zhang Z-Q, et al. (2012) Tracing the origin and diversification of Dipodoidea (Order: Rodentia): Evidence from fossil record and molecular phylogeny. Evol Biol doi: 10.1007/s11692-11012-19167-11696. <ul class="find" data-citedArticleID="5827319" data-doi="10.1007/s11692-012-9167-6"><li><a href="http://dx.doi.org/10.1007/s11692-012-9167-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zhang%5Bauthor%5D+AND+Tracing+the+origin+and+diversification+of+Dipodoidea+%28Order%3A+Rodentia%29%3A+Evidence+from+fossil+record+and+molecular+phylogeny" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AZhang+%22Tracing+the+origin+and+diversification+of+Dipodoidea+%28Order%3A+Rodentia%29%3A+Evidence+from+fossil+record+and+molecular+phylogeny%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0046445-Tong1" id="pone.0046445-Tong1"></a>Tong Y-S (1997) Middle Eocene small mammals from Liguanqiao Basin of Henan Province and Yuanqu Basin of Shanxi Province, central China. Paleont Sinica, N S C 26. <ul class="find" data-citedArticleID="5827321"><li><a href="http://www.crossref.org/guestquery/?auth2=Tong&amp;atitle2=Middle+Eocene+small+mammals+from+Liguanqiao+Basin+of+Henan+Province+and+Yuanqu+Basin+of+Shanxi+Province%2C+central+China&amp;auth=Tong&amp;atitle=Middle+Eocene+small+mammals+from+Liguanqiao+Basin+of+Henan+Province+and+Yuanqu+Basin+of+Shanxi+Province%2C+central+China" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tong%5Bauthor%5D+AND+Middle+Eocene+small+mammals+from+Liguanqiao+Basin+of+Henan+Province+and+Yuanqu+Basin+of+Shanxi+Province%2C+central+China" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATong+%22Middle+Eocene+small+mammals+from+Liguanqiao+Basin+of+Henan+Province+and+Yuanqu+Basin+of+Shanxi+Province%2C+central+China%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0046445-Tong2" id="pone.0046445-Tong2"></a>Tong Y-S (1992) <em>Papporicetodon</em>, a pre-Oligocene cricetid genus (Rodentia) from central China. Vert PalAsiat 30: 1–16. <ul class="find" data-citedArticleID="5827323"><li><a href="http://www.crossref.org/guestquery/?auth2=Tong&amp;atitle2=Papporicetodon%2C+a+pre-Oligocene+cricetid+genus+%28Rodentia%29+from+central+China&amp;auth=Tong&amp;atitle=Papporicetodon%2C+a+pre-Oligocene+cricetid+genus+%28Rodentia%29+from+central+China" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tong%5Bauthor%5D+AND+Papporicetodon%2C+a+pre-Oligocene+cricetid+genus+%28Rodentia%29+from+central+China" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATong+%22Papporicetodon%2C+a+pre-Oligocene+cricetid+genus+%28Rodentia%29+from+central+China%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0046445-Jacobs1" id="pone.0046445-Jacobs1"></a>Jacobs LJ, Flynn LJ (2005) Of mice again: the Siwalik rodent record, murine distribution, and molecular clocks. In: Lieberman DE, Simth RJ, Kelley SJ, editors. Interpreting the past: Essays on human, primate, and mammal evolution in honor of David Pilbeam. Boston, MA: Brill Academic Publishers. pp. 63–80. <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0046445-Walton1" id="pone.0046445-Walton1"></a>Walton AH (1997) Rodents. In: Kay RF, Madden RH, Cifelli RL, Flynn JJ, editors. Vertebrate paleontology in the Neotropics The Miocene Fauna of La Venta, Colombia. Washington, DC: Smithsonian Institution Press. pp. 392–409. <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0046445-Wyss1" id="pone.0046445-Wyss1"></a>Wyss AR, Flynn JJ, Norell MA, Swisher CC, Charrier R, et al. (1993) South America's earliest rodent and recognition of a new interval of mammalian evolution. Nature 365: 434–437.
        doi: 10.1038/365434a0 <ul class="find" data-citedArticleID="5827329" data-doi="10.1038/365434a0"><li><a href="http://dx.doi.org/10.1038/365434a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wyss%5Bauthor%5D+AND+South+America%27s+earliest+rodent+and+recognition+of+a+new+interval+of+mammalian+evolution" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWyss+%22South+America%27s+earliest+rodent+and+recognition+of+a+new+interval+of+mammalian+evolution%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0046445-Luo1" id="pone.0046445-Luo1"></a>Luo Z-X, Crompton AW, Sun A-L (2001) A New Mammaliaform from the Early Jurassic and Evolution of Mammalian Characteristics. Science 292: 1535–1540.
        doi: 10.1126/science.1058476 <ul class="find" data-citedArticleID="5827331" data-doi="10.1126/science.1058476"><li><a href="http://dx.doi.org/10.1126/science.1058476" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Luo%5Bauthor%5D+AND+A+New+Mammaliaform+from+the+Early+Jurassic+and+Evolution+of+Mammalian+Characteristics" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALuo+%22A+New+Mammaliaform+from+the+Early+Jurassic+and+Evolution+of+Mammalian+Characteristics%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0046445-Luo2" id="pone.0046445-Luo2"></a>Luo Z-X, Yuan C-X, Meng Q-J, Ji Q (2011) A Jurassic eutherian mammal and the divergence of marsupials and placentals. Nature 476: 442–445.
        doi: 10.1038/nature10291 <ul class="find" data-citedArticleID="5827333" data-doi="10.1038/nature10291"><li><a href="http://dx.doi.org/10.1038/nature10291" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Luo%5Bauthor%5D+AND+A+Jurassic+eutherian+mammal+and+the+divergence+of+marsupials+and+placentals" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALuo+%22A+Jurassic+eutherian+mammal+and+the+divergence+of+marsupials+and+placentals%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0046445-Tomiya1" id="pone.0046445-Tomiya1"></a>Tomiya S (2011) A new basal Caniform (Mammalia: Carnivora) from the middle Eocene of North America and remarks on the phylogeny of early carnivorans. PLoS one 6: e24146.
        doi: 10.1371/journal.pone.0024146 <ul class="find" data-citedArticleID="5827335" data-doi="10.1371/journal.pone.0024146"><li><a href="http://dx.doi.org/10.1371/journal.pone.0024146" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tomiya%5Bauthor%5D+AND+A+new+basal+Caniform+%28Mammalia%3A+Carnivora%29+from+the+middle+Eocene+of+North+America+and+remarks+on+the+phylogeny+of+early+carnivorans" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATomiya+%22A+new+basal+Caniform+%28Mammalia%3A+Carnivora%29+from+the+middle+Eocene+of+North+America+and+remarks+on+the+phylogeny+of+early+carnivorans%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0046445-Drummond1" id="pone.0046445-Drummond1"></a>Drummond AJ, Rambaut A (2007) BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evol Biol 7: 214.
        doi: 10.1186/1471-2148-7-214 <ul class="find" data-citedArticleID="5827337" data-doi="10.1186/1471-2148-7-214"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-214" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drummond%5Bauthor%5D+AND+BEAST%3A+Bayesian+evolutionary+analysis+by+sampling+trees" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADrummond+%22BEAST%3A+Bayesian+evolutionary+analysis+by+sampling+trees%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0046445-Ronquist1" id="pone.0046445-Ronquist1"></a>Ronquist F, Huelsenback JP (2003) Bayesian phylogenetic inference under mixed models. Bioinformations 19: 1572–1574.
        doi: 10.1093/bioinformatics/btg180 <ul class="find" data-citedArticleID="5827339" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ronquist%5Bauthor%5D+AND+Bayesian+phylogenetic+inference+under+mixed+models" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARonquist+%22Bayesian+phylogenetic+inference+under+mixed+models%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0046445-Guindon1" id="pone.0046445-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol 52: 696–704. <ul class="find" data-citedArticleID="5827341"><li><a href="http://www.crossref.org/guestquery/?auth2=Guindon&amp;atitle2=A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood&amp;auth=Guindon&amp;atitle=A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Guindon%5Bauthor%5D+AND+A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGuindon+%22A+simple%2C+fast+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0046445-Sanderson1" id="pone.0046445-Sanderson1"></a>Sanderson MJ (2003) r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock. Bioinformatics 19: 301–302.
        doi: 10.1093/bioinformatics/19.2.301 <ul class="find" data-citedArticleID="5827343" data-doi="10.1093/bioinformatics/19.2.301"><li><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.301" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sanderson%5Bauthor%5D+AND+r8s%3A+inferring+absolute+rates+of+molecular+evolution+and+divergence+times+in+the+absence+of+a+molecular+clock" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASanderson+%22r8s%3A+inferring+absolute+rates+of+molecular+evolution+and+divergence+times+in+the+absence+of+a+molecular+clock%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0046445-Asher1" id="pone.0046445-Asher1"></a>Asher RJ, Meng J, Wible JR, McKenna MC, Rougier GW, et al. (2005) Stem Lagomorpha and the Antiquity of Glires. Science 307: 1091–1094.
        doi: 10.1126/science.1107808 <ul class="find" data-citedArticleID="5827345" data-doi="10.1126/science.1107808"><li><a href="http://dx.doi.org/10.1126/science.1107808" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asher%5Bauthor%5D+AND+Stem+Lagomorpha+and+the+Antiquity+of+Glires" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAsher+%22Stem+Lagomorpha+and+the+Antiquity+of+Glires%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0046445-Li3" id="pone.0046445-Li3"></a>Li C-K, Chow M-C (1994) The origin of rodents. In: Tomida Y, editor. Rodent and lagomorph families of Asian origins and diversification. Tokyo, Japan: National Science Museum. <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0046445-Hartenberger1" id="pone.0046445-Hartenberger1"></a>Hartenberger JL (1982) A review of the Eocene rodents of Pakistan. Univ Mich Mus Paleo Contr 26: 19–35. <ul class="find" data-citedArticleID="5827349"><li><a href="http://www.crossref.org/guestquery/?auth2=Hartenberger&amp;atitle2=A+review+of+the+Eocene+rodents+of+Pakistan&amp;auth=Hartenberger&amp;atitle=A+review+of+the+Eocene+rodents+of+Pakistan" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hartenberger%5Bauthor%5D+AND+A+review+of+the+Eocene+rodents+of+Pakistan" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHartenberger+%22A+review+of+the+Eocene+rodents+of+Pakistan%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0046445-Sallam1" id="pone.0046445-Sallam1"></a>Sallam HM, Seiffert ER, Steiper ME, Simons EL (2009) Fossil and molecular evidence constain scenarios for the early evolutionary and biogeographic history of hystricognathous rodents. Proc Natl Acad Sci 106: 16722–16727.
        doi: 10.1073/pnas.0908702106 <ul class="find" data-citedArticleID="5827351" data-doi="10.1073/pnas.0908702106"><li><a href="http://dx.doi.org/10.1073/pnas.0908702106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sallam%5Bauthor%5D+AND+Fossil+and+molecular+evidence+constain+scenarios+for+the+early+evolutionary+and+biogeographic+history+of+hystricognathous+rodents" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASallam+%22Fossil+and+molecular+evidence+constain+scenarios+for+the+early+evolutionary+and+biogeographic+history+of+hystricognathous+rodents%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0046445-Fitch1" id="pone.0046445-Fitch1"></a>Fitch WM, Beintema JJ (1990) Correcting parsimonious trees for unseen nucleotide substitutions: The effect of dense branching as exemplified by ribnuclease. Mol Biol Evol 7: 438–443. <ul class="find" data-citedArticleID="5827353"><li><a href="http://www.crossref.org/guestquery/?auth2=Fitch&amp;atitle2=Correcting+parsimonious+trees+for+unseen+nucleotide+substitutions%3A+The+effect+of+dense+branching+as+exemplified+by+ribnuclease&amp;auth=Fitch&amp;atitle=Correcting+parsimonious+trees+for+unseen+nucleotide+substitutions%3A+The+effect+of+dense+branching+as+exemplified+by+ribnuclease" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitch%5Bauthor%5D+AND+Correcting+parsimonious+trees+for+unseen+nucleotide+substitutions%3A+The+effect+of+dense+branching+as+exemplified+by+ribnuclease" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFitch+%22Correcting+parsimonious+trees+for+unseen+nucleotide+substitutions%3A+The+effect+of+dense+branching+as+exemplified+by+ribnuclease%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0046445-Fitch2" id="pone.0046445-Fitch2"></a>Fitch WM, Bruschi M (1987) The evolution of prokaryotic ferredoxins-with a general method correcting for unobserved substitutions in less branched lineages. Mol Biol Evol 4: 381–394. <ul class="find" data-citedArticleID="5827355"><li><a href="http://www.crossref.org/guestquery/?auth2=Fitch&amp;atitle2=The+evolution+of+prokaryotic+ferredoxins-with+a+general+method+correcting+for+unobserved+substitutions+in+less+branched+lineages&amp;auth=Fitch&amp;atitle=The+evolution+of+prokaryotic+ferredoxins-with+a+general+method+correcting+for+unobserved+substitutions+in+less+branched+lineages" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitch%5Bauthor%5D+AND+The+evolution+of+prokaryotic+ferredoxins-with+a+general+method+correcting+for+unobserved+substitutions+in+less+branched+lineages" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFitch+%22The+evolution+of+prokaryotic+ferredoxins-with+a+general+method+correcting+for+unobserved+substitutions+in+less+branched+lineages%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0046445-Venditti1" id="pone.0046445-Venditti1"></a>Venditti C, Meade A, Pagel M (2006) Detecting the node-sensity artifact in phylogeny reconstruction. Syst Biol 55: 637–643. <ul class="find" data-citedArticleID="5827357"><li><a href="http://www.crossref.org/guestquery/?auth2=Venditti&amp;atitle2=Detecting+the+node-sensity+artifact+in+phylogeny+reconstruction&amp;auth=Venditti&amp;atitle=Detecting+the+node-sensity+artifact+in+phylogeny+reconstruction" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Venditti%5Bauthor%5D+AND+Detecting+the+node-sensity+artifact+in+phylogeny+reconstruction" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AVenditti+%22Detecting+the+node-sensity+artifact+in+phylogeny+reconstruction%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0046445-Ji1" id="pone.0046445-Ji1"></a>Ji Q, Luo ZX, Yuan CX, Wible JR, Zhang JP, et al. (2002) The earliest known eutherian mammal. Nature 816–822.
        doi: 10.1038/416816a <ul class="find" data-citedArticleID="5827359" data-doi="10.1038/416816a"><li><a href="http://dx.doi.org/10.1038/416816a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ji%5Bauthor%5D+AND+The+earliest+known+eutherian+mammal" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AJi+%22The+earliest+known+eutherian+mammal%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0046445-KielanJaworowska1" id="pone.0046445-KielanJaworowska1"></a>Kielan-Jaworowska Z, Cifelli RL, Luo ZX (2004) Mammals from the age of dinosaurs: origin, evolution, and structure. New York: Columbia University Press. <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0046445-Marivaux1" id="pone.0046445-Marivaux1"></a>Marivaux L, Vianey-Liaud M, Jaeger J (2004) High-level phylogeny of early Tertiary rodents: dental evidence. Zool J Linnean Soc 142: 105–134.
        doi: 10.1111/j.1096-3642.2004.00131.x <ul class="find" data-citedArticleID="5827363" data-doi="10.1111/j.1096-3642.2004.00131.x"><li><a href="http://dx.doi.org/10.1111/j.1096-3642.2004.00131.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Marivaux%5Bauthor%5D+AND+High-level+phylogeny+of+early+Tertiary+rodents%3A+dental+evidence" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMarivaux+%22High-level+phylogeny+of+early+Tertiary+rodents%3A+dental+evidence%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0046445-Huchon3" id="pone.0046445-Huchon3"></a>Huchon D, Madsen O, Sibbald MJJB, Ament K, Stanhope MJ, et al. (2002) Rodent phylogeny and a timescale for the evolution of Glires: Evidence from an extensive taxon sampling using three nuclear genes. Mol Biol Evol 19: 1053–1065.
        doi: 10.1093/oxfordjournals.molbev.a004164 <ul class="find" data-citedArticleID="5827365" data-doi="10.1093/oxfordjournals.molbev.a004164"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004164" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Huchon%5Bauthor%5D+AND+Rodent+phylogeny+and+a+timescale+for+the+evolution+of+Glires%3A+Evidence+from+an+extensive+taxon+sampling+using+three+nuclear+genes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHuchon+%22Rodent+phylogeny+and+a+timescale+for+the+evolution+of+Glires%3A+Evidence+from+an+extensive+taxon+sampling+using+three+nuclear+genes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0046445-BlangaKanfi1" id="pone.0046445-BlangaKanfi1"></a>Blanga-Kanfi S, Miranda H, Penn O, Pupko T, DeBry RW, et al. (2009) Rodent phylogeny revised: analysis of six nuclear genes from all major rodent clades. BMC Evol Biol 9: 71.
        doi: 10.1186/1471-2148-9-71 <ul class="find" data-citedArticleID="5827367" data-doi="10.1186/1471-2148-9-71"><li><a href="http://dx.doi.org/10.1186/1471-2148-9-71" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Blanga-Kanfi%5Bauthor%5D+AND+Rodent+phylogeny+revised%3A+analysis+of+six+nuclear+genes+from+all+major+rodent+clades" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABlanga-Kanfi+%22Rodent+phylogeny+revised%3A+analysis+of+six+nuclear+genes+from+all+major+rodent+clades%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0046445-Wible1" id="pone.0046445-Wible1"></a>Wible JR, Rougier GW, Novacek MJ, Asher RJ (2007) Cretaceous eutherians and Laurasian origin for placental mammals near the K/T boundary. Nature 447: 1003–1006.
        doi: 10.1038/nature05854 <ul class="find" data-citedArticleID="5827369" data-doi="10.1038/nature05854"><li><a href="http://dx.doi.org/10.1038/nature05854" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wible%5Bauthor%5D+AND+Cretaceous+eutherians+and+Laurasian+origin+for+placental+mammals+near+the+K%2FT+boundary" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWible+%22Cretaceous+eutherians+and+Laurasian+origin+for+placental+mammals+near+the+K%2FT+boundary%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0046445-Cartwright1" id="pone.0046445-Cartwright1"></a>Cartwright P, Collins A (2007) Fossil and phylogenies: integrating multiple lines of evidence to investigate the origin of early major metazoan lineages. Integr Comp Biol 47: 744–751.
        doi: 10.1093/icb/icm071 <ul class="find" data-citedArticleID="5827371" data-doi="10.1093/icb/icm071"><li><a href="http://dx.doi.org/10.1093/icb/icm071" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cartwright%5Bauthor%5D+AND+Fossil+and+phylogenies%3A+integrating+multiple+lines+of+evidence+to+investigate+the+origin+of+early+major+metazoan+lineages" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACartwright+%22Fossil+and+phylogenies%3A+integrating+multiple+lines+of+evidence+to+investigate+the+origin+of+early+major+metazoan+lineages%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0046445-Maddison1" id="pone.0046445-Maddison1"></a>Maddison DR, Maddison WP (2003) MacClade 4: analysis of phylogeny and character evolution, version 4.06. Sunderland, MA: Sinauer Associates. <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0046445-Posada1" id="pone.0046445-Posada1"></a>Posada D, Crandal K (1998) MODELTEST: testing the model of DNA substitution. Bioinformatics 14: 817.
        doi: 10.1093/bioinformatics/14.9.817 <ul class="find" data-citedArticleID="5827375" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Posada%5Bauthor%5D+AND+MODELTEST%3A+testing+the+model+of+DNA+substitution" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APosada+%22MODELTEST%3A+testing+the+model+of+DNA+substitution%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0046445-Ronquist2" id="pone.0046445-Ronquist2"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics 19: 1572.
        doi: 10.1093/bioinformatics/btg180 <ul class="find" data-citedArticleID="5827377" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ronquist%5Bauthor%5D+AND+MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARonquist+%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0046445-Drummond2" id="pone.0046445-Drummond2"></a>Drummond AJ, Ho SYW, Phillips MJ, Rambaut A (2006) Relaxed phylogenetics and dating with confidence. PLos Biology 4: e88.
        doi: 10.1371/journal.pbio.0040088 <ul class="find" data-citedArticleID="5827379" data-doi="10.1371/journal.pbio.0040088"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drummond%5Bauthor%5D+AND+Relaxed+phylogenetics+and+dating+with+confidence" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADrummond+%22Relaxed+phylogenetics+and+dating+with+confidence%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0046445-Sun1" id="pone.0046445-Sun1"></a>Sun B, Yue L-P, Wang Y-Q, Meng J, Wang J-Q, et al. (2009) Magnetostratigraphy of the Early Paleogene in the Erlian Basin. J Stratigraphy 33: 62–68. <ul class="find" data-citedArticleID="5827381"><li><a href="http://www.crossref.org/guestquery/?auth2=Sun&amp;atitle2=Magnetostratigraphy+of+the+Early+Paleogene+in+the+Erlian+Basin&amp;auth=Sun&amp;atitle=Magnetostratigraphy+of+the+Early+Paleogene+in+the+Erlian+Basin" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sun%5Bauthor%5D+AND+Magnetostratigraphy+of+the+Early+Paleogene+in+the+Erlian+Basin" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASun+%22Magnetostratigraphy+of+the+Early+Paleogene+in+the+Erlian+Basin%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0046445-Sun2" id="pone.0046445-Sun2"></a>Sun J-M, Ye J, Wu W-Y, Ni X-J, Bi S-D, et al. (2010) Late Oligocene-Miocene mid-latitude aridification and wind patterns in Asian interior. Geology 38: 515–518.
        doi: 10.1130/g30776.1 <ul class="find" data-citedArticleID="5827383" data-doi="10.1130/g30776.1"><li><a href="http://dx.doi.org/10.1130/g30776.1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sun%5Bauthor%5D+AND+Late+Oligocene-Miocene+mid-latitude+aridification+and+wind+patterns+in+Asian+interior" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASun+%22Late+Oligocene-Miocene+mid-latitude+aridification+and+wind+patterns+in+Asian+interior%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0046445-Meng2" id="pone.0046445-Meng2"></a>Meng J, Ye J, Wu W-Y, Ni X-J, Bi S-D (2008) The Neogene Dingshanyanchi Formation in northern Junggar basin of Xinjiang and its stratigraphic implications. Vertebrata PalAsiat 46: 90–110. <ul class="find" data-citedArticleID="5827385"><li><a href="http://www.crossref.org/guestquery/?auth2=Meng&amp;atitle2=The+Neogene+Dingshanyanchi+Formation+in+northern+Junggar+basin+of+Xinjiang+and+its+stratigraphic+implications&amp;auth=Meng&amp;atitle=The+Neogene+Dingshanyanchi+Formation+in+northern+Junggar+basin+of+Xinjiang+and+its+stratigraphic+implications" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Meng%5Bauthor%5D+AND+The+Neogene+Dingshanyanchi+Formation+in+northern+Junggar+basin+of+Xinjiang+and+its+stratigraphic+implications" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMeng+%22The+Neogene+Dingshanyanchi+Formation+in+northern+Junggar+basin+of+Xinjiang+and+its+stratigraphic+implications%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0046445-Qiu1" id="pone.0046445-Qiu1"></a>Qiu Z-D (1996) Middle Miocene micromammalian fauna from the Tunggur, Nei Mongol. Beijing: Science Press. <ul class="find-nolinks"></ul></li><li><span class="label">55.
              </span><a name="pone.0046445-Wu1" id="pone.0046445-Wu1"></a>Wu W-Y, Meng J, Ye J, N iX-J, Bi S-D, et al. (2009) The Miocene mammals from Dingshanyanchi Formation of northern Junggar Basin, Xinjiang. Vertebrata PalAsiat 47: 208. <ul class="find" data-citedArticleID="5827389"><li><a href="http://www.crossref.org/guestquery/?auth2=Wu&amp;atitle2=The+Miocene+mammals+from+Dingshanyanchi+Formation+of+northern+Junggar+Basin%2C+Xinjiang&amp;auth=Wu&amp;atitle=The+Miocene+mammals+from+Dingshanyanchi+Formation+of+northern+Junggar+Basin%2C+Xinjiang" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wu%5Bauthor%5D+AND+The+Miocene+mammals+from+Dingshanyanchi+Formation+of+northern+Junggar+Basin%2C+Xinjiang" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWu+%22The+Miocene+mammals+from+Dingshanyanchi+Formation+of+northern+Junggar+Basin%2C+Xinjiang%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0046445-Zazhigin1" id="pone.0046445-Zazhigin1"></a>Zazhigin VS, Lopatin AV (2000) The history of the Dipodoidea (Rodentia, Mammalia) in the Miocene of Asia: 3. Allactaginae. Paleont J 34: 553–565. <ul class="find" data-citedArticleID="5827391"><li><a href="http://www.crossref.org/guestquery/?auth2=Zazhigin&amp;atitle2=The+history+of+the+Dipodoidea+%28Rodentia%2C+Mammalia%29+in+the+Miocene+of+Asia%3A+3.+Allactaginae&amp;auth=Zazhigin&amp;atitle=The+history+of+the+Dipodoidea+%28Rodentia%2C+Mammalia%29+in+the+Miocene+of+Asia%3A+3.+Allactaginae" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zazhigin%5Bauthor%5D+AND+The+history+of+the+Dipodoidea+%28Rodentia%2C+Mammalia%29+in+the+Miocene+of+Asia%3A+3.+Allactaginae" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AZazhigin+%22The+history+of+the+Dipodoidea+%28Rodentia%2C+Mammalia%29+in+the+Miocene+of+Asia%3A+3.+Allactaginae%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.XML" value="149589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.PDF" value="747082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g001.TIF" value="949350"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g001.PNG_I" value="43681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g001.PNG_S" value="13306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g001.PNG_M" value="99477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g001.PNG_L" value="615475"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g002.TIF" value="1456414"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g002.PNG_I" value="65948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g002.PNG_S" value="17832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g002.PNG_M" value="135527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g002.PNG_L" value="957897"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.t001.TIF" value="649168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.t001.PNG_I" value="31852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.t001.PNG_S" value="4320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.t001.PNG_M" value="75839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.t001.PNG_L" value="163681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.t002.TIF" value="720908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.t002.PNG_I" value="43411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.t002.PNG_S" value="4793"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.t002.PNG_M" value="106270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.t002.PNG_L" value="214839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g003.TIF" value="625822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g003.PNG_I" value="39379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g003.PNG_S" value="14307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g003.PNG_M" value="59222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.g003.PNG_L" value="238067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s001.TIF" value="1170078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s002.TIF" value="1868756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s003.TIF" value="295982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s004.TIF" value="378234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s005.TIF" value="327046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s006.PDF" value="44936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s007.PDF" value="95454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s008.PDF" value="46570"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s009.PDF" value="54787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s010.PDF" value="97981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0046445.s011.PDF" value="71470"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0046445&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0046445" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0046445&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0046445&volume=&issue=&title=Molecular and Paleontological Evidence for a Post-Cretaceous Origin of Rodents&author_name=Shaoyuan%20Wu%2C%20Wenyu%20Wu%2C%20Fuchun%20Zhang%2C%20Jie%20Ye%2C%20Xijun%20Ni%2C%20Jimin%20Sun%2C%20Scott%20V.%20Edwards%2C%20Jin%20Meng%2C%20Chris%20L.%20Organ&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0046445" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0046445" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0046445" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0046445&amp;t=Molecular%20and%20Paleontological%20Evidence%20for%20a%20Post-Cretaceous%20Origin%20of%20Rodents" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0046445&title=Molecular%20and%20Paleontological%20Evidence%20for%20a%20Post-Cretaceous%20Origin%20of%20Rodents&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0046445&amp;title=Molecular%20and%20Paleontological%20Evidence%20for%20a%20Post-Cretaceous%20Origin%20of%20Rodents" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0046445" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0046445&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Molecular and Paleontological Evidence for a Post-Cretaceous Origin of Rodents';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0046445';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Molecular%20and%20Paleontological%20Evidence%20for%20a%20Post-Cretaceous%20Origin%20of%20Rodents http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0046445" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0046445" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bayes+theorem%22" title="Search for articles in the subject area:'Bayes theorem'"><div class="flagText">Bayes theorem</div></a>
              <div data-categoryid="32251" data-articleid="123781"
                   data-categoryname="Bayes theorem"
                   class="flagImage" title="Flag 'Bayes theorem' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cretaceous+period%22" title="Search for articles in the subject area:'Cretaceous period'"><div class="flagText">Cretaceous period</div></a>
              <div data-categoryid="27195" data-articleid="123781"
                   data-categoryname="Cretaceous period"
                   class="flagImage" title="Flag 'Cretaceous period' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fossil+calibration%22" title="Search for articles in the subject area:'Fossil calibration'"><div class="flagText">Fossil calibration</div></a>
              <div data-categoryid="33891" data-articleid="123781"
                   data-categoryname="Fossil calibration"
                   class="flagImage" title="Flag 'Fossil calibration' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fossil+record%22" title="Search for articles in the subject area:'Fossil record'"><div class="flagText">Fossil record</div></a>
              <div data-categoryid="20963" data-articleid="123781"
                   data-categoryname="Fossil record"
                   class="flagImage" title="Flag 'Fossil record' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleogenetics%22" title="Search for articles in the subject area:'Paleogenetics'"><div class="flagText">Paleogenetics</div></a>
              <div data-categoryid="33815" data-articleid="123781"
                   data-categoryname="Paleogenetics"
                   class="flagImage" title="Flag 'Paleogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleozoology%22" title="Search for articles in the subject area:'Paleozoology'"><div class="flagText">Paleozoology</div></a>
              <div data-categoryid="17645" data-articleid="123781"
                   data-categoryname="Paleozoology"
                   class="flagImage" title="Flag 'Paleozoology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="123781"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Rodents%22" title="Search for articles in the subject area:'Rodents'"><div class="flagText">Rodents</div></a>
              <div data-categoryid="42763" data-articleid="123781"
                   data-categoryname="Rodents"
                   class="flagImage" title="Flag 'Rodents' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3328'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5950'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4470&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
