

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Live and Let Die - The Bsister MADS-Box Gene OsMADS29 Controls the Degeneration of Cells in Maternal Tissues during Seed Development of Rice (Oryza sativa)</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0051435"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0051435" />

    <meta name="citation_title" content="Live and Let Die - The Bsister MADS-Box Gene OsMADS29 Controls the Degeneration of Cells in Maternal Tissues during Seed Development of Rice (Oryza sativa)"/>
    <meta itemprop="name" content="Live and Let Die - The Bsister MADS-Box Gene OsMADS29 Controls the Degeneration of Cells in Maternal Tissues during Seed Development of Rice (Oryza sativa)"/>

      <meta name="citation_author" content="Xuelian Yang"/>
            <meta name="citation_author_institution" content="Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China"/>
            <meta name="citation_author_institution" content="Graduate School, Chinese Academy of Sciences, Beijing, People’s Republic of China"/>
      <meta name="citation_author" content="Feng Wu"/>
            <meta name="citation_author_institution" content="Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China"/>
      <meta name="citation_author" content="Xuelei Lin"/>
            <meta name="citation_author_institution" content="Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China"/>
            <meta name="citation_author_institution" content="Graduate School, Chinese Academy of Sciences, Beijing, People’s Republic of China"/>
      <meta name="citation_author" content="Xiaoqiu Du"/>
            <meta name="citation_author_institution" content="Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China"/>
      <meta name="citation_author" content="Kang Chong"/>
            <meta name="citation_author_institution" content="Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China"/>
      <meta name="citation_author" content="Lydia Gramzow"/>
            <meta name="citation_author_institution" content="Department of Genetics, Friedrich Schiller University Jena, Jena, Germany"/>
      <meta name="citation_author" content="Susanne Schilling"/>
            <meta name="citation_author_institution" content="Department of Genetics, Friedrich Schiller University Jena, Jena, Germany"/>
      <meta name="citation_author" content="Annette Becker"/>
            <meta name="citation_author_institution" content="Plant Evo-Devo Group, The Institute of Botany, Justus-Liebig-University Gießen, Gießen, Germany"/>
      <meta name="citation_author" content="Günter Theißen"/>
            <meta name="citation_author_institution" content="Department of Genetics, Friedrich Schiller University Jena, Jena, Germany"/>
      <meta name="citation_author" content="Zheng Meng"/>
            <meta name="citation_author_institution" content="Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China"/>

    <meta name="citation_date" content="2012/12/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0051435.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e51435"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="7"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The ABCs of floral homeotic genes; citation_volume=78; citation_number=1; citation_pages=203-209; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The war of the whorls: genetic interactions controlling flower development; citation_volume=353; citation_number=2; citation_pages=31-37; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Analysis of the C-terminal region of Arabidopsis thaliana APETALA1 as a transcription activation domain; citation_volume=40; citation_number=3; citation_pages=419-429; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=MADS domain proteins in plant development; citation_volume=378; citation_number=4; citation_pages=1079-1118; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Specific interactions between the K domains of AG and AGLs, members of the MADS domain family of DNA binding proteins; citation_volume=12; citation_number=5; citation_pages=999-1010; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A novel MADS-box gene subfamily with a sister-group relationship to class B floral homeotic genes; citation_volume=266; citation_number=6; citation_pages=942-950; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Class D and Bsister MADS-box genes are associated with ectopic ovule formation in the pistil-like stamens of alloplasmic wheat (Triticum aestivum L.); citation_volume=71; citation_number=7; citation_pages=1-14; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=A Bsister MADS-box gene involved in ovule and seed development in petunia and Arabidopsis; citation_volume=47; citation_number=8; citation_pages=934-946; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The TRANSPARENT TESTA16 locus encodes the ARABIDOPSIS BSISTER MADS domain protein and is required for proper development and pigmentation of the seed coat; citation_volume=14; citation_number=9; citation_pages=2463-2479; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis B-sister MADS-box protein, GORDITA, represses fruit growth and contributes to integument development; citation_volume=62; citation_number=10; citation_pages=203-214; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=GORDITA (AGL63) is a young paralog of the Arabidopsis thaliana Bsister MADS box gene ABS (TT16) that has undergone neofunctionalization; citation_volume=63; citation_number=11; citation_pages=914-924; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Mutant analysis, protein–protein interactions and subcellular localization of the Arabidopsis Bsister (ABS) protein; citation_volume=274; citation_number=12; citation_pages=103-118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The MADS box genes SEEDSTICK and ARABIDOPSIS Bsister play a maternal role in fertilization and seed development; citation_volume=70; citation_number=13; citation_pages=409-420; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=MADS-box gene family in rice: genome-wide identification, organization and expression profiling during reproductive development and stress; citation_volume=8; citation_number=14; citation_pages=242; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Systematic reverse genetic screening of T-DNA tagged genes in rice for functional genomic analyses: MADS-box genes as a test case; citation_volume=44; citation_number=15; citation_pages=1403-1411; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The MADS29 transcription factor regulates the degradation of the nucellus and the nucellar projection during rice seed development; citation_volume=24; citation_number=16; citation_pages=1049-1065; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Anthesis, fertilization and development of caryopsis; citation_volume=1; citation_number=17; citation_pages=339-376; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Expression profiling of genes involved in starch synthesis in sink and source organs of rice; citation_volume=56; citation_number=18; citation_pages=3229-3244; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Study on programmed cell death and dynamic changes of starch accumulation in pericarp cells of Triticum aestivum L. Protoplasma; citation_volume=236; citation_number=19; citation_pages=49-58; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=The nucellus degenerates by a process of programmed cell death during the early stages of wheat grain development; citation_volume=213; citation_number=20; citation_pages=352-360; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Programmed cell death during vascular system formation; citation_volume=4; citation_number=21; citation_pages=684-688; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Bcl-2 suppresses hydrogen peroxide-induced programmed cell death via OsVPE2 and OsVPE3, but not via OsVPE1 and OsVPE4, in rice; citation_volume=278; citation_number=22; citation_pages=4797-4810; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Development of maternal seed tissue in barley is mediated by regulated cell expansion and cell disintegration and coordinated with endosperm growth; citation_volume=62; citation_number=23; citation_pages=1217-1227; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Vacuolar processing enzyme: an executor of plant cell death; citation_volume=8; citation_number=24; citation_pages=404-408; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Uncovering the role of VDAC in the regulation of cell life and death; citation_volume=40; citation_number=25; citation_pages=183-191; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=VDAC is a conserved element of death pathways in plant and animal systems; citation_volume=1642; citation_number=26; citation_pages=87-96; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The RNase activity of rice probenazole-induced protein1 (PBZ1) plays a key role in cell death in plants; citation_volume=31; citation_number=27; citation_pages=25-31; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning and characterization of a novel jasmonate inducible pathogenesis-related class 10 protein gene, JIOsPR10, from rice (Oryza sativa L.) seedling leaves; citation_volume=286; citation_number=28; citation_pages=973-983; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DNA-binding properties of arabidopsis MADS domain homeotic proteins APETALA1, APETALA3, PISTILLATA and AGAMOUS; citation_volume=24; citation_number=29; citation_pages=3134-3141; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Evolution of the angiosperms: calibrating the family tree; citation_volume=268; citation_number=30; citation_pages=2211-2220; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Dinosaur coprolites and the early evolution of grasses and grazers; citation_volume=310; citation_number=31; citation_pages=1177-1180; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolutionary dynamics of grass genomes; citation_volume=154; citation_number=32; citation_pages=15-28; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolutionary history of the grasses; citation_volume=125; citation_number=33; citation_pages=1198-1205; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MADS-Box gene diversity in seed plants 300 Million Years Ago; citation_volume=17; citation_number=34; citation_pages=1425-1434; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The D-lineage MADS-box gene OsMADS13 controls ovule identity in rice; citation_volume=52; citation_number=35; citation_pages=690-699; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rice expression atlas in reproductive development; citation_volume=51; citation_number=36; citation_pages=2060-2081; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Structural and histochemical studies on grain-filling in the caryopsis of rice (Oryza sativa L.); citation_volume=28; citation_number=37; citation_pages=455-469; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Sink anatomy in relation to solute movement in rice (Oryza sativa L.): A summary of findings; citation_volume=2; citation_number=38; citation_pages=297-307; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Transport of assimilates in the developing caryopsis of rice (Oryza sativa L.); citation_volume=151; citation_number=39; citation_pages=561-573; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Functional conservation and diversification of class E floral homeotic genes in rice (Oryza sativa); citation_volume=61; citation_number=40; citation_pages=767-781; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=PAUP: A computer program for phylogenetic inference using maximum parsimony; citation_volume=102; citation_number=41; citation_pages=9A; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=ProbCons: Probabilistic consistency-based multiple sequence alignment; citation_volume=15; citation_number=42; citation_pages=330-340; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RevTrans: multiple alignment of coding DNA from aligned amino acid sequences; citation_volume=31; citation_number=43; citation_pages=3537-3539; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution; citation_volume=14; citation_number=44; citation_pages=817-818; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The general stochastic model of nucleotide substitution; citation_volume=142; citation_number=45; citation_pages=485-501; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models; citation_volume=19; citation_number=46; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MOSAIC FLORAL ORGANS1, an AGL6-Like MADS box gene, regulates floral organ identity and meristem fate in rice; citation_volume=21; citation_number=47; citation_pages=3008-3025; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Rice plant development: from zygote to spikelet; citation_volume=46; citation_number=48; citation_pages=23-47; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0051435" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Live and Let Die - The Bsister MADS-Box Gene OsMADS29 Controls the Degeneration of Cells in Maternal Tissues during Seed Development of Rice (Oryza sativa)"/>
    <meta name="twitter:description" content="Bsister genes have been identified as the closest relatives of class B floral homeotic genes. Previous studies have shown that Bsister genes from eudicots are involved in cell differentiation during ovule and seed development. However, the complete function of Bsister genes in eudicots is masked by redundancy with other genes and little is known about the function of Bsister genes in monocots, and about the evolution of Bsister gene functions. Here we characterize OsMADS29, one of three MADS-box Bsister genes in rice. Our analyses show that OsMADS29 is expressed in female reproductive organs including the ovule, ovule vasculature, and the whole seed except for the outer layer cells of the pericarp. Knock-down of OsMADS29 by double-stranded RNA-mediated interference (RNAi) results in shriveled and/or aborted seeds. Histological analyses of the abnormal seeds at 7 days after pollination (DAP) indicate that the symplastic continuity, including the ovular vascular trace and the nucellar projection, which is the nutrient source for the filial tissue at early development stages, is affected. Moreover, degeneration of all the maternal tissues in the transgenic seeds, including the pericarp, ovular vascular trace, integuments, nucellar epidermis and nucellar projection, is blocked as compared to control plants. Our results suggest that OsMADS29 has important functions in seed development of rice by regulating cell degeneration of maternal tissues. Our findings provide important insights into the ancestral function of Bsister genes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0051435.g006"/>

  <meta property="og:title" content="Live and Let Die - The Bsister MADS-Box Gene OsMADS29 Controls the Degeneration of Cells in Maternal Tissues during Seed Development of Rice (Oryza sativa)" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0051435" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=366'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7483'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6907&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0051435">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0051435" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0051435&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Live and Let Die - The B<sub>sister</sub> MADS-Box Gene <i>OsMADS29</i> Controls the Degeneration of Cells in Maternal Tissues during Seed Development of Rice (<i>Oryza sativa</i>)
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xuelian Yang
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Xuelian Yang, Feng Wu</p>

              
              

                <p>Affiliations:
                  Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China, 
                  Graduate School, Chinese Academy of Sciences, Beijing, People’s Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Feng Wu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Xuelian Yang, Feng Wu</p>

              
              

                <p>Affiliation:
                  Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xuelei Lin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China, 
                  Graduate School, Chinese Academy of Sciences, Beijing, People’s Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiaoqiu Du, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kang Chong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lydia Gramzow, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, Friedrich Schiller University Jena, Jena, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Susanne Schilling, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, Friedrich Schiller University Jena, Jena, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Annette Becker, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Plant Evo-Devo Group, The Institute of Botany, Justus-Liebig-University Gießen, Gießen, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Günter Theißen
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:guenter.theissen@uni-jena.de">guenter.theissen@uni-jena.de</a> (GT); <a href="mailto:zhmeng@ibcas.ac.cn">zhmeng@ibcas.ac.cn</a> (ZM)</p>

                <p>Affiliation:
                  Department of Genetics, Friedrich Schiller University Jena, Jena, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zheng Meng
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:guenter.theissen@uni-jena.de">guenter.theissen@uni-jena.de</a> (GT); <a href="mailto:zhmeng@ibcas.ac.cn">zhmeng@ibcas.ac.cn</a> (ZM)</p>

                <p>Affiliation:
                  Key Laboratory of Plant Molecular Physiology, Institute of Botany, Chinese Academy of Sciences, Beijing, People’s Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 12, 2012</li>
    <li>DOI: 10.1371/journal.pone.0051435</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0051435" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0051435" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0051435" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0051435" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0051435-g001" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0051435-g002" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0051435-g003" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0051435-t001" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0051435-g004" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0051435-g005" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0051435-g006" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0051435">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0051435" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>B<sub>sister</sub> genes have been identified as the closest relatives of class B floral homeotic genes. Previous studies have shown that B<sub>sister</sub> genes from eudicots are involved in cell differentiation during ovule and seed development. However, the complete function of B<sub>sister</sub> genes in eudicots is masked by redundancy with other genes and little is known about the function of B<sub>sister</sub> genes in monocots, and about the evolution of B<sub>sister</sub> gene functions. Here we characterize <em>OsMADS29</em>, one of three MADS-box B<sub>sister</sub> genes in rice. Our analyses show that <em>OsMADS29</em> is expressed in female reproductive organs including the ovule, ovule vasculature, and the whole seed except for the outer layer cells of the pericarp. Knock-down of <em>OsMADS29</em> by double-stranded RNA-mediated interference (RNAi) results in shriveled and/or aborted seeds. Histological analyses of the abnormal seeds at 7 days after pollination (DAP) indicate that the symplastic continuity, including the ovular vascular trace and the nucellar projection, which is the nutrient source for the filial tissue at early development stages, is affected. Moreover, degeneration of all the maternal tissues in the transgenic seeds, including the pericarp, ovular vascular trace, integuments, nucellar epidermis and nucellar projection, is blocked as compared to control plants. Our results suggest that <em>OsMADS29</em> has important functions in seed development of rice by regulating cell degeneration of maternal tissues. Our findings provide important insights into the ancestral function of B<sub>sister</sub> genes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Yang X, Wu F, Lin X, Du X, Chong K, et al.  (2012) Live and Let Die - The B<sub>sister</sub> MADS-Box Gene <em>OsMADS29</em> Controls the Degeneration of Cells in Maternal Tissues during Seed Development of Rice (<em>Oryza sativa</em>). PLoS ONE 7(12):
          e51435.
            doi:10.1371/journal.pone.0051435</p><p><strong>Editor: </strong>Joshua L. Heazlewood, 
        Lawrence Berkeley National Laboratory, United States of America
      </p><p><strong>Received:</strong> August 10, 2012; <strong>Accepted:</strong> November 1, 2012; <strong>Published:</strong> December 12, 2012</p><p><strong>Copyright:</strong> © 2012 Yang et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work is supported by the Ministry of Science and Technology of China (grants 2011CB100405; 2011ZX08009-004) and National Nature Science Foundation of China (grants 31100867 and 31121065). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Background"></a><h3>Background</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Genetic and functional analyses of floral homeotic mutants in the model eudicot plants <em>Arabidopsis thaliana</em> and <em>Antirrhinum majus</em> led to the formulation of the ABC model, which was proposed to explain the determination of floral organ identities <a href="#pone.0051435-Weigel1">[1]</a>–<a href="#pone.0051435-Coen1">[2]</a>. According to this model, class A genes specify the identity of sepals, class A and B genes specify petal identity, class B and C genes determine stamen identity, and class C genes determine carpel identity. Most of the floral homeotic genes that play a role in the ABC model are MIKC-type MADS-box genes. These genes were named after the domain structure of the encoded transcription factors consisting of a conserved DNA-binding MADS (M) domain, a less conserved intervening (I) region, a moderately conserved keratin-like (K) domain and a highly variable C-terminal (C) region <a href="#pone.0051435-Cho1">[3]</a>–<a href="#pone.0051435-Fan1">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In the ABC model, class B genes specify petal and stamen identity in combination with class A and C genes, respectively <a href="#pone.0051435-Coen1">[2]</a>. Several years ago, the sister clade of class B genes (<em>DEF</em>/<em>GLO</em>-like genes, also known as <em>AP3</em>/<em>PI</em>-like genes), termed B<sub>sister</sub> genes, was identified in both angiosperms and gymnosperms <a href="#pone.0051435-Becker1">[6]</a>. Protein sequence alignments with other MADS-domain proteins indicated that compared to other MIKC-type proteins, the proteins encoded by B<sub>sister</sub> genes share a shorter I domain, a sub-terminal PI Motif-derived sequence and, in some cases, also a PaleoAP3 Motif in the C-terminal region with the AP3/PI-like proteins of gymnosperms and angiosperms <a href="#pone.0051435-Becker1">[6]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In contrast to class B genes, which are predominantly expressed in male reproductive organs (and in angiosperm petals), B<sub>sister</sub> genes were found to be mainly transcribed in female reproductive organs (ovules, carpel walls) and in developing seeds <a href="#pone.0051435-Yamada1">[7]</a>–<a href="#pone.0051435-Nesi1">[9]</a>. Furthermore, the sequences of B<sub>sister</sub> genes are highly conserved. Together these findings suggest that B<sub>sister</sub> genes play an important role in ovule and/or seed development, which has been conserved for about 300 million years <a href="#pone.0051435-Becker1">[6]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In <em>Arabidopsis</em>, two B<sub>sister</sub> genes, <em>ARABIDOPSIS BSISTER</em> (<em>ABS</em>; also known as <em>TRANSPARENT TESTA 16</em> (<em>TT16</em>) and <em>AGL32</em>) and <em>GORDITA (GOA,</em> formerly known as <em>AGL63</em>), were identified <a href="#pone.0051435-Nesi1">[9]</a>–<a href="#pone.0051435-Kaufmann1">[12]</a>. Expression analyses showed that <em>ABS</em> is expressed in the endothelial layer of the inner integuments of mature ovules <a href="#pone.0051435-Mizzotti1">[13]</a>. In line with this, the <em>abs</em> (<em>tt16</em>) mutant seeds showed a loss of pigmentation and defect inner integument, indicating that <em>ABS</em> (<em>TT16</em>) regulates anthocyanidin accumulation and inner endothelial cell differentiation in <em>Arabidopsis</em> <a href="#pone.0051435-Nesi1">[9]</a>, <a href="#pone.0051435-Kaufmann1">[12]</a>. Recently it was shown that <em>ABS</em> acts redundantly with the class D floral homeotic gene <em>SEEDSTICK</em> (<em>STK</em>); <em>abs</em>/<em>stk</em> double mutants are characterized by a total absence of the endothelium and by massive starch accumulation in the embryo sac resulting in a low number of viable seeds <a href="#pone.0051435-Mizzotti1">[13]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The closest relative of <em>ABS</em> in <em>Arabidopsis</em> is <em>GOA</em>. <em>GOA</em> has a broader expression pattern compared to <em>ABS</em>. It has undergone neofunctionalization and has non-redundant functions to <em>ABS</em> in ovule outer integument development and the regulation of fruit longitudinal growth <a href="#pone.0051435-Prasad1">[10]</a>, <a href="#pone.0051435-Erdmann1">[11]</a>. In <em>Petunia</em>, the <em>FLORAL BINDING PROTEIN 24</em> gene (<em>FBP24)</em> is expressed in ovule primodia, nucelli and integuments. Later, its expression is restricted to the endothelium of mature ovules and seeds. An <em>fbp24</em> knock-out mutant did not show alterations in development, indicating that <em>FBP24</em> acts redundantly with other genes. <em>fbp24</em> knock-down lines in which so far unknown other genes are likely co-suppressed, are affected in endothelium layer development similar to <em>abs</em> mutants, which suggests that the <em>FBP24</em> function is similar to that of <em>ABS</em> in <em>Arabidopsis</em> <a href="#pone.0051435-deFolter1">[8]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>So far, functional studies have been conducted for B<sub>sister</sub> genes from core eudicots only. However, expression patterns have been determined also for B<sub>sister</sub> genes from a number of other species, as summarized in <a href="#pone-0051435-g001">Figure 1</a>. The B<sub>sister</sub> gene <em>GGM13</em> of the gymnosperm <em>Gnetum gnemon</em> is expressed in the developing nucellus and inner envelope of female reproductive units <a href="#pone.0051435-Becker1">[6]</a>. Expression analyses of B<sub>sister</sub> genes in monocots were carried out in several species. The wheat B<sub>sister</sub> gene, <em>WBsis</em>, is expressed in the endothelial layer of the inner integument of the ovule <a href="#pone.0051435-Yamada1">[7]</a>, resembling the expression pattern of <em>ABS</em> in <em>Arabidopsis</em> <a href="#pone.0051435-Mizzotti1">[13]</a>. However, weak expression of <em>WBsis</em> was detected also in the nucellus and in the outer integument. The maize B<sub>sister</sub> gene <em>ZMM17</em> is initially expressed broadly in all organ primordia of the female spikelet, but at later developmental stages expression is restricted to the ovule and the developing silk <a href="#pone.0051435-Becker1">[6]</a>. Three B<sub>sister</sub> genes, <em>OsMADS29</em>, <em>OsMADS30,</em> and <em>OsMADS31,</em> were identified in the rice genome <a href="#pone.0051435-Arora1">[14]</a>. Both RT-PCR and microarray analyses showed that transcription of <em>OsMADS29</em> is restricted to developing seeds, <em>OsMADS30</em> is expressed throughout all organs of the rice plant, and that expression of <em>OsMADS31</em> is below detection limit of the methods being used <a href="#pone.0051435-Arora1">[14]</a>, <a href="#pone.0051435-Lee1">[15]</a>.</p>
<div class="figure" id="pone-0051435-g001"><div class="img"><a name="pone-0051435-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Phylogeny and expression patterns of B<sub>sister</sub> genes.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Bayesian phylogeny of class D, class B and B<sub>sister</sub> genes in seed plants. Posterior probabilities are indicated on the nodes. The expression patterns of B<sub>sister</sub> and class D genes are depicted as far as they are known. Colors indicate expression intensity: white, no expression; light grey, weak expression; dark grey, strong expression; asterisk, expression was detected particularly in endothelial cells of inner integument; n, no expression data available; cross, tissue or organ is not present in respective species.</p>
<span>doi:10.1371/journal.pone.0051435.g001</span></div><a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Since within angiosperms the monocot rice is only distantly related to the core eudicots <em>Arabidopsis</em> and <em>Petunia</em>, functional conservation and diversification of B<sub>sister</sub> genes between monocots and eudicots can be investigated employing these species and along with this provide clues about the ancestral B<sub>sister</sub> function. Here we investigate <em>OsMADS29</em>, because among the three B<sub>sister</sub> genes of rice it is the only one exclusively and highly expressed during ovule and seed development resembling the expression pattern of B<sub>sister</sub> genes of other species. We performed detailed expression analyses of <em>OsMADS29</em> and explored its functions in rice by a reverse genetics approach employing RNAi. Our results show that <em>OsMADS29</em> is mainly expressed in the developing ovule and seed. Knock-down of <em>OsMADS29</em> leads to aborted and/or shriveled seeds with deficient accumulation of starch in the endosperm. The severe phenotype of the <em>OsMADS29</em> knock-down lines suggests that, it plays an important role during rice seed development and controls at least some aspects non-redundantly with other genes, which is in contrast to typical findings about B<sub>sister</sub> genes in eudicots. Our results thus corroborate many of the findings recently reported by Yin and Xue (2012) <a href="#pone.0051435-Yin1">[16]</a>. While these authors, however, focused on transgenic lines with a quite moderate reduction in <em>OsMADS29</em> mRNA levels, we report here in more detail the phenotypes of lines with stronger knock-down effects. Moreover, we provide novel insights into the evolution of the B<sub>sister</sub> gene function and expression in flowering plants and reveal the crucial role of B<sub>sister</sub> genes in the seed development.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Phylogeny of B<sub>sister</sub> Genes from Monocots</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In order to gain insight into the evolutionary relationships between <em>OsMADS29</em> and other B<sub>sister</sub> genes, a comprehensive search for B<sub>sister</sub> genes of monocots was carried out in GenBank and Phytozome (<a href="http://www.phytozome.net">www.phytozome.net</a>) (<a href="#pone.0051435.s001">Figure S1</a>). A phylogenetic tree was then constructed with the identified genes and representative B<sub>sister</sub> genes from core eudicots (<a href="#pone-0051435-g001">Figure 1</a>). The topology of the phylogenetic tree shows that B<sub>sister</sub> genes of core eudicots and monocots constitute two different clades, suggesting that the most recent common ancestor of both taxa contained one B<sub>sister</sub> gene only. The clade of grass B<sub>sister</sub> genes is divided into three subclades, termed the <em>OsMADS29</em>, <em>OsMADS30</em>, and <em>OsMADS31</em> subclade. These clades comprise rice B<sub>sister</sub> genes together with putative orthologs from other grass species such as barely (<em>Hordeum vulgare</em>), wheat (<em>Triticum aestivum</em>), sorghum (<em>Sorghum bicolor</em>), maize (<em>Zea mays</em>), <em>Brachypodium distachyon</em>, and <em>Setaria italica</em> (<a href="#pone-0051435-g001">Figure 1</a>), indicating that the gene duplication events that generated the three clades occurred before grass diversification.</p>


<h4><em>OsMADS29</em> is Specifically Expressed in Ovules and Developing Seeds</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To analyze the expression patterns of <em>OsMADS29</em> in detail, RT-PCR, quantitative reverse transcription PCR (qRT-PCR), and RNA <em>in situ</em> hybridization analyses were performed using diverse tissues at different developmental stages.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>RT-PCR analyses show that the expression of <em>OsMADS29</em> initiates at panicles 0.1~5 cm, then increases gradually with the development of panicles at 6–22 cm. After pollination, the expression level reaches the maximum value at 5 DAP-7 DAP, then decreases from 9 DAP. No expression of <em>OsMADS29</em> is detected in the vegetative organs (<a href="#pone-0051435-g002">Figure 2A</a>). Results of qRT-PCR show similar expression profiles of <em>OsMADS29</em> to those of the RT-PCR (<a href="#pone-0051435-g002">Figure 2B</a>).</p>
<div class="figure" id="pone-0051435-g002"><div class="img"><a name="pone-0051435-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Expression analyses of <em>OsMADS29</em>.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p><strong>(A).</strong> RT-PCR analyses of <em>OsMADS29</em> expression at different developmental stages of wild type plants. DAP, days after pollination. <em>ACTIN1</em> was used as control. <strong>(B).</strong> Relative expression levels of <em>OsMADS29</em> at different developmental stages. Amounts of transcripts are shown as relative values to those of <em>UBQ</em>. Error bars show SD (n = 3). <strong>(C).</strong> RT-PCR analyses of <em>OsMADS29</em> expression in various floral organs of wild type plants at heading date stage. <em>APT1</em> was used as a control <a href="#pone.0051435-Ohmori1">[47]</a>. <strong>(D–L).</strong> <em>In situ</em> hybridization analyses of <em>OsMADS29</em> in ovules and developing seeds. <strong>(D).</strong> Longitudinal section of the ovule. <strong>(E–G).</strong> Longitudinal sections of seed at 1 DAP (E), and embryo of seed at 3 DAP (F) and 5 DAP (G). <strong>(H–I).</strong> Transverse sections of the mid-region of the ovary at a few hours before anthesis (H) and seed at 3 DAP (I). The arrows head in (H) and (I) indicate the inner layer cells of the pericarp. <strong>(J).</strong> Negative control with sense probes of <em>OsMADS29</em>. <strong>(K–L).</strong> A magnification of the transverse section of a seed at 7 DAP. (K). The ovular vascular trace in the seed at 7 DAP. (L). Part of pericarp of seed at 7 DAP. CC, cross-cells; EM, embryo; EN, endosperm; IN, integuments; II, inner layer of the inner integument; NE, nucellar epidermis; NP, nucellar projection; NU, nucellus; OV, ovular vascular trace; TC, tube-cells. Bars = 100 µm in (D) to (J), and 50 µm in (K) and (L).</p>
<span>doi:10.1371/journal.pone.0051435.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>RT-PCR was also performed to examine the expression of <em>OsMADS29</em> in different floral organs. Like other known B<sub>sister</sub> genes, <em>OsMADS29</em> is specifically expressed in the pistils (<a href="#pone-0051435-g002">Figure 2C</a>).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p><em>In situ</em> hybridization was performed to locate the <em>OsMADS29</em> expression at the cellular level (<a href="#pone-0051435-g002">Figure 2D–L</a>). Transcripts of <em>OsMADS29</em> are localized in the ovule, including integuments (IN) and nucellus (NU) throughout ovule development, from the primordial stage to maturity (<a href="#pone-0051435-g002">Figure 2D and H</a>), as well as in the ovule vasculature. After pollination, <em>OsMADS29</em> is expressed evenly throughout the whole seed (<a href="#pone-0051435-g002">Figure 2I and K</a>) and the embryo (EM) (<a href="#pone-0051435-g002">Figure 2E, F and G</a>) except for some cell layers in the outer epidermis of the pericarp. Notably, the gene is expressed in some tissue types which degenerate at later stages of seed development. These tissues include the inner epidermis of the pericarp (<a href="#pone-0051435-g002">Figure 2H and I</a>, indicated by an arrow), the tube-cells (TC) and cross-cells (CC) which differentiate from the inner epidermis at 7 DAP (<a href="#pone-0051435-g002">Figure 2L</a>), the inner and outer integument II (<a href="#pone-0051435-g002">Figure 2D and H</a>), the ovular vascular trace (OV) (<a href="#pone-0051435-g002">Figure 2D, E, H, I and K</a>), the nucellar projection (NP) (<a href="#pone-0051435-g002">Figure 2I, K</a>), and the nucellar epidermis (NE) (<a href="#pone-0051435-g002">Figure 2I, K and L</a>).</p>


<h4>Silencing of <em>OsMADS29</em> Results in Seed Abortion and Deficiency in Starch Accumulation in the Endosperm</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To explore the functions of <em>OsMADS29</em>, we generated transgenic plants in which <em>OsMADS29</em> expression is specifically silenced at different levels by RNAi (<a href="#pone-0051435-g003">Figure 3J and K</a>). Statistical evaluations of data from mature grains shows that there is a usual abortion rate of about 9% in the control plants (<a href="#pone-0051435-g003">Figure 3L</a>). In contrast, of fifteen independent transgenic lines, five lines with strong <em>OsMADS29</em> down regulation produced strong phenotypes with 100% abnormal seeds, including, on average, 66% shriveled seeds (mildly affected seeds with deficient starch accumulation in the endosperm, defined as m-seeds) (<a href="#pone-0051435-g003">Figure 3D, E, F, L</a> and <a href="#pone-0051435-g004">Figure 4B</a>) and 34% aborted seeds (severely affected seeds which are fully aborted before 7 DAP, s-seeds) (<a href="#pone-0051435-g003">Figure 3G, H, I, L</a> and <a href="#pone-0051435-g004">Figure 4C</a>). Seven lines with moderate <em>OsMADS29</em> down regulation exhibited phenotypes with 63% abnormal seeds, including, on average, 25% m-seeds and 38% s-seeds, and 37% normal grains (n-seeds) (<a href="#pone-0051435-g003">Figure 3L</a>). In the three lines with weak down regulation, most of the grains are n-seeds except for, on average, 17% s-seeds (<a href="#pone-0051435-g003">Figure 3L</a>).</p>
<div class="figure" id="pone-0051435-g003"><div class="img"><a name="pone-0051435-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Phenotypes of <em>OsMADS29</em> RNAi lines and molecular detections.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p><strong>(A–I).</strong> Profiles of seeds at 7 DAP, 10 DAP and mature stages, respectively. (A–C), n-seeds of control plants. (D–F), m-seeds with deficient starch accumulation in the endosperm. (G–I), s-seeds that are aborted before 7 DAP. Bars = 1 mm. <strong>(J).</strong> Relative expression levels of <em>OsMADS29</em> in control plants and three typical <em>OsMADS29</em> RNAi lines (L5, L8 and L9). Total RNA was pooled from stage Ov10 <a href="#pone.0051435-Itoh1">[48]</a>. The <em>ACTIN1</em> was used as the internal control. Error bars show SD (n = 3). <strong>(K).</strong> RT-PCR analyses of <em>OsMADS29</em> and the other two related B<sub>sister</sub> genes of rice, <em>OsMADS30</em> and <em>OsMADS31</em>, in control plants and RNAi plants. Total RNA was pooled from stage Ov10. The <em>ACTIN1</em> was used as the internal control. <strong>(L).</strong> Percentage of different kinds of seeds in one panicle in control plants and different <em>OsMADS29</em> RNAi lines. The error bar indicates the SE (n = 3).</p>
<span>doi:10.1371/journal.pone.0051435.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Considering that rice seeds reach their maximum length at about 5–6 DAP <a href="#pone.0051435-Hoshikawa1">[17]</a>, 7 DAP were chosen as first time point for investigation of seeds. At 7 DAP, the s-seeds are aborted already (<a href="#pone-0051435-g003">Figure 3G</a>), while the m-seeds are still similar to the control seeds (<a href="#pone-0051435-g003">Figure 3A</a>) except for being slightly smaller in size (<a href="#pone-0051435-g003">Figure 3D</a>). At 10 DAP, unlike n-seeds that show an increasing width, s-seeds did not show any further development (<a href="#pone-0051435-g003">Figure 3H</a>), while the apical part of the m-seeds reveals initial signs of shrinking possibly caused by a failure to accumulate sufficient levels of starch (<a href="#pone-0051435-g003">Figure 3E</a>). One month later, compared with the fully developed n-seeds, the s-seeds did not show much further development compared to 7 DAP (<a href="#pone-0051435-g003">Figure 3I</a>), and the mature m-seeds appear shriveled with wrinkles on their pericarps (<a href="#pone-0051435-g003">Figure 3F</a>).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The agronomic traits of seeds of control and <em>OsMADS29</em> RNAi transgenic lines were analyzed except for the s-seeds that had been aborted at 7 DAP (<a href="#pone-0051435-t001">Table 1</a>). In line 5 (L5), in which the <em>OsMADS29</em> expression was severely down-regulated, the length of m-seeds is 4.84 mm on average, while control seeds are about 16.7% longer (5.65 mm). The average width of control seeds was 3.15 mm, about 18.9% wider than that of the m-seeds (2.65 mm). The average thickness of m-seeds (1.00 mm) was reduced to about 50% of that of the control (1.98 mm) (P-value &lt;0.01). These differences in length, width, and thickness are probably caused by a deficiency in starch filling. As expected, the 1000-grain-weight is also extremely reduced in m-seeds (2.15 g), with the weight being only about 10% of that of the control seeds (23.48 g) (P-value &lt;0.01).</p>
<div class="figure" id="pone-0051435-t001"><div class="img"><a name="pone-0051435-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0051435.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0051435.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Agronomic traits of seeds in control plants and <em>OsMADS29</em> RNAi transgenic lines.</span></strong></p><span>doi:10.1371/journal.pone.0051435.t001</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>To correlate the phenotypes with <em>OsMADS29</em> transcript abundance, qRT-PCR experiments were performed. As shown in <a href="#pone-0051435-g003">Figure 3J</a>, the transcript level of <em>OsMADS29</em> was highly reduced to 26.4% of that of the wild-type in L5 exhibiting a strong phenotype, significantly reduced to 35.3% in L8 with a moderate phenotype, and slightly reduced to 71.7% in L9 showing only a weak phenotype. In contrast, the expression levels of the other two B<sub>sister</sub> genes in rice, <em>OsMADS30</em> and <em>OsMADS31</em> remained unchanged in the <em>OsMADS29</em> knock-down lines (<a href="#pone-0051435-g003">Figure 3K</a>). These results indicate that the observed phenotypes of the transgenic plants are caused by specific silencing of <em>OsMADS29.</em></p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Given that the starch content is substantially decreased in mature seeds of transgenic plants (<a href="#pone-0051435-g004">Figure 4B and C</a>), the expression levels of four genes, including <em>OsAGPS1</em>, <em>OsAGPL2</em>, <em>OsAGPL3</em>, and <em>OsAGPS2a</em> which encode the rate-limiting-step enzymes in the starch synthesis pathway <a href="#pone.0051435-Ohdan1">[18]</a>, were examined in developing seeds of control plants and <em>OsMADS29</em> RNAi lines at 10 DAP. qRT-PCR results showed that the expression levels of those four genes in transgenic seeds are similar to the ones of the control seeds (<a href="#pone.0051435.s002">Figure S2</a>), indicating that starch synthesis itself is not affected.</p>
<div class="figure" id="pone-0051435-g004"><div class="img"><a name="pone-0051435-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Histological analyses of control and transgenic seeds.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p><strong>(A).</strong> Starch granules were seen in control seeds at 7 DAP. <strong>(B, C).</strong> m- and s-seeds at 7 DAP without starch granules. <strong>(D–F).</strong> A higher magnification of the OV at 7 DAP in the control seeds (D), m-seeds (E) and s-seeds (F), respectively. The double arrow between the two transverse lines indicates the cell layers of the OV. <strong>(G, H).</strong> Semi-thin transverse sections from the mid-region of mature control seeds (G) and m-seeds (H) at 30 DAP. The red rectangle in (G) and (H) indicates the position of ovular vascular trace. The arrowhead in (H) indicates the remained pericarp. <strong>(I, J).</strong> Higher magnification for part of the pericarp in mature control seeds (I) and m-seeds (J) at 30 DAP. It shows that the pericarp cuticular layer in (I) and persisted pericarp cells in (J). The red dots indicate the cuticular seed coat in control seeds (I) and the cells of integuments and nucellar epidermis in m-seeds are not degenerated (J). <strong>(K, L).</strong> Higher magnification of the OV in control seeds (K) and m-seeds (L), which showed in the red rectangle in (G) and (H). The red circle in (L) indicates the large OV in m-seeds. AL, aleurone; NP, nucellar projecton; OV, ovular vascular trace; P, pericarp; PH, phloem; PS, the pigment strand; XY, xylem; EN, endosperm. Bars = 200 µm in (A–C), 50 µm in (D–F), 500 µm in (G) and (H), 100µm in (I) and (J), and 50 µm in (K) and (L).</p>
<span>doi:10.1371/journal.pone.0051435.g004</span></div>

<h4>Silencing of <em>OsMADS29</em> Affects Vascular Trace Development and the Degeneration of Tissues in Rice Grains</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Down regulation of <em>OsMADS29</em> does not affect ovule development but causes seeds to abort or shrivel as they are most likely deficient in endosperm development and starch accumulation (<a href="#pone.0051435.s003">Figure S3F</a>, <a href="#pone-0051435-g003">Figure 3A–I</a>, <a href="#pone-0051435-g004">Figure 4B and C</a>). To observe the internal organization of developing seeds, mid-region transverse sections of seeds at 7 DAP were made. While the seeds of control plants are entirely filled with endosperm cells that produce starch granules (<a href="#pone-0051435-g004">Figure 4A</a>), no endosperm cells can be observed in the m- and s-seeds of the transgenic lines at 7 DAP (<a href="#pone-0051435-g004">Figure 4B and C</a>). In control plants, the xylem cells are large, have a circular shape and are centered in the middle of the ovular vascular trace (OV), and the phloem cells are small and arranged in a cluster at the periphery of the OV (<a href="#pone-0051435-g004">Figure 4D</a>). In the s-seeds, the xylem cells are smaller in size and fewer in number than that of the control seeds. Also a lower number of phloem cells can be seen at the margin of the OV (<a href="#pone-0051435-g004">Figure 4F</a>). In m-seeds, the size and number of xylem cells are similar to those of the s-seeds, whereas arrangement style of phloem cells is scattered in contrast to the phloem cell clusters of the control plants (<a href="#pone-0051435-g004">Figure 4E</a>). Moreover, the number of cell layers of the OV in the transverse orientation is higher in the <em>OsMADS29</em> RNAi transgenic lines than in the control lines (marked by double arrows brackets in <a href="#pone-0051435-g004">Figure 4D–F</a>), suggesting that adjacent cells in the OV probably fail to integrate into the xylem and phloem. Another tissue type recognized as abnormal in the transgenic lines is the nucellar projection (NP), which is a small zone of persisting nucellar cells attached to the chalaza. The NP cells degenerate in the control seeds at 7 DAP such that of originally 4–5 cell layers only 2–3 layers remain. The respective cells are small and flat, and constitute a tissue required for nutrient transfer to the endosperm and embryo (<a href="#pone-0051435-g004">Figure 4D</a>; <a href="#pone.0051435.s004">Figure S4</a>). In contrast, the NP cells of m- and s-seeds at 7 DAP remain large and circular, without any obvious indication of cell degeneration (<a href="#pone-0051435-g004">Figure 4E and F</a>).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In mature grains, the structure of the embryo was not affected in the m-seeds, except for the fact that the embryos were considerably smaller than in control seeds (<a href="#pone-0051435-g005">Figure 5E and H</a>). Notably in the n-seeds at 30 DAP, the maternal tissues (pericarp, ovular vascular trace, integuments, and nucellar epidermis) were almost degenerated to form the cuticula (<a href="#pone-0051435-g004">Figure 4G, I and K</a>) along with the maturity of filial tissue (endosperm and embryo). In contrast, all the maternal tissues remained in the mature seeds in the <em>OsMADS29</em> RNAi transgenic lines at this stage (<a href="#pone-0051435-g004">Figure 4H, J and L</a>).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Longitudinal histological sections of seeds at 7 DAP showed that development of the embryo in transgenic seeds severely lagged behind that of the wild-type. Compared to the completed embryo morphology in control seeds at 7 DAP (<a href="#pone-0051435-g005">Figure 5A</a>), embryos of s-seeds at 7 DAP appear to cease development at a stage corresponding to 3 DAP of wild-type seeds, and remain in this stage (<a href="#pone-0051435-g005">Figure 5B</a> and <a href="#pone.0051435.s003">Figure S3C</a>); embryos of m-seeds at 7 DAP grew retarded to a stage corresponding to almost 5 DAP of control seeds (<a href="#pone-0051435-g005">Figure 5C</a> and <a href="#pone.0051435.s003">Figure S3D</a>). In mature grains, the embryo of m-seeds is considerably smaller than the one in control seeds (<a href="#pone-0051435-g005">Figure 5E and H</a>). Considering that the mature embryo of m-seeds failed to germinate even on the 1/2 MS culture medium (<a href="#pone-0051435-g005">Figure 5G</a>), mid-region transverse sections of embryos were carried out to thoroughly observe the structure of the vascular system, which also developed abnormally in m- and s-seeds at 7 DAP. While vascular bundles are obvious between the shoot apical meristem and radicle of control embryos, no vascular bundles were observed in the embryos of m-seeds (<a href="#pone-0051435-g005">Figure 5F and I</a>).</p>
<div class="figure" id="pone-0051435-g005"><div class="img"><a name="pone-0051435-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Histological structure of embryo and germination of mature seeds.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p><strong>(A).</strong> Longitudinal sections of control embryo at 7DAP. <strong>(B, C)</strong> Longitudinal sections of embryo of s-seeds (B) and m-seeds (C) at 7 DAP, respectively. <strong>(D).</strong> The normally germinated seed of control plant. <strong>(E).</strong> Longitudinal sections of the mature embryo in control seed. The red line referred to the position of vascular bundle. <strong>(F).</strong> Magnifying observation of cross sections from the red line marked parts in (E). The arrow showed the clear vascular bundle. <strong>(G).</strong> The m-seeds failed to germinate. <strong>(H).</strong> Longitudinal sections of the mature embryo in m-seeds. <strong>(I).</strong> Magnifying observation of cross sections from the red line marked parts in (H). No obvious vascular bundles were observed at the arrow referred place. VB, vascular bundles. Bars = 20 µm in (A–C), 5 mm in (D) and (G), 0.2 mm in (E) and (H), 20 µm in (F) and (I).</p>
<span>doi:10.1371/journal.pone.0051435.g005</span></div><a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Previous studies have shown that the differentiation of the tracheary elements as well as the degeneration of nucellus and pericarp cells are also brought about by PCD <a href="#pone.0051435-Zhou1">[19]</a>–<a href="#pone.0051435-Fukuda1">[21]</a>. To analyze the relationship between <em>OsMADS29</em> and cell degeneration caused by PCD during seed development, an Evan’s blue staining experiment, which can dye the dead cells to be blue, was performed. Results showed that cell death of the NP and endosperm tissues in the m-seeds obviously occurs later than that of control seeds during the stage from 15 DAP to 26 DAP (<a href="#pone-0051435-g006">Figure 6A–H</a>). In addition, qRT-PCR expression analyses were carried out with three classes of genes known to positively regulate PCD through three different pathways, including <em>VPE1-4</em> <a href="#pone.0051435-Deng1">[22]</a>–<a href="#pone.0051435-Ikuko1">[24]</a>, <em>VDAC1-3</em> <a href="#pone.0051435-Varda1">[25]</a>, <a href="#pone.0051435-Godbole1">[26]</a>, and <em>PBZ1</em> <a href="#pone.0051435-Kim1">[27]</a>, <a href="#pone.0051435-Jwa1">[28]</a>. Our results show that <em>OsVPE1</em> (Os04g45470) is the only gene which is significantly down-regulated in <em>OsMADS29</em> knock-down transgenic seeds (<a href="#pone-0051435-g006">Figure 6I</a>). A putative CArG-box with the consensus sequence C(A/T)<sub>8</sub>G was found at position −1827 in the upstream region of <em>OsVPE1</em> (<a href="#pone.0051435.s005">Figure S5</a>). As MADS-domain proteins have been shown to bind to CArG-boxes of their target genes to regulate expression <a href="#pone.0051435-Riechmann2">[29]</a>, this may indicate direct regulation of <em>OsVPE1</em> by <em>OsMADS29</em>.</p>
<div class="figure" id="pone-0051435-g006"><div class="img"><a name="pone-0051435-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0051435" data-uri="info:doi/10.1371/journal.pone.0051435.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0051435.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0051435.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0051435.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0051435.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Detections of tissue degeneration.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p><strong>(A–D).</strong> Evan’s Blue staining of control seeds. (A-C) Transverse sections at 15 DAP, 20 DAP, and 26 DAP, respectively. (D) The longitudinal section at 26 DAP. The blue tissues indicate dead endosperm cells. <strong>(E–H).</strong> Evan’s Blue staining of m-seeds. (E–G) Transverse sections at 15 DAP, 20 DAP, and 26 DAP respectively. H Longitudinal sections at 26 DAP. Bar = 500 µm in (A) to (H). EM, embryo. <strong>(I).</strong> Quantitative real-time PCR analyses of PCD-related genes in 7 DAP seeds of control and transgenic plants. <em>ACTIN1</em> was used as an internal control. Error bar indicated the SD (n = 3). ** indicates an extremely significant difference (P&lt;0.01) between control and transgenic lines.</p>
<span>doi:10.1371/journal.pone.0051435.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4><em>OsMADS29</em> is a Canonical B<sub>sister</sub> Gene in Rice</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>As revealed by phylogenetic analyses previously, B<sub>sister</sub> genes are the sister clade of the <em>DEF</em>/<em>GLO</em>-like (or <em>AP3</em>/<em>PI</em>-like) genes comprising class B floral organ identity genes <a href="#pone.0051435-Becker1">[6]</a>. Our phylogeny suggest that the two gene duplications that gave rise to the three B<sub>sister</sub> genes <em>OsMADS29</em>, <em>OsMADS30</em> and <em>OsMADS31</em> in rice occurred after the divergence of the lineages that led to extant eudicots and monocots about 150 MYA <a href="#pone.0051435-Wikstrm1">[30]</a>, but predated the diversification of grasses from a common ancestor about 55–70 MYA <a href="#pone.0051435-Prasad2">[31]</a>–<a href="#pone.0051435-Kellogg1">[33]</a>. In our phylogeny, the branch lengths are relatively short for all genes belonging to the <em>OsMADS29</em> clade as compared to the branch lengths for the genes belonging to the other two monocot B<sub>sister</sub> clades (<a href="#pone-0051435-g001">Figure 1</a>). This suggests that purifying selection is acting on the genes in the <em>OsMADS29</em> clade. Furthermore, the expression pattern of the genes in the <em>OsMADS29</em> subclade seems to be conserved, as the expression of <em>OsMADS29</em> in the developing ovule including integuments and nucellus is similar to that of the previously described expression patterns of <em>WBsis</em> and <em>ZMM17</em> (Figure1) <a href="#pone.0051435-Becker1">[6]</a>, <a href="#pone.0051435-Yamada1">[7]</a>. Taken together, this provides evidence for the functional importance of the genes in the <em>OsMADS29</em> clade. Protein sequence alignments of the C-terminal domains show that OsMADS29 and OsMADS31 are typical B<sub>sister</sub> proteins with respect to their protein sequence as they both share a conserved sub-terminal ‘PI Motif-Derived sequence’ and a terminal ‘Paleo AP3 Motif’ with other B<sub>sister</sub> proteins. In contrast, OsMADS30 does not include these motifs characteristic for B<sub>sister</sub> proteins (<a href="#pone.0051435.s006">Figure S6</a>). Expression analyses reveal that <em>OsMADS31b</em> (the alternative splice isoform of <em>OsMADS31</em> reported by Lee <em>et al</em>. (2003) <a href="#pone.0051435-Lee1">[15]</a>) is broadly expressed during all developmental stages of rice, whereas <em>OsMADS30</em> expression is hardly detectable at all (<a href="#pone.0051435.s007">Figure S7</a>), which are unusual expression patterns for B<sub>sister</sub> genes. Only <em>OsMADS29</em>, which is specifically expressed in mature florets, especially in pistils and in developing seeds, exhibits an expression pattern typical for B<sub>sister</sub> genes (<a href="#pone-0051435-g002">Figure 2A–C</a>). After pollination, <em>OsMADS29</em> is highly expressed in developing seeds (<a href="#pone-0051435-g002">Figure 2D–L</a>). Expression during seed development has been reported for <em>ABS</em>, <em>GOA</em> and <em>FBP24</em> as well <a href="#pone.0051435-deFolter1">[8]</a>–<a href="#pone.0051435-Prasad1">[10]</a>. These results suggest functional conservation of B<sub>sister</sub> genes between eudicots and monocots in ovule and seed development. Furthermore, our study shows that <em>OsMADS29</em> is probably the canonical representative of these functions in rice.</p>


<h4>Conserved Functions of B<sub>sister</sub> Genes in Ovule and Seed Development</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Given that all B<sub>sister</sub> genes investigated so far, including <em>OsMADS29</em>, are expressed during early ovule development, it was hypothesized that members of the B<sub>sister</sub> subfamily play important roles in female reproductive organ (ovule) development <a href="#pone.0051435-Becker2">[34]</a>. In our study, down-regulation of <em>OsMADS29</em> does not affect the structure of the ovule (<a href="#pone.0051435.s003">Figure S3A and F</a>). Accordingly, either <em>OsMADS29</em> is not required for ovule specification in rice or its function in ovule development is obscured by redundancy with another gene, e.g. with <em>OsMADS21</em>, which is a class D gene. The expression of <em>OsMADS21</em> overlaps with <em>OsMADS29</em> and single mutants of <em>osmads21</em> do not have a mutant phenotype <a href="#pone.0051435-Dreni1">[35]</a>. Redundancy between B<sub>sister</sub> and D class genes would resemble recent findings for <em>ABS/TT16</em> and <em>STK</em> in <em>Arabidopsis</em> <a href="#pone.0051435-Mizzotti1">[13]</a>. Redundancy with the other two B<sub>sister</sub> genes <em>OsMADS30</em> and <em>OsMADS31</em> may also obscure the spectrum of functional significance of <em>OsMADS29</em> during ovule development. Recently, some microarray data about ovule development can be used from the public database, which can give us some new clues for the ovule development of <em>OsMADS29</em> in our future work <a href="#pone.0051435-Fujita1">[36]</a>.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>In contrast to ovule development, the effect on seed development is stronger for <em>OsMADS29</em> than for the <em>Arabidopsis</em> B<sub>sister</sub> genes. While the phenotypes of the single mutants <em>abs</em> (<em>tt16</em>) and <em>goa</em> of <em>Arabidopsis</em> are mild, given that they still produce seeds which germinate properly <a href="#pone.0051435-Nesi1">[9]</a>, <a href="#pone.0051435-Erdmann1">[11]</a>, <a href="#pone.0051435-Kaufmann1">[12]</a>, silencing of <em>OsMADS29</em> in rice leads to a severe phenotype with sterile seeds which are aborted or shriveled. The mild phenotype of the <em>abs</em> single mutant may again be explained by redundancy of <em>ABS</em> (<em>TT16</em>) and <em>STK</em> in <em>Arabidopsis</em> <a href="#pone.0051435-Mizzotti1">[13]</a>. In rice, this redundancy between B<sub>sister</sub> and class D genes in seed development may have been lost or not gained in evolution, leading to a severe phenotype in <em>OsMADS29</em> knock down lines. Compared to control seeds which are full of starch granules, cross-sections of abnormal seeds at 7 DAP show no visible starch granules. Interestingly, starch accumulation is also deregulated in the <em>abs</em>/<em>stk</em> double mutant. However, in this case, an excess of starch was observed in the embryo sac and the developing seed <a href="#pone.0051435-Mizzotti1">[13]</a>. Although some shriveled seeds of the <em>OsMADS29</em> knock down lines have embryos with normal shape, they have no vascular bundles and fail to germinate, possibly because of the lack of nutrition transport (<a href="#pone-0051435-g005">Figure 5D–I</a>).</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The mutant phenotypes of B<sub>sister</sub> genes from eudicots and rice differ during ovule and seed development. This may indicate that the function of <em>OsMADS29</em> has diverged from that of B<sub>sister</sub> genes in eudicots during the evolution of monocots. However, the differences in the mutant phenotypes can also, at least partially, be explained by redundancy to other genes. Hence, it is also possible that the function of <em>OsMADS29</em> resembles the ancestral function of B<sub>sister</sub> genes while this ancestral function is masked in eudicots due to other genes with redundant functions.</p>


<h4><em>OsMADS29</em> is Required for Vascular Trace Development and Cell Degeneration in Rice Seeds</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Previous studies have shown that there is a symplastic continuity, including the ovular vascular trace (OV), pigment strand (PS) and nucellar projection (NP), which is the nutrient source for the filial tissues (endosperm and embryo), between the rice maternal tissues and the filial tissues <a href="#pone.0051435-Krishnan1">[37]</a>–<a href="#pone.0051435-Oparka2">[39]</a>. Therefore the path among the OV, PS and NP plays an important role in grain filling and seed development. In control plants, the nucellus was absorbed by the filial tissues during early seed development (<a href="#pone-0051435-g004">Figure 4D</a>). In contrast, the structures of the nucellus, including NP and NE are still visible in the transgenic grains at 7 DAP and in mature seeds (<a href="#pone-0051435-g004">Figure 4E, F, H, J and L</a>). This indicates that the process of degeneration of their cells is partially suppressed. Similar results concerning the defect structure of NE and NP have recently been reported by Yin and Xue (2012) when this manuscript was in preparation <a href="#pone.0051435-Yin1">[16]</a>. However, we also observed defects in other tissues, such as the OV, which may lead towards a more comprehensive understanding of the function of <em>OsMADS29.</em> In the transgenic abnormal seeds at 7 DAP, the shapes of the OV are irregular and the cell numbers of xylem and phloem are less than those in the control plants (<a href="#pone-0051435-g004">Figure 4E, F</a>), indicating that transport paths for nutrients are strongly affected. Xylem and phloem are differentiated from the cambium cells, revealing that their formation is a process involving PCD <a href="#pone.0051435-Fukuda1">[21]</a>. Hence, fewer cells in the xylem and phloem of transgenic abnormal seeds at 7 DAP imply that cell degeneration of the tracheary tissue (including xylem and phloem) is suppressed to a great extent. This further indicates that the defect tracheary system may be another important cause, besides the failed degradation of NE and NP, for the deficient starch accumulation, as no tracheary system is observed in the m-seeds at mature stage. Therefore, the loss of nutrient transport may be the possible direct reason for the small embryo and the germination failure. This result also shows that <em>OsMADS29</em> affects not only the cell degeneration of maternal tissues but also that of filial tissues. Furthermore, the pericarp cells in the <em>OsMADS29</em> transgenic m-seeds remain in their original state without any sign of degeneration (<a href="#pone-0051435-g003">Figure 3B and E</a>). However, the pericarp of the wild type mature seed is degenerated in a PCD dependent process to become the cuticula which protects the filial tissue (<a href="#pone-0051435-g004">Figure 4I</a>) <a href="#pone.0051435-Zhou1">[19]</a>. Collectively, these results show that the degeneration of cells is localized in those tissues of seeds where gene expression of <em>OsMADS29</em> is found.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>As expected, cell degeneration was indeed shown to be suppressed during seed development of transgenic plants. Evan's blue staining indicates that cell death of NP and endosperm tissue in transgenic seeds is obviously delayed as compared to that of control seeds (<a href="#pone-0051435-g006">Figure 6A–H</a>), suggesting that the process of PCD is delayed and suppressed to some extent. Since molecular markers for cell death in seeds are not available at present, the expression of three kinds of genes, <em>VPEs</em>, <em>VADC</em>, and <em>PBZ1</em>, known to be involved in PCD through different pathways, was investigated. Previous studies showed that the <em>VPEs</em> are involved in vacuole mediated cell death in both defense and development, regulating PCD in <em>Arabidopsis</em>, rice, and barley <a href="#pone.0051435-Deng1">[22]</a>–<a href="#pone.0051435-Ikuko1">[24]</a>. Only four <em>VPE</em> genes were found in rice, among which <em>OsVPE1</em> and <em>OsVPE4</em> are seed-type genes, whereas <em>OsVPE2</em> and <em>OsVPE3</em> are vegetative-type genes <a href="#pone.0051435-Deng1">[22]</a>. In this study, only the transcript amount of <em>OsVPE1</em> was found to be substantially reduced in abnormal transgenic seeds (<a href="#pone-0051435-g006">Figure 6I</a>). In contrast, the expression levels of the <em>VDAC</em> genes, which are elements of the mitochondrial death machinery <a href="#pone.0051435-Varda1">[25]</a>, <a href="#pone.0051435-Godbole1">[26]</a>, and <em>OsPBZ1</em>, which is involved in PCD induced by abiotic stresses <a href="#pone.0051435-Kim1">[27]</a>, <a href="#pone.0051435-Jwa1">[28]</a>, were the same in abnormal transgenic as in control seeds (<a href="#pone-0051435-g006">Figure 6I</a>). A Cys protease involved in regulating PCD of maternal tissues has been identified as a direct target gene of <em>OsMADS29</em> by Yin and Xue (2012), but the pathway of PCD regulated by <em>OsMADS29</em> has remained unknown <a href="#pone.0051435-Yin1">[16]</a>. Our molecular analyses together with the histological structures of the abnormal seeds in our transgenic plants suggest that PCD of the tracheary system and pericarp in rice caryopses regulated by <em>OsMADS29</em> occurs through a pathway mediated by vacuolar processing. The expression levels of genes encoding the rate-limiting-step enzymes of the starch synthesis pathway, <em>OsAGPS</em>1, <em>OsAGPS2a</em>, <em>OsAGPL2</em>, and <em>OsAGPL3</em>, are not affected (<a href="#pone.0051435.s002">Figure S2</a>). Taken together, our results reveal that the failure of the degeneration process in maternal tissues and thus insufficient development of xylem and phloem leading to an undersupply of filial tissue, rather than the starch synthesis process itself, is the original cause for the abnormal development of transgenic seeds. Interestingly, as mentioned above, also the eudicot B<sub>sister</sub> genes <em>ABS</em> and <em>FBP24</em> are functioning in the maternal tissue that supplies the filial tissue with nutrients <a href="#pone.0051435-deFolter1">[8]</a>, <a href="#pone.0051435-Nesi1">[9]</a>, <a href="#pone.0051435-Kaufmann1">[12]</a>, <a href="#pone.0051435-Mizzotti1">[13]</a>. Hence, developmental control of the tissue responsible for nutrient supply to the embryo may be one of the ancestral functions of B<sub>sister</sub> genes.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Our molecular analyses and structural observations provide evidence for the function of <em>OsMADS29</em> in cell degeneration during seed development and may enable a better understanding of the process of PCD during seed development. The severe phenotype of <em>OsMADS29</em> knockdown lines indicates that this B<sub>sister</sub> gene at least in part functions non-redundantly to other genes during seed development which is in contrast to the B<sub>sister</sub> genes of eudicots which have been studied so far. Furthermore, the expression pattern of <em>OsMADS29</em> is similar to that of the gymnosperm B<sub>sister</sub> gene <em>GGM13</em> and resembles, more or less, a combination of the expression patterns known from eudicot B<sub>sister</sub> genes. Hence, <em>OsMADS29</em> may have largely retained the ancestral expression pattern and its functions as revealed here may represent the ancestral function of B<sub>sister</sub> genes. On the other hand, different functions and regulatory mechanisms of eudicot B<sub>sister</sub> genes in cell expansion, pigment accumulation or integument specification suggest that B<sub>sister</sub> genes may have undergone sub- and neo-functionalization in various species during the evolution of flowering plants to control different aspects of seed development.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Plant Materials</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The rice cultivar ‘Zhonghua 11′ (<em>Oryza sativa</em> L. ssp. <em>japonica</em>) was planted in local paddy-fields of the Institute of Botany, the Chinese Academy of Sciences.</p>


<h4>Knockdown Vector Construction of <em>OsMADS29</em> and Rice Transformation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The <em>OsMADS29</em> full length cDNA clone (Accession NO. AK109522) was obtained from the Japan NIAS KOME stock center (<a href="http://cdna01.dna.affrc.go.jp/cDNA/">http://cdna01.dna.affrc.go.jp/cDNA/</a>). To generate the <em>OsMADS29</em> knock-down vector, a 488 bp long specific region (nucleotide positions 474 - 962 counted from ATG) was amplified, using the PCR primer pairs 29RNAiF (<em>Bam</em> HI and <em>Spe</em> I) and 29RNAiR (<em>Nco</em> I and <em>Hpa</em> I). The intron and nos terminator cassette of pJawohl3-RNAi (GenBank Accession No. AF404854) was transferred with <em>Bam</em> HI/<em>Not</em> I site to the pBluescript SK (Stratagene, La Jolla, CA, USA), termed pBJWl3 as an intermediate vector. The RNAi-fragment of <em>OsMADS29</em> was cloned into two sides of the intron region of the pBJWl3 vector in sense (<em>Bam</em> HI/<em>Nco</em> I) and antisense (<em>Spe</em> I/<em>Hpa</em> I) orientation, respectively. Finally, the dsRNAi cassette containing two oppositely orientated coding sequences and the nos terminator was mobilized with restriction enzymes <em>Bam</em> HI and <em>Sac</em> I, and was introduced into the vector pCAMBIA1301-Ubi, in which the maize (<em>Zea mays</em>) <em>Ubiquitin</em> promoter was inserted using the <em>Hind</em> III and <em>Bam</em> HI sites, resulting in the final plasmid pUOsM29I. The construct was verified by sequencing and restriction mapping. Rice transformation was performed according to the previous methods <a href="#pone.0051435-Cui1">[40]</a>.</p>


<h4><em>In situ</em> Hybridization</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Fresh wild-type flowers and seeds (1–7 DAP) were fixed immediately and embedded in paraffin (Sigma). 8 µm-thick sections were hybridized with the <em>OsMADS29</em> specific sequences (nucleotides 695–962 counted from the start codon ATG) according to the method described by Cui <em>et al</em>., (2010) <a href="#pone.0051435-Cui1">[40]</a>.</p>


<h4>RT-PCR and Quantitative Real-time RT-PCR Analyses</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Total RNA was isolated from different rice tissues using TRIzol reagent (Invitrogen, Carlsbad, USA) according to the manufacturer’s instructions. Reverse transcription was performed using Superscript-III Reverse Transcriptase (Invitrogen, Carlsbad, USA). The diluted cDNA samples were used as templates for RT-PCR and real-time PCR. The internal control genes for RT-PCR were <em>ACTIN1</em> and <em>APT1</em> (<em>Adenine Phosphoribosyltransferase1</em>). Real-time PCR were performed using SYBR Premix Ex Taq (Takara, Dalian, China) on a Rotor-Gene 3000 (Corbett Research, QIAGEN, Hilden, Germany) detection system and software according to the manufacturer’s instructions. The <em>ACTIN1</em> and <em>UBQ</em> were used as an internal control. Gene-specific primers are shown in <a href="#pone.0051435.s008">Table S1</a>.</p>


<h4>Histochemical Analyses</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Samples were embedded in paraffin (Sigma-Aldrich, St. Louis, USA) for histological sections (8-µm thickness) and in LR white resin (Sigma-Aldrich, St. Louis, USA) for semi-thin slices (1-µm thickness), respectively. Slices were stained with 0.1% toluidine blue and observed with a light microscope.</p>


<h4>Evans Blue Staining</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Fresh caryopses at different days after pollination were cut into longitudinal and cross-sections with a sharp double-edged blade. The sections were stained with Evans Blue solution (0.1% in H<sub>2</sub>O) for 2 min and washed with water for 60 min, then photographed using a Leica microscope (Leica DM4500B).</p>


<h4>Phylogenetic Analyses</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>B<sub>sister</sub> genes from monocotyledonous plants were searched using the blastn algorithm (<a href="#pone.0051435.s001">Figure S1</a>). The amino acid sequences of the three B<sub>sister</sub> proteins from <em>Oryza sativa</em>, OsMADS29, OsMADS30 and OsMADS31, were used as query sequences. The databases “Nucleotide collection (nr/nt)” and “non-human, non-mouse ESTs (est_others)” were searched. Furthermore, the whole genome sequences of <em>Setaria italica</em>, <em>Brachypodium distachyon</em>, <em>Sorghum bicolor</em> and <em>Panicum virgatum</em> were searched at phytozome (<a href="http://www.phytozome.net">www.phytozome.net</a>). The BLAST results were combined with known B<sub>sister</sub> genes and representative MIKC-type MADS-box genes of other clades in a neighbor joining tree constructed using PAUP* version 4.0b10 <a href="#pone.0051435-Swofford1">[41]</a> to identify B<sub>sister</sub> genes and to remove redundancies. Amino acid sequences of known B<sub>sister</sub> proteins and the newly identified B<sub>sister</sub> proteins were aligned with ProbCons <a href="#pone.0051435-Do1">[42]</a>. The ProbCons alignment was reverse translated into a nucleotide alignment using RevTrans1.4 <a href="#pone.0051435-Wernersson1">[43]</a>. The best fitting nucleotide substitution model to this alignment was determined using PAUP* version 4.0b10 <a href="#pone.0051435-Swofford1">[41]</a> and Modeltest3.7 <a href="#pone.0051435-Posada1">[44]</a> where positions 1 to 24 and 508 to the end of the alignment were excluded. With the best-fitting model of nucleotide substitution (GTR+I+G) <a href="#pone.0051435-Rodrguez1">[45]</a>, a MrBayes v3.1.2 <a href="#pone.0051435-Ronquist1">[46]</a> phylogeny was determined, excluding the same positions as for the Modeltest, using AP3 of <em>Arabidopsis thaliana</em> as outgroup, generating 3,000,000 trees, sampling every 100th generation and discarding 7,500 of the sampled trees. The final phylogeny is a completely resolved consensus tree with Bayesian posterior probabilities.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0051435"></div><a name="pone.0051435.s001" id="pone.0051435.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0051435.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Accession numbers of genes used in the phylogeny trees.</p>
<p class="siDoi">doi:10.1371/journal.pone.0051435.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0051435.s002" id="pone.0051435.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0051435.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Expression detections for the selected genes related to the starch synthesis. Quantitative real-time PCR analyses of ADP-glucose pyrophosphorylase genes in young seeds (10 DAP) of control and RNAi transgenic plants. <em>ACTIN1</em> was used as an internal control. Error bars indicate the SD (n = 3).</p>
<p class="siDoi">doi:10.1371/journal.pone.0051435.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0051435.s003" id="pone.0051435.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0051435.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Longitudinal sections of ovule and embryo of control and RNAi transgenic plants. (<strong>A</strong>)<strong>.</strong> Longitudinal sections of mature ovule (OV10) in control plant. (<strong>B–E</strong>)<strong>.</strong> Longitudinal sections of embryo of developing seeds in control plants at 1, 3, 5, and 7 DAP, respectively. (<strong>F</strong>)<strong>.</strong> Longitudinal sections of mature ovule (OV10) in RNAi transgenic plant. Bar = 20 µm.</p>
<p class="siDoi">doi:10.1371/journal.pone.0051435.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0051435.s004" id="pone.0051435.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0051435.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Transverse sections in the mid-region of control seeds at 0–7 DAP. (<strong>A–C</strong>)<strong>.</strong> Transverse sections of control seeds at stages 0, 1, 2 DAP. (<strong>D–F</strong>)<strong>.</strong> Magnification of OV and NU at stages 0, 1, 2 DAP, respectively. (<strong>G–H</strong>)<strong>.</strong> Transverse sections of control seeds at stages 3, 5, 7 DAP, respectively. (<strong>J–I</strong>)<strong>.</strong> Magnification of OV and NP at stages 3, 5, 7 DAP, respectively. NP, nucellar projection; NU, nucellus; OV, ovular vascular trace. Bar = 50 µm in (A–C), 25 µm in (D–F), 100 µm in (G–H), 200 µm in I, and 50 µm in (J–L), respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0051435.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0051435.s005" id="pone.0051435.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0051435.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">The promoter of <em>OsVPE1</em> upstream about 3000 bp. A putative CArG-box at position -1827 in the upstream region of <em>OsVPE1.</em></p>
<p class="siDoi">doi:10.1371/journal.pone.0051435.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0051435.s006" id="pone.0051435.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0051435.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">C-terminal sequence alignment of B<sub>sister</sub> Proteins.</p>
<p class="siDoi">doi:10.1371/journal.pone.0051435.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0051435.s007" id="pone.0051435.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0051435.s007">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">RT-PCR analyses of <em>OsMADS30</em> and <em>OsMADS31</em>. (<strong>A</strong>). RT-PCR analyses of <em>OsMADS30</em> and <em>OsMADS31</em> at different development stages. DAP, days after pollination. <em>ACTIN1</em> was used as control. (<strong>B</strong>). RT-PCR analyses of <em>OsMADS30</em> and <em>OsMADS31</em> expression in various floral organs of wild type plants at heading date stage. <em>APT1</em> was used as a control.</p>
<p class="siDoi">doi:10.1371/journal.pone.0051435.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0051435.s008" id="pone.0051435.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0051435.s008">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Gene specific primers used in this study.</p>
<p class="siDoi">doi:10.1371/journal.pone.0051435.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(DOCX)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Dr. Chaoying He for his valuable comments on the manuscript, and Dr. Qijiang Xu for his help in ‘<em>in situ</em> hybridization’.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: XY ZM. Performed the experiments: XY FW LG. Analyzed the data: XY FW. Contributed reagents/materials/analysis tools: XL XD. Wrote the paper: XY FW ZM GT. Gave suggestions for the draft manuscript: KC SS AB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0051435-Weigel1" id="pone.0051435-Weigel1"></a>Weigel D, Meyerowitz EM (1994) The ABCs of floral homeotic genes. Cell 78: 203–209.
        doi: 10.1016/0092-8674(94)90291-7 <ul class="find" data-citedArticleID="6354897" data-doi="10.1016/0092-8674(94)90291-7"><li><a href="http://dx.doi.org/10.1016/0092-8674(94)90291-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Weigel%5Bauthor%5D+AND+The+ABCs+of+floral+homeotic+genes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWeigel+%22The+ABCs+of+floral+homeotic+genes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0051435-Coen1" id="pone.0051435-Coen1"></a>Coen ES, Meyerowitz EM (1991) The war of the whorls: genetic interactions controlling flower development. Nature 353: 31–37.
        doi: 10.1038/353031a0 <ul class="find" data-citedArticleID="6354899" data-doi="10.1038/353031a0"><li><a href="http://dx.doi.org/10.1038/353031a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coen%5Bauthor%5D+AND+The+war+of+the+whorls%3A+genetic+interactions+controlling+flower+development" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACoen+%22The+war+of+the+whorls%3A+genetic+interactions+controlling+flower+development%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0051435-Cho1" id="pone.0051435-Cho1"></a>Cho S, Jang S, Chae S, Chung K, Moon YH, et al. (1999) Analysis of the C-terminal region of <em>Arabidopsis thaliana APETALA1</em> as a transcription activation domain. Plant Mol Biol 40: 419–429. <ul class="find" data-citedArticleID="6354901"><li><a href="http://www.crossref.org/guestquery/?auth2=Cho&amp;atitle2=Analysis+of+the+C-terminal+region+of+Arabidopsis+thaliana+APETALA1+as+a+transcription+activation+domain&amp;auth=Cho&amp;atitle=Analysis+of+the+C-terminal+region+of+Arabidopsis+thaliana+APETALA1+as+a+transcription+activation+domain" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cho%5Bauthor%5D+AND+Analysis+of+the+C-terminal+region+of+Arabidopsis+thaliana+APETALA1+as+a+transcription+activation+domain" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACho+%22Analysis+of+the+C-terminal+region+of+Arabidopsis+thaliana+APETALA1+as+a+transcription+activation+domain%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0051435-Riechmann1" id="pone.0051435-Riechmann1"></a>Riechmann JL, Meyerowitz EM (1997) MADS domain proteins in plant development. Biol Chem 378: 1079–1118. <ul class="find" data-citedArticleID="6354903"><li><a href="http://www.crossref.org/guestquery/?auth2=Riechmann&amp;atitle2=MADS+domain+proteins+in+plant+development&amp;auth=Riechmann&amp;atitle=MADS+domain+proteins+in+plant+development" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Riechmann%5Bauthor%5D+AND+MADS+domain+proteins+in+plant+development" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARiechmann+%22MADS+domain+proteins+in+plant+development%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0051435-Fan1" id="pone.0051435-Fan1"></a>Fan HY, Hu Y, Tudor M, Ma H (1997) Specific interactions between the K domains of AG and AGLs, members of the MADS domain family of DNA binding proteins. Plant J 12: 999–1010.
        doi: 10.1046/j.1365-313x.1997.12050999.x <ul class="find" data-citedArticleID="6354905" data-doi="10.1046/j.1365-313x.1997.12050999.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1997.12050999.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fan%5Bauthor%5D+AND+Specific+interactions+between+the+K+domains+of+AG+and+AGLs%2C+members+of+the+MADS+domain+family+of+DNA+binding+proteins" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFan+%22Specific+interactions+between+the+K+domains+of+AG+and+AGLs%2C+members+of+the+MADS+domain+family+of+DNA+binding+proteins%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0051435-Becker1" id="pone.0051435-Becker1"></a>Becker A, Kaufmann K, Freialdenhoven A, Vincent C, Li MA, et al. (2002) A novel MADS-box gene subfamily with a sister-group relationship to class B floral homeotic genes. Mol Genet Genomics 266: 942–950.
        doi: 10.1007/s00438-001-0615-8 <ul class="find" data-citedArticleID="6354907" data-doi="10.1007/s00438-001-0615-8"><li><a href="http://dx.doi.org/10.1007/s00438-001-0615-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Becker%5Bauthor%5D+AND+A+novel+MADS-box+gene+subfamily+with+a+sister-group+relationship+to+class+B+floral+homeotic+genes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABecker+%22A+novel+MADS-box+gene+subfamily+with+a+sister-group+relationship+to+class+B+floral+homeotic+genes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0051435-Yamada1" id="pone.0051435-Yamada1"></a>Yamada K, Saraike T, Shitsukawa N, Hirabayashi C, Takumi S, et al. (2009) Class D and B<sub>sister</sub> MADS-box genes are associated with ectopic ovule formation in the pistil-like stamens of alloplasmic wheat (<em>Triticum aestivum</em> L.). Plant Mol Biol 71: 1–14.
        doi: 10.1007/s11103-009-9504-z <ul class="find" data-citedArticleID="6354909" data-doi="10.1007/s11103-009-9504-z"><li><a href="http://dx.doi.org/10.1007/s11103-009-9504-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yamada%5Bauthor%5D+AND+Class+D+and+Bsister+MADS-box+genes+are+associated+with+ectopic+ovule+formation+in+the+pistil-like+stamens+of+alloplasmic+wheat+%28Triticum+aestivum+L.%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AYamada+%22Class+D+and+Bsister+MADS-box+genes+are+associated+with+ectopic+ovule+formation+in+the+pistil-like+stamens+of+alloplasmic+wheat+%28Triticum+aestivum+L.%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0051435-deFolter1" id="pone.0051435-deFolter1"></a>de Folter S, Shchennikova AV, Franken J, Busscher M, Baskar R, et al. (2006) A B<sub>sister</sub> MADS-box gene involved in ovule and seed development in petunia and Arabidopsis. Plant J 47: 934–946.
        doi: 10.1111/j.1365-313x.2006.02846.x <ul class="find" data-citedArticleID="6354911" data-doi="10.1111/j.1365-313x.2006.02846.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02846.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=de+Folter%5Bauthor%5D+AND+A+Bsister+MADS-box+gene+involved+in+ovule+and+seed+development+in+petunia+and+Arabidopsis" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3Ade+Folter+%22A+Bsister+MADS-box+gene+involved+in+ovule+and+seed+development+in+petunia+and+Arabidopsis%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0051435-Nesi1" id="pone.0051435-Nesi1"></a>Nesi N, Debeaujon I, Jond C, Stewart AJ, Jenkins GI, et al. (2002) The <em>TRANSPARENT TESTA16</em> locus encodes the ARABIDOPSIS BSISTER MADS domain protein and is required for proper development and pigmentation of the seed coat. Plant Cell 14: 2463–2479.
        doi: 10.1105/tpc.004127 <ul class="find" data-citedArticleID="6354913" data-doi="10.1105/tpc.004127"><li><a href="http://dx.doi.org/10.1105/tpc.004127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nesi%5Bauthor%5D+AND+The+TRANSPARENT+TESTA16+locus+encodes+the+ARABIDOPSIS+BSISTER+MADS+domain+protein+and+is+required+for+proper+development+and+pigmentation+of+the+seed+coat" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ANesi+%22The+TRANSPARENT+TESTA16+locus+encodes+the+ARABIDOPSIS+BSISTER+MADS+domain+protein+and+is+required+for+proper+development+and+pigmentation+of+the+seed+coat%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0051435-Prasad1" id="pone.0051435-Prasad1"></a>Prasad K, Zhang X, Tobón E, Ambrose BA (2010) The Arabidopsis B-sister MADS-box protein, GORDITA, represses fruit growth and contributes to integument development. Plant J 62: 203–214.
        doi: 10.1111/j.1365-313x.2010.04139.x <ul class="find" data-citedArticleID="6354915" data-doi="10.1111/j.1365-313x.2010.04139.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2010.04139.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prasad%5Bauthor%5D+AND+The+Arabidopsis+B-sister+MADS-box+protein%2C+GORDITA%2C+represses+fruit+growth+and+contributes+to+integument+development" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APrasad+%22The+Arabidopsis+B-sister+MADS-box+protein%2C+GORDITA%2C+represses+fruit+growth+and+contributes+to+integument+development%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0051435-Erdmann1" id="pone.0051435-Erdmann1"></a>Erdmann R, Gramzow L, Melzer R, Theißen G, Becker A (2010) <em>GORDITA</em> (<em>AGL63</em>) is a young paralog of the <em>Arabidopsis thaliana</em> B<sub>sister</sub> MADS box gene <em>ABS</em> (<em>TT16</em>) that has undergone neofunctionalization. Plant J 63: 914–924.
        doi: 10.1111/j.1365-313x.2010.04290.x <ul class="find" data-citedArticleID="6354917" data-doi="10.1111/j.1365-313x.2010.04290.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2010.04290.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Erdmann%5Bauthor%5D+AND+GORDITA+%28AGL63%29+is+a+young+paralog+of+the+Arabidopsis+thaliana+Bsister+MADS+box+gene+ABS+%28TT16%29+that+has+undergone+neofunctionalization" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AErdmann+%22GORDITA+%28AGL63%29+is+a+young+paralog+of+the+Arabidopsis+thaliana+Bsister+MADS+box+gene+ABS+%28TT16%29+that+has+undergone+neofunctionalization%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0051435-Kaufmann1" id="pone.0051435-Kaufmann1"></a>Kaufmann K, Anfang N, Saedler H, Theissen G (2005) Mutant analysis, protein–protein interactions and subcellular localization of the Arabidopsis B<sub>sister</sub> (ABS) protein. Mol Genet Genomics 274: 103–118.
        doi: 10.1007/s00438-005-0010-y <ul class="find" data-citedArticleID="6354919" data-doi="10.1007/s00438-005-0010-y"><li><a href="http://dx.doi.org/10.1007/s00438-005-0010-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kaufmann%5Bauthor%5D+AND+Mutant+analysis%2C+protein%E2%80%93protein+interactions+and+subcellular+localization+of+the+Arabidopsis+Bsister+%28ABS%29+protein" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKaufmann+%22Mutant+analysis%2C+protein%E2%80%93protein+interactions+and+subcellular+localization+of+the+Arabidopsis+Bsister+%28ABS%29+protein%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0051435-Mizzotti1" id="pone.0051435-Mizzotti1"></a>Mizzotti C, Mendes MA, Caporali E, Schnittger A, Kater MM, et al. (2012) The MADS box genes <em>SEEDSTICK</em> and <em>ARABIDOPSIS</em> B<sub>sister</sub> play a maternal role in fertilization and seed development. Plant J 70: 409–420.
        doi: 10.1111/j.1365-313x.2011.04878.x <ul class="find" data-citedArticleID="6354921" data-doi="10.1111/j.1365-313x.2011.04878.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2011.04878.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mizzotti%5Bauthor%5D+AND+The+MADS+box+genes+SEEDSTICK+and+ARABIDOPSIS+Bsister+play+a+maternal+role+in+fertilization+and+seed+development" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMizzotti+%22The+MADS+box+genes+SEEDSTICK+and+ARABIDOPSIS+Bsister+play+a+maternal+role+in+fertilization+and+seed+development%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0051435-Arora1" id="pone.0051435-Arora1"></a>Arora R, Agarwal P, Ray S, Singh A, Singh V, et al. (2007) MADS-box gene family in rice: genome-wide identification, organization and expression profiling during reproductive development and stress. BMC Genomics 8: 242.
        doi: 10.1186/1471-2164-8-242 <ul class="find" data-citedArticleID="6354923" data-doi="10.1186/1471-2164-8-242"><li><a href="http://dx.doi.org/10.1186/1471-2164-8-242" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arora%5Bauthor%5D+AND+MADS-box+gene+family+in+rice%3A+genome-wide+identification%2C+organization+and+expression+profiling+during+reproductive+development+and+stress" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AArora+%22MADS-box+gene+family+in+rice%3A+genome-wide+identification%2C+organization+and+expression+profiling+during+reproductive+development+and+stress%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0051435-Lee1" id="pone.0051435-Lee1"></a>Lee S, Kim J, Son JS, Nam J, Jeong DH, et al. (2003) Systematic reverse genetic screening of T-DNA tagged genes in rice for functional genomic analyses: MADS-box genes as a test case. Plant Cell Physiol 44: 1403–1411.
        doi: 10.1093/pcp/pcg156 <ul class="find" data-citedArticleID="6354925" data-doi="10.1093/pcp/pcg156"><li><a href="http://dx.doi.org/10.1093/pcp/pcg156" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lee%5Bauthor%5D+AND+Systematic+reverse+genetic+screening+of+T-DNA+tagged+genes+in+rice+for+functional+genomic+analyses%3A+MADS-box+genes+as+a+test+case" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALee+%22Systematic+reverse+genetic+screening+of+T-DNA+tagged+genes+in+rice+for+functional+genomic+analyses%3A+MADS-box+genes+as+a+test+case%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0051435-Yin1" id="pone.0051435-Yin1"></a>Yin L, Xue H (2012) The <em>MADS29</em> transcription factor regulates the degradation of the nucellus and the nucellar projection during rice seed development. Plant Cell 24: 1049–1065.
        doi: 10.1105/tpc.111.094854 <ul class="find" data-citedArticleID="6354927" data-doi="10.1105/tpc.111.094854"><li><a href="http://dx.doi.org/10.1105/tpc.111.094854" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yin%5Bauthor%5D+AND+The+MADS29+transcription+factor+regulates+the+degradation+of+the+nucellus+and+the+nucellar+projection+during+rice+seed+development" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AYin+%22The+MADS29+transcription+factor+regulates+the+degradation+of+the+nucellus+and+the+nucellar+projection+during+rice+seed+development%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0051435-Hoshikawa1" id="pone.0051435-Hoshikawa1"></a>Hoshikawa K (1993) Anthesis, fertilization and development of caryopsis. Science of the rice plant 1: 339–376. <ul class="find" data-citedArticleID="6354929"><li><a href="http://www.crossref.org/guestquery/?auth2=Hoshikawa&amp;atitle2=Anthesis%2C+fertilization+and+development+of+caryopsis&amp;auth=Hoshikawa&amp;atitle=Anthesis%2C+fertilization+and+development+of+caryopsis" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hoshikawa%5Bauthor%5D+AND+Anthesis%2C+fertilization+and+development+of+caryopsis" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHoshikawa+%22Anthesis%2C+fertilization+and+development+of+caryopsis%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0051435-Ohdan1" id="pone.0051435-Ohdan1"></a>Ohdan T, Francisco PB, Sawada T, Hirose T, Terao T, et al. (2005) Expression profiling of genes involved in starch synthesis in sink and source organs of rice. J Exp Bot 56: 3229–3244.
        doi: 10.1093/jxb/eri292 <ul class="find" data-citedArticleID="6354931" data-doi="10.1093/jxb/eri292"><li><a href="http://dx.doi.org/10.1093/jxb/eri292" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ohdan%5Bauthor%5D+AND+Expression+profiling+of+genes+involved+in+starch+synthesis+in+sink+and+source+organs+of+rice" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AOhdan+%22Expression+profiling+of+genes+involved+in+starch+synthesis+in+sink+and+source+organs+of+rice%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0051435-Zhou1" id="pone.0051435-Zhou1"></a>Zhou Z, Wang L, Li J, Song X, Yang C (2009) Study on programmed cell death and dynamic changes of starch accumulation in pericarp cells of <em>Triticum aestivum</em> L. Protoplasma. 236: 49–58.
        doi: 10.1007/s00709-009-0046-7 <ul class="find" data-citedArticleID="6354933" data-doi="10.1007/s00709-009-0046-7"><li><a href="http://dx.doi.org/10.1007/s00709-009-0046-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zhou%5Bauthor%5D+AND+Study+on+programmed+cell+death+and+dynamic+changes+of+starch+accumulation+in+pericarp+cells+of+Triticum+aestivum+L.+Protoplasma" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AZhou+%22Study+on+programmed+cell+death+and+dynamic+changes+of+starch+accumulation+in+pericarp+cells+of+Triticum+aestivum+L.+Protoplasma%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0051435-Dominguez1" id="pone.0051435-Dominguez1"></a>Dominguez F, Moreno J, Cejudo FJ (2001) The nucellus degenerates by a process of programmed cell death during the early stages of wheat grain development. Planta 213: 352–360.
        doi: 10.1007/s004250000517 <ul class="find" data-citedArticleID="6354935" data-doi="10.1007/s004250000517"><li><a href="http://dx.doi.org/10.1007/s004250000517" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dominguez%5Bauthor%5D+AND+The+nucellus+degenerates+by+a+process+of+programmed+cell+death+during+the+early+stages+of+wheat+grain+development" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADominguez+%22The+nucellus+degenerates+by+a+process+of+programmed+cell+death+during+the+early+stages+of+wheat+grain+development%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0051435-Fukuda1" id="pone.0051435-Fukuda1"></a>Fukuda H (1997) Programmed cell death during vascular system formation. Cell Death Differ 4: 684–688.
        doi: 10.1038/sj.cdd.4400310 <ul class="find" data-citedArticleID="6354937" data-doi="10.1038/sj.cdd.4400310"><li><a href="http://dx.doi.org/10.1038/sj.cdd.4400310" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fukuda%5Bauthor%5D+AND+Programmed+cell+death+during+vascular+system+formation" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFukuda+%22Programmed+cell+death+during+vascular+system+formation%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0051435-Deng1" id="pone.0051435-Deng1"></a>Deng M, Bian H, Xie Y, Kim Y, Wang W, et al. (2011) Bcl-2 suppresses hydrogen peroxide-induced programmed cell death via <em>OsVPE2</em> and <em>OsVPE3</em>, but not via <em>OsVPE1</em> and <em>OsVPE4</em>, in rice. FEBS J 278: 4797–4810.
        doi: 10.1111/j.1742-4658.2011.08380.x <ul class="find" data-citedArticleID="6354939" data-doi="10.1111/j.1742-4658.2011.08380.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2011.08380.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deng%5Bauthor%5D+AND+Bcl-2+suppresses+hydrogen+peroxide-induced+programmed+cell+death+via+OsVPE2+and+OsVPE3%2C+but+not+via+OsVPE1+and+OsVPE4%2C+in+rice" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADeng+%22Bcl-2+suppresses+hydrogen+peroxide-induced+programmed+cell+death+via+OsVPE2+and+OsVPE3%2C+but+not+via+OsVPE1+and+OsVPE4%2C+in+rice%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0051435-Radchuk1" id="pone.0051435-Radchuk1"></a>Radchuk V, Weier D, Radchuk R, Weschke W, Weber H (2010) Development of maternal seed tissue in barley is mediated by regulated cell expansion and cell disintegration and coordinated with endosperm growth. J Exp Bot 62: 1217–1227.
        doi: 10.1093/jxb/erq348 <ul class="find" data-citedArticleID="6354941" data-doi="10.1093/jxb/erq348"><li><a href="http://dx.doi.org/10.1093/jxb/erq348" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Radchuk%5Bauthor%5D+AND+Development+of+maternal+seed+tissue+in+barley+is+mediated+by+regulated+cell+expansion+and+cell+disintegration+and+coordinated+with+endosperm+growth" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARadchuk+%22Development+of+maternal+seed+tissue+in+barley+is+mediated+by+regulated+cell+expansion+and+cell+disintegration+and+coordinated+with+endosperm+growth%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0051435-Ikuko1" id="pone.0051435-Ikuko1"></a>Ikuko HN, Noriyuki H, Satoru N, Miwa K, Mikio N (2005) Vacuolar processing enzyme: an executor of plant cell death. Curr Opin Plant Biol 8: 404–408.
        doi: 10.1016/j.pbi.2005.05.016 <ul class="find" data-citedArticleID="6354943" data-doi="10.1016/j.pbi.2005.05.016"><li><a href="http://dx.doi.org/10.1016/j.pbi.2005.05.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ikuko%5Bauthor%5D+AND+Vacuolar+processing+enzyme%3A+an+executor+of+plant+cell+death" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AIkuko+%22Vacuolar+processing+enzyme%3A+an+executor+of+plant+cell+death%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0051435-Varda1" id="pone.0051435-Varda1"></a>Varda SB, Nurit K, Hilal Z (2008) Uncovering the role of VDAC in the regulation of cell life and death. J Bioenerg Biomembr 40: 183–191.
        doi: 10.1007/s10863-008-9147-9 <ul class="find" data-citedArticleID="6354945" data-doi="10.1007/s10863-008-9147-9"><li><a href="http://dx.doi.org/10.1007/s10863-008-9147-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Varda%5Bauthor%5D+AND+Uncovering+the+role+of+VDAC+in+the+regulation+of+cell+life+and+death" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AVarda+%22Uncovering+the+role+of+VDAC+in+the+regulation+of+cell+life+and+death%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0051435-Godbole1" id="pone.0051435-Godbole1"></a>Godbole A, Varghese J, Sarin A, Mathew MK (2003) VDAC is a conserved element of death pathways in plant and animal systems. Biochimica et Biophysica Acta (BBA) - Molecular Cell Research 1642: 87–96.
        doi: 10.1016/s0167-4889(03)00102-2 <ul class="find" data-citedArticleID="6354947" data-doi="10.1016/s0167-4889(03)00102-2"><li><a href="http://dx.doi.org/10.1016/s0167-4889(03)00102-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Godbole%5Bauthor%5D+AND+VDAC+is+a+conserved+element+of+death+pathways+in+plant+and+animal+systems" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGodbole+%22VDAC+is+a+conserved+element+of+death+pathways+in+plant+and+animal+systems%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0051435-Kim1" id="pone.0051435-Kim1"></a>Kim S, Kim S, Wang Y, Yu S, Choi I, et al. (2011) The RNase activity of rice probenazole-induced protein1 (PBZ1) plays a key role in cell death in plants. Mol Cells 31: 25–31.
        doi: 10.1007/s10059-011-0004-z <ul class="find" data-citedArticleID="6354949" data-doi="10.1007/s10059-011-0004-z"><li><a href="http://dx.doi.org/10.1007/s10059-011-0004-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kim%5Bauthor%5D+AND+The+RNase+activity+of+rice+probenazole-induced+protein1+%28PBZ1%29+plays+a+key+role+in+cell+death+in+plants" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKim+%22The+RNase+activity+of+rice+probenazole-induced+protein1+%28PBZ1%29+plays+a+key+role+in+cell+death+in+plants%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0051435-Jwa1" id="pone.0051435-Jwa1"></a>Jwa NS, Kumar Agrawal G, Rakwal R, Park CH, Prasad Agrawal V (2001) Molecular cloning and characterization of a novel jasmonate inducible pathogenesis-related class 10 protein gene, <em>JIOsPR10</em>, from rice (<em>Oryza sativa</em> L.) seedling leaves. Biochem Bioph Res Co 286: 973–983.
        doi: 10.1006/bbrc.2001.5507 <ul class="find" data-citedArticleID="6354951" data-doi="10.1006/bbrc.2001.5507"><li><a href="http://dx.doi.org/10.1006/bbrc.2001.5507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Jwa%5Bauthor%5D+AND+Molecular+cloning+and+characterization+of+a+novel+jasmonate+inducible+pathogenesis-related+class+10+protein+gene%2C+JIOsPR10%2C+from+rice+%28Oryza+sativa+L.%29+seedling+leaves" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AJwa+%22Molecular+cloning+and+characterization+of+a+novel+jasmonate+inducible+pathogenesis-related+class+10+protein+gene%2C+JIOsPR10%2C+from+rice+%28Oryza+sativa+L.%29+seedling+leaves%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0051435-Riechmann2" id="pone.0051435-Riechmann2"></a>Riechmann JL, Wang M, Meyerowitz EM (1996) DNA-binding properties of arabidopsis MADS domain homeotic proteins APETALA1, APETALA3, PISTILLATA and AGAMOUS. Nucleic Acids Res 24: 3134–3141.
        doi: 10.1093/nar/24.16.3134 <ul class="find" data-citedArticleID="6354953" data-doi="10.1093/nar/24.16.3134"><li><a href="http://dx.doi.org/10.1093/nar/24.16.3134" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Riechmann%5Bauthor%5D+AND+DNA-binding+properties+of+arabidopsis+MADS+domain+homeotic+proteins+APETALA1%2C+APETALA3%2C+PISTILLATA+and+AGAMOUS" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARiechmann+%22DNA-binding+properties+of+arabidopsis+MADS+domain+homeotic+proteins+APETALA1%2C+APETALA3%2C+PISTILLATA+and+AGAMOUS%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0051435-Wikstrm1" id="pone.0051435-Wikstrm1"></a>Wikström N, Savolainen V, Chase MW (2001) Evolution of the angiosperms: calibrating the family tree. Proc R Soc Lond B Biol Sci 268: 2211–2220.
        doi: 10.1098/rspb.2001.1782 <ul class="find" data-citedArticleID="6354955" data-doi="10.1098/rspb.2001.1782"><li><a href="http://dx.doi.org/10.1098/rspb.2001.1782" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wikstr%C3%B6m%5Bauthor%5D+AND+Evolution+of+the+angiosperms%3A+calibrating+the+family+tree" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWikstr%C3%B6m+%22Evolution+of+the+angiosperms%3A+calibrating+the+family+tree%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0051435-Prasad2" id="pone.0051435-Prasad2"></a>Prasad V, Strömberg CAE, Alimohammadian H, Sahni A (2005) Dinosaur coprolites and the early evolution of grasses and grazers. Science 310: 1177–1180.
        doi: 10.1126/science.1118806 <ul class="find" data-citedArticleID="6354957" data-doi="10.1126/science.1118806"><li><a href="http://dx.doi.org/10.1126/science.1118806" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prasad%5Bauthor%5D+AND+Dinosaur+coprolites+and+the+early+evolution+of+grasses+and+grazers" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APrasad+%22Dinosaur+coprolites+and+the+early+evolution+of+grasses+and+grazers%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0051435-Gaut1" id="pone.0051435-Gaut1"></a>Gaut BS (2002) Evolutionary dynamics of grass genomes. New Phytol 154: 15–28.
        doi: 10.1046/j.1469-8137.2002.00352.x <ul class="find" data-citedArticleID="6354959" data-doi="10.1046/j.1469-8137.2002.00352.x"><li><a href="http://dx.doi.org/10.1046/j.1469-8137.2002.00352.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gaut%5Bauthor%5D+AND+Evolutionary+dynamics+of+grass+genomes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGaut+%22Evolutionary+dynamics+of+grass+genomes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0051435-Kellogg1" id="pone.0051435-Kellogg1"></a>Kellogg EA (2001) Evolutionary history of the grasses. Plant Physiol 125: 1198–1205.
        doi: 10.1104/pp.125.3.1198 <ul class="find" data-citedArticleID="6354961" data-doi="10.1104/pp.125.3.1198"><li><a href="http://dx.doi.org/10.1104/pp.125.3.1198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kellogg%5Bauthor%5D+AND+Evolutionary+history+of+the+grasses" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKellogg+%22Evolutionary+history+of+the+grasses%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0051435-Becker2" id="pone.0051435-Becker2"></a>Becker A, Winter KU, Meyer B, Saedler H, Theißen G (2000) MADS-Box gene diversity in seed plants 300 Million Years Ago. Mol Biol Evol 17: 1425–1434.
        doi: 10.1093/oxfordjournals.molbev.a026243 <ul class="find" data-citedArticleID="6354963" data-doi="10.1093/oxfordjournals.molbev.a026243"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026243" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Becker%5Bauthor%5D+AND+MADS-Box+gene+diversity+in+seed+plants+300+Million+Years+Ago" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABecker+%22MADS-Box+gene+diversity+in+seed+plants+300+Million+Years+Ago%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0051435-Dreni1" id="pone.0051435-Dreni1"></a>Dreni L, Jacchia S, Fornara F, Fornari M, Ouwerkerk PBF, et al. (2007) The D-lineage MADS-box gene <em>OsMADS13</em> controls ovule identity in rice. Plant J 52: 690–699.
        doi: 10.1111/j.1365-313x.2007.03272.x <ul class="find" data-citedArticleID="6354965" data-doi="10.1111/j.1365-313x.2007.03272.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2007.03272.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dreni%5Bauthor%5D+AND+The+D-lineage+MADS-box+gene+OsMADS13+controls+ovule+identity+in+rice" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADreni+%22The+D-lineage+MADS-box+gene+OsMADS13+controls+ovule+identity+in+rice%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0051435-Fujita1" id="pone.0051435-Fujita1"></a>Fujita M, Horiuchi Y, Ueda Y, Mizuta Y, Kubo T, et al. (2010) Rice expression atlas in reproductive development. Plant Cell Physiol 51: 2060–2081.
        doi: 10.1093/pcp/pcq165 <ul class="find" data-citedArticleID="6354967" data-doi="10.1093/pcp/pcq165"><li><a href="http://dx.doi.org/10.1093/pcp/pcq165" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fujita%5Bauthor%5D+AND+Rice+expression+atlas+in+reproductive+development" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFujita+%22Rice+expression+atlas+in+reproductive+development%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0051435-Krishnan1" id="pone.0051435-Krishnan1"></a>Krishnan S, Dayanandan P (2003) Structural and histochemical studies on grain-filling in the caryopsis of rice (<em>Oryza sativa</em> L.). J Bioscience 28: 455–469.
        doi: 10.1007/bf02705120 <ul class="find" data-citedArticleID="6354969" data-doi="10.1007/bf02705120"><li><a href="http://dx.doi.org/10.1007/bf02705120" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Krishnan%5Bauthor%5D+AND+Structural+and+histochemical+studies+on+grain-filling+in+the+caryopsis+of+rice+%28Oryza+sativa+L.%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKrishnan+%22Structural+and+histochemical+studies+on+grain-filling+in+the+caryopsis+of+rice+%28Oryza+sativa+L.%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0051435-Oparka1" id="pone.0051435-Oparka1"></a>Oparka KJ, Gates P (1984) Sink anatomy in relation to solute movement in rice (<em>Oryza sativa</em> L.): A summary of findings. Plant Growth Regul 2: 297–307.
        doi: 10.1007/bf00027289 <ul class="find" data-citedArticleID="6354971" data-doi="10.1007/bf00027289"><li><a href="http://dx.doi.org/10.1007/bf00027289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oparka%5Bauthor%5D+AND+Sink+anatomy+in+relation+to+solute+movement+in+rice+%28Oryza+sativa+L.%29%3A+A+summary+of+findings" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AOparka+%22Sink+anatomy+in+relation+to+solute+movement+in+rice+%28Oryza+sativa+L.%29%3A+A+summary+of+findings%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0051435-Oparka2" id="pone.0051435-Oparka2"></a>Oparka KJ, Gates P (1981) Transport of assimilates in the developing caryopsis of rice (<em>Oryza sativa</em> L.). Planta 151: 561–573.
        doi: 10.1007/bf00387436 <ul class="find" data-citedArticleID="6354973" data-doi="10.1007/bf00387436"><li><a href="http://dx.doi.org/10.1007/bf00387436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oparka%5Bauthor%5D+AND+Transport+of+assimilates+in+the+developing+caryopsis+of+rice+%28Oryza+sativa+L.%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AOparka+%22Transport+of+assimilates+in+the+developing+caryopsis+of+rice+%28Oryza+sativa+L.%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0051435-Cui1" id="pone.0051435-Cui1"></a>Cui R, Han J, Zhao S, Su K, Wu F, et al. (2010) Functional conservation and diversification of class E floral homeotic genes in rice (<em>Oryza sativa</em>). Plant J 61: 767–781.
        doi: 10.1111/j.1365-313x.2009.04101.x <ul class="find" data-citedArticleID="6354975" data-doi="10.1111/j.1365-313x.2009.04101.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2009.04101.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cui%5Bauthor%5D+AND+Functional+conservation+and+diversification+of+class+E+floral+homeotic+genes+in+rice+%28Oryza+sativa%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACui+%22Functional+conservation+and+diversification+of+class+E+floral+homeotic+genes+in+rice+%28Oryza+sativa%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0051435-Swofford1" id="pone.0051435-Swofford1"></a>Swofford D (1993) PAUP: A computer program for phylogenetic inference using maximum parsimony. J Gen Physiol 102: 9A. <ul class="find" data-citedArticleID="6354977"><li><a href="http://www.crossref.org/guestquery/?auth2=Swofford&amp;atitle2=PAUP%3A+A+computer+program+for+phylogenetic+inference+using+maximum+parsimony&amp;auth=Swofford&amp;atitle=PAUP%3A+A+computer+program+for+phylogenetic+inference+using+maximum+parsimony" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Swofford%5Bauthor%5D+AND+PAUP%3A+A+computer+program+for+phylogenetic+inference+using+maximum+parsimony" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASwofford+%22PAUP%3A+A+computer+program+for+phylogenetic+inference+using+maximum+parsimony%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0051435-Do1" id="pone.0051435-Do1"></a>Do CB, Mahabhashyam MSP, Brudno M, Batzoglou S (2005) ProbCons: Probabilistic consistency-based multiple sequence alignment. Genome Res 15: 330–340.
        doi: 10.1101/gr.2821705 <ul class="find" data-citedArticleID="6354979" data-doi="10.1101/gr.2821705"><li><a href="http://dx.doi.org/10.1101/gr.2821705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Do%5Bauthor%5D+AND+ProbCons%3A+Probabilistic+consistency-based+multiple+sequence+alignment" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADo+%22ProbCons%3A+Probabilistic+consistency-based+multiple+sequence+alignment%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0051435-Wernersson1" id="pone.0051435-Wernersson1"></a>Wernersson R, Pedersen AG (2003) RevTrans: multiple alignment of coding DNA from aligned amino acid sequences. Nucleic Acids Res 31: 3537–3539.
        doi: 10.1093/nar/gkg609 <ul class="find" data-citedArticleID="6354981" data-doi="10.1093/nar/gkg609"><li><a href="http://dx.doi.org/10.1093/nar/gkg609" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wernersson%5Bauthor%5D+AND+RevTrans%3A+multiple+alignment+of+coding+DNA+from+aligned+amino+acid+sequences" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWernersson+%22RevTrans%3A+multiple+alignment+of+coding+DNA+from+aligned+amino+acid+sequences%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0051435-Posada1" id="pone.0051435-Posada1"></a>Posada D, Crandall KA (1998) MODELTEST: testing the model of DNA substitution. Bioinformatics 14: 817–818.
        doi: 10.1093/bioinformatics/14.9.817 <ul class="find" data-citedArticleID="6354983" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Posada%5Bauthor%5D+AND+MODELTEST%3A+testing+the+model+of+DNA+substitution" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APosada+%22MODELTEST%3A+testing+the+model+of+DNA+substitution%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0051435-Rodrguez1" id="pone.0051435-Rodrguez1"></a>Rodríguez F, Oliver JL, Marín A, Medina JR (1990) The general stochastic model of nucleotide substitution. J Theor Biol 142: 485–501.
        doi: 10.1016/s0022-5193(05)80104-3 <ul class="find" data-citedArticleID="6354985" data-doi="10.1016/s0022-5193(05)80104-3"><li><a href="http://dx.doi.org/10.1016/s0022-5193(05)80104-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rodr%C3%ADguez%5Bauthor%5D+AND+The+general+stochastic+model+of+nucleotide+substitution" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARodr%C3%ADguez+%22The+general+stochastic+model+of+nucleotide+substitution%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0051435-Ronquist1" id="pone.0051435-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics 19: 1572–1574.
        doi: 10.1093/bioinformatics/btg180 <ul class="find" data-citedArticleID="6354987" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ronquist%5Bauthor%5D+AND+MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARonquist+%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0051435-Ohmori1" id="pone.0051435-Ohmori1"></a>Ohmori S, Kimizu M, Sugita M, Miyao A, Hirochika H, et al. (2009) <em>MOSAIC FLORAL ORGANS1</em>, an <em>AGL6</em>-Like MADS box gene, regulates floral organ identity and meristem fate in rice. Plant Cell 21: 3008–3025.
        doi: 10.1105/tpc.109.068742 <ul class="find" data-citedArticleID="6354989" data-doi="10.1105/tpc.109.068742"><li><a href="http://dx.doi.org/10.1105/tpc.109.068742" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ohmori%5Bauthor%5D+AND+MOSAIC+FLORAL+ORGANS1%2C+an+AGL6-Like+MADS+box+gene%2C+regulates+floral+organ+identity+and+meristem+fate+in+rice" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AOhmori+%22MOSAIC+FLORAL+ORGANS1%2C+an+AGL6-Like+MADS+box+gene%2C+regulates+floral+organ+identity+and+meristem+fate+in+rice%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0051435-Itoh1" id="pone.0051435-Itoh1"></a>Itoh JI, Nonomura KI, Ikeda K, Yamaki S, Inukai Y, et al. (2005) Rice plant development: from zygote to spikelet. Plant Cell Physiol 46: 23–47.
        doi: 10.1093/pcp/pci501 <ul class="find" data-citedArticleID="6354991" data-doi="10.1093/pcp/pci501"><li><a href="http://dx.doi.org/10.1093/pcp/pci501" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Itoh%5Bauthor%5D+AND+Rice+plant+development%3A+from+zygote+to+spikelet" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AItoh+%22Rice+plant+development%3A+from+zygote+to+spikelet%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.XML" value="117187"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.PDF" value="2478653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g001.TIF" value="1557038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g001.PNG_I" value="29152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g001.PNG_S" value="9887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g001.PNG_M" value="43003"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g001.PNG_L" value="582703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g002.TIF" value="4015144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g002.PNG_I" value="197913"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g002.PNG_S" value="22758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g002.PNG_M" value="270598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g002.PNG_L" value="4293865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g003.TIF" value="1413506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g003.PNG_I" value="78570"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g003.PNG_S" value="18438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g003.PNG_M" value="209998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g003.PNG_L" value="1168036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.t001.TIF" value="124810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.t001.PNG_I" value="6379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.t001.PNG_S" value="1880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.t001.PNG_M" value="14425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.t001.PNG_L" value="29726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g004.TIF" value="7971590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g004.PNG_I" value="377718"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g004.PNG_S" value="32666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g004.PNG_M" value="547366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g004.PNG_L" value="9143530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g005.TIF" value="4506374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g005.PNG_I" value="227306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g005.PNG_S" value="25964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g005.PNG_M" value="657519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g005.PNG_L" value="6322725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g006.TIF" value="2061824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g006.PNG_I" value="200753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g006.PNG_S" value="22904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g006.PNG_M" value="406264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.g006.PNG_L" value="2529055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.s001.TIF" value="885482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.s002.TIF" value="248896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.s003.TIF" value="2477692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.s004.TIF" value="3286952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.s005.TIF" value="2259168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.s006.TIF" value="955158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.s007.TIF" value="471676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0051435.s008.DOCX" value="16120"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0051435&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0051435" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0051435&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0051435&volume=&issue=&title=Live and Let Die - The Bsister MADS-Box Gene OsMADS29 Controls the Degeneration of Cells in Maternal Tissues during Seed Development of Rice (Oryza sativa)&author_name=Xuelian%20Yang%2C%20Feng%20Wu%2C%20Xuelei%20Lin%2C%20Xiaoqiu%20Du%2C%20Kang%20Chong%2C%20Lydia%20Gramzow%2C%20Susanne%20Schilling%2C%20Annette%20Becker%2C%20G%C3%BCnter%20Thei%C3%9Fen%2C%20Zheng%20Meng&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0051435" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0051435" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0051435" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0051435&amp;t=Live%20and%20Let%20Die%20-%20The%20Bsister%20MADS-Box%20Gene%20OsMADS29%20Controls%20the%20Degeneration%20of%20Cells%20in%20Maternal%20Tissues%20during%20Seed%20Development%20of%20Rice%20(Oryza%20sativa)" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0051435&title=Live%20and%20Let%20Die%20-%20The%20Bsister%20MADS-Box%20Gene%20OsMADS29%20Controls%20the%20Degeneration%20of%20Cells%20in%20Maternal%20Tissues%20during%20Seed%20Development%20of%20Rice%20(Oryza%20sativa)&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0051435&amp;title=Live%20and%20Let%20Die%20-%20The%20Bsister%20MADS-Box%20Gene%20OsMADS29%20Controls%20the%20Degeneration%20of%20Cells%20in%20Maternal%20Tissues%20during%20Seed%20Development%20of%20Rice%20(Oryza%20sativa)" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0051435" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0051435&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Live and Let Die - The Bsister MADS-Box Gene OsMADS29 Controls the Degeneration of Cells in Maternal Tissues during Seed Development of Rice (Oryza sativa)';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0051435';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Live%20and%20Let%20Die%20-%20The%20Bsister%20MADS-Box%20Gene%20OsMADS29%20Controls%20the%20Degeneration%20of%20Cells%20in%20Maternal%20Tissues%20during%20Seed%20Development%20of%20Rice%20(Oryza%20sativa) http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0051435" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0051435" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Embryos%22" title="Search for articles in the subject area:'Embryos'"><div class="flagText">Embryos</div></a>
              <div data-categoryid="17321" data-articleid="134447"
                   data-categoryname="Embryos"
                   class="flagImage" title="Flag 'Embryos' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Integumentary+system%22" title="Search for articles in the subject area:'Integumentary system'"><div class="flagText">Integumentary system</div></a>
              <div data-categoryid="50079" data-articleid="134447"
                   data-categoryname="Integumentary system"
                   class="flagImage" title="Flag 'Integumentary system' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ovules%22" title="Search for articles in the subject area:'Ovules'"><div class="flagText">Ovules</div></a>
              <div data-categoryid="18707" data-articleid="134447"
                   data-categoryname="Ovules"
                   class="flagImage" title="Flag 'Ovules' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pericarp%22" title="Search for articles in the subject area:'Pericarp'"><div class="flagText">Pericarp</div></a>
              <div data-categoryid="24927" data-articleid="134447"
                   data-categoryname="Pericarp"
                   class="flagImage" title="Flag 'Pericarp' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Rice%22" title="Search for articles in the subject area:'Rice'"><div class="flagText">Rice</div></a>
              <div data-categoryid="31915" data-articleid="134447"
                   data-categoryname="Rice"
                   class="flagImage" title="Flag 'Rice' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Seeds%22" title="Search for articles in the subject area:'Seeds'"><div class="flagText">Seeds</div></a>
              <div data-categoryid="18605" data-articleid="134447"
                   data-categoryname="Seeds"
                   class="flagImage" title="Flag 'Seeds' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Starches%22" title="Search for articles in the subject area:'Starches'"><div class="flagText">Starches</div></a>
              <div data-categoryid="25599" data-articleid="134447"
                   data-categoryname="Starches"
                   class="flagImage" title="Flag 'Starches' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transgenic+plants%22" title="Search for articles in the subject area:'Transgenic plants'"><div class="flagText">Transgenic plants</div></a>
              <div data-categoryid="20465" data-articleid="134447"
                   data-categoryname="Transgenic plants"
                   class="flagImage" title="Flag 'Transgenic plants' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3507'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1133'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8940&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
