

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Adaptation of Organisms by Resonance of RNA Transcription with the Cellular Redox Cycle</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0025270"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0025270" />

    <meta name="citation_title" content="Adaptation of Organisms by Resonance of RNA Transcription with the Cellular Redox Cycle"/>
    <meta itemprop="name" content="Adaptation of Organisms by Resonance of RNA Transcription with the Cellular Redox Cycle"/>

      <meta name="citation_author" content="Viktor Stolc"/>
            <meta name="citation_author_institution" content="NASA Ames Research Center, Moffett Field, California, United States of America"/>
      <meta name="citation_author" content="Alena Shmygelska"/>
            <meta name="citation_author_institution" content="Carnegie Mellon University Silicon Valley, Moffett Field, California, United States of America"/>
      <meta name="citation_author" content="Yuri Griko"/>
            <meta name="citation_author_institution" content="NASA Ames Research Center, Moffett Field, California, United States of America"/>

    <meta name="citation_date" content="2011/9/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0025270.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e25270"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="6"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Redox rhythmicity: clocks at the core of temporal coherence.; citation_author=D Lloyd; citation_author=DB Murray; citation_journal_title=Bioessays; citation_volume=29; citation_number=1; citation_pages=465-473; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The temporal architecture of eukaryotic growth.; citation_author=D Lloyd; citation_author=DB Murray; citation_journal_title=FEBS Lett; citation_volume=580; citation_number=2; citation_pages=2830-2835; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The microbial engines that drive Earth's biogeochemical cycles.; citation_author=PG Falkowski; citation_author=T Fenchel; citation_author=EF Delong; citation_journal_title=Science; citation_volume=320; citation_number=3; citation_pages=34-39; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=How to Flip the (Redox) Switch.; citation_author=G Georgiou; citation_journal_title=Cell; citation_volume=11; citation_number=4; citation_pages=607-610; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structural basis for the redox control of plant glutamate cysteine ligase.; citation_author=M Hothorn; citation_author=A Wachter; citation_author=R Gromes; citation_author=T Stuwe; citation_author=T Rausch; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=5; citation_pages=27557-2765; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microbial H2O2 sensors as archetypical redox signaling modules.; citation_author=MB Toledano; citation_author=A Delaunay; citation_author=L Monceau; citation_author=F Tacnet; citation_journal_title=Trends Biochem Sci; citation_volume=29; citation_number=6; citation_pages=351-357; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A genomewide oscillation in transcription gates DNA replication and cell cycle.; citation_author=RR Klevecz; citation_author=J Bolen; citation_author=G Forrest; citation_author=DB Murray; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=7; citation_pages=200-205; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Calculus of Logic, Cambridge and Dublin Mathematical Journal; citation_author=George Boole; citation_volume=3; citation_number=8; citation_pages=183-198; citation_date=1848; " />
      <meta name="citation_reference" content="citation_title=Fluctuation theorem for Hamiltonian systems - Le Chatelier's principle.; citation_author=DJ Evans; citation_author=DJ Searles; citation_author=E Mittag; citation_journal_title=Phys Rev E Stat Nonlin Soft Matter Phys; citation_volume=63; citation_number=9; citation_pages=051105; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A rapid genome-scale response of the transcriptional oscillator to perturbation reveals a period-doubling path to phenotypic change.; citation_author=CM Li; citation_author=RR Klevecz; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=10; citation_pages=16254-16259; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence of carbon monoxide-mediated phase advancement of the yeast metabolic cycle.; citation_author=BP Tu; citation_author=SL McKnight; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=106; citation_number=11; citation_pages=14293-14296; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Oxidants, oxidative stress and the biology of ageing.; citation_author=T Finkel; citation_author=NJ Holbrook; citation_journal_title=Nature; citation_volume=408; citation_number=12; citation_pages=239-247; citation_date=2000; " />
      <meta name="citation_reference" content="citation_author=N Eldredge; citation_author=SJ Gould; citation_title=Punctuated equilibria: an alternative to phyletic gradualism In Schopf TJM, editor, Models in Paleobiology; citation_number=13; citation_pages=82-15; citation_date=1972; citation_publisher=Freeman Cooper; " />
      <meta name="citation_reference" content="citation_author=P Ward; citation_title=Out of Thin Air: Dinosaurs, Birds, and Earth's Ancient Atmosphere; citation_number=14; citation_date=2006; citation_publisher=Joseph Henry Press; " />
      <meta name="citation_reference" content="citation_title=The Origin of Life.; citation_author=J Trefil; citation_author=HJ Morowitz; citation_author=E Smith; citation_journal_title=American Scientist; citation_volume=97; citation_number=15; citation_pages=206-213; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Estimating the per-base-pair mutation rate in the yeast Saccharomyces cerevisiae.; citation_author=GI Lang; citation_author=AW Murray; citation_journal_title=Genetics; citation_volume=178; citation_number=16; citation_pages=67-82; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Microbial genetics: Hypermutation under stress.; citation_author=BA Bridges; citation_journal_title=Nature; citation_volume=387; citation_number=17; citation_pages=557-558; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Free radicals, exercise, apoptosis, and heat shock proteins.; citation_author=E Fehrenbach; citation_author=H Northoff; citation_journal_title=Exerc Immonol Rev; citation_volume=7; citation_number=18; citation_pages=66-89; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=On the dependence of spontaneous mutation rates on the functional state of genes.; citation_author=VI Korogodin; citation_author=VL Korogodina; citation_author=C Fajszi; citation_author=AI Chepurnoy; citation_author=N Mikhova-Tsenova; citation_journal_title=Yeast; citation_volume=7; citation_number=19; citation_pages=105-117; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=A biochemical mechanism for nonrandom mutations and evolution.; citation_author=BE Wright; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=20; citation_pages=2993-3001; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Involvement of oxidative stress in the regulation of H2S production during ultradian metabolic oscillation of Saccharomyces cerevisiae.; citation_author=WJ Kwak; citation_author=GS Kwon; citation_author=I Jin; citation_author=H Kuriyama; citation_author=HY Sohn; citation_journal_title=FEMS Microbiology Letters; citation_volume=219; citation_number=21; citation_pages=99-104; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Regulation of yeast oscillatory dynamics.; citation_author=DB Murray; citation_author=M Beckmann; citation_author=H Kitano; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=22; citation_pages=2241-2246; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cyclic changes in metabolic state during the life of a yeast cell.; citation_author=BP Tu; citation_author=RE Mohler; citation_author=JC Liu; citation_author=KM Dombek; citation_author=ET Young; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=23; citation_pages=16886-16891; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Logic of the yeast metabolic cycle: temporal compartmentalization of cellular processes.; citation_author=BP Tu; citation_author=A Kudlicki; citation_author=M Rowicka; citation_author=SL McKnight; citation_journal_title=Science; citation_volume=310; citation_number=24; citation_pages=1152-1158; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Restriction of DNA replication to the reductive phase of the metabolic cycle protects genome integrity.; citation_author=Z Chen; citation_author=EA Odstrcil; citation_author=BP Tu; citation_author=SL McKnight; citation_journal_title=Science; citation_volume=316; citation_number=25; citation_pages=1916-199; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=H2S cytotoxicity mechanism involves reactive oxygen species formation and mitochondrial depolarisation.; citation_author=MA Eghbal; citation_author=PS Pennefather; citation_author=PJ O'Brien; citation_journal_title=Toxicology; citation_volume=203; citation_number=26; citation_pages=69-76; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Peroxiredoxin-null yeast cells are hypersensitive to oxidative stress and are genomically unstable.; citation_author=CM Wong; citation_author=KL Siu; citation_author=DY Jin; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=27; citation_pages=23207-23213; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Yeast oxidative stress response. Influences of cytosolic thioredoxin peroxidase I and of the mitochondrial functional state.; citation_author=AP Demasi; citation_author=GA Pereira; citation_author=LE Netto; citation_journal_title=FEBS J; citation_volume=27; citation_number=28; citation_pages=805-816; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Oxygen metabolism and reactive oxygen species cause chromosomal rearrangements and cell death.; citation_author=S Ragu; citation_author=G Faye; citation_author=I Iraqui; citation_author=A Masurel-Heneman; citation_author=RD Kolodner; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=29; citation_pages=9747-9752; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Peroxiredoxin Tsa1 is the key peroxidase suppressing genome instability and protecting against cell death in Saccharomyces cerevisiae.; citation_author=I Iraqui; citation_author=G Kienda; citation_author=J Soeur; citation_author=G Faye; citation_author=G Baldacci; citation_journal_title=PLoS Genet; citation_volume=5; citation_number=30; citation_pages=e1000524; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Chronic oxidative DNA damage due to DNA repair defects causes chromosomal instability in Saccharomyces cerevisiae.; citation_author=NP Degtyareva; citation_author=L Chen; citation_author=P Mieczkowski; citation_author=TD Petes; citation_author=PW Doetsch; citation_journal_title=Mol Cell Biol; citation_volume=28; citation_number=31; citation_pages=5432-5445; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Spontaneous DNA damage in Saccharomyces cerevisiae elicits phenotypic properties similar to cancer cells.; citation_author=BA Evert; citation_author=TB Salmon; citation_author=B Song; citation_author=L Jingjing; citation_author=W Siede; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=32; citation_pages=22585-22594; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Highly mutagenic and severely imbalanced dNTP pools can escape detection by the S-phase checkpoint.; citation_author=D Kumar; citation_author=J Viberg; citation_author=AK Nilsson; citation_author=A Chabes; citation_journal_title=Nucleic Acids Res; citation_volume=38; citation_number=33; citation_pages=3975-3983; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of mutagenesis in vivo due to imbalanced dNTP pools.; citation_author=D Kumar; citation_author=AL Abdulovic; citation_author=J Viberg; citation_author=AK Nilsson; citation_author=TA Kunkel; citation_journal_title=Nucleic Acids Res; citation_volume=39; citation_number=34; citation_pages=1360-1371; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Evolving responsively: adaptive mutation.; citation_author=SM Rosenberg; citation_journal_title=Nat Rev Genet; citation_volume=2; citation_number=35; citation_pages=504-515; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The origin of mutants.; citation_author=J Cairns; citation_author=J Overbaugh; citation_author=S Miller; citation_journal_title=Nature; citation_volume=335; citation_number=36; citation_pages=142-145; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Transcriptional bias: a non-Lamarckian mechanism for substrate-induced mutations.; citation_author=BD Davis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=86; citation_number=37; citation_pages=5005-5009; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Origin of mutants disputed.; citation_author=D Charlesworth; citation_author=B Charlesworth; citation_journal_title=Nature; citation_volume=336; citation_number=38; citation_pages=525-528; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Stochastic switching as a survival strategy in fluctuating environments.; citation_author=M Acar; citation_author=JT Mettetal; citation_author=A van Oudenaarden; citation_journal_title=Nat Genet; citation_volume=40; citation_number=39; citation_pages=471-475; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=The codon Adaptation Index–a measure of directional synonymous codon usage bias, and its potential applications.; citation_author=PM Sharp; citation_author=WH Li; citation_journal_title=Nucleic Acids Res; citation_volume=15; citation_number=40; citation_pages=1281-1295; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Dynamic transcriptome analysis measures rates of mRNA synthesis and decay in yeast.; citation_author=C Miller; citation_author=B Schwalb; citation_author=K Maier; citation_author=D Schulz; citation_author=S Dümcke; citation_journal_title=Mol Syst Biol; citation_volume=7; citation_number=41; citation_pages=458; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Base excision of oxidative purine and pyrimidine DNA damage in Saccharomyces cerevisiae by a DNA glycosylase with sequence similarity to endonuclease III from Escherichia coli.; citation_author=L Eide; citation_author=M Bjørås; citation_author=M Pirovano; citation_author=I Alseth; citation_author=KG Berdal; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=42; citation_pages=10735-10740; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Metabolic cycling in single yeast cells from unsynchronized steady-state populations limited on glucose or phosphate.; citation_author=SJ Silverman; citation_author=AA Petti; citation_author=N Slavov; citation_author=L Parsons; citation_author=R Briehof; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=107; citation_number=43; citation_pages=6946-6951; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Codon preferences in free-living microorganisms.; citation_author=SG Andersson; citation_author=CG Kurland; citation_journal_title=Microbiol Rev; citation_volume=54; citation_number=44; citation_pages=198-210; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Co-variation of tRNA abundance and codon usage in Escherichia coli at different growth rates.; citation_author=H Dong; citation_author=L Nilsson; citation_author=CG Kurland; citation_journal_title=J Mol Biol; citation_volume=260; citation_number=45; citation_pages=649-663; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Marginal fitness contributions of nonessential genes in yeast.; citation_author=JW Thatcher; citation_author=JM Shaw; citation_author=WJ Dickinson; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=95; citation_number=46; citation_pages=253-257; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=SGD project. Saccharomyces Genome Database.; citation_number=47; " />
      <meta name="citation_reference" content="citation_title=Global identification of noncoding RNAs in Saccharomyces cerevisiae by modulating an essential RNA processing pathway.; citation_author=MP Samanta; citation_author=W Tongprasit; citation_author=H Sethi; citation_author=CS Chin; citation_author=V Stolc; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=48; citation_pages=4192-4197; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.; citation_author=J Schacherer; citation_author=JA Shapiro; citation_author=DM Ruderfer; citation_author=L Kruglyak; citation_journal_title=Nature; citation_volume=458; citation_number=49; citation_pages=342-345; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=A catalog of neutral and deleterious polymorphism in yeast.; citation_author=SW Doniger; citation_author=HS Kim; citation_author=D Swain; citation_author=D Corcuera; citation_author=M Williams; citation_journal_title=PLoS Genet; citation_volume=4; citation_number=50; citation_pages=e1000183; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Exploration of essential gene functions via titratable promoter alleles.; citation_author=S Mnaimneh; citation_author=AP Davierwala; citation_author=J Haynes; citation_author=J Moffat; citation_author=WT Peng; citation_journal_title=Cell; citation_volume=118; citation_number=51; citation_pages=31-44; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The molecular basis of genetic dominance.; citation_author=AO Wilkie; citation_journal_title=J Med Genet; citation_volume=31; citation_number=52; citation_pages=89-98; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Yeast mitochondrial biogenesis: a role for the PUF RNA-binding protein Puf3p in mRNA localization.; citation_author=Y Saint-Georges; citation_author=M Garcia; citation_author=T Delaveau; citation_author=L Jourdren; citation_author=S Le Crom; citation_journal_title=PLoS One; citation_volume=3; citation_number=53; citation_pages=e2293; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Large-scale comparison of intron positions among animal, plant, and fungal genes.; citation_author=A Fedorov; citation_author=AF Merican; citation_author=W Gilbert; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=54; citation_pages=16128-16133; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Predicting functional upstream open reading frames in Saccharomyces cerevisiae.; citation_author=S Selpi; citation_author=CH Bryant; citation_author=GJ Kemp; citation_author=J Sarv; citation_author=E Kristiansson; citation_journal_title=BMC Bioinformatics; citation_volume=10; citation_number=55; citation_pages=451; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=The Transcriptional Landscape of the Yeast Genome Defined by RNA Sequencing.; citation_author=U Nagalakshmi; citation_author=Z Wang; citation_author=K Waern; citation_author=C Shou; citation_author=D Raha; citation_journal_title=Science; citation_volume=320; citation_number=56; citation_pages=1344-1349; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=On the Origin Of Species: A Facsimile of the First Edition; citation_number=57; citation_pages=11, 43, 134, 135, 136, 137, 447, 454, 455, 472, 479, and 480; citation_date=1964; citation_publisher=Harvard University Press; " />
      <meta name="citation_reference" content="citation_author=JG Mendel; citation_number=58; citation_pages=3-47; citation_date=1866; " />
      <meta name="citation_reference" content="citation_author=RW Burkhardt; citation_title=The Zoological Philosophy of J. B. Lamarck. Zoological Philosophy: An Exposition with Regard to the Natural History of Animals. By Lamarck JB. Trans. Hugh Elliot; citation_number=59; citation_date=1984; citation_publisher=The University of Chicago Press; " />
      <meta name="citation_reference" content="citation_author=J Chambers; citation_author=W Cleveland; citation_author=B Kleiner; citation_author=P Tukey; citation_number=60; citation_date=1983; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0025270" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Adaptation of Organisms by Resonance of RNA Transcription with the Cellular Redox Cycle"/>
    <meta name="twitter:description" content="
        Sequence variation in organisms differs across the genome and the majority of mutations are caused by oxidation, yet its origin is not fully understood. It has also been shown that the reduction-oxidation reaction cycle is the fundamental biochemical cycle that coordinates the timing of all biochemical processes in the cell, including energy production, DNA replication, and RNA transcription. We show that the temporal resonance of transcriptome biosynthesis with the oscillating binary state of the reduction-oxidation reaction cycle serves as a basis for non-random sequence variation at specific genome-wide coordinates that change faster than by accumulation of chance mutations. This work demonstrates evidence for a universal, persistent and iterative feedback mechanism between the environment and heredity, whereby acquired variation between cell divisions can outweigh inherited variation.
      "/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0025270.g003"/>

  <meta property="og:title" content="Adaptation of Organisms by Resonance of RNA Transcription with the Cellular Redox Cycle" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0025270" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6057'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6135'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9438&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025270">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0025270" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0025270&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Adaptation of Organisms by Resonance of RNA Transcription with the Cellular Redox Cycle
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Viktor Stolc
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:vstolc@mail.arc.nasa.gov">vstolc@mail.arc.nasa.gov</a></p>

                <p>Affiliation:
                  NASA Ames Research Center, Moffett Field, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alena Shmygelska, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Carnegie Mellon University Silicon Valley, Moffett Field, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yuri Griko
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  NASA Ames Research Center, Moffett Field, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 28, 2011</li>
    <li>DOI: 10.1371/journal.pone.0025270</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0025270" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0025270" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0025270" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0025270" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0025270-g001" data-doi="info:doi/10.1371/journal.pone.0025270" data-uri="info:doi/10.1371/journal.pone.0025270.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0025270-g002" data-doi="info:doi/10.1371/journal.pone.0025270" data-uri="info:doi/10.1371/journal.pone.0025270.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0025270-g003" data-doi="info:doi/10.1371/journal.pone.0025270" data-uri="info:doi/10.1371/journal.pone.0025270.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0025270-t001" data-doi="info:doi/10.1371/journal.pone.0025270" data-uri="info:doi/10.1371/journal.pone.0025270.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0025270">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0025270" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Sequence variation in organisms differs across the genome and the majority of mutations are caused by oxidation, yet its origin is not fully understood. It has also been shown that the reduction-oxidation reaction cycle is the fundamental biochemical cycle that coordinates the timing of all biochemical processes in the cell, including energy production, DNA replication, and RNA transcription. We show that the temporal resonance of transcriptome biosynthesis with the oscillating binary state of the reduction-oxidation reaction cycle serves as a basis for non-random sequence variation at specific genome-wide coordinates that change faster than by accumulation of chance mutations. This work demonstrates evidence for a universal, persistent and iterative feedback mechanism between the environment and heredity, whereby acquired variation between cell divisions can outweigh inherited variation.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Stolc V, Shmygelska A, Griko Y (2011) Adaptation of Organisms by Resonance of RNA Transcription with the Cellular Redox Cycle. PLoS ONE 6(9):
          e25270.
            doi:10.1371/journal.pone.0025270</p><p><strong>Editor: </strong>Jason E. Stajich, University of California Riverside, United States of America</p><p><strong>Received:</strong> December 17, 2010; <strong>Accepted:</strong> August 30, 2011; <strong>Published:</strong> September 28, 2011</p><p>This is an open-access article, free of all copyright, and may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose. The work is made available under the Creative Commons CC0 public domain dedication.</p><p><strong>Funding: </strong>This work was supported by the NASA Human Research Program, and NASA contract to the Carnegie Mellon University Silicon Valley #NNX08BA52A. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Time is an essential non-random variable of the information content and processing functions in cellular systems. Temporal coordination is fundamentally maintained by the oscillating reduction-oxidation reactions, i.e. the redox cycle. The redox cycle consists of successive forward and reverse transfers of electrons and protons in which reduced nicotinamide nucleotides drive the reduction of soluble low molecular weight thiols (e.g. glutathione), or disulfide bridges in proteins (e.g. thioredoxin and glutaredoxin) <a href="#pone.0025270-Lloyd1">[1]</a>. Molecular oxygen (O<sub>2</sub>) and hydrogen sulfide (H<sub>2</sub>S) act reversibly as diffusible allosteric gas effectors, affecting the redox cycle in opposite direction <a href="#pone.0025270-Lloyd1">[1]</a>, <a href="#pone.0025270-Lloyd2">[2]</a>. The successive and reversible transfer of electrons and protons alternatively reverses functionally active or inactive states in many, if not most protein and RNA enzymes. While the activating functions of metabolism are easily recognizable, they are indivisible from the more subtle inactivating function of metabolic stasis. Both functions are reversibly maintained by the endogenous production of H<sub>2</sub>S, which inhibits the electron transport in most aerobic organisms by antagonism of the oxygen binding sites with sulfur. The reoxidation is catalyzed by endogenously generated reactive oxygen species (ROS) or by reaction with molecular oxygen (O<sub>2</sub>) in aerobic organisms <a href="#pone.0025270-Lloyd1">[1]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Importantly, the redox cycle is universal and intrinsic among the three domains of life (archaea, bacteria, and eukaryotes), and despite its antiquity, it is mediated by highly conserved components that are genetically encoded and invariant to phylogenetic detection. The ubiquitous nature of the redox cycle presently extends to the Earth's redox state, which is an emergent property of microbial life on the planetary scale <a href="#pone.0025270-Falkowski1">[3]</a>. In cells, it is used for reversible activation or inactivation of proteins by altering their folding, structure, and function <a href="#pone.0025270-Georgiou1">[4]</a>–<a href="#pone.0025270-Toledano1">[6]</a>. The cycle not only alternatively changes the functional activity of individual proteins by direct flipping of their “on" and “off" functional states, but also causes alternating global reprogramming of the “on/off" functional state of all proteins and RNAs within a given cell by a rhythmic RNA transcription mechanism <a href="#pone.0025270-Lloyd1">[1]</a>. The redox cycle appears to be generated endogenously, at least in part, by the rhythm of the transcriptome biosynthesis, which can oscillate multiple times between cell divisions <a href="#pone.0025270-Klevecz1">[7]</a>. Thus, many biological functions are possible from a single chemical function that oscillates in a binary state that is reduced or oxidized as the only possible values and operations on such values, remarkably synonymous with Boolean logic <a href="#pone.0025270-Boole1">[8]</a>. The ability to reversibly alternate between the “on" and “off" functional states enables gene products to compartmentalize essential reactions and even entire pathways in cellular metabolism temporally to avoid fundamental chemical conflict with each other.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The effect of a change in concentration, temperature, or other environmental conditions on the chemical equilibrium of the redox cycle, specifically its chemical products that are inherently reactive, unstable, and self-regenerating, shifts the equilibrium to counter-act the imposed change according to the Le Chatelier's Principle <a href="#pone.0025270-Evans1">[9]</a>. Importantly, environment-directed variation of heredity is a fundamental consequence of the Le Chatelier's principle because cellular functions, which are modulated by the redox cycle, including transcriptome biosynthesis, can be not only modulated in amplitude but also synchronized temporally by signals that originate in the external physical environment, particularly when such perturbations are punctuated in phase, or in resonance, with the endogenous redox cycle, which oscillates many times in a day <a href="#pone.0025270-Lloyd1">[1]</a>. For example, external perturbations of the redox cycle can modulate its amplitude and period (e.g. highly diffusible drugs, H<sub>2</sub>S, CO, and acetaldehyde) <a href="#pone.0025270-Lloyd1">[1]</a>, <a href="#pone.0025270-Li1">[10]</a>–<a href="#pone.0025270-Tu1">[11]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>A periodic increase in the oxidized cellular state by the environment may result in a universal responsive variation of the genetic code and therefore provide a substrate for speciation. This hypothesis is supported by the accumulation of oxidant-damaged nuclear DNA in aging cells and organisms <a href="#pone.0025270-Finkel1">[12]</a>, and the discontinuity of species originally detected as the absence of gradualism in the fossil record. The later observation led to the punctuated equilibrium theory, which attempts to explain static that dominates the history of most fossil records <a href="#pone.0025270-Eldredge1">[13]</a>. Additionally, the emergence and extinction of species has been correlated with the time of transitions in the geological cycles of carbon, sulfur, and oxygen <a href="#pone.0025270-Ward1">[14]</a>. Consistently, the present universal cellular metabolism responds equivalently to appropriate concentrations of O<sub>2</sub> with activity, and by inactivity to H<sub>2</sub>S in aerobic organisms, or <em>vice-versa</em> in anaerobic organisms. The fundamental mechanism for the reversal of active and inactive metabolic states is encoded by the reversible direction of successive transfers of electrons and protons in the universal cellular metabolism <a href="#pone.0025270-Trefil1">[15]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Several experimental results suggest that mutation rate is non-uniform and varies across the genome by at least an order of magnitude in some organisms; however, such mutations appeared to be random <a href="#pone.0025270-Lang1">[16]</a>. Moreover, hypermutation under metabolic stress is increased significantly in microorganisms and in human cells <a href="#pone.0025270-Bridges1">[17]</a>, <a href="#pone.0025270-Fehrenbach1">[18]</a>. The unexpectedly high rate of mutation during nutritional stress may be caused by RNA transcription <a href="#pone.0025270-Korogodin1">[19]</a>, which may be a major cause of transition mutations in nature <a href="#pone.0025270-Wright1">[20]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>We hypothesized that the temporal nature of the redox cycle and its chemical products, such as the endogenously generated ROS produced during the oxidative phase and thiols produced during the reductive phase, may not only balance the chemical equilibrium of the redox cycle, but also accumulate non-random DNA sequence variation, i.e. directed with respect to selective conditions of the environment, during RNA transcription. The mechanism is based on the asymmetric chemical reactivity in the transcribed DNA strands. Specifically, an exposure of the non-transcribed strand and coincident protection of the transcribed single stranded DNA:RNA hybrid is caused by the transcription complex. The asymmetry results in differential reactivity of the two DNA strands to oxidation at actively transcribed loci. In contrast, DNA replication does not result in an asymmetric chemical reactivity of the two strands.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>This effect would oscillate periodically and differentially for each gene, and thus non-randomly among genes, because the level of ROS varies during the redox cycle and decreases the reducing equivalents of NAD(P)H and H<sub>2</sub>S correspondingly to a minimum level in the oxidative phase when a subset of genes are transcribed <a href="#pone.0025270-Lloyd2">[2]</a>. Analysis of H<sub>2</sub>S production profiles during perturbation of the redox cycle previously revealed that the amount of H<sub>2</sub>S production is closely linked with cellular oxidative stress by periodic inhibition of respiratory activity, which correspondingly increases ROS periodically in the oxidative phase <a href="#pone.0025270-Kwak1">[21]</a>. Moreover, the biochemical nature of the reductive and oxidative phases has been comprehensively analyzed and defined using gas and mass spectrometry of reduced and oxidized metabolites, which oscillate correspondingly in the appropriate phases of the redox cycle <a href="#pone.0025270-Murray1">[22]</a>, <a href="#pone.0025270-Tu2">[23]</a>.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>Thus, the frequency of RNA transcription-induced mutation would accumulate in genes that are transcribed periodically, restricted to the time when the chemically reactive potential generated by the oxidative phase of the cycle is at its maximum. In contrast, genes transcribed exclusively during the reductive phase of the cycle would be less subject to chemically induced mutation, as was shown previously for the frequency of mutation from genome replication restricted to the reductive phase <a href="#pone.0025270-Klevecz1">[7]</a>, <a href="#pone.0025270-Tu3">[24]</a>–<a href="#pone.0025270-Chen1">[25]</a>. As a result, the temporal resonance of RNA transcription with the oscillating binary state of the redox cycle would provide a fundamental and constant template for adaptation of the genetic code to the environment, “overwriting" any replication-based errors. Specifically, replication was previously shown to be gated by not one, but multiple redox cycles <a href="#pone.0025270-Klevecz1">[7]</a>, during which H<sub>2</sub>S acts in a manner similar to cyanide by binding to the heme in cytochrome c oxidase and inhibits electron transport and ATP production, which leads to an increased ROS production <a href="#pone.0025270-Eghbal1">[26]</a>. This fundamental and universal mechanism of H<sub>2</sub>S-induced ROS production during the redox cycle applies to all aerobic cells. Thus, the maximum ROS production is induced persistently in each oxidative phase of the redox cycle as a response to the production of H<sub>2</sub>S in the preceding reductive phase. Consistently, the <em>cys4</em> mutant yeast strain that is a partial loss-of-function deficiency in H<sub>2</sub>S synthesis lacks a metabolic cycle, supporting the relationship of mutual induction between H<sub>2</sub>S and ROS <a href="#pone.0025270-Chen1">[25]</a>.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>The actual limit in the buffering capacity of the reducing equivalents to protect the genome from mutation by ROS is similar to the limited fidelity of other biochemical reactions, and it has been specifically demonstrated by decreasing the reductive power and increasing ROS in a yeast strain with a deletion of <em>TSA1</em> and/or <em>TSA2</em> genes that code the Thioredoxin peroxidases. The mutant strains fail to adequately neutralize ROS, which results in a mutator phenotype with as much as 2000% higher rate of mutation than the wild type strain <a href="#pone.0025270-Wong1">[27]</a>. Yeast cells that have the <em>TSA1</em> gene deleted are specifically sensitive to oxidative stress when the mitochondrial function is inhibited <a href="#pone.0025270-Demasi1">[28]</a>, as is persistently repeated in the redox cycle by H<sub>2</sub>S <a href="#pone.0025270-Kwak1">[21]</a>. In general, blocking the electron transport chain by metabolites, such as H<sub>2</sub>S, results in ROS production <a href="#pone.0025270-Kwak1">[21]</a>. Thus, a consequence of the redox cycle can be single-base-pair mutations caused by oxidative damage. Additionally, even large-scale genome instabilities can be the result of oxidative damage, and consistently, such large-scale genome changes can also result from a defect in the repair of abasic sites caused by oxidation, or as a consequence of imbalance in the nucleotide pool, rather than necessarily be independent events <a href="#pone.0025270-Ragu1">[29]</a>–<a href="#pone.0025270-Kumar2">[34]</a>. Remarkably, oxidation is an intrinsically periodic product of metabolism that is fundamental for adaptation of organisms because modulation of its amplitude beyond the buffering capacity of the reductive phase by resonance with the dynamically changing environment can result in changes of the genome sequence that can specify persistence <em>vs.</em> extinction.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>Previously, circumstantial evidence for an environmentally responsive and simultaneously adaptive nature of mutation has been suggested <a href="#pone.0025270-Rosenberg1">[35]</a>–<a href="#pone.0025270-Davis1">[37]</a> and actively debated <a href="#pone.0025270-Charlesworth1">[38]</a>. A hierarchical, yet stochastic, and complex regulation of multiple cellular timing mechanisms had been suggested for adaptation without a heritable component <a href="#pone.0025270-Acar1">[39]</a>. In contrast, we present systematic evidence that the cellular redox cycle enables a simple, deterministic and heritable mechanism of phenotypic adaptation in response to an arbitrary challenge by the environment, consistently with the Le Chatelier's principle.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>To determine whether or not the universal feedback loop between H<sub>2</sub>S and ROS enables variation of the genome sequence and adaptation, the yeast genome and transcriptome were examined according to various self-consistent and coherent criteria correlated at discrete time intervals of the redox cycle to determine their phase of variation relative to the oxidized and reduced states. The chemically reactive potential generated by the oxidative phase of the redox cycle is at its maximum during the transition between high and low residual dissolved oxygen, and is at its minimum during the transition between low and high accumulated dissolved hydrogen sulfide <a href="#pone.0025270-Lloyd1">[1]</a>, <a href="#pone.0025270-Lloyd2">[2]</a>. Specifically, we examined codon bias, i.e. codon adaptation index (CAI) <a href="#pone.0025270-Sharp1">[40]</a> (<a href="#pone-0025270-g001"><strong>Fig. 1A</strong></a>), variations of genome-wide RNA transcript levels, the number of genes (<a href="#pone-0025270-g002"><strong>Fig. 2A</strong></a>, <a href="#pone-0025270-g002"><strong>Fig. 2B</strong></a>, and <strong><a href="#pone.0025270.s006">fig. S6</a></strong>), and the frequency of sequence variation between regions of the <em>S. cerevisiae</em> genome transcribed in the oxidative <em>vs.</em> reductive phase of the redox cycle (<a href="#pone-0025270-g003"><strong>Fig. 3</strong></a><strong> and <a href="#pone.0025270.s004">fig. S4</a></strong>). We also examined the degree of adaptation in the <em>S. cerevisiae</em> genome by analysis of phylogenetic sequence variation in all known yeast genes (<a href="#pone-0025270-g001"><strong>Fig. 1B</strong></a><strong>. and <a href="#pone.0025270.s003">fig. S3</a></strong>).</p>
<div class="figure" id="pone-0025270-g001"><div class="img"><a name="pone-0025270-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0025270" data-uri="info:doi/10.1371/journal.pone.0025270.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0025270.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0025270.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Yeast codons and phylogenetic conservation biased by the redox state.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>The relationship between dissolved oxygen (blue curve), hydrogen sulfide (red curve) (from reference 2), codon adaptation index (CAI, black line), phylogenetic protein-coding sequence similarity, the number of genes expressed during the redox cycle, and Spearman's correlation between CAI and log-normalized RNA expression during the redox cycle of yeast <em>S. cerevisiae</em>. Yellow color (time points 1–4) is the oxidative phase, and the blue color (time points 5–12) is the reductive phase. Boxes show median values with statistical significance with notches, (if the notches in the two boxes do not overlap, then it is a ‘strong evidence’ that their medians differ <a href="#pone.0025270-Chambers1">[60]</a>, first quantile (25%) and third quantile (75%); whiskers indicate minimum and maximum values. <strong>A.</strong> The oscillation of CAI values matching the period of the fluctuations in the oxidation state. <strong>B.</strong> The oscillation of yeast phylogenetic sequence similarity matching the period of the fluctuations in the oxidation state. Protein sequence conservation in <em>Saccharomyces sensu stricto</em> clade is graphed relative to the <em>S. cerevisiae</em> redox cycle.</p>
<span>doi:10.1371/journal.pone.0025270.g001</span></div><div class="figure" id="pone-0025270-g002"><div class="img"><a name="pone-0025270-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0025270" data-uri="info:doi/10.1371/journal.pone.0025270.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0025270.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0025270.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Transcript amplitudes and CAI correlate, and the numbers of expressed genes oscillate with the redox state.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p><strong>A.</strong> The oscillation of Spearman's correlation between CAI values and log-normalized RNA expression shows that a stronger correlation matches the oxidative phase. <strong>B.</strong> The oscillation in the number of genes expressed during the redox cycle. The number of essential genes (right panel) peak during the oxidative phase and the number of non-essential genes (left panel) peak during the reductive phase.</p>
<span>doi:10.1371/journal.pone.0025270.g002</span></div><div class="figure" id="pone-0025270-g003"><div class="img"><a name="pone-0025270-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0025270" data-uri="info:doi/10.1371/journal.pone.0025270.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0025270.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0025270.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Genome-wide frequency of sequence variation biased differentially in gene structure elements by the redox state.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>The relationship between dissolved oxygen (blue dots), hydrogen sulfide (red dots) (from reference 2), and the frequency of sequence variation during the redox cycle of yeast <em>S. cerevisiae</em>. The opposite phased oscillations of the frequency of sequence variation values (black lines) in ORFs (<strong>A</strong>), and introns (<strong>B</strong>), match the period of the fluctuations in the redox states. The frequency of sequence variation is based on 101,343 distinct segregating sites of SNPs from 63 ecologically and geographically diverse types of laboratory-derived and wild type yeast strains.</p>
<span>doi:10.1371/journal.pone.0025270.g003</span></div>
<h4>Resonance of the yeast transcriptome synthesis with the redox cycle</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We analyzed yeast <em>S. cerevisiae</em> data from the transcriptional redox attractor cycle (TRAC) <a href="#pone.0025270-Li1">[10]</a>, and transcriptome data from similarly oscillating respiratory bursts during the yeast metabolic cycle (YMC) <a href="#pone.0025270-Tu3">[24]</a>. We identified individual open reading frames (ORFs) that were expressed periodically and aperiodically during each measured time interval in the redox cycle of both data sets (<a href="#pone.0025270.s013">Supplemental Data S1</a>). We treated the microarray data from the short (TRAC) and long-period (YMC) respiratory oscillations separately to identify agreement and disagreement between the datasets for each open reading frame (ORF) individually (<a href="#pone.0025270.s013">Supplemental Data S1</a>). Our method of analysis, described in <a href="#pone.0025270.s014">Appendix S1</a>, revealed a significantly greater degree of coherence between the two datasets than was previously reported. Our results resolve the issue about which genes peak at what phase of the cycle in favor of phase coherence in gene expression between the two datasets despite the difference in the length of the cycle periods.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Periodic ORFs oscillating with the redox cycle composed 3,614 out of 4,231 ORFs (~85.4%) having consistent expression between the two data sets <a href="#pone.0025270-Li1">[10]</a>, <a href="#pone.0025270-Tu3">[24]</a>, while 617 genes (~14.6%) were non-periodic (see the <a href="#s4">Methods</a> section for how the consistency of expression and periodicity was defined). The number of non-essential genes was 3,217 (~76.0%) and number of essential genes was 1,014 (~24.0%). Most non-essential genes are transcribed during the reductive phase, while most of the essential genes are transcribed in the oxidative phase (<a href="#pone-0025270-g002"><strong>Fig. 2B</strong></a>), resulting in separate consideration.</p>


<h4>Resonance of the yeast codon bias with the redox cycle</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>ORFs transcribed at the time when the chemically reactive potential generated by the oxidative phase of the redox cycle is at its maximum, possess maximum bias (<a href="#pone-0025270-g001"><strong>Fig. 1A</strong></a>). Conversely, the minimum bias is encoded in ORFs transcribed at the time when the chemically reactive potential generated by the oxidative phase is at its minimum. CAI values at synonymous sites of <em>S. cerevisiae</em> are on average ~0.03 higher in the oxidative phase than in the reductive phase (<em>p</em>-value&lt;2.2×10<sup>−16</sup>, one-sided Wilcoxon rank sum test). This corresponds to ~9% increase in the mean CAI value for genes transcribed during the oxidative phase over the mean CAI value of genes transcribed during the reductive phase. The values given for the second cycle were chosen as representative. Similar oscillations were observed in all three cycles, <strong><a href="#pone.0025270.s010">Table S1</a></strong>. The increase in CAI values indicates that the two different frequencies of sequence variation at synonymous codons of genes transcribed in the oxidative and reductive phases are not equal, nor arbitrary.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The average CAI values for essential genes are ~0.03 higher than for non-essential genes since the number of essential genes peaks at the oxidative phase (<a href="#pone-0025270-g002"><strong>Fig. 2B</strong></a>), reflecting the difference in CAI values observed between the genes transcribed in the oxidative <em>vs.</em> reductive phase. Despite the differences in the pattern of gene number oscillation between the essential and non-essential genes, i.e. the number of essential genes peak during the oxidative phase and the number of non-essential genes peak during the reductive phase, the CAI value oscillation and oscillations of RNA expressions are the same for both groups.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Moreover, the CAI values are biased in genes expressed during the oxidative phase of the cycle, despite the fact that the relative expression levels of genes are similar or even higher in the reductive phase of the redox cycle <a href="#pone.0025270-Klevecz1">[7]</a>. This result shows that the level of transcription does not drive sequence variation independently of the redox phase. A higher rate of sequence variation may at first suggest that the CAI values would loose bias, because biased codons represent only a single option out of several codon redundancy options that are likely produced by the variation of sequence through the repair of oxidized bases. In reality, the persistent variation of sequence by repair of oxidized bases maintains codon bias when combined with selection. For example, a comparison of CAI values for genes encoding the <em>S. cerevisiae</em> cytoplasmic and mitochondrial ribosomes shows that CAI values are biased by the oxidative phase of the redox cycle, and not by some other independent factor. Specifically, genes encoding the cytoplasmic ribosome subunits are co-transcribed in the oxidative phase and have expression levels similar to genes that encode the mitochondrial ribosome, which, in contrast, is encoded by genes that are co-transcribed in the reductive phase of the redox cycle <a href="#pone.0025270-Tu3">[24]</a>. Yet, despite the similar levels of gene expression between the two different types of ribosomes-coding genes, the cytoplasmic ribosome has more than double the average CAI value of the mitochondrial ribosome, i.e. 0.761 <em>vs.</em> 0.297. This result clearly demonstrates that CAI values are biased by the oxidative phase, and not by transcription level independent of the redox phase in which the corresponding genes are transcribed. Consistently with the CAI bias, the average amino acid sequence similarity between the yeast and human ribosomes is 60% in the cytoplamsic ribosome <em>vs.</em> 33% in the mitochondrial ribosome. The least conserved yeast cytoplasmic ribosome protein, Rpp1B, has 37% sequence similarity with its human orthologue, and the most conserved yeast cytoplasmic ribosome protein, Rpl40B, has 90% sequence similarity with its human orthologue. In contrast, the least conserved yeast mitochondrial ribosome protein, Mrpl3, has 20% sequence similarity with its human orthologue, and the most conserved yeast protein, Mrps12, has 55% sequence similarity.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Remarkably, the correlation between the amplitude of genome-wide RNA transcripts and CAI values also oscillates with the redox cycle (Spearman's rank method, <em>p</em>-value&lt;10<sup>−10</sup> for all 36 time points for both essential and non-essential genes, <a href="#pone-0025270-g002"><strong>Fig. 2A</strong></a>). The correlation is stronger during the oxidative phase and relaxes during the reductive phase, suggesting that CAI values during the oxidative phase are more representative of gene transcription levels, consistent with a higher RNA turnover in the oxidative environment. <strong><a href="#pone.0025270.s009">Figure S9</a></strong> shows the result of modeling the relationship between CAI values and the total log-normalized RNA expression level in the oxidative phase for each gene in <em>S. cerevisiae</em>.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Additionally, because RNA turnover is higher in the oxidative <em>vs.</em> reductive phase of the redox cycle, it was verified that the rate of RNA synthesis alone does not have a role in differentiating CAI values for the two types of ribosomes-coding genes independently of the redox phases. The rates of yeast mRNA synthesis, determined previously <a href="#pone.0025270-Miller1">[41]</a>, were compared for all the ribosomal genes to show that the turnover of the mRNAs that code for the cytoplasmic and mitochondrial ribosomes are the same despite the fact that the induced rate of mRNA synthesis on average is two times higher in the oxidative phase for the cytoplasmic ribosome-coding ORFs compared to the reductive phase for the mitochondrial ribosome-coding ORFs. However, this result is due to an equivalent difference in the length of the oxidative <em>vs.</em> reductive phase during which the respective genes are transcribed. Specifically, the duration of the oxidative phase is half of the reductive phase, and this difference also accounts for the same maximum amplitude of RNA transcripts for the two different ribosomes, which are expressed exclusively in the opposite phases of the redox cycle. Consistent with the product of duration and rate of synthesis for both types of ribosomes, mRNA transcripts of the cytoplasmic ribsosome accumulate as short duration sharp burst peaks exclusively during the oxidative phase, whereas all mRNA transcripts for the mitochondrial ribosome accumulate as broad plateau peaks exclusively during the reductive phase.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>To further investigate the differences in CAI values between ORFs transcribed in the oxidative <em>vs.</em> reductive phases, we examined the base identity at the third positions within synonymous codons for all ORFs. The identity of the third bases in codons of all periodically expressed ORFs oscillates with the redox cycle, and it is biased against guanine (G) and adenine (A), i.e. purine bases, and for thymine (T) and cytosine (C), i.e. pyrimidine bases, by ~1% with all <em>p</em>-values&lt;10<sup>−3</sup> (<strong><a href="#pone.0025270.s008">fig. S8</a></strong>). A higher depurination in the oxidative phase is consistent with an oxidative damage to the DNA <a href="#pone.0025270-Eide1">[42]</a>. Moreover, repair of oxidative DNA damage <em>in vivo</em> results in transversions, which change the chemical nature of the base, thus enabling maximum mutagenic potential and the most rapid genetic variation possible. Interestingly, despite adenine being most efficiently inserted when abasic sites are bypassed by DNA polymerases, it is clear that the fidelity of translation is selected over the enzymatic specificity for purines.</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>Additionally, we examined CAI values in <em>Saccharomyces sensu stricto</em> clade (<em>S. paradoxus, S. mikatae, S. kudriavzevii, S. bayanus, S. castellii, S. kluyvery</em>) genes and in human protein-coding genes that are similar in their amino acid sequence coding potential to <em>S. cerevisiae</em> ORFs (Blast <em>e</em>-value&lt;10<sup>−50</sup>) (Supporting online material, <strong><a href="#pone.0025270.s001">fig. S1</a>, <a href="#pone.0025270.s002">S2</a>, <a href="#pone.0025270.s010">Table S1</a></strong>). The results show a periodic oscillation of CAI values for orthologous ORFs in other species of yeast as well as in human ORFs.</p>
<a id="article1.body1.sec2.sec2.p8" name="article1.body1.sec2.sec2.p8"></a><p>The frequency of codon bias is enumerated between two classes of transcribed regions defined non-randomly by temporal segregation between incompatible functions of genes in the redox cycle within an individual organism, as well as, consistently, in different species due to the universal nature of the redox cycle in living cells. Thus, the CAI bias is an intrinsic consequence of iteration between heredity and the environment for all cells, and it is consistent with the observation that the redox cycle is independent of either population dynamics such as synchronization among cells or external limitation of growth by the carbon source <a href="#pone.0025270-Silverman1">[43]</a>.</p>
<a id="article1.body1.sec2.sec2.p9" name="article1.body1.sec2.sec2.p9"></a><p>Also, while the tRNA pool contains differential amounts of isoacceptors, i.e. tRNAs with different anticodons but incorporating the same amino acid in protein synthesis, each tRNA isoacceptor amount matches its condon bias, as was shown for yeast and <em>E. coli</em> <a href="#pone.0025270-Andersson1">[44]</a>, <a href="#pone.0025270-Dong1">[45]</a>. Therefore, the oscillating CAI bias among all ORFs is a result of the relative timing of expression of mRNAs in the redox cycle, rather than biased by different amounts of the tRNA isoacceptors, which accumulate proportionally to the CAI bias.</p>


<h4>Resonance of the yeast phylogenetic conservation with the redox cycle</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>A pair wise sequence similarity comparison between <em>S. cerevisiae</em> ORFs and all ORFs in the <em>Saccharomyces sensu stricto</em> clade matching the period of the redox cycle revealed that, remarkably, the sequence similarity also oscillates with the redox cycle. This result indicates that genes of other yeast species are likely similarly organized in their expression relative to the <em>S. cerevisiae</em> redox phases because variation of homologous sequences is maximal for genes expressed in the oxidative phase, when ROS reaches maximum value <a href="#pone.0025270-Kwak1">[21]</a>. The cycle is slightly shifted; presumably due to variation in the time of expression for orthologous ORFs in <em>S. cerevisiae</em> relative to the other yeast species (<a href="#pone-0025270-g001"><strong>Fig. 1B</strong></a> and <strong><a href="#pone.0025270.s011">Table S2</a></strong>). The sequence similarity on average is ~1.0% higher in the oxidative phase compared to the reductive phase (<em>p</em> -value&lt;2.2×10<sup>−16</sup>, one-sided Wilcoxon rank sum test), and is ~3.0% higher in essential genes <em>vs.</em> non-essential genes. The conserved sequences are an indication of functional importance, and are expected to be under a stronger purifying and/or fixing selection in non-synonymous sites during the oxidative <em>vs.</em> the reductive phase. A similar trend holds for a pair wise sequence similarity of human genes that are similar in amino acid sequence to <em>S. cerevisiae</em> non-essential genes (~1% higher in oxidative phase, <em>p</em>-value&lt;10<sup>−4</sup>, one sided Wilcoxon rank sum test), however, for human genes that are similar in amino acid sequence to <em>S. cerevisiae</em> essential ORFs, there is no significant difference in variation of sequence conservation between oxidative and reductive phases (<strong><a href="#pone.0025270.s002">fig. S2</a></strong> and <strong><a href="#pone.0025270.s011">Table S2</a></strong>).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>These results may seem counterintuitive because a higher rate of sequence variation would imply a lower potential for sequence conservation, but this would only be true in the absence of selection. In reality, amino acid sequence conservation and CAI bias both result from a high rate of sequence variation that is restrained by selection of orthologous biological function.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Should chance alone be the mutation frequency determinant, there would be no basis for periodic oscillation of the phylogenetic sequence similarities, or oscillation and bias of CAI values in phase with the <em>S. cerevisiae</em> redox cycle. These results indicate that the fidelity of translation and biological functions are both optimized by genetic variation induced through RNA transcription in addition to DNA replication rather than through variation of the genetic code by the imperfect fidelity of DNA replication alone. Interestingly, the CAI bias is also present in non-essential ORFs, clearly indicating that their functions are non-trivial and actively optimized by selection, as previously detected in a phenotypic assay that revealed a selection pressure on non-essential ORFs <a href="#pone.0025270-Thatcher1">[46]</a>.</p>


<h4>Non-random, RNA transcription-induced frequency of sequence variation in the yeast genome</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We examined the difference in the experimentally determined frequency of sequence variation between genes transcribed in the oxidative and reductive phases of the cycle and estimated the frequency of sequence variation induced by RNA transcription in yeast directly (<a href="#pone-0025270-t001"><strong>Table 1</strong></a>). This was done by comparing the frequency of sequence variation among ORFs, introns, and un-translated regions (UTRs), i.e. in both 5′UTRs and 3′UTRs <a href="#pone.0025270-SGD1">[47]</a> transcribed in the oxidative and reductive phases of the redox cycle. In addition, we compared the frequency of sequence variation in transcribed DNA (ORFs, introns, 5′ and 3′UTRs) with the estimated frequency of sequence variation in un-transcribed regions <a href="#pone.0025270-Samanta1">[48]</a>. We used previously mapped whole-genome single nucleotide polymorphisms (SNPs) in non-repetitive regions that were determined using tiling microarrays for 63 ecologically and geographically diverse <em>S. cerevisiae</em> strains <a href="#pone.0025270-Schacherer1">[49]</a>. The most comprehensively validated SNP dataset that contains 101,343 distinct segregating sites mapped in wild type yeast genotypes was used because the yeast metabolic cycle is an intrinsic property of yeast metabolism and does not depend on either synchronization or external limitation of growth by the carbon source <a href="#pone.0025270-Silverman1">[43]</a>.</p>
<div class="figure" id="pone-0025270-t001"><div class="img"><a name="pone-0025270-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0025270" data-uri="info:doi/10.1371/journal.pone.0025270.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0025270.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0025270.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0025270.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0025270.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0025270.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Table of the mean frequencies of sequence variation (in ORFs, introns, 5′UTRs and 3′UTRs) per nucleotide for the oxidative and reductive phases.</span></strong></p><span>doi:10.1371/journal.pone.0025270.t001</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p><a href="#pone-0025270-t001"><strong>Table 1</strong></a> is a summary for the frequency of sequence variation per nucleotide of genes expressed periodically at the oxidative or the reductive phases of the <em>S. cerevisiae</em> redox cycle over the unknown period of time required for divergence of the 63 yeast strains. The observed frequencies of sequence variation within ORFs, introns, 5′UTRs and 3′UTRs persistently oscillate in phase with the redox cycle and are the result of appropriate interplay between sequence variation and selection, where the effect of selection is believed to be the strongest in ORFs, and nearly neutral in introns, 5′UTRs and 3′UTRs. Consistently, in the protein-coding portions of genes where selection is expected to play the biggest role for adaptation, the frequency of sequence variation has the lowest value, indicating a limited potential for productive mutation (<a href="#pone-0025270-g003"><strong>Fig. 3A</strong></a> and <strong><a href="#pone.0025270.s012">Table S3</a></strong>). While synonymous codons (i.e. CAI values) can be positively biased, the negative bias against non-synonymous codons reveals a fundamental restriction in the potential for variation of amino acid identity. This result indicates a non-trivial limitation for gradualism as a mechanism of adaptation to the dynamic environment by speciation.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The frequency of sequence variation is anti-correlated with CAI values in ORFs and 3′UTRs, and positively correlated in introns and 5′UTRs, consistent with sequence variation being mostly deleterious to the function of coding sequences <a href="#pone.0025270-Doniger1">[50]</a>–<a href="#pone.0025270-Wilkie1">[52]</a> (<a href="#pone-0025270-g003"><strong>Fig. 3</strong></a> and <strong><a href="#pone.0025270.s004">fig. S4</a></strong>). The frequency of sequence variation in 3′UTRs, which encode structured RNA-recognition sites for protein binding, <em>e.g.</em> mitochondrial targeting sequences <a href="#pone.0025270-SaintGeorges1">[53]</a>, is anti-correlated with the oxidative phase, indicating a negative selection against variation, while the 5′UTRs do not show this trend. Consistent with this difference, 3′UTRs overlap adjacent ORF-coding regions, which are anti-correlated with the sequence variation frequency, while the 5′UTRs do not show this trend. In addition, a diminishing gradient in the frequency of sequence variation along the gene lengths is consistent with a reduction in processivity of RNA transcription elongation. Unfailingly, the occupancy of RNA Polymerase II along gene lengths has been mapped on a genome-wide scale in yeast and shown to decrease from the transcription start sites at the 5′ end toward the 3′ end of genes. An apparent increase in occupancy of RNA Polymerase II at the polyadenylation sites near the 3′ end of genes was also consistently mapped due to the overlap of 3′UTRs with adjacent ORF-coding regions <a href="#pone.0025270-Miller1">[41]</a>. Thus, the failure of the transcription complex to translocate successfully to the 3′ end results in a higher frequency of sequence variation in the 5′ UTRs than in the 3′ UTRs, as detected (<a href="#pone-0025270-t001"><strong>Table 1</strong></a><strong>, <a href="#pone.0025270.s012">Table S3</a></strong>, and <strong><a href="#pone.0025270.s004">fig. S4</a></strong>).</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>The analysis of the difference in the frequency of sequence variation between oxidative and reductive states among 158 out of 308 known <em>S. cerevisiae</em> intron-containing genes <a href="#pone.0025270-SGD1">[47]</a>, which are presumed to be neutral to selection in their intron-coding sequences <a href="#pone.0025270-Fedorov1">[54]</a>, revealed that intron sequences transcribed during the oxidative phase contained 39.0% higher mean frequency of sequence variation per nucleotide over the 63 strain divergence time than introns transcribed during the reductive phase of the cycle. The known occurrence of yeast intron sequences that can be affected by selection is limited to a few bases in comparison with the total length of yeast introns, i.e. 66,180 bases in ORF-containing introns, and therefore does not affect the general assumption that intron sequences are neutral to the effect of selection. For example, the intron encoded polypyrimidine tract and a single adenosine in the branch point of introns, in addition to a single small RNA (snR191) within the intron of an ORF called YNR053C, contribute less than a few percent of selection-affected bases to the total number of considered bases. In comparison to introns, the difference between the frequency of sequence variation in 5′UTRs of genes transcribed in the oxidative <em>vs.</em> the reductive phase was 15.2%. The lower frequency of sequence variation in 5′UTRs <em>vs.</em> introns is consistent with a higher content of selectable sequence encoded within 5′UTRs by upstream ORFs, i.e. uORFs <a href="#pone.0025270-Selpi1">[55]</a>, <a href="#pone.0025270-Nagalakshmi1">[56]</a>. Because uORFs encode amino acid identity, the effect of selection is against their variation. Thus, the frequency of sequence variation is higher in the oxidative <em>vs.</em> the reductive phase in nearly neutral sites, confirming RNA transcription as a templating mechanism that increases the mutation potential among genes transcribed in the oxidative <em>vs.</em> reductive phase of the redox cycle.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>Additionally, both the number and lengths of the selection-neutral genetic elements and ORFs oscillate during the redox cycle, matching the period of the fluctuations in the oxidation state (<strong><a href="#pone.0025270.s005">fig. S5</a> and <a href="#pone.0025270.s007">fig. S7</a></strong>). This result is consistent with propagation of the length and number of genetic elements by the repair of their oxidized bases, which can lead to base extensions at the damaged sites.</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>Finally, we estimated the difference in the frequencies of sequence variation between transcribed DNA (ORFs, introns, 5′ and 3′UTRs) and un-transcribed DNA. The coordinates of the un-transcribed regions were identified by incorporating data from 36-mer oligonucleotides mapped to regions with nearly absent expression levels <a href="#pone.0025270-Samanta1">[48]</a> and checking for overlap with non-coding regions <a href="#pone.0025270-Doniger1">[50]</a>, absence of coding elements <a href="#pone.0025270-SGD1">[47]</a>, and absence of 5′ and 3′UTRs <a href="#pone.0025270-Nagalakshmi1">[56]</a>. In total, out of nearly 400,000 unique 36-mers tiled across the genome, 584 specific 36-mers were identified as potential un-transcribed regions, and an upper bound frequency of sequence variation of 2.50×10<sup>−3</sup>±5.18×10<sup>−3</sup>(SD, N = 584) per nucleotide was derived for the un-transcribed DNA over the divergence time of the 63 strains. Based on this estimate, the ratio of the frequencies of sequence variation between transcribed introns, i.e. selection nearly neutral DNA, and un-transcribed DNA is at least 1.26(C.I. 95%, 0.98±1.62, N = 101) for essential genes transcribed during the oxidative phase <em>vs.</em> 0.90(C.I. 95%, 0.69±1.18, N = 111) for essential genes transcribed in the reductive phase. The same ratio for the frequency of sequence variation in introns of non-essential genes is at least 1.01(C.I. 95%, 0.82±1.25, N = 212) for genes transcribed in the oxidative phase <em>vs.</em> 0.71(C.I. 95%, 0.56±0.89, N = 233) for genes transcribed in the reductive phase. Thus, the estimated ratios indicate that RNA transcription-induced variation of sequence significantly increases the total rate of mutation.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In contrast to the common presumption that the passage of time alone serves as a template for random variation of heredity, we showed that the variation of genetic sequence is specified persistently and non-randomly by the temporal resonance of RNA transcription with the binary chemical state of the redox cycle. The rate of genetic variation by RNA transcription persistently increases the total rate of mutation and oscillates asymmetrically with the redox cycle among genes. Moreover, the measured RNA transcription-induced frequency of sequence variation (<a href="#pone-0025270-t001"><strong>Table 1</strong></a>) is an underestimation because (1) determining the frequency of sequence variation in un-transcribed regions is challenging due to their scarcity in <em>S. cerevisiae</em> genome, and (2) grouping of individual time points in the two phases of the redox cycle introduces dilution of the relative values at the points of chemical extrema. Additionally, negative selection reduces the observable frequencies of sequence variation at the two phases of the redox cycle. Importantly, a random-based asymmetry in the rate of sequence variation would result in a constant variation among genome-wide transcribed regions relative to the period of the redox cycle, rather than oscillate in phase with the timing of the two phases of the redox cycle. Because time is an essential non-random variable of the information content and processing functions in cellular systems, the redox cycle clearly provides a temporal template for asymmetry in the genome-wide rate of sequence variation.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>A biochemical mechanism that can account for the increased total frequency of transcription-induced sequence variation over the replication-based mutation is the asymmetric chemical reactivity between the transcribed and replicated DNA strands. Specifically, the confinement of a single round of replication to the reductive phase of the cycle is protective against variation of sequence by base oxidation <a href="#pone.0025270-Klevecz1">[7]</a>, <a href="#pone.0025270-Tu3">[24]</a>, while multiple rounds of genome-wide RNA transcription between cell divisions oscillate in the chemical environment of the reactive and unstable products of the oxidizing phase, which results in non-uniform and varied frequency of mutation across the genome. Additionally, it is important to recognize that the periodic oscillation in the frequency of sequence variation for all relevant genetic elements is appropriately in agreement with the central dogma of molecular biology. In contrast to the reality of our results, a random process as previously contemplated by others could only generate aperiodic or irregular values for the same variables of heredity. Moreover, the paradox of biological adaptation by means of such supposedly passive and random events is <em>a priory</em> inconsistent, because it is incorrect to assume that a random process can generate asymmetry without a template, especially for a periodic asymmetry of the redox cycle. Perhaps because the environment is dynamic, it can be non-obvious that the environment varies periodically on a number of different time-scales and its impact is matched by responsively dynamic changes in cellular functions according to the Le Chatelier's principle. The temporally ordered biochemical events and asymmetry of the redox cycle chemistry on DNA <em>vs.</em> RNA provide the non-random basis for a dynamic heredity in response to the environment, consistently with the Le Chatelier's principle.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The concept of adaptation through use and disuse of characteristics, whereby acquired variation outweighs inherited variation was admitted by Darwin <a href="#pone.0025270-Mayr1">[57]</a>. However, Gregor Mendel's subsequent definition of static genetic inheritance <a href="#pone.0025270-Mendel1">[58]</a> supplanted the dynamic notion of inheritance of acquired traits <a href="#pone.0025270-Burkhardt1">[59]</a>. Our results may fundamentally change the common understanding about the underlying mechanism at work in adaptation and persistence of organisms. While the basis of the observed genetic and phenotypic discontinuity is broadly attributed to a continuum of evolution by passive accumulation of chance mutation, i.e. random alternations followed by selection and determination of a choice, we present evidence for acquired genetic variation that is actively induced by the environment between generations. The discrete and iterative feedback between the environment and heredity is mediated by the redox cycle, which may serve as a template mechanism for spontaneously acquired adaptation and speciation of organisms toward persistence in a dynamically changing environment.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Computational protocols and statistical methods of analysis used for evaluating microarray data, calculations of periodicity, codon usage bias, amino acid sequence similarity, and the frequency of sequence variation are described in the Supporting materials index.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0025270"></div><a name="pone.0025270.s001" id="pone.0025270.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI"><strong>The relationship between dissolved oxygen (blue curve), hydrogen sulfide (red curve) (from reference 1), codon adaptation index (CAI, black line) of </strong><strong><em>sensu stricto</em></strong><strong> clade genes (</strong><strong><em>S. paradoxus, S. mikatae, S. kudriavzevii, S. bayanus, S. castellii, S. kluyvery</em></strong><strong>) similar in amino acid sequences to </strong><strong><em>S. cerevisiae</em></strong><strong> ORFs expressed periodically during the redox cycle.</strong> Yellow color (time points 1–4) is the oxidative phase, and the blue color (time points 5–12) is the reductive phase. Boxes show median values with statistical significance with notches (if two boxes' notches do not overlap this is a ‘strong evidence’ that their medians differ <a href="#pone.0025270-Lloyd2">[2]</a>), first quantile (25%) and third quantile (75%); whiskers indicate minimum and maximum values.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s002" id="pone.0025270.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI"><strong>The relationship between codon adaptation index (CAI, black line) for </strong><strong><em>H. sapiens</em></strong><strong> genes that are similar in amino acid sequence to </strong><strong><em>S. cerevisiae</em></strong><strong> genes expressed periodically in the redox cycle.</strong> Yellow color (time points 3–6) is the suggested oxidative phase, and the blue color (time points 1–2 and 7–12) is the suggested reductive phase. Boxes show median values with statistical significance with notches (if two boxes' notches do not overlap this is a ‘strong evidence’ that their medians differ <a href="#pone.0025270-Lloyd2">[2]</a>), first quantile (25%) and third quantile (75%); whiskers indicate minimum and maximum values.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s003" id="pone.0025270.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI"><strong>The relationship between sequence similarity of </strong><strong><em>H. sapiens</em></strong><strong> genes to </strong><strong><em>S. cerevisiae</em></strong><strong> genes periodically expressed in the redox cycle.</strong> Yellow color (time points 3–6) is the suggested oxidative phase, and the blue color (time points 1–2 and 7–12) is the suggested reductive phase. Boxes show median values with statistical significance with notches (if two boxes' notches do not overlap this is a ‘strong evidence’ that their medians differ <a href="#pone.0025270-Lloyd2">[2]</a>), first quantile (25%) and third quantile (75%); whiskers indicate minimum and maximum values.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s004" id="pone.0025270.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI"><strong>The relationship between dissolved oxygen (blue dots), hydrogen sulfide (red dots) (from reference 1), and the frequency of sequence variation during the redox cycle of yeast </strong><strong><em>S. cerevisiae</em></strong><strong>.</strong> The opposite phased oscillations for the frequency of sequence variation values in 5′UTRs and 3′UTRs (black lines) match the period of the fluctuations in the redox states.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s005" id="pone.0025270.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI"><strong>A. The relationship between number and length of introns, 5′ and 3′UTRs in </strong><strong><em>S. cerevisiae</em></strong><strong> genes and the redox cycle.</strong> Yellow color (time points 1–4) is the oxidative phase, and the blue color (time points 5–12) is the reductive phase. Boxes show median values with statistical significance with notches (if two boxes' notches do not overlap this is a ‘strong evidence’ that their medians differ <a href="#pone.0025270-Lloyd2">[2]</a>), first quantile (25%) and third quantile (75%); whiskers indicate minimum and maximum values. <strong>A.</strong> The oscillation of intron number and length matching the period of the fluctuations in the oxidation state. <strong>B. The oscillation of 5′UTR number and length matching the period of the fluctuations in the oxidation state.</strong> <strong>C. The oscillation of 3′UTR number and length matching the period of the fluctuations in the oxidation state.</strong></p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s006" id="pone.0025270.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI"><strong>The relationship between mean RNA transcript levels, total RNA transcript levels of </strong><strong><em>S. cerevisiae</em></strong><strong> genes and the redox cycle.</strong> Yellow color (time points 1–4) is the oxidative phase, and the blue color (time points 5–12) is the reductive phase. Boxes show median values with statistical significance with notches (if two boxes' notches do not overlap this is a ‘strong evidence’ that their medians differ <a href="#pone.0025270-Lloyd2">[2]</a>), first quantile (25%) and third quantile (75%); whiskers indicate minimum and maximum values.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s007" id="pone.0025270.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s007">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI"><strong>The relationship between ORF length and the redox cycle of </strong><strong><em>S. cerevisiae</em></strong><strong>. Yellow color (time points 1–4) is the oxidative phase, and the blue color (time points 5–12) is the reductive phase.</strong> Boxes show median values with statistical significance with notches (if two boxes' notches do not overlap this is a ‘strong evidence’ that their medians differ <a href="#pone.0025270-Lloyd2">[2]</a>), first quantile (25%) and third quantile (75%); whiskers indicate minimum and maximum values.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s008" id="pone.0025270.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s008">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI"><strong>The relationship between mean occurrence of a specific nucleotide in the third base in synonymous codons of </strong><strong><em>S. cerevisiae</em></strong><strong> genes and the redox cycle of </strong><strong><em>S. cerevisiae</em></strong><strong>.</strong> Yellow color (time points 1–4) is the oxidative phase, and the blue color (time points 5–12) is the reductive phase. Boxes show median values with statistical significance with notches (if two boxes' notches do not overlap this is a ‘strong evidence’ that their medians differ <a href="#pone.0025270-Lloyd2">[2]</a>), first quantile (25%) and third quantile (75%); whiskers indicate minimum and maximum values.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s009" id="pone.0025270.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s009">Figure S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI"><strong>The relationship between CAI values and total log-normalized RNA expression level in the oxidative phase (time points 1–4) for each gene in </strong><strong><em>S. cerevisiae</em></strong><strong>.</strong> Red points represent the measured values, and the black points represent the best non-linear fit.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s010" id="pone.0025270.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s010">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Table of mean CAI values for <em>S. cerevisiae</em>, <em>Saccharomyces sensu stricto</em> and <em>H. sapiens</em>, mean log-normalized RNA expressions, standard deviations, sample gene number, and two-tailed Wilcoxon <em>p</em>-values for oxidative and reductive phases of <em>S. cerevisiae</em> cycle.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s011" id="pone.0025270.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s011">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Table of mean pair wise phylogenetic sequence similarity values between <em>S. cerevisiae</em> and <em>Saccharomyces sensu stricto</em>, and between <em>S. cerevisiae</em> and <em>H. sapiens</em>, standard deviations, sample gene number, and two-tailed Wilcoxon <em>p</em>-values for oxidative and reductive phases of <em>S. cerevisiae</em> cycle.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s012" id="pone.0025270.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s012">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Table of mean frequencies of sequence variation per nucleotide over the time of divergence that resulted in the 63 strains (in ORFs, introns, 5′ and 3′UTRs), standard deviations, sample gene number, and two-tailed Wilcoxon <em>p</em>-values for oxidative and reductive phases of <em>S. cerevisiae</em> cycle.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0025270.s013" id="pone.0025270.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s013">Data S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Supplementary data file contains open reading frames (ORFs), agreement about periodicity of gene expression between TRAC and YMC data sets, the periodicity of RNA expression and its false discovery rate (fdr), and ORF viability designation.</p>
<p class="siDoi">doi:10.1371/journal.pone.0025270.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0025270.s014" id="pone.0025270.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0025270.s014">Appendix S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0025270.s014</p><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1"></a><p class="postSiDOI">(DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors acknowledge and thank Douglas B. Murray for helpful discussions, and for sharing his understanding of the redox cycle. The authors also thank Chris Webb and Jonah Harley for reviewing the manuscript. <em>SDG</em></p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: VS. Performed the experiments: VS AS. Analyzed the data: VS AS YG. Contributed reagents/materials/analysis tools: VS AS. Wrote the paper: VS AS YG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0025270-Lloyd1" id="pone.0025270-Lloyd1"></a>Lloyd D, Murray DB (2007) Redox rhythmicity: clocks at the core of temporal coherence. Bioessays  29: 465–473.  <ul class="find" data-citedArticleID="3272922" data-doi="10.1002/bies.20575"><li><a href="http://dx.doi.org/10.1002/bies.20575" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Redox+rhythmicity%3A+clocks+at+the+core+of+temporal+coherence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Redox+rhythmicity%3A+clocks+at+the+core+of+temporal+coherence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0025270-Lloyd2" id="pone.0025270-Lloyd2"></a>Lloyd D, Murray DB (2006) The temporal architecture of eukaryotic growth. FEBS Lett  580: 2830–2835.  <ul class="find" data-citedArticleID="3272918" data-doi="10.1016/j.febslet.2006.07.026"><li><a href="http://dx.doi.org/10.1016/j.febslet.2006.07.026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+temporal+architecture+of+eukaryotic+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+temporal+architecture+of+eukaryotic+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0025270-Falkowski1" id="pone.0025270-Falkowski1"></a>Falkowski PG, Fenchel T, Delong EF (2008) The microbial engines that drive Earth's biogeochemical cycles. Science  320: 34–39.  <ul class="find" data-citedArticleID="3272950" data-doi="10.1126/science.1153213"><li><a href="http://dx.doi.org/10.1126/science.1153213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+microbial+engines+that+drive+Earth%27s+biogeochemical+cycles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+microbial+engines+that+drive+Earth%27s+biogeochemical+cycles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0025270-Georgiou1" id="pone.0025270-Georgiou1"></a>Georgiou G (2002) How to Flip the (Redox) Switch. Cell  11: 607–610.  <ul class="find" data-citedArticleID="3272876" data-doi="10.1016/s0092-8674(02)01165-0"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)01165-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+to+Flip+the+%28Redox%29+Switch." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+to+Flip+the+%28Redox%29+Switch.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0025270-Hothorn1" id="pone.0025270-Hothorn1"></a>Hothorn M, Wachter A, Gromes R, Stuwe T, Rausch T, et al.  (2006) Structural basis for the redox control of plant glutamate cysteine ligase. J Biol Chem  281: 27557–2765.  <ul class="find" data-citedArticleID="3272894" data-doi="10.1074/jbc.m602770200"><li><a href="http://dx.doi.org/10.1074/jbc.m602770200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+the+redox+control+of+plant+glutamate+cysteine+ligase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+the+redox+control+of+plant+glutamate+cysteine+ligase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0025270-Toledano1" id="pone.0025270-Toledano1"></a>Toledano MB, Delaunay A, Monceau L, Tacnet F (2004) Microbial H<sub>2</sub>O<sub>2</sub> sensors as archetypical redox signaling modules. Trends Biochem Sci  29: 351–357.  <ul class="find" data-citedArticleID="3272938" data-doi="10.1016/j.tibs.2004.05.005"><li><a href="http://dx.doi.org/10.1016/j.tibs.2004.05.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+H2O2+sensors+as+archetypical+redox+signaling+modules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+H2O2+sensors+as+archetypical+redox+signaling+modules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0025270-Klevecz1" id="pone.0025270-Klevecz1"></a>Klevecz RR, Bolen J, Forrest G, Murray DB (2004) A genomewide oscillation in transcription gates DNA replication and cell cycle. Proc Natl Acad Sci U S A  101: 200–205.  <ul class="find" data-citedArticleID="3272956" data-doi="10.1073/pnas.0306490101"><li><a href="http://dx.doi.org/10.1073/pnas.0306490101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomewide+oscillation+in+transcription+gates+DNA+replication+and+cell+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomewide+oscillation+in+transcription+gates+DNA+replication+and+cell+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0025270-Boole1" id="pone.0025270-Boole1"></a>Boole George (1848) The Calculus of Logic, Cambridge and Dublin Mathematical Journal 3: 183–198.  <ul class="find" data-citedArticleID="3272862"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Calculus+of+Logic%2C+Cambridge+and+Dublin+Mathematical+Journal&amp;auth=&amp;atitle=The+Calculus+of+Logic%2C+Cambridge+and+Dublin+Mathematical+Journal" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Calculus+of+Logic%2C+Cambridge+and+Dublin+Mathematical+Journal" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Calculus+of+Logic%2C+Cambridge+and+Dublin+Mathematical+Journal%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0025270-Evans1" id="pone.0025270-Evans1"></a>Evans DJ, Searles DJ, Mittag E (2001) Fluctuation theorem for Hamiltonian systems - Le Chatelier's principle. Phys Rev E Stat Nonlin Soft Matter Phys  63(5 Pt 1): 051105.  <ul class="find" data-citedArticleID="3272892" data-doi="10.1103/physreve.63.051105"><li><a href="http://dx.doi.org/10.1103/physreve.63.051105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fluctuation+theorem+for+Hamiltonian+systems+-+Le+Chatelier%27s+principle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fluctuation+theorem+for+Hamiltonian+systems+-+Le+Chatelier%27s+principle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0025270-Li1" id="pone.0025270-Li1"></a>Li CM, Klevecz RR (2006) A rapid genome-scale response of the transcriptional oscillator to perturbation reveals a period-doubling path to phenotypic change. Proc Natl Acad Sci U S A  103: 16254–16259.  <ul class="find" data-citedArticleID="3272866" data-doi="10.1073/pnas.0604860103"><li><a href="http://dx.doi.org/10.1073/pnas.0604860103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+rapid+genome-scale+response+of+the+transcriptional+oscillator+to+perturbation+reveals+a+period-doubling+path+to+phenotypic+change." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+rapid+genome-scale+response+of+the+transcriptional+oscillator+to+perturbation+reveals+a+period-doubling+path+to+phenotypic+change.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0025270-Tu1" id="pone.0025270-Tu1"></a>Tu BP, McKnight SL (2009) Evidence of carbon monoxide-mediated phase advancement of the yeast metabolic cycle. Proc Natl Acad Sci U S A  106: 14293–14296.  <ul class="find" data-citedArticleID="3272936" data-doi="10.1073/pnas.0907786106"><li><a href="http://dx.doi.org/10.1073/pnas.0907786106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+of+carbon+monoxide-mediated+phase+advancement+of+the+yeast+metabolic+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+of+carbon+monoxide-mediated+phase+advancement+of+the+yeast+metabolic+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0025270-Finkel1" id="pone.0025270-Finkel1"></a>Finkel T, Holbrook NJ (2000) Oxidants, oxidative stress and the biology of ageing. Nature  408: 239–247.  <ul class="find" data-citedArticleID="3272906" data-doi="10.1038/35041687"><li><a href="http://dx.doi.org/10.1038/35041687" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oxidants%2C+oxidative+stress+and+the+biology+of+ageing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oxidants%2C+oxidative+stress+and+the+biology+of+ageing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0025270-Eldredge1" id="pone.0025270-Eldredge1"></a>Eldredge N, Gould SJ (1972) Punctuated equilibria: an alternative to phyletic gradualism In Schopf TJM, editor, Models in Paleobiology. San Francisco: Freeman Cooper. pp. 82–15.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0025270-Ward1" id="pone.0025270-Ward1"></a>Ward P (2006) Out of Thin Air: Dinosaurs, Birds, and Earth's Ancient Atmosphere. Washington, D.C.: Joseph Henry Press. 296 p.  <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0025270-Trefil1" id="pone.0025270-Trefil1"></a>Trefil J, Morowitz HJ, Smith E (2009) The Origin of Life. American Scientist  97: 206–213.  <ul class="find" data-citedArticleID="3272878" data-doi="10.1511/2009.78.206"><li><a href="http://dx.doi.org/10.1511/2009.78.206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Origin+of+Life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Origin+of+Life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0025270-Lang1" id="pone.0025270-Lang1"></a>Lang GI, Murray AW (2008) Estimating the per-base-pair mutation rate in the yeast <em>Saccharomyces cerevisiae</em>. Genetics  178: 67–82.  <ul class="find" data-citedArticleID="3272920" data-doi="10.1534/genetics.107.071506"><li><a href="http://dx.doi.org/10.1534/genetics.107.071506" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+the+per-base-pair+mutation+rate+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+the+per-base-pair+mutation+rate+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0025270-Bridges1" id="pone.0025270-Bridges1"></a>Bridges BA (1997) Microbial genetics: Hypermutation under stress. Nature  387: 557–558.  <ul class="find" data-citedArticleID="3272900"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Microbial+genetics%3A+Hypermutation+under+stress.&amp;auth=&amp;atitle=Microbial+genetics%3A+Hypermutation+under+stress." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+genetics%3A+Hypermutation+under+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+genetics%3A+Hypermutation+under+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0025270-Fehrenbach1" id="pone.0025270-Fehrenbach1"></a>Fehrenbach E, Northoff H (2001) Free radicals, exercise, apoptosis, and heat shock proteins. Exerc Immonol Rev  7: 66–89.  <ul class="find" data-citedArticleID="3272912"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Free+radicals%2C+exercise%2C+apoptosis%2C+and+heat+shock+proteins.&amp;auth=&amp;atitle=Free+radicals%2C+exercise%2C+apoptosis%2C+and+heat+shock+proteins." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Free+radicals%2C+exercise%2C+apoptosis%2C+and+heat+shock+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Free+radicals%2C+exercise%2C+apoptosis%2C+and+heat+shock+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0025270-Korogodin1" id="pone.0025270-Korogodin1"></a>Korogodin VI, Korogodina VL, Fajszi C, Chepurnoy AI, Mikhova-Tsenova N, et al.  (1991) On the dependence of spontaneous mutation rates on the functional state of genes. Yeast  7: 105–117.  <ul class="find" data-citedArticleID="3272886"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+the+dependence+of+spontaneous+mutation+rates+on+the+functional+state+of+genes.&amp;auth=&amp;atitle=On+the+dependence+of+spontaneous+mutation+rates+on+the+functional+state+of+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+dependence+of+spontaneous+mutation+rates+on+the+functional+state+of+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+dependence+of+spontaneous+mutation+rates+on+the+functional+state+of+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0025270-Wright1" id="pone.0025270-Wright1"></a>Wright BE (2000) A biochemical mechanism for nonrandom mutations and evolution. J Bacteriol  182: 2993–3001.  <ul class="find" data-citedArticleID="3272940" data-doi="10.1128/jb.182.11.2993-3001.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.11.2993-3001.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+biochemical+mechanism+for+nonrandom+mutations+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+biochemical+mechanism+for+nonrandom+mutations+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0025270-Kwak1" id="pone.0025270-Kwak1"></a>Kwak WJ, Kwon GS, Jin I, Kuriyama H, Sohn HY (2003) Involvement of oxidative stress in the regulation of H<sub>2</sub>S production during ultradian metabolic oscillation of <em>Saccharomyces cerevisiae</em>. FEMS Microbiology Letters  219: 99–104.  <ul class="find" data-citedArticleID="3272964" data-doi="10.1016/s0378-1097(02)01198-9"><li><a href="http://dx.doi.org/10.1016/s0378-1097(02)01198-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Involvement+of+oxidative+stress+in+the+regulation+of+H2S+production+during+ultradian+metabolic+oscillation+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Involvement+of+oxidative+stress+in+the+regulation+of+H2S+production+during+ultradian+metabolic+oscillation+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0025270-Murray1" id="pone.0025270-Murray1"></a>Murray DB, Beckmann M, Kitano H (2007) Regulation of yeast oscillatory dynamics. Proc Natl Acad Sci U S A  104: 2241–2246.  <ul class="find" data-citedArticleID="3272902" data-doi="10.1073/pnas.0606677104"><li><a href="http://dx.doi.org/10.1073/pnas.0606677104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+yeast+oscillatory+dynamics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+yeast+oscillatory+dynamics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0025270-Tu2" id="pone.0025270-Tu2"></a>Tu BP, Mohler RE, Liu JC, Dombek KM, Young ET, et al.  (2007) Cyclic changes in metabolic state during the life of a yeast cell. Proc Natl Acad Sci U S A  104: 16886–16891.  <ul class="find" data-citedArticleID="3272880" data-doi="10.1073/pnas.0708365104"><li><a href="http://dx.doi.org/10.1073/pnas.0708365104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cyclic+changes+in+metabolic+state+during+the+life+of+a+yeast+cell." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cyclic+changes+in+metabolic+state+during+the+life+of+a+yeast+cell.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0025270-Tu3" id="pone.0025270-Tu3"></a>Tu BP, Kudlicki A, Rowicka M, McKnight SL (2005) Logic of the yeast metabolic cycle: temporal compartmentalization of cellular processes. Science  310: 1152–1158.  <ul class="find" data-citedArticleID="3272890" data-doi="10.1126/science.1120499"><li><a href="http://dx.doi.org/10.1126/science.1120499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Logic+of+the+yeast+metabolic+cycle%3A+temporal+compartmentalization+of+cellular+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Logic+of+the+yeast+metabolic+cycle%3A+temporal+compartmentalization+of+cellular+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0025270-Chen1" id="pone.0025270-Chen1"></a>Chen Z, Odstrcil EA, Tu BP, McKnight SL (2007) Restriction of DNA replication to the reductive phase of the metabolic cycle protects genome integrity. Science  316: 1916–199.  <ul class="find" data-citedArticleID="3272864" data-doi="10.1126/science.1140958"><li><a href="http://dx.doi.org/10.1126/science.1140958" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Restriction+of+DNA+replication+to+the+reductive+phase+of+the+metabolic+cycle+protects+genome+integrity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Restriction+of+DNA+replication+to+the+reductive+phase+of+the+metabolic+cycle+protects+genome+integrity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0025270-Eghbal1" id="pone.0025270-Eghbal1"></a>Eghbal MA, Pennefather PS, O'Brien PJ (2004) H<sub>2</sub>S cytotoxicity mechanism involves reactive oxygen species formation and mitochondrial depolarisation. Toxicology  203: 69–76.  <ul class="find" data-citedArticleID="3272968" data-doi="10.1016/j.tox.2004.05.020"><li><a href="http://dx.doi.org/10.1016/j.tox.2004.05.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=H2S+cytotoxicity+mechanism+involves+reactive+oxygen+species+formation+and+mitochondrial+depolarisation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22H2S+cytotoxicity+mechanism+involves+reactive+oxygen+species+formation+and+mitochondrial+depolarisation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0025270-Wong1" id="pone.0025270-Wong1"></a>Wong CM, Siu KL, Jin DY (2004) Peroxiredoxin-null yeast cells are hypersensitive to oxidative stress and are genomically unstable. J Biol Chem  279: 23207–23213.  <ul class="find" data-citedArticleID="3272946" data-doi="10.1074/jbc.m402095200"><li><a href="http://dx.doi.org/10.1074/jbc.m402095200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peroxiredoxin-null+yeast+cells+are+hypersensitive+to+oxidative+stress+and+are+genomically+unstable." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peroxiredoxin-null+yeast+cells+are+hypersensitive+to+oxidative+stress+and+are+genomically+unstable.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0025270-Demasi1" id="pone.0025270-Demasi1"></a>Demasi AP, Pereira GA, Netto LE (2006) Yeast oxidative stress response. Influences of cytosolic thioredoxin peroxidase I and of the mitochondrial functional state. FEBS J  27: 805–816.  <ul class="find" data-citedArticleID="3272954" data-doi="10.1111/j.1742-4658.2006.05116.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2006.05116.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yeast+oxidative+stress+response.+Influences+of+cytosolic+thioredoxin+peroxidase+I+and+of+the+mitochondrial+functional+state." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yeast+oxidative+stress+response.+Influences+of+cytosolic+thioredoxin+peroxidase+I+and+of+the+mitochondrial+functional+state.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0025270-Ragu1" id="pone.0025270-Ragu1"></a>Ragu S, Faye G, Iraqui I, Masurel-Heneman A, Kolodner RD, et al.  (2007) Oxygen metabolism and reactive oxygen species cause chromosomal rearrangements and cell death. Proc Natl Acad Sci U S A  104: 9747–9752.  <ul class="find" data-citedArticleID="3272858" data-doi="10.1073/pnas.0703192104"><li><a href="http://dx.doi.org/10.1073/pnas.0703192104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oxygen+metabolism+and+reactive+oxygen+species+cause+chromosomal+rearrangements+and+cell+death." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oxygen+metabolism+and+reactive+oxygen+species+cause+chromosomal+rearrangements+and+cell+death.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0025270-Iraqui1" id="pone.0025270-Iraqui1"></a>Iraqui I, Kienda G, Soeur J, Faye G, Baldacci G, et al.  (2009) Peroxiredoxin Tsa1 is the key peroxidase suppressing genome instability and protecting against cell death in Saccharomyces cerevisiae. PLoS Genet  5(6): e1000524.  <ul class="find" data-citedArticleID="3272868" data-doi="10.1371/journal.pgen.1000524"><li><a href="http://dx.doi.org/10.1371/journal.pgen.1000524" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peroxiredoxin+Tsa1+is+the+key+peroxidase+suppressing+genome+instability+and+protecting+against+cell+death+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peroxiredoxin+Tsa1+is+the+key+peroxidase+suppressing+genome+instability+and+protecting+against+cell+death+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0025270-Degtyareva1" id="pone.0025270-Degtyareva1"></a>Degtyareva NP, Chen L, Mieczkowski P, Petes TD, Doetsch PW (2008) Chronic oxidative DNA damage due to DNA repair defects causes chromosomal instability in Saccharomyces cerevisiae. Mol Cell Biol  28: 5432–5445.  <ul class="find" data-citedArticleID="3272874" data-doi="10.1128/mcb.00307-08"><li><a href="http://dx.doi.org/10.1128/mcb.00307-08" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chronic+oxidative+DNA+damage+due+to+DNA+repair+defects+causes+chromosomal+instability+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chronic+oxidative+DNA+damage+due+to+DNA+repair+defects+causes+chromosomal+instability+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0025270-Evert1" id="pone.0025270-Evert1"></a>Evert BA, Salmon TB, Song B, Jingjing L, Siede W, et al.  (2004) Spontaneous DNA damage in Saccharomyces cerevisiae elicits phenotypic properties similar to cancer cells. J Biol Chem  279: 22585–22594.  <ul class="find" data-citedArticleID="3272898" data-doi="10.1074/jbc.m400468200"><li><a href="http://dx.doi.org/10.1074/jbc.m400468200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spontaneous+DNA+damage+in+Saccharomyces+cerevisiae+elicits+phenotypic+properties+similar+to+cancer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spontaneous+DNA+damage+in+Saccharomyces+cerevisiae+elicits+phenotypic+properties+similar+to+cancer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0025270-Kumar1" id="pone.0025270-Kumar1"></a>Kumar D, Viberg J, Nilsson AK, Chabes A (2010) Highly mutagenic and severely imbalanced dNTP pools can escape detection by the S-phase checkpoint. Nucleic Acids Res  38: 3975–3983.  <ul class="find" data-citedArticleID="3272976" data-doi="10.1093/nar/gkq128"><li><a href="http://dx.doi.org/10.1093/nar/gkq128" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+mutagenic+and+severely+imbalanced+dNTP+pools+can+escape+detection+by+the+S-phase+checkpoint." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+mutagenic+and+severely+imbalanced+dNTP+pools+can+escape+detection+by+the+S-phase+checkpoint.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0025270-Kumar2" id="pone.0025270-Kumar2"></a>Kumar D, Abdulovic AL, Viberg J, Nilsson AK, Kunkel TA, et al.  (2011) Mechanisms of mutagenesis in vivo due to imbalanced dNTP pools. Nucleic Acids Res  39: 1360–1371.  <ul class="find" data-citedArticleID="3272896" data-doi="10.1093/nar/gkq829"><li><a href="http://dx.doi.org/10.1093/nar/gkq829" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+mutagenesis+in+vivo+due+to+imbalanced+dNTP+pools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+mutagenesis+in+vivo+due+to+imbalanced+dNTP+pools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0025270-Rosenberg1" id="pone.0025270-Rosenberg1"></a>Rosenberg SM (2001) Evolving responsively: adaptive mutation. Nat Rev Genet  2: 504–515.  <ul class="find" data-citedArticleID="3272966" data-doi="10.1038/35080556"><li><a href="http://dx.doi.org/10.1038/35080556" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolving+responsively%3A+adaptive+mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolving+responsively%3A+adaptive+mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0025270-Cairns1" id="pone.0025270-Cairns1"></a>Cairns J, Overbaugh J, Miller S (1988) The origin of mutants. Nature  335: 142–145.  <ul class="find" data-citedArticleID="3272942" data-doi="10.1038/335142a0"><li><a href="http://dx.doi.org/10.1038/335142a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+origin+of+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+origin+of+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0025270-Davis1" id="pone.0025270-Davis1"></a>Davis BD (1989) Transcriptional bias: a non-Lamarckian mechanism for substrate-induced mutations. Proc Natl Acad Sci U S A  86: 5005–5009.  <ul class="find" data-citedArticleID="3272914" data-doi="10.1073/pnas.86.13.5005"><li><a href="http://dx.doi.org/10.1073/pnas.86.13.5005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+bias%3A+a+non-Lamarckian+mechanism+for+substrate-induced+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+bias%3A+a+non-Lamarckian+mechanism+for+substrate-induced+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0025270-Charlesworth1" id="pone.0025270-Charlesworth1"></a>Charlesworth D, Charlesworth B (1988) Origin of mutants disputed. Nature  336: 525–528.  <ul class="find" data-citedArticleID="3272882" data-doi="10.1038/336525a0"><li><a href="http://dx.doi.org/10.1038/336525a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+mutants+disputed." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+mutants+disputed.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0025270-Acar1" id="pone.0025270-Acar1"></a>Acar M, Mettetal JT, van Oudenaarden A (2008) Stochastic switching as a survival strategy in fluctuating environments. Nat Genet  40: 471–475.  <ul class="find" data-citedArticleID="3272944" data-doi="10.1038/ng.110"><li><a href="http://dx.doi.org/10.1038/ng.110" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+switching+as+a+survival+strategy+in+fluctuating+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+switching+as+a+survival+strategy+in+fluctuating+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0025270-Sharp1" id="pone.0025270-Sharp1"></a>Sharp PM, Li WH (1987) The codon Adaptation Index–a measure of directional synonymous codon usage bias, and its potential applications. Nucleic Acids Res  15: 1281–1295.  <ul class="find" data-citedArticleID="3272970" data-doi="10.1093/nar/15.3.1281"><li><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+codon+Adaptation+Index%E2%80%93a+measure+of+directional+synonymous+codon+usage+bias%2C+and+its+potential+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+codon+Adaptation+Index%E2%80%93a+measure+of+directional+synonymous+codon+usage+bias%2C+and+its+potential+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0025270-Miller1" id="pone.0025270-Miller1"></a>Miller C, Schwalb B, Maier K, Schulz D, Dümcke S, et al.  (2011) Dynamic transcriptome analysis measures rates of mRNA synthesis and decay in yeast. Mol Syst Biol  7: 458.  <ul class="find" data-citedArticleID="3272924" data-doi="10.1038/msb.2010.112"><li><a href="http://dx.doi.org/10.1038/msb.2010.112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+transcriptome+analysis+measures+rates+of+mRNA+synthesis+and+decay+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+transcriptome+analysis+measures+rates+of+mRNA+synthesis+and+decay+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0025270-Eide1" id="pone.0025270-Eide1"></a>Eide L, Bjørås M, Pirovano M, Alseth I, Berdal KG, et al.  (1996) Base excision of oxidative purine and pyrimidine DNA damage in <em>Saccharomyces cerevisiae</em> by a DNA glycosylase with sequence similarity to endonuclease III from Escherichia coli. Proc Natl Acad Sci U S A  93: 10735–10740.  <ul class="find" data-citedArticleID="3272962" data-doi="10.1073/pnas.93.20.10735"><li><a href="http://dx.doi.org/10.1073/pnas.93.20.10735" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base+excision+of+oxidative+purine+and+pyrimidine+DNA+damage+in+Saccharomyces+cerevisiae+by+a+DNA+glycosylase+with+sequence+similarity+to+endonuclease+III+from+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base+excision+of+oxidative+purine+and+pyrimidine+DNA+damage+in+Saccharomyces+cerevisiae+by+a+DNA+glycosylase+with+sequence+similarity+to+endonuclease+III+from+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0025270-Silverman1" id="pone.0025270-Silverman1"></a>Silverman SJ, Petti AA, Slavov N, Parsons L, Briehof R, et al.  (2010) Metabolic cycling in single yeast cells from unsynchronized steady-state populations limited on glucose or phosphate. Proc Natl Acad Sci USA  107: 6946–6951.  <ul class="find" data-citedArticleID="3272904" data-doi="10.1073/pnas.1002422107"><li><a href="http://dx.doi.org/10.1073/pnas.1002422107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolic+cycling+in+single+yeast+cells+from+unsynchronized+steady-state+populations+limited+on+glucose+or+phosphate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolic+cycling+in+single+yeast+cells+from+unsynchronized+steady-state+populations+limited+on+glucose+or+phosphate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0025270-Andersson1" id="pone.0025270-Andersson1"></a>Andersson SG, Kurland CG (1990) Codon preferences in free-living microorganisms. Microbiol Rev  54: 198–210.  <ul class="find" data-citedArticleID="3272926"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon+preferences+in+free-living+microorganisms.&amp;auth=&amp;atitle=Codon+preferences+in+free-living+microorganisms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+preferences+in+free-living+microorganisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+preferences+in+free-living+microorganisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0025270-Dong1" id="pone.0025270-Dong1"></a>Dong H, Nilsson L, Kurland CG (1996) Co-variation of tRNA abundance and codon usage in Escherichia coli at different growth rates. J Mol Biol  260: 649–663.  <ul class="find" data-citedArticleID="3272884" data-doi="10.1006/jmbi.1996.0428"><li><a href="http://dx.doi.org/10.1006/jmbi.1996.0428" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Co-variation+of+tRNA+abundance+and+codon+usage+in+Escherichia+coli+at+different+growth+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Co-variation+of+tRNA+abundance+and+codon+usage+in+Escherichia+coli+at+different+growth+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0025270-Thatcher1" id="pone.0025270-Thatcher1"></a>Thatcher JW, Shaw JM, Dickinson WJ (1998) Marginal fitness contributions of nonessential genes in yeast. Proc Natl Acad Sci USA  95: 253–257.  <ul class="find" data-citedArticleID="3272958" data-doi="10.1073/pnas.95.1.253"><li><a href="http://dx.doi.org/10.1073/pnas.95.1.253" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Marginal+fitness+contributions+of+nonessential+genes+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Marginal+fitness+contributions+of+nonessential+genes+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0025270-SGD1" id="pone.0025270-SGD1"></a>SGD project. Saccharomyces Genome Database.   Available: <a href="http://downloads.yeastgenome.org/01/2010">http://downloads.yeastgenome.org/01/2010</a>; Accessed 2010 December 1.  <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0025270-Samanta1" id="pone.0025270-Samanta1"></a>Samanta MP, Tongprasit W, Sethi H, Chin CS, Stolc V (2006) Global identification of noncoding RNAs in Saccharomyces cerevisiae by modulating an essential RNA processing pathway. Proc Natl Acad Sci USA  103: 4192–4197.  <ul class="find" data-citedArticleID="3272916" data-doi="10.1073/pnas.0507669103"><li><a href="http://dx.doi.org/10.1073/pnas.0507669103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+identification+of+noncoding+RNAs+in+Saccharomyces+cerevisiae+by+modulating+an+essential+RNA+processing+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+identification+of+noncoding+RNAs+in+Saccharomyces+cerevisiae+by+modulating+an+essential+RNA+processing+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0025270-Schacherer1" id="pone.0025270-Schacherer1"></a>Schacherer J, Shapiro JA, Ruderfer DM, Kruglyak L (2009) Comprehensive polymorphism survey elucidates population structure of <em>Saccharomyces cerevisiae</em>. Nature  458: 342–345.  <ul class="find" data-citedArticleID="3272860" data-doi="10.1038/nature07670"><li><a href="http://dx.doi.org/10.1038/nature07670" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+polymorphism+survey+elucidates+population+structure+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+polymorphism+survey+elucidates+population+structure+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0025270-Doniger1" id="pone.0025270-Doniger1"></a>Doniger SW, Kim HS, Swain D, Corcuera D, Williams M, et al.  (2008) A catalog of neutral and deleterious polymorphism in yeast. PLoS Genet  4: e1000183.  <ul class="find" data-citedArticleID="3272930" data-doi="10.1371/journal.pgen.1000183"><li><a href="http://dx.doi.org/10.1371/journal.pgen.1000183" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+catalog+of+neutral+and+deleterious+polymorphism+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+catalog+of+neutral+and+deleterious+polymorphism+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0025270-Mnaimneh1" id="pone.0025270-Mnaimneh1"></a>Mnaimneh S, Davierwala AP, Haynes J, Moffat J, Peng WT, et al.  (2004) Exploration of essential gene functions via titratable promoter alleles. Cell  118: 31–44.  <ul class="find" data-citedArticleID="3272934" data-doi="10.1016/j.cell.2004.06.013"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.06.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploration+of+essential+gene+functions+via+titratable+promoter+alleles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploration+of+essential+gene+functions+via+titratable+promoter+alleles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0025270-Wilkie1" id="pone.0025270-Wilkie1"></a>Wilkie AO (1994) The molecular basis of genetic dominance. J Med Genet  31: 89–98.  <ul class="find" data-citedArticleID="3272928" data-doi="10.1136/jmg.31.2.89"><li><a href="http://dx.doi.org/10.1136/jmg.31.2.89" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+basis+of+genetic+dominance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+basis+of+genetic+dominance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0025270-SaintGeorges1" id="pone.0025270-SaintGeorges1"></a>Saint-Georges Y, Garcia M, Delaveau T, Jourdren L, Le Crom S, et al.  (2008) Yeast mitochondrial biogenesis: a role for the PUF RNA-binding protein Puf3p in mRNA localization. PLoS One  3: e2293.  <ul class="find" data-citedArticleID="3272932" data-doi="10.1371/journal.pone.0002293"><li><a href="http://dx.doi.org/10.1371/journal.pone.0002293" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yeast+mitochondrial+biogenesis%3A+a+role+for+the+PUF+RNA-binding+protein+Puf3p+in+mRNA+localization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yeast+mitochondrial+biogenesis%3A+a+role+for+the+PUF+RNA-binding+protein+Puf3p+in+mRNA+localization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0025270-Fedorov1" id="pone.0025270-Fedorov1"></a>Fedorov A, Merican AF, Gilbert W (2002) Large-scale comparison of intron positions among animal, plant, and fungal genes. Proc Natl Acad Sci USA  99: 16128–16133.  <ul class="find" data-citedArticleID="3272952" data-doi="10.1073/pnas.242624899"><li><a href="http://dx.doi.org/10.1073/pnas.242624899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+comparison+of+intron+positions+among+animal%2C+plant%2C+and+fungal+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+comparison+of+intron+positions+among+animal%2C+plant%2C+and+fungal+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0025270-Selpi1" id="pone.0025270-Selpi1"></a>Selpi S, Bryant CH, Kemp GJ, Sarv J, Kristiansson E, et al.  (2009) Predicting functional upstream open reading frames in Saccharomyces cerevisiae. BMC Bioinformatics  10: 451.  <ul class="find" data-citedArticleID="3272908" data-doi="10.1186/1471-2105-10-451"><li><a href="http://dx.doi.org/10.1186/1471-2105-10-451" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+functional+upstream+open+reading+frames+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+functional+upstream+open+reading+frames+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0025270-Nagalakshmi1" id="pone.0025270-Nagalakshmi1"></a>Nagalakshmi U, Wang Z, Waern K, Shou C, Raha D, et al.  (2008) The Transcriptional Landscape of the Yeast Genome Defined by RNA Sequencing. Science  320: 1344–1349.  <ul class="find" data-citedArticleID="3272948" data-doi="10.1126/science.1158441"><li><a href="http://dx.doi.org/10.1126/science.1158441" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Transcriptional+Landscape+of+the+Yeast+Genome+Defined+by+RNA+Sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Transcriptional+Landscape+of+the+Yeast+Genome+Defined+by+RNA+Sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0025270-Mayr1" id="pone.0025270-Mayr1"></a> Mayr E, editor.  (1964) On the Origin Of Species: A Facsimile of the First Edition. Cambridge, Massachusetts: Harvard University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">58.
              </span><a name="pone.0025270-Mendel1" id="pone.0025270-Mendel1"></a>Mendel JG (1866) pp. 3–47.  <em>Versuche über Pflanzenhybriden</em> Verhandlungen des naturforschenden Vereines in Brünn, Bd. IV für das Jahr, 1865 Abhandlungen.  <ul class="find-nolinks"></ul></li><li><span class="label">59.
              </span><a name="pone.0025270-Burkhardt1" id="pone.0025270-Burkhardt1"></a>Burkhardt RW (1984) The Zoological Philosophy of J. B. Lamarck. Zoological Philosophy: An Exposition with Regard to the Natural History of Animals. By Lamarck JB. Trans. Hugh Elliot. Chicago: The University of Chicago Press.   <ul class="find-nolinks"></ul></li><li><span class="label">60.
              </span><a name="pone.0025270-Chambers1" id="pone.0025270-Chambers1"></a>Chambers J, Cleveland W, Kleiner B, Tukey P (1983) 63 p.   <em>Graphical Methods for Data Analysis</em>, Wadsworth International Group.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.XML" value="140991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.PDF" value="1618413"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g001.PNG_L" value="1186972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g001.PNG_M" value="241565"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g001.PNG_S" value="26971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g001.TIF" value="2351874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g001.PNG_I" value="99733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g002.PNG_L" value="681093"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g002.PNG_M" value="162922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g002.PNG_S" value="18671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g002.TIF" value="1319484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g002.PNG_I" value="54330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g003.PNG_L" value="1259926"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g003.PNG_M" value="250485"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g003.PNG_S" value="27203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g003.TIF" value="2474260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.g003.PNG_I" value="102303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.t001.PNG_L" value="211051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.t001.PNG_M" value="36978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.t001.PNG_S" value="8754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.t001.TIF" value="12440272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.t001.PNG_I" value="11054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s001.DOC" value="147456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s002.DOC" value="126976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s003.DOC" value="125440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s004.DOC" value="282624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s005.DOC" value="441856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s006.DOC" value="226816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s007.DOC" value="142848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s008.DOC" value="503808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s009.DOC" value="81408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s010.DOC" value="55296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s011.DOC" value="43520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s012.DOC" value="55808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s013.PDF" value="393445"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0025270.s014.DOC" value="1918976"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0025270&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0025270" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0025270&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0025270&volume=&issue=&title=Adaptation of Organisms by Resonance of RNA Transcription with the Cellular Redox Cycle&author_name=Viktor%20Stolc%2C%20Alena%20Shmygelska%2C%20Yuri%20Griko&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0025270" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0025270" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0025270" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0025270&amp;t=Adaptation%20of%20Organisms%20by%20Resonance%20of%20RNA%20Transcription%20with%20the%20Cellular%20Redox%20Cycle" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0025270&title=Adaptation%20of%20Organisms%20by%20Resonance%20of%20RNA%20Transcription%20with%20the%20Cellular%20Redox%20Cycle&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0025270&amp;title=Adaptation%20of%20Organisms%20by%20Resonance%20of%20RNA%20Transcription%20with%20the%20Cellular%20Redox%20Cycle" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0025270" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0025270&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Adaptation of Organisms by Resonance of RNA Transcription with the Cellular Redox Cycle';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0025270';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Adaptation%20of%20Organisms%20by%20Resonance%20of%20RNA%20Transcription%20with%20the%20Cellular%20Redox%20Cycle http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0025270" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0025270" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="72984"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="72984"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="72984"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+oscillators%22" title="Search for articles in the subject area:'Genetic oscillators'"><div class="flagText">Genetic oscillators</div></a>
              <div data-categoryid="19703" data-articleid="72984"
                   data-categoryname="Genetic oscillators"
                   class="flagImage" title="Flag 'Genetic oscillators' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="72984"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oxidation-reduction+reactions%22" title="Search for articles in the subject area:'Oxidation-reduction reactions'"><div class="flagText">Oxidation-reduction reactions</div></a>
              <div data-categoryid="20673" data-articleid="72984"
                   data-categoryname="Oxidation-reduction reactions"
                   class="flagImage" title="Flag 'Oxidation-reduction reactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Saccharomyces+cerevisiae%22" title="Search for articles in the subject area:'Saccharomyces cerevisiae'"><div class="flagText">Saccharomyces cerevisiae</div></a>
              <div data-categoryid="43143" data-articleid="72984"
                   data-categoryname="Saccharomyces cerevisiae"
                   class="flagImage" title="Flag 'Saccharomyces cerevisiae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="72984"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2220'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6301'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1544&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
