

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Reconstruction of Family-Level Phylogenetic Relationships within Demospongiae (Porifera) Using Nuclear Encoded Housekeeping Genes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0050437"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0050437" />

    <meta name="citation_title" content="Reconstruction of Family-Level Phylogenetic Relationships within Demospongiae (Porifera) Using Nuclear Encoded Housekeeping Genes"/>
    <meta itemprop="name" content="Reconstruction of Family-Level Phylogenetic Relationships within Demospongiae (Porifera) Using Nuclear Encoded Housekeeping Genes"/>

      <meta name="citation_author" content="Malcolm S. Hill"/>
            <meta name="citation_author_institution" content="Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America"/>
      <meta name="citation_author" content="April L. Hill"/>
            <meta name="citation_author_institution" content="Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America"/>
      <meta name="citation_author" content="Jose Lopez"/>
            <meta name="citation_author_institution" content="Nova Southeastern University Oceanographic Center, Dania Beach, Florida, United States of America"/>
      <meta name="citation_author" content="Kevin J. Peterson"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Dartmouth College, Hanover, New Hampshire, United States of America"/>
      <meta name="citation_author" content="Shirley Pomponi"/>
            <meta name="citation_author_institution" content="Harbor Branch Oceanographic Institute, Florida Atlantic University, Fort Pierce, Florida, United States of America"/>
      <meta name="citation_author" content="Maria C. Diaz"/>
            <meta name="citation_author_institution" content="Museo Marino de Margarita, Boulevard de Boca Del Rio, Boca del Rio, Nueva Esparta, Venezuela"/>
      <meta name="citation_author" content="Robert W. Thacker"/>
            <meta name="citation_author_institution" content="Department of Biology, University of Alabama at Birmingham, Birmingham, Alabama, United States of America"/>
      <meta name="citation_author" content="Maja Adamska"/>
            <meta name="citation_author_institution" content="Sars International Centre for Marine Molecular Biology, Thormøhlensgt, Bergen, Norway"/>
      <meta name="citation_author" content="Nicole Boury-Esnault"/>
            <meta name="citation_author_institution" content="IMBE-UMR7263 CNRS, Université d'Aix-Marseille, Station marine d'Endoume, Marseille, France"/>
      <meta name="citation_author" content="Paco Cárdenas"/>
            <meta name="citation_author_institution" content="Department of Systematic Biology, Evolutionary Biology Centre, Uppsala University, Uppsala, Sweden"/>
      <meta name="citation_author" content="Andia Chaves-Fonnegra"/>
            <meta name="citation_author_institution" content="Nova Southeastern University Oceanographic Center, Dania Beach, Florida, United States of America"/>
      <meta name="citation_author" content="Elizabeth Danka"/>
            <meta name="citation_author_institution" content="Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America"/>
      <meta name="citation_author" content="Bre-Onna De Laine"/>
            <meta name="citation_author_institution" content="Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America"/>
      <meta name="citation_author" content="Dawn Formica"/>
            <meta name="citation_author_institution" content="Nova Southeastern University Oceanographic Center, Dania Beach, Florida, United States of America"/>
      <meta name="citation_author" content="Eduardo Hajdu"/>
            <meta name="citation_author_institution" content="Departamento de Invertebrados, Museu Nacional/Universidade Federal do Rio de Janeiro, Rio de Janeiro, Rio de Janeiro, Brazil"/>
      <meta name="citation_author" content="Gisele Lobo-Hajdu"/>
            <meta name="citation_author_institution" content="Departamento de Genética, IBRAG, Universidade do Estado do Rio de Janeiro, Rio de Janeiro, Rio de Janeiro, Brazil"/>
      <meta name="citation_author" content="Sarah Klontz"/>
            <meta name="citation_author_institution" content="Department of Invertebrate Zoology, National Museum of Natural History, Smithsonian Institution, Washington, D.C., United States of America"/>
      <meta name="citation_author" content="Christine C. Morrow"/>
            <meta name="citation_author_institution" content="School of Biological Sciences, MBC, Queen's University, Belfast, United Kingdom"/>
      <meta name="citation_author" content="Jignasa Patel"/>
            <meta name="citation_author_institution" content="Nova Southeastern University Oceanographic Center, Dania Beach, Florida, United States of America"/>
      <meta name="citation_author" content="Bernard Picton"/>
            <meta name="citation_author_institution" content="National Museums Northern Ireland, Holywood, Northern Ireland, United Kingdom"/>
      <meta name="citation_author" content="Davide Pisani"/>
            <meta name="citation_author_institution" content="School of Earth Sciences and School of Biological Sciences, The University of Bristol, Bristol, United Kingdom"/>
      <meta name="citation_author" content="Deborah Pohlmann"/>
            <meta name="citation_author_institution" content="Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America"/>
      <meta name="citation_author" content="Niamh E. Redmond"/>
            <meta name="citation_author_institution" content="Department of Invertebrate Zoology, National Museum of Natural History, Smithsonian Institution, Washington, D.C., United States of America"/>
      <meta name="citation_author" content="John Reed"/>
            <meta name="citation_author_institution" content="Harbor Branch Oceanographic Institute, Florida Atlantic University, Fort Pierce, Florida, United States of America"/>
      <meta name="citation_author" content="Stacy Richey"/>
            <meta name="citation_author_institution" content="Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America"/>
      <meta name="citation_author" content="Ana Riesgo"/>
            <meta name="citation_author_institution" content="Museum of Comparative Zoology, Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Ewelina Rubin"/>
            <meta name="citation_author_institution" content="Nova Southeastern University Oceanographic Center, Dania Beach, Florida, United States of America"/>
      <meta name="citation_author" content="Zach Russell"/>
            <meta name="citation_author_institution" content="Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America"/>
      <meta name="citation_author" content="Klaus Rützler"/>
            <meta name="citation_author_institution" content="Department of Invertebrate Zoology, National Museum of Natural History, Smithsonian Institution, Washington, D.C., United States of America"/>
      <meta name="citation_author" content="Erik A. Sperling"/>
            <meta name="citation_author_institution" content="Harvard University, Department of Earth and Planetary Science, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Michael di Stefano"/>
            <meta name="citation_author_institution" content="Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America"/>
      <meta name="citation_author" content="James E. Tarver"/>
            <meta name="citation_author_institution" content="School of Earth Sciences, University of Bristol, Bristol, United Kingdom"/>
      <meta name="citation_author" content="Allen G. Collins"/>
            <meta name="citation_author_institution" content="Department of Invertebrate Zoology, National Museum of Natural History, Smithsonian Institution, Washington, D.C., United States of America"/>
            <meta name="citation_author_institution" content="National Systematics Laboratory of NOAA's Fisheries Service, National Museum of Natural History, Smithsonian Institution, Washington, D.C., United States of America"/>

    <meta name="citation_date" content="2013/1/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0050437.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e50437"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="8"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Long expected sponges from the neoproterozoic Ediacara fauna of South Australia; citation_volume=2; citation_number=1; citation_pages=185-195; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Fossil steroids record the appearance of Demospongiae during the Cryogenian period; citation_volume=457; citation_number=2; citation_pages=718-721; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=The first animals: ca. 760-million-year-old sponge-like fossils from Namibia; citation_volume=108; citation_number=3; citation_pages=658; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Where's the glass? Biomarkers, molecular clocks, and microRNAs suggest a 200-Myr missing Precambrian fossil record of siliceous sponge spicules; citation_volume=8; citation_number=4; citation_pages=24-36; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=The Cambrian conundrum: early divergence and later ecological success in the early history of animals; citation_volume=334; citation_number=5; citation_pages=1091-1097; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Discovery of a “living dinosaur”: Globally unique modern hexactinellid sponge reefs off British Columbia, Canada; citation_volume=44; citation_number=6; citation_pages=265-282; citation_date=2001; " />
      <meta name="citation_reference" content="citation_number=7; " />
      <meta name="citation_reference" content="citation_title=Spongivory on Caribbean reefs releases corals from competition with sponges; citation_volume=117; citation_number=8; citation_pages=143-150; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Sponges On Coral Reefs: A Community Shaped By Competitive Cooperation; citation_volume=68; citation_number=9; citation_pages=85-148; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Major bulk dissolved organic carbon (DOC) removal by encrusting coral reef cavity sponges; citation_volume=357; citation_number=10; citation_pages=139-151; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Cryptic diversity of the symbiotic cyanobacterium Synechococcus spongiarum among sponge hosts; citation_volume=17; citation_number=11; citation_pages=2937-47; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=The use of marine sponge Haliclona tenuiramosa as bioindicator to monitor heavy metal pollution in the coasts of Gulf of Mannar, India; citation_volume=156; citation_number=12; citation_pages=451-459; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Florida Reef Sponges Harbor Coral Disease-Associated Bacteria; citation_volume=51; citation_number=13; citation_pages=117-129; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=The chemical ecology of sponges on Caribbean reefs: Natural products shape natural systems; citation_volume=61; citation_number=14; citation_pages=888-898; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Marine sponges and their microbial symbionts: love and other relationships; citation_volume=14; citation_number=15; citation_pages=335-346; citation_date=2012; " />
      <meta name="citation_reference" content="citation_number=16; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of Demospongiae: implications for classification and scenarios of character evolution; citation_volume=32; citation_number=17; citation_pages=823-837; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic-signal dissection of nuclear housekeeping genes supports the paraphyly of sponges and the monophyly of Eumetazoa; citation_volume=26; citation_number=18; citation_pages=2261-2274; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Phylogenomics revives traditional views on deep animal relationships; citation_volume=19; citation_number=19; citation_pages=706-712; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Resolving difficult phylogenetic questions: why more sequences are not enough; citation_volume=9(3); citation_number=20; citation_pages=e1000602; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Two new genera of hadromerid sponges (Porifera, Demospongiae); citation_volume=22; citation_number=21; citation_pages=337-3444; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Two Pione species (Hadromerida, Clionaidae) from the Red Sea: a taxonomical challenge; citation_volume=10; citation_number=22; citation_pages=275-285; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Morphology and molecules on opposite sides of the diversity gradient: Four cryptic species of the Cliona celata (Porifera, Demospongiae) complex in South America revealed by mitochondrial and nuclear markers; citation_volume=62; citation_number=23; citation_pages=529-541; citation_date=2012; " />
      <meta name="citation_reference" content="citation_number=24; " />
      <meta name="citation_reference" content="citation_title=An analysis of partial 28S ribosomal RNA sequences suggests early radiations of sponges; citation_volume=28; citation_number=25; citation_pages=139-151; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogenetic position of the hexactinellid sponges in relation to the Protista and Demospongiae; citation_volume=2; citation_number=26; citation_pages=71-75; citation_date=1993; " />
      <meta name="citation_reference" content="citation_number=27; " />
      <meta name="citation_reference" content="citation_title=Evaluating multiple alternative hypotheses for the origin of Bilateria: An analysis of 18S rRNA molecular evidence; citation_volume=95; citation_number=28; citation_pages=15458-15463; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Toward a phylogenetic classification of the Mycalids with anisochelae (Demospongiae: Poecilosclerida), and comments on the status of Naviculina Gray, 1867; citation_volume=44; citation_number=29; citation_pages=225-238; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic relationships of the family Axinellidae (Porifera: Demospongiae) using morphological and molecular data; citation_volume=29; citation_number=30; citation_pages=169-198; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular Systematics of sponges (Porifera); citation_volume=420; citation_number=31; citation_pages=15-27; citation_date=2000; " />
      <meta name="citation_reference" content="citation_number=32; " />
      <meta name="citation_reference" content="citation_title=Poriferan mtDNA and animal phylogeny based on mitochondrial gene arrangements; citation_volume=54; citation_number=33; citation_pages=651-659; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Biodiversity, molecular ecology and phylogeography of marine sponges: patterns, implications and outlooks; citation_volume=43; citation_number=34; citation_pages=377-385; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Reassessment of the classification of the Order Haplosclerida (Class Demospongiae, Phylum Porifera) using 18S rRNA gene sequence data; citation_volume=43; citation_number=35; citation_pages=344-352; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Polyphyly of the genus Axinella and of the family Axinellidae (Porifera: DemospongiaeP); citation_volume=57; citation_number=36; citation_pages=35-47; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic Relationships of the Marine Haplosclerida (Phylum Porifera) Employing Ribosomal (28S rRNA) and Mitochondrial (cox1, nad1) Gene Sequence Data; citation_volume=6; citation_number=37; citation_pages=e24344; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Congruence between nuclear and mitochondrial genes in Demospongiae: A new hypothesis for relationships within the G4 clade (Porifera: Demospongiae); citation_volume=62; citation_number=38; citation_pages=174-190; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Phylum Porifera Grant, 1826. In: Zhang, Z-Q, editor. Animal biodiversity: An outline of higher-level classification and survey of taxonomic richness; citation_volume=3148; citation_number=39; citation_pages=13-18; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Sponge systematics facing new challenges; citation_volume=61; citation_number=40; citation_pages=79-209; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Deep phylogeny and evolution of sponges (Phylum Porifera); citation_volume=61; citation_number=41; citation_pages=1-78; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=An evaluation of support for order-level monophyly and interrelationships within the class Demospongiae using partial data from the large subunit rDNA and cytochrome oxidase subunit I; citation_volume=34; citation_number=42; citation_pages=81-96; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Reconstructing ordinal relationships in the Demospongiae using mitochondrial genomic data; citation_volume=49; citation_number=43; citation_pages=111-124; citation_date=2008; " />
      <meta name="citation_reference" content="citation_number=44; " />
      <meta name="citation_reference" content="citation_title=Estimating metazoan divergence times with a molecular clock; citation_volume=101; citation_number=45; citation_pages=6536-6541; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Origin of the Eumetazoa: testing ecological predictions of molecular clocks against the Proterozoic fossil record; citation_volume=102; citation_number=46; citation_pages=9547-9552; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Biodiversity and distribution of deep and shallow water sponges in the Bahamas; citation_volume=2; citation_number=47; citation_pages=1387-1392; citation_date=1997; " />
      <meta name="citation_reference" content="citation_number=48; " />
      <meta name="citation_reference" content="citation_title=ProtTest: Selection of best-fit models of protein evolution; citation_volume=21; citation_number=49; citation_pages=2104-2105; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=LG: An Improved, General Amino-Acid Replacement Matrix; citation_volume=25; citation_number=50; citation_pages=1307-1320; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing; citation_volume=18; citation_number=51; citation_pages=502-504; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cross-validatory choice and assessment of statistical prediction; citation_volume=36; citation_number=52; citation_pages=111-147; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Matrix representation with parsimony, taxonomic congruence, and total evidence; citation_volume=51; citation_number=53; citation_pages=151-155; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Phylogeny and evolution of glass sponges (Porifera, Hexactinellida); citation_volume=57; citation_number=54; citation_pages=388-405; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Phylogeny and evolution of calcareous sponges: Monophyly of Calcinea and Calcaronea, high level of morphological homoplasy, and the primitive nature of axial symmetry; citation_volume=52; citation_number=55; citation_pages=311-333; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Non-monophyly of most supraspecific taxa of calcareous sponges (Porifera, Calcarea) revealed by increased taxon sampling and partitioned Bayesian analysis of ribosomal DNA; citation_volume=40; citation_number=56; citation_pages=830-843; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phylogeny and Morphogenesis of Dictyoceratida (Porifera: Demospongiae); citation_volume=46; citation_number=57; citation_pages=480-484; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecular and morphological description of a new species of Halisarca (Demospongiae: Halisarcida) from Mediterranean Sea and a redescription of the type species Halisarca dujardini; citation_volume=2768; citation_number=58; citation_pages=5-31; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Horny sponges and their affairs: On the phylogenetic relationships of keratose sponges; citation_volume=63; citation_number=59; citation_pages=809-816; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Morphological, chemical, and biochemical characterization of a new species of sponges without skeleton (Porifera, Demospongiae) from the Mediterranean Sea; citation_volume=22; citation_number=60; citation_pages=313-326; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analyses of marine sponges within the order Verongida: a comparison of morphological and molecular data; citation_volume=126; citation_number=61; citation_pages=220-234; citation_date=2007; " />
      <meta name="citation_reference" content="citation_number=62; " />
      <meta name="citation_reference" content="citation_number=63; " />
      <meta name="citation_reference" content="citation_title=Major discrepancy between phylogenetic hypotheses based on molecular and morphological criteria within the Order Haplosclerida (Phylum Porifera: Class Demospongiae); citation_volume=40; citation_number=64; citation_pages=237-240; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial Cytochrome oxidase 1 phylogeny supports alternative taxonomic scheme for the marine Haplosclerida; citation_volume=87; citation_number=65; citation_pages=1577-1584; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Amphimedon queenslandica genome and the evolution of animal complexity; citation_number=66; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic relationships of freshwater sponges (Porifera, Spongillina) inferred from analyses of 18S rDNA, COI mtDNA, and ITS2 rDNA sequences; citation_volume=34; citation_number=67; citation_pages=549-557; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genus-specific associations of marine sponges with group I crenaarchaeotes; citation_volume=150; citation_number=68; citation_pages=759-772; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Towards a DNA taxonomy of Caribbean demosponges: a gene tree reconstructed from partial mitochondrial CO1 gene sequences supports previous rDNA phylogenies and provides a new perspective on the systematics of Demospongiae; citation_volume=87; citation_number=69; citation_pages=1563-1570; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of the Astrophorida (Porifera, Demospongiae(p)) reveals an unexpected high level of spicule homoplasy; citation_volume=6; citation_number=70; citation_pages=e18318; citation_date=2011; " />
      <meta name="citation_reference" content="citation_number=71; " />
      <meta name="citation_reference" content="citation_number=72; " />
      <meta name="citation_reference" content="citation_title=Phylogeny and classification of lithistid sponges (Porifera: Demospongiae): a preliminary assessment using ribosomal DNA sequence comparisons; citation_volume=3; citation_number=73; citation_pages=87-103; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=On the phylogenetic relationships of Hadromerid and Poecilosclerid sponges; citation_volume=87; citation_number=74; citation_pages=1585-1598; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The systematics of Raspailiidae (Demospongiae, Poecilosclerida, Microcionina) reanalysed with a ribosomal marker; citation_volume=87; citation_number=75; citation_pages=1571-1576; citation_date=2007; " />
      <meta name="citation_reference" content="citation_number=76; " />
      <meta name="citation_reference" content="citation_number=77; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; citation_volume=25; citation_number=78; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput; citation_volume=32; citation_number=79; citation_pages=1792-1797; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building; citation_volume=27; citation_number=80; citation_pages=221-224; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Optimization of preservation and storage time of sponge tissue samples to obtain quality mRNA for Next-Generation Sequencing with Illumina platforms; citation_volume=12; citation_number=81; citation_pages=312-322; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of the sox family in the calcareous sponge Sycon ciliatum: multiple genes with unique expression patterns; citation_volume=3; citation_number=82; citation_pages=14; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Likelihood-mapping: a simple method to visualize phylogenetic content of a sequence alignment; citation_volume=94; citation_number=83; citation_pages=6815-6819; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models; citation_volume=22; citation_number=84; citation_pages=2688-2690; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PhyloBayes 3: a Bayesian software package for phylogenetic reconstruction and molecular dating; citation_volume=25; citation_number=85; citation_pages=2286-2288; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=A Bayesian Mixture Model for Across-Site Heterogeneities in the Amino-Acid Replacement Process; citation_volume=21; citation_number=86; citation_pages=1095-1109; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=TOPD/FMTS: a new software to compare phylogenetic trees; citation_volume=23; citation_number=87; citation_pages=1556-1558; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Clann: Investigating phylogenetic information through supertree analyses; citation_volume=21; citation_number=88; citation_pages=390-392; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A method for inferring the rate of evolution of homologous characters that can potentially improve phylogenetic inference, resolve deep divergence and correct systematic biases; citation_volume=60; citation_number=89; citation_pages=833-844; citation_date=2011; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0050437" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Reconstruction of Family-Level Phylogenetic Relationships within Demospongiae (Porifera) Using Nuclear Encoded Housekeeping Genes"/>
    <meta name="twitter:description" content="BackgroundDemosponges are challenging for phylogenetic systematics because of their plastic and relatively simple morphologies and many deep divergences between major clades. To improve understanding of the phylogenetic relationships within Demospongiae, we sequenced and analyzed seven nuclear housekeeping genes involved in a variety of cellular functions from a diverse group of sponges.Methodology/Principal FindingsWe generated data from each of the four sponge classes (i.e., Calcarea, Demospongiae, Hexactinellida, and Homoscleromorpha), but focused on family-level relationships within demosponges. With data for 21 newly sampled families, our Maximum Likelihood and Bayesian-based approaches recovered previously phylogenetically defined taxa: Keratosap, Myxospongiaep, Spongillidap, Haploscleromorphap (the marine haplosclerids) and Democlaviap. We found conflicting results concerning the relationships of Keratosap and Myxospongiaep to the remaining demosponges, but our results strongly supported a clade of Haploscleromorphap+Spongillidap+Democlaviap. In contrast to hypotheses based on mitochondrial genome and ribosomal data, nuclear housekeeping gene data suggested that freshwater sponges (Spongillidap) are sister to Haploscleromorphap rather than part of Democlaviap. Within Keratosap, we found equivocal results as to the monophyly of Dictyoceratida. Within Myxospongiaep, Chondrosida and Verongida were monophyletic. A well-supported clade within Democlaviap, Tetractinellidap, composed of all sampled members of Astrophorina and Spirophorina (including the only lithistid in our analysis), was consistently revealed as the sister group to all other members of Democlaviap. Within Tetractinellidap, we did not recover monophyletic Astrophorina or Spirophorina. Our results also reaffirmed the monophyly of order Poecilosclerida (excluding Desmacellidae and Raspailiidae), and polyphyly of Hadromerida and Halichondrida.Conclusions/SignificanceThese results, using an independent nuclear gene set, confirmed many hypotheses based on ribosomal and/or mitochondrial genes, and they also identified clades with low statistical support or clades that conflicted with traditional morphological classification. Our results will serve as a basis for future exploration of these outstanding questions using more taxon- and gene-rich datasets."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0050437.g002"/>

  <meta property="og:title" content="Reconstruction of Family-Level Phylogenetic Relationships within Demospongiae (Porifera) Using Nuclear Encoded Housekeeping Genes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0050437" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8819'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8425'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7555&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0050437">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0050437" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0050437&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Reconstruction of Family-Level Phylogenetic Relationships within Demospongiae (Porifera) Using Nuclear Encoded Housekeeping Genes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Malcolm S. Hill, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            April L. Hill, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jose Lopez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Nova Southeastern University Oceanographic Center, Dania Beach, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kevin J. Peterson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Dartmouth College, Hanover, New Hampshire, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shirley Pomponi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harbor Branch Oceanographic Institute, Florida Atlantic University, Fort Pierce, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maria C. Diaz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Museo Marino de Margarita, Boulevard de Boca Del Rio, Boca del Rio, Nueva Esparta, Venezuela
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robert W. Thacker, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, University of Alabama at Birmingham, Birmingham, Alabama, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maja Adamska, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Sars International Centre for Marine Molecular Biology, Thormøhlensgt, Bergen, Norway
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nicole Boury-Esnault, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  IMBE-UMR7263 CNRS, Université d'Aix-Marseille, Station marine d'Endoume, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paco Cárdenas, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Systematic Biology, Evolutionary Biology Centre, Uppsala University, Uppsala, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andia Chaves-Fonnegra, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Nova Southeastern University Oceanographic Center, Dania Beach, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elizabeth Danka, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bre-Onna De Laine, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dawn Formica, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Nova Southeastern University Oceanographic Center, Dania Beach, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eduardo Hajdu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departamento de Invertebrados, Museu Nacional/Universidade Federal do Rio de Janeiro, Rio de Janeiro, Rio de Janeiro, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gisele Lobo-Hajdu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departamento de Genética, IBRAG, Universidade do Estado do Rio de Janeiro, Rio de Janeiro, Rio de Janeiro, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sarah Klontz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Invertebrate Zoology, National Museum of Natural History, Smithsonian Institution, Washington, D.C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christine C. Morrow, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biological Sciences, MBC, Queen's University, Belfast, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jignasa Patel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Nova Southeastern University Oceanographic Center, Dania Beach, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bernard Picton, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Museums Northern Ireland, Holywood, Northern Ireland, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Davide Pisani, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Earth Sciences and School of Biological Sciences, The University of Bristol, Bristol, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Deborah Pohlmann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Niamh E. Redmond, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Invertebrate Zoology, National Museum of Natural History, Smithsonian Institution, Washington, D.C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John Reed, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harbor Branch Oceanographic Institute, Florida Atlantic University, Fort Pierce, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stacy Richey, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ana Riesgo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Museum of Comparative Zoology, Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ewelina Rubin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Nova Southeastern University Oceanographic Center, Dania Beach, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zach Russell, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Klaus Rützler, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Invertebrate Zoology, National Museum of Natural History, Smithsonian Institution, Washington, D.C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Erik A. Sperling, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard University, Department of Earth and Planetary Science, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael di Stefano, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Gottwald Science Center, University of Richmond, Richmond, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James E. Tarver, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Earth Sciences, University of Bristol, Bristol, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Allen G. Collins
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:collinsa@si.edu">collinsa@si.edu</a></p>

                <p>Affiliations:
                  Department of Invertebrate Zoology, National Museum of Natural History, Smithsonian Institution, Washington, D.C., United States of America, 
                  National Systematics Laboratory of NOAA's Fisheries Service, National Museum of Natural History, Smithsonian Institution, Washington, D.C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 23, 2013</li>
    <li>DOI: 10.1371/journal.pone.0050437</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0050437" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0050437" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0050437" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0050437" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0050437-t001" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0050437-t002" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0050437-t003" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0050437-t004" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0050437-g001" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0050437-g002" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0050437-t005" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0050437-t006" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0050437">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0050437" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Demosponges are challenging for phylogenetic systematics because of their plastic and relatively simple morphologies and many deep divergences between major clades. To improve understanding of the phylogenetic relationships within Demospongiae, we sequenced and analyzed seven nuclear housekeeping genes involved in a variety of cellular functions from a diverse group of sponges.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We generated data from each of the four sponge classes (i.e., Calcarea, Demospongiae, Hexactinellida, and Homoscleromorpha), but focused on family-level relationships within demosponges. With data for 21 newly sampled families, our Maximum Likelihood and Bayesian-based approaches recovered previously phylogenetically defined taxa: <em>Keratosa<sup>p</sup></em>, <em>Myxospongiae<sup>p</sup></em>, <em>Spongillida<sup>p</sup></em>, <em>Haploscleromorpha<sup>p</sup></em> (the marine haplosclerids) and <em>Democlavia<sup>p</sup></em>. We found conflicting results concerning the relationships of <em>Keratosa<sup>p</sup></em> and <em>Myxospongiae<sup>p</sup></em> to the remaining demosponges, but our results strongly supported a clade of <em>Haploscleromorpha<sup>p</sup></em>+<em>Spongillida<sup>p</sup></em>+<em>Democlavia<sup>p</sup></em>. In contrast to hypotheses based on mitochondrial genome and ribosomal data, nuclear housekeeping gene data suggested that freshwater sponges (<em>Spongillida<sup>p</sup></em>) are sister to <em>Haploscleromorpha<sup>p</sup></em> rather than part of <em>Democlavia<sup>p</sup></em>. Within <em>Keratosa<sup>p</sup></em>, we found equivocal results as to the monophyly of Dictyoceratida. Within <em>Myxospongiae<sup>p</sup></em>, Chondrosida and Verongida were monophyletic. A well-supported clade within <em>Democlavia<sup>p</sup></em>, <em>Tetractinellida<sup>p</sup></em>, composed of all sampled members of Astrophorina and Spirophorina (including the only lithistid in our analysis), was consistently revealed as the sister group to all other members of <em>Democlavia<sup>p</sup></em>. Within <em>Tetractinellida<sup>p</sup></em>, we did not recover monophyletic Astrophorina or Spirophorina. Our results also reaffirmed the monophyly of order Poecilosclerida (excluding Desmacellidae and Raspailiidae), and polyphyly of Hadromerida and Halichondrida.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>These results, using an independent nuclear gene set, confirmed many hypotheses based on ribosomal and/or mitochondrial genes, and they also identified clades with low statistical support or clades that conflicted with traditional morphological classification. Our results will serve as a basis for future exploration of these outstanding questions using more taxon- and gene-rich datasets.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hill MS, Hill AL, Lopez J, Peterson KJ, Pomponi S, et al.  (2013) Reconstruction of Family-Level Phylogenetic Relationships within Demospongiae (Porifera) Using Nuclear Encoded Housekeeping Genes. PLoS ONE 8(1):
          e50437.
            doi:10.1371/journal.pone.0050437</p><p><strong>Editor: </strong>Senjie Lin, 
        University of Connecticut, United States of America
      </p><p><strong>Received:</strong> May 1, 2012; <strong>Accepted:</strong> October 22, 2012; <strong>Published:</strong> January 23, 2013</p><p>This is an open-access article, free of all copyright, and may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose. The work is made available under the Creative Commons CC0 public domain dedication.</p><p><strong>Funding: </strong>This work was supported by the National Science Foundation's Assembling the Tree of Life Program (<a href="http://www.nsf.gov/funding/pgm_summ.jsp?pims_id=5129">http://www.nsf.gov/funding/pgm_summ.jsp?​pims_id=5129</a>; DEB awards 0829763, 0829783, 0829791, and 0829986). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Sponges belong to an ancient metazoan lineage with a fossil record that stretches back to the late Cryogenian &gt;635 Myr ago <a href="#pone.0050437-Gehling1">[1]</a>–<a href="#pone.0050437-Brain1">[3]</a>. Some estimates place their appearance at nearly 800 Myr ago <a href="#pone.0050437-Sperling1">[4]</a>, <a href="#pone.0050437-Erwin1">[5]</a>. As a sister group (or groups) to all the other animals in the metazoan tree of life, sponges represent a fulcrum point in the history of animal life lying at the junction between single-celled ancestors and the rest of Metazoa. Sponges have also been important ecosystem engineers throughout much of their history, e.g., as major reef-builders during the Upper Devonian, Upper Permian, and through a major portion of the Jurassic <a href="#pone.0050437-Krautter1">[6]</a>, <a href="#pone.0050437-Milsom1">[7]</a>. In modern oceans, poriferans continue to perform important ecological functions as water filterers, bioeroders, structural habitat providers, microbial symbiont incubators, dissolved organic carbon sinks, natural product biosynthesizers, chemical accumulators, and potential marine pathogen reservoirs <a href="#pone.0050437-Hill1">[8]</a>–<a href="#pone.0050437-Webster1">[15]</a>. As one of the most diverse taxa of extant sessile invertebrates <a href="#pone.0050437-Hooper1">[16]</a>, a detailed exploration of poriferan evolutionary relationships will yield important insights into many phases of metazoan history.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Due to their simple bodies with a paucity of easily accessible morphological traits, sponges are notoriously resistant to attempts at taxonomic classification <a href="#pone.0050437-Hooper1">[16]</a>. Indeed, taxonomic controversy extends from the highest levels of classification (e.g., whether the phylum Porifera is monophyletic <a href="#pone.0050437-Borchiellini1">[17]</a>–<a href="#pone.0050437-Philippe2">[20]</a>) to whether particular genera belong to one or another family (e.g., <a href="#pone.0050437-Rtzler2">[21]</a>), or even whether different nominal species are truly distinct (e.g., <a href="#pone.0050437-Ferrario1">[22]</a>, <a href="#pone.0050437-dePaula1">[23]</a>). In the mid-1980s, van Soest <a href="#pone.0050437-vanSoest1">[24]</a> presented a call to include explicitly phylogenetic perspectives in sponge systematics through cladistic analysis. Since that time, phylogenetic classification has permeated the field of sponge taxonomy (e.g., <a href="#pone.0050437-Lafay1">[25]</a>–<a href="#pone.0050437-Morrow1">[38]</a>). As currently envisioned, four classes comprise the phylum Porifera: Calcarea = (Calcispongiae plus the fossil group Heteractinida), Demospongiae, Homoscleromorpha, and Hexactinellida <a href="#pone.0050437-Hooper2">[39]</a>. Ample evidence exists to conclude that each of these classes is monophyletic, and so each has been provided with an explicit phylogenetic definition <a href="#pone.0050437-Crdenas1">[40]</a>. Indeed, substantial evidence is accumulating for the existence of various sponge clades at different levels <a href="#pone.0050437-Crdenas1">[40]</a>, <a href="#pone.0050437-Wrheide2">[41]</a>, and throughout this paper, we will differentiate between Linnean taxa and those clades that have been provided with explicitly phylogenetic definitions by italicizing phylogenetically defined taxa and following them with a superscript p, as in <em>Demospongiae<sup>p</sup></em> (i.e., <em>PhyloCode</em> designations).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A major challenge to scientists working in this field has been the identification of appropriate markers for addressing the daunting task of dealing with ancient divergences among the diverse assortment of poriferan taxa. Evolutionary relationships across the most diverse class of Porifera, Demospongiae, have mainly been addressed with three sets of phylogenetic markers: ribosomal DNA sequences <a href="#pone.0050437-Borchiellini1">[17]</a>, <a href="#pone.0050437-Nichols1">[42]</a>, complete mitochondrial genome sequences <a href="#pone.0050437-Lavrov2">[43]</a>, and amino acid sequences that code for seven nuclear housekeeping genes <a href="#pone.0050437-Sperling2">[18]</a>, <a href="#pone.0050437-Sperling3">[44]</a>. A broad correspondence in inferences about demosponge phylogeny exists between these three sets of data (see <a href="#s3">discussion</a> below), but both of the latter two sets of data have been sampled from a far more limited number of taxa. The Porifera Tree of Life project (<a href="http://www.portol.org">www.portol.org</a>) employs a variety of tools to integrate morphological and molecular data and to expand the diversity of sponge taxa used to elucidate all levels of sponge phylogeny. In this study, we report findings based on a significant expansion (38 new samples from 38 species representing 30 families, including 21 families newly sampled) of the nuclear housekeeping gene dataset first developed for metazoan-wide phylogenetic and molecular dating analyses <a href="#pone.0050437-Peterson1">[45]</a>, <a href="#pone.0050437-Peterson2">[46]</a> and later applied by Sperling et al. <a href="#pone.0050437-Sperling2">[18]</a>, <a href="#pone.0050437-Sperling3">[44]</a> to sponges, with a thorough taxonomic vetting process and a slightly modified phylogenetic analysis focused on relationships within <em>Demospongiae<sup>p</sup></em>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Extraction of high quality RNA for subsequent cDNA synthesis and cloning was a significant hurdle, curtailing use of some samples (e.g., lithistids), even though a large number of archived specimens were available for potential study <a href="#pone.0050437-Reed1">[47]</a>. Several hundred cDNAs were cloned and sequenced, but only 159 usable sequences were generated due to the amplification of non-sponge contaminants (<a href="#pone-0050437-t001">Tables 1</a>–<a href="#pone-0050437-t002">2</a>). We evaluated single gene phylogenies (ALD, ATPB, etc.) including all the members of each gene family that could be identified in GenBank (via reciprocal blasting) to identify and remove potential paralogs. Our dataset for phylogenetic analysis contains 2,033 amino acid characters and a total of 68 sponge species representing 48 of 137 accepted and recently proposed families of Porifera <a href="#pone.0050437-Morrow1">[38]</a>, <a href="#pone.0050437-Crdenas1">[40]</a>, <a href="#pone.0050437-vanSoest2">[48]</a>, including 51 species from 37 of 91 families recognized for Demospongiae (<a href="#pone-0050437-t001">Table 1</a>). The most appropriate models of amino acid evolution, as determined by ProtTest <a href="#pone.0050437-Abascal1">[49]</a> for the various datasets (i.e., all genes, each individual gene, etc.), nearly always involved some variant of the LG matrix <a href="#pone.0050437-Le1">[50]</a> (<a href="#pone-0050437-t003">Table 3</a>). Maximum likelihood mapping, performed for each gene under the best fitting model, among those implemented in Treepuzzle <a href="#pone.0050437-Schmidt1">[51]</a>, showed that each of the seven considered genes convey enough phylogenetic signal to be considered potentially useful phylogenetic markers to resolve the relationships within Demospongiae (<a href="#pone.0050437.s001">Figs. S1</a>, <a href="#pone.0050437.s002">S2</a>, <a href="#pone.0050437.s003">S3</a>, <a href="#pone.0050437.s004">S4</a>, <a href="#pone.0050437.s005">S5</a>, <a href="#pone.0050437.s006">S6</a>, <a href="#pone.0050437.s007">S7</a>). Bayesian cross-validation <a href="#pone.0050437-Stone1">[52]</a> analyses showed that the CAT based models (CAT and CAT-GTR) fit our dataset significantly better than any empirical site-homogeneous time reversible model tested (WAG+G, and LG+G). Cross-validation also showed that the CAT-based models fit the data better than the more complex site-homogeneous time reversible model: the mechanistic amino acid-GTR (<a href="#pone-0050437-t004">Table 4</a>) model. Accordingly, hypothesized relationships obtained with homogeneous time-reversible models (e.g. LG or GTR), where differing from those obtained in our CAT and particularly CAT-GTR analysis, could be considered inferior. That said, just five of the resolved nodes in the Bayesian analysis contradict those in the ML-based topology and none of these have pp values&gt;0.90.</p>
<div class="figure" id="pone-0050437-t001"><div class="img"><a name="pone-0050437-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Annotated list of samples and sequences used for analysis. New sequences and samples are indicated in bold.</span></strong></p><span>doi:10.1371/journal.pone.0050437.t001</span></div><div class="figure" id="pone-0050437-t002"><div class="img"><a name="pone-0050437-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Summary of genes and taxa for analysis<span class="xref"><sup>*</sup></span> by poriferan clade.</span></strong></p><span>doi:10.1371/journal.pone.0050437.t002</span></div><div class="figure" id="pone-0050437-t003"><div class="img"><a name="pone-0050437-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Amino acid model selection, used for maximum likelihood searches on different datasets<span class="xref"><sup>*</sup></span>.</span></strong></p><span>doi:10.1371/journal.pone.0050437.t003</span></div><div class="figure" id="pone-0050437-t004"><div class="img"><a name="pone-0050437-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Model cross validation performed using CAT-GTR as the reference model.</span></strong></p><span>doi:10.1371/journal.pone.0050437.t004</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>The partitioned ML analysis of the combined data had the same topology as that found when assuming a single model of amino acid evolution (LG+F+G). Additionally, no major differences were found when comparing a Bayesian analysis performed under LG+G, the ML analysis performed using LG+F+G, and the ML analysis performed using multiple partitions. We used this topology as the reference point for comparing the different analyses (<a href="#pone-0050437-g001">Fig. 1</a>). The Bayesian topology (<a href="#pone-0050437-g002">Fig. 2</a>) is highly consistent with the ML-based topology (<a href="#pone-0050437-t005">Table 5</a>). Each of the single-gene ML topologies (<a href="#pone.0050437.s008">Figs. S8</a>, <a href="#pone.0050437.s009">S9</a>, <a href="#pone.0050437.s010">S10</a>, <a href="#pone.0050437.s011">S11</a>, <a href="#pone.0050437.s012">S12</a>, <a href="#pone.0050437.s013">S13</a>, <a href="#pone.0050437.s014">S14</a>) differs from that derived from the combined dataset. An ordered ranking of how well the single-gene topologies match our overall hypothesis, based on nodal difference is: PFK, TPI, ALD, MAT, ATPB, CAT and EF1A (<a href="#pone-0050437-t005">Table 5</a>). This performance is also reflected in a tabulation of whether notable clades were recovered in the single-gene topologies (<a href="#pone-0050437-t006">Table 6</a>), where ATPB, CAT and EF1A recovers less than half of a set of reference clades in the topology based on the combined data. ML analyses serially excluding CAT, EF1A, and ATPB resulted in topologies (<a href="#pone.0050437.s015">Figs. S15</a>, <a href="#pone.0050437.s016">S16</a>, <a href="#pone.0050437.s017">S17</a>) that are highly consistent with the tree based on the analysis of combined data (<a href="#pone-0050437-t005">Table 5</a>–<a href="#pone-0050437-t006">6</a>). A supertree analysis was performed to evaluate the extent to which the principal signal <a href="#pone.0050437-Pisani1">[53]</a> in the single-gene partitions differed from the signal in the gene concatenation and the results showed a substantial level of agreement (<a href="#pone.0050437.s018">Fig. S18</a>).</p>
<div class="figure" id="pone-0050437-g001"><div class="img"><a name="pone-0050437-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0050437.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0050437.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Hypothesis of demosponge relationships based on maximum likelihood analysis of seven nuclear housekeeping genes.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Topology rooted on three cnidarians and the placozoan <em>Trichoplax</em>. Bootstrap indices (400 replicates) are shown at each node, with those exceeding 70 in bold. New taxa added as part of the PorToL project are indicated in bold; new taxa added from EST/genomics projects are indicated with a single asterisk; and taxa with new identifications after examination of the voucher specimen are marked with two asterisks. Clade names in italics followed by a superscript p have been phylogenetically defined in other studies (see text).</p>
<span>doi:10.1371/journal.pone.0050437.g001</span></div><div class="figure" id="pone-0050437-g002"><div class="img"><a name="pone-0050437-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0050437.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0050437.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Hypothesis of demosponge relationships based on Bayesian analysis of seven nuclear housekeeping genes.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Topology rooted on three cnidarians and the placozoan <em>Trichoplax</em>. Posterior probabilities are shown at each node, with those exceeding 0.90 in bold. New taxa added as part of the PorToL project are indicated in bold; new taxa added from EST/genomics projects are indicated with a single asterisk; and taxa with new identifications after examination of the voucher specimen are marked with two asterisks. Clade names in italics followed by a superscript p have been phylogenetically defined in other studies (see text).</p>
<span>doi:10.1371/journal.pone.0050437.g002</span></div><div class="figure" id="pone-0050437-t005"><div class="img"><a name="pone-0050437-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Nodal differences between reference topology (ML assuming LG+G+F) and topologies derived from different datasets<span class="xref"><sup>*</sup></span> and analyses.</span></strong></p><span>doi:10.1371/journal.pone.0050437.t005</span></div><div class="figure" id="pone-0050437-t006"><div class="img"><a name="pone-0050437-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0050437" data-uri="info:doi/10.1371/journal.pone.0050437.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0050437.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0050437.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0050437.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0050437.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Comparison of clades found in NHK7<span class="xref"><sup>*</sup></span> ML topology with those revealed in single-gene and other analyses.<span class="xref"><sup>*</sup></span></span></strong></p><span>doi:10.1371/journal.pone.0050437.t006</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>Nodal support for the ML-based phylogenetic hypothesis (<a href="#pone-0050437-g001">Fig. 1</a>) varies widely; 46 of 70 nodes have bootstrap support (bs) exceeding 70%. Similarly, although generally higher in magnitude, posterior probability (pp) values in the Bayesian topology are not universally high, with 44 of 70 nodes having values exceeding 0.90 (<a href="#pone-0050437-g002">Fig. 2</a>).</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>To test whether some of our results could be attributed to tree reconstruction artifacts we performed a variety of analyses. We first built trees using differently fitting models (WAG, LG, GTR, CAT, and CAT-GTR) and compared their results. This analysis indicated an important area of disagreement with reference to the relationships between <em>Keratosa<sup>p</sup></em> and <em>Myxospongiae<sup>p</sup></em> (see <a href="#s3">discussion</a>). We performed a posterior predictive analysis to identify compositionally heterogeneous taxa. This analysis indicated that many taxa in the dataset are, indeed, compositionally heterogeneous (<a href="#pone.0050437.s024">Table S1</a>). The 6-categories Dayhoff recoding strategy is commonly used to ease compositional heterogeneity. We recoded our dataset using the 6-categories Dayhoff strategy and performed a posterior predictive analysis and found that the Dayhoff recoding eliminated almost all heterogeneity from the data (<a href="#pone.0050437.s025">Table S2</a>). CAT-GTR analyses of the Dayhoff recoded dataset found a tree (<a href="#pone.0050437.s019">Fig. S19</a>) that is highly comparable with the CAT-GTR tree of <a href="#pone-0050437-g002">Fig. 2</a> (non-recoded data). However, results of the Bayesian analysis using Dayhoff recoded data and assuming GTR (<a href="#pone.0050437.s020">Fig. S20</a>) contains a key difference. In the Dayhoff recoded GTR analysis <em>Myxospongiae<sup>p</sup></em> is not the sister group of <em>Keratosa<sup>p</sup></em> but the sister group of all the other Demospongiae (albeit with a low PP). Analyses performed after excluding compositionally heterogeneous species, fast-evolving sites, or the outgroups consistently reiterate the results of our Bayesian analysis (compare <a href="#pone-0050437-g002">Fig. 2</a> with <a href="#pone.0050437.s021">Figs. S21</a>, <a href="#pone.0050437.s022">S22</a>, <a href="#pone.0050437.s023">S23</a>).</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Sponge Classes</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Analyses of the seven nuclear housekeeping gene dataset provide strong support for each of the four major clades of sponges assigned the rank of class (Calcarea, Demospongiae, Hexactinellida, and Homoscleromorpha). Because we did not include non-metazoan outgroups our results cannot be used to assess sponge monophyly. Concerning the relationships among the four sponge classes, support is generally poor. Our tree does not recover <em>Silicea<sup>p</sup></em> (<em>Demospongiae<sup>p</sup></em>+<em>Hexactinellida<sup>p</sup></em>), which has been supported in a great deal of other works based on disparate datasets <a href="#pone.0050437-Sperling1">[4]</a>, <a href="#pone.0050437-Sperling2">[18]</a>, <a href="#pone.0050437-Philippe1">[19]</a>, <a href="#pone.0050437-Collins1">[28]</a>, <a href="#pone.0050437-Dohrmann1">[54]</a>, but instead places <em>Calcispongiae<sup>p</sup></em> with <em>Hexactinellida<sup>p</sup></em> (<a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>), most likely erroneously with low support (bs = 74%; pp = 0.68). Relationships within <em>Calcispongiae<sup>p</sup></em> and <em>Hexactinellida<sup>p</sup></em> are consistent with previous analyses <a href="#pone.0050437-Dohrmann1">[54]</a>–<a href="#pone.0050437-Dohrmann2">[56]</a>. As designed, our analyses do not provide any basis for inferring relationships among the sponge classes (as they do not include non-metazoan outgroups), but rather elucidate phylogenetic relationships within <em>Demospongiae<sup>p</sup></em> (<a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>).</p>


<h4>Major Demosponge Clades</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Hypotheses derived from our analyses of nuclear housekeeping gene data (<a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>) are fairly consistent with the so-called “G clades” originally derived from analysis of ribosomal DNA data <a href="#pone.0050437-Borchiellini1">[17]</a>, and largely recovered by mitochondrial genome <a href="#pone.0050437-Lavrov2">[43]</a> and nuclear housekeeping gene data <a href="#pone.0050437-Sperling2">[18]</a>. G1 and G2 correspond to <em>Keratosa<sup>p</sup></em> and <em>Myxospongiae<sup>p</sup></em>, respectively, following the names of Borchiellini et al. <a href="#pone.0050437-Borchiellini1">[17]</a>. One key difference between the results of these studies concerns the placement of the clade containing all freshwater sponges, <em>Spongillida<sup>p</sup></em>, phylogenetically defined in Cárdenas et al. <a href="#pone.0050437-Crdenas1">[40]</a>. Traditionally, these sponges were classified as the suborder Spongillina within the order Haplosclerida. However, ribosomal DNA and mitochondrial genome data suggested that <em>Spongillida<sup>p</sup></em> falls as the earliest diverging lineage of the “G4” clade. Sperling et al. <a href="#pone.0050437-Sperling2">[18]</a> found a similar clade, for which they provided a phylogenetic definition and the name <em>Democlavia<sup>p</sup></em> ( = subclass Heteroscleromorpha of Cárdenas et al. <a href="#pone.0050437-Crdenas1">[40]</a>), with the exception that <em>Spongillida<sup>p</sup></em> was found as the sister group of the marine haplosclerids. The marine haplosclerid taxa have consistently been shown to be a well-supported clade that has recently been phylogenetically defined and named <em>Haploscleromorpha<sup>p</sup></em> <a href="#pone.0050437-Crdenas1">[40]</a>.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>This study finds strong support at nearly all deep nodes within <em>Demospongiae<sup>p</sup></em> (<a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>), even with our more diverse taxon sampling. The clear distinction of these clades indicates that the divergence among these groups is likely ancient <a href="#pone.0050437-Sperling1">[4]</a>. Thus, future genomic exploration within <em>Demospongiae<sup>p</sup></em> will be guided by these emerging phylogenetic results so as to make best use of the comparative method. To be especially useful for rank-based taxonomy and nomenclature, type species within genera and type genera within families (e.g., our sampling of <em>Spongia officinalis</em>, <em>Halisarca dujardini</em>, and <em>Desmacella pumilio</em>) should be targeted whenever possible. Also, to the extent possible, type species should be collected from their respective type localities for maximum taxonomic and nomenclatural utility. For phylogenetic nomenclature, ‘specifiers’ (i.e., species, specimens or apomorphies used in <em>PhyloCode</em> definitions) should be targeted. Of course, when species are used as specifiers (which has so far usually been the case for poriferan names), their name-bearing type specimens are <em>de facto</em> specifiers (<em>PhyloCode</em>, Note 13.2.2.).</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Nuclear housekeeping gene data strongly support an as yet unnamed clade containing the groups of demosponges with silica-mineralized skeletons: <em>Democlavia<sup>p</sup></em>, <em>Haploscleromorpha<sup>p</sup></em>, and <em>Spongillida<sup>p</sup></em> (<a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>), in accordance with other analyses of ribosomal genes <a href="#pone.0050437-Borchiellini1">[17]</a>, complete mitochondrial genomes <a href="#pone.0050437-Lavrov2">[43]</a>, and a smaller dataset of nuclear housekeeping genes <a href="#pone.0050437-Sperling2">[18]</a>. Our ML and Bayesian analyses provide equivocal results concerning the phylogenetic relationships of <em>Keratosa<sup>p</sup></em> and <em>Myxospongiae<sup>p</sup></em>. A sister group relationship between <em>Keratosa<sup>p</sup></em> and <em>Myxospongiae<sup>p</sup></em> has been suggested, with only modest support, based on analyses of 18S rRNA genes <a href="#pone.0050437-Borchiellini1">[17]</a> and complete mitochondrial genomes <a href="#pone.0050437-Lavrov2">[43]</a> but has also been contradicted by earlier Bayesian analyses of nuclear housekeeping genes <a href="#pone.0050437-Sperling1">[4]</a>, <a href="#pone.0050437-Sperling2">[18]</a>, <a href="#pone.0050437-Sperling3">[44]</a>. Our ML topology (<a href="#pone-0050437-g001">Fig. 1</a>) shows <em>Keratosa<sup>p</sup></em> and <em>Myxospongiae<sup>p</sup></em> [which both lack mineralized skeletons (with the exception of siliceous microscleres in <em>Chondrilla</em> within <em>Myxospongiae<sup>p</sup></em>: Chondrosida)] as a clade that is sister to the mineralized sponges. In contrast, the Bayesian analysis (<a href="#pone-0050437-g002">Fig. 2</a>) identifies <em>Myxospongiae<sup>p</sup></em> as the earliest diverging clade of <em>Demospongiae<sup>p</sup></em>, and shows <em>Keratosa<sup>p</sup></em> as the sister group to the mineralized groups. It is important to note, however, that all site-homogeneous models (LG and GTR) display the <em>Keratosa<sup>p</sup></em>+<em>Myxospongiae<sup>p</sup></em> clade, while the site-heterogeneous CAT and CAT-GTR models (which fit the data better) support <em>Myxospongiae<sup>p</sup></em> as the sister group of all the other demosponges. Thus, model selection is responsible for this disagreement. Because the best fitting models suggest <em>Myxospongiae<sup>p</sup></em> is sister to the remaining demosponges, the contradicting results obtained using LG, GTR and WAG (<em>Keratosa<sup>p</sup></em>+<em>Myxospongiae<sup>p</sup></em>) are likely artifactual.</p>


<h4>
          <em>Keratosa<sup>p</sup></em>
        </h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>This clade is composed of members of the demosponge orders Dictyoceratida and Dendroceratida. Our sampling includes members of five of the six families: Dysideidae, Irciniidae, Spongiidae and Thorectidae in the former, Dictyodendrillidae in the latter. Ribosomal data <a href="#pone.0050437-Borchiellini1">[17]</a> indicate that Dendroceratida is monophyletic, but our results rely on a single genus (<em>Igernella</em>) so we cannot support or refute that result. The nuclear housekeeping gene data also fail to provide support for the monophyly of Dictyoceratida, a result that has also been obtained through the analysis of ribosomal data <a href="#pone.0050437-Redmond1">[35]</a>, <a href="#pone.0050437-Ma1">[57]</a>. We have conflicting results concerning Dictyoceratida, with our ML-topology (<a href="#pone-0050437-g001">Fig. 1</a>) suggesting that dendroceratids are derived from within a paraphyletic Dictyoceratida and the Bayesian tree having a poorly supported monophyletic Dictyoceratida. The key taxon, from the perspective of this analysis, is the representative of Dysideidae. All the other dictyoceratids in our study, representing Irciniidae, Spongiidae, and Thorectidae, always form a well-supported clade. It is interesting to note that when the worst performing markers (CAT, EF1A, and ATPB) are sequentially removed from analysis, Dictyoceratida, including our representative of Dysideidae, forms a monophyletic group with strong support (<a href="#pone.0050437.s015">Figs. S15</a>, <a href="#pone.0050437.s016">S16</a>, <a href="#pone.0050437.s017">S17</a>).</p>


<h4>
          <em>Myxospongiae<sup>p</sup></em>
        </h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Members of the orders Chondrosida and Verongida make up <em>Myxospongiae<sup>p</sup></em>. Our sampling includes both families of Chondrosida (Chondrillidae and Halisarcidae), the latter of which was previously placed in its own order Halisarcida (e.g., <a href="#pone.0050437-Ereskovsky1">[58]</a>). Within Verongida, just one of the four families of Verongida (i.e., Aplysinidae) is sampled. With the present taxon sampling, our analyses support monophyly of Chondrosida, a result not obtained by some analyses of ribosomal data <a href="#pone.0050437-Borchiellini1">[17]</a>, <a href="#pone.0050437-Erpenbeck1">[59]</a>, but found in others <a href="#pone.0050437-Redmond1">[35]</a>, <a href="#pone.0050437-Vacelet1">[60]</a>. However, our analysis lacks a representative of <em>Chondrosia</em>, which has proven to be a difficult taxon in relation to the question of Chondrosida monophyly <a href="#pone.0050437-Borchiellini1">[17]</a>, <a href="#pone.0050437-Erpenbeck1">[59]</a>. Similarly lacking a representative of the problematic <em>Chondrosia</em>, an analysis of complete mitochondrial genome data also supports a monophyletic <em>Chondrosida<sup>p</sup></em> <a href="#pone.0050437-Lavrov2">[43]</a>, which has nevertheless recently been given a phylogenetic definition <a href="#pone.0050437-Crdenas1">[40]</a>. Within Verongida, nuclear housekeeping genes support monophyly of Aplysinidae, for which we were able to sample each of its component genera (<a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>). Relationships among the three aplysinid genera (<em>Verongula</em>, <em>Aplysina</em>, and <em>Aiolochroia</em>), however, are not well supported. Based on ribosomal data, Erwin and Thacker <a href="#pone.0050437-Erwin3">[61]</a> found that Aplysinidae is not monophyletic because <em>Verongula</em> grouped with members of Pseudoceratinidae and members of <em>Aiolochroia</em> grouped with Ianthellidae and Aplysinellidae. The absence of pseudoceratinids, ianthellids and aplysinellids from our samples prevents our analyses from testing these hypotheses, but if Erwin and Thacker's <a href="#pone.0050437-Erwin3">[61]</a> findings are true, they would suggest that our sampling represents a more disparate group of Verongida (<em>Aplysina</em> in Aplysinidae and <em>Verongula</em> in Pseudoceratinidae) than is suggested by current taxonomy (<em>Aplysina</em> and <em>Verongula</em> in Aplysinidae). Indeed, this phylogenetic result (i.e., that <em>Aplysina</em> and <em>Verongula</em> belong to distinct families) was recently verified with mitochondrial and nuclear markers by Erpenbeck et al. <a href="#pone.0050437-Erpenbeck1">[59]</a>.</p>


<h4><em>Haploscleromorpha<sup>p</sup></em> &amp; <em>Spongillida<sup>p</sup></em></h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>From a broad perspective, one of the most important outstanding questions in demosponge phylogenetics is the phylogenetic placement of the freshwater sponges, <em>Spongillida<sup>p</sup></em>, which is phylogenetically defined in Cárdenas et al. <a href="#pone.0050437-Crdenas1">[40]</a>. Traditional taxonomy based on morphology <a href="#pone.0050437-vanSoest3">[62]</a> and earlier analyses of nuclear housekeeping genes <a href="#pone.0050437-Sperling2">[18]</a> suggest a close relationship between <em>Spongillida<sup>p</sup></em> and the marine haplosclerids, <em>Haploscleromorpha<sup>p</sup></em>. In contrast, both mitochondrial genome and ribosomal data suggest that <em>Spongillida<sup>p</sup></em> is sister to the rest of the <em>Democlavia<sup>p</sup></em> <a href="#pone.0050437-Borchiellini1">[17]</a>, <a href="#pone.0050437-Redmond1">[35]</a>, <a href="#pone.0050437-Lavrov2">[43]</a>, <a href="#pone.0050437-Itskovich1">[63]</a>. The results here, for the most part, agree with the former hypothesis and specifically indicate that <em>Spongillida<sup>p</sup></em> is the earliest diverging lineage of the traditional order Haplosclerida (with high support, <a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>). An exception to this result is one of the single gene analyses (ALD, <a href="#pone.0050437.s008">Fig. S8</a>), which found <em>Spongillida<sup>p</sup></em> branching among democlaviid taxa, albeit with no support. Limited taxon sampling, and in particular, the fact that our analyses do not include any representatives of the democlaviid family Scopalinidae (which was recently suggested by Morrow et al. <a href="#pone.0050437-Morrow1">[38]</a> to have a close relationship to the freshwater sponges), could explain these contradicting results. In any event, it is fairly clear that <em>Spongillida<sup>p</sup></em> is a distinct lineage from the marine haplosclerids.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>Our sampling within <em>Haploscleromorpha<sup>p</sup></em> represents five of the six accepted families. Monophyletic haplosclerid suborders Petrosina and Haplosclerina were not recovered (although support values are somewhat low at some of the deeper branches of the clade), corroborating the results of McCormack et al. <a href="#pone.0050437-McCormack1">[64]</a> and Redmond et al. <a href="#pone.0050437-Redmond1">[35]</a>, <a href="#pone.0050437-Redmond2">[37]</a>. Not surprisingly, given that studies with denser taxon sampling have shown widespread polyphyly of subtaxa within this group <a href="#pone.0050437-Redmond1">[35]</a>, <a href="#pone.0050437-Redmond2">[37]</a>, <a href="#pone.0050437-Raleigh1">[65]</a>, we find both Petrosiidae and Niphatidae to be polyphyletic. Even at the generic level, <em>Amphimedon</em> (Niphatidae) is revealed to be polyphyletic. <em>Amphimedon queenslandica</em>, whose genome has been sequenced <a href="#pone.0050437-Srivastava1">[66]</a>, clusters with <em>Callyspongia vaginalis</em> (Callyspongiidae) with high support, suggesting that the taxonomy of this important model organism remains confused, corroborating evidence from ribosomal data <a href="#pone.0050437-Redmond1">[35]</a>, <a href="#pone.0050437-Redmond2">[37]</a>.</p>


<h4>
          <em>Democlavia<sup>p</sup></em>
        </h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p><em>Democlavia<sup>p</sup></em> is the most species-rich (roughly 75% of demosponge species; <a href="#pone.0050437-Morrow1">[38]</a>) and diverse of the major demosponge clades, and includes the traditional orders Agelasida, Astrophorida, Hadromerida, Halichondrida, Poecilosclerida, and Spirophorida <a href="#pone.0050437-vanSoest2">[48]</a>, several of which are already thought to not be monophyletic (as discussed below). As such, the systematics of <em>Democlavia<sup>p</sup></em> presents many challenges, but important breakthroughs are being made in understanding the phylogeny of this clade based on increasingly taxon-rich analyses of ribosomal RNA and mitochondrial CO1 data <a href="#pone.0050437-Morrow1">[38]</a>. Our nuclear housekeeping gene dataset and analyses provide an opportunity to test hypotheses arising from these alternative sets of data and suggest new hypotheses where previous results have provided no resolution.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>Our analyses reveal a well-supported clade containing members of Astrophorina and Spirophorina (suborder designations for these taxa, following <a href="#pone.0050437-Crdenas1">[40]</a>), including our only sampled lithistid (<em>Microscleroderma</em> sp. nov.). Other analyses of ribosomal and mitochondrial data have revealed the same clade <a href="#pone.0050437-Borchiellini1">[17]</a>, <a href="#pone.0050437-Redmond1">[35]</a>, <a href="#pone.0050437-Nichols1">[42]</a>, <a href="#pone.0050437-Addis1">[67]</a>–<a href="#pone.0050437-Erpenbeck2">[69]</a>, the phylogenetically defined <em>Tetractinellida<sup>p</sup></em> <a href="#pone.0050437-Borchiellini1">[17]</a>, <a href="#pone.0050437-Crdenas1">[40]</a>. Although modest in support, our analyses always suggest that T<em>etractinellida<sup>p</sup></em> is sister to the remaining members of <em>Democlavia<sup>p</sup></em>. Our sampling of sub-order Astrophorina includes two of the six families, Ancorinidae (<em>Dercitus</em>, recently transferred from Pachastrellidae by Cárdenas et al. <a href="#pone.0050437-Crdenas2">[70]</a>) and Geodiidae (<em>Geodia tumulosa</em> and <em>Geostelletta<sup>p</sup> fibrosa</em>), as well as an <em>incertae sedis</em> taxon, <em>Characella</em> aff. <em>connectens</em>, which was also formerly assigned to family Pachastrellidae. The latter three species form a well-supported clade, but no specific position for our representative of Ancorinidae within <em>Tetractinellida<sup>p</sup></em> is supported (<a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>). The family Pachastrellidae <em>sensu</em> Maldonado <a href="#pone.0050437-Maldonado1">[71]</a> is based on a plesiomorphic character (streptasters; <a href="#pone.0050437-Crdenas2">[70]</a>) so it is no surprise that our results confirm that <em>Characella</em> and <em>Dercitus</em> do not have an especially close relationship.</p>
<a id="article1.body1.sec3.sec6.p3" name="article1.body1.sec3.sec6.p3"></a><p>Our analyses include two representatives of Spirophorina – <em>Cinachyrella</em> sp., representing the family Tetillidae, and the lithistid <em>Microscleroderma</em> sp. nov., representing the family Scleritodermidae – but there is no support for the group being monophyletic. The lithistids are a taxonomically rich group sharing a common growth form (skeleton of interlocked desmas), with 13 recognized families. Lithistids have always presented taxonomic challenges from morphological perspectives (see 72) and the redistribution of its members to different sponge clades has been proposed for quite some time <a href="#pone.0050437-Pisera1">[72]</a>, <a href="#pone.0050437-KellyBorges1">[73]</a> and continues <a href="#pone.0050437-Crdenas1">[40]</a>. In this vein, the lithistid family Desmanthidae appears to be closely related to Dictyonellidae <a href="#pone.0050437-Morrow1">[38]</a>. The presence of sigmaspires in Scleritodermidae <a href="#pone.0050437-Pisera1">[72]</a> is consistent with this group being reallocated to Spirophorina within <em>Tetractinellida<sup>p</sup></em> <a href="#pone.0050437-Crdenas1">[40]</a>.</p>
<a id="article1.body1.sec3.sec6.p4" name="article1.body1.sec3.sec6.p4"></a><p>Another well-supported alliance of taxa includes most members of order Poecilosclerida that we have sampled, specifically representatives of Coelosphaeridae, Crambeidae, Hymedesmiidae, Microcionidae, Mycalidae, and Tedaniidae (<a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>). Monophyly of Poecilosclerida has been found in several analyses of ribosomal data <a href="#pone.0050437-Borchiellini1">[17]</a>, <a href="#pone.0050437-Redmond1">[35]</a>, <a href="#pone.0050437-Nichols1">[42]</a>, <a href="#pone.0050437-Kober1">[74]</a>, but more recent studies with greater taxon sampling have shown the group to be polyphyletic <a href="#pone.0050437-Morrow1">[38]</a>, <a href="#pone.0050437-Erpenbeck3">[75]</a>, as found here. Morrow et al. <a href="#pone.0050437-Morrow1">[38]</a> demonstrated that the families Desmacellidae and Raspailiidae should be removed from Poecilosclerida. Our results support this action, as our representatives of these families branch deeper within <em>Democlavia<sup>p</sup></em> (<a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>). Unfortunately, these data do not provide strong support for relationships within this poecilosclerid group, which remains the most species-rich order and therefore one of the more challenging clades within <em>Demospongiae<sup>p</sup></em>.</p>
<a id="article1.body1.sec3.sec6.p5" name="article1.body1.sec3.sec6.p5"></a><p>The sister group to Poecilosclerida (<em>sensu</em> 38) consists of most of our sampled hadromerids as well as the family Halichondriidae from the order Halichondrida. A similar relationship was derived in Morrow et al. <a href="#pone.0050437-Morrow1">[38]</a>. Within this clade, three hadromerids, <em>Cliona</em> (Clionaidae), <em>Placospongia</em> (Placospongiidae), and <em>Spirastrella</em> (Spirastrellidae) form a well-supported clade. In turn, this clade is revealed to have a relatively well-supported relationship with the families Halichondriidae and Suberitidae. The latter two families, currently classified within Halichondrida and Hadromerida, respectively, have long been known to have a close relationship <a href="#pone.0050437-Chombard1">[27]</a>. Interestingly, the hadromerid <em>Tethya</em> (Tethyidae) consistently branches with this alliance of Suberitidae, Halichondriidae, and the hadromerids (representing Clionaidae, Placospongiidae and Spirastrellidae) albeit with limited support. One other hadromerid in our analysis, <em>Polymastia tenax</em>, falls outside this clade, a peculiar result given that Polymastiidae is considered among the “core” components of Hadromerida <a href="#pone.0050437-Hooper3">[76]</a>. In the 28S-based analysis of Morrow et al. <a href="#pone.0050437-Morrow1">[38]</a>, Polymastiidae emerged as a distinct clade, sister to Suberitidae plus Halichondriidae but with low support, whereas their analysis of CO1 data recovered a clade with Polymastiidae sister to the hadromerid families Tethyidae, Hemiasterellidae, and Clionaidae, but again with only low support.</p>
<a id="article1.body1.sec3.sec6.p6" name="article1.body1.sec3.sec6.p6"></a><p>The monophyly of <em>Agelasida<sup>p</sup></em> is well supported. This result is obtained only after taking into account recent findings made by Gazave et al. <a href="#pone.0050437-Gazave1">[36]</a>, who provided a phylogenetic definition of the clade, and corroborated by Morrow et al. <a href="#pone.0050437-Morrow1">[38]</a>. In light of polyphyly of <em>Axinella</em> (order Axinellida), Gazave et al. <a href="#pone.0050437-Gazave1">[36]</a> erected the taxon <em>Cymbaxinella<sup>p</sup></em> for those species, including <em>Axinella corrugata</em> sampled here, with a close relationship to <em>Agelas</em> (family Agelasidae). With broader taxon sampling, Morrow et al. <a href="#pone.0050437-Morrow1">[38]</a> established the new family Hymerhabdiidae for this same clade within <em>Agelasida<sup>p</sup></em>. In contrast with this study <a href="#pone.0050437-Morrow1">[38]</a>, however, nuclear housekeeping gene data do not provide further support for a sister group relationship between <em>Agelasida<sup>p</sup></em> and the clade containing the core poecilosclerids, hadromerids and Halichondriidae. The only representative of order Axinellida in our analysis is <em>Ectyoplasia</em>; the species belongs to the family Raspailiidae, which was moved from Poecilosclerida to Axinellida by Morrow et al. <a href="#pone.0050437-Morrow1">[38]</a>. That study <a href="#pone.0050437-Morrow1">[38]</a> also found that representatives of Desmacellidae fell in two groups, a finding we also recovered given that <em>Desmacella</em> and <em>Biemna</em> did not exhibit a particularly close relationship. It is important to note that our analysis includes the type species of <em>Desmacella</em>. Nuclear housekeeping gene data provide modest support for a relationship between <em>Desmacella</em> and the family Dictyonellidae (<a href="#pone-0050437-g001">Figs. 1</a>–<a href="#pone-0050437-g002">2</a>).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Conclusions"></a><h3>Conclusions</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>As with any phylogenetic analysis, the hypotheses presented here do not represent the final statement on demosponge phylogeny. In particular, the aforementioned gaps in taxonomic sampling limit the extent to which these analyses are able to assess interesting and relevant hypotheses of demosponge relationships. Nonetheless, this analysis makes several important strides forward. First, our results bolster previous claims of the efficacy of the nuclear housekeeping gene marker set <a href="#pone.0050437-Sperling3">[44]</a>, albeit at a high cost in effort. Analyses of these data with enhanced taxon sampling confirm numerous phylogenetic hypotheses derived from ribosomal DNA and mitochondrial markers. Most importantly, this boosts overall confidence in the emerging picture of demosponge systematics and phylogenetics that has largely been based on ribosomal and mitochondrial markers, which are more readily obtained from sponge samples. Nevertheless, there are still key points of difference, for example the position of the freshwater <em>Spongillida<sup>p</sup></em> clade, that remain to be tested by new datasets, and numerous open questions not yet satisfactorily answered by any phylogenetic analyses, such as the position of <em>Tetractinellida<sup>p</sup></em> within <em>Democlavia<sup>p</sup></em>, and the relationships among <em>Keratosa<sup>p</sup></em>, <em>Myxospongiae<sup>p</sup></em>, and the clade consisting of <em>Democlavia<sup>p</sup></em>, <em>Haploscleromorpha<sup>p</sup></em>, and <em>Spongillida<sup>p</sup></em>. A final important advance of this study is that incorporates a diverse set of sponge systematicists engaged in transforming the taxonomy (both <em>PhyloCode</em>-based and more traditional approaches) used to describe demosponge diversity. As a new understanding of demosponge relationships emerges, the names – and possibly the rules by which we erect and use them – must change <a href="#pone.0050437-Morrow1">[38]</a>–<a href="#pone.0050437-Wrheide2">[41]</a>.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Ethics Statement</h4>
<a id="article1.body1.sec5.sec1.p1" name="article1.body1.sec5.sec1.p1"></a><p>In accordance with policy and legal requirements associated with specimens vouchered in the collections of the Smithsonian US National Museum of Natural History (NMNH), Harbor Branch Oceanographic Institute (HBOI), Harvard Museum of Comparative Zoology (MCZ), and Zoological Museum Bergen Norway (ZMBN), all collections involved in this study were obtained with all appropriate and relevant permits. Specifically, samples from Panama were collected under a Marine Collecting Permit provided by The Republic of Panama; samples from the State of Florida were collected under a Florida recreational resident saltwater fishing license issued from Florida Fish and Wildlife Conservation Commission; and one sample from Honduras was collected with the permission of Rosa del Carmen Garcia, Directora General de Pesca y Acuicultura. No permits were required to collect sponge specimens in US territorial waters outside state boundaries, the Catalan coast of Spain, Vancouver Island, Canada, or Norway.</p>


<h4>Sample and sequence collection</h4>
<a id="article1.body1.sec5.sec2.p1" name="article1.body1.sec5.sec2.p1"></a><p>Samples were collected from a variety of locations and stored as described below or obtained from frozen collections at the Harbor Branch Oceanographic Institute-Florida Atlantic University (<a href="#pone-0050437-t001">Table 1</a>; <a href="http://PorToL.org/NHK7data">http://PorToL.org/NHK7data</a>). To obtain RNA of sufficient quality and quantity, when possible, fresh material was collected and preserved via one of several methods. One involved placing fresh material in cold 75% ethanol with liquid changes occurring after 15 min, 1 hour and 4 hours. When available, material was also placed in RNAlater (Invitrogen), directly in TRIzol® (Invitrogen) reagent, following the manufacturer's instructions, or in liquid nitrogen. In most cases, the tissue placed directly in TRIzol® or frozen in liquid nitrogen yielded the highest quality and/or quantity of RNA. However, the most practical storage method in the field was 75% ethanol preservation and in most cases this was suitable for RNA extraction and subsequent polymerase chain reaction (PCR) amplifications from cDNA.</p>
<a id="article1.body1.sec5.sec2.p2" name="article1.body1.sec5.sec2.p2"></a><p>Following Sperling et al. <a href="#pone.0050437-Sperling2">[18]</a>, <a href="#pone.0050437-Sperling3">[44]</a> total RNA was isolated using a one-step TRIzol® method (Invitrogen), and cDNA was synthesized from 1–2 µg RNA using RETROSCRIPT® (Ambion) reverse transcriptase using both random decamers and oligo dT primers, which were then pooled. PCR was used to amplify 7 nuclear-encoded genes: aldolase (ALD), ATP synthase beta chain (ATPB), catalase (CAT), elongation factor 1-alpha (EF1alpha), methionine adenosyltransferase (MAT), phosphofructokinase (PFK), and triose-phosphate isomerase (TPI). All primer sequences for initial PCR of housekeeping genes can be found in Sperling et al. <a href="#pone.0050437-Sperling3">[44]</a>. In many cases, however, it was necessary to use nested PCR primers if amplification and re-amplification of housekeeping gene products was not possible. <a href="#pone.0050437.s026">Table S3</a> provides primer sequences for nested amplifications of individual housekeeping genes. Primary or nested amplification products were cloned into PCR cloning vectors (pGEM®-T, Promega or TOPO TA®, Invitrogen) and individual clones were prepared for DNA sequencing using standard protocols.</p>
<a id="article1.body1.sec5.sec2.p3" name="article1.body1.sec5.sec2.p3"></a><p>After editing and trimming vector sequences with GENEIOUS <a href="#pone.0050437-Drummond1">[77]</a>, DNA sequences were assessed for gene and sponge identity via BLASTX or BLASTP queries <a href="#pone.0050437-Altschul1">[78]</a>, followed by preliminary single-gene phylogenetic analyses under the likelihood framework described below. The identification of likely paralogs followed standard procedures based on the generation of trees including all the members of each gene family that could be identified in GenBank (via reciprocal blasting). Within the context of these trees, paralogy groups were identified and only the sequences nesting within the selected orthology group were used. New sequences generated in this study have been submitted to GenBank (<a href="#pone-0050437-t001">Table 1</a>). Sequences are also available via the Porifera Tree of Life database (PorToL.org). In addition, voucher specimens for many of the sequences presented in Sperling et al. <a href="#pone.0050437-Sperling2">[18]</a>, <a href="#pone.0050437-Sperling3">[44]</a> were examined, resulting in several instances of updated taxonomic identification and classification (<a href="#pone-0050437-t001">Table 1</a>).</p>
<a id="article1.body1.sec5.sec2.p4" name="article1.body1.sec5.sec2.p4"></a><p>Nucleotide sequences were translated and aligned using MUSCLE <a href="#pone.0050437-Edgar1">[79]</a> and visualized in SEAVIEW (v. 4.3) <a href="#pone.0050437-Gouy1">[80]</a>. In addition to the new sequences, the initial alignment included data for sponges that had already been published (<a href="#pone-0050437-t001">Table 1</a>). Also, five species for which transcriptome data exist were also added to the dataset. Both mRNA and cDNA from <em>Corticium candelabrum</em>, <em>Petrosia ficiformis</em> and <em>Sycon coactum</em> were obtained using protocols available in Riesgo et al. <a href="#pone.0050437-Riesgo1">[81]</a>. <em>Sycon ciliatum</em> and <em>Leucosolenia complicata</em> sequences are derived from current genome and transcriptome sequencing projects for these species <a href="#pone.0050437-Fortunato1">[82]</a> and Adamska, unpublished). <em>De novo</em> assemblies of the reads obtained with Illumina GA (Illumina, Solexa, USA) were built with CLC Genomics Workbench 4 (CLCbio, MA, USA). Local blasts against the contig lists generated were used to search for the housekeeping genes. Initially, 50 outgroup taxa representing Bilateria, Ctenophora, Cnidaria, Placozoa and non-metazoan Opisthokonta were included in the analyses. However, preliminary phylogenetic analyses, conducted as described below, indicated that inferred demosponge relationships were robust to outgroup choice and therefore outgroups in the final dataset were reduced to the cnidarian taxa (<em>Acropora</em>, <em>Metridium</em> and <em>Nematostella</em>) and the placozoan <em>Trichoplax</em>. Approximately 40 positions in the alignment were manually excluded from analyses because they represented insertions present in one or a small number (&lt;5) of taxa.</p>


<h4>Phylogenetic Analyses</h4>
<a id="article1.body1.sec5.sec3.p1" name="article1.body1.sec5.sec3.p1"></a><p>For all gene trees we investigated the presence of significant clustering information using Maximum Likelihood Mapping <a href="#pone.0050437-Strimmer1">[83]</a> as implemented in Treepuzzle V. 5.2 <a href="#pone.0050437-Schmidt1">[51]</a>. The dataset was analyzed in both Bayesian and Maximum Likelihood (ML) frameworks. For the ML analyses, appropriate models of amino acid evolution were assessed using the Akaike Information Criterion (AIC), as implemented in ProtTest (v.2.4) <a href="#pone.0050437-Abascal1">[49]</a>. The computing cluster of the Smithsonian's Laboratories of Analytical Biology was used to run the parallelized version of RaxML <a href="#pone.0050437-Stamatakis1">[84]</a> to search for maximum likelihood (ML) topologies. We assumed the model that best fit our data according to the second-order AIC (AICc-1) with the exception that a proportion of invariant sites was not estimated (according to a recommendation in the RaxML manual). We also used RaxML to conduct bootstrap analyses (400 replicate searches) to assess nodal support. We searched for ML topologies using each gene separately as well as all genes combined. We analyzed the combined data a) assuming a single model for all the data and b) by assigning most appropriate models to each gene partition (mixed models).</p>
<a id="article1.body1.sec5.sec3.p2" name="article1.body1.sec5.sec3.p2"></a><p>Bayesian analyses were performed using the site-heterogeneous CAT-GTR+gamma in Phylobayes 3.3b <a href="#pone.0050437-Lartillot1">[85]</a>. This model was selected because Bayesian model selection, performed using 10-fold cross-validation <a href="#pone.0050437-Lartillot2">[86]</a>, showed that CAT-GTR best fitted our dataset, outperforming CAT, GTR and LG. The considered models were: WAG, LG, GTR, CAT, and CAT-GTR (all models used a gamma correction to account for rate heterogeneity among sites). The CAT based models (in this case CAT and CAT-GTR <a href="#pone.0050437-Lartillot2">[86]</a>) are mixture models developed to better take into account site-specific features of protein evolution. These models are thus expected to fit the data better than homogeneous time reversible models like LG and GTR <a href="#pone.0050437-Lartillot2">[86]</a>. Indeed, CAT based models have previously been shown to fit amino acid datasets better than other models and they have been shown to be highly effective at reducing systematic biases, like long branch attraction, which are well known to be very pervasive in deep time phylogenetics. In Phylobayes two independent analyses were run for 30,000 cycles sampling every 100 points. The analyses were considered converged when the largest discrepancy observed across all bipartitions (i.e. the maxdiff statistics) dropped below 0.15, despite the Phylobayes manual's suggestion that a chain has reached convergence when maxdiff &lt;0.3. Support values for the nodes recovered in the CAT-GTR analysis are expressed as posterior probabilities.</p>
<a id="article1.body1.sec5.sec3.p3" name="article1.body1.sec5.sec3.p3"></a><p>Comparisons were made between the different single-gene topologies and the Bayesian topology to the ML tree derived from the overall data. In addition, nodal differences were calculated, as measured by the root-mean-squared distance, in Topd (v.3.3) <a href="#pone.0050437-Puigb1">[87]</a>. Taxa that were missing data for some genes were pruned from the combined tree prior to calculating nodal differences. Topd was also used to conduct randomization analyses to test whether similarities between the various topologies and the combined ML topology were not greater than expected by chance. Finally, further ML searches were conducted by sequentially excluding the three genes that subtend the trees that are most distant from the tree derived from the concatenated dataset, as measured by subtracting the random nodal difference from the actual nodal difference. To further investigate the extent to which the principal signal <a href="#pone.0050437-Pisani1">[53]</a> in the single-genes corroborated the results of concatenated Bayesian and ML analyses, we performed a supertree analysis. The supertree was built using the Matrix Representation with Parsimony method <a href="#pone.0050437-Pisani1">[53]</a>. Input trees used for this analysis were, for each gene, the 400 bootstrap trees derived (see above) under ML. This set of 2800 input trees was bootstrapped to generate 100 replicate datasets, each of which scored 2800 trees using the software CLANN <a href="#pone.0050437-Creevey1">[88]</a>. For each bootstrapped dataset a bootstrap supertree was recovered and a majority rule consensus of the recovered bootstrap supertrees was built to estimate nodal support.</p>
<a id="article1.body1.sec5.sec3.p4" name="article1.body1.sec5.sec3.p4"></a><p>Finally, analyses were performed to test for tree reconstruction artifacts. More precisely we investigated the potential effect of long-branch attraction and compositional attraction on our results. We first investigated the effect of using alternative model of evolution on our results. We thus built trees (within a Bayesian framework) using models (WAG, LG, GTR, CAT, and CAT-GTR, each with a gamma correction) providing different levels of fit to the data and compared the trees we obtained. We tested whether the taxa in our dataset were compositionally heterogeneous performing a posterior predictive analysis (see for example <a href="#pone.0050437-Sperling2">[18]</a>) of compositional heterogeneity using Phylobayes under the CAT-GTR model. The posterior predictive analysis indicated that several taxa displayed a biased composition of their sites. This, if not addressed, can cause compositional artifacts. To test whether our results were affected by compositional biases we performed two analyses. First we analysed (under CAT-GTR) a dataset from which all compositionally heterogeneous taxa were excluded. This experiment has the downside of excluding potentially important taxa. Accordingly, a second experiment was performed in which our dataset was recoded using the Dayhoff scheme. Dayhoff recoding can alleviate compositional artifact, and a posterior predictive analysis of our Dayhoff-recoded dataset was performed (under CAT-GTR) to evaluate whether further compositionally biased taxa remained after the application of Dayhoff recoding. Finally, our Dayhoff recoded dataset was analysed using both a site-homogeneous (GTR) and a site heterogeneous (CAT-GTR) model.</p>
<a id="article1.body1.sec5.sec3.p5" name="article1.body1.sec5.sec3.p5"></a><p>To test for the potential effect of long-branch attraction artifacts we identified fast evolving sites in our dataset using the program Tiger <a href="#pone.0050437-Cummins1">[89]</a>. After that, sites that Tiger deemed as being fast evolving (bins 7 to 10) were excluded and the slowly evolving sites analysed in isolation. In addition to the site-stripping analysis, we also performed an analysis where all the outgroups to Demospongiae (including Hexactinellida) were removed.</p>

</div>

<div id="section6" class="section"><a id="s6" name="s6" toc="s6" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0050437"></div><a name="pone.0050437.s001" id="pone.0050437.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s001">Figure S1. </a></strong></p><a id="article1.body1.sec6.supplementary-material1.caption1.p1" name="article1.body1.sec6.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood Mapping shows ALD has signal to resolve unambiguously over 90% of the quartets that make up the ALD-derived tree. ALD cannot resolve 4.4% of the quartets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s001</p><a id="article1.body1.sec6.supplementary-material1.caption1.p2" name="article1.body1.sec6.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s002" id="pone.0050437.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s002">Figure S2. </a></strong></p><a id="article1.body1.sec6.supplementary-material2.caption1.p1" name="article1.body1.sec6.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood Mapping shows ATPB has signal to resolve unambiguously over 82% of the quartets that make up the ATPB-derived tree. ATPB cannot resolve 8% of the quartets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s002</p><a id="article1.body1.sec6.supplementary-material2.caption1.p2" name="article1.body1.sec6.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s003" id="pone.0050437.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s003">Figure S3. </a></strong></p><a id="article1.body1.sec6.supplementary-material3.caption1.p1" name="article1.body1.sec6.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood Mapping shows CAT has signal to resolve unambiguously over 82% of the quartets that make up the CAT-derived tree. CAT cannot resolve 9% of the quartets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s003</p><a id="article1.body1.sec6.supplementary-material3.caption1.p2" name="article1.body1.sec6.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s004" id="pone.0050437.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s004">Figure S4. </a></strong></p><a id="article1.body1.sec6.supplementary-material4.caption1.p1" name="article1.body1.sec6.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood Mapping shows EF1a has signal to resolve unambiguously over 76% of the quartets that make up the EF1a-derived tree. EF1a cannot resolve 12.3% of the quartets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s004</p><a id="article1.body1.sec6.supplementary-material4.caption1.p2" name="article1.body1.sec6.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s005" id="pone.0050437.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s005">Figure S5. </a></strong></p><a id="article1.body1.sec6.supplementary-material5.caption1.p1" name="article1.body1.sec6.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood Mapping shows MAT has signal to resolve unambiguously nearly 83% of the quartets that make up the MAT-derived tree. MAT cannot resolve 10.2% of the quartets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s005</p><a id="article1.body1.sec6.supplementary-material5.caption1.p2" name="article1.body1.sec6.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s006" id="pone.0050437.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s006">Figure S6. </a></strong></p><a id="article1.body1.sec6.supplementary-material6.caption1.p1" name="article1.body1.sec6.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood Mapping shows PFK has signal to resolve unambiguously over 71% of the quartets that make up the PFKtree. PFK cannot resolve 20.6% of the quartets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s006</p><a id="article1.body1.sec6.supplementary-material6.caption1.p2" name="article1.body1.sec6.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s007" id="pone.0050437.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s007">Figure S7. </a></strong></p><a id="article1.body1.sec6.supplementary-material7.caption1.p1" name="article1.body1.sec6.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood Mapping shows TPI has signal to resolve unambiguously over 76% of the quartets that make up the TPI-derived tree. TPI cannot resolve 15.8% of the quartets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s007</p><a id="article1.body1.sec6.supplementary-material7.caption1.p2" name="article1.body1.sec6.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s008" id="pone.0050437.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s008">Figure S8. </a></strong></p><a id="article1.body1.sec6.supplementary-material8.caption1.p1" name="article1.body1.sec6.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood topology based on ALD, with assumed model of LG+gamma.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s008</p><a id="article1.body1.sec6.supplementary-material8.caption1.p2" name="article1.body1.sec6.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s009" id="pone.0050437.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s009">Figure S9. </a></strong></p><a id="article1.body1.sec6.supplementary-material9.caption1.p1" name="article1.body1.sec6.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood topology based on ATPB, with assumed model of WAG+gamma.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s009</p><a id="article1.body1.sec6.supplementary-material9.caption1.p2" name="article1.body1.sec6.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s010" id="pone.0050437.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s010">Figure S10. </a></strong></p><a id="article1.body1.sec6.supplementary-material10.caption1.p1" name="article1.body1.sec6.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood topology based on CAT, with assumed model of LG+gamma.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s010</p><a id="article1.body1.sec6.supplementary-material10.caption1.p2" name="article1.body1.sec6.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s011" id="pone.0050437.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s011">Figure S11. </a></strong></p><a id="article1.body1.sec6.supplementary-material11.caption1.p1" name="article1.body1.sec6.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood topology based on EF1A, with assumed model of LG+F+gamma.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s011</p><a id="article1.body1.sec6.supplementary-material11.caption1.p2" name="article1.body1.sec6.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s012" id="pone.0050437.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s012">Figure S12. </a></strong></p><a id="article1.body1.sec6.supplementary-material12.caption1.p1" name="article1.body1.sec6.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood topology based on MAT, with assumed model of LG+gamma.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s012</p><a id="article1.body1.sec6.supplementary-material12.caption1.p2" name="article1.body1.sec6.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s013" id="pone.0050437.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s013">Figure S13. </a></strong></p><a id="article1.body1.sec6.supplementary-material13.caption1.p1" name="article1.body1.sec6.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood topology based on PFK, with assumed model of LG+gamma.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s013</p><a id="article1.body1.sec6.supplementary-material13.caption1.p2" name="article1.body1.sec6.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s014" id="pone.0050437.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s014">Figure S14. </a></strong></p><a id="article1.body1.sec6.supplementary-material14.caption1.p1" name="article1.body1.sec6.supplementary-material14.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood topology based on TPI, with assumed model of LG+gamma.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s014</p><a id="article1.body1.sec6.supplementary-material14.caption1.p2" name="article1.body1.sec6.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s015" id="pone.0050437.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s015">Figure S15. </a></strong></p><a id="article1.body1.sec6.supplementary-material15.caption1.p1" name="article1.body1.sec6.supplementary-material15.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood topology based on NHK6, with assumed model of LG+gamma.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s015</p><a id="article1.body1.sec6.supplementary-material15.caption1.p2" name="article1.body1.sec6.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s016" id="pone.0050437.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s016">Figure S16. </a></strong></p><a id="article1.body1.sec6.supplementary-material16.caption1.p1" name="article1.body1.sec6.supplementary-material16.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood topology based on NHK5, with assumed model of LG+gamma.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s016</p><a id="article1.body1.sec6.supplementary-material16.caption1.p2" name="article1.body1.sec6.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s017" id="pone.0050437.s017"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s017">Figure S17. </a></strong></p><a id="article1.body1.sec6.supplementary-material17.caption1.p1" name="article1.body1.sec6.supplementary-material17.caption1.p1"></a><p class="preSiDOI">Maximum Likelihood topology based on NHK4, with assumed model of LG+gamma.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s017</p><a id="article1.body1.sec6.supplementary-material17.caption1.p2" name="article1.body1.sec6.supplementary-material17.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s018" id="pone.0050437.s018"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s018">Figure S18. </a></strong></p><a id="article1.body1.sec6.supplementary-material18.caption1.p1" name="article1.body1.sec6.supplementary-material18.caption1.p1"></a><p class="preSiDOI">Consensus supertree derived from the input trees that represents the signal in the collection of the individual trees.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s018</p><a id="article1.body1.sec6.supplementary-material18.caption1.p2" name="article1.body1.sec6.supplementary-material18.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s019" id="pone.0050437.s019"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s019">Figure S19. </a></strong></p><a id="article1.body1.sec6.supplementary-material19.caption1.p1" name="article1.body1.sec6.supplementary-material19.caption1.p1"></a><p class="preSiDOI">Bayesian analysis of Dayhoff recoded data using CAT-GTR.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s019</p><a id="article1.body1.sec6.supplementary-material19.caption1.p2" name="article1.body1.sec6.supplementary-material19.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s020" id="pone.0050437.s020"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s020">Figure S20. </a></strong></p><a id="article1.body1.sec6.supplementary-material20.caption1.p1" name="article1.body1.sec6.supplementary-material20.caption1.p1"></a><p class="preSiDOI">Bayesian analysis of Dayhoff recoded data using GTR.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s020</p><a id="article1.body1.sec6.supplementary-material20.caption1.p2" name="article1.body1.sec6.supplementary-material20.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s021" id="pone.0050437.s021"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s021">Figure S21. </a></strong></p><a id="article1.body1.sec6.supplementary-material21.caption1.p1" name="article1.body1.sec6.supplementary-material21.caption1.p1"></a><p class="preSiDOI">Bayesian analysis using CAT-GTR, with all compositionally heterogenous taxa excluded.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s021</p><a id="article1.body1.sec6.supplementary-material21.caption1.p2" name="article1.body1.sec6.supplementary-material21.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s022" id="pone.0050437.s022"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s022">Figure S22. </a></strong></p><a id="article1.body1.sec6.supplementary-material22.caption1.p1" name="article1.body1.sec6.supplementary-material22.caption1.p1"></a><p class="preSiDOI">Bayesian analysis using CAT-GTR, excluding fast-evolving sites with Tiger software (“SlowFast Tree”).</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s022</p><a id="article1.body1.sec6.supplementary-material22.caption1.p2" name="article1.body1.sec6.supplementary-material22.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s023" id="pone.0050437.s023"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s023">Figure S23. </a></strong></p><a id="article1.body1.sec6.supplementary-material23.caption1.p1" name="article1.body1.sec6.supplementary-material23.caption1.p1"></a><p class="preSiDOI">Bayesian analysis using CAT-GTR, with no outgroups.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s023</p><a id="article1.body1.sec6.supplementary-material23.caption1.p2" name="article1.body1.sec6.supplementary-material23.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s024" id="pone.0050437.s024"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s024">Table S1. </a></strong></p><a id="article1.body1.sec6.supplementary-material24.caption1.p1" name="article1.body1.sec6.supplementary-material24.caption1.p1"></a><p class="preSiDOI">Results of the Posterior Predictive Analysis of the combined data set (all 7 genes) under the CAT GTR model. Taxa with a star are heterogeneous in composition.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s024</p><a id="article1.body1.sec6.supplementary-material24.caption1.p2" name="article1.body1.sec6.supplementary-material24.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s025" id="pone.0050437.s025"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s025">Table S2. </a></strong></p><a id="article1.body1.sec6.supplementary-material25.caption1.p1" name="article1.body1.sec6.supplementary-material25.caption1.p1"></a><p class="preSiDOI">An analysis of the Dayhoff recoded dataset (still under CAT-GTR). As expected, nearly all the heterogeneity is gone (compared to <a href="#pone.0050437.s024">Table S1</a>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s025</p><a id="article1.body1.sec6.supplementary-material25.caption1.p2" name="article1.body1.sec6.supplementary-material25.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0050437.s026" id="pone.0050437.s026"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0050437.s026">Table S3. </a></strong></p><a id="article1.body1.sec6.supplementary-material26.caption1.p1" name="article1.body1.sec6.supplementary-material26.caption1.p1"></a><p class="preSiDOI">Nested primers used to facilitate amplifications of 5 of the 7 genes analyzed in this work.</p>
<p class="siDoi">doi:10.1371/journal.pone.0050437.s026</p><a id="article1.body1.sec6.supplementary-material26.caption1.p2" name="article1.body1.sec6.supplementary-material26.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are indebted to the many sponge biologists who contributed samples, expertise, and sequences to this project. Special thanks go to Sebastian Joffre and Crystal Romero for their assistance in sponge RNA preparations, Sebastian Basile, Tandace Burkhart and Renee Potens with sequence data curation and bioinformatics analyses, and Chris Freeman for assistance in collecting specimens. We thank Martin Dohrmann for identifying hexactinellid samples and providing helpful comments on an earlier version of this MS. We also thank Gonzalo Giribet and Sally Leys for guidance provided to obtain all the transcriptomic sequencing for the species <em>Petrosia ficiformis</em>, <em>Sycon coactum</em>, and <em>Corticium candelabrum</em>.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MSH ALH JL KJP SP MCD RWT EAS AGC. Performed the experiments: MSH ALH JL MA AC ED BD DF JP D. Pohlmann NER SR AR ER ZR EAS MdS JET. Analyzed the data: MSH D. Pisani AGC. Contributed reagents/materials/analysis tools: SP JR. Wrote the paper: MSH ALH JL KJP MCD RWT MA NBE PC EH GLH SK CCM D. Pisani NER AR KR EAS AGC BP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0050437-Gehling1" id="pone.0050437-Gehling1"></a>Gehling JG, Rigby JK (1996) Long expected sponges from the neoproterozoic Ediacara fauna of South Australia. J Paleo 2: 185–195. <ul class="find" data-citedArticleID="6679805"><li><a href="http://www.crossref.org/guestquery/?auth2=Gehling&amp;atitle2=Long+expected+sponges+from+the+neoproterozoic+Ediacara+fauna+of+South+Australia&amp;auth=Gehling&amp;atitle=Long+expected+sponges+from+the+neoproterozoic+Ediacara+fauna+of+South+Australia" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gehling%5Bauthor%5D+AND+Long+expected+sponges+from+the+neoproterozoic+Ediacara+fauna+of+South+Australia" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGehling+%22Long+expected+sponges+from+the+neoproterozoic+Ediacara+fauna+of+South+Australia%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0050437-Love1" id="pone.0050437-Love1"></a>Love GD, Grosjean E, Stalvies C, Fike DA, Grotzinger JP, et al. (2009) Fossil steroids record the appearance of Demospongiae during the Cryogenian period. Nature 457: 718–721.
        doi: 10.1038/nature07673 <ul class="find" data-citedArticleID="6679807" data-doi="10.1038/nature07673"><li><a href="http://dx.doi.org/10.1038/nature07673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Love%5Bauthor%5D+AND+Fossil+steroids+record+the+appearance+of+Demospongiae+during+the+Cryogenian+period" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALove+%22Fossil+steroids+record+the+appearance+of+Demospongiae+during+the+Cryogenian+period%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0050437-Brain1" id="pone.0050437-Brain1"></a>Brain CKB, Prave AR, Hoffmann K-H, Fallick AE, Botha A, et al. (2012) The first animals: ca. 760-million-year-old sponge-like fossils from Namibia. S Afr J Sci 108: 658.
        doi: 10.4102/sajs.v108i1/2.658 <ul class="find" data-citedArticleID="6679809" data-doi="10.4102/sajs.v108i1/2.658"><li><a href="http://dx.doi.org/10.4102/sajs.v108i1/2.658" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Brain%5Bauthor%5D+AND+The+first+animals%3A+ca.+760-million-year-old+sponge-like+fossils+from+Namibia" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABrain+%22The+first+animals%3A+ca.+760-million-year-old+sponge-like+fossils+from+Namibia%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0050437-Sperling1" id="pone.0050437-Sperling1"></a>Sperling EA, Robinson JM, Pisani D, Peterson KJ (2010) Where's the glass? Biomarkers, molecular clocks, and microRNAs suggest a 200-Myr missing Precambrian fossil record of siliceous sponge spicules. Geobiology 8: 24–36.
        doi: 10.1111/j.1472-4669.2009.00225.x <ul class="find" data-citedArticleID="6679811" data-doi="10.1111/j.1472-4669.2009.00225.x"><li><a href="http://dx.doi.org/10.1111/j.1472-4669.2009.00225.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sperling%5Bauthor%5D+AND+Where%27s+the+glass%3F+Biomarkers%2C+molecular+clocks%2C+and+microRNAs+suggest+a+200-Myr+missing+Precambrian+fossil+record+of+siliceous+sponge+spicules" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASperling+%22Where%27s+the+glass%3F+Biomarkers%2C+molecular+clocks%2C+and+microRNAs+suggest+a+200-Myr+missing+Precambrian+fossil+record+of+siliceous+sponge+spicules%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0050437-Erwin1" id="pone.0050437-Erwin1"></a>Erwin DH, Laflamme M, Tweedt SM, Sperling EA, Pisani D, et al. (2011) The Cambrian conundrum: early divergence and later ecological success in the early history of animals. Science 334: 1091–1097.
        doi: 10.1126/science.1206375 <ul class="find" data-citedArticleID="6679813" data-doi="10.1126/science.1206375"><li><a href="http://dx.doi.org/10.1126/science.1206375" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Erwin%5Bauthor%5D+AND+The+Cambrian+conundrum%3A+early+divergence+and+later+ecological+success+in+the+early+history+of+animals" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AErwin+%22The+Cambrian+conundrum%3A+early+divergence+and+later+ecological+success+in+the+early+history+of+animals%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0050437-Krautter1" id="pone.0050437-Krautter1"></a>Krautter M, Conway KW, Barrie JV, Neuweiler M (2001) Discovery of a “living dinosaur”: Globally unique modern hexactinellid sponge reefs off British Columbia, Canada. Facies 44: 265–282.
        doi: 10.1007/bf02668178 <ul class="find" data-citedArticleID="6679815" data-doi="10.1007/bf02668178"><li><a href="http://dx.doi.org/10.1007/bf02668178" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Krautter%5Bauthor%5D+AND+Discovery+of+a+%E2%80%9Cliving+dinosaur%E2%80%9D%3A+Globally+unique+modern+hexactinellid+sponge+reefs+off+British+Columbia%2C+Canada" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKrautter+%22Discovery+of+a+%E2%80%9Cliving+dinosaur%E2%80%9D%3A+Globally+unique+modern+hexactinellid+sponge+reefs+off+British+Columbia%2C+Canada%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0050437-Milsom1" id="pone.0050437-Milsom1"></a>Milsom C, Rigby S (2010) Fossils at a glance, 2<sup>nd</sup> Edition. Wiley-Blackwell. <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0050437-Hill1" id="pone.0050437-Hill1"></a>Hill MS (1998) Spongivory on Caribbean reefs releases corals from competition with sponges. Oecologia 117: 143–150.
        doi: 10.1007/s004420050642 <ul class="find" data-citedArticleID="6679819" data-doi="10.1007/s004420050642"><li><a href="http://dx.doi.org/10.1007/s004420050642" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hill%5Bauthor%5D+AND+Spongivory+on+Caribbean+reefs+releases+corals+from+competition+with+sponges" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHill+%22Spongivory+on+Caribbean+reefs+releases+corals+from+competition+with+sponges%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0050437-Rtzler1" id="pone.0050437-Rtzler1"></a>Rützler K (2004) Sponges On Coral Reefs: A Community Shaped By Competitive Cooperation. Boll Mus Ist Biol Univ Genova 68: 85–148. <ul class="find" data-citedArticleID="6679821"><li><a href="http://www.crossref.org/guestquery/?auth2=R%FCtzler&amp;atitle2=Sponges+On+Coral+Reefs%3A+A+Community+Shaped+By+Competitive+Cooperation&amp;auth=R%FCtzler&amp;atitle=Sponges+On+Coral+Reefs%3A+A+Community+Shaped+By+Competitive+Cooperation" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=R%C3%BCtzler%5Bauthor%5D+AND+Sponges+On+Coral+Reefs%3A+A+Community+Shaped+By+Competitive+Cooperation" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AR%C3%BCtzler+%22Sponges+On+Coral+Reefs%3A+A+Community+Shaped+By+Competitive+Cooperation%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0050437-DeGoeij1" id="pone.0050437-DeGoeij1"></a>De Goeij JM, van den Berg H, van Oostveen MM, Epping EHG, van Duyl FC (2008) Major bulk dissolved organic carbon (DOC) removal by encrusting coral reef cavity sponges. Mar Ecol Prog Ser 357: 139–151.
        doi: 10.3354/meps07403 <ul class="find" data-citedArticleID="6679823" data-doi="10.3354/meps07403"><li><a href="http://dx.doi.org/10.3354/meps07403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=De+Goeij%5Bauthor%5D+AND+Major+bulk+dissolved+organic+carbon+%28DOC%29+removal+by+encrusting+coral+reef+cavity+sponges" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADe+Goeij+%22Major+bulk+dissolved+organic+carbon+%28DOC%29+removal+by+encrusting+coral+reef+cavity+sponges%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0050437-Erwin2" id="pone.0050437-Erwin2"></a>Erwin PM, Thacker RW (2008) Cryptic diversity of the symbiotic cyanobacterium <em>Synechococcus spongiarum</em> among sponge hosts. Mol Ecol 17: 2937–47.
        doi: 10.1111/j.1365-294x.2008.03808.x <ul class="find" data-citedArticleID="6679825" data-doi="10.1111/j.1365-294x.2008.03808.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2008.03808.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Erwin%5Bauthor%5D+AND+Cryptic+diversity+of+the+symbiotic+cyanobacterium+Synechococcus+spongiarum+among+sponge+hosts" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AErwin+%22Cryptic+diversity+of+the+symbiotic+cyanobacterium+Synechococcus+spongiarum+among+sponge+hosts%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0050437-VenkateswaraRao1" id="pone.0050437-VenkateswaraRao1"></a>Venkateswara Rao J, Srikanth K, Pallela R, Gnaneshwar Rao T (2009) The use of marine sponge <em>Haliclona tenuiramosa</em> as bioindicator to monitor heavy metal pollution in the coasts of Gulf of Mannar, India. Environ Monit Assess 156: 451–459.
        doi: 10.1007/s10661-008-0497-x <ul class="find" data-citedArticleID="6679827" data-doi="10.1007/s10661-008-0497-x"><li><a href="http://dx.doi.org/10.1007/s10661-008-0497-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Venkateswara+Rao%5Bauthor%5D+AND+The+use+of+marine+sponge+Haliclona+tenuiramosa+as+bioindicator+to+monitor+heavy+metal+pollution+in+the+coasts+of+Gulf+of+Mannar%2C+India" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AVenkateswara+Rao+%22The+use+of+marine+sponge+Haliclona+tenuiramosa+as+bioindicator+to+monitor+heavy+metal+pollution+in+the+coasts+of+Gulf+of+Mannar%2C+India%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0050437-Negandhi1" id="pone.0050437-Negandhi1"></a>Negandhi K, Blackwelder PL, Ereskovsky AV, Lopez JV (2010) Florida Reef Sponges Harbor Coral Disease-Associated Bacteria. Symbiosis 51: 117–129.
        doi: 10.1007/s13199-010-0059-1 <ul class="find" data-citedArticleID="6679829" data-doi="10.1007/s13199-010-0059-1"><li><a href="http://dx.doi.org/10.1007/s13199-010-0059-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Negandhi%5Bauthor%5D+AND+Florida+Reef+Sponges+Harbor+Coral+Disease-Associated+Bacteria" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ANegandhi+%22Florida+Reef+Sponges+Harbor+Coral+Disease-Associated+Bacteria%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0050437-Pawlik1" id="pone.0050437-Pawlik1"></a>Pawlik JR (2011) The chemical ecology of sponges on Caribbean reefs: Natural products shape natural systems. BioScience 61: 888–898.
        doi: 10.1525/bio.2011.61.11.8 <ul class="find" data-citedArticleID="6679831" data-doi="10.1525/bio.2011.61.11.8"><li><a href="http://dx.doi.org/10.1525/bio.2011.61.11.8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pawlik%5Bauthor%5D+AND+The+chemical+ecology+of+sponges+on+Caribbean+reefs%3A+Natural+products+shape+natural+systems" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APawlik+%22The+chemical+ecology+of+sponges+on+Caribbean+reefs%3A+Natural+products+shape+natural+systems%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0050437-Webster1" id="pone.0050437-Webster1"></a>Webster NS, Taylor MW (2012) Marine sponges and their microbial symbionts: love and other relationships. Environ Microbiol 14: 335–346  doi:10.1111/j.1462-2920.2011.02460.x. <ul class="find" data-citedArticleID="6679833" data-doi="10.1111/j.1462-2920.2011.02460.x"><li><a href="http://dx.doi.org/10.1111/j.1462-2920.2011.02460.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Webster%5Bauthor%5D+AND+Marine+sponges+and+their+microbial+symbionts%3A+love+and+other+relationships" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWebster+%22Marine+sponges+and+their+microbial+symbionts%3A+love+and+other+relationships%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0050437-Hooper1" id="pone.0050437-Hooper1"></a>Hooper JNA, van Soest RWM (2002) Systema Porifera. A guide to the classification of sponges. New York: Kluwer Academic/Plenum Publishers. 1810 p. <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0050437-Borchiellini1" id="pone.0050437-Borchiellini1"></a>Borchiellini C, Chombard C, Manuel M, Alivon E, Vacelet J, et al. (2004) Molecular phylogeny of Demospongiae: implications for classification and scenarios of character evolution. Mol Phylogenet Evol 32: 823–837.
        doi: 10.1016/j.ympev.2004.02.021 <ul class="find" data-citedArticleID="6679837" data-doi="10.1016/j.ympev.2004.02.021"><li><a href="http://dx.doi.org/10.1016/j.ympev.2004.02.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Borchiellini%5Bauthor%5D+AND+Molecular+phylogeny+of+Demospongiae%3A+implications+for+classification+and+scenarios+of+character+evolution" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABorchiellini+%22Molecular+phylogeny+of+Demospongiae%3A+implications+for+classification+and+scenarios+of+character+evolution%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0050437-Sperling2" id="pone.0050437-Sperling2"></a>Sperling EA, Peterson KJ, Pisani D (2009) Phylogenetic-signal dissection of nuclear housekeeping genes supports the paraphyly of sponges and the monophyly of Eumetazoa. Mol Biol Evol 26: 2261–2274.
        doi: 10.1093/molbev/msp148 <ul class="find" data-citedArticleID="6679839" data-doi="10.1093/molbev/msp148"><li><a href="http://dx.doi.org/10.1093/molbev/msp148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sperling%5Bauthor%5D+AND+Phylogenetic-signal+dissection+of+nuclear+housekeeping+genes+supports+the+paraphyly+of+sponges+and+the+monophyly+of+Eumetazoa" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASperling+%22Phylogenetic-signal+dissection+of+nuclear+housekeeping+genes+supports+the+paraphyly+of+sponges+and+the+monophyly+of+Eumetazoa%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0050437-Philippe1" id="pone.0050437-Philippe1"></a>Philippe H, Derelle R, Lopez P, Pick K, Borchiellini C, et al. (2009) Phylogenomics revives traditional views on deep animal relationships. Curr Biol 19: 706–712.
        doi: 10.1016/j.cub.2009.02.052 <ul class="find" data-citedArticleID="6679841" data-doi="10.1016/j.cub.2009.02.052"><li><a href="http://dx.doi.org/10.1016/j.cub.2009.02.052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Philippe%5Bauthor%5D+AND+Phylogenomics+revives+traditional+views+on+deep+animal+relationships" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APhilippe+%22Phylogenomics+revives+traditional+views+on+deep+animal+relationships%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0050437-Philippe2" id="pone.0050437-Philippe2"></a>Philippe H, Brinkmann H, Lavrov DV, Littlewood DTJ, Manuel M, et al. (2011) Resolving difficult phylogenetic questions: why more sequences are not enough. PLoS Biology 9(3): e1000602  doi:10.1371/journal.pbio.1000602. <ul class="find" data-citedArticleID="6679843" data-doi="10.1371/journal.pbio.1000602"><li><a href="http://dx.doi.org/10.1371/journal.pbio.1000602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Philippe%5Bauthor%5D+AND+Resolving+difficult+phylogenetic+questions%3A+why+more+sequences+are+not+enough" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APhilippe+%22Resolving+difficult+phylogenetic+questions%3A+why+more+sequences+are+not+enough%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0050437-Rtzler2" id="pone.0050437-Rtzler2"></a>Rützler K, Hooper JNA (2000) Two new genera of hadromerid sponges (Porifera, Demospongiae). Zoosystema 22: 337–3444. <ul class="find" data-citedArticleID="6679845"><li><a href="http://www.crossref.org/guestquery/?auth2=R%FCtzler&amp;atitle2=Two+new+genera+of+hadromerid+sponges+%28Porifera%2C+Demospongiae%29&amp;auth=R%FCtzler&amp;atitle=Two+new+genera+of+hadromerid+sponges+%28Porifera%2C+Demospongiae%29" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=R%C3%BCtzler%5Bauthor%5D+AND+Two+new+genera+of+hadromerid+sponges+%28Porifera%2C+Demospongiae%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AR%C3%BCtzler+%22Two+new+genera+of+hadromerid+sponges+%28Porifera%2C+Demospongiae%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0050437-Ferrario1" id="pone.0050437-Ferrario1"></a>Ferrario F, Calcinai B, Erpenbeck D, Galli P, Wörheide G (2010) Two <em>Pione</em> species (Hadromerida, Clionaidae) from the Red Sea: a taxonomical challenge. Org Divers Evol 10: 275–285.
        doi: 10.1007/s13127-010-0027-x <ul class="find" data-citedArticleID="6679847" data-doi="10.1007/s13127-010-0027-x"><li><a href="http://dx.doi.org/10.1007/s13127-010-0027-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ferrario%5Bauthor%5D+AND+Two+Pione+species+%28Hadromerida%2C+Clionaidae%29+from+the+Red+Sea%3A+a+taxonomical+challenge" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFerrario+%22Two+Pione+species+%28Hadromerida%2C+Clionaidae%29+from+the+Red+Sea%3A+a+taxonomical+challenge%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0050437-dePaula1" id="pone.0050437-dePaula1"></a>de Paula , Zilberberg C, Hajdu E, Lôbo-Hajdu G (2012) Morphology and molecules on opposite sides of the diversity gradient: Four cryptic species of the <em>Cliona celata</em> (Porifera, Demospongiae) complex in South America revealed by mitochondrial and nuclear markers. Mol Phylogenet Evol 62: 529–541.
        doi: 10.1016/j.ympev.2011.11.001 <ul class="find" data-citedArticleID="6679849" data-doi="10.1016/j.ympev.2011.11.001"><li><a href="http://dx.doi.org/10.1016/j.ympev.2011.11.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=de+Paula%5Bauthor%5D+AND+Morphology+and+molecules+on+opposite+sides+of+the+diversity+gradient%3A+Four+cryptic+species+of+the+Cliona+celata+%28Porifera%2C+Demospongiae%29+complex+in+South+America+revealed+by+mitochondrial+and+nuclear+markers" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3Ade+Paula+%22Morphology+and+molecules+on+opposite+sides+of+the+diversity+gradient%3A+Four+cryptic+species+of+the+Cliona+celata+%28Porifera%2C+Demospongiae%29+complex+in+South+America+revealed+by+mitochondrial+and+nuclear+markers%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0050437-vanSoest1" id="pone.0050437-vanSoest1"></a>van Soest RWM (1990) Towards a phylogenetic classification of sponges. In: Rützler K, editor. New Perspectives in Sponge Biology. Proceedings of the 3rd International Conference on the Biology of Sponges, Woods Hole, 1985. Washington: Smithsonian Institution Press. pp. 344–350. <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0050437-Lafay1" id="pone.0050437-Lafay1"></a>Lafay B, Boury-Esnault N, Vacelet J, Christen R (1992) An analysis of partial 28S ribosomal RNA sequences suggests early radiations of sponges. BioSys 28: 139–151.
        doi: 10.1016/0303-2647(92)90016-r <ul class="find" data-citedArticleID="6679853" data-doi="10.1016/0303-2647(92)90016-r"><li><a href="http://dx.doi.org/10.1016/0303-2647(92)90016-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lafay%5Bauthor%5D+AND+An+analysis+of+partial+28S+ribosomal+RNA+sequences+suggests+early+radiations+of+sponges" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALafay+%22An+analysis+of+partial+28S+ribosomal+RNA+sequences+suggests+early+radiations+of+sponges%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0050437-West1" id="pone.0050437-West1"></a>West L, Powers D (1993) Molecular phylogenetic position of the hexactinellid sponges in relation to the Protista and Demospongiae. Mol Mar Biol Biotech 2: 71–75. <ul class="find" data-citedArticleID="6679855"><li><a href="http://www.crossref.org/guestquery/?auth2=West&amp;atitle2=Molecular+phylogenetic+position+of+the+hexactinellid+sponges+in+relation+to+the+Protista+and+Demospongiae&amp;auth=West&amp;atitle=Molecular+phylogenetic+position+of+the+hexactinellid+sponges+in+relation+to+the+Protista+and+Demospongiae" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=West%5Bauthor%5D+AND+Molecular+phylogenetic+position+of+the+hexactinellid+sponges+in+relation+to+the+Protista+and+Demospongiae" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWest+%22Molecular+phylogenetic+position+of+the+hexactinellid+sponges+in+relation+to+the+Protista+and+Demospongiae%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0050437-Chombard1" id="pone.0050437-Chombard1"></a>Chombard C (1998) Les Demospongiae à asters: essai de phylogénie moléculaire. Homologie du caractère ‘aster’. Ph.D. Thesis, Museum national d'Histoire naturelle de Paris. <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0050437-Collins1" id="pone.0050437-Collins1"></a>Collins AG (1998) Evaluating multiple alternative hypotheses for the origin of Bilateria: An analysis of 18S rRNA molecular evidence. Proc Natl Acad Sci 95: 15458–15463.
        doi: 10.1073/pnas.95.26.15458 <ul class="find" data-citedArticleID="6679859" data-doi="10.1073/pnas.95.26.15458"><li><a href="http://dx.doi.org/10.1073/pnas.95.26.15458" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collins%5Bauthor%5D+AND+Evaluating+multiple+alternative+hypotheses+for+the+origin+of+Bilateria%3A+An+analysis+of+18S+rRNA+molecular+evidence" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACollins+%22Evaluating+multiple+alternative+hypotheses+for+the+origin+of+Bilateria%3A+An+analysis+of+18S+rRNA+molecular+evidence%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0050437-Hajdu1" id="pone.0050437-Hajdu1"></a>Hajdu E (1999) Toward a phylogenetic classification of the Mycalids with anisochelae (Demospongiae: Poecilosclerida), and comments on the status of <em>Naviculina</em> Gray, 1867. Mem Queensland Mus 44: 225–238. <ul class="find" data-citedArticleID="6679861"><li><a href="http://www.crossref.org/guestquery/?auth2=Hajdu&amp;atitle2=Toward+a+phylogenetic+classification+of+the+Mycalids+with+anisochelae+%28Demospongiae%3A+Poecilosclerida%29%2C+and+comments+on+the+status+of+Naviculina+Gray%2C+1867&amp;auth=Hajdu&amp;atitle=Toward+a+phylogenetic+classification+of+the+Mycalids+with+anisochelae+%28Demospongiae%3A+Poecilosclerida%29%2C+and+comments+on+the+status+of+Naviculina+Gray%2C+1867" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hajdu%5Bauthor%5D+AND+Toward+a+phylogenetic+classification+of+the+Mycalids+with+anisochelae+%28Demospongiae%3A+Poecilosclerida%29%2C+and+comments+on+the+status+of+Naviculina+Gray%2C+1867" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHajdu+%22Toward+a+phylogenetic+classification+of+the+Mycalids+with+anisochelae+%28Demospongiae%3A+Poecilosclerida%29%2C+and+comments+on+the+status+of+Naviculina+Gray%2C+1867%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0050437-Alvarez1" id="pone.0050437-Alvarez1"></a>Alvarez B, Crisp MD, Driver F, Hooper JNA, van Soest RWM (2000) Phylogenetic relationships of the family Axinellidae (Porifera: Demospongiae) using morphological and molecular data. Zool Scr 29: 169–198.
        doi: 10.1046/j.1463-6409.2000.00029.x <ul class="find" data-citedArticleID="6679863" data-doi="10.1046/j.1463-6409.2000.00029.x"><li><a href="http://dx.doi.org/10.1046/j.1463-6409.2000.00029.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alvarez%5Bauthor%5D+AND+Phylogenetic+relationships+of+the+family+Axinellidae+%28Porifera%3A+Demospongiae%29+using+morphological+and+molecular+data" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAlvarez+%22Phylogenetic+relationships+of+the+family+Axinellidae+%28Porifera%3A+Demospongiae%29+using+morphological+and+molecular+data%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0050437-Borchiellini2" id="pone.0050437-Borchiellini2"></a>Borchiellini C, Chombard C, Lafay B, Boury-Esnault N (2000) Molecular Systematics of sponges (Porifera). Hydrobiologia 420: 15–27.
        doi: 10.1007/978-94-017-2184-4_2 <ul class="find" data-citedArticleID="6679865" data-doi="10.1007/978-94-017-2184-4_2"><li><a href="http://dx.doi.org/10.1007/978-94-017-2184-4_2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Borchiellini%5Bauthor%5D+AND+Molecular+Systematics+of+sponges+%28Porifera%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABorchiellini+%22Molecular+Systematics+of+sponges+%28Porifera%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0050437-Hajdu2" id="pone.0050437-Hajdu2"></a>Hajdu E, van Soest RWM, Hooper JNA (1994) Proposal for a phylogenetic subordinal classification of poecilosclerid sponges. In: van Soest RWM, van Kempen TMG, Braekman J-C, editors. Sponges in Time and Space - Proceedings of the 4th International Porifera Congress. Rotterdam: Balkema. pp. 123–139. <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0050437-Lavrov1" id="pone.0050437-Lavrov1"></a>Lavrov DV, Lang BF (2005) Poriferan mtDNA and animal phylogeny based on mitochondrial gene arrangements. Syst Biol 54: 651–659. <ul class="find" data-citedArticleID="6679869"><li><a href="http://www.crossref.org/guestquery/?auth2=Lavrov&amp;atitle2=Poriferan+mtDNA+and+animal+phylogeny+based+on+mitochondrial+gene+arrangements&amp;auth=Lavrov&amp;atitle=Poriferan+mtDNA+and+animal+phylogeny+based+on+mitochondrial+gene+arrangements" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lavrov%5Bauthor%5D+AND+Poriferan+mtDNA+and+animal+phylogeny+based+on+mitochondrial+gene+arrangements" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALavrov+%22Poriferan+mtDNA+and+animal+phylogeny+based+on+mitochondrial+gene+arrangements%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0050437-Wrheide1" id="pone.0050437-Wrheide1"></a>Wörheide G, Solé-Cava AM, Hooper JNA (2005) Biodiversity, molecular ecology and phylogeography of marine sponges: patterns, implications and outlooks. Integr Comp Biol 43: 377–385.
        doi: 10.1093/icb/45.2.377 <ul class="find" data-citedArticleID="6679871" data-doi="10.1093/icb/45.2.377"><li><a href="http://dx.doi.org/10.1093/icb/45.2.377" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=W%C3%B6rheide%5Bauthor%5D+AND+Biodiversity%2C+molecular+ecology+and+phylogeography+of+marine+sponges%3A+patterns%2C+implications+and+outlooks" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AW%C3%B6rheide+%22Biodiversity%2C+molecular+ecology+and+phylogeography+of+marine+sponges%3A+patterns%2C+implications+and+outlooks%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0050437-Redmond1" id="pone.0050437-Redmond1"></a>Redmond NE, van Soest RWM, Kelly M, Raleigh J, Travers SAA, et al. (2007) Reassessment of the classification of the Order Haplosclerida (Class Demospongiae, Phylum Porifera) using 18S rRNA gene sequence data. Mol Phylogenet Evol 43: 344–352.
        doi: 10.1016/j.ympev.2006.10.021 <ul class="find" data-citedArticleID="6679873" data-doi="10.1016/j.ympev.2006.10.021"><li><a href="http://dx.doi.org/10.1016/j.ympev.2006.10.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Redmond%5Bauthor%5D+AND+Reassessment+of+the+classification+of+the+Order+Haplosclerida+%28Class+Demospongiae%2C+Phylum+Porifera%29+using+18S+rRNA+gene+sequence+data" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARedmond+%22Reassessment+of+the+classification+of+the+Order+Haplosclerida+%28Class+Demospongiae%2C+Phylum+Porifera%29+using+18S+rRNA+gene+sequence+data%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0050437-Gazave1" id="pone.0050437-Gazave1"></a>Gazave E, Carteron S, Chenuil A, Richelle-Maurer E, Boury-Esnault N, et al. (2010) Polyphyly of the genus <em>Axinella</em> and of the family Axinellidae (Porifera: <em>Demospongiae<sup>P</sup></em>). Mol Phylogenet Evol 57: 35–47.
        doi: 10.1016/j.ympev.2010.05.028 <ul class="find" data-citedArticleID="6679875" data-doi="10.1016/j.ympev.2010.05.028"><li><a href="http://dx.doi.org/10.1016/j.ympev.2010.05.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gazave%5Bauthor%5D+AND+Polyphyly+of+the+genus+Axinella+and+of+the+family+Axinellidae+%28Porifera%3A+DemospongiaeP%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGazave+%22Polyphyly+of+the+genus+Axinella+and+of+the+family+Axinellidae+%28Porifera%3A+DemospongiaeP%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0050437-Redmond2" id="pone.0050437-Redmond2"></a>Redmond NE, Raleigh J, van Soest RWM, Kelly M, Travers SAA, et al. (2011) Phylogenetic Relationships of the Marine Haplosclerida (Phylum Porifera) Employing Ribosomal (28S rRNA) and Mitochondrial (<em>cox1, nad1</em>) Gene Sequence Data. PLoS ONE 6(9): e24344.
        doi: 10.1371/journal.pone.0024344 <ul class="find" data-citedArticleID="6679877" data-doi="10.1371/journal.pone.0024344"><li><a href="http://dx.doi.org/10.1371/journal.pone.0024344" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Redmond%5Bauthor%5D+AND+Phylogenetic+Relationships+of+the+Marine+Haplosclerida+%28Phylum+Porifera%29+Employing+Ribosomal+%2828S+rRNA%29+and+Mitochondrial+%28cox1%2C+nad1%29+Gene+Sequence+Data" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARedmond+%22Phylogenetic+Relationships+of+the+Marine+Haplosclerida+%28Phylum+Porifera%29+Employing+Ribosomal+%2828S+rRNA%29+and+Mitochondrial+%28cox1%2C+nad1%29+Gene+Sequence+Data%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0050437-Morrow1" id="pone.0050437-Morrow1"></a>Morrow CC, Picton BE, Erpenbeck D, Boury-Esnault N, Maggs CA, et al. (2012) Congruence between nuclear and mitochondrial genes in Demospongiae: A new hypothesis for relationships within the G4 clade (Porifera: Demospongiae). Mol Phylogenet Evol 62: 174–190.
        doi: 10.1016/j.ympev.2011.09.016 <ul class="find" data-citedArticleID="6679879" data-doi="10.1016/j.ympev.2011.09.016"><li><a href="http://dx.doi.org/10.1016/j.ympev.2011.09.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Morrow%5Bauthor%5D+AND+Congruence+between+nuclear+and+mitochondrial+genes+in+Demospongiae%3A+A+new+hypothesis+for+relationships+within+the+G4+clade+%28Porifera%3A+Demospongiae%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMorrow+%22Congruence+between+nuclear+and+mitochondrial+genes+in+Demospongiae%3A+A+new+hypothesis+for+relationships+within+the+G4+clade+%28Porifera%3A+Demospongiae%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0050437-Hooper2" id="pone.0050437-Hooper2"></a>Hooper JNA, van Soest RWM, Pisera A (2011) Phylum Porifera Grant, 1826. In: Zhang, Z-Q, editor. Animal biodiversity: An outline of higher-level classification and survey of taxonomic richness. Zootaxa 3148: 13–18. <ul class="find" data-citedArticleID="6679881"><li><a href="http://www.crossref.org/guestquery/?auth2=Hooper&amp;atitle2=Phylum+Porifera+Grant%2C+1826.+In%3A+Zhang%2C+Z-Q%2C+editor.+Animal+biodiversity%3A+An+outline+of+higher-level+classification+and+survey+of+taxonomic+richness&amp;auth=Hooper&amp;atitle=Phylum+Porifera+Grant%2C+1826.+In%3A+Zhang%2C+Z-Q%2C+editor.+Animal+biodiversity%3A+An+outline+of+higher-level+classification+and+survey+of+taxonomic+richness" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hooper%5Bauthor%5D+AND+Phylum+Porifera+Grant%2C+1826.+In%3A+Zhang%2C+Z-Q%2C+editor.+Animal+biodiversity%3A+An+outline+of+higher-level+classification+and+survey+of+taxonomic+richness" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHooper+%22Phylum+Porifera+Grant%2C+1826.+In%3A+Zhang%2C+Z-Q%2C+editor.+Animal+biodiversity%3A+An+outline+of+higher-level+classification+and+survey+of+taxonomic+richness%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0050437-Crdenas1" id="pone.0050437-Crdenas1"></a>Cárdenas P, Pérez T, Boury-Esnault N (2012) Sponge systematics facing new challenges. Adv Mar Biol 61: 79–209.
        doi: 10.1016/b978-0-12-387787-1.00010-6 <ul class="find" data-citedArticleID="6679883" data-doi="10.1016/b978-0-12-387787-1.00010-6"><li><a href="http://dx.doi.org/10.1016/b978-0-12-387787-1.00010-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=C%C3%A1rdenas%5Bauthor%5D+AND+Sponge+systematics+facing+new+challenges" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AC%C3%A1rdenas+%22Sponge+systematics+facing+new+challenges%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0050437-Wrheide2" id="pone.0050437-Wrheide2"></a>Wörheide G, Dohrmann M, Erpenbeck D, Larroux C, Maldonado M, et al. (2012) Deep phylogeny and evolution of sponges (Phylum Porifera). Adv Mar Biol 61: 1–78.
        doi: 10.1016/b978-0-12-387787-1.00007-6 <ul class="find" data-citedArticleID="6679885" data-doi="10.1016/b978-0-12-387787-1.00007-6"><li><a href="http://dx.doi.org/10.1016/b978-0-12-387787-1.00007-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=W%C3%B6rheide%5Bauthor%5D+AND+Deep+phylogeny+and+evolution+of+sponges+%28Phylum+Porifera%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AW%C3%B6rheide+%22Deep+phylogeny+and+evolution+of+sponges+%28Phylum+Porifera%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0050437-Nichols1" id="pone.0050437-Nichols1"></a>Nichols SA (2005) An evaluation of support for order-level monophyly and interrelationships within the class Demospongiae using partial data from the large subunit rDNA and cytochrome oxidase subunit I. Mol Phylogenet Evol 34: 81–96.
        doi: 10.1016/j.ympev.2004.08.019 <ul class="find" data-citedArticleID="6679887" data-doi="10.1016/j.ympev.2004.08.019"><li><a href="http://dx.doi.org/10.1016/j.ympev.2004.08.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nichols%5Bauthor%5D+AND+An+evaluation+of+support+for+order-level+monophyly+and+interrelationships+within+the+class+Demospongiae+using+partial+data+from+the+large+subunit+rDNA+and+cytochrome+oxidase+subunit+I" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ANichols+%22An+evaluation+of+support+for+order-level+monophyly+and+interrelationships+within+the+class+Demospongiae+using+partial+data+from+the+large+subunit+rDNA+and+cytochrome+oxidase+subunit+I%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0050437-Lavrov2" id="pone.0050437-Lavrov2"></a>Lavrov DV, Wang X, Kelly M (2008) Reconstructing ordinal relationships in the Demospongiae using mitochondrial genomic data. Mol Phylogenet Evol 49: 111–124.
        doi: 10.1016/j.ympev.2008.05.014 <ul class="find" data-citedArticleID="6679889" data-doi="10.1016/j.ympev.2008.05.014"><li><a href="http://dx.doi.org/10.1016/j.ympev.2008.05.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lavrov%5Bauthor%5D+AND+Reconstructing+ordinal+relationships+in+the+Demospongiae+using+mitochondrial+genomic+data" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALavrov+%22Reconstructing+ordinal+relationships+in+the+Demospongiae+using+mitochondrial+genomic+data%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0050437-Sperling3" id="pone.0050437-Sperling3"></a>Sperling EA, Pisani D, Peterson KJ (2007) Poriferan paraphyly and its implications for Precambrian paleobiology. In Vickers-Rich P, Komarower P, editors. The Rise and Fall of the Ediacaran Biota. Geol Soc, Special Publications, London, pp. 355–368. <ul class="find-nolinks"></ul></li><li><span class="label">45.
              </span><a name="pone.0050437-Peterson1" id="pone.0050437-Peterson1"></a>Peterson KJ, Lyons JB, Nowak KS, Takacs CM, Wargo MT, et al. (2004) Estimating metazoan divergence times with a molecular clock. Proc Natl Acad Sci USA 101: 6536–6541.
        doi: 10.1073/pnas.0401670101 <ul class="find" data-citedArticleID="6679893" data-doi="10.1073/pnas.0401670101"><li><a href="http://dx.doi.org/10.1073/pnas.0401670101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peterson%5Bauthor%5D+AND+Estimating+metazoan+divergence+times+with+a+molecular+clock" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APeterson+%22Estimating+metazoan+divergence+times+with+a+molecular+clock%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0050437-Peterson2" id="pone.0050437-Peterson2"></a>Peterson KJ, Butterfield NJ (2005) Origin of the Eumetazoa: testing ecological predictions of molecular clocks against the Proterozoic fossil record. Proc Natl Acad Sci USA 102: 9547–9552.
        doi: 10.1073/pnas.0503660102 <ul class="find" data-citedArticleID="6679895" data-doi="10.1073/pnas.0503660102"><li><a href="http://dx.doi.org/10.1073/pnas.0503660102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peterson%5Bauthor%5D+AND+Origin+of+the+Eumetazoa%3A+testing+ecological+predictions+of+molecular+clocks+against+the+Proterozoic+fossil+record" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APeterson+%22Origin+of+the+Eumetazoa%3A+testing+ecological+predictions+of+molecular+clocks+against+the+Proterozoic+fossil+record%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0050437-Reed1" id="pone.0050437-Reed1"></a>Reed JK, Pomponi SA (1997) Biodiversity and distribution of deep and shallow water sponges in the Bahamas. Proc 8th Int Coral Reef Symp 2: 1387–1392. <ul class="find" data-citedArticleID="6679897"><li><a href="http://www.crossref.org/guestquery/?auth2=Reed&amp;atitle2=Biodiversity+and+distribution+of+deep+and+shallow+water+sponges+in+the+Bahamas&amp;auth=Reed&amp;atitle=Biodiversity+and+distribution+of+deep+and+shallow+water+sponges+in+the+Bahamas" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reed%5Bauthor%5D+AND+Biodiversity+and+distribution+of+deep+and+shallow+water+sponges+in+the+Bahamas" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AReed+%22Biodiversity+and+distribution+of+deep+and+shallow+water+sponges+in+the+Bahamas%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0050437-vanSoest2" id="pone.0050437-vanSoest2"></a>van Soest, RWM, Boury-Esnault N, Hooper JNA, Rützler K, de Voogd NJ, et al.. (2012) World Porifera database. Available: <a href="http://www.marinespecies.org/porifera">http://www.marinespecies.org/porifera</a>. Accessed 2012 Aug 1. <ul class="find-nolinks"></ul></li><li><span class="label">49.
              </span><a name="pone.0050437-Abascal1" id="pone.0050437-Abascal1"></a>Abascal F, Zardoya R, Posada D (2005) ProtTest: Selection of best-fit models of protein evolution. Bioinformatics 21: 2104–2105  doi:10.1093/bioinformatics/bti263. <ul class="find" data-citedArticleID="6679901" data-doi="10.1093/bioinformatics/bti263"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Abascal%5Bauthor%5D+AND+ProtTest%3A+Selection+of+best-fit+models+of+protein+evolution" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAbascal+%22ProtTest%3A+Selection+of+best-fit+models+of+protein+evolution%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0050437-Le1" id="pone.0050437-Le1"></a>Le SQ, Gascuel O (2008) LG: An Improved, General Amino-Acid Replacement Matrix. Mol Biol Evol 25: 1307–1320.
        doi: 10.1093/molbev/msn067 <ul class="find" data-citedArticleID="6679903" data-doi="10.1093/molbev/msn067"><li><a href="http://dx.doi.org/10.1093/molbev/msn067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Le%5Bauthor%5D+AND+LG%3A+An+Improved%2C+General+Amino-Acid+Replacement+Matrix" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALe+%22LG%3A+An+Improved%2C+General+Amino-Acid+Replacement+Matrix%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0050437-Schmidt1" id="pone.0050437-Schmidt1"></a>Schmidt HA, Strimmer K, Vingron M, von Haeseler A (2002) TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing. Bioinformatics 18: 502–504.
        doi: 10.1093/bioinformatics/18.3.502 <ul class="find" data-citedArticleID="6679905" data-doi="10.1093/bioinformatics/18.3.502"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.3.502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Schmidt%5Bauthor%5D+AND+TREE-PUZZLE%3A+maximum+likelihood+phylogenetic+analysis+using+quartets+and+parallel+computing" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASchmidt+%22TREE-PUZZLE%3A+maximum+likelihood+phylogenetic+analysis+using+quartets+and+parallel+computing%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0050437-Stone1" id="pone.0050437-Stone1"></a>Stone M (1974) Cross-validatory choice and assessment of statistical prediction. J R Stat Soc Ser B 36: 111–147. <ul class="find" data-citedArticleID="6679907"><li><a href="http://www.crossref.org/guestquery/?auth2=Stone&amp;atitle2=Cross-validatory+choice+and+assessment+of+statistical+prediction&amp;auth=Stone&amp;atitle=Cross-validatory+choice+and+assessment+of+statistical+prediction" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stone%5Bauthor%5D+AND+Cross-validatory+choice+and+assessment+of+statistical+prediction" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AStone+%22Cross-validatory+choice+and+assessment+of+statistical+prediction%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0050437-Pisani1" id="pone.0050437-Pisani1"></a>Pisani D, Wilkinson M (2002) Matrix representation with parsimony, taxonomic congruence, and total evidence. Syst Biol 51: 151–155. <ul class="find" data-citedArticleID="6679909"><li><a href="http://www.crossref.org/guestquery/?auth2=Pisani&amp;atitle2=Matrix+representation+with+parsimony%2C+taxonomic+congruence%2C+and+total+evidence&amp;auth=Pisani&amp;atitle=Matrix+representation+with+parsimony%2C+taxonomic+congruence%2C+and+total+evidence" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pisani%5Bauthor%5D+AND+Matrix+representation+with+parsimony%2C+taxonomic+congruence%2C+and+total+evidence" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APisani+%22Matrix+representation+with+parsimony%2C+taxonomic+congruence%2C+and+total+evidence%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0050437-Dohrmann1" id="pone.0050437-Dohrmann1"></a>Dohrmann M, Janussen D, Reitner J, Collins AG, Wörheide G (2008) Phylogeny and evolution of glass sponges (Porifera, Hexactinellida). Syst Biol 57: 388–405. <ul class="find" data-citedArticleID="6679911"><li><a href="http://www.crossref.org/guestquery/?auth2=Dohrmann&amp;atitle2=Phylogeny+and+evolution+of+glass+sponges+%28Porifera%2C+Hexactinellida%29&amp;auth=Dohrmann&amp;atitle=Phylogeny+and+evolution+of+glass+sponges+%28Porifera%2C+Hexactinellida%29" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dohrmann%5Bauthor%5D+AND+Phylogeny+and+evolution+of+glass+sponges+%28Porifera%2C+Hexactinellida%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADohrmann+%22Phylogeny+and+evolution+of+glass+sponges+%28Porifera%2C+Hexactinellida%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0050437-Manuel1" id="pone.0050437-Manuel1"></a>Manuel M, Borchiellini C, Alivon E, Le Parco Y, Vacelet J, et al. (2003) Phylogeny and evolution of calcareous sponges: Monophyly of Calcinea and Calcaronea, high level of morphological homoplasy, and the primitive nature of axial symmetry. Syst Biol 52: 311–333. <ul class="find" data-citedArticleID="6679913"><li><a href="http://www.crossref.org/guestquery/?auth2=Manuel&amp;atitle2=Phylogeny+and+evolution+of+calcareous+sponges%3A+Monophyly+of+Calcinea+and+Calcaronea%2C+high+level+of+morphological+homoplasy%2C+and+the+primitive+nature+of+axial+symmetry&amp;auth=Manuel&amp;atitle=Phylogeny+and+evolution+of+calcareous+sponges%3A+Monophyly+of+Calcinea+and+Calcaronea%2C+high+level+of+morphological+homoplasy%2C+and+the+primitive+nature+of+axial+symmetry" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Manuel%5Bauthor%5D+AND+Phylogeny+and+evolution+of+calcareous+sponges%3A+Monophyly+of+Calcinea+and+Calcaronea%2C+high+level+of+morphological+homoplasy%2C+and+the+primitive+nature+of+axial+symmetry" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AManuel+%22Phylogeny+and+evolution+of+calcareous+sponges%3A+Monophyly+of+Calcinea+and+Calcaronea%2C+high+level+of+morphological+homoplasy%2C+and+the+primitive+nature+of+axial+symmetry%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0050437-Dohrmann2" id="pone.0050437-Dohrmann2"></a>Dohrmann M, Voigt O, Erpenbeck D, Wörheide G (2006) Non-monophyly of most supraspecific taxa of calcareous sponges (Porifera, Calcarea) revealed by increased taxon sampling and partitioned Bayesian analysis of ribosomal DNA. Mol Phylogenet Evol 40: 830–843.
        doi: 10.1016/j.ympev.2006.04.016 <ul class="find" data-citedArticleID="6679915" data-doi="10.1016/j.ympev.2006.04.016"><li><a href="http://dx.doi.org/10.1016/j.ympev.2006.04.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dohrmann%5Bauthor%5D+AND+Non-monophyly+of+most+supraspecific+taxa+of+calcareous+sponges+%28Porifera%2C+Calcarea%29+revealed+by+increased+taxon+sampling+and+partitioned+Bayesian+analysis+of+ribosomal+DNA" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADohrmann+%22Non-monophyly+of+most+supraspecific+taxa+of+calcareous+sponges+%28Porifera%2C+Calcarea%29+revealed+by+increased+taxon+sampling+and+partitioned+Bayesian+analysis+of+ribosomal+DNA%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0050437-Ma1" id="pone.0050437-Ma1"></a>Ma J, Yang Q (2007) Phylogeny and Morphogenesis of Dictyoceratida (Porifera: Demospongiae). Acta Palaeontologica Sinica 46: 480–484. <ul class="find" data-citedArticleID="6679917"><li><a href="http://www.crossref.org/guestquery/?auth2=Ma&amp;atitle2=Phylogeny+and+Morphogenesis+of+Dictyoceratida+%28Porifera%3A+Demospongiae%29&amp;auth=Ma&amp;atitle=Phylogeny+and+Morphogenesis+of+Dictyoceratida+%28Porifera%3A+Demospongiae%29" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ma%5Bauthor%5D+AND+Phylogeny+and+Morphogenesis+of+Dictyoceratida+%28Porifera%3A+Demospongiae%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMa+%22Phylogeny+and+Morphogenesis+of+Dictyoceratida+%28Porifera%3A+Demospongiae%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0050437-Ereskovsky1" id="pone.0050437-Ereskovsky1"></a>Ereskovsky A, Lavrov DV, Boury-Esnault N, Vacelet J (2011) Molecular and morphological description of a new species of <em>Halisarca</em> (Demospongiae: Halisarcida) from Mediterranean Sea and a redescription of the type species <em>Halisarca dujardini</em>. Zootaxa 2768: 5–31. <ul class="find" data-citedArticleID="6679919"><li><a href="http://www.crossref.org/guestquery/?auth2=Ereskovsky&amp;atitle2=Molecular+and+morphological+description+of+a+new+species+of+Halisarca+%28Demospongiae%3A+Halisarcida%29+from+Mediterranean+Sea+and+a+redescription+of+the+type+species+Halisarca+dujardini&amp;auth=Ereskovsky&amp;atitle=Molecular+and+morphological+description+of+a+new+species+of+Halisarca+%28Demospongiae%3A+Halisarcida%29+from+Mediterranean+Sea+and+a+redescription+of+the+type+species+Halisarca+dujardini" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ereskovsky%5Bauthor%5D+AND+Molecular+and+morphological+description+of+a+new+species+of+Halisarca+%28Demospongiae%3A+Halisarcida%29+from+Mediterranean+Sea+and+a+redescription+of+the+type+species+Halisarca+dujardini" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AEreskovsky+%22Molecular+and+morphological+description+of+a+new+species+of+Halisarca+%28Demospongiae%3A+Halisarcida%29+from+Mediterranean+Sea+and+a+redescription+of+the+type+species+Halisarca+dujardini%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0050437-Erpenbeck1" id="pone.0050437-Erpenbeck1"></a>Erpenbeck D, Sutcliffe P, Cook SdC, Dietzel A, Maldonado M, et al. (2012) Horny sponges and their affairs: On the phylogenetic relationships of keratose sponges. Mol Phylog Evol 63: 809–816.
        doi: 10.1016/j.ympev.2012.02.024 <ul class="find" data-citedArticleID="6679921" data-doi="10.1016/j.ympev.2012.02.024"><li><a href="http://dx.doi.org/10.1016/j.ympev.2012.02.024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Erpenbeck%5Bauthor%5D+AND+Horny+sponges+and+their+affairs%3A+On+the+phylogenetic+relationships+of+keratose+sponges" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AErpenbeck+%22Horny+sponges+and+their+affairs%3A+On+the+phylogenetic+relationships+of+keratose+sponges%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0050437-Vacelet1" id="pone.0050437-Vacelet1"></a>Vacelet J, Borchiellini C, Pérez T, Bultel-Poncé V, Brouard J-P, et al. (2000) Morphological, chemical, and biochemical characterization of a new species of sponges without skeleton (Porifera, Demospongiae) from the Mediterranean Sea. Zoosystema 22: 313–326. <ul class="find" data-citedArticleID="6679923"><li><a href="http://www.crossref.org/guestquery/?auth2=Vacelet&amp;atitle2=Morphological%2C+chemical%2C+and+biochemical+characterization+of+a+new+species+of+sponges+without+skeleton+%28Porifera%2C+Demospongiae%29+from+the+Mediterranean+Sea&amp;auth=Vacelet&amp;atitle=Morphological%2C+chemical%2C+and+biochemical+characterization+of+a+new+species+of+sponges+without+skeleton+%28Porifera%2C+Demospongiae%29+from+the+Mediterranean+Sea" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vacelet%5Bauthor%5D+AND+Morphological%2C+chemical%2C+and+biochemical+characterization+of+a+new+species+of+sponges+without+skeleton+%28Porifera%2C+Demospongiae%29+from+the+Mediterranean+Sea" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AVacelet+%22Morphological%2C+chemical%2C+and+biochemical+characterization+of+a+new+species+of+sponges+without+skeleton+%28Porifera%2C+Demospongiae%29+from+the+Mediterranean+Sea%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0050437-Erwin3" id="pone.0050437-Erwin3"></a>Erwin PM, Thacker RW (2007) Phylogenetic analyses of marine sponges within the order Verongida: a comparison of morphological and molecular data. Invertebr Biol 126: 220–234.
        doi: 10.1111/j.1744-7410.2007.00092.x <ul class="find" data-citedArticleID="6679925" data-doi="10.1111/j.1744-7410.2007.00092.x"><li><a href="http://dx.doi.org/10.1111/j.1744-7410.2007.00092.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Erwin%5Bauthor%5D+AND+Phylogenetic+analyses+of+marine+sponges+within+the+order+Verongida%3A+a+comparison+of+morphological+and+molecular+data" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AErwin+%22Phylogenetic+analyses+of+marine+sponges+within+the+order+Verongida%3A+a+comparison+of+morphological+and+molecular+data%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0050437-vanSoest3" id="pone.0050437-vanSoest3"></a>van Soest RWM, Hooper JNA (2002) Order Haplosclerida Topsent, 1928. In: Hooper JNA, van Soest RWM, editors. Systema Porifera. A guide to the classification of sponges. New York: Kluwer Academic/Plenum Publishers. pp. 831–832. <ul class="find-nolinks"></ul></li><li><span class="label">63.
              </span><a name="pone.0050437-Itskovich1" id="pone.0050437-Itskovich1"></a>Itskovich VB, Belikov SI, Efremova SM, Masuda Y, Perez T, et al.. (2007) Phylogenetic relationships between freshwater and marine Haplosclerida (Porifera, Demospongiae) based on full-length 18S rRNA and partial COXI gene sequences. In: Custódio MR, Lôbo-Hadju G, Hajdu Muricy G, editors. Porifera Research: Biodiversity, Innovation and Sustainability. Série Livros. Museu Nacional, Rio de Janeiro. pp. 383–391. <ul class="find-nolinks"></ul></li><li><span class="label">64.
              </span><a name="pone.0050437-McCormack1" id="pone.0050437-McCormack1"></a>McCormack GP, Erpenbeck D, van Soest R (2002) Major discrepancy between phylogenetic hypotheses based on molecular and morphological criteria within the Order Haplosclerida (Phylum Porifera: Class Demospongiae). J Zoolog System Evol Res 40: 237–240.
        doi: 10.1046/j.1439-0469.2002.00204.x <ul class="find" data-citedArticleID="6679931" data-doi="10.1046/j.1439-0469.2002.00204.x"><li><a href="http://dx.doi.org/10.1046/j.1439-0469.2002.00204.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=McCormack%5Bauthor%5D+AND+Major+discrepancy+between+phylogenetic+hypotheses+based+on+molecular+and+morphological+criteria+within+the+Order+Haplosclerida+%28Phylum+Porifera%3A+Class+Demospongiae%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMcCormack+%22Major+discrepancy+between+phylogenetic+hypotheses+based+on+molecular+and+morphological+criteria+within+the+Order+Haplosclerida+%28Phylum+Porifera%3A+Class+Demospongiae%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0050437-Raleigh1" id="pone.0050437-Raleigh1"></a>Raleigh J, Redmond NE, Delehan E, Torpey S, van Soest RWM, et al. (2007) Mitochondrial Cytochrome oxidase 1 phylogeny supports alternative taxonomic scheme for the marine Haplosclerida. J Mar Biol Ass UK 87: 1577–1584.
        doi: 10.1017/s0025315407058341 <ul class="find" data-citedArticleID="6679933" data-doi="10.1017/s0025315407058341"><li><a href="http://dx.doi.org/10.1017/s0025315407058341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Raleigh%5Bauthor%5D+AND+Mitochondrial+Cytochrome+oxidase+1+phylogeny+supports+alternative+taxonomic+scheme+for+the+marine+Haplosclerida" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARaleigh+%22Mitochondrial+Cytochrome+oxidase+1+phylogeny+supports+alternative+taxonomic+scheme+for+the+marine+Haplosclerida%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0050437-Srivastava1" id="pone.0050437-Srivastava1"></a>Srivastava M, Simakov O, Chapman J, Fahey B, Gauthier MEA, et al. (2010) The <em>Amphimedon queenslandica</em> genome and the evolution of animal complexity. Nature  doi:10.1038/nature09201. <ul class="find" data-citedArticleID="6679935" data-doi="10.1038/nature09201"><li><a href="http://dx.doi.org/10.1038/nature09201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Srivastava%5Bauthor%5D+AND+The+Amphimedon+queenslandica+genome+and+the+evolution+of+animal+complexity" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASrivastava+%22The+Amphimedon+queenslandica+genome+and+the+evolution+of+animal+complexity%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0050437-Addis1" id="pone.0050437-Addis1"></a>Addis JS, Peterson KJ (2005) Phylogenetic relationships of freshwater sponges (Porifera, Spongillina) inferred from analyses of 18S rDNA, COI mtDNA, and ITS2 rDNA sequences. Zool Scr 34: 549–557.
        doi: 10.1111/j.1463-6409.2005.00211.x <ul class="find" data-citedArticleID="6679937" data-doi="10.1111/j.1463-6409.2005.00211.x"><li><a href="http://dx.doi.org/10.1111/j.1463-6409.2005.00211.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Addis%5Bauthor%5D+AND+Phylogenetic+relationships+of+freshwater+sponges+%28Porifera%2C+Spongillina%29+inferred+from+analyses+of+18S+rDNA%2C+COI+mtDNA%2C+and+ITS2+rDNA+sequences" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAddis+%22Phylogenetic+relationships+of+freshwater+sponges+%28Porifera%2C+Spongillina%29+inferred+from+analyses+of+18S+rDNA%2C+COI+mtDNA%2C+and+ITS2+rDNA+sequences%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0050437-Holmes1" id="pone.0050437-Holmes1"></a>Holmes B, Blanch H (2007) Genus-specific associations of marine sponges with group I crenaarchaeotes. Mar Biol 150: 759–772.
        doi: 10.1007/s00227-006-0361-x <ul class="find" data-citedArticleID="6679939" data-doi="10.1007/s00227-006-0361-x"><li><a href="http://dx.doi.org/10.1007/s00227-006-0361-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Holmes%5Bauthor%5D+AND+Genus-specific+associations+of+marine+sponges+with+group+I+crenaarchaeotes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHolmes+%22Genus-specific+associations+of+marine+sponges+with+group+I+crenaarchaeotes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0050437-Erpenbeck2" id="pone.0050437-Erpenbeck2"></a>Erpenbeck D, Duran S, Rützler K, Paul V, Hooper JNA, et al. (2007) Towards a DNA taxonomy of Caribbean demosponges: a gene tree reconstructed from partial mitochondrial CO1 gene sequences supports previous rDNA phylogenies and provides a new perspective on the systematics of Demospongiae. J Mar Biol Assoc UK 87: 1563–1570.
        doi: 10.1017/s0025315407058195 <ul class="find" data-citedArticleID="6679941" data-doi="10.1017/s0025315407058195"><li><a href="http://dx.doi.org/10.1017/s0025315407058195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Erpenbeck%5Bauthor%5D+AND+Towards+a+DNA+taxonomy+of+Caribbean+demosponges%3A+a+gene+tree+reconstructed+from+partial+mitochondrial+CO1+gene+sequences+supports+previous+rDNA+phylogenies+and+provides+a+new+perspective+on+the+systematics+of+Demospongiae" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AErpenbeck+%22Towards+a+DNA+taxonomy+of+Caribbean+demosponges%3A+a+gene+tree+reconstructed+from+partial+mitochondrial+CO1+gene+sequences+supports+previous+rDNA+phylogenies+and+provides+a+new+perspective+on+the+systematics+of+Demospongiae%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0050437-Crdenas2" id="pone.0050437-Crdenas2"></a>Cárdenas P, Xavier JR, Reveillaud J, Schander C, Rapp HT (2011) Molecular phylogeny of the Astrophorida (Porifera, <em>Demospongiae</em>(p)) reveals an unexpected high level of spicule homoplasy. PLoS ONE 6(4): e18318  doi:10.1371/journal.pone.0018318. <ul class="find" data-citedArticleID="6679943" data-doi="10.1371/journal.pone.0018318"><li><a href="http://dx.doi.org/10.1371/journal.pone.0018318" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=C%C3%A1rdenas%5Bauthor%5D+AND+Molecular+phylogeny+of+the+Astrophorida+%28Porifera%2C+Demospongiae%28p%29%29+reveals+an+unexpected+high+level+of+spicule+homoplasy" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AC%C3%A1rdenas+%22Molecular+phylogeny+of+the+Astrophorida+%28Porifera%2C+Demospongiae%28p%29%29+reveals+an+unexpected+high+level+of+spicule+homoplasy%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0050437-Maldonado1" id="pone.0050437-Maldonado1"></a>Maldonado M (2002) Family Pachastrellidae Carter, 1875. In: Hooper JNA, van Soest RWM, editors. Systema Porifera. A guide to the classification of sponges. New York: Kluwer Academic/Plenum Publishers. pp. 141–162. <ul class="find-nolinks"></ul></li><li><span class="label">72.
              </span><a name="pone.0050437-Pisera1" id="pone.0050437-Pisera1"></a>Pisera A, Lévi C (2002) ‘Lithistid’ Demospongiae. In: Hooper JNA, van Soest RWM, editors. Systema Porifera. A guide to the classification of sponges. New York: Kluwer Academic/Plenum Publishers. pp. 299–301. <ul class="find-nolinks"></ul></li><li><span class="label">73.
              </span><a name="pone.0050437-KellyBorges1" id="pone.0050437-KellyBorges1"></a>Kelly-Borges M, Pomponi S (1994) Phylogeny and classification of lithistid sponges (Porifera: Demospongiae): a preliminary assessment using ribosomal DNA sequence comparisons. Mol Mar Biol Biotechnol 3: 87–103. <ul class="find" data-citedArticleID="6679949"><li><a href="http://www.crossref.org/guestquery/?auth2=Kelly-Borges&amp;atitle2=Phylogeny+and+classification+of+lithistid+sponges+%28Porifera%3A+Demospongiae%29%3A+a+preliminary+assessment+using+ribosomal+DNA+sequence+comparisons&amp;auth=Kelly-Borges&amp;atitle=Phylogeny+and+classification+of+lithistid+sponges+%28Porifera%3A+Demospongiae%29%3A+a+preliminary+assessment+using+ribosomal+DNA+sequence+comparisons" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kelly-Borges%5Bauthor%5D+AND+Phylogeny+and+classification+of+lithistid+sponges+%28Porifera%3A+Demospongiae%29%3A+a+preliminary+assessment+using+ribosomal+DNA+sequence+comparisons" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKelly-Borges+%22Phylogeny+and+classification+of+lithistid+sponges+%28Porifera%3A+Demospongiae%29%3A+a+preliminary+assessment+using+ribosomal+DNA+sequence+comparisons%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0050437-Kober1" id="pone.0050437-Kober1"></a>Kober KM, Nichols SA (2007) On the phylogenetic relationships of Hadromerid and Poecilosclerid sponges. J Mar Biol Assoc UK 87: 1585–1598.
        doi: 10.1017/s0025315407058237 <ul class="find" data-citedArticleID="6679951" data-doi="10.1017/s0025315407058237"><li><a href="http://dx.doi.org/10.1017/s0025315407058237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kober%5Bauthor%5D+AND+On+the+phylogenetic+relationships+of+Hadromerid+and+Poecilosclerid+sponges" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKober+%22On+the+phylogenetic+relationships+of+Hadromerid+and+Poecilosclerid+sponges%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0050437-Erpenbeck3" id="pone.0050437-Erpenbeck3"></a>Erpenbeck D, List-Armitage SE, Alvarez B, Degnan BM, Hooper JNA, et al. (2007) The systematics of Raspailiidae (Demospongiae, Poecilosclerida, Microcionina) reanalysed with a ribosomal marker. J Mar Biol Soc UK 87: 1571–1576.
        doi: 10.1017/s0025315407058201 <ul class="find" data-citedArticleID="6679953" data-doi="10.1017/s0025315407058201"><li><a href="http://dx.doi.org/10.1017/s0025315407058201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Erpenbeck%5Bauthor%5D+AND+The+systematics+of+Raspailiidae+%28Demospongiae%2C+Poecilosclerida%2C+Microcionina%29+reanalysed+with+a+ribosomal+marker" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AErpenbeck+%22The+systematics+of+Raspailiidae+%28Demospongiae%2C+Poecilosclerida%2C+Microcionina%29+reanalysed+with+a+ribosomal+marker%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0050437-Hooper3" id="pone.0050437-Hooper3"></a>Hooper JNA, van Soest RWM (2002) Order Hadromerida Topsent, 1894. In: Hooper JNA, van Soest RWM, editors. Systema Porifera. A guide to the classification of sponges. New York: Kluwer Academic/Plenum Publishers. pp. 169–172. <ul class="find-nolinks"></ul></li><li><span class="label">77.
              </span><a name="pone.0050437-Drummond1" id="pone.0050437-Drummond1"></a>Drummond AJ, Kearse M, Heled J, Moir R, Thierer T, et al.. (2006) Geneious v1.2.1, Available: <a href="http://www.geneious.com/">http://www.geneious.com/</a>. <ul class="find-nolinks"></ul></li><li><span class="label">78.
              </span><a name="pone.0050437-Altschul1" id="pone.0050437-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al. (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res 25: 3389–3402.
        doi: 10.1093/nar/25.17.3389 <ul class="find" data-citedArticleID="6679959" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Altschul%5Bauthor%5D+AND+Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAltschul+%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0050437-Edgar1" id="pone.0050437-Edgar1"></a>Edgar RC (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res 32: 1792–1797.
        doi: 10.1093/nar/gkh340 <ul class="find" data-citedArticleID="6679961" data-doi="10.1093/nar/gkh340"><li><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Edgar%5Bauthor%5D+AND+MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AEdgar+%22MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0050437-Gouy1" id="pone.0050437-Gouy1"></a>Gouy M, Guindon S, Gascuel O (2010) SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building. Mol Biol Evol 27: 221–224.
        doi: 10.1093/molbev/msp259 <ul class="find" data-citedArticleID="6679963" data-doi="10.1093/molbev/msp259"><li><a href="http://dx.doi.org/10.1093/molbev/msp259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gouy%5Bauthor%5D+AND+SeaView+version+4%3A+a+multiplatform+graphical+user+interface+for+sequence+alignment+and+phylogenetic+tree+building" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGouy+%22SeaView+version+4%3A+a+multiplatform+graphical+user+interface+for+sequence+alignment+and+phylogenetic+tree+building%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0050437-Riesgo1" id="pone.0050437-Riesgo1"></a>Riesgo A, Pérez-Porro AR, Carmona S, Leys SP, Giribet G (2012) Optimization of preservation and storage time of sponge tissue samples to obtain quality mRNA for Next-Generation Sequencing with Illumina platforms. Mol Ecol Resour 12: 312–322.
        doi: 10.1111/j.1755-0998.2011.03097.x <ul class="find" data-citedArticleID="6679965" data-doi="10.1111/j.1755-0998.2011.03097.x"><li><a href="http://dx.doi.org/10.1111/j.1755-0998.2011.03097.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Riesgo%5Bauthor%5D+AND+Optimization+of+preservation+and+storage+time+of+sponge+tissue+samples+to+obtain+quality+mRNA+for+Next-Generation+Sequencing+with+Illumina+platforms" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARiesgo+%22Optimization+of+preservation+and+storage+time+of+sponge+tissue+samples+to+obtain+quality+mRNA+for+Next-Generation+Sequencing+with+Illumina+platforms%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0050437-Fortunato1" id="pone.0050437-Fortunato1"></a>Fortunato S, Adamski M, Bergum B, Guder C, Jordal S, et al. (2012) Genome-wide analysis of the sox family in the calcareous sponge <em>Sycon ciliatum</em>: multiple genes with unique expression patterns. Evodevo 3: 14.
        doi: 10.1186/2041-9139-3-14 <ul class="find" data-citedArticleID="6679967" data-doi="10.1186/2041-9139-3-14"><li><a href="http://dx.doi.org/10.1186/2041-9139-3-14" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fortunato%5Bauthor%5D+AND+Genome-wide+analysis+of+the+sox+family+in+the+calcareous+sponge+Sycon+ciliatum%3A+multiple+genes+with+unique+expression+patterns" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFortunato+%22Genome-wide+analysis+of+the+sox+family+in+the+calcareous+sponge+Sycon+ciliatum%3A+multiple+genes+with+unique+expression+patterns%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0050437-Strimmer1" id="pone.0050437-Strimmer1"></a>Strimmer K, von Haeseler A (1997) Likelihood-mapping: a simple method to visualize phylogenetic content of a sequence alignment. Proc Natl Acad Sci USA 94: 6815–6819.
        doi: 10.1073/pnas.94.13.6815 <ul class="find" data-citedArticleID="6679969" data-doi="10.1073/pnas.94.13.6815"><li><a href="http://dx.doi.org/10.1073/pnas.94.13.6815" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strimmer%5Bauthor%5D+AND+Likelihood-mapping%3A+a+simple+method+to+visualize+phylogenetic+content+of+a+sequence+alignment" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AStrimmer+%22Likelihood-mapping%3A+a+simple+method+to+visualize+phylogenetic+content+of+a+sequence+alignment%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0050437-Stamatakis1" id="pone.0050437-Stamatakis1"></a>Stamatakis A (2006) RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models. Bioinformatics 22: 2688–2690  doi:10.1093/bioinformatics/btl446. <ul class="find" data-citedArticleID="6679971" data-doi="10.1093/bioinformatics/btl446"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl446" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stamatakis%5Bauthor%5D+AND+RAxML-VI-HPC%3A+maximum+likelihood-based+phylogenetic+analyses+with+thousands+of+taxa+and+mixed+models" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AStamatakis+%22RAxML-VI-HPC%3A+maximum+likelihood-based+phylogenetic+analyses+with+thousands+of+taxa+and+mixed+models%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0050437-Lartillot1" id="pone.0050437-Lartillot1"></a>Lartillot N, Lepage T, Blanquart S (2009) PhyloBayes 3: a Bayesian software package for phylogenetic reconstruction and molecular dating. Bioinformatics 25: 2286–2288  doi:10.1093/bioinformatics/btp368. <ul class="find" data-citedArticleID="6679973" data-doi="10.1093/bioinformatics/btp368"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btp368" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lartillot%5Bauthor%5D+AND+PhyloBayes+3%3A+a+Bayesian+software+package+for+phylogenetic+reconstruction+and+molecular+dating" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALartillot+%22PhyloBayes+3%3A+a+Bayesian+software+package+for+phylogenetic+reconstruction+and+molecular+dating%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0050437-Lartillot2" id="pone.0050437-Lartillot2"></a>Lartillot N, Philippe H (2004) A Bayesian Mixture Model for Across-Site Heterogeneities in the Amino-Acid Replacement Process. Mol Biol Evol 21: 1095–1109.
        doi: 10.1093/molbev/msh112 <ul class="find" data-citedArticleID="6679975" data-doi="10.1093/molbev/msh112"><li><a href="http://dx.doi.org/10.1093/molbev/msh112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lartillot%5Bauthor%5D+AND+A+Bayesian+Mixture+Model+for+Across-Site+Heterogeneities+in+the+Amino-Acid+Replacement+Process" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALartillot+%22A+Bayesian+Mixture+Model+for+Across-Site+Heterogeneities+in+the+Amino-Acid+Replacement+Process%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0050437-Puigb1" id="pone.0050437-Puigb1"></a>Puigbò P, Garcia-Vallve S, McInerney JO (2007) TOPD/FMTS: a new software to compare phylogenetic trees. Bioinformatics 23: 1556–1558  doi:10.1093/bioinformatics/btm135. <ul class="find" data-citedArticleID="6679977" data-doi="10.1093/bioinformatics/btm135"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btm135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Puigb%C3%B2%5Bauthor%5D+AND+TOPD%2FFMTS%3A+a+new+software+to+compare+phylogenetic+trees" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APuigb%C3%B2+%22TOPD%2FFMTS%3A+a+new+software+to+compare+phylogenetic+trees%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0050437-Creevey1" id="pone.0050437-Creevey1"></a>Creevey CJ, McInerney JO (2005) Clann: Investigating phylogenetic information through supertree analyses. Bioinformatics 21: 390–392.
        doi: 10.1093/bioinformatics/bti020 <ul class="find" data-citedArticleID="6679979" data-doi="10.1093/bioinformatics/bti020"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Creevey%5Bauthor%5D+AND+Clann%3A+Investigating+phylogenetic+information+through+supertree+analyses" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACreevey+%22Clann%3A+Investigating+phylogenetic+information+through+supertree+analyses%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0050437-Cummins1" id="pone.0050437-Cummins1"></a>Cummins CA, McInerney JO (2011) A method for inferring the rate of evolution of homologous characters that can potentially improve phylogenetic inference, resolve deep divergence and correct systematic biases. Syst Biol 60: 833–844. <ul class="find" data-citedArticleID="6679981"><li><a href="http://www.crossref.org/guestquery/?auth2=Cummins&amp;atitle2=A+method+for+inferring+the+rate+of+evolution+of+homologous+characters+that+can+potentially+improve+phylogenetic+inference%2C+resolve+deep+divergence+and+correct+systematic+biases&amp;auth=Cummins&amp;atitle=A+method+for+inferring+the+rate+of+evolution+of+homologous+characters+that+can+potentially+improve+phylogenetic+inference%2C+resolve+deep+divergence+and+correct+systematic+biases" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cummins%5Bauthor%5D+AND+A+method+for+inferring+the+rate+of+evolution+of+homologous+characters+that+can+potentially+improve+phylogenetic+inference%2C+resolve+deep+divergence+and+correct+systematic+biases" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACummins+%22A+method+for+inferring+the+rate+of+evolution+of+homologous+characters+that+can+potentially+improve+phylogenetic+inference%2C+resolve+deep+divergence+and+correct+systematic+biases%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.XML" value="301495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.PDF" value="861524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t001.TIF" value="3660694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t001.PNG_I" value="88495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t001.PNG_S" value="14389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t001.PNG_M" value="82590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t001.PNG_L" value="1425116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t002.TIF" value="242753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t002.PNG_I" value="10857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t002.PNG_S" value="2561"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t002.PNG_M" value="23505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t002.PNG_L" value="50056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t003.TIF" value="114779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t003.PNG_I" value="21559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t003.PNG_S" value="3596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t003.PNG_M" value="47809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t003.PNG_L" value="37686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t004.TIF" value="90769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t004.PNG_I" value="21338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t004.PNG_S" value="3140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t004.PNG_M" value="49266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t004.PNG_L" value="36305"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.g001.TIF" value="562510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.g001.PNG_I" value="51199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.g001.PNG_S" value="16890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.g001.PNG_M" value="101243"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.g001.PNG_L" value="330974"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.g002.TIF" value="1012694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.g002.PNG_I" value="53356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.g002.PNG_S" value="17065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.g002.PNG_M" value="108459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.g002.PNG_L" value="686172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t005.TIF" value="238405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t005.PNG_I" value="9064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t005.PNG_S" value="2185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t005.PNG_M" value="20640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t005.PNG_L" value="50096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t006.TIF" value="355017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t006.PNG_I" value="17116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t006.PNG_S" value="2973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t006.PNG_M" value="38998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.t006.PNG_L" value="72064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s001.PDF" value="586539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s002.PDF" value="660475"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s003.PDF" value="718948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s004.PDF" value="798248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s005.PDF" value="690353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s006.PDF" value="774870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s007.PDF" value="761028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s008.PDF" value="59574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s009.PDF" value="60334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s010.PDF" value="59790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s011.PDF" value="59908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s012.PDF" value="58619"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s013.PDF" value="58504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s014.PDF" value="59591"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s015.PDF" value="61451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s016.PDF" value="61337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s017.PDF" value="61365"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s018.PDF" value="424434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s019.PDF" value="231776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s020.PDF" value="298272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s021.PDF" value="309292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s022.PDF" value="338942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s023.PDF" value="369937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s024.PDF" value="65219"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s025.PDF" value="71519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0050437.s026.PDF" value="58341"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0050437&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0050437" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0050437&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0050437&volume=&issue=&title=Reconstruction of Family-Level Phylogenetic Relationships within Demospongiae (Porifera) Using Nuclear Encoded Housekeeping Genes&author_name=Malcolm%20S.%20Hill%2C%20April%20L.%20Hill%2C%20Jose%20Lopez%2C%20Kevin%20J.%20Peterson%2C%20Shirley%20Pomponi%2C%20Maria%20C.%20Diaz%2C%20Robert%20W.%20Thacker%2C%20Maja%20Adamska%2C%20Nicole%20Boury-Esnault%2C%20Paco%20C%C3%A1rdenas%2C%20Andia%20Chaves-Fonnegra%2C%20Elizabeth%20Danka%2C%20Bre-Onna%20De%20Laine%2C%20Dawn%20Formica%2C%20Eduardo%20Hajdu%2C%20Gisele%20Lobo-Hajdu%2C%20Sarah%20Klontz%2C%20Christine%20C.%20Morrow%2C%20Jignasa%20Patel%2C%20Bernard%20Picton%2C%20Davide%20Pisani%2C%20Deborah%20Pohlmann%2C%20Niamh%20E.%20Redmond%2C%20John%20Reed%2C%20Stacy%20Richey%2C%20Ana%20Riesgo%2C%20Ewelina%20Rubin%2C%20Zach%20Russell%2C%20Klaus%20R%C3%BCtzler%2C%20Erik%20A.%20Sperling%2C%20Michael%20di%20Stefano%2C%20James%20E.%20Tarver%2C%20Allen%20G.%20Collins&start_page=1&end_page=16" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0050437" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0050437" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0050437" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0050437&amp;t=Reconstruction%20of%20Family-Level%20Phylogenetic%20Relationships%20within%20Demospongiae%20(Porifera)%20Using%20Nuclear%20Encoded%20Housekeeping%20Genes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0050437&title=Reconstruction%20of%20Family-Level%20Phylogenetic%20Relationships%20within%20Demospongiae%20(Porifera)%20Using%20Nuclear%20Encoded%20Housekeeping%20Genes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0050437&amp;title=Reconstruction%20of%20Family-Level%20Phylogenetic%20Relationships%20within%20Demospongiae%20(Porifera)%20Using%20Nuclear%20Encoded%20Housekeeping%20Genes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0050437" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0050437&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Reconstruction of Family-Level Phylogenetic Relationships within Demospongiae (Porifera) Using Nuclear Encoded Housekeeping Genes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0050437';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Reconstruction%20of%20Family-Level%20Phylogenetic%20Relationships%20within%20Demospongiae%20(Porifera)%20Using%20Nuclear%20Encoded%20Housekeeping%20Genes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0050437" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0050437" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+analysis%22" title="Search for articles in the subject area:'Amino acid analysis'"><div class="flagText">Amino acid analysis</div></a>
              <div data-categoryid="47251" data-articleid="141095"
                   data-categoryname="Amino acid analysis"
                   class="flagImage" title="Flag 'Amino acid analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+phylogenetics%22" title="Search for articles in the subject area:'Animal phylogenetics'"><div class="flagText">Animal phylogenetics</div></a>
              <div data-categoryid="20335" data-articleid="141095"
                   data-categoryname="Animal phylogenetics"
                   class="flagImage" title="Flag 'Animal phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bayes+theorem%22" title="Search for articles in the subject area:'Bayes theorem'"><div class="flagText">Bayes theorem</div></a>
              <div data-categoryid="32251" data-articleid="141095"
                   data-categoryname="Bayes theorem"
                   class="flagImage" title="Flag 'Bayes theorem' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="141095"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="141095"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="141095"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sponges%22" title="Search for articles in the subject area:'Sponges'"><div class="flagText">Sponges</div></a>
              <div data-categoryid="43951" data-articleid="141095"
                   data-categoryname="Sponges"
                   class="flagImage" title="Flag 'Sponges' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Topology%22" title="Search for articles in the subject area:'Topology'"><div class="flagText">Topology</div></a>
              <div data-categoryid="32019" data-articleid="141095"
                   data-categoryname="Topology"
                   class="flagImage" title="Flag 'Topology' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2654'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5326'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4701&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
