

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genetic Structure and Demographic History Reveal Migration of the Diamondback Moth Plutella xylostella (Lepidoptera: Plutellidae) from the Southern to Northern Regions of China</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0059654"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0059654" />

    <meta name="citation_title" content="Genetic Structure and Demographic History Reveal Migration of the Diamondback Moth Plutella xylostella (Lepidoptera: Plutellidae) from the Southern to Northern Regions of China"/>
    <meta itemprop="name" content="Genetic Structure and Demographic History Reveal Migration of the Diamondback Moth Plutella xylostella (Lepidoptera: Plutellidae) from the Southern to Northern Regions of China"/>

      <meta name="citation_author" content="Shu-Jun Wei"/>
            <meta name="citation_author_institution" content="Institute of Plant and Environmental Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China"/>
      <meta name="citation_author" content="Bao-Cai Shi"/>
            <meta name="citation_author_institution" content="Institute of Plant and Environmental Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China"/>
      <meta name="citation_author" content="Ya-Jun Gong"/>
            <meta name="citation_author_institution" content="Institute of Plant and Environmental Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China"/>
      <meta name="citation_author" content="Gui-Hua Jin"/>
            <meta name="citation_author_institution" content="Institute of Plant and Environmental Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China"/>
      <meta name="citation_author" content="Xue-Xin Chen"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Rice Biology and Ministry of Agriculture Key Laboratory of Agricultural Entomology, Institute of Insect Sciences, Zhejiang University, Hangzhou, China"/>
      <meta name="citation_author" content="Xiang-Feng Meng"/>
            <meta name="citation_author_institution" content="Zhumadian Academy of Agricultural Sciences, Zhumadian, China"/>

    <meta name="citation_date" content="2013/4/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0059654.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e59654"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="8"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Migration strategies of insects; citation_volume=175; citation_number=1; citation_pages=1327; citation_date=1972; " />
      <meta name="citation_reference" content="citation_title=Lowland-upland migration of sauropod dinosaurs during the Late Jurassic epoch; citation_volume=480; citation_number=2; citation_pages=513-515; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Long-distance migration: evolution and determinants; citation_volume=103; citation_number=3; citation_pages=247-260; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Animal migration and infectious disease risk; citation_volume=331; citation_number=4; citation_pages=296; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Seasonal migration to high latitudes results in major reproductive benefits in an insect; citation_volume=109; citation_number=5; citation_pages=14924-14929; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Lack of genetic differentiation between monarch butterflies with divergent migration destinations; citation_volume=21; citation_number=6; citation_pages=3433-3444; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Microsatellites isolated from diamondback moth, Plutella xylostella (L.), for studies of dispersal in Australian populations; citation_volume=5; citation_number=7; citation_pages=51-53; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Improving insect pest management through population genetic data: a case study of the mosquito Ochlerotatus caspius (Pallas); citation_volume=44; citation_number=8; citation_pages=682-691; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Use of DNA markers to study bird migration; citation_volume=147; citation_number=9; citation_pages=234-244; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Out of Africa again and again; citation_volume=416; citation_number=10; citation_pages=45-51; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Use of Y chromosome and mitochondrial DNA population structure in tracing human migrations; citation_volume=41; citation_number=11; citation_pages=539-564; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Lower predation risk for migratory birds at high latitudes; citation_volume=327; citation_number=12; citation_pages=326-327; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Recent insights from radar studies of insect flight; citation_volume=56; citation_number=13; citation_pages=337-356; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial genomes reveal the global phylogeography and dispersal routes of the migratory locust; citation_volume=21; citation_number=14; citation_pages=4344-4358; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Migration and genetic structure of the grain aphid (Sitobion avenae) in Britain related to climate and clonal fluctuation as revealed using microsatellites; citation_volume=12; citation_number=15; citation_pages=21-34; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic variation among the geographic population of the grain aphid, Sitobion avenae (Hemiptera: Aphididae) in China inferred from mitochondrial COI gene sequence; citation_volume=10; citation_number=16; citation_pages=1041-1048; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity and widespread haplotypes in a migratory dragonfly, the common green darner Anax junius; citation_volume=28; citation_number=17; citation_pages=413-421; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic analysis of cabbage loopers, Trichoplusia ni (Lepidoptera: Noctuidae), a seasonal migrant in western North America; citation_volume=4; citation_number=18; citation_pages=89-99; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Biology, ecology, and management of the diamondback moth; citation_volume=38; citation_number=19; citation_pages=275-301; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Diamondback Moth Ecology and Management: Problems, Progress and Prospects; citation_volume=58; citation_number=20; citation_pages=517-541; citation_date=2013; " />
      <meta name="citation_reference" content="citation_title=Biology of cabbage caterpillars in eastern Ontario; citation_volume=93; citation_number=21; citation_pages=61-75; citation_date=1963; " />
      <meta name="citation_reference" content="citation_title=Origin of the diamondback moth (Lepidoptera : Plutellidae); citation_volume=91; citation_number=22; citation_pages=164-167; citation_date=1998; " />
      <meta name="citation_reference" content="citation_number=23; " />
      <meta name="citation_reference" content="citation_number=24; " />
      <meta name="citation_reference" content="citation_title=Cold tolerance and overwintering of the diamondback moth (Lepidoptera: Plutellidae) in Southeastern Australia; citation_volume=38; citation_number=25; citation_pages=524-529; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Overw intering of the diamondback moth, Plutella xylostella in temperate countries; citation_volume=30; citation_number=26; citation_pages=3628-3636; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=The prevention of egg-laying on turnips by the diamondback moth; citation_volume=22; citation_number=27; citation_pages=222-226; citation_date=1915; " />
      <meta name="citation_reference" content="citation_title=Evidence for dispersal of diamondback moth Plutella xylostella (Lepidoptera: Plutellidae) into southern Ontario; citation_volume=113; citation_number=28; citation_pages=21-27; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Long distance movement of two migrant Lepidoptera in relation to synoptic weather conditions; citation_volume=2; citation_number=29; citation_pages=565-569; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=Low mtDNA diversity among widespread Australian diamondback moth Plutella xylostella (L.) suggests isolation and a founder effect; citation_volume=13; citation_number=30; citation_pages=365-373; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microsatellites reveal a lack of structure in Australian populations of the diamondback moth, Plutella xylostella (L.); citation_volume=15; citation_number=31; citation_pages=107-118; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic variation in the diamondback moth, Plutella xylostella (Lepidoptera: Yponomeutidae) in China inferred from mitochondrial COI gene sequence; citation_volume=103; citation_number=32; citation_pages=605-611; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic variability of Plutella xylostella (Lepidoptera: Plutellidae) among five field populations in China; citation_volume=L; citation_number=33; citation_pages=29-33; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA sequence variation among geographic strains of diamondback moth (Lepidoptera: Plutellidae); citation_volume=90; citation_number=34; citation_pages=590-595; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genetic difference between two field populations of Plutella xylostella ((Linné) based on four polymorphic allozymes; citation_volume=2; citation_number=35; citation_pages=1-5; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genetic population studies in diamondback moth Plutella xylostella L.(Lepidoptera; Yponomeutidae) in crucifers; citation_volume=8; citation_number=36; citation_pages=1265-1280; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial COI gene sequence-based population genetic structure of the diamondback moth, Plutella xylostella, in Korea; citation_volume=25; citation_number=37; citation_pages=155-170; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Allozymes used to estimate gene flow among populations of diamondback moth (Lepidoptera: Plutellidae) in Hawaii; citation_volume=21; citation_number=38; citation_pages=808-816; citation_date=1992; " />
      <meta name="citation_reference" content="citation_number=39; " />
      <meta name="citation_reference" content="citation_title=Radar observations of the spring migration into northeastern China of the oriental armyworm moth, Mythimna separata, and other insects; citation_volume=14; citation_number=40; citation_pages=149-162; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Radar observations of the autumn migration of the beet armyworm Spodoptera exigua (Lepidoptera: Noctuidae) and other moths in northern China; citation_volume=93; citation_number=41; citation_pages=115-124; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Spring migration and summer dispersal of Loxostege sticticalis (Lepidoptera: Pyralidae) and other insects observed with radar in northern China; citation_volume=33; citation_number=42; citation_pages=1253-1265; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Northward migration of Helicoverpa armigera (Lepidoptera: Noctuidae) and other moths in early summer observed with radar in northern China; citation_volume=97; citation_number=43; citation_pages=1874-1883; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The long-distance migration of Nilaparvata lugens (Stål)(Delphacidae) in China: radar observations of mass return flight in the autumn; citation_volume=16; citation_number=44; citation_pages=471-489; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Population genetic structure of Chilo suppressalis (Walker) (Lepidoptera: Crambidae): strong subdivision in China inferred from microsatellite markers and mtDNA gene sequences; citation_volume=17; citation_number=45; citation_pages=2880-2897; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Gene flow and population history in high dispersal marine invertebrates: mitochondrial DNA analysis of Holothuria nobilis (Echinodermata: Holothuroidea) populations from the Indo-Pacific; citation_volume=12; citation_number=46; citation_pages=2635-2648; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolution and connectivity in the world-wide migration system of the mallard: Inferences from mitochondrial DNA; citation_volume=12; citation_number=47; citation_pages=99; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Plutella maculipennis, Curt., its natural and biological control in England; citation_volume=29; citation_number=48; citation_pages=343-372; citation_date=1938; " />
      <meta name="citation_reference" content="citation_title=Diamondback moth, Plutella xylostella (Linnaeus) (Insecta: Lepidoptera: Plutellidae); citation_volume=EENY-119; citation_number=49; citation_pages=1-4; citation_date=2000; " />
      <meta name="citation_reference" content="citation_number=50; " />
      <meta name="citation_reference" content="citation_title=An exceptionally high nucleotide and haplotype diversity and a signature of positive selection for the eIF4E resistance gene in barley are revealed by allele mining and phylogenetic analyses of natural populations; citation_volume=20; citation_number=51; citation_pages=3653-3688; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Monitoring of diamondback moth (Lepidoptera : Plutellidae) resistance to spinosad, indoxacarb, and emamectin benzoate; citation_volume=99; citation_number=52; citation_pages=176-181; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mannose phosphate isomerase isoenzymes in Plutella xylostella support common genetic bases of resistance to Bacillus thuringiensis toxins in lepidopteran species; citation_volume=67; citation_number=53; citation_pages=979; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Effects of abamectin selection on the genetic differentiation within Plutella xylostella (Lepidoptera: Plutellidae) based on amplified fragment length polymorphism; citation_volume=17; citation_number=54; citation_pages=353-360; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Randomly amplified polymorphic DNA differences between strains of diamondback moth (Lepidoptera: Plutellidae) susceptible or resistant to Bacillus thuringiensis; citation_volume=88; citation_number=55; citation_pages=531-537; citation_date=1995; " />
      <meta name="citation_reference" content="citation_number=56; " />
      <meta name="citation_reference" content="citation_title=Life history of Plutella maculipennis: the diamond-back moth; citation_volume=10; citation_number=57; citation_pages=1-10; citation_date=1917; " />
      <meta name="citation_reference" content="citation_title=The diamond back moth (Plutella maculipennis); citation_volume=20; citation_number=58; citation_pages=1-7; citation_date=1926; " />
      <meta name="citation_reference" content="citation_number=59; " />
      <meta name="citation_reference" content="citation_number=60; " />
      <meta name="citation_reference" content="citation_title=Effects of low and subzero temperature on a Plutella xylostella laboratory population; citation_volume=15; citation_number=61; citation_pages=99-102; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lack of genetic and plumage differentiation in the red-billed quelea Quelea quelea across a migratory divide in southern Africa; citation_volume=12; citation_number=62; citation_pages=345-353; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Return migration of Helicoverpa armigera (Lepidoptera: Noctuidae) during autumn in northern China; citation_volume=95; citation_number=63; citation_pages=361-370; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dispersal and migration; citation_volume=7; citation_number=64; citation_pages=223-242; citation_date=1962; " />
      <meta name="citation_reference" content="citation_title=Animal migration: seasonal reversals of migrant moths; citation_volume=18; citation_number=65; citation_pages=R1007-R1009; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Dispersal and movement of insect pests; citation_volume=28; citation_number=66; citation_pages=319-335; citation_date=1983; " />
      <meta name="citation_reference" content="citation_number=67; " />
      <meta name="citation_reference" content="citation_title=Population dynamics and generation differentiating of the diamondback moth, Plutella xylostella L. in Gong zhuling; citation_volume=17; citation_number=68; citation_pages=27-31; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Behavior and ecological genetics of wind-borne migration by insects; citation_volume=42; citation_number=69; citation_pages=475-502; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Migration of Vanessa cardui and Plutella xylostella (Lepidoptera) to Spitsbergen in the summer 1978; citation_volume=58; citation_number=70; citation_pages=121-123; citation_date=1978; " />
      <meta name="citation_reference" content="citation_number=71; " />
      <meta name="citation_reference" content="citation_title=Genetic variation among Asian populations of rice planthoppers, Nilaparvata lugens and Sogatella furcifera (Hemiptera : Delphacidae): mitochondrial DNA sequences; citation_volume=89; citation_number=72; citation_pages=245-253; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Current control methods for diamondback moth and other brassica insect pests and the prospects for improved management with lepidopteran-resistant Bt vegetable brassicas in Asia and Africa; citation_volume=29; citation_number=73; citation_pages=68-79; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Research progress of the resistance management and sustainable control of diamondback moth (Plutella xylostella) in China; citation_volume=48; citation_number=74; citation_pages=247-253; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Primer premier: program for design of degenerate primers from a protein sequence; citation_volume=24; citation_number=75; citation_pages=318-319; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of trinucleotide repeat microsatellite markers for Plutella xylostella L. Molecular Ecology Notes; citation_volume=6; citation_number=76; citation_pages=1246-1248; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An economic method for the fluorescent labeling of PCR fragments; citation_volume=18; citation_number=77; citation_pages=233-234; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; citation_volume=22; citation_number=78; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods; citation_volume=28; citation_number=79; citation_pages=2731-2739; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows; citation_volume=10; citation_number=80; citation_pages=564-567; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Micro-checker: software for identifying and correcting genotyping errors in microsatellite data; citation_volume=4; citation_number=81; citation_pages=535-538; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Microsatellite analyser (MSA): a platform independent analysis tool for large microsatellite data sets; citation_volume=3; citation_number=82; citation_pages=167-169; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=GENEPOP (version 1.2): population genetics software for exact tests and ecumenicism; citation_volume=86; citation_number=83; citation_pages=248-249; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A simulated annealing approach to define the genetic structure of populations; citation_volume=11; citation_number=84; citation_pages=2571-2581; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data; citation_volume=155; citation_number=85; citation_pages=945-959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method; citation_volume=4; citation_number=86; citation_pages=359-361; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=The detection of disease clustering and a generalized regression approach; citation_volume=27; citation_number=87; citation_pages=209; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=zt: a software tool for simple and partial Mantel tests; citation_volume=7; citation_number=88; citation_pages=1-12; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood estimation of a migration matrix and effective population sizes in n subpopulations by using a coalescent approach; citation_volume=98; citation_number=89; citation_pages=4563; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models; citation_volume=19; citation_number=90; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Selection of models of DNA evolution with jModelTest; citation_volume=537; citation_number=91; citation_pages=93-112; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=The complete mitochondrial genome of Leucoptera malifoliella Costa (Lepidoptera: Lyonetiidae); citation_volume=31; citation_number=92; citation_pages=1508-1522; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Median-joining networks for inferring intraspecific phylogenies; citation_volume=16; citation_number=93; citation_pages=37-48; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic star contraction applied to Asian and Papuan mtDNA evolution; citation_volume=18; citation_number=94; citation_pages=1864-1881; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=On Steiner trees and minimum spanning trees in hypergraphs; citation_volume=31; citation_number=95; citation_pages=12-20; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism; citation_volume=123; citation_number=96; citation_pages=585-595; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Statistical tests of neutrality of mutations against population growth, hitchhiking and background selection; citation_volume=147; citation_number=97; citation_pages=915-925; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Pairwise comparisons of mitochondrial DNA sequences in stable and exponentially growing populations; citation_volume=129; citation_number=98; citation_pages=555-562; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Estimation of past demographic parameters from the distribution of pairwise differences when the mutation rates vary among sites: application to human mitochondrial DNA; citation_volume=152; citation_number=99; citation_pages=1079-1089; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=BEAST: Bayesian evolutionary analysis by sampling trees; citation_volume=7; citation_number=100; citation_pages=214; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Bayesian coalescent inference of past population dynamics from molecular sequences; citation_volume=22; citation_number=101; citation_pages=1185-1192; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0059654" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genetic Structure and Demographic History Reveal Migration of the Diamondback Moth Plutella xylostella (Lepidoptera: Plutellidae) from the Southern to Northern Regions of China"/>
    <meta name="twitter:description" content="The diamondback moth Plutella xylostella (Linnaeus) (Lepidoptera: Plutellidae) is one of the most destructive insect pests of cruciferous plants worldwide. Biological, ecological and genetic studies have indicated that this moth is migratory in many regions around the world. Although outbreaks of this pest occur annually in China and cause heavy damage, little is known concerning its migration. To better understand its migration pattern, we investigated the population genetic structure and demographic history of the diamondback moth by analyzing 27 geographical populations across China using four mitochondrial genes and nine microsatellite loci. The results showed that high haplotype diversity and low nucleotide diversity occurred in the diamondback moth populations, a finding that is typical for migratory species. No genetic differentiation among all populations and no correlation between genetic and geographical distance were found. However, pairwise analysis of the mitochondrial genes has indicated that populations from the southern region were more differentiated than those from the northern region. Gene flow analysis revealed that the effective number of migrants per generation into populations of the northern region is very high, whereas that into populations of the southern region is quite low. Neutrality testing, mismatch distribution and Bayesian Skyline Plot analyses based on mitochondrial genes all revealed that deviation from Hardy-Weinberg equilibrium and sudden expansion of the effective population size were present in populations from the northern region but not in those from the southern region. In conclusion, all our analyses strongly demonstrated that the diamondback moth migrates within China from the southern to northern regions with rare effective migration in the reverse direction. Our research provides a successful example of using population genetic approaches to resolve the seasonal migration of insects."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0059654.g005"/>

  <meta property="og:title" content="Genetic Structure and Demographic History Reveal Migration of the Diamondback Moth Plutella xylostella (Lepidoptera: Plutellidae) from the Southern to Northern Regions of China" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0059654" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9727'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=57'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3039&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0059654">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0059654" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0059654&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genetic Structure and Demographic History Reveal Migration of the Diamondback Moth <i>Plutella xylostella</i> (Lepidoptera: Plutellidae) from the Southern to Northern Regions of China
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shu-Jun Wei
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:shujun268@163.com">shujun268@163.com</a></p>

                <p>Affiliation:
                  Institute of Plant and Environmental Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bao-Cai Shi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Plant and Environmental Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ya-Jun Gong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Plant and Environmental Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gui-Hua Jin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Plant and Environmental Protection, Beijing Academy of Agriculture and Forestry Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xue-Xin Chen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  State Key Laboratory of Rice Biology and Ministry of Agriculture Key Laboratory of Agricultural Entomology, Institute of Insect Sciences, Zhejiang University, Hangzhou, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiang-Feng Meng
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Zhumadian Academy of Agricultural Sciences, Zhumadian, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 02, 2013</li>
    <li>DOI: 10.1371/journal.pone.0059654</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0059654" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0059654" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0059654" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0059654" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0059654-t001" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0059654-t002" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0059654-g001" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0059654-g002" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0059654-g003" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0059654-g004" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0059654-t003" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0059654-g005" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0059654-t004" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0059654">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0059654" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The diamondback moth <em>Plutella xylostella</em> (Linnaeus) (Lepidoptera: Plutellidae) is one of the most destructive insect pests of cruciferous plants worldwide. Biological, ecological and genetic studies have indicated that this moth is migratory in many regions around the world. Although outbreaks of this pest occur annually in China and cause heavy damage, little is known concerning its migration. To better understand its migration pattern, we investigated the population genetic structure and demographic history of the diamondback moth by analyzing 27 geographical populations across China using four mitochondrial genes and nine microsatellite loci. The results showed that high haplotype diversity and low nucleotide diversity occurred in the diamondback moth populations, a finding that is typical for migratory species. No genetic differentiation among all populations and no correlation between genetic and geographical distance were found. However, pairwise analysis of the mitochondrial genes has indicated that populations from the southern region were more differentiated than those from the northern region. Gene flow analysis revealed that the effective number of migrants per generation into populations of the northern region is very high, whereas that into populations of the southern region is quite low. Neutrality testing, mismatch distribution and Bayesian Skyline Plot analyses based on mitochondrial genes all revealed that deviation from Hardy-Weinberg equilibrium and sudden expansion of the effective population size were present in populations from the northern region but not in those from the southern region. In conclusion, all our analyses strongly demonstrated that the diamondback moth migrates within China from the southern to northern regions with rare effective migration in the reverse direction. Our research provides a successful example of using population genetic approaches to resolve the seasonal migration of insects.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wei S-J, Shi B-C, Gong Y-J, Jin G-H, Chen X-X, et al.  (2013) Genetic Structure and Demographic History Reveal Migration of the Diamondback Moth <em>Plutella xylostella</em> (Lepidoptera: Plutellidae) from the Southern to Northern Regions of China. PLoS ONE 8(4):
          e59654.
            doi:10.1371/journal.pone.0059654</p><p><strong>Editor: </strong>Keith A. Crandall, George Washington University, United States of America</p><p><strong>Received:</strong> November 6, 2012; <strong>Accepted:</strong> February 16, 2013; <strong>Published:</strong> April 2, 2013</p><p><strong>Copyright:</strong> © 2013 Wei et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The research was funded jointly by the National Basic Research Program of China (no. 2013CB127600) to XXC and SJW, the National Science Foundation of China (no. 31101661), Beijing New Star Program on Science and Technology (no. 2010B027), Special Fund for the Promotion and Innovation of Beijing Academy of Agriculture and Forestry Sciences (no. KJCX201104009) and Beijing Excellent Talents Program (no. 2010D002020000010) to SJW. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Many animals undergo seasonal migration to escape deteriorating habits, colonize new resources, avoid predation, competition and parasitism and benefit reproduction <a href="#pone.0059654-Dingle1">[1]</a>–<a href="#pone.0059654-Chapman1">[5]</a>. Understanding of the migration mechanisms is meaningful for species evolution, conservation biology and pest management <a href="#pone.0059654-Dingle1">[1]</a>, <a href="#pone.0059654-Lyons1">[6]</a>–<a href="#pone.0059654-Porretta1">[8]</a>. Compared with mammals and birds, migration pathways and destinations are poorly understood in insects <a href="#pone.0059654-Wink1">[9]</a>–<a href="#pone.0059654-McKinnon1">[12]</a>. It is difficult to apply conventional approaches–e.g., fluorescent marker dyes, radio-isotopes and radar monitoring <a href="#pone.0059654-Chapman2">[13]</a>–to study the migration of insects, due to the very long distances over which they can fly, their small size and short lifespan and rapid aerial population dilution. Population genetic approaches have the potential to overcome many of those challenges <a href="#pone.0059654-Ma1">[14]</a>, <a href="#pone.0059654-Llewellyn1">[15]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>However, migration usually homogenizes the genetic differentiation among populations <a href="#pone.0059654-Lyons1">[6]</a>, <a href="#pone.0059654-Endersby1">[7]</a>, <a href="#pone.0059654-Xu1">[16]</a>, leading to difficulty in deducing migration routes. Presently, most studies employing insect species, of which the migration patterns were well established, to test population genetic theories <a href="#pone.0059654-Lyons1">[6]</a>, <a href="#pone.0059654-Freeland1">[17]</a>, <a href="#pone.0059654-Franklin1">[18]</a>. However, very few studies could reveal the seasonal migration pattern of insects using population genetic approaches <a href="#pone.0059654-Endersby1">[7]</a>. Successful application of population genetic approaches to explore the pattern of seasonal migration requires a proper scale of the research area, time-dependent sampling, and polymorphic molecular markers, as well as the retrospective pattern of migration that the species displays.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The diamondback moth <em>Plutella xylostella</em> (Linnaeus) (Lepidoptera: Plutellidae) is one of the most destructive insect pests that affects cruciferous plants worldwide <a href="#pone.0059654-Talekar1">[19]</a>. Combined management costs and yield losses due to the diamondback moth are estimated to be 4 to 5 billion United States dollars annually worldwide <a href="#pone.0059654-Furlong1">[20]</a>. This moth is most likely of Mediterranean or South Africa origin <a href="#pone.0059654-Harcourt1">[21]</a>, <a href="#pone.0059654-Kfir1">[22]</a>, but it is now recorded everywhere where crucifers are grown. Biological studies and field surveys have found that the diamondback moth cannot successfully overwinter in temperate regions where crucifers are not grown year-round, such as in western Canada <a href="#pone.0059654-Dosdall1">[23]</a> and northern Japan <a href="#pone.0059654-Honda1">[24]</a>. In the tropics and sub-tropics where crucifers are planted throughout the year, all life stages of this moth appear year-round <a href="#pone.0059654-Gu1">[25]</a>, <a href="#pone.0059654-Ma2">[26]</a>. The diamondback moth was known to be a transoceanic migrant insect in Europe and American during its large-scale outbreaks in temperate regions <a href="#pone.0059654-HarperGray1">[27]</a>–<a href="#pone.0059654-French1">[29]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Population genetic approaches have been previously used to reveal the migration of the diamondback moth. All studies have detected no genetic differentiation within the populations of Australia and New Zealand <a href="#pone.0059654-Saw1">[30]</a>, <a href="#pone.0059654-Endersby2">[31]</a>, China <a href="#pone.0059654-Li1">[32]</a>, <a href="#pone.0059654-Wang1">[33]</a>, the United States <a href="#pone.0059654-Chang1">[34]</a>, Korea <a href="#pone.0059654-Kim1">[35]</a>–<a href="#pone.0059654-Kim2">[37]</a> and Hawaii <a href="#pone.0059654-Caprio1">[38]</a>. However, none of these studies could directly reveal the migration pattern of the diamondback moth.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In China, where the climate ranges from temperate to subtropical to tropical, diamondback moth outbreaks occur yearly and cause heavy damage. Although Ma and Chen <a href="#pone.0059654-Ma3">[39]</a> speculated that the diamondback moth populations occurring in northern China were migrant from the southern region, the moth is still considered a local pest. Compared with previously researched regions, such as England, Australia and Korea, China offers an excellent opportunity for studying the regional migration of the diamondback moth because of its large size and range of climate types.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>In the present study, we investigated the population genetic structure and demographic history of the diamondback moth across China by intense sampling, using both mitochondrial genes and microsatellite loci. The aim was to (1) apply genetic approaches to resolve the migration pattern of this insect without much background information and (2) reveal the migration pattern of the diamondback moth in China. If the migration of the diamondback moth is confirmed, the details of its life cycle and overwinter abilities in northern China should be reconsidered, and new controlling strategies based on forecasting systems could be developed for regional management of this insect pest. Additionally, the sampling and analysis methods used here would be a potential model to reveal the migration of other insects, such as the oriental armyworm <em>Mythimna separata</em> (Lepidoptera: Noctuidae) <a href="#pone.0059654-Chen1">[40]</a>, the beet armyworm <em>Spodoptera exigua</em> (Lepidoptera: Noctuidae) <a href="#pone.0059654-Feng1">[41]</a>, the meadow moth <em>Loxostege sticticalis</em> (Lepidoptera: Pyralidae) <a href="#pone.0059654-Feng2">[42]</a>, the cotton bollworm <em>Helicoverpa armigera</em> (Lepidoptera: Noctuidae) <a href="#pone.0059654-Feng3">[43]</a> and the planthopper <em>Nilaparvata lugens</em> (Hemiptera: Delphacidae) <a href="#pone.0059654-Riley1">[44]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Genetic Variation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Five mitochondrial genes, <em>cox1</em>, <em>trnD</em>, <em>atp8</em>, <em>atp6</em> and <em>nad5</em>, from the 27 populations of the diamondback moth showed different genetic variation. We observed 205, 2, 12, 60 and 93 haplotypes in the <em>cox1</em>, <em>trnD</em>, <em>atp8</em>, <em>atp6</em> and <em>nad5</em> genes, respectively, in the 794 individuals (GenBank Accession numbers: KC154863-KC155245). The <em>cox1</em> alignment had the highest variable and parsimony-informative sites, haplotype (<em>h</em>) and nucleotide (π) diversity, followed by the <em>nad5</em>, <em>atp6</em> and <em>atp8</em> alignments, suggesting that these genes possess a large amount of genetic diversity. By contrast, <em>trnD</em> demonstrated low values of the diversity parameters (<a href="#pone.0059654.s007">Table S1</a>). Thus, sequences of the <em>trnD</em> gene were excluded from the subsequent analysis because of their short sequence length and rare genetic variation.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>When the genes <em>cox1</em>, <em>atp6</em>, <em>atp8</em> and <em>nad5</em> were combined (2100 bp), 350 haplotypes were observed in 794 individuals. Among the 350 haplotypes, 246 were unique to one individual, 56 were unique to individuals from the same population, and 48 were shared by individuals from different populations. There were 272 (13.0%) variable characters, of which 156 (7.4%) were parsimony informative. The haplotype (<em>h</em>) and nucleotide (π) diversity were 0.9814 and 0.0024, respectively.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Based on the combined genes, all 27 populations showed high haplotype diversity and low nucleotide diversity. However, a relatively low value of the haplotype diversity was observed in the JSLY (0.5030), GXLZ (0.7330), JSNJ (0.7520) and FJQZ (0.7540) populations (<a href="#pone-0059654-t001">Table 1</a>).</p>
<div class="figure" id="pone-0059654-t001"><div class="img"><a name="pone-0059654-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0059654.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0059654.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Parameters of genetic diversity and demographic analysis of 27 populations of the <em>Plutella xylostella.</em></span></strong></p><span>doi:10.1371/journal.pone.0059654.t001</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>All microsatellite loci showed high levels of variation. The number of alleles ranged from 77 to 135, and the observed heterozygosity ranged from 0.17 to 0.90 in the 27 populations. There was no significant genotypic disequilibrium among the nine loci. Tests for Hardy-Weinberg equilibrium within populations and loci showed significant deviation (<a href="#pone.0059654.s008">Table S2</a>).</p>


<h4>Population Genetic Structure</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The pairwise <em>F<sub>ST</sub></em> difference based on mitochondrial genes showed significant differentiation in 521 of the 702 population pairs (<a href="#pone-0059654-t002">Table 2</a>). Most southern populations, such as FJXM, GDGZ, GXLZ and FJQZ, were significantly different from all other populations, as indicated by the <em>p</em> value of <em>F<sub>ST</sub></em>. However, there were less differentiated population pairs in the northern region than in the southern regions. The microsatellite data showed significant pairwise differentiation in 13 of the 702 population pairs.</p>
<div class="figure" id="pone-0059654-t002"><div class="img"><a name="pone-0059654-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0059654.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0059654.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Pairwise <em>F<sub>ST</sub></em> values of 27 populations of the <em>Plutella xylostella</em> based on the combined mitochondrial genes of <em>cox1</em>, <em>atp8</em>, <em>atp6</em> and <em>nad5</em>.</span></strong></p><span>doi:10.1371/journal.pone.0059654.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Next, we tested the division of the 27 populations into different geographical groups. The SAMOVA results based on mitochondrial genes showed that the <em>F<sub>ST</sub></em>, <em>F<sub>CT</sub></em> and <em>F<sub>SC</sub></em> values decreased as the group number increased from 2 to 16 (<a href="#pone.0059654.s001">Figure S1</a>). When the populations were divided into 20 groups, the biggest group consisted of the BJYQ, HBBS, JLSP, LNSY, SDQD and SDYT populations from the northern region. The STRUCTURE program results based on microsatellite data divided the 794 individuals into two clusters. However, the two clusters were almost evenly distributed among all geographical populations and different regions (<a href="#pone-0059654-g001">Figure 1</a>). The global test of <em>F<sub>ST</sub></em> in ARLEQUIN (<em>P</em> = 0.49447, 100000 Markov steps) based on microsatellite data also showed no differentiation among populations. Our analyses indicated that no geographical differentiation exists among populations in China.</p>
<div class="figure" id="pone-0059654-g001"><div class="img"><a name="pone-0059654-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0059654.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0059654.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Membership coefficients of 27 <em>Plutella xylostella</em> populations divided into two inferred clusters obtained from the STRUCTURE program.</span></strong></p><span>doi:10.1371/journal.pone.0059654.g001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Finally, the Mantel test results produced an r value of 0.16075 for combined mitochondrial genes (<em>P</em> = 0.036496) and 0.0975 for microsatellite data (<em>P</em> = 0.1584) (<a href="#pone-0059654-g002">Figure 2</a>), indicating that no correlation was found between genetic and geographical distance among the populations of the diamondback moth in China.</p>
<div class="figure" id="pone-0059654-g002"><div class="img"><a name="pone-0059654-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0059654.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0059654.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Scatter plots of genetic distance vs. geographical distance for pairwise population comparisons (for combined mitochondrial genes: r = 0.16075, <em>P</em> = 0.036496; for nine microsatellite loci: r = 0.0975, <em>P</em> = 0.1584; both analyses are calculated from 10000 randomizations).</span></strong></p><span>doi:10.1371/journal.pone.0059654.g002</span></div>

<h4>Haplotype Phylogeny</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>When the 350 haplotypes of the combined genes from China were analyzed, two obvious mtDNA groups were present in the BI and NJ trees. One group contained mostly haplotypes, whereas the other group contained 13 haplotypes that were distributed from the southern to northern regions of China (<a href="#pone-0059654-g003">Figure 3A</a>). When we compared the haplotypes of the <em>cox1</em> gene (372 bp) worldwide, 10 of the 15 haplotypes from Korea, one from Australia and one of seven from Canada were identical to haplotypes from China. The phylogenetic analysis also divided the haplotypes worldwide into two groups (<a href="#pone-0059654-g003">Figure 3B</a>). One group contained mostly haplotypes from China and all haplotypes from Korea, whereas the other one contained one haplotype from China and India, one from China, one from India, one from USA, and six from Canada.</p>
<div class="figure" id="pone-0059654-g003"><div class="img"><a name="pone-0059654-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0059654.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0059654.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Neighbor-Joining phylogenetic trees of the haplotypes of the diamondback moth from China based on the combined genes of <em>cox1</em>, <em>atp8</em>, <em>atp6</em> and <em>nad5</em> (A), and globally based on the <em>cox1</em> gene (B).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>When the haplotype was unique to one individual, the population name with a number was used as the haplotype name; When the haplotype was unique to individuals from the same population, the population name with a letter was used as the haplotype name; When the haplotype was shared by different populations from China, the “Hap” with a number was used as the haplotype name; When the haplotype was found in other countries, the country name was added/used in the haplotype name.</p>
<span>doi:10.1371/journal.pone.0059654.g003</span></div>

<h4>Haplotype Network</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The haplotype network tree of combined genes and <em>cox1</em> formed four major groups (<a href="#pone-0059654-g004">Figure 4</a>), whereas that of <em>atp6</em> and <em>nad5</em> obviously displayed a star-like pattern with the most common haplotypes in the star’s center (<a href="#pone.0059654.s002">Figure S2</a>, <a href="#pone.0059654.s003">S3</a>, <a href="#pone.0059654.s004">S4</a>). All the dominant haplotypes were evenly distributed in the southern, middle and northern regions of China, except the third dominant haplotype in the network of combined genes and <em>cox1</em>, most of which were mainly observed in the southern region of China (<a href="#pone.0059654.s010">Table S4</a>). In addition, most of the terminal haplotypes were from southern China.</p>
<div class="figure" id="pone-0059654-g004"><div class="img"><a name="pone-0059654-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0059654.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0059654.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The haplotype network of the combined mitochondrial genes of <em>cox1</em>, <em>atp8</em>, <em>atp6</em> and <em>nad5</em>.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The area of a circle is proportional to the number of observed individuals. Colors within the nodes: red, southern China; green, middle regions of China; blue, northern China.</p>
<span>doi:10.1371/journal.pone.0059654.g004</span></div>

<h4>Gene Flow</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The analysis between each pair of the 27 populations showed that high gene flow was present. Unidirectional estimates of <em>M</em> ranged from 331.7 (from QHXN to FJQZ) to 662.8 (from FJLY to LNSY) (<a href="#pone.0059654.s009">Table S3</a>). When the <em>M</em> values were translated into effective migrants per generation (<em>N<sub>e</sub>m</em>), the numbers of migrants into northern populations were high, ranging from 9.8 to 36.6 for each population. By contrast, the numbers of effective migrants into southern populations were lower than 4.8 for each population, except HNDZ and GXBS (<a href="#pone-0059654-t003">Table 3</a>). When the three geographical groups were analyzed, the migrants from the southern and middle regions to the northern regions were high, with values of 76.7 and 92.1, respectively, whereas the migrants from the northern to southern and middle regions were low, with values of 21.7 and 6.1, respectively.</p>
<div class="figure" id="pone-0059654-t003"><div class="img"><a name="pone-0059654-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0059654.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0059654.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Numbers of effective migrants per generation (<em>N<sub>e</sub>m</em>) in the 27 <em>Plutella xylostella</em> populations based on the combined genes of <em>cox1</em>, <em>atp8</em>, <em>atp6</em> and <em>nad5.</em></span></strong></p><span>doi:10.1371/journal.pone.0059654.t003</span></div>

<h4>Haplotype Distribution</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Several haplotypes were dominant and evenly distributed among the different regions of China (<a href="#pone.0059654.s010">Tables S4</a>), whereas most haplotypes were unique to individuals and populations. Both of the above haplotype types were uninformative for directly revealing the migration route. Here, we compared the distribution of two moderately dominant haplotypes (Hap31 and Hap3). In the southern region, the Hap3 haplotype was found in populations from the Fujian and Hainan provinces, whereas the Hap31 haplotype was found in the GXBS, YNQJ and HNSY populations. Furthermore, both haplotypes were found in most populations in the middle and northern regions (<a href="#pone-0059654-g005">Figure 5</a>).</p>
<div class="figure" id="pone-0059654-g005"><div class="img"><a name="pone-0059654-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0059654.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0059654.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Distributions of two dominant haplotypes of the combined mitochondrial genes of <em>cox1</em>, <em>atp8</em>, <em>atp6</em> and <em>nad5</em>.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Detailed locations and codes of the populations are shown in <a href="#pone-0059654-t004">Table 4</a>. The area of the circle indicates the number of the haplotypes Hap3 (blue-green) and Hap31 (pink). The red circle indicates the other collection locations without the haplotypes of Hap3 and Hap 31. The blue line on the map indicates the Yangtze River. Populations north of the Yangtze river could not overwinter. The arrows indicate the possible migration routes of the two haplotypes from the southern to northern regions of China.</p>
<span>doi:10.1371/journal.pone.0059654.g005</span></div>

<h4>Demographic History</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Neutrality tests were conducted using Tajima’s D and Fu’s F statistics. When the 27 populations were considered as one group, Tajima’s D was significantly negative for the genes <em>cox1</em>, <em>atp8</em>, <em>atp6</em> and <em>nad5</em>. We then calculated Tajima’s D for each of the 27 populations using the combined genes and found that ‘it was not significantly negative in all populations from southern China except the populations HNDZ (<em>p</em> = 0.0140), GDGZ (<em>p</em> = 0.003) and FJLY (<em>p</em> = 0.0160), whereas Tajima’s D was statistically negative in all populations from northern China. Similar tendencies were found using Fu’s F statistics (<a href="#pone-0059654-t001">Table 1</a>).</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>Mismatch distributions for each population obtained from the combined genes were unimodal in all populations from the northern regions, most populations from the middle regions (exceptions are JSNJ and JSLY), and four populations from the southern regions (HNDZ, HNSY, GDGZ and FJLY), indicating that a sudden demographic expansion occurred in these populations (<a href="#pone.0059654.s005">Figure S5</a>).</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>The effective population size estimated for each population showed that an obvious expansion occurred in populations from the northern regions (except HBCL and QHXN) and GXBS from the southern regions, whereas a decrease or no variation of effective population size was observed in populations from the middle and southern regions (<a href="#pone.0059654.s006">Figure S6</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Recent Introduction and Adaption of the Diamondback Moth in China</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>In the diamondback moth, low mitochondrial DNA diversity among Australian populations were found based on a 257-bp <em>cox1</em> sequence <a href="#pone.0059654-Saw1">[30]</a>. When a long <em>cox1</em> gene sequence was used for analysis of Chinese and Korea populations, the haplotype diversity became high <a href="#pone.0059654-Li1">[32]</a>, <a href="#pone.0059654-Kim2">[37]</a>. The gene sequences used in our study represent all three types of protein-coding genes (cytochrome c oxidase, ATP synthase F0 and NADH dehydrogenase subunit) and the 22 tRNA genes. These genes are located evenly throughout the circular molecules of the mitochondrial genome, which are powerful for studying the population genetic structure of the diamondback moth, as previously demonstrated in other insects <a href="#pone.0059654-Ma1">[14]</a>, <a href="#pone.0059654-Meng1">[45]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Our study revealed high haplotype diversity (<em>h</em>) with low nucleotide diversity (π) in the populations of the diamondback moth from China. A similar situation was previously found in many species, with a high migration rate and small effective population size <a href="#pone.0059654-Uthicke1">[46]</a>, <a href="#pone.0059654-Kraus1">[47]</a>. Low nucleotide diversity but high haplotype diversity might be indicative of genetic bottleneck events. The diamondback moth is believed to have originated in the Mediterranean area, the source of some of our most important crucifers <a href="#pone.0059654-Harcourt1">[21]</a>, <a href="#pone.0059654-Hardy1">[48]</a>, or in southern Africa, with a large number of indigenous plants from the Brassicaceae, and is rich in parasitoids <a href="#pone.0059654-Kfir1">[22]</a>. This pest became worldwide in the recent 156 years <a href="#pone.0059654-Capinera1">[49]</a>. For example, in Taiwan, China, the diamondback moth was first reported as a pest over 100 years ago <a href="#pone.0059654-Hori1">[50]</a>. This relatively recent introduction of the diamondback moth followed by its population expansion might be the cause of the low nucleotide diversity observed.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>A plausible explanation for the high overall degree of haplotype diversity identified might be that these haplotypes are important for adaptation to different local habitats <a href="#pone.0059654-Hofinger1">[51]</a>. The diamondback moth has developed resistance to various insecticides in the field <a href="#pone.0059654-Zhao1">[52]</a>. Variation in pesticide resistance was found between populations geographically nearby <a href="#pone.0059654-Caprio1">[38]</a>. Genetic difference was found between pesticide-resistant and susceptible strains <a href="#pone.0059654-Herrero1">[53]</a>–<a href="#pone.0059654-Heckel1">[55]</a> and between populations at different temperatures and altitudes <a href="#pone.0059654-Noran1">[56]</a>. In the tropics and subtropics, the continuous cultivation and short life cycle of the diamondback moth can result in more than 25 generations a year being exposed to synthetic insecticides routinely used by growers. This high level of selection pressure of pesticide and local climate might be the key factors leading to high haplotype diversity. This is congruent with the network analysis, in which the terminal haplotypes usually originated from the southern region. Additionally, demographic analysis showed that the populations from the southern region were in selective neutrality with a stable effective population size and no sudden demographic expansion. These analyses indicated that the diamondback moth had reached stability in gene frequency in southern China.</p>


<h4>Evidence for Migration of the Diamondback Moth in China</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Early studies presumed that the diamondback moth hibernates in host-plant debris through the winter in temperate regions where crucifers are not grown year-round <a href="#pone.0059654-Marsh1">[57]</a>, <a href="#pone.0059654-Theobald1">[58]</a>. However, in none of these studies were insects collected during the coldest months and brought out of hibernation <a href="#pone.0059654-Talekar1">[19]</a>. Presently, many studies have demonstrated that the diamondback moth cannot overwinter in temperate regions <a href="#pone.0059654-Zalucki1">[59]</a>, <a href="#pone.0059654-Saito1">[60]</a>. In China, Ma and Chen <a href="#pone.0059654-Ma3">[39]</a> first proved that the diamondback moth could not overwinter in the northeast region using indoor and outdoor data during the winter and consequently proposed the hypothesis that this moth migrates with the south-western airflow into the north-eastern area of China based on their long distance flight ability, aerial-trapped samples, and sudden increase of trapping number in the spring. Chen and Liu <a href="#pone.0059654-Chen2">[61]</a> domestrated that the lethal time for 90% (LT90) population mortality of the diamondback moth exposed to −10°C was shorter than 69 hours. Those biological and ecological studies indicated that the diamondbakc moth could not overwinter in the cold regions of China.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>In the present study, we did not find geographical differentiation among the 27 populations of the diamondback moth in China based on both mitochondrial sequences and microsatellite data. A consequent Mantel test revealed no isolation by distance, as in Australia and New Zealand revealed by microsatellite markers <a href="#pone.0059654-Endersby1">[7]</a>. The network analysis showed that all the dominant haplotypes were evenly distributed in the southern, middle and northern regions of China. These genetic features are evidence for migration, as demonstrated in other migratory species <a href="#pone.0059654-Lyons1">[6]</a>, <a href="#pone.0059654-Llewellyn1">[15]</a>, <a href="#pone.0059654-Endersby2">[31]</a>, <a href="#pone.0059654-Uthicke1">[46]</a>, <a href="#pone.0059654-Dallimer1">[62]</a>.</p>


<h4>The Migration Route within China</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>In temperate regions of the northern hemisphere, migrations of moths are often directed northward in spring and summer and southward in autumn <a href="#pone.0059654-Chapman1">[5]</a>, <a href="#pone.0059654-Feng4">[63]</a>–<a href="#pone.0059654-Card1">[65]</a>. Our analysis of gene flow indicates that the diamondback moth migrated from the southern to northern regions of China. However, rare effective migration was detected in the reverse direction. The unidirectional migration with seldom return of the progeny has been found in other insect species <a href="#pone.0059654-Stinner1">[66]</a>. Our finding is congruent with the yearly occurrence of the diamondback moth in China. First, in the northern region, the population of diamondback moth increases quickly starting in May as the temperature increases <a href="#pone.0059654-Ma4">[67]</a>, during which time the first generations are found in the southern region in field <a href="#pone.0059654-Ma5">[68]</a>. Second, many insects employ the wind to facilitate their long-distance migration because of their small body size <a href="#pone.0059654-Schneider1">[64]</a>, <a href="#pone.0059654-Gatehouse1">[69]</a>. For example, Lokki et al. <a href="#pone.0059654-Lokki1">[70]</a> reported mass migration of the diamondback moth to Spitsbergen from South Finland and Finnish Lapland, carried by a strong south-southeastern storm. Light trapping of moths on the northern coast of Tasmania has also revealed peaks in abundance of the diamondback moth in spring and summer, sometimes associated with storm fronts <a href="#pone.0059654-Endersby2">[31]</a>. The East Asian subtropical monsoon that occurs from the southeastern and southwestern to northern directions in the spring and summer facilitates migration of this insect in China from the southern to northern regions. Third, the outbreak of this pest in the northern region usually occurs in June and July. However, during mid-summer and autumn, the diamondback moth causes relative less damage with a low population quantity likely caused by the increase of natural enemies <a href="#pone.0059654-Ma5">[68]</a>, unsuitable high temperature and frequent rainfall during the summer <a href="#pone.0059654-Ma6">[71]</a>. The low population density might lead to decreased effective migration from the northern region back to the southern region.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>The detailed migration route of the diamondback moth is complicated. The effective migrants per generation from multiple populations into each northern population were high, indicating that northern populations resulted from an admixture of multiple southern populations. The effective migrants per generation into two southern populations–i.e., HNDZ and GXBS–were relatively higher than those into other southern populations, indicating migration amongst populations in the southern region. However, the high value of effective migrants per generation among northern populations might be caused by the shared sources from the southern region. When we marked the two moderately distributed haplotypes on a map, the migration route was obviously congruent with the predicted pattern based on statistical results (<a href="#pone-0059654-g005">Figure 5</a>).</p>


<h4>Origin of the Diamondback Moth in China</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>There are two groups of the diamondback moth in China and worldwide (<a href="#pone-0059654-g003">Figure 3</a>). The haplotypes from the major groups are dominant and widely distributed in China. The phylogenetic tree inferred from all haplotypes of China could represent the proportion of the two types of haplotype. However, the worldwide phylogenetic tree just represents the division of the haplotypes worldwide. Frequent migration between China and Korea is reasonable for their near distance, as revealed in rice planthoppers <a href="#pone.0059654-Mun1">[72]</a>. It is congruent with our results that all the haplotypes found in Korea and Australia were classified into the major group of China. Additionally, the same haplotypes of the diamondback moth between Australia and Korea have been reported <a href="#pone.0059654-Saw1">[30]</a>. Within the phylogenetic tree, although individuals from the North America and those from Asia and Australia formed two different groups, the haplotypes from the North American group could be found in China from the southern to northern regions. Nevertheless, the Chinese and global structures of the haplotypes might indicate that most of the diamondback moths presently occurred in China might be from the southern regions of China and/or Australia, with minor individuals introduced from other regions, such as Canada and American. It is also possible that the two groups of the diamondback moth were introduced into China and occurred afterward in the mixture.</p>


<h4>Implications for Pest Management</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Our data show that the diamondback moth migrates from southern to northern within China with rare effective migration in reverse direction. This migration has many implications for the management of this pest. Although this species develops resistance to insecticides rapidly, presently, applications of synthetic insecticides remain overwhelmingly the most common control strategy <a href="#pone.0059654-Grzywacz1">[73]</a>. Therefore, it should be considered that insecticide resistance alleles arising in the southern region might spread quickly into the northern region via migration.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>Generation predictions in the northern region should not merely depend on the development rate and the accumulated temperature; instead, it should also take into consideration the timing of the arrival of new migrants from the southern region. Aerial detection of the adult dynamics should be incorporated into the forecast of the diamondback moth.</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>In conclusion, we investigated the population genetic structure and demographic history of the diamondback moth in China. Both the mitochondrial sequences and microsatellite data showed no differentiation or isolation by distance in the 27 geographical populations, indicating migration among the diamondback moth populations in China. Pairwise differentiation based on mitochondrial sequences, gene flow, and demographic history strongly showed that the migration pattern of this moth is in the northward direction with rare effective reverse re-migration. Phylogenetic analysis of the mitochondrial gene haplotypes indicated that one dominant and one minor group were present in China in the mixture. Our research provides a successful example of a method for understanding the seasonal migration of insects.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Ethics Statement</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The diamondback moth is a pest insect of vegetables. The study of this pest is welcomed by farmers because understanding the behavior this moth may be helpful to protect their vegetables from pest damage. Thus, no specific permits were required for the described field studies. Additionally, the field studies did not involve endangered or protected species.</p>


<h4>Insect Sampling</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>In China, biological studies revealed that the diamondback moth could not overwinter in regions north of the Yangtze River <a href="#pone.0059654-Zalucki1">[59]</a>, <a href="#pone.0059654-Feng5">[74]</a>. In our study, 794 diamondback moth larvae were collected from cabbage fields at 27 locations. The sampled populations included nine collected from the south of the Yangtze River, nine from the north of the Yangtze River, and nine around the east of the Yangtze River, covering all representative regions of China (<a href="#pone-0059654-t004">Table 4</a> and <a href="#pone-0059654-g005">Figure 5</a>). Five populations were collected in the year 2010, and 22 populations were collected in 2011. Most populations from southern China were collected in the early part of 2011 when the population density of the diamondback moth was low in the field, whereas most populations from northern China were collected immediately when the population reached a high density. All specimens were preserved in absolute ethanol and stored at −80°C prior to DNA extraction.</p>
<div class="figure" id="pone-0059654-t004"><div class="img"><a name="pone-0059654-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0059654" data-uri="info:doi/10.1371/journal.pone.0059654.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0059654.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0059654.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0059654.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0059654.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0059654.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Collection details of the <em>Plutella xylostella</em> populations used in this study.</span></strong></p><span>doi:10.1371/journal.pone.0059654.t004</span></div>

<h4>DNA Extraction and Sequencing</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Total genomic DNA was extracted from one abdominal segment of individual larva using the DNeasy Blood and Tissue Kit (Qiagen, Germany). The gut was removed prior to DNA extraction.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>To choose the gene region for the population genetic study and design the primers for amplification, we sequenced the complete mitochondrial genome of the diamondback moth as the reference sequence (GenBank accession No. JF911819). Three segments from mitochondrial genomes, including a partial sequence of <em>cox1</em> and <em>nad5</em> and full-length sequences of <em>atp8</em>, <em>atp6</em> and <em>trnD</em>, were used as molecular markers for the current study according to the types of the mitochondrial protein-coding genes and previous studies <a href="#pone.0059654-Meng1">[45]</a>. Primers were designed using Primer Premier version 5 <a href="#pone.0059654-Singh1">[75]</a> according to the reference mitochondrial genome (<a href="#pone.0059654.s011">Table S5</a>). Polymerase chain reaction (PCR) was conducted using the Mastercycler pro system (Eppendorf, Germany) under the following conditions: an initial denaturation for 2 min at 94°C, followed by 35 cycles of 10 s at 96°C, 15 s at 48°C for the region of <em>cox1</em>-<em>cox2</em> and 54°C for the regions of <em>atp8</em>-<em>atp6</em> and <em>nad5</em>, and 1 min at 72°C, and a subsequent final extension for 10 min at 72°C. PCR components were added as recommended by the manufacturer of Takara LA Taq (Takara Biomedical, Japan). Amplified products were purified and sequenced directly from both strands using the ABI 3730xl DNA Analyzer by Sanboyuanzhi Biotechnology Co., Ltd (Beijing, China).</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>Two sets of microsatellite primers have been reported for the diamondback moth <a href="#pone.0059654-Endersby1">[7]</a>, <a href="#pone.0059654-Esselink1">[76]</a>. Those developed by Endersby et al. <a href="#pone.0059654-Endersby1">[7]</a> were found to produce many null alleles in populations outside Australia <a href="#pone.0059654-Endersby2">[31]</a>. Thus, we used nine microsatellite loci from Esselink et al. <a href="#pone.0059654-Esselink1">[76]</a> (<a href="#pone.0059654.s012">Table S6</a>). All loci were fluorescently labeled and amplified following the method of Schuelke <a href="#pone.0059654-Schuelke1">[77]</a> under the annealing temperatures recommended by Esselink et al. <a href="#pone.0059654-Esselink1">[76]</a> for the first 30 cycles. The sizes of the amplified PCR products were determined using the ABI 3730xl DNA Analyzer (Applied Biosystems) with GeneScan 500 LIZ size standard (Applied Biosystems), performed by Tsingke Biotechnology Co., Ltd (Beijing, China). Allele designation was obtained using the software GENEMAPPER, version 4.0 (Applied Biosystems).</p>


<h4>Statistical Analysis</h4>
<h5>Genetic diversity.</h5><a id="article1.body1.sec4.sec4.sec1.p1" name="article1.body1.sec4.sec4.sec1.p1"></a><p>Sequencing results determined from both strands were assembled using the SEQMAN program within the LASERGENE suite version 7.1.2 (DNASTAR, Inc., USA). Sequences of the five genes were aligned independently using CLUSTALW <a href="#pone.0059654-Thompson1">[78]</a> implemented in MEGA version 5 <a href="#pone.0059654-Tamura1">[79]</a> with default parameters. Alignment of nucleotide sequences of the protein-coding genes was inferred from the amino acid alignment. The number of polymorphic sites (<em>S</em>), haplotype diversity (<em>h</em>), nucleotide diversity (<em>π</em>) and the average number of pairwise differences (<em>P</em>) were calculated using the ARLEQUIN suite, version 3.5 <a href="#pone.0059654-Excoffier1">[80]</a>.</p>
<a id="article1.body1.sec4.sec4.sec1.p2" name="article1.body1.sec4.sec4.sec1.p2"></a><p>The microsatellite data determined from GENEMAPPER were first checked for stuttering, large allele dropout and null alleles using MICRO-CHECKER version 2.2.3 <a href="#pone.0059654-vanOosterhout1">[81]</a>. Allele frequency, the number of alleles, observed and expected heterozygosity, <em>F<sub>ST</sub></em> and genetic distances were calculated using MICROSATELLITE ANALYSER (MSA) version 4.0.5 <a href="#pone.0059654-Dieringer1">[82]</a>. Tests for deviation from Hardy-Weinberg equilibrium at each locus for each population were performed using GENEPOP version 4.0.11 <a href="#pone.0059654-Raymond1">[83]</a>.</p>

<h5>Population structure.</h5><a id="article1.body1.sec4.sec4.sec2.p1" name="article1.body1.sec4.sec4.sec2.p1"></a><p>For mitochondrial genes, the pairwise <em>F<sub>ST</sub></em> values between each pair of the 27 populations were calculated using the ARLEQUIN suite, version 3.5 <a href="#pone.0059654-Excoffier1">[80]</a>. The method of simulated annealing was implemented in the SAMOVA program version 1.0 <a href="#pone.0059654-Dupanloup1">[84]</a> to examine the genetic structure. The number of groups was set from 2 to 20, and the number of initial conditions was set at 100. The pairwise difference was used to calculate the molecular distance. The values of fixation indices among populations within groups (<em>F<sub>SC</sub></em>), among groups (<em>F<sub>CT</sub></em>) and within populations (<em>F<sub>ST</sub></em>) were compared among different group numbers within each dataset. For microsatellite data, population differentiation was investigated using the Bayesian clustering approach implemented in the program STRUCTURE, version 2.3.3 <a href="#pone.0059654-Pritchard1">[85]</a>. The admixture ancestry model and the correlated allele frequency model with a burn-in period of 50000 iterations and 1 million Markov chain Monte Carlo repetitions were used to calculate the probable number of genetic clusters (<em>K</em>). We performed 35 independent runs for each <em>K</em> (from 1 to 20) to confirm consistency across runs. The most accurate number of groups (<em>K</em>) was visually examined when plotting <em>K</em> against Δ<em>K</em> and using the Evanno method in the online program STRUCTURE HARVESTER <a href="#pone.0059654-Earl1">[86]</a>.</p>
<a id="article1.body1.sec4.sec4.sec2.p2" name="article1.body1.sec4.sec4.sec2.p2"></a><p>To establish whether any isolation-by-distance effect occurred, matrices of genetic distance data [<em>F<sub>ST</sub></em>/(1-<em>F<sub>ST</sub></em>)] and the logarithms of geographical distance (ln Km) between all the sampling sites were constructed. These matrices were analyzed for their degree of correlation using a Mantel test <a href="#pone.0059654-Mantel1">[87]</a> implemented in the software ZT version, 1.1 <a href="#pone.0059654-Bonnet1">[88]</a>.</p>

<h5>Gene flow.</h5><a id="article1.body1.sec4.sec4.sec3.p1" name="article1.body1.sec4.sec4.sec3.p1"></a><p>To test whether there was asymmetric dispersal between populations, the mutation-scaled population size (θ = <em>N<sub>e</sub></em>µ, where μ is the mutation rate per site per generation) and the mutation-scaled migration rate (<em>M</em> = <em>m</em>/µ, where m is the migration rate) were calculated using Bayesian search strategies and the software MIGRATE, version 3.2.16 <a href="#pone.0059654-Beerli1">[89]</a>. The 27 populations were either directly used or divided into three geographical groups–i.e., south, middle and north (<a href="#pone-0059654-t004">Table 4</a>)–for analysis. The effective number of migrants entering and leaving each population per generation <em>xN<sub>e</sub>m</em> is θ<em>M</em> (<em>x</em> is a multiplier that depends on the ploidy and inheritance of the data, and here, <em>x</em> is one for mitochondrial DNA). The parameters used in the calculations are as follows: long-chains = 1, long-inc = 20, long-sample = 1000000, burn-in = 1000000, heating = YES: 1: (1.0, 1.5, 3.0, 6.0), heated-swap = YES and replicate = YES: 5. Four runs of MIGRATE analysis were conducted to verify the consistency in our results. For each run, we changed the random number seed and starting values for θ and <em>M</em>. In the first run, θ and <em>M</em> were estimated from <em>F<sub>ST</sub></em> values and in the subsequent runs. In the subsequent runs, Bayesian estimates of θ and <em>M</em> from the previous run were used. The estimated θ, <em>M</em> and θ<em>M</em> from the final run are reported here.</p>

<h5>Haplotype phylogeny and network analysis.</h5><a id="article1.body1.sec4.sec4.sec4.p1" name="article1.body1.sec4.sec4.sec4.p1"></a><p>The phylogenetic relationship among the 350 haplotypes from the combined genes of <em>cox1</em>, <em>atp8</em>, <em>atp6</em> and <em>nad5</em> sequenced in our study were constructed. To compare the phylogenetic relationship of the haplotypes from China and other regions, the partial sequence of the <em>cox1</em> gene of the diamondback moth was downloaded from GenBank. There are 163 sequences with homologous regions to the <em>cox1</em> segment sequenced in our study. A total of 106 sequences with information regarding sampling location were left for further analysis. Phylogenetic inferences were conducted using the Bayesian method (BI) in the software MRBAYES, version 3.2 <a href="#pone.0059654-Ronquist1">[90]</a> and the Neigbor-Joining (NJ) method in the software MEGA version 5 <a href="#pone.0059654-Tamura1">[79]</a>. The software jModeltest version 0.1.1 was used to aid the selection of the best-fit nucleotide substitution model <a href="#pone.0059654-Posada1">[91]</a>. The GTR+I+G model for four-state nucleotide sequences was used in the BI analysis. The data matrix was partitioned by three codon positions. Bayesian analyses were conducted with eight independent Markov chains that were run for 10,000,000 Metropolis-coupled MCMC generations, with tree sampling every 1000 generations and a burn-in of 2500 trees. Trees were visualized and produced using FIGTREE software version 1.3.1 (<a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtre​e/</a>). For the NJ method, the LogDet (Tamura-Kumar) model was used for analysis. The species <em>Leucoptera malifoliella</em> (Lepidoptera: Yponomeutoidea: Lyonetiidae) was selected as outgroup, because this is the evolutionary nearest species to the diamondback moth with sequenced mitochondrial genome sequence <a href="#pone.0059654-Wu1">[92]</a>.</p>
<a id="article1.body1.sec4.sec4.sec4.p2" name="article1.body1.sec4.sec4.sec4.p2"></a><p>The haplotype network of each <em>cox1</em>, <em>atp6</em>, <em>nad5</em> gene and the combined genes were inferred using the median-joining algorithm <a href="#pone.0059654-Bandelt1">[93]</a>. Before the calculation, the star contraction method with a maximum star radius value of 10 was used to simplify the data matrix of the <em>cox1</em> gene and the combined genes <a href="#pone.0059654-Forster1">[94]</a>. The “Frequency&gt;1” criterion, which simplifies networks by ignoring unique haplotypes in the data set, was used, and the epsilon value was set to 0 for the calculation. After the calculation, the MP calculation was used to identify and clean up unnecessary median vectors and links <a href="#pone.0059654-Polzin1">[95]</a>. All the above calculations were conducted using the software Network, version 4.6.1.0 (Fluxus Technology Ltd, England). The network’s results were drawn and prepared using the software Network Publisher, version 1.3.0.0 (Fluxus Technology Ltd, England). The shortest trees with median vectors were shown.</p>

<h5>Demographic analysis.</h5><a id="article1.body1.sec4.sec4.sec5.p1" name="article1.body1.sec4.sec4.sec5.p1"></a><p>The demographic history of each population was examined. Background selection using Tajima’s D <a href="#pone.0059654-Tajima1">[96]</a>, and Fu’s F statistics <a href="#pone.0059654-Fu1">[97]</a> were tested. The distribution of pairwise differences between individual sequences was analyzed using mismatch distribution analysis <a href="#pone.0059654-Slatkin1">[98]</a>, <a href="#pone.0059654-Schneider2">[99]</a>. Both analyses were conducted using the program ARLEQUIN suite, version 3.5 <a href="#pone.0059654-Excoffier1">[80]</a>.</p>
<a id="article1.body1.sec4.sec4.sec5.p2" name="article1.body1.sec4.sec4.sec5.p2"></a><p>Bayesian Skyline Plot implemented in the software BEAST, version 1.7.2 <a href="#pone.0059654-Drummond1">[100]</a>, was used to investigate the population history of the diamondback moth by estimating changes in the effective population size over time <a href="#pone.0059654-Drummond2">[101]</a>. Each of the 27 populations was analyzed independently. Because all individuals were collected around the year 2010 or 2011, we set the date value as 0 to be default. The SRD06 model was used for analysis. This model links 1st and 2nd codon positions but allows the 3rd positions to have a different relative rate of substitution, transition-transversion ratio and gamma-distributed rate heterogeneity. The strict clock model was used with the estimated rate. The prior uniform distribution was used for the rate clock with a lower value of 0 and upper value of 1. Two independent runs with 100 million generations were performed. Samples from the Markov Chain were taken every 10000 steps. The output of BEAST was compared and analyzed using Tracer, version 1.5 (<a href="http://tree.bio.ed.ac.uk/software/tracer/">http://tree.bio.ed.ac.uk/software/tracer​/</a>). When the two runs were converging on the same distribution, one of them was used for Bayesian Skyline Reconstruction analysis. Otherwise, additional runs were performed with increased generations until two of them were converging.</p>


</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0059654"></div><a name="pone.0059654.s001" id="pone.0059654.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI"><strong>The values of fixation indices among populations within groups (</strong><strong><em>F<sub>SC</sub></em></strong><strong>), among groups (</strong><strong><em>F<sub>CT</sub></em></strong><strong>) and within populations (</strong><strong><em>F<sub>ST</sub></em></strong><strong>) when the 27 populations are divided into 2 to 20 geographical groups, calculated using SAMOVA based on the combined mitochondrial genes of </strong><strong><em>cox1</em></strong><strong>, </strong><strong><em>atp8</em></strong><strong>, </strong><strong><em>atp6</em></strong><strong> and </strong><strong><em>nad5</em></strong><strong>.</strong></p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0059654.s002" id="pone.0059654.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI"><strong>The haplotype network of the gene </strong><strong><em>cox1</em></strong><strong>.</strong> The area of a circle is proportional to the number of observed individuals. Colors within the nodes indicate the following: red, southern China; green, middle regions of China; blue, northern China.</p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0059654.s003" id="pone.0059654.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI"><strong>The haplotype network of the gene </strong><strong><em>atp6</em></strong><strong>.</strong> The area of a circle is proportional to the number of observed individuals. Colors within the nodes indicate the following: red, southern China; green, middle regions of China; blue, northern China.</p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0059654.s004" id="pone.0059654.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI"><strong>The haplotype network of the gene </strong><strong><em>nad5</em></strong><strong>.</strong> The area of a circle is proportional to the number of observed individuals. Colors within the nodes indicate the following: red, southern China; green, middle regions of China; blue, northern China.</p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0059654.s005" id="pone.0059654.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI"><strong>Mismatch distribution of the combined genes of </strong><strong><em>cox1</em></strong><strong>, </strong><strong><em>atp8</em></strong><strong>, </strong><strong><em>atp6</em></strong><strong> and </strong><strong><em>nad5</em></strong><strong> in the 27 populations of the </strong><strong><em>Plutella xylostella</em></strong><strong> from China using the ARLEQUIN suite, version 3.5</strong></p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0059654.s006" id="pone.0059654.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI"><strong>Bayesian Skyline Plot of the 27 populations of the </strong><strong><em>Plutella xylostella</em></strong><strong> from China based on the combined genes of </strong><strong><em>cox1</em></strong><strong>, </strong><strong><em>atp8</em></strong><strong>, </strong><strong><em>atp6</em></strong><strong> and </strong><strong><em>nad5</em></strong><strong>.</strong></p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(PDF)</p>
<a name="pone.0059654.s007" id="pone.0059654.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s007">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI"><strong>Genetic diversity of the five mitochondrial genes and demographic analysis of the 27 </strong><strong><em>Plutella xylostella</em></strong><strong> populations as one group.</strong></p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(DOCX)</p>
<a name="pone.0059654.s008" id="pone.0059654.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s008">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI"><strong>Summary statistics of the nine microsatellite loci examined in the 27 populations of the </strong><strong><em>Plutella xylostella</em></strong><strong>.</strong></p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(DOCX)</p>
<a name="pone.0059654.s009" id="pone.0059654.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s009">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI"><strong>Estimates (</strong><strong><em>M</em></strong><strong> and θ) of the migration among 27 </strong><strong><em>Plutella xylostella</em></strong><strong> populations from China based on combined genes.</strong></p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0059654.s010" id="pone.0059654.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s010">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI"><strong>The number of the dominant haplotypes in the 27 populations.</strong> Hap4, Hap31 and Hap8 are the three most dominant haplotypes in the combined genes, and Hap4, Hap2 and Hap2 are the most dominant haplotypes in the genes <em>cox1</em>, <em>atp6</em> and <em>nad5</em>, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(DOCX)</p>
<a name="pone.0059654.s011" id="pone.0059654.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s011">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI"><strong>Mitochondrial genes and their amplification conditions used in this study.</strong></p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(DOCX)</p>
<a name="pone.0059654.s012" id="pone.0059654.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0059654.s012">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI"><strong>Microsatellite loci used in this study.</strong></p>
<p class="siDoi">doi:10.1371/journal.pone.0059654.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(DOCX)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Cheng-ming Shi (Institute of Zoology, Chinese Academy of Sciences), Li-jun He (East China Normal University), Chun-sen Ma and Bo Zhang (Chinese Academy of Agricultural Sciences) for their help on the data analysis, and Dr. Anthony Partridge for his language correction. We also thank those who contributed to sampling of the diamondback moth for this research, including Li-jun Cao, Qing-sheng Lin, Zhen-qiang Qin, Jin-ming Zhang, Jia-ying Zhu, Peng Xu, Ming-qing Gao, Ming Gu, Yu-zhou Du, Guan-hong Wang, Xiao-qiang Chu, Xiao Fan, Rong-shan Zhou, Hui Huang and Xiu-dong Nie.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SJW BCS YJG. Performed the experiments: SJW GHJ. Analyzed the data: SJW XXC XFM. Contributed reagents/materials/analysis tools: SJW. Wrote the paper: SJW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0059654-Dingle1" id="pone.0059654-Dingle1"></a>Dingle H (1972) Migration strategies of insects. Science 175: 1327.
        doi: 10.1126/science.175.4028.1327 <ul class="find" data-citedArticleID="7327479" data-doi="10.1126/science.175.4028.1327"><li><a href="http://dx.doi.org/10.1126/science.175.4028.1327" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dingle%5Bauthor%5D+AND+Migration+strategies+of+insects" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADingle+%22Migration+strategies+of+insects%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0059654-Fricke1" id="pone.0059654-Fricke1"></a>Fricke HC, Hencecroth J, Hoerner ME (2011) Lowland-upland migration of sauropod dinosaurs during the Late Jurassic epoch. Nature 480: 513–515.
        doi: 10.1038/nature10570 <ul class="find" data-citedArticleID="7327481" data-doi="10.1038/nature10570"><li><a href="http://dx.doi.org/10.1038/nature10570" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fricke%5Bauthor%5D+AND+Lowland-upland+migration+of+sauropod+dinosaurs+during+the+Late+Jurassic+epoch" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFricke+%22Lowland-upland+migration+of+sauropod+dinosaurs+during+the+Late+Jurassic+epoch%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0059654-Alerstam1" id="pone.0059654-Alerstam1"></a>Alerstam T, Hedenström A, Åkesson S (2003) Long-distance migration: evolution and determinants. Oikos 103: 247–260.
        doi: 10.1034/j.1600-0706.2003.12559.x <ul class="find" data-citedArticleID="7327483" data-doi="10.1034/j.1600-0706.2003.12559.x"><li><a href="http://dx.doi.org/10.1034/j.1600-0706.2003.12559.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alerstam%5Bauthor%5D+AND+Long-distance+migration%3A+evolution+and+determinants" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAlerstam+%22Long-distance+migration%3A+evolution+and+determinants%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0059654-Altizer1" id="pone.0059654-Altizer1"></a>Altizer S, Bartel R, Han BA (2011) Animal migration and infectious disease risk. Science 331: 296.
        doi: 10.1126/science.1194694 <ul class="find" data-citedArticleID="7327485" data-doi="10.1126/science.1194694"><li><a href="http://dx.doi.org/10.1126/science.1194694" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Altizer%5Bauthor%5D+AND+Animal+migration+and+infectious+disease+risk" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAltizer+%22Animal+migration+and+infectious+disease+risk%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0059654-Chapman1" id="pone.0059654-Chapman1"></a>Chapman JW, Bell JR, Burgin LE, Reynolds DR, Pettersson LB, et al. (2012) Seasonal migration to high latitudes results in major reproductive benefits in an insect. Proceedings of the National Academy of Sciences 109: 14924–14929.
        doi: 10.1073/pnas.1207255109 <ul class="find" data-citedArticleID="7327487" data-doi="10.1073/pnas.1207255109"><li><a href="http://dx.doi.org/10.1073/pnas.1207255109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chapman%5Bauthor%5D+AND+Seasonal+migration+to+high+latitudes+results+in+major+reproductive+benefits+in+an+insect" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AChapman+%22Seasonal+migration+to+high+latitudes+results+in+major+reproductive+benefits+in+an+insect%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0059654-Lyons1" id="pone.0059654-Lyons1"></a>Lyons JI, Pierce AA, Barribeau SM, Sternberg ED, Mongue AJ, et al. (2012) Lack of genetic differentiation between monarch butterflies with divergent migration destinations. Molecular Ecology 21: 3433–3444.
        doi: 10.1111/j.1365-294x.2012.05613.x <ul class="find" data-citedArticleID="7327489" data-doi="10.1111/j.1365-294x.2012.05613.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2012.05613.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lyons%5Bauthor%5D+AND+Lack+of+genetic+differentiation+between+monarch+butterflies+with+divergent+migration+destinations" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALyons+%22Lack+of+genetic+differentiation+between+monarch+butterflies+with+divergent+migration+destinations%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0059654-Endersby1" id="pone.0059654-Endersby1"></a>Endersby NM, McKechnie SW, Vogel H, Gahan LJ, Baxter SW, et al. (2005) Microsatellites isolated from diamondback moth, Plutella xylostella (L.), for studies of dispersal in Australian populations. Molecular Ecology Notes 5: 51–53.
        doi: 10.1111/j.1471-8286.2004.00827.x <ul class="find" data-citedArticleID="7327491" data-doi="10.1111/j.1471-8286.2004.00827.x"><li><a href="http://dx.doi.org/10.1111/j.1471-8286.2004.00827.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endersby%5Bauthor%5D+AND+Microsatellites+isolated+from+diamondback+moth%2C+Plutella+xylostella+%28L.%29%2C+for+studies+of+dispersal+in+Australian+populations" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AEndersby+%22Microsatellites+isolated+from+diamondback+moth%2C+Plutella+xylostella+%28L.%29%2C+for+studies+of+dispersal+in+Australian+populations%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0059654-Porretta1" id="pone.0059654-Porretta1"></a>Porretta D, Canestrelli D, Bellini R, Celli G, Urbanelli S (2007) Improving insect pest management through population genetic data: a case study of the mosquito <em>Ochlerotatus caspius</em> (Pallas). Journal of Applied Ecology 44: 682–691.
        doi: 10.1111/j.1365-2664.2007.01301.x <ul class="find" data-citedArticleID="7327493" data-doi="10.1111/j.1365-2664.2007.01301.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2664.2007.01301.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Porretta%5Bauthor%5D+AND+Improving+insect+pest+management+through+population+genetic+data%3A+a+case+study+of+the+mosquito+Ochlerotatus+caspius+%28Pallas%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APorretta+%22Improving+insect+pest+management+through+population+genetic+data%3A+a+case+study+of+the+mosquito+Ochlerotatus+caspius+%28Pallas%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0059654-Wink1" id="pone.0059654-Wink1"></a>Wink M (2006) Use of DNA markers to study bird migration. Journal of Ornithology 147: 234–244.
        doi: 10.1007/s10336-006-0065-5 <ul class="find" data-citedArticleID="7327495" data-doi="10.1007/s10336-006-0065-5"><li><a href="http://dx.doi.org/10.1007/s10336-006-0065-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wink%5Bauthor%5D+AND+Use+of+DNA+markers+to+study+bird+migration" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWink+%22Use+of+DNA+markers+to+study+bird+migration%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0059654-Templeton1" id="pone.0059654-Templeton1"></a>Templeton A (2002) Out of Africa again and again. Nature 416: 45–51.
        doi: 10.1038/416045a <ul class="find" data-citedArticleID="7327497" data-doi="10.1038/416045a"><li><a href="http://dx.doi.org/10.1038/416045a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Templeton%5Bauthor%5D+AND+Out+of+Africa+again+and+again" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATempleton+%22Out+of+Africa+again+and+again%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0059654-Underhill1" id="pone.0059654-Underhill1"></a>Underhill PA, Kivisild T (2007) Use of Y chromosome and mitochondrial DNA population structure in tracing human migrations. Annual Review of Genetics 41: 539–564.
        doi: 10.1146/annurev.genet.41.110306.130407 <ul class="find" data-citedArticleID="7327499" data-doi="10.1146/annurev.genet.41.110306.130407"><li><a href="http://dx.doi.org/10.1146/annurev.genet.41.110306.130407" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Underhill%5Bauthor%5D+AND+Use+of+Y+chromosome+and+mitochondrial+DNA+population+structure+in+tracing+human+migrations" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AUnderhill+%22Use+of+Y+chromosome+and+mitochondrial+DNA+population+structure+in+tracing+human+migrations%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0059654-McKinnon1" id="pone.0059654-McKinnon1"></a>McKinnon L, Smith P, Nol E, Martin J, Doyle F, et al. (2010) Lower predation risk for migratory birds at high latitudes. Science 327: 326–327.
        doi: 10.1126/science.1183010 <ul class="find" data-citedArticleID="7327501" data-doi="10.1126/science.1183010"><li><a href="http://dx.doi.org/10.1126/science.1183010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=McKinnon%5Bauthor%5D+AND+Lower+predation+risk+for+migratory+birds+at+high+latitudes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMcKinnon+%22Lower+predation+risk+for+migratory+birds+at+high+latitudes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0059654-Chapman2" id="pone.0059654-Chapman2"></a>Chapman JW, Drake VA, Reynolds DR (2011) Recent insights from radar studies of insect flight. Annual Review of Entomology 56: 337–356.
        doi: 10.1146/annurev-ento-120709-144820 <ul class="find" data-citedArticleID="7327503" data-doi="10.1146/annurev-ento-120709-144820"><li><a href="http://dx.doi.org/10.1146/annurev-ento-120709-144820" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chapman%5Bauthor%5D+AND+Recent+insights+from+radar+studies+of+insect+flight" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AChapman+%22Recent+insights+from+radar+studies+of+insect+flight%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0059654-Ma1" id="pone.0059654-Ma1"></a>Ma C, Yang P, Jiang F, Chapuis MP, Shali Y, et al. (2012) Mitochondrial genomes reveal the global phylogeography and dispersal routes of the migratory locust. Molecular Ecology 21: 4344–4358.
        doi: 10.1111/j.1365-294x.2012.05684.x <ul class="find" data-citedArticleID="7327505" data-doi="10.1111/j.1365-294x.2012.05684.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2012.05684.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ma%5Bauthor%5D+AND+Mitochondrial+genomes+reveal+the+global+phylogeography+and+dispersal+routes+of+the+migratory+locust" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMa+%22Mitochondrial+genomes+reveal+the+global+phylogeography+and+dispersal+routes+of+the+migratory+locust%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0059654-Llewellyn1" id="pone.0059654-Llewellyn1"></a>Llewellyn K, Loxdale H, Harrington R, Brookes C, Clark S, et al. (2003) Migration and genetic structure of the grain aphid (<em>Sitobion avenae</em>) in Britain related to climate and clonal fluctuation as revealed using microsatellites. Molecular Ecology 12: 21–34.
        doi: 10.1046/j.1365-294x.2003.01703.x <ul class="find" data-citedArticleID="7327507" data-doi="10.1046/j.1365-294x.2003.01703.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2003.01703.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Llewellyn%5Bauthor%5D+AND+Migration+and+genetic+structure+of+the+grain+aphid+%28Sitobion+avenae%29+in+Britain+related+to+climate+and+clonal+fluctuation+as+revealed+using+microsatellites" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALlewellyn+%22Migration+and+genetic+structure+of+the+grain+aphid+%28Sitobion+avenae%29+in+Britain+related+to+climate+and+clonal+fluctuation+as+revealed+using+microsatellites%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0059654-Xu1" id="pone.0059654-Xu1"></a>Xu ZH, Chen JL, Cheng DF, Liu Y, Francis F (2011) Genetic variation among the geographic population of the grain aphid, <em>Sitobion avenae</em> (Hemiptera: Aphididae) in China inferred from mitochondrial COI gene sequence. Agricultural Sciences in China 10: 1041–1048.
        doi: 10.1016/s1671-2927(11)60092-8 <ul class="find" data-citedArticleID="7327509" data-doi="10.1016/s1671-2927(11)60092-8"><li><a href="http://dx.doi.org/10.1016/s1671-2927(11)60092-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Xu%5Bauthor%5D+AND+Genetic+variation+among+the+geographic+population+of+the+grain+aphid%2C+Sitobion+avenae+%28Hemiptera%3A+Aphididae%29+in+China+inferred+from+mitochondrial+COI+gene+sequence" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AXu+%22Genetic+variation+among+the+geographic+population+of+the+grain+aphid%2C+Sitobion+avenae+%28Hemiptera%3A+Aphididae%29+in+China+inferred+from+mitochondrial+COI+gene+sequence%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0059654-Freeland1" id="pone.0059654-Freeland1"></a>Freeland JR, May M, Lodge R, Conrad KF (2003) Genetic diversity and widespread haplotypes in a migratory dragonfly, the common green darner <em>Anax junius</em>. Ecological Entomology 28: 413–421.
        doi: 10.1046/j.1365-2311.2003.00521.x <ul class="find" data-citedArticleID="7327511" data-doi="10.1046/j.1365-2311.2003.00521.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2311.2003.00521.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Freeland%5Bauthor%5D+AND+Genetic+diversity+and+widespread+haplotypes+in+a+migratory+dragonfly%2C+the+common+green+darner+Anax+junius" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFreeland+%22Genetic+diversity+and+widespread+haplotypes+in+a+migratory+dragonfly%2C+the+common+green+darner+Anax+junius%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0059654-Franklin1" id="pone.0059654-Franklin1"></a>Franklin MT, Ritland CE, Myers JH (2011) Genetic analysis of cabbage loopers, <em>Trichoplusia ni</em> (Lepidoptera: Noctuidae), a seasonal migrant in western North America. Evolutionary Applications 4: 89–99.
        doi: 10.1111/j.1752-4571.2010.00135.x <ul class="find" data-citedArticleID="7327513" data-doi="10.1111/j.1752-4571.2010.00135.x"><li><a href="http://dx.doi.org/10.1111/j.1752-4571.2010.00135.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Franklin%5Bauthor%5D+AND+Genetic+analysis+of+cabbage+loopers%2C+Trichoplusia+ni+%28Lepidoptera%3A+Noctuidae%29%2C+a+seasonal+migrant+in+western+North+America" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFranklin+%22Genetic+analysis+of+cabbage+loopers%2C+Trichoplusia+ni+%28Lepidoptera%3A+Noctuidae%29%2C+a+seasonal+migrant+in+western+North+America%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0059654-Talekar1" id="pone.0059654-Talekar1"></a>Talekar NS, Shelton AM (1993) Biology, ecology, and management of the diamondback moth. Annual Review of Entomology 38: 275–301.
        doi: 10.1146/annurev.en.38.010193.001423 <ul class="find" data-citedArticleID="7327515" data-doi="10.1146/annurev.en.38.010193.001423"><li><a href="http://dx.doi.org/10.1146/annurev.en.38.010193.001423" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Talekar%5Bauthor%5D+AND+Biology%2C+ecology%2C+and+management+of+the+diamondback+moth" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATalekar+%22Biology%2C+ecology%2C+and+management+of+the+diamondback+moth%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0059654-Furlong1" id="pone.0059654-Furlong1"></a>Furlong MJ, Wright DJ, Dosdall LM (2013) Diamondback Moth Ecology and Management: Problems, Progress and Prospects. Annual Review of Entomology 58: 517–541.
        doi: 10.1146/annurev-ento-120811-153605 <ul class="find" data-citedArticleID="7327517" data-doi="10.1146/annurev-ento-120811-153605"><li><a href="http://dx.doi.org/10.1146/annurev-ento-120811-153605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Furlong%5Bauthor%5D+AND+Diamondback+Moth+Ecology+and+Management%3A+Problems%2C+Progress+and+Prospects" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFurlong+%22Diamondback+Moth+Ecology+and+Management%3A+Problems%2C+Progress+and+Prospects%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0059654-Harcourt1" id="pone.0059654-Harcourt1"></a>Harcourt DG (1963) Biology of cabbage caterpillars in eastern Ontario. Proceedings of the Entomological Society of Ontario 93: 61–75. <ul class="find" data-citedArticleID="7327519"><li><a href="http://www.crossref.org/guestquery/?auth2=Harcourt&amp;atitle2=Biology+of+cabbage+caterpillars+in+eastern+Ontario&amp;auth=Harcourt&amp;atitle=Biology+of+cabbage+caterpillars+in+eastern+Ontario" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Harcourt%5Bauthor%5D+AND+Biology+of+cabbage+caterpillars+in+eastern+Ontario" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHarcourt+%22Biology+of+cabbage+caterpillars+in+eastern+Ontario%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0059654-Kfir1" id="pone.0059654-Kfir1"></a>Kfir R (1998) Origin of the diamondback moth (Lepidoptera : Plutellidae). Annals of the Entomological Society of America 91: 164–167. <ul class="find" data-citedArticleID="7327521"><li><a href="http://www.crossref.org/guestquery/?auth2=Kfir&amp;atitle2=Origin+of+the+diamondback+moth+%28Lepidoptera+%3A+Plutellidae%29&amp;auth=Kfir&amp;atitle=Origin+of+the+diamondback+moth+%28Lepidoptera+%3A+Plutellidae%29" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kfir%5Bauthor%5D+AND+Origin+of+the+diamondback+moth+%28Lepidoptera+%3A+Plutellidae%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKfir+%22Origin+of+the+diamondback+moth+%28Lepidoptera+%3A+Plutellidae%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0059654-Dosdall1" id="pone.0059654-Dosdall1"></a>Dosdall L, Mason P, Olfert O, Kaminski L, Keddie B (2001) The origins of infestations of diamondback moth, <em>Plutella xylostella</em> (L.), in canola in western Canada. In: Endersby NM, Ridland PM, editors. The Fourth International Workshop on the Management of Diamondback Moth and Other Cruciferous Pests, 26–29 November. Melbourne, Australia. 95–100. <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0059654-Honda1" id="pone.0059654-Honda1"></a>Honda K (1992) Hibernation and migration of diamondback moth in northern Japan. In: Taleker NS, editor. Proceedings of the Second International Workshop, 10–15 December. Asian Vegetable Research and Development Center, Tainan, Taiwan. 43–50. <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0059654-Gu1" id="pone.0059654-Gu1"></a>Gu H (2009) Cold tolerance and overwintering of the diamondback moth (Lepidoptera: Plutellidae) in Southeastern Australia. Environmental entomology 38: 524–529.
        doi: 10.1603/022.038.0303 <ul class="find" data-citedArticleID="7327527" data-doi="10.1603/022.038.0303"><li><a href="http://dx.doi.org/10.1603/022.038.0303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gu%5Bauthor%5D+AND+Cold+tolerance+and+overwintering+of+the+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29+in+Southeastern+Australia" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGu+%22Cold+tolerance+and+overwintering+of+the+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29+in+Southeastern+Australia%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0059654-Ma2" id="pone.0059654-Ma2"></a>Ma CS, Ma G, Yang HP (2010) Overw intering of the diamondback moth, <em>Plutella xylostella</em> in temperate countries. Acta Ecologica Sinica 30: 3628–3636. <ul class="find" data-citedArticleID="7327529"><li><a href="http://www.crossref.org/guestquery/?auth2=Ma&amp;atitle2=Overw+intering+of+the+diamondback+moth%2C+Plutella+xylostella+in+temperate+countries&amp;auth=Ma&amp;atitle=Overw+intering+of+the+diamondback+moth%2C+Plutella+xylostella+in+temperate+countries" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ma%5Bauthor%5D+AND+Overw+intering+of+the+diamondback+moth%2C+Plutella+xylostella+in+temperate+countries" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMa+%22Overw+intering+of+the+diamondback+moth%2C+Plutella+xylostella+in+temperate+countries%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0059654-HarperGray1" id="pone.0059654-HarperGray1"></a>Harper Gray R (1915) The prevention of egg-laying on turnips by the diamondback moth. Journal of the Board of Agriculture and Fisheries 22: 222–226. <ul class="find" data-citedArticleID="7327531"><li><a href="http://www.crossref.org/guestquery/?auth2=Harper+Gray&amp;atitle2=The+prevention+of+egg-laying+on+turnips+by+the+diamondback+moth&amp;auth=Harper+Gray&amp;atitle=The+prevention+of+egg-laying+on+turnips+by+the+diamondback+moth" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Harper+Gray%5Bauthor%5D+AND+The+prevention+of+egg-laying+on+turnips+by+the+diamondback+moth" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHarper+Gray+%22The+prevention+of+egg-laying+on+turnips+by+the+diamondback+moth%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0059654-Smith1" id="pone.0059654-Smith1"></a>Smith DB, Sears MK (1982) Evidence for dispersal of diamondback moth <em>Plutella xylostella</em> (Lepidoptera: Plutellidae) into southern Ontario. Proceedings of the Entomological Society of Ontario 113: 21–27.
        doi: 10.1007/springerreference_86090 <ul class="find" data-citedArticleID="7327533" data-doi="10.1007/springerreference_86090"><li><a href="http://dx.doi.org/10.1007/springerreference_86090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Smith%5Bauthor%5D+AND+Evidence+for+dispersal+of+diamondback+moth+Plutella+xylostella+%28Lepidoptera%3A+Plutellidae%29+into+southern+Ontario" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASmith+%22Evidence+for+dispersal+of+diamondback+moth+Plutella+xylostella+%28Lepidoptera%3A+Plutellidae%29+into+southern+Ontario%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0059654-French1" id="pone.0059654-French1"></a>French RA (1967) Long distance movement of two migrant Lepidoptera in relation to synoptic weather conditions. Biometeorology 2: 565–569. <ul class="find" data-citedArticleID="7327535"><li><a href="http://www.crossref.org/guestquery/?auth2=French&amp;atitle2=Long+distance+movement+of+two+migrant+Lepidoptera+in+relation+to+synoptic+weather+conditions&amp;auth=French&amp;atitle=Long+distance+movement+of+two+migrant+Lepidoptera+in+relation+to+synoptic+weather+conditions" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=French%5Bauthor%5D+AND+Long+distance+movement+of+two+migrant+Lepidoptera+in+relation+to+synoptic+weather+conditions" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFrench+%22Long+distance+movement+of+two+migrant+Lepidoptera+in+relation+to+synoptic+weather+conditions%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0059654-Saw1" id="pone.0059654-Saw1"></a>Saw J, Endersby NM, McKechnie SW (2006) Low mtDNA diversity among widespread Australian diamondback moth <em>Plutella xylostella</em> (L.) suggests isolation and a founder effect. Insect Science 13: 365–373.
        doi: 10.1111/j.1744-7917.2006.00105.x <ul class="find" data-citedArticleID="7327537" data-doi="10.1111/j.1744-7917.2006.00105.x"><li><a href="http://dx.doi.org/10.1111/j.1744-7917.2006.00105.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Saw%5Bauthor%5D+AND+Low+mtDNA+diversity+among+widespread+Australian+diamondback+moth+Plutella+xylostella+%28L.%29+suggests+isolation+and+a+founder+effect" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASaw+%22Low+mtDNA+diversity+among+widespread+Australian+diamondback+moth+Plutella+xylostella+%28L.%29+suggests+isolation+and+a+founder+effect%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0059654-Endersby2" id="pone.0059654-Endersby2"></a>Endersby NM, McKechnie SW, Ridland PM, Weeks AR (2006) Microsatellites reveal a lack of structure in Australian populations of the diamondback moth, <em>Plutella xylostella</em> (L.). Molecular Ecology 15: 107–118.
        doi: 10.1111/j.1365-294x.2005.02789.x <ul class="find" data-citedArticleID="7327539" data-doi="10.1111/j.1365-294x.2005.02789.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2005.02789.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endersby%5Bauthor%5D+AND+Microsatellites+reveal+a+lack+of+structure+in+Australian+populations+of+the+diamondback+moth%2C+Plutella+xylostella+%28L.%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AEndersby+%22Microsatellites+reveal+a+lack+of+structure+in+Australian+populations+of+the+diamondback+moth%2C+Plutella+xylostella+%28L.%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0059654-Li1" id="pone.0059654-Li1"></a>Li JH, Zhao F, Choi YS, Kim I, Sohn HD, et al. (2006) Genetic variation in the diamondback moth, <em>Plutella xylostella</em> (Lepidoptera: Yponomeutidae) in China inferred from mitochondrial COI gene sequence. European Journal of Entomology 103: 605–611.
        doi: 10.14411/eje.2006.081 <ul class="find" data-citedArticleID="7327541" data-doi="10.14411/eje.2006.081"><li><a href="http://dx.doi.org/10.14411/eje.2006.081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Li%5Bauthor%5D+AND+Genetic+variation+in+the+diamondback+moth%2C+Plutella+xylostella+%28Lepidoptera%3A+Yponomeutidae%29+in+China+inferred+from+mitochondrial+COI+gene+sequence" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALi+%22Genetic+variation+in+the+diamondback+moth%2C+Plutella+xylostella+%28Lepidoptera%3A+Yponomeutidae%29+in+China+inferred+from+mitochondrial+COI+gene+sequence%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0059654-Wang1" id="pone.0059654-Wang1"></a>Wang SL, Sheng CF, Qiao CL (2005) Genetic variability of <em>Plutella xylostella</em> (Lepidoptera: Plutellidae) among five field populations in China. In: Shanghai, China: Medimond S R Xiong XG, Yan ZZ, Liang WY, editors. Proceedings of the 13th International Congress on Genes, Gene Families and Isozymes. L: 29–33. <ul class="find" data-citedArticleID="7327543"><li><a href="http://www.crossref.org/guestquery/?auth2=Wang&amp;atitle2=Genetic+variability+of+Plutella+xylostella+%28Lepidoptera%3A+Plutellidae%29+among+five+field+populations+in+China&amp;auth=Wang&amp;atitle=Genetic+variability+of+Plutella+xylostella+%28Lepidoptera%3A+Plutellidae%29+among+five+field+populations+in+China" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wang%5Bauthor%5D+AND+Genetic+variability+of+Plutella+xylostella+%28Lepidoptera%3A+Plutellidae%29+among+five+field+populations+in+China" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWang+%22Genetic+variability+of+Plutella+xylostella+%28Lepidoptera%3A+Plutellidae%29+among+five+field+populations+in+China%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0059654-Chang1" id="pone.0059654-Chang1"></a>Chang W, Tabashnik B, Artelt B, Malvar T, Ballester V, et al. (1997) Mitochondrial DNA sequence variation among geographic strains of diamondback moth (Lepidoptera: Plutellidae). Annals of the Entomological Society of America 90: 590–595. <ul class="find" data-citedArticleID="7327545"><li><a href="http://www.crossref.org/guestquery/?auth2=Chang&amp;atitle2=Mitochondrial+DNA+sequence+variation+among+geographic+strains+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29&amp;auth=Chang&amp;atitle=Mitochondrial+DNA+sequence+variation+among+geographic+strains+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chang%5Bauthor%5D+AND+Mitochondrial+DNA+sequence+variation+among+geographic+strains+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AChang+%22Mitochondrial+DNA+sequence+variation+among+geographic+strains+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0059654-Kim1" id="pone.0059654-Kim1"></a>Kim Y, Kim K, Kim N (1999) Genetic difference between two field populations of <em>Plutella xylostella</em> ((Linné) based on four polymorphic allozymes. Journal of Asia-Pacific Entomology 2: 1–5.
        doi: 10.1016/s1226-8615(08)60024-2 <ul class="find" data-citedArticleID="7327547" data-doi="10.1016/s1226-8615(08)60024-2"><li><a href="http://dx.doi.org/10.1016/s1226-8615(08)60024-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kim%5Bauthor%5D+AND+Genetic+difference+between+two+field+populations+of+Plutella+xylostella+%28%28Linn%C3%A9%29+based+on+four+polymorphic+allozymes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKim+%22Genetic+difference+between+two+field+populations+of+Plutella+xylostella+%28%28Linn%C3%A9%29+based+on+four+polymorphic+allozymes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0059654-Laude1" id="pone.0059654-Laude1"></a>Laude R, Aseo S, Fajardo G, Dinulos R, Juliano A, et al. (1992) Genetic population studies in diamondback moth <em>Plutella xylostella</em> L.(Lepidoptera; Yponomeutidae) in crucifers. Philippine Entomologist 8: 1265–1280. <ul class="find" data-citedArticleID="7327549"><li><a href="http://www.crossref.org/guestquery/?auth2=Laude&amp;atitle2=Genetic+population+studies+in+diamondback+moth+Plutella+xylostella+L.%28Lepidoptera%3B+Yponomeutidae%29+in+crucifers&amp;auth=Laude&amp;atitle=Genetic+population+studies+in+diamondback+moth+Plutella+xylostella+L.%28Lepidoptera%3B+Yponomeutidae%29+in+crucifers" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Laude%5Bauthor%5D+AND+Genetic+population+studies+in+diamondback+moth+Plutella+xylostella+L.%28Lepidoptera%3B+Yponomeutidae%29+in+crucifers" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALaude+%22Genetic+population+studies+in+diamondback+moth+Plutella+xylostella+L.%28Lepidoptera%3B+Yponomeutidae%29+in+crucifers%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0059654-Kim2" id="pone.0059654-Kim2"></a>Kim I, Bae AS, Lee KS, Kim ES, Lee HS, et al. (2003) Mitochondrial COI gene sequence-based population genetic structure of the diamondback moth, <em>Plutella xylostella</em>, in Korea. Korean Journal of Genetics 25: 155–170. <ul class="find" data-citedArticleID="7327551"><li><a href="http://www.crossref.org/guestquery/?auth2=Kim&amp;atitle2=Mitochondrial+COI+gene+sequence-based+population+genetic+structure+of+the+diamondback+moth%2C+Plutella+xylostella%2C+in+Korea&amp;auth=Kim&amp;atitle=Mitochondrial+COI+gene+sequence-based+population+genetic+structure+of+the+diamondback+moth%2C+Plutella+xylostella%2C+in+Korea" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kim%5Bauthor%5D+AND+Mitochondrial+COI+gene+sequence-based+population+genetic+structure+of+the+diamondback+moth%2C+Plutella+xylostella%2C+in+Korea" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKim+%22Mitochondrial+COI+gene+sequence-based+population+genetic+structure+of+the+diamondback+moth%2C+Plutella+xylostella%2C+in+Korea%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0059654-Caprio1" id="pone.0059654-Caprio1"></a>Caprio MA, Tabashnik BE (1992) Allozymes used to estimate gene flow among populations of diamondback moth (Lepidoptera: Plutellidae) in Hawaii. Environmental Entomology 21: 808–816. <ul class="find" data-citedArticleID="7327553"><li><a href="http://www.crossref.org/guestquery/?auth2=Caprio&amp;atitle2=Allozymes+used+to+estimate+gene+flow+among+populations+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29+in+Hawaii&amp;auth=Caprio&amp;atitle=Allozymes+used+to+estimate+gene+flow+among+populations+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29+in+Hawaii" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Caprio%5Bauthor%5D+AND+Allozymes+used+to+estimate+gene+flow+among+populations+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29+in+Hawaii" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACaprio+%22Allozymes+used+to+estimate+gene+flow+among+populations+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29+in+Hawaii%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0059654-Ma3" id="pone.0059654-Ma3"></a>Ma CS, Chen RL (1991) Studies on the overwinter and migration of diamondbacn moth. In: Protection CSoP, Agriculture ECoCAo, editors. Proceedings of the First National Plant Protection Congress Attended by Middle-aged and Youth Scientists. Beijing: China: Scientech Press. 294–299. <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0059654-Chen1" id="pone.0059654-Chen1"></a>Chen RL, Bao XZ, Drake VA, Farrow RA, Wang SY, et al. (1989) Radar observations of the spring migration into northeastern China of the oriental armyworm moth, <em>Mythimna separata</em>, and other insects. Ecological Entomology 14: 149–162.
        doi: 10.1111/j.1365-2311.1989.tb00765.x <ul class="find" data-citedArticleID="7327557" data-doi="10.1111/j.1365-2311.1989.tb00765.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2311.1989.tb00765.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chen%5Bauthor%5D+AND+Radar+observations+of+the+spring+migration+into+northeastern+China+of+the+oriental+armyworm+moth%2C+Mythimna+separata%2C+and+other+insects" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AChen+%22Radar+observations+of+the+spring+migration+into+northeastern+China+of+the+oriental+armyworm+moth%2C+Mythimna+separata%2C+and+other+insects%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0059654-Feng1" id="pone.0059654-Feng1"></a>Feng HQ, Wu KM, Cheng DF, Guo YY (2003) Radar observations of the autumn migration of the beet armyworm <em>Spodoptera exigua</em> (Lepidoptera: Noctuidae) and other moths in northern China. Bulletin of Entomological Research 93: 115–124.
        doi: 10.1079/ber2002221 <ul class="find" data-citedArticleID="7327559" data-doi="10.1079/ber2002221"><li><a href="http://dx.doi.org/10.1079/ber2002221" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feng%5Bauthor%5D+AND+Radar+observations+of+the+autumn+migration+of+the+beet+armyworm+Spodoptera+exigua+%28Lepidoptera%3A+Noctuidae%29+and+other+moths+in+northern+China" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFeng+%22Radar+observations+of+the+autumn+migration+of+the+beet+armyworm+Spodoptera+exigua+%28Lepidoptera%3A+Noctuidae%29+and+other+moths+in+northern+China%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0059654-Feng2" id="pone.0059654-Feng2"></a>Feng H, Wu K, Cheng D, Guo Y (2004) Spring migration and summer dispersal of <em>Loxostege sticticalis</em> (Lepidoptera: Pyralidae) and other insects observed with radar in northern China. Environmental Entomology 33: 1253–1265.
        doi: 10.1603/0046-225x-33.5.1253 <ul class="find" data-citedArticleID="7327561" data-doi="10.1603/0046-225x-33.5.1253"><li><a href="http://dx.doi.org/10.1603/0046-225x-33.5.1253" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feng%5Bauthor%5D+AND+Spring+migration+and+summer+dispersal+of+Loxostege+sticticalis+%28Lepidoptera%3A+Pyralidae%29+and+other+insects+observed+with+radar+in+northern+China" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFeng+%22Spring+migration+and+summer+dispersal+of+Loxostege+sticticalis+%28Lepidoptera%3A+Pyralidae%29+and+other+insects+observed+with+radar+in+northern+China%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0059654-Feng3" id="pone.0059654-Feng3"></a>Feng HQ, Wu KM, Cheng DF, Guo YY (2004) Northward migration of <em>Helicoverpa armigera</em> (Lepidoptera: Noctuidae) and other moths in early summer observed with radar in northern China. Journal of Economic Entomology 97: 1874–1883.
        doi: 10.1603/0022-0493-97.6.1874 <ul class="find" data-citedArticleID="7327563" data-doi="10.1603/0022-0493-97.6.1874"><li><a href="http://dx.doi.org/10.1603/0022-0493-97.6.1874" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feng%5Bauthor%5D+AND+Northward+migration+of+Helicoverpa+armigera+%28Lepidoptera%3A+Noctuidae%29+and+other+moths+in+early+summer+observed+with+radar+in+northern+China" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFeng+%22Northward+migration+of+Helicoverpa+armigera+%28Lepidoptera%3A+Noctuidae%29+and+other+moths+in+early+summer+observed+with+radar+in+northern+China%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0059654-Riley1" id="pone.0059654-Riley1"></a>Riley JR, Cheng XN, Xiao-Xi Z, Reynolds DR, Guo-Min X, et al. (1991) The long-distance migration of <em>Nilaparvata lugens</em> (Stål)(Delphacidae) in China: radar observations of mass return flight in the autumn. Ecological Entomology 16: 471–489.
        doi: 10.1111/j.1365-2311.1991.tb00240.x <ul class="find" data-citedArticleID="7327565" data-doi="10.1111/j.1365-2311.1991.tb00240.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2311.1991.tb00240.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Riley%5Bauthor%5D+AND+The+long-distance+migration+of+Nilaparvata+lugens+%28St%C3%A5l%29%28Delphacidae%29+in+China%3A+radar+observations+of+mass+return+flight+in+the+autumn" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARiley+%22The+long-distance+migration+of+Nilaparvata+lugens+%28St%C3%A5l%29%28Delphacidae%29+in+China%3A+radar+observations+of+mass+return+flight+in+the+autumn%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0059654-Meng1" id="pone.0059654-Meng1"></a>Meng XF, Shi M, Chen XX (2008) Population genetic structure of <em>Chilo suppressalis</em> (Walker) (Lepidoptera: Crambidae): strong subdivision in China inferred from microsatellite markers and mtDNA gene sequences. Mol Ecol 17: 2880–2897.
        doi: 10.1111/j.1365-294x.2008.03792.x <ul class="find" data-citedArticleID="7327567" data-doi="10.1111/j.1365-294x.2008.03792.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2008.03792.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Meng%5Bauthor%5D+AND+Population+genetic+structure+of+Chilo+suppressalis+%28Walker%29+%28Lepidoptera%3A+Crambidae%29%3A+strong+subdivision+in+China+inferred+from+microsatellite+markers+and+mtDNA+gene+sequences" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMeng+%22Population+genetic+structure+of+Chilo+suppressalis+%28Walker%29+%28Lepidoptera%3A+Crambidae%29%3A+strong+subdivision+in+China+inferred+from+microsatellite+markers+and+mtDNA+gene+sequences%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0059654-Uthicke1" id="pone.0059654-Uthicke1"></a>Uthicke S, Benzie J (2003) Gene flow and population history in high dispersal marine invertebrates: mitochondrial DNA analysis of <em>Holothuria nobilis</em> (Echinodermata: Holothuroidea) populations from the Indo-Pacific. Molecular Ecology 12: 2635–2648.
        doi: 10.1046/j.1365-294x.2003.01954.x <ul class="find" data-citedArticleID="7327569" data-doi="10.1046/j.1365-294x.2003.01954.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2003.01954.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uthicke%5Bauthor%5D+AND+Gene+flow+and+population+history+in+high+dispersal+marine+invertebrates%3A+mitochondrial+DNA+analysis+of+Holothuria+nobilis+%28Echinodermata%3A+Holothuroidea%29+populations+from+the+Indo-Pacific" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AUthicke+%22Gene+flow+and+population+history+in+high+dispersal+marine+invertebrates%3A+mitochondrial+DNA+analysis+of+Holothuria+nobilis+%28Echinodermata%3A+Holothuroidea%29+populations+from+the+Indo-Pacific%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0059654-Kraus1" id="pone.0059654-Kraus1"></a>Kraus RHS, Zeddeman A, van Hooft P, Sartakov D, Soloviev SA, et al. (2011) Evolution and connectivity in the world-wide migration system of the mallard: Inferences from mitochondrial DNA. BMC Genetics 12: 99.
        doi: 10.1186/1471-2156-12-99 <ul class="find" data-citedArticleID="7327571" data-doi="10.1186/1471-2156-12-99"><li><a href="http://dx.doi.org/10.1186/1471-2156-12-99" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kraus%5Bauthor%5D+AND+Evolution+and+connectivity+in+the+world-wide+migration+system+of+the+mallard%3A+Inferences+from+mitochondrial+DNA" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKraus+%22Evolution+and+connectivity+in+the+world-wide+migration+system+of+the+mallard%3A+Inferences+from+mitochondrial+DNA%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0059654-Hardy1" id="pone.0059654-Hardy1"></a>Hardy JE (1938) <em>Plutella maculipennis</em>, Curt., its natural and biological control in England. Bulletin of Entomological Research 29: 343–372.
        doi: 10.1017/s0007485300026274 <ul class="find" data-citedArticleID="7327573" data-doi="10.1017/s0007485300026274"><li><a href="http://dx.doi.org/10.1017/s0007485300026274" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hardy%5Bauthor%5D+AND+Plutella+maculipennis%2C+Curt.%2C+its+natural+and+biological+control+in+England" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHardy+%22Plutella+maculipennis%2C+Curt.%2C+its+natural+and+biological+control+in+England%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0059654-Capinera1" id="pone.0059654-Capinera1"></a>Capinera JL (2000) Diamondback moth, <em>Plutella xylostella</em> (Linnaeus) (Insecta: Lepidoptera: Plutellidae). Featured Creatures from the Entomology and Nematology Department, Florida Cooperative Extension Service, Institute of Food and Agricultural Sciences, University of Florida EENY-119: 1–4. <ul class="find" data-citedArticleID="7327575"><li><a href="http://www.crossref.org/guestquery/?auth2=Capinera&amp;atitle2=Diamondback+moth%2C+Plutella+xylostella+%28Linnaeus%29+%28Insecta%3A+Lepidoptera%3A+Plutellidae%29&amp;auth=Capinera&amp;atitle=Diamondback+moth%2C+Plutella+xylostella+%28Linnaeus%29+%28Insecta%3A+Lepidoptera%3A+Plutellidae%29" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Capinera%5Bauthor%5D+AND+Diamondback+moth%2C+Plutella+xylostella+%28Linnaeus%29+%28Insecta%3A+Lepidoptera%3A+Plutellidae%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACapinera+%22Diamondback+moth%2C+Plutella+xylostella+%28Linnaeus%29+%28Insecta%3A+Lepidoptera%3A+Plutellidae%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0059654-Hori1" id="pone.0059654-Hori1"></a>Hori K, Shiraki T (1910) Investigation of pest insect in Taiwan. Special Report Formosa Agricultural Experiment Station: 228. <ul class="find-nolinks"></ul></li><li><span class="label">51.
              </span><a name="pone.0059654-Hofinger1" id="pone.0059654-Hofinger1"></a>Hofinger BJ, Russell JR, Bass CG, Baldwin T, Dos Reis M, et al. (2011) An exceptionally high nucleotide and haplotype diversity and a signature of positive selection for the eIF4E resistance gene in barley are revealed by allele mining and phylogenetic analyses of natural populations. Molecular Ecology 20: 3653–3688.
        doi: 10.1111/j.1365-294x.2011.05201.x <ul class="find" data-citedArticleID="7327579" data-doi="10.1111/j.1365-294x.2011.05201.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2011.05201.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hofinger%5Bauthor%5D+AND+An+exceptionally+high+nucleotide+and+haplotype+diversity+and+a+signature+of+positive+selection+for+the+eIF4E+resistance+gene+in+barley+are+revealed+by+allele+mining+and+phylogenetic+analyses+of+natural+populations" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHofinger+%22An+exceptionally+high+nucleotide+and+haplotype+diversity+and+a+signature+of+positive+selection+for+the+eIF4E+resistance+gene+in+barley+are+revealed+by+allele+mining+and+phylogenetic+analyses+of+natural+populations%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0059654-Zhao1" id="pone.0059654-Zhao1"></a>Zhao JZ, Collins HL, Li YX, Mau RFL, Thompson GD, et al. (2006) Monitoring of diamondback moth (Lepidoptera : Plutellidae) resistance to spinosad, indoxacarb, and emamectin benzoate. Journal of Economic Entomology 99: 176–181.
        doi: 10.1603/0022-0493(2006)099[0176:modmlp]2.0.co;2 <ul class="find" data-citedArticleID="7327581" data-doi="10.1603/0022-0493(2006)099[0176:modmlp]2.0.co;2"><li><a href="http://dx.doi.org/10.1603/0022-0493(2006)099[0176:modmlp]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zhao%5Bauthor%5D+AND+Monitoring+of+diamondback+moth+%28Lepidoptera+%3A+Plutellidae%29+resistance+to+spinosad%2C+indoxacarb%2C+and+emamectin+benzoate" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AZhao+%22Monitoring+of+diamondback+moth+%28Lepidoptera+%3A+Plutellidae%29+resistance+to+spinosad%2C+indoxacarb%2C+and+emamectin+benzoate%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0059654-Herrero1" id="pone.0059654-Herrero1"></a>Herrero S, Ferré J, Escriche B (2001) Mannose phosphate isomerase isoenzymes in <em>Plutella xylostella</em> support common genetic bases of resistance to <em>Bacillus thuringiensis</em> toxins in lepidopteran species. Applied and Environmental Microbiology 67: 979.
        doi: 10.1128/aem.67.2.979-981.2001 <ul class="find" data-citedArticleID="7327583" data-doi="10.1128/aem.67.2.979-981.2001"><li><a href="http://dx.doi.org/10.1128/aem.67.2.979-981.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Herrero%5Bauthor%5D+AND+Mannose+phosphate+isomerase+isoenzymes+in+Plutella+xylostella+support+common+genetic+bases+of+resistance+to+Bacillus+thuringiensis+toxins+in+lepidopteran+species" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHerrero+%22Mannose+phosphate+isomerase+isoenzymes+in+Plutella+xylostella+support+common+genetic+bases+of+resistance+to+Bacillus+thuringiensis+toxins+in+lepidopteran+species%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0059654-Zhou1" id="pone.0059654-Zhou1"></a>Zhou XM, Wu QJ, Zhang YJ, Bai LY, Huang XY (2010) Effects of abamectin selection on the genetic differentiation within <em>Plutella xylostella</em> (Lepidoptera: Plutellidae) based on amplified fragment length polymorphism. Insect Science 17: 353–360.
        doi: 10.1111/j.1744-7917.2009.01307.x <ul class="find" data-citedArticleID="7327585" data-doi="10.1111/j.1744-7917.2009.01307.x"><li><a href="http://dx.doi.org/10.1111/j.1744-7917.2009.01307.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zhou%5Bauthor%5D+AND+Effects+of+abamectin+selection+on+the+genetic+differentiation+within+Plutella+xylostella+%28Lepidoptera%3A+Plutellidae%29+based+on+amplified+fragment+length+polymorphism" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AZhou+%22Effects+of+abamectin+selection+on+the+genetic+differentiation+within+Plutella+xylostella+%28Lepidoptera%3A+Plutellidae%29+based+on+amplified+fragment+length+polymorphism%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0059654-Heckel1" id="pone.0059654-Heckel1"></a>Heckel D, Gahan L, Tabashnik B, Johnson M (1995) Randomly amplified polymorphic DNA differences between strains of diamondback moth (Lepidoptera: Plutellidae) susceptible or resistant to Bacillus thuringiensis. Annals of the Entomological Society of America 88: 531–537. <ul class="find" data-citedArticleID="7327587"><li><a href="http://www.crossref.org/guestquery/?auth2=Heckel&amp;atitle2=Randomly+amplified+polymorphic+DNA+differences+between+strains+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29+susceptible+or+resistant+to+Bacillus+thuringiensis&amp;auth=Heckel&amp;atitle=Randomly+amplified+polymorphic+DNA+differences+between+strains+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29+susceptible+or+resistant+to+Bacillus+thuringiensis" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heckel%5Bauthor%5D+AND+Randomly+amplified+polymorphic+DNA+differences+between+strains+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29+susceptible+or+resistant+to+Bacillus+thuringiensis" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHeckel+%22Randomly+amplified+polymorphic+DNA+differences+between+strains+of+diamondback+moth+%28Lepidoptera%3A+Plutellidae%29+susceptible+or+resistant+to+Bacillus+thuringiensis%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0059654-Noran1" id="pone.0059654-Noran1"></a>Noran A, Tang P (1996) Allozymic polymorphism among three populations of <em>Plutella xylostella</em>. In: Sivapragasam A, Loke W, Hussan A, Lim G, editors. Proceedings of the Third International Workshop. Malaysian Agrcultural Research and Development Institute, Kuala Lumpur, Malaysia. 3. <ul class="find-nolinks"></ul></li><li><span class="label">57.
              </span><a name="pone.0059654-Marsh1" id="pone.0059654-Marsh1"></a>Marsh HO (1917) Life history of <em>Plutella maculipennis</em>: the diamond-back moth. Journal of Agricultural Research 10: 1–10. <ul class="find" data-citedArticleID="7327591"><li><a href="http://www.crossref.org/guestquery/?auth2=Marsh&amp;atitle2=Life+history+of+Plutella+maculipennis%3A+the+diamond-back+moth&amp;auth=Marsh&amp;atitle=Life+history+of+Plutella+maculipennis%3A+the+diamond-back+moth" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Marsh%5Bauthor%5D+AND+Life+history+of+Plutella+maculipennis%3A+the+diamond-back+moth" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMarsh+%22Life+history+of+Plutella+maculipennis%3A+the+diamond-back+moth%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0059654-Theobald1" id="pone.0059654-Theobald1"></a>Theobald F (1926) The diamond back moth (<em>Plutella maculipennis</em>). J Kent Farmers Union 20: 1–7. <ul class="find" data-citedArticleID="7327593"><li><a href="http://www.crossref.org/guestquery/?auth2=Theobald&amp;atitle2=The+diamond+back+moth+%28Plutella+maculipennis%29&amp;auth=Theobald&amp;atitle=The+diamond+back+moth+%28Plutella+maculipennis%29" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Theobald%5Bauthor%5D+AND+The+diamond+back+moth+%28Plutella+maculipennis%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATheobald+%22The+diamond+back+moth+%28Plutella+maculipennis%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0059654-Zalucki1" id="pone.0059654-Zalucki1"></a>Zalucki M, Furlong M (2008) Predicting outbreaks of a migratory pest: An analysis of DBM distribution and abundance. China Agricultural Science and Technology Press. 122–131. <ul class="find-nolinks"></ul></li><li><span class="label">60.
              </span><a name="pone.0059654-Saito1" id="pone.0059654-Saito1"></a>Saito O, Mizushima S, Okuyama S, Hanada T, Torikura H, et al.. (1998) Biology of the diamondback moth, <em>Plutella xylostella</em> (L.), in Hokkaido. Research Bulletin of the Hokkaido National Agricultural Experiment Station 167. <ul class="find-nolinks"></ul></li><li><span class="label">61.
              </span><a name="pone.0059654-Chen2" id="pone.0059654-Chen2"></a>Chen FZ, Liu SS (2004) Effects of low and subzero temperature on a Plutella xylostella laboratory population. Chinese Journal of Applied Ecology 15: 99–102. <ul class="find" data-citedArticleID="7327599"><li><a href="http://www.crossref.org/guestquery/?auth2=Chen&amp;atitle2=Effects+of+low+and+subzero+temperature+on+a+Plutella+xylostella+laboratory+population&amp;auth=Chen&amp;atitle=Effects+of+low+and+subzero+temperature+on+a+Plutella+xylostella+laboratory+population" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chen%5Bauthor%5D+AND+Effects+of+low+and+subzero+temperature+on+a+Plutella+xylostella+laboratory+population" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AChen+%22Effects+of+low+and+subzero+temperature+on+a+Plutella+xylostella+laboratory+population%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0059654-Dallimer1" id="pone.0059654-Dallimer1"></a>Dallimer M, Jones P, Pemberton J, Cheke R (2003) Lack of genetic and plumage differentiation in the red-billed quelea <em>Quelea quelea</em> across a migratory divide in southern Africa. Molecular Ecology 12: 345–353.
        doi: 10.1046/j.1365-294x.2003.01733.x <ul class="find" data-citedArticleID="7327601" data-doi="10.1046/j.1365-294x.2003.01733.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2003.01733.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dallimer%5Bauthor%5D+AND+Lack+of+genetic+and+plumage+differentiation+in+the+red-billed+quelea+Quelea+quelea+across+a+migratory+divide+in+southern+Africa" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADallimer+%22Lack+of+genetic+and+plumage+differentiation+in+the+red-billed+quelea+Quelea+quelea+across+a+migratory+divide+in+southern+Africa%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0059654-Feng4" id="pone.0059654-Feng4"></a>Feng HQ, Wu KM, Ni YX, Cheng DF, Guo YY (2005) Return migration of <em>Helicoverpa armigera</em> (Lepidoptera: Noctuidae) during autumn in northern China. Bulletin of Entomological Research 95: 361–370.
        doi: 10.1079/ber2005367 <ul class="find" data-citedArticleID="7327603" data-doi="10.1079/ber2005367"><li><a href="http://dx.doi.org/10.1079/ber2005367" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feng%5Bauthor%5D+AND+Return+migration+of+Helicoverpa+armigera+%28Lepidoptera%3A+Noctuidae%29+during+autumn+in+northern+China" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFeng+%22Return+migration+of+Helicoverpa+armigera+%28Lepidoptera%3A+Noctuidae%29+during+autumn+in+northern+China%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0059654-Schneider1" id="pone.0059654-Schneider1"></a>Schneider F (1962) Dispersal and migration. Annual Review of Entomology 7: 223–242.
        doi: 10.1146/annurev.en.07.010162.001255 <ul class="find" data-citedArticleID="7327605" data-doi="10.1146/annurev.en.07.010162.001255"><li><a href="http://dx.doi.org/10.1146/annurev.en.07.010162.001255" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Schneider%5Bauthor%5D+AND+Dispersal+and+migration" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASchneider+%22Dispersal+and+migration%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0059654-Card1" id="pone.0059654-Card1"></a>Cardé RT (2008) Animal migration: seasonal reversals of migrant moths. Current Biology 18: R1007–R1009.
        doi: 10.1016/j.cub.2008.09.013 <ul class="find" data-citedArticleID="7327607" data-doi="10.1016/j.cub.2008.09.013"><li><a href="http://dx.doi.org/10.1016/j.cub.2008.09.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Card%C3%A9%5Bauthor%5D+AND+Animal+migration%3A+seasonal+reversals+of+migrant+moths" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACard%C3%A9+%22Animal+migration%3A+seasonal+reversals+of+migrant+moths%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0059654-Stinner1" id="pone.0059654-Stinner1"></a>Stinner R, Barfield C, Stimac J, Dohse L (1983) Dispersal and movement of insect pests. Annual Review of Entomology 28: 319–335.
        doi: 10.1146/annurev.en.28.010183.001535 <ul class="find" data-citedArticleID="7327609" data-doi="10.1146/annurev.en.28.010183.001535"><li><a href="http://dx.doi.org/10.1146/annurev.en.28.010183.001535" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stinner%5Bauthor%5D+AND+Dispersal+and+movement+of+insect+pests" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AStinner+%22Dispersal+and+movement+of+insect+pests%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0059654-Ma4" id="pone.0059654-Ma4"></a>Ma Q, Zhang YF, Xie YZ (2009) The occurence and control stratergy of the diamondback moth <em>Plutella xylostella</em> in the Cruciferous vegetables. Journal of Zhejiang Agricultural Sciences: 956–959. <ul class="find-nolinks"></ul></li><li><span class="label">68.
              </span><a name="pone.0059654-Ma5" id="pone.0059654-Ma5"></a>Ma CS, Chen RL (1995) Population dynamics and generation differentiating of the diamondback moth, <em>Plutella xylostella</em> L. in Gong zhuling. Journal of Jilin Agriculture University 17: 27–31. <ul class="find" data-citedArticleID="7327613"><li><a href="http://www.crossref.org/guestquery/?auth2=Ma&amp;atitle2=Population+dynamics+and+generation+differentiating+of+the+diamondback+moth%2C+Plutella+xylostella+L.+in+Gong+zhuling&amp;auth=Ma&amp;atitle=Population+dynamics+and+generation+differentiating+of+the+diamondback+moth%2C+Plutella+xylostella+L.+in+Gong+zhuling" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ma%5Bauthor%5D+AND+Population+dynamics+and+generation+differentiating+of+the+diamondback+moth%2C+Plutella+xylostella+L.+in+Gong+zhuling" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMa+%22Population+dynamics+and+generation+differentiating+of+the+diamondback+moth%2C+Plutella+xylostella+L.+in+Gong+zhuling%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0059654-Gatehouse1" id="pone.0059654-Gatehouse1"></a>Gatehouse A (1997) Behavior and ecological genetics of wind-borne migration by insects. Annual Review of Entomology 42: 475–502.
        doi: 10.1146/annurev.ento.42.1.475 <ul class="find" data-citedArticleID="7327615" data-doi="10.1146/annurev.ento.42.1.475"><li><a href="http://dx.doi.org/10.1146/annurev.ento.42.1.475" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gatehouse%5Bauthor%5D+AND+Behavior+and+ecological+genetics+of+wind-borne+migration+by+insects" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGatehouse+%22Behavior+and+ecological+genetics+of+wind-borne+migration+by+insects%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0059654-Lokki1" id="pone.0059654-Lokki1"></a>Lokki J, Malmström K, Suomalainen E (1978) Migration of <em>Vanessa cardui</em> and <em>Plutella xylostella</em> (Lepidoptera) to Spitsbergen in the summer 1978. Notulae Entomologicae 58: 121–123. <ul class="find" data-citedArticleID="7327617"><li><a href="http://www.crossref.org/guestquery/?auth2=Lokki&amp;atitle2=Migration+of+Vanessa+cardui+and+Plutella+xylostella+%28Lepidoptera%29+to+Spitsbergen+in+the+summer+1978&amp;auth=Lokki&amp;atitle=Migration+of+Vanessa+cardui+and+Plutella+xylostella+%28Lepidoptera%29+to+Spitsbergen+in+the+summer+1978" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lokki%5Bauthor%5D+AND+Migration+of+Vanessa+cardui+and+Plutella+xylostella+%28Lepidoptera%29+to+Spitsbergen+in+the+summer+1978" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALokki+%22Migration+of+Vanessa+cardui+and+Plutella+xylostella+%28Lepidoptera%29+to+Spitsbergen+in+the+summer+1978%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0059654-Ma6" id="pone.0059654-Ma6"></a>Ma CS, Chen RL (1993) The effect of tempetature on the development and reproduction of the dimondback moth <em>Plutella xylostella</em> L. Journal of Jilin Agriculture University: 44–49. <ul class="find-nolinks"></ul></li><li><span class="label">72.
              </span><a name="pone.0059654-Mun1" id="pone.0059654-Mun1"></a>Mun JH, Song YH, Heong KL, Roderick GK (1999) Genetic variation among Asian populations of rice planthoppers, Nilaparvata lugens and Sogatella furcifera (Hemiptera : Delphacidae): mitochondrial DNA sequences. Bulletin of Entomological Research 89: 245–253.
        doi: 10.1017/s000748539900036x <ul class="find" data-citedArticleID="7327621" data-doi="10.1017/s000748539900036x"><li><a href="http://dx.doi.org/10.1017/s000748539900036x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mun%5Bauthor%5D+AND+Genetic+variation+among+Asian+populations+of+rice+planthoppers%2C+Nilaparvata+lugens+and+Sogatella+furcifera+%28Hemiptera+%3A+Delphacidae%29%3A+mitochondrial+DNA+sequences" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMun+%22Genetic+variation+among+Asian+populations+of+rice+planthoppers%2C+Nilaparvata+lugens+and+Sogatella+furcifera+%28Hemiptera+%3A+Delphacidae%29%3A+mitochondrial+DNA+sequences%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0059654-Grzywacz1" id="pone.0059654-Grzywacz1"></a>Grzywacz D, Rossbach A, Rauf A, Russell DA, Srinivasan R, et al. (2010) Current control methods for diamondback moth and other brassica insect pests and the prospects for improved management with lepidopteran-resistant Bt vegetable brassicas in Asia and Africa. Crop Protection 29: 68–79.
        doi: 10.1016/j.cropro.2009.08.009 <ul class="find" data-citedArticleID="7327623" data-doi="10.1016/j.cropro.2009.08.009"><li><a href="http://dx.doi.org/10.1016/j.cropro.2009.08.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Grzywacz%5Bauthor%5D+AND+Current+control+methods+for+diamondback+moth+and+other+brassica+insect+pests+and+the+prospects+for+improved+management+with+lepidopteran-resistant+Bt+vegetable+brassicas+in+Asia+and+Africa" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGrzywacz+%22Current+control+methods+for+diamondback+moth+and+other+brassica+insect+pests+and+the+prospects+for+improved+management+with+lepidopteran-resistant+Bt+vegetable+brassicas+in+Asia+and+Africa%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0059654-Feng5" id="pone.0059654-Feng5"></a>Feng X, Li ZY, Wu QJ, Chen AD, Wu YD, et al. (2011) Research progress of the resistance management and sustainable control of diamondback moth (<em>Plutella xylostella</em>) in China. Chinese Journal of Applied Entomology 48: 247–253. <ul class="find" data-citedArticleID="7327625"><li><a href="http://www.crossref.org/guestquery/?auth2=Feng&amp;atitle2=Research+progress+of+the+resistance+management+and+sustainable+control+of+diamondback+moth+%28Plutella+xylostella%29+in+China&amp;auth=Feng&amp;atitle=Research+progress+of+the+resistance+management+and+sustainable+control+of+diamondback+moth+%28Plutella+xylostella%29+in+China" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feng%5Bauthor%5D+AND+Research+progress+of+the+resistance+management+and+sustainable+control+of+diamondback+moth+%28Plutella+xylostella%29+in+China" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFeng+%22Research+progress+of+the+resistance+management+and+sustainable+control+of+diamondback+moth+%28Plutella+xylostella%29+in+China%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0059654-Singh1" id="pone.0059654-Singh1"></a>Singh VK, Mangalam A, Dwivedi S, Naik S (1998) Primer premier: program for design of degenerate primers from a protein sequence. Biotechniques 24: 318–319. <ul class="find" data-citedArticleID="7327627"><li><a href="http://www.crossref.org/guestquery/?auth2=Singh&amp;atitle2=Primer+premier%3A+program+for+design+of+degenerate+primers+from+a+protein+sequence&amp;auth=Singh&amp;atitle=Primer+premier%3A+program+for+design+of+degenerate+primers+from+a+protein+sequence" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Singh%5Bauthor%5D+AND+Primer+premier%3A+program+for+design+of+degenerate+primers+from+a+protein+sequence" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASingh+%22Primer+premier%3A+program+for+design+of+degenerate+primers+from+a+protein+sequence%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0059654-Esselink1" id="pone.0059654-Esselink1"></a>Esselink GD, Den Belder E, Elderson J, Smulders MJM (2006) Isolation and characterization of trinucleotide repeat microsatellite markers for Plutella xylostella L. Molecular Ecology Notes. 6: 1246–1248.
        doi: 10.1111/j.1471-8286.2006.01504.x <ul class="find" data-citedArticleID="7327629" data-doi="10.1111/j.1471-8286.2006.01504.x"><li><a href="http://dx.doi.org/10.1111/j.1471-8286.2006.01504.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Esselink%5Bauthor%5D+AND+Isolation+and+characterization+of+trinucleotide+repeat+microsatellite+markers+for+Plutella+xylostella+L.+Molecular+Ecology+Notes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AEsselink+%22Isolation+and+characterization+of+trinucleotide+repeat+microsatellite+markers+for+Plutella+xylostella+L.+Molecular+Ecology+Notes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0059654-Schuelke1" id="pone.0059654-Schuelke1"></a>Schuelke M (2000) An economic method for the fluorescent labeling of PCR fragments. Nature Biotechnology 18: 233–234.
        doi: 10.1038/72708 <ul class="find" data-citedArticleID="7327631" data-doi="10.1038/72708"><li><a href="http://dx.doi.org/10.1038/72708" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Schuelke%5Bauthor%5D+AND+An+economic+method+for+the+fluorescent+labeling+of+PCR+fragments" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASchuelke+%22An+economic+method+for+the+fluorescent+labeling+of+PCR+fragments%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0059654-Thompson1" id="pone.0059654-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Research 22: 4673–4680.
        doi: 10.1093/nar/22.22.4673 <ul class="find" data-citedArticleID="7327633" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thompson%5Bauthor%5D+AND+CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AThompson+%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0059654-Tamura1" id="pone.0059654-Tamura1"></a>Tamura K, Peterson D, Peterson N, Stecher G, Nei M, et al. (2011) MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods. Molecular biology and evolution 28: 2731–2739.
        doi: 10.1093/molbev/msr121 <ul class="find" data-citedArticleID="7327635" data-doi="10.1093/molbev/msr121"><li><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tamura%5Bauthor%5D+AND+MEGA5%3A+molecular+evolutionary+genetics+analysis+using+maximum+likelihood%2C+evolutionary+distance%2C+and+maximum+parsimony+methods" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATamura+%22MEGA5%3A+molecular+evolutionary+genetics+analysis+using+maximum+likelihood%2C+evolutionary+distance%2C+and+maximum+parsimony+methods%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0059654-Excoffier1" id="pone.0059654-Excoffier1"></a>Excoffier L, Lischer HEL (2010) Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows. Molecular Ecology Resources 10: 564–567.
        doi: 10.1111/j.1755-0998.2010.02847.x <ul class="find" data-citedArticleID="7327637" data-doi="10.1111/j.1755-0998.2010.02847.x"><li><a href="http://dx.doi.org/10.1111/j.1755-0998.2010.02847.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Excoffier%5Bauthor%5D+AND+Arlequin+suite+ver+3.5%3A+a+new+series+of+programs+to+perform+population+genetics+analyses+under+Linux+and+Windows" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AExcoffier+%22Arlequin+suite+ver+3.5%3A+a+new+series+of+programs+to+perform+population+genetics+analyses+under+Linux+and+Windows%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0059654-vanOosterhout1" id="pone.0059654-vanOosterhout1"></a>van Oosterhout C, Hutchinson WF, WILLS DPM, Shipley P (2004) Micro-checker: software for identifying and correcting genotyping errors in microsatellite data. Molecular Ecology Notes 4: 535–538.
        doi: 10.1111/j.1471-8286.2004.00684.x <ul class="find" data-citedArticleID="7327639" data-doi="10.1111/j.1471-8286.2004.00684.x"><li><a href="http://dx.doi.org/10.1111/j.1471-8286.2004.00684.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=van+Oosterhout%5Bauthor%5D+AND+Micro-checker%3A+software+for+identifying+and+correcting+genotyping+errors+in+microsatellite+data" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3Avan+Oosterhout+%22Micro-checker%3A+software+for+identifying+and+correcting+genotyping+errors+in+microsatellite+data%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0059654-Dieringer1" id="pone.0059654-Dieringer1"></a>Dieringer D, Schlötterer C (2003) Microsatellite analyser (MSA): a platform independent analysis tool for large microsatellite data sets. Molecular Ecology Notes 3: 167–169.
        doi: 10.1046/j.1471-8286.2003.00351.x <ul class="find" data-citedArticleID="7327641" data-doi="10.1046/j.1471-8286.2003.00351.x"><li><a href="http://dx.doi.org/10.1046/j.1471-8286.2003.00351.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dieringer%5Bauthor%5D+AND+Microsatellite+analyser+%28MSA%29%3A+a+platform+independent+analysis+tool+for+large+microsatellite+data+sets" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADieringer+%22Microsatellite+analyser+%28MSA%29%3A+a+platform+independent+analysis+tool+for+large+microsatellite+data+sets%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0059654-Raymond1" id="pone.0059654-Raymond1"></a>Raymond M, Rousset F (1995) GENEPOP (version 1.2): population genetics software for exact tests and ecumenicism. Journal of Heredity 86: 248–249. <ul class="find" data-citedArticleID="7327643"><li><a href="http://www.crossref.org/guestquery/?auth2=Raymond&amp;atitle2=GENEPOP+%28version+1.2%29%3A+population+genetics+software+for+exact+tests+and+ecumenicism&amp;auth=Raymond&amp;atitle=GENEPOP+%28version+1.2%29%3A+population+genetics+software+for+exact+tests+and+ecumenicism" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Raymond%5Bauthor%5D+AND+GENEPOP+%28version+1.2%29%3A+population+genetics+software+for+exact+tests+and+ecumenicism" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARaymond+%22GENEPOP+%28version+1.2%29%3A+population+genetics+software+for+exact+tests+and+ecumenicism%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0059654-Dupanloup1" id="pone.0059654-Dupanloup1"></a>Dupanloup I, Schneider S, Excoffier L (2002) A simulated annealing approach to define the genetic structure of populations. Molecular Ecology 11: 2571–2581.
        doi: 10.1046/j.1365-294x.2002.01650.x <ul class="find" data-citedArticleID="7327645" data-doi="10.1046/j.1365-294x.2002.01650.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2002.01650.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dupanloup%5Bauthor%5D+AND+A+simulated+annealing+approach+to+define+the+genetic+structure+of+populations" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADupanloup+%22A+simulated+annealing+approach+to+define+the+genetic+structure+of+populations%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0059654-Pritchard1" id="pone.0059654-Pritchard1"></a>Pritchard JK, Stephens M, Donnelly P (2000) Inference of population structure using multilocus genotype data. Genetics 155: 945–959. <ul class="find" data-citedArticleID="7327647"><li><a href="http://www.crossref.org/guestquery/?auth2=Pritchard&amp;atitle2=Inference+of+population+structure+using+multilocus+genotype+data&amp;auth=Pritchard&amp;atitle=Inference+of+population+structure+using+multilocus+genotype+data" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pritchard%5Bauthor%5D+AND+Inference+of+population+structure+using+multilocus+genotype+data" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APritchard+%22Inference+of+population+structure+using+multilocus+genotype+data%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0059654-Earl1" id="pone.0059654-Earl1"></a>Earl DA, vonHoldt BM (2012) STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method. Conservation Genetics Resources 4: 359–361.
        doi: 10.1007/s12686-011-9548-7 <ul class="find" data-citedArticleID="7327649" data-doi="10.1007/s12686-011-9548-7"><li><a href="http://dx.doi.org/10.1007/s12686-011-9548-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Earl%5Bauthor%5D+AND+STRUCTURE+HARVESTER%3A+a+website+and+program+for+visualizing+STRUCTURE+output+and+implementing+the+Evanno+method" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AEarl+%22STRUCTURE+HARVESTER%3A+a+website+and+program+for+visualizing+STRUCTURE+output+and+implementing+the+Evanno+method%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0059654-Mantel1" id="pone.0059654-Mantel1"></a>Mantel N (1967) The detection of disease clustering and a generalized regression approach. Cancer research 27: 209. <ul class="find" data-citedArticleID="7327651"><li><a href="http://www.crossref.org/guestquery/?auth2=Mantel&amp;atitle2=The+detection+of+disease+clustering+and+a+generalized+regression+approach&amp;auth=Mantel&amp;atitle=The+detection+of+disease+clustering+and+a+generalized+regression+approach" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mantel%5Bauthor%5D+AND+The+detection+of+disease+clustering+and+a+generalized+regression+approach" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMantel+%22The+detection+of+disease+clustering+and+a+generalized+regression+approach%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0059654-Bonnet1" id="pone.0059654-Bonnet1"></a>Bonnet E, Van de Peer Y (2002) zt: a software tool for simple and partial Mantel tests. Journal of Statistical software 7: 1–12. <ul class="find" data-citedArticleID="7327653"><li><a href="http://www.crossref.org/guestquery/?auth2=Bonnet&amp;atitle2=zt%3A+a+software+tool+for+simple+and+partial+Mantel+tests&amp;auth=Bonnet&amp;atitle=zt%3A+a+software+tool+for+simple+and+partial+Mantel+tests" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bonnet%5Bauthor%5D+AND+zt%3A+a+software+tool+for+simple+and+partial+Mantel+tests" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABonnet+%22zt%3A+a+software+tool+for+simple+and+partial+Mantel+tests%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0059654-Beerli1" id="pone.0059654-Beerli1"></a>Beerli P, Felsenstein J (2001) Maximum likelihood estimation of a migration matrix and effective population sizes in n subpopulations by using a coalescent approach. Proceedings of the National Academy of Sciences of the United States of America 98: 4563.
        doi: 10.1073/pnas.081068098 <ul class="find" data-citedArticleID="7327655" data-doi="10.1073/pnas.081068098"><li><a href="http://dx.doi.org/10.1073/pnas.081068098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beerli%5Bauthor%5D+AND+Maximum+likelihood+estimation+of+a+migration+matrix+and+effective+population+sizes+in+n+subpopulations+by+using+a+coalescent+approach" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABeerli+%22Maximum+likelihood+estimation+of+a+migration+matrix+and+effective+population+sizes+in+n+subpopulations+by+using+a+coalescent+approach%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0059654-Ronquist1" id="pone.0059654-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics 19: 1572–1574.
        doi: 10.1093/bioinformatics/btg180 <ul class="find" data-citedArticleID="7327657" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ronquist%5Bauthor%5D+AND+MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARonquist+%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0059654-Posada1" id="pone.0059654-Posada1"></a>Posada D (2009) Selection of models of DNA evolution with jModelTest. Methods in Molecular Biology 537: 93–112.
        doi: 10.1007/978-1-59745-251-9_5 <ul class="find" data-citedArticleID="7327659" data-doi="10.1007/978-1-59745-251-9_5"><li><a href="http://dx.doi.org/10.1007/978-1-59745-251-9_5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Posada%5Bauthor%5D+AND+Selection+of+models+of+DNA+evolution+with+jModelTest" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APosada+%22Selection+of+models+of+DNA+evolution+with+jModelTest%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0059654-Wu1" id="pone.0059654-Wu1"></a>Wu YP, Zhao JL, Su TJ, Li J, Yu F, et al. (2012) The complete mitochondrial genome of <em>Leucoptera malifoliella</em> Costa (Lepidoptera: Lyonetiidae). DNA and Cell Biology 31: 1508–1522.
        doi: 10.1089/dna.2012.1642 <ul class="find" data-citedArticleID="7327661" data-doi="10.1089/dna.2012.1642"><li><a href="http://dx.doi.org/10.1089/dna.2012.1642" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wu%5Bauthor%5D+AND+The+complete+mitochondrial+genome+of+Leucoptera+malifoliella+Costa+%28Lepidoptera%3A+Lyonetiidae%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWu+%22The+complete+mitochondrial+genome+of+Leucoptera+malifoliella+Costa+%28Lepidoptera%3A+Lyonetiidae%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0059654-Bandelt1" id="pone.0059654-Bandelt1"></a>Bandelt HJ, Forster P, Röhl A (1999) Median-joining networks for inferring intraspecific phylogenies. Molecular biology and evolution 16: 37–48.
        doi: 10.1093/oxfordjournals.molbev.a026036 <ul class="find" data-citedArticleID="7327663" data-doi="10.1093/oxfordjournals.molbev.a026036"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bandelt%5Bauthor%5D+AND+Median-joining+networks+for+inferring+intraspecific+phylogenies" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABandelt+%22Median-joining+networks+for+inferring+intraspecific+phylogenies%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0059654-Forster1" id="pone.0059654-Forster1"></a>Forster P, Torroni A, Renfrew C, Röhl A (2001) Phylogenetic star contraction applied to Asian and Papuan mtDNA evolution. Molecular Biology and Evolution 18: 1864–1881.
        doi: 10.1093/oxfordjournals.molbev.a003728 <ul class="find" data-citedArticleID="7327665" data-doi="10.1093/oxfordjournals.molbev.a003728"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003728" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Forster%5Bauthor%5D+AND+Phylogenetic+star+contraction+applied+to+Asian+and+Papuan+mtDNA+evolution" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AForster+%22Phylogenetic+star+contraction+applied+to+Asian+and+Papuan+mtDNA+evolution%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0059654-Polzin1" id="pone.0059654-Polzin1"></a>Polzin T, Daneshmand SV (2003) On Steiner trees and minimum spanning trees in hypergraphs. Operations Research Letters 31: 12–20.
        doi: 10.1016/s0167-6377(02)00185-2 <ul class="find" data-citedArticleID="7327667" data-doi="10.1016/s0167-6377(02)00185-2"><li><a href="http://dx.doi.org/10.1016/s0167-6377(02)00185-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polzin%5Bauthor%5D+AND+On+Steiner+trees+and+minimum+spanning+trees+in+hypergraphs" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APolzin+%22On+Steiner+trees+and+minimum+spanning+trees+in+hypergraphs%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0059654-Tajima1" id="pone.0059654-Tajima1"></a>Tajima F (1989) Statistical method for testing the neutral mutation hypothesis by DNA polymorphism. Genetics 123: 585–595. <ul class="find" data-citedArticleID="7327669"><li><a href="http://www.crossref.org/guestquery/?auth2=Tajima&amp;atitle2=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism&amp;auth=Tajima&amp;atitle=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tajima%5Bauthor%5D+AND+Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATajima+%22Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">97.
              </span><a name="pone.0059654-Fu1" id="pone.0059654-Fu1"></a>Fu YX (1997) Statistical tests of neutrality of mutations against population growth, hitchhiking and background selection. Genetics 147: 915–925. <ul class="find" data-citedArticleID="7327671"><li><a href="http://www.crossref.org/guestquery/?auth2=Fu&amp;atitle2=Statistical+tests+of+neutrality+of+mutations+against+population+growth%2C+hitchhiking+and+background+selection&amp;auth=Fu&amp;atitle=Statistical+tests+of+neutrality+of+mutations+against+population+growth%2C+hitchhiking+and+background+selection" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fu%5Bauthor%5D+AND+Statistical+tests+of+neutrality+of+mutations+against+population+growth%2C+hitchhiking+and+background+selection" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFu+%22Statistical+tests+of+neutrality+of+mutations+against+population+growth%2C+hitchhiking+and+background+selection%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">98.
              </span><a name="pone.0059654-Slatkin1" id="pone.0059654-Slatkin1"></a>Slatkin M, Hudson RR (1991) Pairwise comparisons of mitochondrial DNA sequences in stable and exponentially growing populations. Genetics 129: 555–562. <ul class="find" data-citedArticleID="7327673"><li><a href="http://www.crossref.org/guestquery/?auth2=Slatkin&amp;atitle2=Pairwise+comparisons+of+mitochondrial+DNA+sequences+in+stable+and+exponentially+growing+populations&amp;auth=Slatkin&amp;atitle=Pairwise+comparisons+of+mitochondrial+DNA+sequences+in+stable+and+exponentially+growing+populations" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Slatkin%5Bauthor%5D+AND+Pairwise+comparisons+of+mitochondrial+DNA+sequences+in+stable+and+exponentially+growing+populations" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASlatkin+%22Pairwise+comparisons+of+mitochondrial+DNA+sequences+in+stable+and+exponentially+growing+populations%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">99.
              </span><a name="pone.0059654-Schneider2" id="pone.0059654-Schneider2"></a>Schneider S, Excoffier L (1999) Estimation of past demographic parameters from the distribution of pairwise differences when the mutation rates vary among sites: application to human mitochondrial DNA. Genetics 152: 1079–1089. <ul class="find" data-citedArticleID="7327675"><li><a href="http://www.crossref.org/guestquery/?auth2=Schneider&amp;atitle2=Estimation+of+past+demographic+parameters+from+the+distribution+of+pairwise+differences+when+the+mutation+rates+vary+among+sites%3A+application+to+human+mitochondrial+DNA&amp;auth=Schneider&amp;atitle=Estimation+of+past+demographic+parameters+from+the+distribution+of+pairwise+differences+when+the+mutation+rates+vary+among+sites%3A+application+to+human+mitochondrial+DNA" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Schneider%5Bauthor%5D+AND+Estimation+of+past+demographic+parameters+from+the+distribution+of+pairwise+differences+when+the+mutation+rates+vary+among+sites%3A+application+to+human+mitochondrial+DNA" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASchneider+%22Estimation+of+past+demographic+parameters+from+the+distribution+of+pairwise+differences+when+the+mutation+rates+vary+among+sites%3A+application+to+human+mitochondrial+DNA%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">100.
              </span><a name="pone.0059654-Drummond1" id="pone.0059654-Drummond1"></a>Drummond A, Rambaut A (2007) BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evolutionary Biology 7: 214.
        doi: 10.1186/1471-2148-7-214 <ul class="find" data-citedArticleID="7327677" data-doi="10.1186/1471-2148-7-214"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-214" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drummond%5Bauthor%5D+AND+BEAST%3A+Bayesian+evolutionary+analysis+by+sampling+trees" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADrummond+%22BEAST%3A+Bayesian+evolutionary+analysis+by+sampling+trees%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">101.
              </span><a name="pone.0059654-Drummond2" id="pone.0059654-Drummond2"></a>Drummond A, Rambaut A, Shapiro B, Pybus O (2005) Bayesian coalescent inference of past population dynamics from molecular sequences. Molecular Biology and Evolution 22: 1185–1192.
        doi: 10.1093/molbev/msi103 <ul class="find" data-citedArticleID="7327679" data-doi="10.1093/molbev/msi103"><li><a href="http://dx.doi.org/10.1093/molbev/msi103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drummond%5Bauthor%5D+AND+Bayesian+coalescent+inference+of+past+population+dynamics+from+molecular+sequences" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADrummond+%22Bayesian+coalescent+inference+of+past+population+dynamics+from+molecular+sequences%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.XML" value="247737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.PDF" value="1354787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t001.TIF" value="659198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t001.PNG_I" value="40424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t001.PNG_S" value="4404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t001.PNG_M" value="101481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t001.PNG_L" value="205233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t002.TIF" value="992698"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t002.PNG_I" value="27193"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t002.PNG_S" value="8754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t002.PNG_M" value="60943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t002.PNG_L" value="318968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g001.TIF" value="158074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g001.PNG_I" value="21388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g001.PNG_S" value="11011"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g001.PNG_M" value="32626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g001.PNG_L" value="24819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g002.TIF" value="259012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g002.PNG_I" value="55957"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g002.PNG_S" value="15956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g002.PNG_M" value="79569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g002.PNG_L" value="56123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g003.TIF" value="733938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g003.PNG_I" value="167735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g003.PNG_S" value="21904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g003.PNG_M" value="136799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g003.PNG_L" value="220836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g004.TIF" value="626910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g004.PNG_I" value="69738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g004.PNG_S" value="20769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g004.PNG_M" value="142661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g004.PNG_L" value="233465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t003.TIF" value="1007562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t003.PNG_I" value="36474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t003.PNG_S" value="10818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t003.PNG_M" value="74882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t003.PNG_L" value="347149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g005.TIF" value="618536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g005.PNG_I" value="48564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g005.PNG_S" value="13852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g005.PNG_M" value="112156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.g005.PNG_L" value="227654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t004.TIF" value="566304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t004.PNG_I" value="29874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t004.PNG_S" value="3906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t004.PNG_M" value="71876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.t004.PNG_L" value="160587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s001.PDF" value="77336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s002.PDF" value="169807"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s003.PDF" value="134695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s004.PDF" value="135574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s005.PDF" value="156222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s006.PDF" value="758460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s007.DOCX" value="15782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s008.DOCX" value="16332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s009.DOC" value="134656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s010.DOCX" value="17357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s011.DOCX" value="15180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0059654.s012.DOCX" value="14834"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0059654&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0059654" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0059654&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0059654&volume=&issue=&title=Genetic Structure and Demographic History Reveal Migration of the Diamondback Moth Plutella xylostella (Lepidoptera: Plutellidae) from the Southern to Northern Regions of China&author_name=Shu-Jun%20Wei%2C%20Bao-Cai%20Shi%2C%20Ya-Jun%20Gong%2C%20Gui-Hua%20Jin%2C%20Xue-Xin%20Chen%2C%20Xiang-Feng%20Meng&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0059654" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0059654" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0059654" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0059654&amp;t=Genetic%20Structure%20and%20Demographic%20History%20Reveal%20Migration%20of%20the%20Diamondback%20Moth%20Plutella%20xylostella%20(Lepidoptera%3A%20Plutellidae)%20from%20the%20Southern%20to%20Northern%20Regions%20of%20China" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0059654&title=Genetic%20Structure%20and%20Demographic%20History%20Reveal%20Migration%20of%20the%20Diamondback%20Moth%20Plutella%20xylostella%20(Lepidoptera%3A%20Plutellidae)%20from%20the%20Southern%20to%20Northern%20Regions%20of%20China&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0059654&amp;title=Genetic%20Structure%20and%20Demographic%20History%20Reveal%20Migration%20of%20the%20Diamondback%20Moth%20Plutella%20xylostella%20(Lepidoptera%3A%20Plutellidae)%20from%20the%20Southern%20to%20Northern%20Regions%20of%20China" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0059654" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0059654&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genetic Structure and Demographic History Reveal Migration of the Diamondback Moth Plutella xylostella (Lepidoptera: Plutellidae) from the Southern to Northern Regions of China';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0059654';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genetic%20Structure%20and%20Demographic%20History%20Reveal%20Migration%20of%20the%20Diamondback%20Moth%20Plutella%20xylostella%20(Lepidoptera%3A%20Plutellidae)%20from%20the%20Southern%20to%20Northern%20Regions%20of%20China http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0059654" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0059654" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+migration%22" title="Search for articles in the subject area:'Animal migration'"><div class="flagText">Animal migration</div></a>
              <div data-categoryid="17269" data-articleid="154293"
                   data-categoryname="Animal migration"
                   class="flagImage" title="Flag 'Animal migration' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22China%22" title="Search for articles in the subject area:'China'"><div class="flagText">China</div></a>
              <div data-categoryid="40325" data-articleid="154293"
                   data-categoryname="China"
                   class="flagImage" title="Flag 'China' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="19681" data-articleid="154293"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="154293"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Moths+and+butterflies%22" title="Search for articles in the subject area:'Moths and butterflies'"><div class="flagText">Moths and butterflies</div></a>
              <div data-categoryid="42981" data-articleid="154293"
                   data-categoryname="Moths and butterflies"
                   class="flagImage" title="Flag 'Moths and butterflies' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="154293"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plutella%22" title="Search for articles in the subject area:'Plutella'"><div class="flagText">Plutella</div></a>
              <div data-categoryid="44359" data-articleid="154293"
                   data-categoryname="Plutella"
                   class="flagImage" title="Flag 'Plutella' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="154293"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5164'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7821'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4278&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
