

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Rapid Sequencing of the Bamboo Mitochondrial Genome Using Illumina Technology and Parallel Episodic Evolution of Organelle Genomes in Grasses</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0030297"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0030297" />

    <meta name="citation_title" content="Rapid Sequencing of the Bamboo Mitochondrial Genome Using Illumina Technology and Parallel Episodic Evolution of Organelle Genomes in Grasses"/>
    <meta itemprop="name" content="Rapid Sequencing of the Bamboo Mitochondrial Genome Using Illumina Technology and Parallel Episodic Evolution of Organelle Genomes in Grasses"/>

      <meta name="citation_author" content="Peng-Fei Ma"/>
            <meta name="citation_author_institution" content="Key Laboratory of Biodiversity and Biogeography, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, People's Republic of China"/>
            <meta name="citation_author_institution" content="Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, People's Republic of China"/>
            <meta name="citation_author_institution" content="Graduate University of Chinese Academy of Sciences, Beijing, People's Republic of China"/>
      <meta name="citation_author" content="Zhen-Hua Guo"/>
            <meta name="citation_author_institution" content="Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, People's Republic of China"/>
      <meta name="citation_author" content="De-Zhu Li"/>
            <meta name="citation_author_institution" content="Key Laboratory of Biodiversity and Biogeography, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, People's Republic of China"/>
            <meta name="citation_author_institution" content="Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, People's Republic of China"/>

    <meta name="citation_date" content="2012/1/17"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0030297.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e30297"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="7"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=‘Next-generation’ sequencing becomes ‘now-generation’.; citation_author=DE Neafsey; citation_author=BJ Haas; citation_journal_title=Genome Biol; citation_volume=12; citation_number=1; citation_pages=303; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Whole genome sequencing of enriched chloroplast DNA using the Illumina GAII platform.; citation_author=RA Atherton; citation_author=BJ McComish; citation_author=LD Shepherd; citation_author=LA Berry; citation_author=NW Albert; citation_journal_title=Plant Methods; citation_volume=6; citation_number=2; citation_pages=22; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Chloroplast genome sequences from total DNA for plant identification.; citation_author=CJ Nock; citation_author=DL Waters; citation_author=MA Edwards; citation_author=SG Bowen; citation_author=N Rice; citation_journal_title=Plant Biotechnol J; citation_volume=9; citation_number=3; citation_pages=328-333; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=The evolution of chloroplast genes and genomes in ferns.; citation_author=PG Wolf; citation_author=JP Der; citation_author=AM Duffy; citation_author=JB Davidson; citation_author=AL Grusz; citation_journal_title=Plant Mol Biol; citation_volume=76; citation_number=4; citation_pages=251-261; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Methods for obtaining and analyzing whole chloroplast genome sequences.; citation_author=RK Jansen; citation_author=LA Raubeson; citation_author=JL Boore; citation_author=CW dePamphilis; citation_author=TW Chumley; citation_journal_title=Methods Enzymol; citation_volume=395; citation_number=5; citation_pages=348-384; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Discovery of global genomic re-organization based on comparison of two newly sequenced rice mitochondrial genomes with cytoplasmic male sterility-related genes.; citation_author=S Fujii; citation_author=T Kazama; citation_author=M Yamada; citation_author=K Toriyama; citation_journal_title=BMC Genomics; citation_volume=11; citation_number=6; citation_pages=209; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Determination of the melon chloroplast and mitochondrial genome sequences reveals that the largest reported mitochondrial genome in plants contains a significant amount of DNA having a nuclear origin.; citation_author=L Rodriguez-Moreno; citation_author=V Gonzalez; citation_author=A Benjak; citation_author=MC Marti; citation_author=P Puigdomenech; citation_journal_title=BMC Genomics; citation_volume=12; citation_number=7; citation_pages=424; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Double-strand break repair processes drive evolution of the mitochondrial genome in Arabidopsis.; citation_author=JI Davila; citation_author=MP Arrieta-Montiel; citation_author=Y Wamboldt; citation_author=J Cao; citation_author=J Hagmann; citation_journal_title=BMC Biol; citation_volume=9; citation_number=8; citation_pages=64; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=High-throughput sequencing of six bamboo chloroplast genomes: phylogenetic implications for temperate woody bamboos (Poaceae: Bambusoideae).; citation_author=YJ Zhang; citation_author=MF Ma; citation_author=DZ Li; citation_journal_title=PLoS ONE; citation_volume=6; citation_number=9; citation_pages=e20596; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Dynamic evolution of plant mitochondrial genomes: mobile genes and introns and highly variable mutation rates.; citation_author=JD Palmer; citation_author=KL Adams; citation_author=Y Cho; citation_author=CL Parkinson; citation_author=YL Qiu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=10; citation_pages=6960-6966; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Punctuated evolution of mitochondrial gene content: high and variable rates of mitochondrial gene loss and transfer to the nucleus during angiosperm evolution.; citation_author=KL Adams; citation_author=YL Qiu; citation_author=M Stoutemyer; citation_author=JD Palmer; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=11; citation_pages=9905-9912; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Angiosperm mitochondrial genomes and mutations.; citation_author=T Kubo; citation_author=KJ Newton; citation_journal_title=Mitochondrion; citation_volume=8; citation_number=12; citation_pages=5-14; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Plant mitochondrial DNA evolves rapidly in structure, but slowly in sequence.; citation_author=JD Palmer; citation_author=LA Herbon; citation_journal_title=J Mol Evol; citation_volume=28; citation_number=13; citation_pages=87-97; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Rates of nucleotide substitution vary greatly among plant mitochondrial, chloroplast, and nuclear DNAs.; citation_author=KH Wolfe; citation_author=WH Li; citation_author=PM Sharp; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=84; citation_number=14; citation_pages=9054-9058; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Evolutionary rate variation at multiple levels of biological organization in plant mitochondrial DNA.; citation_author=DB Sloan; citation_author=CM Barr; citation_author=MS Olson; citation_author=SR Keller; citation_author=DR Taylor; citation_journal_title=Mol Biol Evol; citation_volume=25; citation_number=15; citation_pages=243; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Extensive variation in synonymous substitution rates in mitochondrial genes of seed plants.; citation_author=JP Mower; citation_author=P Touzet; citation_author=JS Gummow; citation_author=LF Delph; citation_author=JD Palmer; citation_journal_title=BMC Evol Biol; citation_volume=7; citation_number=16; citation_pages=135; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial substitution rates are extraordinarily elevated and variable in a genus of flowering plants.; citation_author=Y Cho; citation_author=JP Mower; citation_author=YL Qiu; citation_author=JD Palmer; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=17; citation_pages=17741-17746; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Multiple major increases and decreases in mitochondrial substitution rates in the plant family Geraniaceae.; citation_author=CL Parkinson; citation_author=JP Mower; citation_author=YL Qiu; citation_author=AJ Shirk; citation_author=K Song; citation_journal_title=BMC Evol Biol; citation_volume=5; citation_number=18; citation_pages=73; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analysis of mitochondrial substitution rate variation in the angiosperm tribe Sileneae.; citation_author=DB Sloan; citation_author=B Oxelman; citation_author=A Rautenberg; citation_author=DR Taylor; citation_journal_title=BMC Evol Biol; citation_volume=9; citation_number=19; citation_pages=260; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Parallel rate heterogeneity in chloroplast and mitochondrial genomes of Brazil nut trees (Lecythidaceae) is consistent with lineage effects.; citation_author=DF Soria-Hernanz; citation_author=JM Braverman; citation_author=MB Hamilton; citation_journal_title=Mol Biol Evol; citation_volume=25; citation_number=20; citation_pages=1282-1296; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Correlated rates of synonymous site evolution across plant genomes.; citation_author=A Eyre-Walker; citation_author=BS Gaut; citation_journal_title=Mol Biol Evol; citation_volume=14; citation_number=21; citation_pages=455-460; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Episodic evolution and adaptation of chloroplast genomes in ancestral grasses.; citation_author=B Zhong; citation_author=T Yonezawa; citation_author=Y Zhong; citation_author=M Hasegawa; citation_journal_title=PLoS One; citation_volume=4; citation_number=22; citation_pages=e5297; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Analysis of 81 genes from 64 plastid genomes resolves relationships in angiosperms and identifies genome-scale evolutionary patterns.; citation_author=RK Jansen; citation_author=Z Cai; citation_author=LA Raubeson; citation_author=H Daniell; citation_author=CW Depamphilis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=23; citation_pages=19369-19374; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Using plastid genome-scale data to resolve enigmatic relationships among basal angiosperms.; citation_author=MJ Moore; citation_author=CD Bell; citation_author=PS Soltis; citation_author=DE Soltis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=24; citation_pages=19363-19368; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analysis of 83 plastid genes further resolves the early diversification of eudicots.; citation_author=MJ Moore; citation_author=PS Soltis; citation_author=CD Bell; citation_author=JG Burleigh; citation_author=DE Soltis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=107; citation_number=25; citation_pages=4623-4628; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of the grass family (Poaceae) from rpl16 intron sequence data.; citation_author=W Zhang; citation_journal_title=Mol Phylogenet Evol; citation_volume=15; citation_number=26; citation_pages=135-146; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Large multi-gene phylogenetic trees of the grasses (Poaceae): progress towards complete tribal and generic level sampling.; citation_author=Y Bouchenak-Khelladi; citation_author=N Salamin; citation_author=V Savolainen; citation_author=F Forest; citation_author=M Bank; citation_journal_title=Mol Phylogenet Evol; citation_volume=47; citation_number=27; citation_pages=488-505; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Phylogeny and subfamilial classification of the grasses (Poaceae).; citation_author=G Grass Phylogeny Working; citation_journal_title=Ann Mo Bot Gard; citation_volume=88; citation_number=28; citation_pages=373-457; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Angiosperm phylogeny inferred from sequences of four mitochondrial genes.; citation_author=YL Qiu; citation_author=L Li; citation_author=B Wang; citation_author=JY Xue; citation_author=TA Hendry; citation_journal_title=J Syst Evol; citation_volume=48; citation_number=29; citation_pages=391-425; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Reconstructing the basal angiosperm phylogeny: evaluating information content of mitochondrial genes.; citation_author=YL Qiu; citation_author=L Li; citation_author=TA Hendry; citation_author=R Li; citation_author=DW Taylor; citation_journal_title=Taxon; citation_volume=55; citation_number=30; citation_pages=837-856; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dating the monocot-dicot divergence and the origin of core eudicots using whole chloroplast genomes.; citation_author=SM Chaw; citation_author=CC Chang; citation_author=HL Chen; citation_author=WH Li; citation_journal_title=J Mol Evol; citation_volume=58; citation_number=31; citation_pages=424-441; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Paleontology. Dinosaurs dined on grass.; citation_author=DR Piperno; citation_author=HD Sues; citation_journal_title=Science; citation_volume=310; citation_number=32; citation_pages=1126-1128; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The age of the grasses and clusters of origins of C4 photosynthesis.; citation_author=A Vicentini; citation_author=JC Barber; citation_author=SS Aliscioni; citation_author=LM Giussani; citation_author=EA Kellogg; citation_journal_title=Global Change Biol; citation_volume=14; citation_number=33; citation_pages=2963-2977; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Biogeography of the grasses (Poaceae): a phylogenetic approach to reveal evolutionary history in geographical space and geological time.; citation_author=Y Bouchenak-Khelladi; citation_author=GA Verboom; citation_author=V Savolainen; citation_author=TR Hodkinson; citation_journal_title=Bot J Linn Soc; citation_volume=162; citation_number=34; citation_pages=543-557; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=De novo assembly of human genomes with massively parallel short read sequencing.; citation_author=R Li; citation_author=H Zhu; citation_author=J Ruan; citation_author=W Qian; citation_author=X Fang; citation_journal_title=Genome Res; citation_volume=20; citation_number=35; citation_pages=265-272; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Rapid and accurate pyrosequencing of angiosperm plastid genomes.; citation_author=MJ Moore; citation_author=A Dhingra; citation_author=PS Soltis; citation_author=R Shaw; citation_author=WG Farmerie; citation_journal_title=BMC Plant Biol; citation_volume=6; citation_number=36; citation_pages=17; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multiplex sequencing of plant chloroplast genomes using Solexa sequencing-by-synthesis technology.; citation_author=R Cronn; citation_author=A Liston; citation_author=M Parks; citation_author=DS Gernandt; citation_author=R Shen; citation_journal_title=Nucleic Acids Res; citation_volume=36; citation_number=37; citation_pages=e122; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=The complete sequence of the rice (Oryza sativa L.) mitochondrial genome: frequent DNA sequence acquisition and loss during the evolution of flowering plants.; citation_author=Y Notsu; citation_author=S Masood; citation_author=T Nishikawa; citation_author=N Kubo; citation_author=G Akiduki; citation_journal_title=Mol Genet Genomics; citation_volume=268; citation_number=38; citation_pages=434-445; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structural dynamics of cereal mitochondrial genomes as revealed by complete nucleotide sequencing of the wheat mitochondrial genome.; citation_author=Y Ogihara; citation_author=Y Yamazaki; citation_author=K Murai; citation_author=A Kanno; citation_author=T Terachi; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=39; citation_pages=6235-6250; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ribosomal protein L10 is encoded in the mitochondrial genome of many land plants and green algae.; citation_author=JP Mower; citation_author=L Bonen; citation_journal_title=BMC Evol Biol; citation_volume=9; citation_number=40; citation_pages=265; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Pervasive survival of expressed mitochondrial rps14 pseudogenes in grasses and their relatives for 80 million years following three functional transfers to the nucleus.; citation_author=HC Ong; citation_author=JD Palmer; citation_journal_title=BMC Evol Biol; citation_volume=6; citation_number=41; citation_pages=55; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Linking phylogeny and genetics: Zea mays as a tool for phylogenetic studies.; citation_author=EA Kellogg; citation_author=JA Birchler; citation_journal_title=Syst Biol; citation_volume=42; citation_number=42; citation_pages=415; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A review of long-branch attraction.; citation_author=J Bergsten; citation_journal_title=Cladistics; citation_volume=21; citation_number=43; citation_pages=163-193; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-scale data, angiosperm relationships, and ‘ending incongruence’: a cautionary tale in phylogenetics.; citation_author=DE Soltis; citation_author=VA Albert; citation_author=V Savolainen; citation_author=K Hilu; citation_author=YL Qiu; citation_journal_title=Trends Plant Sci; citation_volume=9; citation_number=44; citation_pages=477-483; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Beyond linear sequence comparisons: the use of genome-level characters for phylogenetic reconstruction.; citation_author=JL Boore; citation_author=SI Fuerstenberg; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=363; citation_number=45; citation_pages=1445-1451; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=The use of genome-level characters for phylogenetic reconstruction.; citation_author=JL Boore; citation_journal_title=Trends Ecol Evol; citation_volume=21; citation_number=46; citation_pages=439-446; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Relative rates of nucleotide substitution at the rbcL locus of monocotyledonous plants.; citation_author=BS Gaut; citation_author=SV Muse; citation_author=WD Clark; citation_author=MT Clegg; citation_journal_title=J Mol Evol; citation_volume=35; citation_number=47; citation_pages=292-303; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=RNA editing in gymnosperms and its impact on the evolution of the mitochondrial coxI gene.; citation_author=MZ Lu; citation_author=AE Szmidt; citation_author=XR Wang; citation_journal_title=Plant Mol Biol; citation_volume=37; citation_number=48; citation_pages=225-234; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Insights into the evolution of mitochondrial genome size from complete sequences of Citrullus lanatus and Cucurbita pepo (Cucurbitaceae).; citation_author=AJ Alverson; citation_author=X Wei; citation_author=DW Rice; citation_author=DB Stern; citation_author=K Barry; citation_journal_title=Mol Biol Evol; citation_volume=27; citation_number=49; citation_pages=1436-1448; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Physical characterization of mitochondrial DNA from cotton.; citation_author=CL Hsu; citation_author=BC Mullin; citation_journal_title=Plant Mol Biol; citation_volume=13; citation_number=50; citation_pages=467-468; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The mitochondrial genome of the legume Vigna radiata and the analysis of recombination across short mitochondrial repeats.; citation_author=AJ Alverson; citation_author=S Zhuo; citation_author=DW Rice; citation_author=DB Sloan; citation_author=JD Palmer; citation_journal_title=PLoS One; citation_volume=6; citation_number=51; citation_pages=e16404; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Accurate whole human genome sequencing using reversible terminator chemistry.; citation_author=DR Bentley; citation_author=S Balasubramanian; citation_author=HP Swerdlow; citation_author=GP Smith; citation_author=J Milton; citation_journal_title=Nature; citation_volume=456; citation_number=52; citation_pages=53-59; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA of Vitis vinifera and the issue of rampant horizontal gene transfer.; citation_author=VV Goremykin; citation_author=F Salamini; citation_author=R Velasco; citation_author=R Viola; citation_journal_title=Mol Biol Evol; citation_volume=26; citation_number=53; citation_pages=99-110; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Heterotachy and tree building: A case study with plastids and eubacteria.; citation_author=P Lockhart; citation_author=P Novis; citation_author=BG Milligan; citation_author=J Riden; citation_author=A Rambau; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=54; citation_pages=40-45; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Loss of different inverted repeat copies from the chloroplast genomes of Pinaceae and Cupressophytes and influence of heterotachy on the evaluation of gemnosperm phylogeny.; citation_author=CS Wu; citation_author=YN Wang; citation_author=CY Hsu; citation_author=CP Lin; citation_author=SM Chaw; citation_journal_title=Genome Biol Evol; citation_number=55; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.; citation_author=B Langmead; citation_author=C Trapnell; citation_author=M Pop; citation_author=SL Salzberg; citation_journal_title=Genome Biol; citation_volume=10; citation_number=56; citation_pages=R25; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0.; citation_author=K Tamura; citation_author=J Dudley; citation_author=M Nei; citation_author=S Kumar; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=57; citation_pages=1596-1599; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The tRNAscan-SE, snoscan and snoGPS web servers for the detection of tRNAs and snoRNAs.; citation_author=P Schattner; citation_author=AN Brooks; citation_author=TM Lowe; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=58; citation_pages=W686-689; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Effects of RNA editing and gene processing on phylogenetic reconstruction.; citation_author=LM Bowe; citation_author=CW dePamphilis; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=59; citation_pages=1159-1166; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Is plant mitochondrial RNA editing a source of phylogenetic incongruence? An answer from in silico and in vivo data sets.; citation_author=E Picardi; citation_author=C Quagliariello; citation_journal_title=BMC Bioinformatics; citation_volume=9; citation_number=60; citation_pages=S14; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.; citation_author=A Stamatakis; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=61; citation_pages=2688-2690; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=62; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=D Swofford; citation_number=63; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=64; citation_pages=1586-1591; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The mitochondrial genome of the gymnosperm Cycas taitungensis contains a novel family of short interspersed elements, Bpu sequences, and abundant RNA editing sites.; citation_author=SM Chaw; citation_author=AC Shih; citation_author=D Wang; citation_author=YW Wu; citation_author=SM Liu; citation_journal_title=Mol Biol Evol; citation_volume=25; citation_number=65; citation_pages=603-615; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Sequence and comparative analysis of the maize NB mitochondrial genome.; citation_author=SW Clifton; citation_author=P Minx; citation_author=CM Fauron; citation_author=M Gibson; citation_author=JO Allen; citation_journal_title=Plant Physiol; citation_volume=136; citation_number=66; citation_pages=3486-3503; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The mitochondrial genome of Arabidopsis thaliana contains 57 genes in 366,924 nucleotides.; citation_author=M Unseld; citation_author=JR Marienfeld; citation_author=P Brandt; citation_author=A Brennicke; citation_journal_title=Nat Genet; citation_volume=15; citation_number=67; citation_pages=57-61; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The complete nucleotide sequence of the mitochondrial genome of sugar beet (Beta vulgaris L.) reveals a novel gene for tRNA(Cys)(GCA).; citation_author=T Kubo; citation_author=S Nishizawa; citation_author=A Sugawara; citation_author=N Itchoda; citation_author=A Estiati; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=68; citation_pages=2571-2576; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The complete nucleotide sequence and RNA editing content of the mitochondrial genome of rapeseed (Brassica napus L.): comparative analysis of the mitochondrial genomes of rapeseed and Arabidopsis thaliana.; citation_author=H Handa; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=69; citation_pages=5907-5916; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The complete nucleotide sequence and multipartite organization of the tobacco mitochondrial genome: comparative analysis of mitochondrial genomes in higher plants.; citation_author=Y Sugiyama; citation_author=Y Watase; citation_author=M Nagase; citation_author=N Makita; citation_author=S Yagura; citation_journal_title=Mol Genet Genomics; citation_volume=272; citation_number=70; citation_pages=603-615; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Extensive loss of translational genes in the structurally dynamic mitochondrial genome of the angiosperm Silene latifolia.; citation_author=DB Sloan; citation_author=AJ Alverson; citation_author=H Storchova; citation_author=JD Palmer; citation_author=DR Taylor; citation_journal_title=BMC Evol Biol; citation_volume=10; citation_number=71; citation_pages=274; citation_date=2010; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0030297" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Rapid Sequencing of the Bamboo Mitochondrial Genome Using Illumina Technology and Parallel Episodic Evolution of Organelle Genomes in Grasses"/>
    <meta name="twitter:description" content="          Background          Compared to their counterparts in animals, the mitochondrial (mt) genomes of angiosperms exhibit a number of unique features. However, unravelling their evolution is hindered by the few completed genomes, of which are essentially Sanger sequenced. While next-generation sequencing technologies have revolutionized chloroplast genome sequencing, they are just beginning to be applied to angiosperm mt genomes. Chloroplast genomes of grasses (Poaceae) have undergone episodic evolution and the evolutionary rate was suggested to be correlated between chloroplast and mt genomes in Poaceae. It is interesting to investigate whether correlated rate change also occurred in grass mt genomes as expected under lineage effects. A time-calibrated phylogenetic tree is needed to examine rate change.                          Methodology/Principal Findings          We determined a largely completed mt genome from a bamboo, Ferrocalamus rimosivaginus (Poaceae), through Illumina sequencing of total DNA. With combination of de novo and reference-guided assembly, 39.5-fold coverage Illumina reads were finally assembled into scaffolds totalling 432,839 bp. The assembled genome contains nearly the same genes as the completed mt genomes in Poaceae. For examining evolutionary rate in grass mt genomes, we reconstructed a phylogenetic tree including 22 taxa based on 31 mt genes. The topology of the well-resolved tree was almost identical to that inferred from chloroplast genome with only minor difference. The inconsistency possibly derived from long branch attraction in mtDNA tree. By calculating absolute substitution rates, we found significant rate change (&sim;4-fold) in mt genome before and after the diversification of Poaceae both in synonymous and nonsynonymous terms. Furthermore, the rate change was correlated with that of chloroplast genomes in grasses.                          Conclusions/Significance          Our result demonstrates that it is a rapid and efficient approach to obtain angiosperm mt genome sequences using Illumina sequencing technology. The parallel episodic evolution of mt and chloroplast genomes in grasses is consistent with lineage effects.        "/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0030297.g005"/>

  <meta property="og:title" content="Rapid Sequencing of the Bamboo Mitochondrial Genome Using Illumina Technology and Parallel Episodic Evolution of Organelle Genomes in Grasses" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0030297" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6192'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1090'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1089&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030297">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0030297" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0030297&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Rapid Sequencing of the Bamboo Mitochondrial Genome Using Illumina Technology and Parallel Episodic Evolution of Organelle Genomes in Grasses
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peng-Fei Ma, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Key Laboratory of Biodiversity and Biogeography, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, People's Republic of China, 
                  Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, People's Republic of China, 
                  Graduate University of Chinese Academy of Sciences, Beijing, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhen-Hua Guo
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:guozhenhua@mail.kib.ac.cn">guozhenhua@mail.kib.ac.cn</a> (Z-HG); <a href="mailto:dzl@mail.kib.ac.cn">dzl@mail.kib.ac.cn</a> (D-ZL)</p>

                <p>Affiliation:
                  Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            De-Zhu Li
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:guozhenhua@mail.kib.ac.cn">guozhenhua@mail.kib.ac.cn</a> (Z-HG); <a href="mailto:dzl@mail.kib.ac.cn">dzl@mail.kib.ac.cn</a> (D-ZL)</p>

                <p>Affiliations:
                  Key Laboratory of Biodiversity and Biogeography, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, People's Republic of China, 
                  Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 17, 2012</li>
    <li>DOI: 10.1371/journal.pone.0030297</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0030297" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0030297" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0030297" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0030297" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0030297-t001" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0030297-t002" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0030297-t003" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0030297-t004" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0030297-g001" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0030297-g002" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0030297-g003" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0030297-g004" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0030297-g005" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0030297">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0030297" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Compared to their counterparts in animals, the mitochondrial (mt) genomes of angiosperms exhibit a number of unique features. However, unravelling their evolution is hindered by the few completed genomes, of which are essentially Sanger sequenced. While next-generation sequencing technologies have revolutionized chloroplast genome sequencing, they are just beginning to be applied to angiosperm mt genomes. Chloroplast genomes of grasses (Poaceae) have undergone episodic evolution and the evolutionary rate was suggested to be correlated between chloroplast and mt genomes in Poaceae. It is interesting to investigate whether correlated rate change also occurred in grass mt genomes as expected under lineage effects. A time-calibrated phylogenetic tree is needed to examine rate change.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We determined a largely completed mt genome from a bamboo, <em>Ferrocalamus rimosivaginus</em> (Poaceae), through Illumina sequencing of total DNA. With combination of <em>de novo</em> and reference-guided assembly, 39.5-fold coverage Illumina reads were finally assembled into scaffolds totalling 432,839 bp. The assembled genome contains nearly the same genes as the completed mt genomes in Poaceae. For examining evolutionary rate in grass mt genomes, we reconstructed a phylogenetic tree including 22 taxa based on 31 mt genes. The topology of the well-resolved tree was almost identical to that inferred from chloroplast genome with only minor difference. The inconsistency possibly derived from long branch attraction in mtDNA tree. By calculating absolute substitution rates, we found significant rate change (~4-fold) in mt genome before and after the diversification of Poaceae both in synonymous and nonsynonymous terms. Furthermore, the rate change was correlated with that of chloroplast genomes in grasses.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our result demonstrates that it is a rapid and efficient approach to obtain angiosperm mt genome sequences using Illumina sequencing technology. The parallel episodic evolution of mt and chloroplast genomes in grasses is consistent with lineage effects.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ma P-F, Guo Z-H, Li D-Z (2012) Rapid Sequencing of the Bamboo Mitochondrial Genome Using Illumina Technology and Parallel Episodic Evolution of Organelle Genomes in Grasses. PLoS ONE 7(1):
          e30297.
            doi:10.1371/journal.pone.0030297</p><p><strong>Editor: </strong>Jonathan H. Badger, J. Craig Venter Institute, United States of America</p><p><strong>Received:</strong> August 24, 2011; <strong>Accepted:</strong> December 13, 2011; <strong>Published:</strong> January 17, 2012</p><p><strong>Copyright:</strong> © 2012 Ma et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This project is supported by National Natural Science Foundation of China (no. 31170204), the Chinese Academy of Sciences under the knowledge innovation program (grants KSCX-YW-N-029 and KSCX2-YW-N-067), and Yunnan Provincial Government through an innovation team program. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Next-generation sequencing that is not only high-throughput but also low-cost has already revolutionized approaches for genome sequencing and is now becoming ‘now-generation’ sequencing <a href="#pone.0030297-Neafsey1">[1]</a>. Recently complete or nearly complete chloroplast (cp) genomes of plants have been successfully recovered by Illumina sequencing or 454 pyrosequencing from total DNA containing cpDNA as well as nuclear and mitochondrial (mt) DNA <a href="#pone.0030297-Atherton1">[2]</a>–<a href="#pone.0030297-Wolf1">[4]</a>. Compared to conventional approaches for cp genome sequencing involving purification or PCR amplification of cpDNA <a href="#pone.0030297-Jansen1">[5]</a>, these methods are more simple and effective <a href="#pone.0030297-Atherton1">[2]</a>–<a href="#pone.0030297-Wolf1">[4]</a>. On the contrary, the next-generation sequencing technologies are just beginning to be applied to sequencing of mt genomes of plants and only a few examples of plant mt genome next-generation sequencing have been published so far <a href="#pone.0030297-Fujii1">[6]</a>–<a href="#pone.0030297-Davila1">[8]</a>. The 21 available mt genomes of higher plants (<a href="#pone-0030297-t001">Table 1</a>) are essentially obtained by Sanger sequencing, involving extracting or PCR amplification of mtDNA and library construction prior to sequencing. This time- and labour-intensive sequencing approach to some extent limits the sequencing of plant mt genomes. Recently, we have sequenced six bamboo cp genomes from total DNA that was enriched for cpDNA using Illumina sequencing <a href="#pone.0030297-Zhang1">[9]</a>. Meanwhile, we found a large number of sequence reads in the total reads being of mtDNA origin. Could the whole or largely completed mt genome be assembled from these reads at the same time? And if so, this would be a rapid and efficient approach to sequence angiosperm mt genomes and more sequenced genomes will be helpful in understanding the extraordinary evolutionary history of angiosperm mt genomes.</p>
<div class="figure" id="pone-0030297-t001"><div class="img"><a name="pone-0030297-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0030297.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0030297.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Mitochondrial genomes of the 22 seed plants included in phylogenetic analyses in this study.</span></strong></p><span>doi:10.1371/journal.pone.0030297.t001</span></div><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The mt genomes of angiosperms exhibit a number of unique features, which distinguish them from their counterparts in animals or other organisms. These features include expanded genome size, frequent structure rearrangement via recombination, ongoing gene loss and transfer to the nuclear genome, uptake of cpDNA and nuclear DNA, and a generally low rate of molecular evolution <a href="#pone.0030297-Palmer1">[10]</a>–<a href="#pone.0030297-Wolfe1">[14]</a>. Among them, the slow mt sequence evolution is probably the most prominent and has long been appreciated <a href="#pone.0030297-Palmer1">[10]</a>, <a href="#pone.0030297-Palmer2">[13]</a>, <a href="#pone.0030297-Wolfe1">[14]</a>. However, recent studies based on one or a few genes have identified several cases of rate acceleration in mt genomes of certain angiosperm lineages <a href="#pone.0030297-Sloan1">[15]</a>–<a href="#pone.0030297-Sloan2">[19]</a>, and some of these rate increases are temporary with rates returning to normally low levels after acceleration <a href="#pone.0030297-Cho1">[17]</a>, <a href="#pone.0030297-Parkinson1">[18]</a>. These changes in evolutionary rate are mostly restricted to mt genome as expected under locus-specific effects without correlated rate changes in cp and/or nuclear genes <a href="#pone.0030297-Sloan1">[15]</a>–<a href="#pone.0030297-Sloan2">[19]</a>. Nevertheless, a few studies have reported parallel rate changes in mt and cp genomes as expected under lineage effects <a href="#pone.0030297-SoriaHernanz1">[20]</a>, <a href="#pone.0030297-EyreWalker1">[21]</a>. Among them a widely cited case demonstrated a faster rate of synonymous substitutions, which was correlated across cp, mt and nuclear loci in grasses relative to palm <a href="#pone.0030297-EyreWalker1">[21]</a>. In addition, Zhong et al. <a href="#pone.0030297-Zhong1">[22]</a> found that episodic rate acceleration of cp genomes occurred in the ancestral grasses and then the rate reverted to the slow rate typical of most monocot species in the descendant lineages. Here, we are interested to investigate whether the similar pattern of rate change occurred in mt genomes during grass evolution.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A well-supported phylogenetic tree that is time calibrated is necessary to examine absolute substitution rates. Although the backbone phylogeny of angiosperms has been established <a href="#pone.0030297-Jansen2">[23]</a>–<a href="#pone.0030297-Moore2">[25]</a> and the major phylogenetic relationships within the grass family (Poaceae) are resolved with the whole family divided into several basal lineages plus two major lineages (the BEP clade and the PACMAD clade, core Poaceae) <a href="#pone.0030297-Zhang1">[9]</a>, <a href="#pone.0030297-Zhang2">[26]</a>–<a href="#pone.0030297-GrassPhylogenyWorking1">[28]</a>, these studies have relied mainly on the cpDNA sequences. However, the mtDNA sequences are also useful for reconstructing phylogeny of angiosperms, especially at deep level <a href="#pone.0030297-Qiu1">[29]</a>, <a href="#pone.0030297-Qiu2">[30]</a>. It would be informative to use signals from the mt genome to evaluate independently these relationships derived from the cp genome. In addition, there are numerous studies that could provide reliable estimate of divergence times during the grass evolution <a href="#pone.0030297-Moore2">[25]</a>, <a href="#pone.0030297-Chaw1">[31]</a>–<a href="#pone.0030297-BouchenakKhelladi2">[34]</a>. Well-resolved phylogenetic relationships and reliable calibration in combination would make it feasible to examine the rate change in mt genomes of grasses.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here we demonstrated a new approach for sequencing angiosperm mt genome using Illumina sequencing-by-synthesis technology and determined a largely completed mt genome from a bamboo, <em>Ferrocalamus rimosivaginus</em> T. H. Wen (Poaceae). A phylogenetic tree including 22 taxa whose mt genomes have been sequenced was reconstructed based on the sequences of 31 mt genes and the topology was nearly congruent with that inferred from the cp genome. By examining the evolutionary rate of grass mt genomes along the time-calibrated tree, we found a parallel rate change in mt and cp genomes of grasses.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Illumina Sequencing, Genome Assembly and PCR Validation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The template DNA for the <em>F. rimosivaginus</em> cp genome sequencing, which was extracted by a rapid and simple procedure from fresh leaves, in fact contained mtDNA as well <a href="#pone.0030297-Zhang1">[9]</a>. We employed a whole-genome shogun sequencing strategy and one paired-end library for Illumina sequencing was constructed with insert size of about 500 bp. The Illumina system produced 1,594,119 usable paired-end (73 bp and 75 bp) reads in one run for genome assembly <a href="#pone.0030297-Zhang1">[9]</a>. These reads were a mixture of reads derived from cp, mt and nuclear genomes.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We assembled the raw reads using the software SOAPdenovo <a href="#pone.0030297-Li1">[35]</a> with optimal parameters. All the assembled scaffolds and contigs larger than 100 bp were first mapped to the reference mt genomes from nine species of the grass family (<a href="#pone-0030297-t001">Table 1</a>), resulting in 68 mapped contigs and scaffolds. Subsequently we searched these 68 contigs and scaffolds for sequences with significant identity (≥90%) to the cp genome of <em>F. rimosivaginus</em>, and the aligned 3 scaffolds and 35 contigs were removed to avoid the impact of the cpDNA reads on our assembly. In addition, a 3,399 bp sequence located at one end of a scaffold and related to the remaining sequence by an estimated 386 bp length gap was deleted from this scaffold for the same reason. At this point, we obtained 16 scaffolds and 14 contigs with an N50 size of 49.6 kb, achieving a total length of 431.7 kb (<a href="#pone-0030297-t002">Table 2</a>, initial assembly) (see details in <a href="#pone.0030297.s004">Table S1</a>). The average sequencing depth was 39.5× (114,891 mt-derived paired-end reads and 7.2% of total), and there was a relatively narrow variation in the 16 scaffolds sequencing coverage, ranging from 31.9× to 46.5× with a median value of 39.3×. Although these scaffolds contained internal gaps as paired-end information was used to join contigs into scaffolds, the total number of gaps was only 24 with a mean estimated size of 151.5 bp (<a href="#pone-0030297-t002">Table 2</a>).</p>
<div class="figure" id="pone-0030297-t002"><div class="img"><a name="pone-0030297-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0030297.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0030297.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Summary of the <em>F. rimosivaginus</em> mitochondrial genome sequencing and assembly.</span></strong></p><span>doi:10.1371/journal.pone.0030297.t002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We further joined the initially assembled contigs and scaffolds into larger scaffolds using the sequence overlap information between them and synteny between the assembled sequences and the reference genomes. This procedure combined 4 scaffolds and 13 contigs in total. All the linkages between them were successfully confirmed by PCR amplification with conventional Sanger sequencing. To close the intra-scaffold gaps, we designed PCR primers and Sanger sequenced the amplified regions. In sum, 23 gaps (an assumed 17 bp gap was proved to be zero-length by PCR analysis and thus not accounted) were closed with a mean size of 172.8 bp which was slightly larger than the estimated size, thereby validating the linkages between regions spanning gaps within scaffolds. The final assembly had 13 scaffolds and one contig with an N50 size of 53.1 kb and a total length of 432.8 kb (<a href="#pone-0030297-t002">Table 2</a>, see details in <a href="#pone.0030297.s004">Table S1</a>), achieving an average sequencing depth of 39.4×.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Validation of linkages and closing gaps by conventional sequencing altogether generated 23,053 bp sequences, of which 16,910 bp sequences could be directly compared to the assembly for accuracy (<a href="#pone.0030297.s005">Table S2</a>). In this comparison, we found 21 mismatched sites distributed in 5 Sanger reads, three of which had low sequencing quality scores. Furthermore, we PCR-amplified and resequenced 16 randomly chosen regions surrounding putative genome rearrangements in comparison with the bamboo <em>Bambusa oldhamii</em> mt genome (EU365401). All the regions were successfully recovered and only one nucleotide substitution was observed in 10,837 bp resequenced regions (<a href="#pone.0030297.s005">Table S2</a>). In total, we tested 27,747 bp sequences by conventional sequencing, validating the accuracy of sequencing and assembly of our mt genome. Only 22 nucleotide substitution errors were found and the error rate was 0.079%, or 0.018% without accounting the errors associated with low Sanger sequence quality. This rate was close to the 0.037–0.056% error rates with the next-generation sequencings reported before <a href="#pone.0030297-Moore3">[36]</a>, <a href="#pone.0030297-Cronn1">[37]</a>.</p>


<h4>Genome Features</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The assembled sequence amounted to 432,839 bp distributed in 13 scaffolds and one contig. Since no estimated size existed for the <em>F. rimosivaginus</em> mt genome in previous studies, we evaluated the degree of sequence completion by comparing the assembly size to the average size (484,329 bp) of mt genomes from three closely related species <em>B. oldhamii</em>, <em>Oryza sativa</em> (NC_011033) <a href="#pone.0030297-Notsu1">[38]</a> and <em>Triticum aestivum</em> (NC_007579) <a href="#pone.0030297-Ogihara1">[39]</a>. Based on this comparison, we could assume that the <em>F. rimosivaginus</em> mt genome has been largely assembled and 14 gaps of unknown size remained while mt genome was considered as a circular molecule <a href="#pone.0030297-Kubo1">[12]</a>. The contig and scaffold sequences were deposited in GenBank under Accession Numbers JQ235166 to JQ235179.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The largely completed <em>F. rimosivaginus</em> mt genome has a GC content of 44.1% (<a href="#pone-0030297-t003">Table 3</a>), which is close to the median value of fully sequenced angiosperm mt genomes. As in other angiosperms <a href="#pone.0030297-Palmer1">[10]</a>, <a href="#pone.0030297-Kubo1">[12]</a>, most of the <em>F. rimosivaginus</em> mt genome sequences are noncoding sequences. The coding and intron sequences comprise only 8.9% (38,642 bp) and 5.7% (24,730 bp) of the total length, respectively, including 34 protein, 19 tRNA, and 3 rRNA genes (<a href="#pone-0030297-t003">Table 3</a>). These annotated genes are scattered throughout 12 of the 13 assembled scaffolds. Eight protein genes contain 22 groupII introns in all, 6 of which are <em>trans</em>-spliced.</p>
<div class="figure" id="pone-0030297-t003"><div class="img"><a name="pone-0030297-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0030297.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0030297.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Main features of the assembled <em>F. rimosivaginus</em> mitochondrial genome.</span></strong></p><span>doi:10.1371/journal.pone.0030297.t003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The <em>F. rimosivaginus</em> mt genome has nearly the same coding capability as those of other grasses (<a href="#pone-0030297-t004">Table 4</a>). All respiratory genes except for <em>sdh3</em> and <em>sdh4</em> are present in the genome, in agreement with the suggested frequent losses of these two genes during angiosperm evolution <a href="#pone.0030297-Adams1">[11]</a>, <a href="#pone.0030297-Mower2">[40]</a>. The other 15 frequently lost genes are all ribosomal protein coding genes <a href="#pone.0030297-Adams1">[11]</a>, <a href="#pone.0030297-Mower2">[40]</a>, among which five genes <em>rpl2</em>, <em>rps8</em>, <em>rps10</em>, <em>rps11</em>, and <em>rps14</em> are absent or appear to be pseudogenes in the sequenced mt genome. Of this group, the <em>rpl2</em> gene is the most intact with just a single 165 bp insertion compared to annotated <em>rpl2</em> gene in other grass mt genomes. Although the insertion does not alter the downstream reading frame, there is a premature stop codon in the sequence of the <em>rpl2</em> gene and thus it may be not functional. Like other grass mt genomes <a href="#pone.0030297-Ong1">[41]</a>, a nearly full-length <em>rpl14</em> pseudogene is retained in the genome and the open reading frame is disrupted by several frameshift mutations. The other three genes have little or no remnant in the genome. The <em>F. rimosivaginus</em> mt genome does not have the translational capacity to recognize all the 61 sense codons even if assuming that U in the third codon/first anticodon wobble position can recognize all bases in the mtDNA, encoding only 14 of the 20 amino acids. Additionally, nearly half of the identified tRNA genes (9 out of 19) are of cp origin, and one of these genes, <em>trnS</em> (<em>gga</em>), has two copies in the genome. All three ribosomal RNA genes (<em>rrn5</em>, <em>rrn18</em>, and <em>rrn26</em>) are present in the mt genome of <em>F. rimosivaginus</em> as other grasses.</p>
<div class="figure" id="pone-0030297-t004"><div class="img"><a name="pone-0030297-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0030297.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0030297.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Comparison of gene content among grass mitochondrial genomes.</span></strong></p><span>doi:10.1371/journal.pone.0030297.t004</span></div>

<h4>Phylogenetic Analyses</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Phylogenetic analyses were performed on a 22-taxon (<a href="#pone-0030297-t001">Table 1</a>), 31 mt genes of 28,728 aligned nucleotide positions using maximum likelihood (ML) method with 3 different partitioning strategies: unpartition, partitioning the data set by gene or codon position. The same tree topology and similar bootstrap support (BS) values were obtained regardless of the partitioning strategies (<a href="#pone-0030297-g001">Figure 1A</a> and <a href="#pone.0030297.s001">Figure S1</a>). To evaluate the influence of different methods imposed on phylogenetic reconstruction, maximum parsimony (MP) and Bayesian inference (BI) were also used. The BI analysis generated the same phylogenetic relationships within angiosperms as those inferred from ML and all nodes of the tree received a posterior probability of 1.0 (<a href="#pone-0030297-g001">Figure 1B</a>). The MP analysis resulted in a single most parsimonious tree with a length of 9,977, a consistency index (CI) of 0.64 (excluding uninformative characters), and a retention index (RI) of 0.85 (<a href="#pone-0030297-g001">Figure 1C</a>). The MP tree differed from the ML tree only in the placement of <em>Vitis vinifera</em>, and overall BS values were slightly lower than those of the ML tree. Furthermore, there were four nodes in the MP tree only receiving weak support (BS = 59% to 67%), two of which involved in the placement of <em>V. vinifera</em>. Below, we would focus on phylogenetic relationships with an emphasis on the ML topology.</p>
<div class="figure" id="pone-0030297-g001"><div class="img"><a name="pone-0030297-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0030297.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0030297.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Phylogenetic trees of 22 seed plants as determined from mitochondrial and chloroplast genomes.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The ML tree (A), the BI tree (B) and the MP tree (C) based on 31 mitochondrial (mt) genes. The topology of the chloroplast tree (D) is constrained by previous studies <a href="#pone.0030297-Zhang1">[9]</a>, <a href="#pone.0030297-Moore2">[25]</a>, <a href="#pone.0030297-Kellogg1">[42]</a>. Numbers at nodes indicate bootstrap support (BS) values ≥50% or BI posterior probabilities. Branch lengths of the ML tree calculated through RAxML analysis, and correspond to scale bar (in units of substitutions/site).</p>
<span>doi:10.1371/journal.pone.0030297.g001</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The ML topology was well supported and phylogenetic resolutions were increased relative to a previous analysis based on only four mt genes with dense taxon sampling <a href="#pone.0030297-Qiu1">[29]</a>, although the taxon sampling was very different between these two studies and comparing them was not straightforward. Phylogenetic relationships inferred from mtDNA were congruent with those based on cpDNA (<a href="#pone-0030297-g001">Figure 1A and 1D</a>) <a href="#pone.0030297-Zhang1">[9]</a>, <a href="#pone.0030297-Moore1">[24]</a>–<a href="#pone.0030297-BouchenakKhelladi1">[27]</a>, <a href="#pone.0030297-Kellogg1">[42]</a> with the exception of one topological difference. Within Poaceae, the three subfamilies Bambusoideae (<em>B. oldhamii</em> and <em>F. rimosivaginus</em>), Ehrhartoideae (<em>O. sativa</em> and <em>O. rufipogon</em>), and Pooideae (<em>T. aestivum</em>) formed the BEP clade which had a sister relationship to the PACMAD clade (the other five grasses in the tree belong to Panicoideae) in all the previous phylogenetic analyses of the cpDNA sequences <a href="#pone.0030297-Zhang1">[9]</a>, <a href="#pone.0030297-Zhang2">[26]</a>–<a href="#pone.0030297-GrassPhylogenyWorking1">[28]</a>. However, ML analysis of mtDNA sequences did not support the monophyly of the BEP clade but a sister relationship of Ehrhartoideae+Panicoideae with 100% bootstrap support, although this sister relationship only received 63% BS support in the MP tree. Furthermore, two relative long branches (the Ehrhartoideae branch and the Panicoideae branch) separated by the short internode (<a href="#pone-0030297-g001">Figure 1A</a>) implied that this sister relationship might be an artifact of phylogeny reconstruction due to long branch attraction (LBA) <a href="#pone.0030297-Bergsten1">[43]</a>, <a href="#pone.0030297-Soltis1">[44]</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>To detect the LBA artifact, we first performed phylogenetic analyses at the amino acid level of the total 31 genes. The BS value for Ehrhartoideae+Panicoideae decreased dramatically from 100% to 58% in the ML tree (<a href="#pone-0030297-g002">Figure 2A</a>), while BS values for other nodes within Poaceae did not decrease proportionally but instead remained high (BS = 84% to 100%). On the other hand, the BEP clade was weakly supported (BS = 52%) as a monophyletic group in the MP tree (<a href="#pone-0030297-g002">Figure 2B</a>). We further partitioned the 31 genes into fast- and slow-evolving ones according to their substitution rates. The genes with lower substitution rate than the average of all the 31 genes (0.060 substitutions/site) were considered as slow-evolving genes and the rest as fast-evolving ones (see <a href="#s4">Materials and Methods</a> for detail). If Ehrhartoideae+Panicoideae were an LBA artifact, then support for this group would be favored by the partitions of fast-evolving genes, whereas it would be minimized in partitions of slow-evolving genes. Consistent with our hypothesis, analyses of the concatenated 12 fast-evolving genes supported Ehrhartoideae+Panicoideae with 100% BS value in the ML tree (<a href="#pone-0030297-g003">Figure 3A</a>), and BS value for this group in the MP tree increased from 63% based on all the 31 genes to 71% (<a href="#pone-0030297-g003">Figure 3B</a> and <a href="#pone-0030297-g001">Figure 1C</a>). Furthermore, the monophyly of the BEP clade (BS = 32%) was recovered in ML analysis of the concatenated 19 slow-evolving genes (<a href="#pone-0030297-g003">Figure 3C</a>), and the relationships of the BEP clade within Poaceae were not resolved in MP analysis of this data (<a href="#pone-0030297-g003">Figure 3D</a>).</p>
<div class="figure" id="pone-0030297-g002"><div class="img"><a name="pone-0030297-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0030297.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0030297.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Phylogenetic trees of 22 seed plants based on amino acid sequences of 31 mitochondrial genes.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>The ML tree (A) and the MP tree (B). Numbers associated with branches are bootstrap support (BS) values. <em>Oryza</em> and related BS values indicated in bold.</p>
<span>doi:10.1371/journal.pone.0030297.g002</span></div><div class="figure" id="pone-0030297-g003"><div class="img"><a name="pone-0030297-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0030297.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0030297.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Phylogenetic trees of 22 seed plants inferred from different datasets.</span></strong></p><a id="article1.body1.sec2.sec3.fig3.caption1.p1" name="article1.body1.sec2.sec3.fig3.caption1.p1"></a><p>The ML tree (A) and the MP tree (B) inferred from the concatenated 12 fast-evolving mitochondrial (mt) genes. The ML tree (C) and the MP tree (D) inferred from the concatenated 19 slow-evolving mt genes. Numbers associated with branches are bootstrap support (BS) values. <em>Oryza</em> and related BS values indicated in bold.</p>
<span>doi:10.1371/journal.pone.0030297.g003</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Rare genomic changes such as gene losses, arrangements of genes, insertions and deletions of introns are less prone to homoplasy than nucleotide sequences and have become an alternative approach for phylogenetic studies <a href="#pone.0030297-Boore1">[45]</a>, <a href="#pone.0030297-Boore2">[46]</a>. Two gene losses, <em>rpl5</em> and <em>rps19</em>, were restricted to the mt genomes of Panicoideae and pseudogene of <em>rps14</em> was only retained in the mt genomes of the BEP clade (<a href="#pone-0030297-t004">Table 4</a>). These rare genomic changes supported the monophyly of the BEP clade as well.</p>


<h4>Pattern of Rate Change in Grass Mitochondrial Genomes</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The long branch leading to Poaceae implied that the mt genomes of this family may undergo an elevated evolutionary rate (<a href="#pone-0030297-g001">Figure 1A</a>), just like the cp genomes of this family <a href="#pone.0030297-Zhong1">[22]</a>, <a href="#pone.0030297-Chaw1">[31]</a>, <a href="#pone.0030297-Gaut1">[47]</a>. To quantify the evolutionary rate in grass mt genomes, we calculated absolute substitution rates (R) in substitutions per site per billion years (SSB) as described by Parkinson et al. <a href="#pone.0030297-Parkinson1">[18]</a> in mtDNA tree with the placement of <em>Oryza</em> constrained by <a href="#pone-0030297-g001">Figure 1D</a>. The values of R for branches involving Poaceae were calculated by dividing the branch length by the length of time for that branch. Divergence times were based on estimates in previous studies <a href="#pone.0030297-Moore2">[25]</a>, <a href="#pone.0030297-Piperno1">[32]</a>–<a href="#pone.0030297-BouchenakKhelladi2">[34]</a>, with separation of monocots/eudicots and origin of core Poaceae setting at 135 Myr and 65 Myr, respectively (other divergence times in <a href="#pone.0030297.s006">Table S3</a>). Because there was no reliable estimate of divergence times within genera <em>Oryza</em> and <em>Zea</em>, R was averaged among them from the terminal species to the common ancestor of <em>Oryza</em>/<em>Triticum</em> or <em>Zea</em>/<em>Tripsacum</em>.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p><a href="#pone-0030297-g004">Figure 4</a> illustrated that the evolutionary rate of mt genome changed many times during grass evolution. Comparison of the fastest and slowest lineages showed that the rate varied by a factor of 17. The elevation of rate occurred on the common branch of Poaceae after they diverged from eudicots, and the majority of rates after diversification of Poaceae were ~4-fold slower than that on the branch leading to Poaceae. For example, the rate along the lineages from monocots/eudicots separation to <em>O. sativa</em> changed from 0.52 to 0.05 to 0.12 SSB. This pattern of rate change resembled that observed in grass cp genomes <a href="#pone.0030297-Zhong1">[22]</a>. Another notable feature of <a href="#pone-0030297-g004">Figure 4</a> was the lower evolutionary rate in the lineages of Bambusoideae compared to the other three subfamilies in Poaceae. To examine the impact of divergence times on demonstrating rate change, we also applied much older divergence times 212 Myr and 72 Myr used in estimating evolutionary rate in grass cp genomes <a href="#pone.0030297-Zhong1">[22]</a> for monocots/eudicots separation and origin of core Poaceae, respectively. Nevertheless, the same tendency for rate change in mt genome was obtained with these calibration points (<a href="#pone.0030297.s002">Figure S2</a>).</p>
<div class="figure" id="pone-0030297-g004"><div class="img"><a name="pone-0030297-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0030297.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0030297.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Absolute substitution rates during the evolutionary history of grasses.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Values above each branch indicate absolute substitution rate (R) in substitutions per site per billion years (SSB) for that branch. Among them, 0.12±0.01 and 0.11±0.01 above and below the branch to <em>Oryza</em> are the mean values along the lineage from the common ancestor of <em>Oryza</em>/<em>Triticum</em> to <em>O. sativa</em> and <em>O. rufipogon</em>, respectively, while 0.21±0.02 above the branch to <em>Zea</em> is the average value of R along the lineage from the common ancestor of <em>Zea</em>/<em>Tripsacum</em> to three <em>Zea</em> species.</p>
<span>doi:10.1371/journal.pone.0030297.g004</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>To check the rate decrease of mt genome after diversification of Poaceae in more detail, we partitioned the substitutions into synonymous and nonsynonymous ones. The same pattern of rate change as that of total substitutions was observed in both types of substitutions (<a href="#pone.0030297.s003">Figure S3</a>). And nonsynonymous/synonymous rate ratio (ω) on the branch leading to Poaceae was 0.42, indicating overall purifying selection operating on these genes during grass evolution. However, we did not exclude the RNA editing sites of mt genes in calculating ω and the existence of RNA editing sites can bias the estimate of it <a href="#pone.0030297-Lu1">[48]</a>.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>The evolutionary rates above were estimated on the concatenated 31-gene data set. However, different genes of mt genome in the same plant lineage could have various evolutionary rates <a href="#pone.0030297-Mower1">[16]</a>, <a href="#pone.0030297-Sloan2">[19]</a>. To explore the rate change in individual genes, the ratio of R before diversification of Poaceae to that on the line from the common ancestor of Poaceae to <em>O. sativa</em> for each gene was calculated (<a href="#pone-0030297-g005">Figure 5</a>). Among them two genes, <em>nad4L</em> and <em>nad7</em>, had no nucleotide substitutions during the period from Poaceae origin to <em>O. sativa</em> and thus the ratios for them were artificially given the average value 6.46 for all the other genes. All the genes except for <em>nad6</em> showed the same pattern of rate change of elevated rate before diversification of Poaceae with subsequent slow-down after diversification (<a href="#pone-0030297-g005">Figure 5</a>) and this pattern was consistent with that based on analysis of combined genes. Furthermore, the rate change exhibited limit of variation between genes and the majority of ratios had a value around 5.00.</p>
<div class="figure" id="pone-0030297-g005"><div class="img"><a name="pone-0030297-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0030297" data-uri="info:doi/10.1371/journal.pone.0030297.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0030297.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0030297.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0030297.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0030297.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0030297.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Rate change variation among genes in Poaceae.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Each bar represents the ratio of absolute substitution rate (R) along the line leading to Poaceae to that on the line from the common ancestor of grasses to <em>O. sativa</em>.</p>
<span>doi:10.1371/journal.pone.0030297.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Due to their high-throughput and low-cost, next-generation sequencing technologies have greatly improved the approaches for genome sequencing. For angiosperm organelle genomes, however, they have been largely restricted to sequencing of cp genomes until now. Using next-generation sequencing platforms for mt genome sequencing has only recently been explored <a href="#pone.0030297-Fujii1">[6]</a>–<a href="#pone.0030297-Davila1">[8]</a>. Furthermore, only a few angiosperms have had their mt genomes sequenced, and more completed genomes are necessary to study the evolution of angiosperm mt genomes. Here we present a largely completed mt genome from the bamboo <em>F. rimosivaginus</em> mainly based on Illumina sequencing, providing the demonstration of the feasibility for sequencing angiosperm mt genomes with Illumina sequencing technique. With successful sequencing of the mt genomes using Illumina and 454 sequencing technologies <a href="#pone.0030297-Fujii1">[6]</a>–<a href="#pone.0030297-Davila1">[8]</a> it has become evident that high-throughput next-generation sequencers could hold promise for the angiosperm mt genomes sequencing in the near future. The Illumina sequence reads for the other five bamboos in <a href="#pone.0030297-Zhang1">[9]</a> are under analysis and similar results are obtained. This rapid and effective approach for the bamboo <em>F. rimosivaginus</em> mt genome sequencing could be an alternative to the established methods for angiosperm mt genomes sequencing.</p>

<h4>Feasibility of Illumina Sequencing of Angiosperm Mitochondrial Genomes</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Given the highly variable sizes of angiosperm mt genomes <a href="#pone.0030297-Kubo1">[12]</a>, <a href="#pone.0030297-Alverson1">[49]</a>, <a href="#pone.0030297-Hsu1">[50]</a>, we could only deduce that the <em>F. rimosivaginus</em> mt genome were largely completed based on mean size of mt genomes from three closely related grasses. Nevertheless, our assembly appears to be 100% complete with regard to gene content, as the housekeeping genes shared by the reference mt genomes are all identified from our assembly. Additionally, the draft genome is of relatively high quality with a N50 size of 53.1 kb and only 14 gaps remained to be finished to complete the genome. However, it should be noted that the availability of closely related reference genomes in the grass family and presumed less large repeats in bamboo mt genomes <a href="#pone.0030297-Alverson2">[51]</a> may have made it relatively easier to assemble than some other angiosperm mt genomes.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>In compared to angiosperm cp genomes, it would be much difficult to assemble mt genome from Illumina short reads because of larger genome size, more repetitive sequences and frequent genome rearrangements. For assembly of the <em>F. rimosivaginus</em> mt genome, there are three probable reasons that could explain why we have not obtained a single, full-length genome sequence. First, the raw reads just do not represent complete coverage of the genome. The sequencing depth could vary throughout the genome due to GC content or other factors <a href="#pone.0030297-Bentley1">[52]</a>. However, in light of the relatively high 39.5× average sequencing depth, we argue that the raw reads may have covered the whole genome. Second, it is failure to retrieve certain assembled contigs and/or scaffolds whose sequences are actually derived from the <em>F. rimosivaginus</em> mt genome during assembly. However, the availability of several closely related reference genomes and the low threshold used in mapping contigs and scaffolds (see details in <a href="#s4">Materials and Methods</a>) would dramatically reduce this possibility. In fact, about 17.9% of the assembled sequences have no similarity to the sequences of any other sequenced mt genomes of plants as well as the sequences in the NCBI non-redundant nucleotide and protein databases (data not shown). Third, the incorporated sequences from cpDNA of <em>F. rimosivaginus</em> in the mt genome were excluded from the assembly. The reported uptakes of cpDNA sequences by angiosperm mt genomes constitute 1.1–11.5% of the genome size <a href="#pone.0030297-Alverson1">[49]</a>, <a href="#pone.0030297-Goremykin1">[53]</a>. These cp-derived sequences are very likely to affect our assembly and thus we removed assembled sequences with significant sequence identity to the cp genome of <em>F. rimosivaginus</em>. This is a potential disadvantage of our approach for mt genome sequencing. Among three explanations above, the last one may contribute mostly to the incompleteness of our assembled genome. In the future, modifying the method used in DNA extraction to reduce the proportion of cpDNA in the total DNA could solve this problem. Advances in next-generation sequencing technologies, such as increase in length and number of sequence reads (more coverage) and paired-end sequencing with larger insert size, will also improve the assembly.</p>


<h4>Gene Content of the <em>F. rimosivaginus</em> Mitochondrial Genome</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The mt genome of <em>F. rimosivaginus</em> is the second sequenced mt genome from a bamboo species in Poaceae. It encodes roughly the same protein genes as mt genomes of other grasses, with the loss of <em>sdh3</em>, <em>sdh4</em> and several ribosomal protein genes which are all prone to loss during angiosperm evolution <a href="#pone.0030297-Adams1">[11]</a>. An <em>rpl14</em> pseudogene is retained in the genome in good agreement with survival of the pseudogene in grasses reported before <a href="#pone.0030297-Ong1">[41]</a>. Like other grass mt genomes, the <em>F. rimosivaginus</em> mt genome does not encode the full set of tRNAs necessary to recognize all codons, and nearly half of them are cp-derived tRNAs. The mt genome contains the identical gene set for tRNAs of mt origin among the ten sequenced grasses, while their genomes do not share the same tRNAs of cp origin, indicating an ongoing process of acquisition and loss of cp-derived tRNAs.</p>


<h4>Parallel Episodic Evolution of Organelle Genomes</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>At present, phylogenies of angiosperms are essentially reconstructed from the cpDNA sequences. If the cp and mt genomes are both strictly maternally inherited in angiosperms, we would expect them to contain identical phylogenetic signals. As expected, the relationships within our reconstructed mtDNA tree were largely congruent with phylogenetic analyses of the cpDNA sequences <a href="#pone.0030297-Zhang1">[9]</a>, <a href="#pone.0030297-Moore1">[24]</a>–<a href="#pone.0030297-BouchenakKhelladi1">[27]</a>, <a href="#pone.0030297-Kellogg1">[42]</a>. The well-supported topology confirms the utility of mtDNA for phylogenetic reconstruction <a href="#pone.0030297-Qiu1">[29]</a>, <a href="#pone.0030297-Qiu2">[30]</a> and mt phylogenomics may be useful for resolving some difficult angiosperm phylogenies. In spite of overall congruence, the angiosperm mtDNA tree is in obvious contrast to cpDNA tree in the monophyly of the BEP clade. Using more conserved protein sequences of mt genes, the monophyly or significant decrease in support for non-monophyly of the BEP clade was obtained in mtDNA tree. Furthermore, the fast-evolving genes were more favored than the slow-evolving genes in non-monophyly of the BEP clade. Genome-level features of mt genome that are less subjective to homoplasy also support the monophyly of the BEP clade. In summary, these results suggest that the sister relationship of Ehrhartoideae+Panicoideae inferred from mtDNA is most likely an LBA artifact. The fast-evolving genes which have high different evolutionary rates among different lineages within Poaceae may largely contribute to the LBA artifact <a href="#pone.0030297-Lockhart1">[54]</a>, <a href="#pone.0030297-Wu1">[55]</a>. In addition, the taxon sampling in our phylogenetic analyses was very sparse. Thus, LBA due to poor taxon sampling <a href="#pone.0030297-Bergsten1">[43]</a>, <a href="#pone.0030297-Soltis1">[44]</a> may also result in this phylogenetic inconsistency.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Unlike prior studies based on a few mt genes <a href="#pone.0030297-Sloan1">[15]</a>–<a href="#pone.0030297-Sloan2">[19]</a>, we used nearly all the protein genes of mt genome to demonstrate episodic evolution of mt genome in grasses. Moreover, all the examined genes except for <em>nad6</em> had the same pattern of rate change. Although completed mt genomes of other monocots are not available to break the long, poorly sampled branch leading to Poaceae, evolutionary rate averaging along the branch is still faster than those after diversification of Poaceae. However, the range of rate variation is much smaller than those recently documented <a href="#pone.0030297-Mower1">[16]</a>–<a href="#pone.0030297-Sloan2">[19]</a>. And in contrast to rate variations being restricted to synonymous substitutions in these studies, nonsynonymous substitutions in grass mt genomes also show the corresponding rate change. The factors that are responsible for this rate change are very likely to be different from those with locus effect proposed before, such as the efficiency of mtDNA repair. Furthermore, that the episodic evolution is correlated between the mt genome and the cp genome in grasses is consistent with lineage effects. The same underlying factors for the rate change may simultaneously operate on the mt and cp genomes of grasses. However, we could not confirm the rate acceleration of mt genome only occurring in the ancestral grasses like that of cp genome with the present data. Completed mt genomes from taxa closely related to Poaceae are required to resolve this question in further studies.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Mitochondrial DNA Isolation, Genome Sequencing and Assembly</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Total DNA enriched for cpDNA was extracted and sequenced using the protocols described in <a href="#pone.0030297-Zhang1">[9]</a>. The raw short reads were firstly assembled using SOAPdenovo <a href="#pone.0030297-Li1">[35]</a> with <em>K</em> = 31 bp and scaffolding contigs with a minimum size of 100 bp. All the assembled scaffolds and contigs (≥100 bp) were mapped to the sequenced mt genomes in Poaceae (<a href="#pone-0030297-t001">Table 1</a>) using BLASTN searches from NCBI with default parameters. Those contigs and scaffolds whose query coverage was greater than 40% were retrieved and then they were used to search against the NCBI non-redundant nucleotide and protein databases with BLASTN (<a href="http://blast.ncbi.nlm.nih.gov/">http://blast.ncbi.nlm.nih.gov/</a>). These without significant identity to sequences from the cp and/or nuclear genomes were deemed as sequences derived from mtDNA. In this procedure, we removed 3 scaffolds and 35 contigs which could be aligned to reference mt genomes but supposed to be of cpDNA origin as they had significant sequences identity (≥90%) to the mt genomes as well as the <em>F. rimosivaginus</em> cp genome. We realigned all the raw reads onto the assembled sequences using software Bowtie <a href="#pone.0030297-Langmead1">[56]</a> with −v = 3. The aligned paired-end reads were used to determine the sequencing depth. A second round of assembly was carried out on the initially assembled contigs and scaffolds. By using the information of overlap of ≥12 bp between contigs and scaffolds and synteny between assembled and reference genomes, we further joined them into larger scaffolds.</p>


<h4>PCR-Based Genome Finishing and Validation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Following the steps above, the candidate linkages of the contigs and/or scaffolds were validated by PCR analysis. We designed primers according to the nucleotide sequences surrounding the linkage and in all 12 primer pairs were used. To close the gaps inside the scaffolds, 20 primer pairs were also designed. Furthermore, we randomly chose 16 regions in the assembly for resequencing using PCR. All the primer sequences were in <a href="#pone.0030297.s007">Table S4</a>. PCR products were sequenced by ABI 3730xl genetic analyzer using standard protocols. Sanger sequences and the assembly were aligned using MEGA 4.0 <a href="#pone.0030297-Tamura1">[57]</a> to determine if there were any differences.</p>


<h4>Genome Annotation</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A preliminary annotation was carried out by mapping BLASTN hits employing known mt genes of grasses as queries and subsequently, by testing for consistency of the open reading frame. The exact gene and exon boundaries were determined by alignment of homologous genes from <em>B. oldhamii</em>. We also used tRNAscan-SE 1.21 <a href="#pone.0030297-Schattner1">[58]</a> to corroborate tRNA boundaries identified by BLASTN. The sequences of identified tRNA genes were BLAST searched against the cp genome of <em>F. rimosivaginus</em> to detect cp-derived tRNA genes.</p>


<h4>Phylogenetic Analyses</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We extracted nucleotide sequences for all protein genes from 22 seed plant mt genomes (<a href="#pone-0030297-t001">Table 1</a>). After excluding several genes (<em>sdh3</em>, <em>sdh4</em>, <em>rpl2</em>, <em>rpl6</em>, <em>rps1</em>, <em>rps2</em>, <em>rps8</em>, <em>rps10</em>, <em>rps11</em>, <em>rps14</em>, and <em>rps19</em>) that are missing from many of the sequenced genomes, 31 genes were retained. Each gene was aligned separately with MEGA 4.0 constrained by its amino acid sequence. Alignments of nucleotide sequences were then manually adjusted, and ambiguously aligned regions were excluded from the analysis. We did not exclude RNA editing sites as several studies suggested that they were not a problem in phylogenetic reconstruction, especially with large sequence data sets <a href="#pone.0030297-Qiu1">[29]</a>, <a href="#pone.0030297-Qiu2">[30]</a>, <a href="#pone.0030297-Bowe1">[59]</a>, <a href="#pone.0030297-Picardi1">[60]</a>. Finally, individual genes were concatenated, and the resulted alignment consisted of 28,728 nucleotides.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>ML analyses of the 31-gene data set were performed with RAxML 7.0.4 <a href="#pone.0030297-Stamatakis1">[61]</a> using three partitioning strategies: unpartitioned, partitioned the data based on gene region, and partitioned the protein genes by each codon position. All ML analyses used the rapid BS algorithm implemented in RAxML 7.0.4 with 1,000 replicates and the general time-reversible (GTR) model of evolution with among-site rate variation. For BI, we used software MrBayes 3.1.2 <a href="#pone.0030297-Ronquist1">[62]</a> with GTR+G+I model. The run started with a random tree, default priors, and four Markov chains, totalling 1 million generations with sampling trees every 100th generation. When convergence was obtained, a consensus tree was calculated after omitting the first 25% of trees as burn-in. For MP, we used PAUP*4.0b10 <a href="#pone.0030297-Swofford1">[63]</a> to implement heuristic searches that consisted of TBR branch swapping, starting from 1,000 trees built by random taxon stepwise addition, and multrees option in effect. Non-parametric bootstrap analysis was conducted under 200 replicates with TBR branch swapping from 100 random taxon addition staring trees. To detect long branch attraction, phylogenetic trees were constructed from the amino acid sequences of 31 genes with both ML and MP analysis conducted like before. ML analysis of protein sequences was performed with the Dayhoff matrix of amino acid substitutions and a discrete gamma distribution with four rate categories. The substitution rates (in number of substitutions per site) of the 31 genes in the 22 sampled species were calculated by MEGA 4.0 under the model of Kimura 2-Parameter, ranging from 0.027 to 0.125 with a mean of 0.060. Nineteen of the 31 genes (<em>nad1</em>, <em>nad2</em>, <em>nad3</em>, <em>nad4</em>, <em>nad4L</em>, <em>nad5</em>, <em>nad6</em>, <em>nad7</em>, <em>nad9</em>, <em>cob</em>, <em>cox1</em>, <em>cox3</em>, <em>ccmB</em>, <em>ccmC</em>, <em>rpl16</em>, <em>rps7</em>, <em>rps12</em>, <em>rps13</em>, and <em>mttB</em>) with substitution rates lower than the average were concatenated to form the 19 slow-evolving genes data and the rest (<em>cox2</em>, <em>atp1</em>, <em>atp4</em>, <em>atp6</em>, <em>atp8</em>, <em>atp9</em>, <em>ccmFC</em>, <em>ccmFN</em>, <em>rpl5</em>, <em>rps3</em>, <em>rps4</em>, and <em>matR</em>) formed the 12 fast-evolving genes data. ML and MP analyses were also performed on the concatenated 12 fast-evolving genes and 19 slow-evolving genes in the same way.</p>


<h4>Estimation of Absolute Substitution Rates</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Absolute substitution rates were calculated for the Poaceae lineages using method that has been described before <a href="#pone.0030297-Parkinson1">[18]</a>. Briefly, branch lengths that represent the number of substitutions per site were determined for concatenation of 31 genes using codeml in PAML v4.4 <a href="#pone.0030297-Yang1">[64]</a> with topologically constrained tree. Codon frequencies were computed by using F3×4 method and separate ω ratios were estimated for each branch. Relationships within the tree were based on <a href="#pone-0030297-g001">Figure 1D</a>. Divergence times used (<a href="#pone.0030297.s006">Table S3</a>) were taken from previous studies <a href="#pone.0030297-Moore2">[25]</a>, <a href="#pone.0030297-Piperno1">[32]</a>–<a href="#pone.0030297-BouchenakKhelladi2">[34]</a>. Then absolute substitution rate per branch was calculated by dividing the branch length by the length of time for that branch. Standard errors were determined as in <a href="#pone.0030297-Parkinson1">[18]</a>. Absolute substitution rates in terms of nonsynonymous and synonymous substitutions were calculated in the same way. For 30 individual genes (<em>rpl5</em> was excluded for loss in Panicoideae), the absolute substitution rate was analyzed in the same manner as that of concatenated genes and then the ratio of absolute substitution rate before diversification of Poaceae to from origin of the common ancestor of Poaceae to <em>O. sativa</em> was calculated.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0030297"></div><a name="pone.0030297.s001" id="pone.0030297.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0030297.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Phylogenetic trees as determined by RAxML based on the 31 mitochondrial genes, under the following partitioning schemes: A) partitioned by gene; B) partitioned by codon position. Numbers at nodes indicate bootstrap support (BS) values.</p>
<p class="siDoi">doi:10.1371/journal.pone.0030297.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0030297.s002" id="pone.0030297.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0030297.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Rate changes in grass mitochondrial genes during evolution with divergence times 212 Myr and 72 Myr for monocots/eudicots separation and origin of core Poaceae, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0030297.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0030297.s003" id="pone.0030297.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0030297.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Rates of nonsynonymous (A) and synonymous (B) substitutions changes in grass mitochondrial genes during evolution.</p>
<p class="siDoi">doi:10.1371/journal.pone.0030297.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0030297.s004" id="pone.0030297.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0030297.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Lists of assembled initial and final contigs and scaffolds.</p>
<p class="siDoi">doi:10.1371/journal.pone.0030297.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(XLS)</p>
<a name="pone.0030297.s005" id="pone.0030297.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0030297.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Comparison of the assembly and Sanger sequences.</p>
<p class="siDoi">doi:10.1371/journal.pone.0030297.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(XLS)</p>
<a name="pone.0030297.s006" id="pone.0030297.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0030297.s006">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Divergence times used in calculating absolute substitution rates of grass mitochondrial genes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0030297.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(DOC)</p>
<a name="pone.0030297.s007" id="pone.0030297.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0030297.s007">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Information on the primers used for the PCR analyses to validate the linkage between the contigs and/or scaffolds and close intra-scaffold gaps.</p>
<p class="siDoi">doi:10.1371/journal.pone.0030297.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are deeply indebted to Yun-Jie Zhang and Jun-Bo Yang for assistances in experiments, to Drs. Chun-Xia Zeng, Yu-Xiao Zhang, and Jin-Mei Lu for helpful discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: D-ZL. Performed the experiments: P-FM. Analyzed the data: P-FM Z-HG. Contributed reagents/materials/analysis tools: P-FM Z-HG. Wrote the paper: P-FM Z-HG D-ZL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0030297-Neafsey1" id="pone.0030297-Neafsey1"></a>Neafsey DE, Haas BJ (2011) ‘Next-generation’ sequencing becomes ‘now-generation’. Genome Biol  12: 303.  <ul class="find" data-citedArticleID="3737230" data-doi="10.1186/gb-2011-12-3-303"><li><a href="http://dx.doi.org/10.1186/gb-2011-12-3-303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%E2%80%98Next-generation%E2%80%99+sequencing+becomes+%E2%80%98now-generation%E2%80%99." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%E2%80%98Next-generation%E2%80%99+sequencing+becomes+%E2%80%98now-generation%E2%80%99.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0030297-Atherton1" id="pone.0030297-Atherton1"></a>Atherton RA, McComish BJ, Shepherd LD, Berry LA, Albert NW, et al.  (2010) Whole genome sequencing of enriched chloroplast DNA using the Illumina GAII platform. Plant Methods  6: 22.  <ul class="find" data-citedArticleID="3737232" data-doi="10.1186/1746-4811-6-22"><li><a href="http://dx.doi.org/10.1186/1746-4811-6-22" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole+genome+sequencing+of+enriched+chloroplast+DNA+using+the+Illumina+GAII+platform." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole+genome+sequencing+of+enriched+chloroplast+DNA+using+the+Illumina+GAII+platform.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0030297-Nock1" id="pone.0030297-Nock1"></a>Nock CJ, Waters DL, Edwards MA, Bowen SG, Rice N, et al.  (2011) Chloroplast genome sequences from total DNA for plant identification. Plant Biotechnol J  9: 328–333.  <ul class="find" data-citedArticleID="3737234" data-doi="10.1111/j.1467-7652.2010.00558.x"><li><a href="http://dx.doi.org/10.1111/j.1467-7652.2010.00558.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chloroplast+genome+sequences+from+total+DNA+for+plant+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chloroplast+genome+sequences+from+total+DNA+for+plant+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0030297-Wolf1" id="pone.0030297-Wolf1"></a>Wolf PG, Der JP, Duffy AM, Davidson JB, Grusz AL, et al.  (2011) The evolution of chloroplast genes and genomes in ferns. Plant Mol Biol  76: 251–261.  <ul class="find" data-citedArticleID="3737236" data-doi="10.1007/s11103-010-9706-4"><li><a href="http://dx.doi.org/10.1007/s11103-010-9706-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+chloroplast+genes+and+genomes+in+ferns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+chloroplast+genes+and+genomes+in+ferns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0030297-Jansen1" id="pone.0030297-Jansen1"></a>Jansen RK, Raubeson LA, Boore JL, dePamphilis CW, Chumley TW, et al.  (2005) Methods for obtaining and analyzing whole chloroplast genome sequences. Methods Enzymol  395: 348–384.  <ul class="find" data-citedArticleID="3737238" data-doi="10.1016/s0076-6879(05)95020-9"><li><a href="http://dx.doi.org/10.1016/s0076-6879(05)95020-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methods+for+obtaining+and+analyzing+whole+chloroplast+genome+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methods+for+obtaining+and+analyzing+whole+chloroplast+genome+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0030297-Fujii1" id="pone.0030297-Fujii1"></a>Fujii S, Kazama T, Yamada M, Toriyama K (2010) Discovery of global genomic re-organization based on comparison of two newly sequenced rice mitochondrial genomes with cytoplasmic male sterility-related genes. BMC Genomics  11: 209.  <ul class="find" data-citedArticleID="3737240" data-doi="10.1186/1471-2164-11-209"><li><a href="http://dx.doi.org/10.1186/1471-2164-11-209" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovery+of+global+genomic+re-organization+based+on+comparison+of+two+newly+sequenced+rice+mitochondrial+genomes+with+cytoplasmic+male+sterility-related+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovery+of+global+genomic+re-organization+based+on+comparison+of+two+newly+sequenced+rice+mitochondrial+genomes+with+cytoplasmic+male+sterility-related+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0030297-RodriguezMoreno1" id="pone.0030297-RodriguezMoreno1"></a>Rodriguez-Moreno L, Gonzalez V, Benjak A, Marti MC, Puigdomenech P, et al.  (2011) Determination of the melon chloroplast and mitochondrial genome sequences reveals that the largest reported mitochondrial genome in plants contains a significant amount of DNA having a nuclear origin. BMC Genomics  12: 424.  <ul class="find" data-citedArticleID="3737242" data-doi="10.1186/1471-2164-12-424"><li><a href="http://dx.doi.org/10.1186/1471-2164-12-424" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determination+of+the+melon+chloroplast+and+mitochondrial+genome+sequences+reveals+that+the+largest+reported+mitochondrial+genome+in+plants+contains+a+significant+amount+of+DNA+having+a+nuclear+origin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determination+of+the+melon+chloroplast+and+mitochondrial+genome+sequences+reveals+that+the+largest+reported+mitochondrial+genome+in+plants+contains+a+significant+amount+of+DNA+having+a+nuclear+origin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0030297-Davila1" id="pone.0030297-Davila1"></a>Davila JI, Arrieta-Montiel MP, Wamboldt Y, Cao J, Hagmann J, et al.  (2011) Double-strand break repair processes drive evolution of the mitochondrial genome in Arabidopsis. BMC Biol  9: 64.  <ul class="find" data-citedArticleID="3737244" data-doi="10.1186/1741-7007-9-64"><li><a href="http://dx.doi.org/10.1186/1741-7007-9-64" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Double-strand+break+repair+processes+drive+evolution+of+the+mitochondrial+genome+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Double-strand+break+repair+processes+drive+evolution+of+the+mitochondrial+genome+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0030297-Zhang1" id="pone.0030297-Zhang1"></a>Zhang YJ, Ma MF, Li DZ (2011) High-throughput sequencing of six bamboo chloroplast genomes: phylogenetic implications for temperate woody bamboos (Poaceae: Bambusoideae). PLoS ONE  6: e20596.  <ul class="find" data-citedArticleID="3737246" data-doi="10.1371/journal.pone.0020596"><li><a href="http://dx.doi.org/10.1371/journal.pone.0020596" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+sequencing+of+six+bamboo+chloroplast+genomes%3A+phylogenetic+implications+for+temperate+woody+bamboos+%28Poaceae%3A+Bambusoideae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+sequencing+of+six+bamboo+chloroplast+genomes%3A+phylogenetic+implications+for+temperate+woody+bamboos+%28Poaceae%3A+Bambusoideae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0030297-Palmer1" id="pone.0030297-Palmer1"></a>Palmer JD, Adams KL, Cho Y, Parkinson CL, Qiu YL, et al.  (2000) Dynamic evolution of plant mitochondrial genomes: mobile genes and introns and highly variable mutation rates. Proc Natl Acad Sci U S A  97: 6960–6966.  <ul class="find" data-citedArticleID="3737248" data-doi="10.1073/pnas.97.13.6960"><li><a href="http://dx.doi.org/10.1073/pnas.97.13.6960" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+evolution+of+plant+mitochondrial+genomes%3A+mobile+genes+and+introns+and+highly+variable+mutation+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+evolution+of+plant+mitochondrial+genomes%3A+mobile+genes+and+introns+and+highly+variable+mutation+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0030297-Adams1" id="pone.0030297-Adams1"></a>Adams KL, Qiu YL, Stoutemyer M, Palmer JD (2002) Punctuated evolution of mitochondrial gene content: high and variable rates of mitochondrial gene loss and transfer to the nucleus during angiosperm evolution. Proc Natl Acad Sci U S A  99: 9905–9912.  <ul class="find" data-citedArticleID="3737250" data-doi="10.1073/pnas.042694899"><li><a href="http://dx.doi.org/10.1073/pnas.042694899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Punctuated+evolution+of+mitochondrial+gene+content%3A+high+and+variable+rates+of+mitochondrial+gene+loss+and+transfer+to+the+nucleus+during+angiosperm+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Punctuated+evolution+of+mitochondrial+gene+content%3A+high+and+variable+rates+of+mitochondrial+gene+loss+and+transfer+to+the+nucleus+during+angiosperm+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0030297-Kubo1" id="pone.0030297-Kubo1"></a>Kubo T, Newton KJ (2008) Angiosperm mitochondrial genomes and mutations. Mitochondrion  8: 5–14.  <ul class="find" data-citedArticleID="3737252" data-doi="10.1016/j.mito.2007.10.006"><li><a href="http://dx.doi.org/10.1016/j.mito.2007.10.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Angiosperm+mitochondrial+genomes+and+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Angiosperm+mitochondrial+genomes+and+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0030297-Palmer2" id="pone.0030297-Palmer2"></a>Palmer JD, Herbon LA (1988) Plant mitochondrial DNA evolves rapidly in structure, but slowly in sequence. J Mol Evol  28: 87–97.  <ul class="find" data-citedArticleID="3737254" data-doi="10.1007/bf02143500"><li><a href="http://dx.doi.org/10.1007/bf02143500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plant+mitochondrial+DNA+evolves+rapidly+in+structure%2C+but+slowly+in+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plant+mitochondrial+DNA+evolves+rapidly+in+structure%2C+but+slowly+in+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0030297-Wolfe1" id="pone.0030297-Wolfe1"></a>Wolfe KH, Li WH, Sharp PM (1987) Rates of nucleotide substitution vary greatly among plant mitochondrial, chloroplast, and nuclear DNAs. Proc Natl Acad Sci U S A  84: 9054–9058.  <ul class="find" data-citedArticleID="3737256" data-doi="10.1073/pnas.84.24.9054"><li><a href="http://dx.doi.org/10.1073/pnas.84.24.9054" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+nucleotide+substitution+vary+greatly+among+plant+mitochondrial%2C+chloroplast%2C+and+nuclear+DNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+nucleotide+substitution+vary+greatly+among+plant+mitochondrial%2C+chloroplast%2C+and+nuclear+DNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0030297-Sloan1" id="pone.0030297-Sloan1"></a>Sloan DB, Barr CM, Olson MS, Keller SR, Taylor DR (2008) Evolutionary rate variation at multiple levels of biological organization in plant mitochondrial DNA. Mol Biol Evol  25: 243.  <ul class="find" data-citedArticleID="3737258" data-doi="10.1093/molbev/msm266"><li><a href="http://dx.doi.org/10.1093/molbev/msm266" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+rate+variation+at+multiple+levels+of+biological+organization+in+plant+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+rate+variation+at+multiple+levels+of+biological+organization+in+plant+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0030297-Mower1" id="pone.0030297-Mower1"></a>Mower JP, Touzet P, Gummow JS, Delph LF, Palmer JD (2007) Extensive variation in synonymous substitution rates in mitochondrial genes of seed plants. BMC Evol Biol  7: 135.  <ul class="find" data-citedArticleID="3737260" data-doi="10.1186/1471-2148-7-135"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+variation+in+synonymous+substitution+rates+in+mitochondrial+genes+of+seed+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+variation+in+synonymous+substitution+rates+in+mitochondrial+genes+of+seed+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0030297-Cho1" id="pone.0030297-Cho1"></a>Cho Y, Mower JP, Qiu YL, Palmer JD (2004) Mitochondrial substitution rates are extraordinarily elevated and variable in a genus of flowering plants. Proc Natl Acad Sci U S A  101: 17741–17746.  <ul class="find" data-citedArticleID="3737262" data-doi="10.1073/pnas.0408302101"><li><a href="http://dx.doi.org/10.1073/pnas.0408302101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+substitution+rates+are+extraordinarily+elevated+and+variable+in+a+genus+of+flowering+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+substitution+rates+are+extraordinarily+elevated+and+variable+in+a+genus+of+flowering+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0030297-Parkinson1" id="pone.0030297-Parkinson1"></a>Parkinson CL, Mower JP, Qiu YL, Shirk AJ, Song K, et al.  (2005) Multiple major increases and decreases in mitochondrial substitution rates in the plant family Geraniaceae. BMC Evol Biol  5: 73.  <ul class="find" data-citedArticleID="3737264"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multiple+major+increases+and+decreases+in+mitochondrial+substitution+rates+in+the+plant+family+Geraniaceae.&amp;auth=&amp;atitle=Multiple+major+increases+and+decreases+in+mitochondrial+substitution+rates+in+the+plant+family+Geraniaceae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+major+increases+and+decreases+in+mitochondrial+substitution+rates+in+the+plant+family+Geraniaceae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+major+increases+and+decreases+in+mitochondrial+substitution+rates+in+the+plant+family+Geraniaceae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0030297-Sloan2" id="pone.0030297-Sloan2"></a>Sloan DB, Oxelman B, Rautenberg A, Taylor DR (2009) Phylogenetic analysis of mitochondrial substitution rate variation in the angiosperm tribe Sileneae. BMC Evol Biol  9: 260.  <ul class="find" data-citedArticleID="3737266" data-doi="10.1186/1471-2148-9-260"><li><a href="http://dx.doi.org/10.1186/1471-2148-9-260" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analysis+of+mitochondrial+substitution+rate+variation+in+the+angiosperm+tribe+Sileneae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analysis+of+mitochondrial+substitution+rate+variation+in+the+angiosperm+tribe+Sileneae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0030297-SoriaHernanz1" id="pone.0030297-SoriaHernanz1"></a>Soria-Hernanz DF, Braverman JM, Hamilton MB (2008) Parallel rate heterogeneity in chloroplast and mitochondrial genomes of Brazil nut trees (Lecythidaceae) is consistent with lineage effects. Mol Biol Evol  25: 1282–1296.  <ul class="find" data-citedArticleID="3737268" data-doi="10.1093/molbev/msn074"><li><a href="http://dx.doi.org/10.1093/molbev/msn074" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+rate+heterogeneity+in+chloroplast+and+mitochondrial+genomes+of+Brazil+nut+trees+%28Lecythidaceae%29+is+consistent+with+lineage+effects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+rate+heterogeneity+in+chloroplast+and+mitochondrial+genomes+of+Brazil+nut+trees+%28Lecythidaceae%29+is+consistent+with+lineage+effects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0030297-EyreWalker1" id="pone.0030297-EyreWalker1"></a>Eyre-Walker A, Gaut BS (1997) Correlated rates of synonymous site evolution across plant genomes. Mol Biol Evol  14: 455–460.  <ul class="find" data-citedArticleID="3737270" data-doi="10.1093/oxfordjournals.molbev.a025781"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025781" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlated+rates+of+synonymous+site+evolution+across+plant+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlated+rates+of+synonymous+site+evolution+across+plant+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0030297-Zhong1" id="pone.0030297-Zhong1"></a>Zhong B, Yonezawa T, Zhong Y, Hasegawa M (2009) Episodic evolution and adaptation of chloroplast genomes in ancestral grasses. PLoS One  4: e5297.  <ul class="find" data-citedArticleID="3737272" data-doi="10.1371/journal.pone.0005297"><li><a href="http://dx.doi.org/10.1371/journal.pone.0005297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Episodic+evolution+and+adaptation+of+chloroplast+genomes+in+ancestral+grasses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Episodic+evolution+and+adaptation+of+chloroplast+genomes+in+ancestral+grasses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0030297-Jansen2" id="pone.0030297-Jansen2"></a>Jansen RK, Cai Z, Raubeson LA, Daniell H, Depamphilis CW, et al.  (2007) Analysis of 81 genes from 64 plastid genomes resolves relationships in angiosperms and identifies genome-scale evolutionary patterns. Proc Natl Acad Sci U S A  104: 19369–19374.  <ul class="find" data-citedArticleID="3737274" data-doi="10.1073/pnas.0709121104"><li><a href="http://dx.doi.org/10.1073/pnas.0709121104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+81+genes+from+64+plastid+genomes+resolves+relationships+in+angiosperms+and+identifies+genome-scale+evolutionary+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+81+genes+from+64+plastid+genomes+resolves+relationships+in+angiosperms+and+identifies+genome-scale+evolutionary+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0030297-Moore1" id="pone.0030297-Moore1"></a>Moore MJ, Bell CD, Soltis PS, Soltis DE (2007) Using plastid genome-scale data to resolve enigmatic relationships among basal angiosperms. Proc Natl Acad Sci U S A  104: 19363–19368.  <ul class="find" data-citedArticleID="3737276" data-doi="10.1073/pnas.0708072104"><li><a href="http://dx.doi.org/10.1073/pnas.0708072104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+plastid+genome-scale+data+to+resolve+enigmatic+relationships+among+basal+angiosperms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+plastid+genome-scale+data+to+resolve+enigmatic+relationships+among+basal+angiosperms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0030297-Moore2" id="pone.0030297-Moore2"></a>Moore MJ, Soltis PS, Bell CD, Burleigh JG, Soltis DE (2010) Phylogenetic analysis of 83 plastid genes further resolves the early diversification of eudicots. Proc Natl Acad Sci U S A  107: 4623–4628.  <ul class="find" data-citedArticleID="3737278" data-doi="10.1073/pnas.0907801107"><li><a href="http://dx.doi.org/10.1073/pnas.0907801107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analysis+of+83+plastid+genes+further+resolves+the+early+diversification+of+eudicots." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analysis+of+83+plastid+genes+further+resolves+the+early+diversification+of+eudicots.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0030297-Zhang2" id="pone.0030297-Zhang2"></a>Zhang W (2000) Phylogeny of the grass family (Poaceae) from <em>rpl16</em> intron sequence data. Mol Phylogenet Evol  15: 135–146.  <ul class="find" data-citedArticleID="3737280" data-doi="10.1006/mpev.1999.0729"><li><a href="http://dx.doi.org/10.1006/mpev.1999.0729" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+the+grass+family+%28Poaceae%29+from+rpl16+intron+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+the+grass+family+%28Poaceae%29+from+rpl16+intron+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0030297-BouchenakKhelladi1" id="pone.0030297-BouchenakKhelladi1"></a>Bouchenak-Khelladi Y, Salamin N, Savolainen V, Forest F, Bank M, et al.  (2008) Large multi-gene phylogenetic trees of the grasses (Poaceae): progress towards complete tribal and generic level sampling. Mol Phylogenet Evol  47: 488–505.  <ul class="find" data-citedArticleID="3737282" data-doi="10.1016/j.ympev.2008.01.035"><li><a href="http://dx.doi.org/10.1016/j.ympev.2008.01.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large+multi-gene+phylogenetic+trees+of+the+grasses+%28Poaceae%29%3A+progress+towards+complete+tribal+and+generic+level+sampling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large+multi-gene+phylogenetic+trees+of+the+grasses+%28Poaceae%29%3A+progress+towards+complete+tribal+and+generic+level+sampling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0030297-GrassPhylogenyWorking1" id="pone.0030297-GrassPhylogenyWorking1"></a>Grass Phylogeny Working G (2001) Phylogeny and subfamilial classification of the grasses (Poaceae). Ann Mo Bot Gard  88: 373–457.  <ul class="find" data-citedArticleID="3737284" data-doi="10.2307/3298585"><li><a href="http://dx.doi.org/10.2307/3298585" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+and+subfamilial+classification+of+the+grasses+%28Poaceae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+and+subfamilial+classification+of+the+grasses+%28Poaceae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0030297-Qiu1" id="pone.0030297-Qiu1"></a>Qiu YL, Li L, Wang B, Xue JY, Hendry TA, et al.  (2010) Angiosperm phylogeny inferred from sequences of four mitochondrial genes. J Syst Evol  48: 391–425.  <ul class="find" data-citedArticleID="3737286" data-doi="10.1111/j.1759-6831.2010.00097.x"><li><a href="http://dx.doi.org/10.1111/j.1759-6831.2010.00097.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Angiosperm+phylogeny+inferred+from+sequences+of+four+mitochondrial+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Angiosperm+phylogeny+inferred+from+sequences+of+four+mitochondrial+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0030297-Qiu2" id="pone.0030297-Qiu2"></a>Qiu YL, Li L, Hendry TA, Li R, Taylor DW, et al.  (2006) Reconstructing the basal angiosperm phylogeny: evaluating information content of mitochondrial genes. Taxon  55: 837–856.  <ul class="find" data-citedArticleID="3737288" data-doi="10.2307/25065680"><li><a href="http://dx.doi.org/10.2307/25065680" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+the+basal+angiosperm+phylogeny%3A+evaluating+information+content+of+mitochondrial+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+the+basal+angiosperm+phylogeny%3A+evaluating+information+content+of+mitochondrial+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0030297-Chaw1" id="pone.0030297-Chaw1"></a>Chaw SM, Chang CC, Chen HL, Li WH (2004) Dating the monocot-dicot divergence and the origin of core eudicots using whole chloroplast genomes. J Mol Evol  58: 424–441.  <ul class="find" data-citedArticleID="3737290" data-doi="10.1007/s00239-003-2564-9"><li><a href="http://dx.doi.org/10.1007/s00239-003-2564-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dating+the+monocot-dicot+divergence+and+the+origin+of+core+eudicots+using+whole+chloroplast+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dating+the+monocot-dicot+divergence+and+the+origin+of+core+eudicots+using+whole+chloroplast+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0030297-Piperno1" id="pone.0030297-Piperno1"></a>Piperno DR, Sues HD (2005) Paleontology. Dinosaurs dined on grass. Science  310: 1126–1128.  <ul class="find" data-citedArticleID="3737292" data-doi="10.1126/science.1121020"><li><a href="http://dx.doi.org/10.1126/science.1121020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paleontology.+Dinosaurs+dined+on+grass." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paleontology.+Dinosaurs+dined+on+grass.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0030297-Vicentini1" id="pone.0030297-Vicentini1"></a>Vicentini A, Barber JC, Aliscioni SS, Giussani LM, Kellogg EA (2008) The age of the grasses and clusters of origins of C4 photosynthesis. Global Change Biol  14: 2963–2977.  <ul class="find" data-citedArticleID="3737294" data-doi="10.1111/j.1365-2486.2008.01688.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2486.2008.01688.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+age+of+the+grasses+and+clusters+of+origins+of+C4+photosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+age+of+the+grasses+and+clusters+of+origins+of+C4+photosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0030297-BouchenakKhelladi2" id="pone.0030297-BouchenakKhelladi2"></a>Bouchenak-Khelladi Y, Verboom GA, Savolainen V, Hodkinson TR (2010) Biogeography of the grasses (Poaceae): a phylogenetic approach to reveal evolutionary history in geographical space and geological time. Bot J Linn Soc  162: 543–557.  <ul class="find" data-citedArticleID="3737296" data-doi="10.1111/j.1095-8339.2010.01041.x"><li><a href="http://dx.doi.org/10.1111/j.1095-8339.2010.01041.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biogeography+of+the+grasses+%28Poaceae%29%3A+a+phylogenetic+approach+to+reveal+evolutionary+history+in+geographical+space+and+geological+time." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biogeography+of+the+grasses+%28Poaceae%29%3A+a+phylogenetic+approach+to+reveal+evolutionary+history+in+geographical+space+and+geological+time.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0030297-Li1" id="pone.0030297-Li1"></a>Li R, Zhu H, Ruan J, Qian W, Fang X, et al.  (2010) <em>De novo</em> assembly of human genomes with massively parallel short read sequencing. Genome Res  20: 265–272.  <ul class="find" data-citedArticleID="3737298" data-doi="10.1101/gr.097261.109"><li><a href="http://dx.doi.org/10.1101/gr.097261.109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=De+novo+assembly+of+human+genomes+with+massively+parallel+short+read+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22De+novo+assembly+of+human+genomes+with+massively+parallel+short+read+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0030297-Moore3" id="pone.0030297-Moore3"></a>Moore MJ, Dhingra A, Soltis PS, Shaw R, Farmerie WG, et al.  (2006) Rapid and accurate pyrosequencing of angiosperm plastid genomes. BMC Plant Biol  6: 17.  <ul class="find" data-citedArticleID="3737300" data-doi="10.1186/1471-2229-6-17"><li><a href="http://dx.doi.org/10.1186/1471-2229-6-17" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+and+accurate+pyrosequencing+of+angiosperm+plastid+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+and+accurate+pyrosequencing+of+angiosperm+plastid+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0030297-Cronn1" id="pone.0030297-Cronn1"></a>Cronn R, Liston A, Parks M, Gernandt DS, Shen R, et al.  (2008) Multiplex sequencing of plant chloroplast genomes using Solexa sequencing-by-synthesis technology. Nucleic Acids Res  36: e122.  <ul class="find" data-citedArticleID="3737302" data-doi="10.1093/nar/gkn502"><li><a href="http://dx.doi.org/10.1093/nar/gkn502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+sequencing+of+plant+chloroplast+genomes+using+Solexa+sequencing-by-synthesis+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+sequencing+of+plant+chloroplast+genomes+using+Solexa+sequencing-by-synthesis+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0030297-Notsu1" id="pone.0030297-Notsu1"></a>Notsu Y, Masood S, Nishikawa T, Kubo N, Akiduki G, et al.  (2002) The complete sequence of the rice (<em>Oryza sativa</em> L.) mitochondrial genome: frequent DNA sequence acquisition and loss during the evolution of flowering plants. Mol Genet Genomics  268: 434–445.  <ul class="find" data-citedArticleID="3737304" data-doi="10.1007/s00438-002-0767-1"><li><a href="http://dx.doi.org/10.1007/s00438-002-0767-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+sequence+of+the+rice+%28Oryza+sativa+L.%29+mitochondrial+genome%3A+frequent+DNA+sequence+acquisition+and+loss+during+the+evolution+of+flowering+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+sequence+of+the+rice+%28Oryza+sativa+L.%29+mitochondrial+genome%3A+frequent+DNA+sequence+acquisition+and+loss+during+the+evolution+of+flowering+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0030297-Ogihara1" id="pone.0030297-Ogihara1"></a>Ogihara Y, Yamazaki Y, Murai K, Kanno A, Terachi T, et al.  (2005) Structural dynamics of cereal mitochondrial genomes as revealed by complete nucleotide sequencing of the wheat mitochondrial genome. Nucleic Acids Res  33: 6235–6250.  <ul class="find" data-citedArticleID="3737306" data-doi="10.1093/nar/gki925"><li><a href="http://dx.doi.org/10.1093/nar/gki925" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+dynamics+of+cereal+mitochondrial+genomes+as+revealed+by+complete+nucleotide+sequencing+of+the+wheat+mitochondrial+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+dynamics+of+cereal+mitochondrial+genomes+as+revealed+by+complete+nucleotide+sequencing+of+the+wheat+mitochondrial+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0030297-Mower2" id="pone.0030297-Mower2"></a>Mower JP, Bonen L (2009) Ribosomal protein L10 is encoded in the mitochondrial genome of many land plants and green algae. BMC Evol Biol  9: 265.  <ul class="find" data-citedArticleID="3737308" data-doi="10.1186/1471-2148-9-265"><li><a href="http://dx.doi.org/10.1186/1471-2148-9-265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+protein+L10+is+encoded+in+the+mitochondrial+genome+of+many+land+plants+and+green+algae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+protein+L10+is+encoded+in+the+mitochondrial+genome+of+many+land+plants+and+green+algae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0030297-Ong1" id="pone.0030297-Ong1"></a>Ong HC, Palmer JD (2006) Pervasive survival of expressed mitochondrial <em>rps14</em> pseudogenes in grasses and their relatives for 80 million years following three functional transfers to the nucleus. BMC Evol Biol  6: 55.  <ul class="find" data-citedArticleID="3737310"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Pervasive+survival+of+expressed+mitochondrial+rps14+pseudogenes+in+grasses+and+their+relatives+for+80+million+years+following+three+functional+transfers+to+the+nucleus.&amp;auth=&amp;atitle=Pervasive+survival+of+expressed+mitochondrial+rps14+pseudogenes+in+grasses+and+their+relatives+for+80+million+years+following+three+functional+transfers+to+the+nucleus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pervasive+survival+of+expressed+mitochondrial+rps14+pseudogenes+in+grasses+and+their+relatives+for+80+million+years+following+three+functional+transfers+to+the+nucleus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pervasive+survival+of+expressed+mitochondrial+rps14+pseudogenes+in+grasses+and+their+relatives+for+80+million+years+following+three+functional+transfers+to+the+nucleus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0030297-Kellogg1" id="pone.0030297-Kellogg1"></a>Kellogg EA, Birchler JA (1993) Linking phylogeny and genetics: <em>Zea mays</em> as a tool for phylogenetic studies. Syst Biol  42: 415.  <ul class="find" data-citedArticleID="3737312"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Linking+phylogeny+and+genetics%3A+Zea+mays+as+a+tool+for+phylogenetic+studies.&amp;auth=&amp;atitle=Linking+phylogeny+and+genetics%3A+Zea+mays+as+a+tool+for+phylogenetic+studies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linking+phylogeny+and+genetics%3A+Zea+mays+as+a+tool+for+phylogenetic+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linking+phylogeny+and+genetics%3A+Zea+mays+as+a+tool+for+phylogenetic+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0030297-Bergsten1" id="pone.0030297-Bergsten1"></a>Bergsten J (2005) A review of long-branch attraction. Cladistics  21: 163–193.  <ul class="find" data-citedArticleID="3737314" data-doi="10.1111/j.1096-0031.2005.00059.x"><li><a href="http://dx.doi.org/10.1111/j.1096-0031.2005.00059.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+review+of+long-branch+attraction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+review+of+long-branch+attraction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0030297-Soltis1" id="pone.0030297-Soltis1"></a>Soltis DE, Albert VA, Savolainen V, Hilu K, Qiu YL, et al.  (2004) Genome-scale data, angiosperm relationships, and ‘ending incongruence’: a cautionary tale in phylogenetics. Trends Plant Sci  9: 477–483.  <ul class="find" data-citedArticleID="3737316" data-doi="10.1016/j.tplants.2004.08.008"><li><a href="http://dx.doi.org/10.1016/j.tplants.2004.08.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-scale+data%2C+angiosperm+relationships%2C+and+%E2%80%98ending+incongruence%E2%80%99%3A+a+cautionary+tale+in+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-scale+data%2C+angiosperm+relationships%2C+and+%E2%80%98ending+incongruence%E2%80%99%3A+a+cautionary+tale+in+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0030297-Boore1" id="pone.0030297-Boore1"></a>Boore JL, Fuerstenberg SI (2008) Beyond linear sequence comparisons: the use of genome-level characters for phylogenetic reconstruction. Philos Trans R Soc Lond B Biol Sci  363: 1445–1451.  <ul class="find" data-citedArticleID="3737318" data-doi="10.1098/rstb.2007.2234"><li><a href="http://dx.doi.org/10.1098/rstb.2007.2234" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beyond+linear+sequence+comparisons%3A+the+use+of+genome-level+characters+for+phylogenetic+reconstruction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Beyond+linear+sequence+comparisons%3A+the+use+of+genome-level+characters+for+phylogenetic+reconstruction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0030297-Boore2" id="pone.0030297-Boore2"></a>Boore JL (2006) The use of genome-level characters for phylogenetic reconstruction. Trends Ecol Evol  21: 439–446.  <ul class="find" data-citedArticleID="3737320" data-doi="10.1016/j.tree.2006.05.009"><li><a href="http://dx.doi.org/10.1016/j.tree.2006.05.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+genome-level+characters+for+phylogenetic+reconstruction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+genome-level+characters+for+phylogenetic+reconstruction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0030297-Gaut1" id="pone.0030297-Gaut1"></a>Gaut BS, Muse SV, Clark WD, Clegg MT (1992) Relative rates of nucleotide substitution at the <em>rbcL</em> locus of monocotyledonous plants. J Mol Evol  35: 292–303.  <ul class="find" data-citedArticleID="3737322" data-doi="10.1007/bf00161167"><li><a href="http://dx.doi.org/10.1007/bf00161167" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relative+rates+of+nucleotide+substitution+at+the+rbcL+locus+of+monocotyledonous+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relative+rates+of+nucleotide+substitution+at+the+rbcL+locus+of+monocotyledonous+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0030297-Lu1" id="pone.0030297-Lu1"></a>Lu MZ, Szmidt AE, Wang XR (1998) RNA editing in gymnosperms and its impact on the evolution of the mitochondrial <em>coxI</em> gene. Plant Mol Biol  37: 225–234.  <ul class="find" data-citedArticleID="3737324"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=RNA+editing+in+gymnosperms+and+its+impact+on+the+evolution+of+the+mitochondrial+coxI+gene.&amp;auth=&amp;atitle=RNA+editing+in+gymnosperms+and+its+impact+on+the+evolution+of+the+mitochondrial+coxI+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+editing+in+gymnosperms+and+its+impact+on+the+evolution+of+the+mitochondrial+coxI+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+editing+in+gymnosperms+and+its+impact+on+the+evolution+of+the+mitochondrial+coxI+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0030297-Alverson1" id="pone.0030297-Alverson1"></a>Alverson AJ, Wei X, Rice DW, Stern DB, Barry K, et al.  (2010) Insights into the evolution of mitochondrial genome size from complete sequences of <em>Citrullus lanatus</em> and <em>Cucurbita pepo</em> (Cucurbitaceae). Mol Biol Evol  27: 1436–1448.  <ul class="find" data-citedArticleID="3737326" data-doi="10.1093/molbev/msq029"><li><a href="http://dx.doi.org/10.1093/molbev/msq029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+the+evolution+of+mitochondrial+genome+size+from+complete+sequences+of+Citrullus+lanatus+and+Cucurbita+pepo+%28Cucurbitaceae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+the+evolution+of+mitochondrial+genome+size+from+complete+sequences+of+Citrullus+lanatus+and+Cucurbita+pepo+%28Cucurbitaceae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0030297-Hsu1" id="pone.0030297-Hsu1"></a>Hsu CL, Mullin BC (1989) Physical characterization of mitochondrial DNA from cotton. Plant Mol Biol  13: 467–468.  <ul class="find" data-citedArticleID="3737328" data-doi="10.1007/bf00015558"><li><a href="http://dx.doi.org/10.1007/bf00015558" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physical+characterization+of+mitochondrial+DNA+from+cotton." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physical+characterization+of+mitochondrial+DNA+from+cotton.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0030297-Alverson2" id="pone.0030297-Alverson2"></a>Alverson AJ, Zhuo S, Rice DW, Sloan DB, Palmer JD (2011) The mitochondrial genome of the legume <em>Vigna radiata</em> and the analysis of recombination across short mitochondrial repeats. PLoS One  6: e16404.  <ul class="find" data-citedArticleID="3737330" data-doi="10.1371/journal.pone.0016404"><li><a href="http://dx.doi.org/10.1371/journal.pone.0016404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mitochondrial+genome+of+the+legume+Vigna+radiata+and+the+analysis+of+recombination+across+short+mitochondrial+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mitochondrial+genome+of+the+legume+Vigna+radiata+and+the+analysis+of+recombination+across+short+mitochondrial+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0030297-Bentley1" id="pone.0030297-Bentley1"></a>Bentley DR, Balasubramanian S, Swerdlow HP, Smith GP, Milton J, et al.  (2008) Accurate whole human genome sequencing using reversible terminator chemistry. Nature  456: 53–59.  <ul class="find" data-citedArticleID="3737332"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Accurate+whole+human+genome+sequencing+using+reversible+terminator+chemistry.&amp;auth=&amp;atitle=Accurate+whole+human+genome+sequencing+using+reversible+terminator+chemistry." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+whole+human+genome+sequencing+using+reversible+terminator+chemistry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+whole+human+genome+sequencing+using+reversible+terminator+chemistry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0030297-Goremykin1" id="pone.0030297-Goremykin1"></a>Goremykin VV, Salamini F, Velasco R, Viola R (2009) Mitochondrial DNA of <em>Vitis vinifera</em> and the issue of rampant horizontal gene transfer. Mol Biol Evol  26: 99–110.  <ul class="find" data-citedArticleID="3737334" data-doi="10.1093/molbev/msn226"><li><a href="http://dx.doi.org/10.1093/molbev/msn226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+of+Vitis+vinifera+and+the+issue+of+rampant+horizontal+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+of+Vitis+vinifera+and+the+issue+of+rampant+horizontal+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0030297-Lockhart1" id="pone.0030297-Lockhart1"></a>Lockhart P, Novis P, Milligan BG, Riden J, Rambau A, et al.  (2006) Heterotachy and tree building: A case study with plastids and eubacteria. Mol Biol Evol  23: 40–45.  <ul class="find" data-citedArticleID="3737336" data-doi="10.1093/molbev/msj005"><li><a href="http://dx.doi.org/10.1093/molbev/msj005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterotachy+and+tree+building%3A+A+case+study+with+plastids+and+eubacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterotachy+and+tree+building%3A+A+case+study+with+plastids+and+eubacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0030297-Wu1" id="pone.0030297-Wu1"></a>Wu CS, Wang YN, Hsu CY, Lin CP, Chaw SM (2011) Loss of different inverted repeat copies from the chloroplast genomes of Pinaceae and Cupressophytes and influence of heterotachy on the evaluation of gemnosperm phylogeny. Genome Biol Evol.   <ul class="find" data-citedArticleID="3737338" data-doi="10.1093/gbe/evr095"><li><a href="http://dx.doi.org/10.1093/gbe/evr095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Loss+of+different+inverted+repeat+copies+from+the+chloroplast+genomes+of+Pinaceae+and+Cupressophytes+and+influence+of+heterotachy+on+the+evaluation+of+gemnosperm+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Loss+of+different+inverted+repeat+copies+from+the+chloroplast+genomes+of+Pinaceae+and+Cupressophytes+and+influence+of+heterotachy+on+the+evaluation+of+gemnosperm+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0030297-Langmead1" id="pone.0030297-Langmead1"></a>Langmead B, Trapnell C, Pop M, Salzberg SL (2009) Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol  10: R25.  <ul class="find" data-citedArticleID="3737340" data-doi="10.1186/gb-2009-10-3-r25"><li><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0030297-Tamura1" id="pone.0030297-Tamura1"></a>Tamura K, Dudley J, Nei M, Kumar S (2007) MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0. Mol Biol Evol  24: 1596–1599.  <ul class="find" data-citedArticleID="3737342" data-doi="10.1093/molbev/msm092"><li><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA4%3A+Molecular+Evolutionary+Genetics+Analysis+%28MEGA%29+software+version+4.0." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA4%3A+Molecular+Evolutionary+Genetics+Analysis+%28MEGA%29+software+version+4.0.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0030297-Schattner1" id="pone.0030297-Schattner1"></a>Schattner P, Brooks AN, Lowe TM (2005) The tRNAscan-SE, snoscan and snoGPS web servers for the detection of tRNAs and snoRNAs. Nucleic Acids Res  33: W686–689.  <ul class="find" data-citedArticleID="3737344" data-doi="10.1093/nar/gki366"><li><a href="http://dx.doi.org/10.1093/nar/gki366" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tRNAscan-SE%2C+snoscan+and+snoGPS+web+servers+for+the+detection+of+tRNAs+and+snoRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tRNAscan-SE%2C+snoscan+and+snoGPS+web+servers+for+the+detection+of+tRNAs+and+snoRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0030297-Bowe1" id="pone.0030297-Bowe1"></a>Bowe LM, dePamphilis CW (1996) Effects of RNA editing and gene processing on phylogenetic reconstruction. Mol Biol Evol  13: 1159–1166.  <ul class="find" data-citedArticleID="3737346" data-doi="10.1093/oxfordjournals.molbev.a025680"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025680" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+RNA+editing+and+gene+processing+on+phylogenetic+reconstruction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+RNA+editing+and+gene+processing+on+phylogenetic+reconstruction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0030297-Picardi1" id="pone.0030297-Picardi1"></a>Picardi E, Quagliariello C (2008) Is plant mitochondrial RNA editing a source of phylogenetic incongruence? An answer from in silico and in vivo data sets. BMC Bioinformatics  9: S14.  <ul class="find" data-citedArticleID="3737348" data-doi="10.1186/1471-2105-9-s2-s14"><li><a href="http://dx.doi.org/10.1186/1471-2105-9-s2-s14" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+plant+mitochondrial+RNA+editing+a+source+of+phylogenetic+incongruence%3F+An+answer+from+in+silico+and+in+vivo+data+sets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+plant+mitochondrial+RNA+editing+a+source+of+phylogenetic+incongruence%3F+An+answer+from+in+silico+and+in+vivo+data+sets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0030297-Stamatakis1" id="pone.0030297-Stamatakis1"></a>Stamatakis A (2006) RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models. Bioinformatics  22: 2688–2690.  <ul class="find" data-citedArticleID="3737350" data-doi="10.1093/bioinformatics/btl446"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl446" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RAxML-VI-HPC%3A+maximum+likelihood-based+phylogenetic+analyses+with+thousands+of+taxa+and+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RAxML-VI-HPC%3A+maximum+likelihood-based+phylogenetic+analyses+with+thousands+of+taxa+and+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0030297-Ronquist1" id="pone.0030297-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="3737352" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0030297-Swofford1" id="pone.0030297-Swofford1"></a>Swofford D (2002)   PAUP* 4.0: phylogenetic analysis using parsimony: Sinauer Associates, Sunderland, Mass.  <ul class="find-nolinks"></ul></li><li><span class="label">64.
              </span><a name="pone.0030297-Yang1" id="pone.0030297-Yang1"></a>Yang Z (2007) PAML 4: phylogenetic analysis by maximum likelihood. Mol Biol Evol  24: 1586–1591.  <ul class="find" data-citedArticleID="3737356" data-doi="10.1093/molbev/msm088"><li><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML+4%3A+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML+4%3A+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0030297-Chaw2" id="pone.0030297-Chaw2"></a>Chaw SM, Shih AC, Wang D, Wu YW, Liu SM, et al.  (2008) The mitochondrial genome of the gymnosperm <em>Cycas taitungensis</em> contains a novel family of short interspersed elements, Bpu sequences, and abundant RNA editing sites. Mol Biol Evol  25: 603–615.  <ul class="find" data-citedArticleID="3737358" data-doi="10.1093/molbev/msn009"><li><a href="http://dx.doi.org/10.1093/molbev/msn009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mitochondrial+genome+of+the+gymnosperm+Cycas+taitungensis+contains+a+novel+family+of+short+interspersed+elements%2C+Bpu+sequences%2C+and+abundant+RNA+editing+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mitochondrial+genome+of+the+gymnosperm+Cycas+taitungensis+contains+a+novel+family+of+short+interspersed+elements%2C+Bpu+sequences%2C+and+abundant+RNA+editing+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0030297-Clifton1" id="pone.0030297-Clifton1"></a>Clifton SW, Minx P, Fauron CM, Gibson M, Allen JO, et al.  (2004) Sequence and comparative analysis of the maize NB mitochondrial genome. Plant Physiol  136: 3486–3503.  <ul class="find" data-citedArticleID="3737360" data-doi="10.1104/pp.104.044602"><li><a href="http://dx.doi.org/10.1104/pp.104.044602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+comparative+analysis+of+the+maize+NB+mitochondrial+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+comparative+analysis+of+the+maize+NB+mitochondrial+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0030297-Unseld1" id="pone.0030297-Unseld1"></a>Unseld M, Marienfeld JR, Brandt P, Brennicke A (1997) The mitochondrial genome of <em>Arabidopsis thaliana</em> contains 57 genes in 366,924 nucleotides. Nat Genet  15: 57–61.  <ul class="find" data-citedArticleID="3737362" data-doi="10.1038/ng0197-57"><li><a href="http://dx.doi.org/10.1038/ng0197-57" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mitochondrial+genome+of+Arabidopsis+thaliana+contains+57+genes+in+366%2C924+nucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mitochondrial+genome+of+Arabidopsis+thaliana+contains+57+genes+in+366%2C924+nucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0030297-Kubo2" id="pone.0030297-Kubo2"></a>Kubo T, Nishizawa S, Sugawara A, Itchoda N, Estiati A, et al.  (2000) The complete nucleotide sequence of the mitochondrial genome of sugar beet (<em>Beta vulgaris</em> L.) reveals a novel gene for <em>tRNA(Cys)(GCA)</em>. Nucleic Acids Res  28: 2571–2576.  <ul class="find" data-citedArticleID="3737364" data-doi="10.1093/nar/28.13.2571"><li><a href="http://dx.doi.org/10.1093/nar/28.13.2571" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+nucleotide+sequence+of+the+mitochondrial+genome+of+sugar+beet+%28Beta+vulgaris+L.%29+reveals+a+novel+gene+for+tRNA%28Cys%29%28GCA%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+nucleotide+sequence+of+the+mitochondrial+genome+of+sugar+beet+%28Beta+vulgaris+L.%29+reveals+a+novel+gene+for+tRNA%28Cys%29%28GCA%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0030297-Handa1" id="pone.0030297-Handa1"></a>Handa H (2003) The complete nucleotide sequence and RNA editing content of the mitochondrial genome of rapeseed (<em>Brassica napus</em> L.): comparative analysis of the mitochondrial genomes of rapeseed and <em>Arabidopsis thaliana</em>. Nucleic Acids Res  31: 5907–5916.  <ul class="find" data-citedArticleID="3737366" data-doi="10.1093/nar/gkg795"><li><a href="http://dx.doi.org/10.1093/nar/gkg795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+nucleotide+sequence+and+RNA+editing+content+of+the+mitochondrial+genome+of+rapeseed+%28Brassica+napus+L.%29%3A+comparative+analysis+of+the+mitochondrial+genomes+of+rapeseed+and+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+nucleotide+sequence+and+RNA+editing+content+of+the+mitochondrial+genome+of+rapeseed+%28Brassica+napus+L.%29%3A+comparative+analysis+of+the+mitochondrial+genomes+of+rapeseed+and+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0030297-Sugiyama1" id="pone.0030297-Sugiyama1"></a>Sugiyama Y, Watase Y, Nagase M, Makita N, Yagura S, et al.  (2005) The complete nucleotide sequence and multipartite organization of the tobacco mitochondrial genome: comparative analysis of mitochondrial genomes in higher plants. Mol Genet Genomics  272: 603–615.  <ul class="find" data-citedArticleID="3737368" data-doi="10.1007/s00438-004-1075-8"><li><a href="http://dx.doi.org/10.1007/s00438-004-1075-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+nucleotide+sequence+and+multipartite+organization+of+the+tobacco+mitochondrial+genome%3A+comparative+analysis+of+mitochondrial+genomes+in+higher+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+nucleotide+sequence+and+multipartite+organization+of+the+tobacco+mitochondrial+genome%3A+comparative+analysis+of+mitochondrial+genomes+in+higher+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0030297-Sloan3" id="pone.0030297-Sloan3"></a>Sloan DB, Alverson AJ, Storchova H, Palmer JD, Taylor DR, et al.  (2010) Extensive loss of translational genes in the structurally dynamic mitochondrial genome of the angiosperm <em>Silene latifolia</em>. BMC Evol Biol  10: 274.  <ul class="find" data-citedArticleID="3737370" data-doi="10.1186/1471-2148-10-274"><li><a href="http://dx.doi.org/10.1186/1471-2148-10-274" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+loss+of+translational+genes+in+the+structurally+dynamic+mitochondrial+genome+of+the+angiosperm+Silene+latifolia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+loss+of+translational+genes+in+the+structurally+dynamic+mitochondrial+genome+of+the+angiosperm+Silene+latifolia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.XML" value="182934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.PDF" value="760314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t001.PNG_L" value="110557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t001.PNG_M" value="43359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t001.PNG_S" value="3035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t001.TIF" value="454534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t001.PNG_I" value="14578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t002.PNG_L" value="43827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t002.PNG_M" value="50716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t002.PNG_S" value="3773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t002.TIF" value="120010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t002.PNG_I" value="18596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t003.PNG_L" value="25591"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t003.PNG_M" value="29084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t003.PNG_S" value="2625"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t003.TIF" value="75490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t003.PNG_I" value="10905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t004.PNG_L" value="111622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t004.PNG_M" value="32109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t004.PNG_S" value="3904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t004.TIF" value="695735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.t004.PNG_I" value="16962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g001.PNG_L" value="283079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g001.PNG_M" value="66384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g001.PNG_S" value="14693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g001.TIF" value="563674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g001.PNG_I" value="37803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g002.PNG_L" value="153693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g002.PNG_M" value="47371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g002.PNG_S" value="11485"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g002.TIF" value="370748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g002.PNG_I" value="14856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g003.PNG_L" value="294837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g003.PNG_M" value="66474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g003.PNG_S" value="15035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g003.TIF" value="578510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g003.PNG_I" value="37072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g004.PNG_L" value="47945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g004.PNG_M" value="32463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g004.PNG_S" value="9969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g004.TIF" value="67328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g004.PNG_I" value="9890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g005.PNG_L" value="39095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g005.PNG_M" value="36124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g005.PNG_S" value="9265"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g005.TIF" value="355462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.g005.PNG_I" value="12073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.s001.TIF" value="343065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.s002.TIF" value="392028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.s003.TIF" value="333256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.s004.XLS" value="20480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.s005.XLS" value="19968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.s006.DOC" value="30208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0030297.s007.XLS" value="28160"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0030297&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0030297" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0030297&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0030297&volume=&issue=&title=Rapid Sequencing of the Bamboo Mitochondrial Genome Using Illumina Technology and Parallel Episodic Evolution of Organelle Genomes in Grasses&author_name=Peng-Fei%20Ma%2C%20Zhen-Hua%20Guo%2C%20De-Zhu%20Li&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0030297" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0030297" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0030297" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0030297&amp;t=Rapid%20Sequencing%20of%20the%20Bamboo%20Mitochondrial%20Genome%20Using%20Illumina%20Technology%20and%20Parallel%20Episodic%20Evolution%20of%20Organelle%20Genomes%20in%20Grasses" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0030297&title=Rapid%20Sequencing%20of%20the%20Bamboo%20Mitochondrial%20Genome%20Using%20Illumina%20Technology%20and%20Parallel%20Episodic%20Evolution%20of%20Organelle%20Genomes%20in%20Grasses&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0030297&amp;title=Rapid%20Sequencing%20of%20the%20Bamboo%20Mitochondrial%20Genome%20Using%20Illumina%20Technology%20and%20Parallel%20Episodic%20Evolution%20of%20Organelle%20Genomes%20in%20Grasses" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0030297" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0030297&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Rapid Sequencing of the Bamboo Mitochondrial Genome Using Illumina Technology and Parallel Episodic Evolution of Organelle Genomes in Grasses';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0030297';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Rapid%20Sequencing%20of%20the%20Bamboo%20Mitochondrial%20Genome%20Using%20Illumina%20Technology%20and%20Parallel%20Episodic%20Evolution%20of%20Organelle%20Genomes%20in%20Grasses http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0030297" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0030297" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Flowering+plants%22" title="Search for articles in the subject area:'Flowering plants'"><div class="flagText">Flowering plants</div></a>
              <div data-categoryid="43015" data-articleid="82716"
                   data-categoryname="Flowering plants"
                   class="flagImage" title="Flag 'Flowering plants' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="82716"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="82716"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Grasses%22" title="Search for articles in the subject area:'Grasses'"><div class="flagText">Grasses</div></a>
              <div data-categoryid="43049" data-articleid="82716"
                   data-categoryname="Grasses"
                   class="flagImage" title="Flag 'Grasses' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="82716"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="82716"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Poaceae%22" title="Search for articles in the subject area:'Poaceae'"><div class="flagText">Poaceae</div></a>
              <div data-categoryid="44811" data-articleid="82716"
                   data-categoryname="Poaceae"
                   class="flagImage" title="Flag 'Poaceae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="82716"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7697'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=928'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1815&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
