

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Cryptic Contamination and Phylogenetic Nonsense</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0002316"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0002316" />

    <meta name="citation_title" content="Cryptic Contamination and Phylogenetic Nonsense"/>
    <meta itemprop="name" content="Cryptic Contamination and Phylogenetic Nonsense"/>

      <meta name="citation_author" content="Anna Linderholm"/>
            <meta name="citation_author_institution" content="Archaeological Research laboratory, Stockholm University, Stockholm, Sweden"/>
      <meta name="citation_author" content="Helena Malmström"/>
            <meta name="citation_author_institution" content="Department of Evolutionary Biology, Uppsala University, Uppsala, Sweden"/>
      <meta name="citation_author" content="Kerstin Lidén"/>
            <meta name="citation_author_institution" content="Archaeological Research laboratory, Stockholm University, Stockholm, Sweden"/>
      <meta name="citation_author" content="Gunilla Holmlund"/>
            <meta name="citation_author_institution" content="Department of Forensic Genetics and Forensic Toxicology, National Board of Forensic Medicine, Linköping, Sweden"/>
      <meta name="citation_author" content="Anders Götherström"/>
            <meta name="citation_author_institution" content="Department of Evolutionary Biology, Uppsala University, Uppsala, Sweden"/>

    <meta name="citation_date" content="2008/5/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0002316.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e2316"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Molecular cloning of Ancient Egyptian mummy DNA.; citation_author=S Pääbo; citation_journal_title=Nature; citation_volume=314; citation_number=1; citation_pages=644-645; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Ancient DNA from the first European farmers in 7500-year-old Neolithic sites.; citation_author=W Haak; citation_author=P Forster; citation_author=B Bramanti; citation_author=S Matsumura; citation_author=G Brandt; citation_journal_title=Science; citation_volume=310; citation_number=2; citation_pages=1016-1018; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ancient DNA.; citation_author=M Hofreiter; citation_author=D Serre; citation_author=HN Poinar; citation_author=M Kuch; citation_author=S Pääbo; citation_journal_title=Nat Rev Genet; citation_volume=2; citation_number=3; citation_pages=353-359; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Authenticating DNA Extracted From Ancient Skeletal Remains.; citation_author=MB Richards; citation_author=BC Sykes; citation_author=REM Hedges; citation_journal_title=Journal of Archaeological Science; citation_volume=22; citation_number=4; citation_pages=291-299; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Extensive human DNA contamination in extracts from ancient dog bones and teeth.; citation_author=H Malmström; citation_author=J Storå; citation_author=L Dalén; citation_author=G Holmlund; citation_author=A Götherström; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=5; citation_pages=2040-2047; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Tracking down human contamination in ancient human teeth.; citation_author=ML Sampietro; citation_author=MT Gilbert; citation_author=O Lao; citation_author=D Caramelli; citation_author=M Lari; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=6; citation_pages=1801-1807; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular genetic analyses of the Tyrolean Ice Man.; citation_author=O Handt; citation_author=M Richards; citation_author=M Trommsdorff; citation_author=C Kilger; citation_author=J Simanainen; citation_journal_title=Science; citation_volume=264; citation_number=7; citation_pages=1775-1778; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=More on contamination: the use of asymmetric molecular behavior to identify authentic ancient human DNA.; citation_author=H Malmström; citation_author=EM Svensson; citation_author=MT Gilbert; citation_author=E Willerslev; citation_author=A Götherström; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=8; citation_pages=998-1004; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Inconsistencies in Neanderthal Genomic DNA Sequences.; citation_author=JD Wall; citation_author=SK Kim; citation_journal_title=PLoS Genet; citation_volume=3; citation_number=9; citation_pages=e175; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Ancient DNA: do it right or not at all.; citation_author=A Cooper; citation_author=HN Poinar; citation_journal_title=Science; citation_volume=289; citation_number=10; citation_pages=1139; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Anthropologists cast doubt on human DNA evidence.; citation_author=A Abbott; citation_journal_title=Nature; citation_volume=423; citation_number=11; citation_pages=468; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evidence for a genetic discontinuity between Neandertals and 24,000-year-old anatomically modern Europeans.; citation_author=D Caramelli; citation_author=C Lalueza-Fox; citation_author=C Vernesi; citation_author=M Lari; citation_author=A Casoli; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=12; citation_pages=6593-6597; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DNA sequence from Cretaceous period bone fragments.; citation_author=SR Woodward; citation_author=NJ Weyand; citation_author=M Bunnell; citation_journal_title=Science; citation_volume=266; citation_number=13; citation_pages=1229-1232; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Detecting dinosaur DNA.; citation_author=H Zischler; citation_author=M Höss; citation_author=O Handt; citation_author=A von Haeseler; citation_author=AC van der Kuyl; citation_journal_title=Science; citation_volume=268; citation_number=14; citation_pages=1192-1193; author reply 1194; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA sequences in ancient Australians: Implications for modern human origins.; citation_author=GJ Adcock; citation_author=ES Dennis; citation_author=S Easteal; citation_author=GA Huttley; citation_author=LS Jermiin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=15; citation_pages=537-542; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Absence of the lactase-persistence-associated allele in early Neolithic Europeans.; citation_author=J Burger; citation_author=M Kirchner; citation_author=B Bramanti; citation_author=W Haak; citation_author=MG Thomas; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=16; citation_pages=3736-3741; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evidence of ancient DNA reveals the first European lineage in Iron Age Central China.; citation_author=CZ Xie; citation_author=CX Li; citation_author=YQ Cui; citation_author=QC Zhang; citation_author=YQ Fu; citation_journal_title=Proc Biol Sci; citation_volume=274; citation_number=17; citation_pages=1597-1601; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Lactase persistence and ovarian carcinoma risk in Finland, Poland and Sweden.; citation_author=M Kuokkanen; citation_author=R Butzow; citation_author=H Rasinpera; citation_author=K Medrek; citation_author=M Nilbert; citation_journal_title=Int J Cancer; citation_volume=117; citation_number=18; citation_pages=90-94; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Technical note: improved DNA extraction from ancient bones using silica-based spin columns.; citation_author=DY Yang; citation_author=B Eng; citation_author=JS Waye; citation_author=JC Dudar; citation_author=SR Saunders; citation_journal_title=Am J Phys Anthropol; citation_volume=105; citation_number=19; citation_pages=539-543; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Tracing genetic change over time using nuclear SNPs in ancient and modern cattle.; citation_author=EM Svensson; citation_author=C Anderung; citation_author=J Baubliene; citation_author=P Persson; citation_author=H Malmström; citation_journal_title=Anim Genet; citation_volume=38; citation_number=20; citation_pages=378-383; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Pyrosequencing sheds light on DNA sequencing.; citation_author=M Ronaghi; citation_journal_title=Genome Res; citation_volume=11; citation_number=21; citation_pages=3-11; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A sequencing method based on real-time pyrophosphate.; citation_author=M Ronaghi; citation_author=M Uhlen; citation_author=P Nyren; citation_journal_title=Science; citation_volume=281; citation_number=22; citation_pages=363, 365; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Analysis of one million base pairs of Neanderthal DNA.; citation_author=RE Green; citation_author=J Krause; citation_author=SE Ptak; citation_author=AW Briggs; citation_author=MT Ronan; citation_journal_title=Nature; citation_volume=444; citation_number=23; citation_pages=330-336; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Animal DNA in PCR reagents plagues ancient DNA research.; citation_author=JA Leonard; citation_author=O Shanks; citation_author=M Hofreiter; citation_author=E Kreuz; citation_author=L Hodges; citation_journal_title=Journal of Archaeological Science; citation_volume=34; citation_number=24; citation_pages=1361-1366; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0002316" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Cryptic Contamination and Phylogenetic Nonsense"/>
    <meta name="twitter:description" content="Ancient human DNA has been treated cautiously ever since the problems related to this type of material were exposed in the early 1990s, but as sequential genetic data from ancient specimens have been key components in several evolutionary and ecological studies, interest in ancient human DNA is on the increase again. It is especially tempting to approach archaeological and anthropological questions through this type of material, but DNA from ancient human tissue is notoriously complicated to work with due to the risk of contamination with modern human DNA. Various ways of authenticating results based on ancient human DNA have been developed to circumvent the problems. One commonly used method is to predict what the contamination is expected to look like and then test whether the ancient human DNA fulfils this prediction. If it does, the results are rejected as contamination, while if it does not, they are often considered authentic. We show here that human contamination in ancient material may well deviate from local allele frequencies or the distributions to be found among the laboratory workers and archaeologists. We conclude that it is not reliable to authenticate ancient human DNA solely by showing that it is different from what would be expected from people who have handled the material."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0002316.g003"/>

  <meta property="og:title" content="Cryptic Contamination and Phylogenetic Nonsense" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0002316" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2275'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2523'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8311&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0002316">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0002316" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0002316&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Cryptic Contamination and Phylogenetic Nonsense
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anna Linderholm
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:anna.linderholm@arklab.su.se">anna.linderholm@arklab.su.se</a></p>

                <p>Affiliation:
                  Archaeological Research laboratory, Stockholm University, Stockholm, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Helena Malmström, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Evolutionary Biology, Uppsala University, Uppsala, Sweden
                </p>

                <p>
                    Current address: Centre for Ancient Genetics, Niels Bohr Institute and Biological Institute, University of Copenhagen, Copenhagen, Denmark
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kerstin Lidén, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Archaeological Research laboratory, Stockholm University, Stockholm, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gunilla Holmlund, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Forensic Genetics and Forensic Toxicology, National Board of Forensic Medicine, Linköping, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anders Götherström
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Evolutionary Biology, Uppsala University, Uppsala, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 28, 2008</li>
    <li>DOI: 10.1371/journal.pone.0002316</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0002316" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0002316" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0002316" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0002316" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0002316-g001" data-doi="info:doi/10.1371/journal.pone.0002316" data-uri="info:doi/10.1371/journal.pone.0002316.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0002316-g002" data-doi="info:doi/10.1371/journal.pone.0002316" data-uri="info:doi/10.1371/journal.pone.0002316.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0002316-t001" data-doi="info:doi/10.1371/journal.pone.0002316" data-uri="info:doi/10.1371/journal.pone.0002316.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0002316-t002" data-doi="info:doi/10.1371/journal.pone.0002316" data-uri="info:doi/10.1371/journal.pone.0002316.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0002316-g003" data-doi="info:doi/10.1371/journal.pone.0002316" data-uri="info:doi/10.1371/journal.pone.0002316.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0002316">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0002316" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Ancient human DNA has been treated cautiously ever since the problems related to this type of material were exposed in the early 1990s, but as sequential genetic data from ancient specimens have been key components in several evolutionary and ecological studies, interest in ancient human DNA is on the increase again. It is especially tempting to approach archaeological and anthropological questions through this type of material, but DNA from ancient human tissue is notoriously complicated to work with due to the risk of contamination with modern human DNA. Various ways of authenticating results based on ancient human DNA have been developed to circumvent the problems. One commonly used method is to predict what the contamination is expected to look like and then test whether the ancient human DNA fulfils this prediction. If it does, the results are rejected as contamination, while if it does not, they are often considered authentic. We show here that human contamination in ancient material may well deviate from local allele frequencies or the distributions to be found among the laboratory workers and archaeologists. We conclude that it is not reliable to authenticate ancient human DNA solely by showing that it is different from what would be expected from people who have handled the material.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Linderholm A, Malmström H, Lidén K, Holmlund G, Götherström A (2008) Cryptic Contamination and Phylogenetic Nonsense. PLoS ONE 3(5):
          e2316.
            doi:10.1371/journal.pone.0002316</p><p><strong>Editor: </strong>Neil John Gemmell, University of Canterbury, New Zealand</p><p><strong>Received:</strong> March 31, 2008; <strong>Accepted:</strong> April 22, 2008; <strong>Published:</strong> May 28, 2008</p><p><strong>Copyright:</strong> © 2008 Linderholm et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>New technologies for working with ancient DNA have increased knowledge and explanatory power in several disciplines bordering on evolution and genetics, and the addressing of anthropological issues through ancient DNA has aroused especial interest. It should be noted that the first major study exploring the power of ancient DNA was concerned with human remains <a href="#pone.0002316-Pbo1">[1]</a>, and that studies of ancient human DNA are still attracting a lot of attention after more than 20 years <a href="#pone.0002316-Haak1">[2]</a>. Ancient human DNA is challenging to work with, however. This was already recognized a decade ago when it was found that modern contamination is present to a high extent in reagents and materials <a href="#pone.0002316-Hofreiter1">[3]</a>, <a href="#pone.0002316-Richards1">[4]</a>. Even more alarming is the evidence that there may be higher concentrations of contaminating DNA than of authentic ancient DNA in specimens from museums and collections <a href="#pone.0002316-Malmstrm1">[5]</a>. Furthermore, the contaminating DNA appears to degrade in a fashion similar to ancient DNA, making it hard to use damage patterns to discriminate between the two <a href="#pone.0002316-Sampietro1">[6]</a>. The only quantifiable difference to emerge to date is the level of DNA fragmentation <a href="#pone.0002316-Handt1">[7]</a>, <a href="#pone.0002316-Malmstrm2">[8]</a>, which has proved useful when authenticating DNA from ancient hominids <a href="#pone.0002316-Wall1">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A list of tests and criteria aimed at establishing the authenticity of ancient DNA results was published in 2000 <a href="#pone.0002316-Cooper1">[10]</a>, and these criteria are often applied in ancient DNA studies. DNA results based on ancient human remains remain controversial in spite of meeting these criteria, however, the best illustration perhaps being the discussion that followed the publication of DNA sequences for the Italian Cro-Magnon remains <a href="#pone.0002316-Abbott1">[11]</a>, <a href="#pone.0002316-Caramelli1">[12]</a>. Thus, one criterion in particular has frequently been used in recent studies of DNA from ancient humans, that stating that sequences generated from the ancient specimens should “make phylogenetic sense”. This criterion was developed to authenticate DNA results obtained from non-human remains, i.e. it makes phylogenetic sense when a DNA sequence from a mammoth clusters with sequences from elephants rather than with human sequences. Initially this criterion was used to test the authenticity of an alleged dinosaur sequence <a href="#pone.0002316-Woodward1">[13]</a> which proved to be a human sequence <a href="#pone.0002316-Zischler1">[14]</a>. This criterion is nevertheless applied in an inverse fashion when studying DNA from ancient human remains. Haplotype and genotype frequencies in the geographical area where the remains are processed are assessed initially and all the laboratory technicians and archaeologists involved are also typed. The ancient specimens are then analysed and the data generated are accepted as being authentic if they deviate significantly from that initial background assessment <a href="#pone.0002316-Haak1">[2]</a>, <a href="#pone.0002316-Adcock1">[15]</a>, <a href="#pone.0002316-Burger1">[16]</a>, <a href="#pone.0002316-Xie1">[17]</a>. It should be pointed out that this mode of authentication is applicable only if every possible source of contamination can be accounted for.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To test this criterion, we selected a genetic marker that has a clear distribution pattern in Sweden. In a modern Swedish population 74% are carriers of a mutation that makes it possible to drink milk as an adult <a href="#pone.0002316-Kuokkanen1">[18]</a>. This high frequency should make it possible to trace any contamination arriving from the surroundings and the people who have handled the material. Next we selected some prehistoric skeletal material from Sweden to study the occurrence of the contamination often found in ancient DNA analyses. We extracted and typed a set of Swedish Neolithic human remains and negative controls for a genetic marker with globally varying allele frequencies in three laboratories and investigated whether the deviating results from the laboratories could be explained by contamination with local alleles or whether there were alternative explanations for the contamination.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The human material originated from two Swedish archaeological sites, Ajvide on the Baltic island of Gotland, and Gökhem a mainland site in southern Sweden (<a href="#pone-0002316-g001">Fig. 1</a>). Both sites represent the middle Neolithic, spanning from 3300 to 2500 BC.</p>
<div class="figure" id="pone-0002316-g001"><div class="img"><a name="pone-0002316-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0002316" data-uri="info:doi/10.1371/journal.pone.0002316.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0002316.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0002316.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Map of Southern and Central Sweden showing the two archaeological sites referred to, 1.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Gökhem, 2. Ajvide.</p>
<span>doi:10.1371/journal.pone.0002316.g001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>Initially we had access to 74 Neolithic human remains and 29 associated non-human specimens for which we already had ancient mitochondrial DNA data. These samples had been decontaminated using various techniques such as bleach pre-treatment prior to extraction of the bone powder and authenticated with real-time PCR quantification (mtDNA fragment of 80bp nt4542-4621, including the primer annealing sites). The measures followed a previously published protocol <a href="#pone.0002316-Malmstrm2">[8]</a> in which the samples were cleaned with bleach, HCl and water, after which the outer surface was removed. About 100 mg of powder from each sample was further soaked in a 0.5% bleach solution for 15 minutes and then washed three times with water prior to the commencement of extraction. The water used was from different companies in all three laboratories, and the extraction method was a modification of silica spin column extraction <a href="#pone.0002316-Yang1">[19]</a>, <a href="#pone.0002316-Svensson1">[20]</a>. DNA was eluted in 90 to 100 µl aliquots of water or TE buffer.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>We selected seven of the 74 human samples that had yielded significantly more human mitochondrial DNA (&gt;1264 copies/5 µl extract) than the non-human samples (&lt;197 copies/5 µl extract) (<a href="#pone-0002316-g002">Fig. 2</a>, <a href="#pone-0002316-t001">Tab. 1</a>), so that these human samples contained on average 198 times more human copies of the mitochondrial fragment tested for (3951±1296 SE) than the 29 non-human specimens (20±7.4 SE). The human samples were then decontaminated and extracted in two laboratories, Linköping (two independent extractions per sample) and Stockholm (a single extraction per sample), the separate extractions being carried out as independent replications performed by different people in each laboratory. Various negative controls were processed in parallel (a minimum of 18 and 4 non-human specimens and 16 and 3 extraction blanks in the Linköping and Stockholm laboratories respectively). DNA amplification was further performed in three laboratories, Linköping, Stockholm and Uppsala. The samples extracted in the Linköping laboratory were also amplified there, while the samples extracted in Stockholm were divided into two aliquots and further processed in Stockholm and Uppsala. All three laboratories are ancient DNA laboratories properly designed according to previously described standards <a href="#pone.0002316-Hofreiter1">[3]</a>, <a href="#pone.0002316-Cooper1">[10]</a>, with airlocks, positive airflow and ceiling UV radiation at night. All three laboratories are located in areas separated from any work with high quality DNA or post-PCR procedures, and the laboratory workers wore full zip suites, facemasks and several layers of gloves when in the laboratories.</p>
<div class="figure" id="pone-0002316-g002"><div class="img"><a name="pone-0002316-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0002316" data-uri="info:doi/10.1371/journal.pone.0002316.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0002316.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0002316.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Presence of an 80bp mtDNA fragment in the 74 screened samples, the seven selected samples and 29 non-human samples.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>The subset of humans was selected from a quantitative pre-screening. Samples yielding sufficient DNA and containing sufficient bone material for repeated re-extraction, and originating from more than one collection were selected for further processing. Note that the same extracts that was used for the mitochondrial pre-screening was used for the -13910 typing in Linköping, while new extracts were produced in Stockholm/Uppsala. A = Human mitochondrial content in all human samples pre-screened for mitochondrial DNA, B = Human mitochondrial content in selected human samples and C = Human mitochondrial content in non-human samples pre-screened for human mitochondrial DNA.</p>
<span>doi:10.1371/journal.pone.0002316.g002</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>Human specific primers (ordered separately for each laboratory from TAG, Copenhagen on three occasions) amplifying a 53 bp fragment were designed manually from a reference sequence (AY220757) where the biotinylated forward primer (<span class="gene">5′→3′ GCTGGCAATACAGATAAGATAATG</span>) and the reverse primer (<span class="gene">5′→3′ GAGGAGAGTTCCTTTGAGGC</span>) target a single nucleotide polymorphism (SNP) situated 13910 bp upstream of the LCT gene. Two PCR amplification protocols were used. In Uppsala 5 µl of extract was used in a 25 µl reaction, containing 2 mM MgCl<sub>2</sub>, 0.2 µM of each primer, 400 µM dNTPs and 2.5U Taq Polymerase (HotStarTaq, Qiagen), whereas in Linköping and Stockholm 5 µl of extract and 300 nM of each primer was added to PCR beads (Illustra Hot Start Mix RTG) in a 25 µl reaction. The PCR bead kit from Illustra was used in both the Linköping and Stockholm laboratories, with new batches used for each PCR amplification setup. The cycling conditions in Stockholm and Linköping (conditions for Uppsala are given in brackets when they deviate) were 95°C for 15 min (or 10 min), followed by 43 (or 47) cycles at 94°C for 30 s, 54°C(or 52°C) for 30 s, 72°C for 30 s, and a final extension at 72°C for 15 min (or 7 min).</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>The amplification was investigated on a 2% agarose gel in Stockholm and Uppsala and samples yielding a visible result were pyrosequenced<sup>TM</sup> on a PSQ™ 96MA sequencer. All the amplified samples from Linköping were pyrosequenced without any verification, as they had been selected from samples known to yield DNA. Pyrosequencing is a real-time sequencing method in which the DNA sequence is identified from light emitted via an enzymatic reaction when bases are incorporated into the DNA molecule <a href="#pone.0002316-Ronaghi1">[21]</a>, <a href="#pone.0002316-Ronaghi2">[22]</a>. In Linköping the PCR products were not checked on a gel prior to SNP typing. A commercial SNP reagent kit (Biotage, Uppsala) was used to examine the PCR products produced in all three laboratories, but different batches were used in each laboratory. In this step 25 µl PCR product from the samples was prepared according to the manufacturer's instructions. The sequencing primer was designed to anneal next to the SNP (<span class="gene">5′→3′ CCTTTGAGGCCAGGG</span>). Nucleotide dispensation was automatically retrieved using the PSQ™ 96MA SNP software (Biotage, Uppsala). The SNPs were scored automatically, edited using the PSQ™ 96MA SNP software and finally checked manually.</p>
<a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>The results from Linköping were based on multiple (3-6) amplifications from two independent extractions, while those from Stockholm/Uppsala were based on single extractions followed by multiple (n = 3) and single amplifications respectively. Each amplicon from the negative controls was treated as an independent observation. The Mann-Whitney U test was used to compare the alleles observed between the datasets. As the results in Stockholm and Uppsala originated from the same extractions, they were pooled. Negative controls were compared with the human samples in both datasets, and the numbers of negative controls that yielded positive results for human samples, i.e. contaminated blanks, were then compared between Linköping and Stockholm/Uppsala. Negative PCR controls were not included in the analyses. All the statistical analyses were performed on STATISTICA 7.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The seven human samples all yielded PCR products with identifiable SNPs in all three laboratories (<a href="#pone-0002316-t001">Table 1</a>). The four non-human samples and the three extraction controls processed in Uppsala and Stockholm were amplified 28 times, and all yielded amplicons with identifiable SNPs (<a href="#pone-0002316-t002">Table 2</a>). The 29 non-human samples and the 25 extraction controls processed in Linköping were amplified 84 times altogether, and yielded 8 amplicons with identifiable SNPs (<a href="#pone-0002316-t002">Table 2</a>).</p>
<div class="figure" id="pone-0002316-t001"><div class="img"><a name="pone-0002316-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0002316" data-uri="info:doi/10.1371/journal.pone.0002316.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0002316.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0002316.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Amount of an 80bp mitochondrial fragment (number of templates per real-time PCR reaction, with 5 µl DNA extracts used in 25 µl reactions, average from two real-time amplifications), and allele status in seven samples of ancient human remains typed in three laboratories (Linköping, Stockholm, and Uppsala).</span></strong></p><span>doi:10.1371/journal.pone.0002316.t001</span></div><div class="figure" id="pone-0002316-t002"><div class="img"><a name="pone-0002316-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0002316" data-uri="info:doi/10.1371/journal.pone.0002316.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0002316.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0002316.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>The non-human samples and negative extraction controls were amplified 84 times in Linköping and 28 times in Stockholm/Uppsala, producing 8 amplicons in Linköping and 28 in Stockholm/Uppsala.</span></strong></p><span>doi:10.1371/journal.pone.0002316.t002</span></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>An obvious discrepancy was detected between the results from the three laboratories, the allele frequency for the derived allele (-13910T) in the seven human samples being 0.36 for Linköping and 0.14 for Uppsala and Stockholm (<a href="#pone-0002316-g003">Fig. 3</a>). The frequency of the -13910T allele in modern Sweden is 74% (21), whereas the only published results obtained for archaeological material showed no individuals carrying the -13910T allele in the eight Neolithic samples from Central Europe (16). In this case we observed a difference in allele frequencies in the contaminated samples by contrast with the frequencies expected on the basis of the modern distribution in the country where the material came from and was analysed.</p>
<div class="figure" id="pone-0002316-g003"><div class="img"><a name="pone-0002316-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0002316" data-uri="info:doi/10.1371/journal.pone.0002316.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0002316.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0002316.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0002316.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0002316.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0002316.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Frequency of the T allele located 13910bp upstream of the LCT gene.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>The groups illustrated are the human samples processed in Linköping, the human samples processed in Stockholm/Uppsala, the negative controls containing human DNA from Linköping, and the negative controls containing human DNA from Stockholm/Uppsala. A = Human samples processed in Linköping, B = Non-human samples processed in Linköping, C = Human samples processed in Stockholm/Uppsala and D = Non-human samples processed in Stockholm/Uppsala.</p>
<span>doi:10.1371/journal.pone.0002316.g003</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We assume that the dataset generated in Linköping is more likely to be authentic than that generated in Stockholm/Uppsala, for three reasons. First, we have quantitative mitochondrial DNA data for the samples processed in Linköping (<a href="#pone-0002316-g002">Fig. 2</a>) showing that there is sufficient human DNA in the human samples and significantly more than in the negative controls, to provide for authentic results. This argument is generally accepted on its own as sufficient for authentication <a href="#pone.0002316-Green1">[23]</a>. Second, significantly fewer negative controls were contaminated with modern human DNA at Linköping than at Stockholm and Uppsala (p = &lt;0.001, Z = 7.149479 in a MWU test). Finally, there was a significant difference in allele frequencies between the negative controls that showed human results and the results obtained from the ancient human samples (p = 0.016, Z = −2.41109, <a href="#pone-0002316-g003">Fig. 3</a>) among those processed in Linköping. The seven human samples had a frequency of 0.36 for the derived allele while the contaminated controls (four out of 43 amplicons from non-human samples and four out of 41 amplicons from extraction controls) had a frequency of 0.88 for the same allele. There is no such difference between the negative controls and the ancient human samples in the results generated in Stockholm and Uppsala (p = 0.93, Z = −0.090854, <a href="#pone-0002316-g003">Fig. 3</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The most striking result, however, is that the contaminating allele (-13910C) present in most of the negative controls and equally many of the samples processed in Stockholm and Uppsala is an allele that is as rare in Scandinavia as in the rest of Europe. None of the laboratory workers who had been closest to the material was a homozygote for the allele appearing in the contamination (AL, HM, and AG, although AG is a heterozygote). Furthermore, the material originates from two collections, making it improbable that one physical anthropologist with a deviating allele could have contaminated all the samples. If this had been the case, there would not have been any discrepancy between the results from Linköping and the other two laboratories. The origin of the contamination could lie in reagents manufactured outside of Europe, for example, and used in the various laboratories. Several of the reagents used in all three laboratories were from the same suppliers but from different batches and with different production dates. The water and the ethanol used in the laboratories were of different origins, and while PCR reagents from the same supplier were used in Stockholm and Linköping, PCR reagents from a different supplier were used in Uppsala (but note that the results from Uppsala did not deviate from those from Stockholm). Contamination problems with PCR reagents have been demonstrated in earlier studies <a href="#pone.0002316-Leonard1">[24]</a>. Other possible sources could be the plastic ware or the gloves, for which the three laboratories use different suppliers.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>More important, the contamination we detected in this material is of a type that we would not have expected from samples excavated and processed in Scandinavia, as the allele present in the negative controls is rare in this area (but note that the allele frequencies in the contaminated samples are similar to those known in parts of Asia, where much of the plastic ware and gloves had been produced). Thus our data raise concerns on how the authentication of ancient human DNA is currently taking place. It is not appropriate to authenticate results solely on the basis of deviation from the modern population or the scientists and laboratory workers who have been in contact with the material. This criterion is not sufficient on its own, but it could possibly be used in combination with other criteria such as real-time qPCR, typing of human DNA in a sufficiently large body of non-human material and massive amplicon cloning.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Cecilia Anderung, Andreas Karlsson, and Ann-Britt Gladh provided useful comments on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AL AG HM. Performed the experiments: AL AG HM. Analyzed the data: AL AG HM. Contributed reagents/materials/analysis tools: AL AG HM. Wrote the paper: KL AL AG HM GH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0002316-Pbo1" id="pone.0002316-Pbo1"></a>Pääbo S (1985) Molecular cloning of Ancient Egyptian mummy DNA. Nature  314: 644–645.  <ul class="find" data-citedArticleID="1122194" data-doi="10.1038/314644a0"><li><a href="http://dx.doi.org/10.1038/314644a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+of+Ancient+Egyptian+mummy+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+of+Ancient+Egyptian+mummy+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0002316-Haak1" id="pone.0002316-Haak1"></a>Haak W, Forster P, Bramanti B, Matsumura S, Brandt G, et al.  (2005) Ancient DNA from the first European farmers in 7500-year-old Neolithic sites. Science  310: 1016–1018.  <ul class="find" data-citedArticleID="1122180" data-doi="10.1126/science.1123936"><li><a href="http://dx.doi.org/10.1126/science.1123936" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+DNA+from+the+first+European+farmers+in+7500-year-old+Neolithic+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+DNA+from+the+first+European+farmers+in+7500-year-old+Neolithic+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0002316-Hofreiter1" id="pone.0002316-Hofreiter1"></a>Hofreiter M, Serre D, Poinar HN, Kuch M, Pääbo S (2001) Ancient DNA. Nat Rev Genet  2: 353–359.  <ul class="find" data-citedArticleID="1122184" data-doi="10.1038/35072071"><li><a href="http://dx.doi.org/10.1038/35072071" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0002316-Richards1" id="pone.0002316-Richards1"></a>Richards MB, Sykes BC, Hedges REM (1995) Authenticating DNA Extracted From Ancient Skeletal Remains. Journal of Archaeological Science  22: 291–299.  <ul class="find" data-citedArticleID="1122196" data-doi="10.1006/jasc.1995.0031"><li><a href="http://dx.doi.org/10.1006/jasc.1995.0031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Authenticating+DNA+Extracted+From+Ancient+Skeletal+Remains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Authenticating+DNA+Extracted+From+Ancient+Skeletal+Remains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0002316-Malmstrm1" id="pone.0002316-Malmstrm1"></a>Malmström H, Storå J, Dalén L, Holmlund G, Götherström A (2005) Extensive human DNA contamination in extracts from ancient dog bones and teeth. Mol Biol Evol  22: 2040–2047.  <ul class="find" data-citedArticleID="1122190" data-doi="10.1093/molbev/msi195"><li><a href="http://dx.doi.org/10.1093/molbev/msi195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+human+DNA+contamination+in+extracts+from+ancient+dog+bones+and+teeth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+human+DNA+contamination+in+extracts+from+ancient+dog+bones+and+teeth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0002316-Sampietro1" id="pone.0002316-Sampietro1"></a>Sampietro ML, Gilbert MT, Lao O, Caramelli D, Lari M, et al.  (2006) Tracking down human contamination in ancient human teeth. Mol Biol Evol  23: 1801–1807.  <ul class="find" data-citedArticleID="1122202" data-doi="10.1093/molbev/msl047"><li><a href="http://dx.doi.org/10.1093/molbev/msl047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tracking+down+human+contamination+in+ancient+human+teeth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tracking+down+human+contamination+in+ancient+human+teeth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0002316-Handt1" id="pone.0002316-Handt1"></a>Handt O, Richards M, Trommsdorff M, Kilger C, Simanainen J, et al.  (1994) Molecular genetic analyses of the Tyrolean Ice Man. Science  264: 1775–1778.  <ul class="find" data-citedArticleID="1122182" data-doi="10.1126/science.8209259"><li><a href="http://dx.doi.org/10.1126/science.8209259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+genetic+analyses+of+the+Tyrolean+Ice+Man." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+genetic+analyses+of+the+Tyrolean+Ice+Man.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0002316-Malmstrm2" id="pone.0002316-Malmstrm2"></a>Malmström H, Svensson EM, Gilbert MT, Willerslev E, Götherström A, et al.  (2007) More on contamination: the use of asymmetric molecular behavior to identify authentic ancient human DNA. Mol Biol Evol  24: 998–1004.  <ul class="find" data-citedArticleID="1122192" data-doi="10.1093/molbev/msm015"><li><a href="http://dx.doi.org/10.1093/molbev/msm015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=More+on+contamination%3A+the+use+of+asymmetric+molecular+behavior+to+identify+authentic+ancient+human+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22More+on+contamination%3A+the+use+of+asymmetric+molecular+behavior+to+identify+authentic+ancient+human+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0002316-Wall1" id="pone.0002316-Wall1"></a>Wall JD, Kim SK (2007) Inconsistencies in Neanderthal Genomic DNA Sequences. PLoS Genet  3: e175.  <ul class="find" data-citedArticleID="1122206" data-doi="10.1371/journal.pgen.0030175.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030175.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inconsistencies+in+Neanderthal+Genomic+DNA+Sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inconsistencies+in+Neanderthal+Genomic+DNA+Sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0002316-Cooper1" id="pone.0002316-Cooper1"></a>Cooper A, Poinar HN (2000) Ancient DNA: do it right or not at all. Science  289: 1139.  <ul class="find" data-citedArticleID="1122176" data-doi="10.1126/science.289.5482.1139b"><li><a href="http://dx.doi.org/10.1126/science.289.5482.1139b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+DNA%3A+do+it+right+or+not+at+all." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+DNA%3A+do+it+right+or+not+at+all.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0002316-Abbott1" id="pone.0002316-Abbott1"></a>Abbott A (2003) Anthropologists cast doubt on human DNA evidence. Nature  423: 468.  <ul class="find" data-citedArticleID="1122168" data-doi="10.1038/423468b"><li><a href="http://dx.doi.org/10.1038/423468b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anthropologists+cast+doubt+on+human+DNA+evidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anthropologists+cast+doubt+on+human+DNA+evidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0002316-Caramelli1" id="pone.0002316-Caramelli1"></a>Caramelli D, Lalueza-Fox C, Vernesi C, Lari M, Casoli A, et al.  (2003) Evidence for a genetic discontinuity between Neandertals and 24,000-year-old anatomically modern Europeans. Proc Natl Acad Sci U S A  100: 6593–6597.  <ul class="find" data-citedArticleID="1122174" data-doi="10.1073/pnas.1130343100"><li><a href="http://dx.doi.org/10.1073/pnas.1130343100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+a+genetic+discontinuity+between+Neandertals+and+24%2C000-year-old+anatomically+modern+Europeans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+a+genetic+discontinuity+between+Neandertals+and+24%2C000-year-old+anatomically+modern+Europeans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0002316-Woodward1" id="pone.0002316-Woodward1"></a>Woodward SR, Weyand NJ, Bunnell M (1994) DNA sequence from Cretaceous period bone fragments. Science  266: 1229–1232.  <ul class="find" data-citedArticleID="1122208" data-doi="10.1126/science.7973705"><li><a href="http://dx.doi.org/10.1126/science.7973705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+sequence+from+Cretaceous+period+bone+fragments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+sequence+from+Cretaceous+period+bone+fragments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0002316-Zischler1" id="pone.0002316-Zischler1"></a>Zischler H, Höss M, Handt O, von Haeseler A, van der Kuyl AC, et al.  (1995) Detecting dinosaur DNA. Science  268: 1192–1193; author reply 1194.  <ul class="find" data-citedArticleID="1122214" data-doi="10.1126/science.7605504"><li><a href="http://dx.doi.org/10.1126/science.7605504" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+dinosaur+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+dinosaur+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0002316-Adcock1" id="pone.0002316-Adcock1"></a>Adcock GJ, Dennis ES, Easteal S, Huttley GA, Jermiin LS, et al.  (2001) Mitochondrial DNA sequences in ancient Australians: Implications for modern human origins. Proc Natl Acad Sci U S A  98: 537–542.  <ul class="find" data-citedArticleID="1122170" data-doi="10.1073/pnas.98.2.537"><li><a href="http://dx.doi.org/10.1073/pnas.98.2.537" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+sequences+in+ancient+Australians%3A+Implications+for+modern+human+origins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+sequences+in+ancient+Australians%3A+Implications+for+modern+human+origins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0002316-Burger1" id="pone.0002316-Burger1"></a>Burger J, Kirchner M, Bramanti B, Haak W, Thomas MG (2007) Absence of the lactase-persistence-associated allele in early Neolithic Europeans. Proc Natl Acad Sci U S A  104: 3736–3741.  <ul class="find" data-citedArticleID="1122172" data-doi="10.1073/pnas.0607187104"><li><a href="http://dx.doi.org/10.1073/pnas.0607187104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Absence+of+the+lactase-persistence-associated+allele+in+early+Neolithic+Europeans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Absence+of+the+lactase-persistence-associated+allele+in+early+Neolithic+Europeans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0002316-Xie1" id="pone.0002316-Xie1"></a>Xie CZ, Li CX, Cui YQ, Zhang QC, Fu YQ, et al.  (2007) Evidence of ancient DNA reveals the first European lineage in Iron Age Central China. Proc Biol Sci  274: 1597–1601.  <ul class="find" data-citedArticleID="1122210" data-doi="10.1098/rspb.2007.0219"><li><a href="http://dx.doi.org/10.1098/rspb.2007.0219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+of+ancient+DNA+reveals+the+first+European+lineage+in+Iron+Age+Central+China." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+of+ancient+DNA+reveals+the+first+European+lineage+in+Iron+Age+Central+China.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0002316-Kuokkanen1" id="pone.0002316-Kuokkanen1"></a>Kuokkanen M, Butzow R, Rasinpera H, Medrek K, Nilbert M, et al.  (2005) Lactase persistence and ovarian carcinoma risk in Finland, Poland and Sweden. Int J Cancer  117: 90–94.  <ul class="find" data-citedArticleID="1122186" data-doi="10.1002/ijc.21130"><li><a href="http://dx.doi.org/10.1002/ijc.21130" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lactase+persistence+and+ovarian+carcinoma+risk+in+Finland%2C+Poland+and+Sweden." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lactase+persistence+and+ovarian+carcinoma+risk+in+Finland%2C+Poland+and+Sweden.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0002316-Yang1" id="pone.0002316-Yang1"></a>Yang DY, Eng B, Waye JS, Dudar JC, Saunders SR (1998) Technical note: improved DNA extraction from ancient bones using silica-based spin columns. Am J Phys Anthropol  105: 539–543.  <ul class="find" data-citedArticleID="1122212" data-doi="10.1002/(sici)1096-8644(199804)105:4<539::aid-ajpa10&gt;3.3.co;2-2"><li><a href="http://dx.doi.org/10.1002/(sici)1096-8644(199804)105:4<539::aid-ajpa10&gt;3.3.co;2-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Technical+note%3A+improved+DNA+extraction+from+ancient+bones+using+silica-based+spin+columns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Technical+note%3A+improved+DNA+extraction+from+ancient+bones+using+silica-based+spin+columns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0002316-Svensson1" id="pone.0002316-Svensson1"></a>Svensson EM, Anderung C, Baubliene J, Persson P, Malmström H, et al.  (2007) Tracing genetic change over time using nuclear SNPs in ancient and modern cattle. Anim Genet  38: 378–383.  <ul class="find" data-citedArticleID="1122204" data-doi="10.1111/j.1365-2052.2007.01620.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2052.2007.01620.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tracing+genetic+change+over+time+using+nuclear+SNPs+in+ancient+and+modern+cattle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tracing+genetic+change+over+time+using+nuclear+SNPs+in+ancient+and+modern+cattle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0002316-Ronaghi1" id="pone.0002316-Ronaghi1"></a>Ronaghi M (2001) Pyrosequencing sheds light on DNA sequencing. Genome Res  11: 3–11.  <ul class="find" data-citedArticleID="1122198" data-doi="10.1101/gr.11.1.3"><li><a href="http://dx.doi.org/10.1101/gr.11.1.3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pyrosequencing+sheds+light+on+DNA+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pyrosequencing+sheds+light+on+DNA+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0002316-Ronaghi2" id="pone.0002316-Ronaghi2"></a>Ronaghi M, Uhlen M, Nyren P (1998) A sequencing method based on real-time pyrophosphate. Science  281: 363, 365.  <ul class="find" data-citedArticleID="1122200" data-doi="10.1126/science.281.5375.363"><li><a href="http://dx.doi.org/10.1126/science.281.5375.363" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+sequencing+method+based+on+real-time+pyrophosphate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+sequencing+method+based+on+real-time+pyrophosphate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0002316-Green1" id="pone.0002316-Green1"></a>Green RE, Krause J, Ptak SE, Briggs AW, Ronan MT, et al.  (2006) Analysis of one million base pairs of Neanderthal DNA. Nature  444: 330–336.  <ul class="find" data-citedArticleID="1122178" data-doi="10.1038/nature05336"><li><a href="http://dx.doi.org/10.1038/nature05336" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+one+million+base+pairs+of+Neanderthal+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+one+million+base+pairs+of+Neanderthal+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0002316-Leonard1" id="pone.0002316-Leonard1"></a>Leonard JA, Shanks O, Hofreiter M, Kreuz E, Hodges L, et al.  (2007) Animal DNA in PCR reagents plagues ancient DNA research. Journal of Archaeological Science  34: 1361–1366.  <ul class="find" data-citedArticleID="1122188" data-doi="10.1016/j.jas.2006.10.023"><li><a href="http://dx.doi.org/10.1016/j.jas.2006.10.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Animal+DNA+in+PCR+reagents+plagues+ancient+DNA+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Animal+DNA+in+PCR+reagents+plagues+ancient+DNA+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.XML" value="52612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.PDF" value="131642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g001.PNG_L" value="172514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g001.PNG_M" value="35795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g001.PNG_S" value="8432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g001.TIF" value="248706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g001.PNG_I" value="28094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g002.PNG_L" value="206189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g002.PNG_M" value="17183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g002.PNG_S" value="4553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g002.TIF" value="379428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g002.PNG_I" value="6330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.t001.PNG_L" value="86845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.t001.PNG_M" value="50860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.t001.PNG_S" value="9436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.t001.TIF" value="473160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.t001.PNG_I" value="16052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.t002.PNG_L" value="54300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.t002.PNG_M" value="33125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.t002.PNG_S" value="8278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.t002.TIF" value="289758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.t002.PNG_I" value="10047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g003.PNG_L" value="54786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g003.PNG_M" value="16097"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g003.PNG_S" value="4406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g003.TIF" value="96004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0002316.g003.PNG_I" value="5871"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0002316&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0002316" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0002316&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0002316&volume=&issue=&title=Cryptic Contamination and Phylogenetic Nonsense&author_name=Anna%20Linderholm%2C%20Helena%20Malmstr%C3%B6m%2C%20Kerstin%20Lid%C3%A9n%2C%20Gunilla%20Holmlund%2C%20Anders%20G%C3%B6therstr%C3%B6m&start_page=1&end_page=5" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0002316" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0002316" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0002316" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0002316&amp;t=Cryptic%20Contamination%20and%20Phylogenetic%20Nonsense" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0002316&title=Cryptic%20Contamination%20and%20Phylogenetic%20Nonsense&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0002316&amp;title=Cryptic%20Contamination%20and%20Phylogenetic%20Nonsense" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0002316" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0002316&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Cryptic Contamination and Phylogenetic Nonsense';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0002316';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Cryptic%20Contamination%20and%20Phylogenetic%20Nonsense http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0002316" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0002316" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ancient+DNA%22" title="Search for articles in the subject area:'Ancient DNA'"><div class="flagText">Ancient DNA</div></a>
              <div data-categoryid="35013" data-articleid="27456"
                   data-categoryname="Ancient DNA"
                   class="flagImage" title="Flag 'Ancient DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+extraction%22" title="Search for articles in the subject area:'DNA extraction'"><div class="flagText">DNA extraction</div></a>
              <div data-categoryid="19965" data-articleid="27456"
                   data-categoryname="DNA extraction"
                   class="flagImage" title="Flag 'DNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="27456"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+amplification%22" title="Search for articles in the subject area:'Gene amplification'"><div class="flagText">Gene amplification</div></a>
              <div data-categoryid="19947" data-articleid="27456"
                   data-categoryname="Gene amplification"
                   class="flagImage" title="Flag 'Gene amplification' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="27456"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondrial+DNA%22" title="Search for articles in the subject area:'Mitochondrial DNA'"><div class="flagText">Mitochondrial DNA</div></a>
              <div data-categoryid="32451" data-articleid="27456"
                   data-categoryname="Mitochondrial DNA"
                   class="flagImage" title="Flag 'Mitochondrial DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleogenetics%22" title="Search for articles in the subject area:'Paleogenetics'"><div class="flagText">Paleogenetics</div></a>
              <div data-categoryid="33815" data-articleid="27456"
                   data-categoryname="Paleogenetics"
                   class="flagImage" title="Flag 'Paleogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="27456"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4894'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1308'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=987&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
