

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Mitochondrial Genomes Reveal Slow Rates of Molecular Evolution and the Timing of Speciation in Beavers (Castor), One of the Largest Rodent Species</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0014622"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0014622" />

    <meta name="citation_title" content="Mitochondrial Genomes Reveal Slow Rates of Molecular Evolution and the Timing of Speciation in Beavers (Castor), One of the Largest Rodent Species"/>
    <meta itemprop="name" content="Mitochondrial Genomes Reveal Slow Rates of Molecular Evolution and the Timing of Speciation in Beavers (Castor), One of the Largest Rodent Species"/>

      <meta name="citation_author" content="Susanne Horn"/>
            <meta name="citation_author_institution" content="Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany"/>
      <meta name="citation_author" content="Walter Durka"/>
            <meta name="citation_author_institution" content="Department for Community Ecology, Helmholtz-Centre for Environmental Research-UFZ, Halle, Germany"/>
      <meta name="citation_author" content="Ronny Wolf"/>
            <meta name="citation_author_institution" content="Department for Molecular Evolution and Animal Systematics, University of Leipzig, Leipzig, Germany"/>
      <meta name="citation_author" content="Aslak Ermala"/>
            <meta name="citation_author_institution" content="Finnish Game and Fisheries Research Institute, Helsinki, Finland"/>
      <meta name="citation_author" content="Annegret Stubbe"/>
            <meta name="citation_author_institution" content="Institute of Zoology, University of Halle-Wittenberg, Wittenberg, Germany"/>
      <meta name="citation_author" content="Michael Stubbe"/>
            <meta name="citation_author_institution" content="Institute of Zoology, University of Halle-Wittenberg, Wittenberg, Germany"/>
      <meta name="citation_author" content="Michael Hofreiter"/>
            <meta name="citation_author_institution" content="Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany"/>

    <meta name="citation_date" content="2011/1/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0014622.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e14622"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="6"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Beaver (Castor canadensis).; citation_author=BW Baker; citation_author=EP Hill; citation_number=1; citation_date=2003; citation_publisher=biology, management, and conservation Baltimore, MD The Johns Hopkins University Press; " />
      <meta name="citation_reference" content="citation_title=Die Biber der alten und neuen Welt.; citation_author=WW Djoshkin; citation_author=WG Safonow; citation_number=2; citation_date=1972; citation_publisher=Die Neue Brehm-BÃ¼cherei, A Ziemsen Verlag; " />
      <meta name="citation_reference" content="citation_title=Multiple molecular evidences for a living mammalian fossil.; citation_author=De Huchon; citation_author=P Chevret; citation_author=U Jordan; citation_author=CW Kilpatrick; citation_author=V Ranwez; citation_journal_title=Proceedings of the National Academy of Sciences; citation_volume=104; citation_number=3; citation_pages=7495-7499; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Higher-level systematics of rodents and divergence time estimates based on two congruent nuclear genes.; citation_author=RM Adkins; citation_author=AH Walton; citation_author=RL Honeycutt; citation_journal_title=Molecular Phylogenetics and Evolution; citation_volume=26; citation_number=4; citation_pages=409-420; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Rodent phylogeny revised: analysis of six nuclear genes from all major rodent clades.; citation_author=S Blanga-Kanfi; citation_author=H Miranda; citation_author=O Penn; citation_author=T Pupko; citation_author=R DeBry; citation_journal_title=BMC Evolutionary Biology; citation_volume=9; citation_number=5; citation_pages=71; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Suprafamilial relationships among Rodentia and the phylogenetic effect of removing fast-evolving nucleotides in mitochondrial, exon and intron fragments.; citation_author=C Montgelard; citation_author=E Forty; citation_author=V Arnal; citation_author=C Matthee; citation_journal_title=BMC Evolutionary Biology; citation_volume=8; citation_number=6; citation_pages=321; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Automated Scanning for Phylogenetically Informative Transposed Elements in Rodents.; citation_author=A Farwick; citation_author=U Jordan; citation_author=G Fuellen; citation_author=D Huchon; citation_author=F Catzeflis; citation_journal_title=Syst Biol; citation_volume=55; citation_number=7; citation_pages=936-948; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Rodent Evolution: Back to the Root.; citation_author=G Churakov; citation_author=MK Sadasivuni; citation_author=KR Rosenbloom; citation_author=D Huchon; citation_author=J Brosius; citation_journal_title=Mol Biol Evol; citation_volume=27; citation_number=8; citation_pages=1315-1326; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=A biochemical-genetic discrimination method for the two beaver species, Castor fiber and Castor canadensis, as a tool for conservation.; citation_author=J Sieber; citation_author=F Suchentrunk; citation_author=GB HARTL; citation_number=9; citation_pages=61-65; citation_date=1999; citation_publisher=Beaver Protection, Management, and Utilization in Europe and North America Kluver Academic/Plenum Publishers; " />
      <meta name="citation_reference" content="citation_title=Cytogenetics of beavers - a case of speciation by monobrachial centric fusions.; citation_author=OG Ward; citation_author=AS Graphodatsky; citation_author=DH Wursterhill; citation_author=VR Eremina; citation_author=JP Park; citation_journal_title=Genome; citation_volume=34; citation_number=10; citation_pages=324-328; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Castorid Phylogenetics: Implications for the Evolution of Swimming and Tree-Exploitation in Beavers.; citation_author=N Rybczynski; citation_journal_title=Journal of Mammalian Evolution; citation_volume=14; citation_number=11; citation_pages=1-35; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Castoroidea. In: Evolution of Tertiary Mammals of North America.; citation_author=LJ Flynn; citation_author=LL Jacobs; citation_journal_title=Cambridge University Press; citation_volume=2; citation_number=12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Blancan-Hemphillian Land Mammal Ages and Late Cenozoic Mammal Dispersal Events.; citation_author=EH Lindsay; citation_author=ND Opdyke; citation_author=NM Johnson; citation_journal_title=Annual Review of Earth and Planetary Sciences; citation_volume=12; citation_number=13; citation_pages=445-488; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Fossil evidence for the origin of behavioral strategies in early Miocene Castoridae, and their role in the evolution of the family.; citation_author=M Hugueney; citation_author=F Escuillie; citation_journal_title=Paleobiology; citation_volume=22; citation_number=14; citation_pages=507-513; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Evolution of Chinese Castoridae.; citation_author=X Xu; citation_journal_title=National Science Museum monographs; citation_volume=8; citation_number=15; citation_pages=77-97; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Glavne napravlenia evolutsi semeistva Castoridae. [Basic directions in the evolution of the family Castoridae].; citation_author=G Lychev; citation_journal_title=[In:] Istoria i evolutsya sivremennoy fauny gryzunov SSSR, Nauka, Moskva (In Russian, German translation; citation_number=16; citation_pages=179-203; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Evolutionary development of the genus Castor and taxonomy of the contemporary beavers in Eurasia.; citation_author=LS Lavrov; citation_journal_title=Acta Zool Fennica; citation_volume=174; citation_number=17; citation_pages=87-90; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analyses of complete mitochondrial genome sequences suggest a basal divergence of the enigmatic rodent Anomalurus.; citation_author=D Horner; citation_author=K Lefkimmiatis; citation_author=A Reyes; citation_author=C Gissi; citation_author=C Saccone; citation_journal_title=BMC Evolutionary Biology; citation_volume=7; citation_number=18; citation_pages=16; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Pika and vole mitochondrial genomes increase support for both rodent monophyly and glires.; citation_author=Y-H Lin; citation_author=PJ Waddell; citation_author=D Penny; citation_journal_title=Gene; citation_volume=294; citation_number=19; citation_pages=119-129; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Relaxed Phylogenetics and Dating with Confidence.; citation_author=AJ Drummond; citation_author=SYW Ho; citation_author=MJ Phillips; citation_author=A Rambaut; citation_journal_title=PLoS Biology; citation_volume=4; citation_number=20; citation_pages=e88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Correlates of substitution rate variation in mammalian protein-coding sequences.; citation_author=J Welch; citation_author=O Bininda-Emonds; citation_author=L Bromham; citation_journal_title=BMC Evolutionary Biology; citation_volume=8; citation_number=21; citation_pages=53; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=There is no universal molecular clock for invertebrates, but rate variation does not scale with body size.; citation_author=JA Thomas; citation_author=JJ Welch; citation_author=M Woolfit; citation_author=L Bromham; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=22; citation_pages=7366-7371; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Site specific rates of mitochondrial genomes and the phylogeny of eutheria.; citation_author=K Kjer; citation_author=R Honeycutt; citation_journal_title=BMC Evolutionary Biology; citation_volume=7; citation_number=23; citation_pages=8; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rodent Monophyly: Pitfalls of Molecular Phylogenies.; citation_author=P HervÃ©; citation_journal_title=Journal of Molecular Evolution; citation_volume=45; citation_number=24; citation_pages=712-715; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Rodent phylogeny and a timescale for the evolution of Glires: evidence from an extensive taxon sampling using three nuclear genes.; citation_author=D Huchon; citation_author=O Madsen; citation_author=MJJ Sibbald; citation_author=K Ament; citation_author=MJ Stanhope; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=25; citation_pages=1053-1065; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Higher-level systematics of rodents and divergence time estimates based on two congruent nuclear genes.; citation_author=RM Adkins; citation_author=AH Walton; citation_author=RL Honeycutt; citation_journal_title=Mol Phylogenet Evol; citation_volume=26; citation_number=26; citation_pages=409-420; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Eocene Radiation and Phylogeny of the Rodents.; citation_author=AE Wood; citation_journal_title=Evolution; citation_volume=13; citation_number=27; citation_pages=354-361; citation_date=1959; " />
      <meta name="citation_reference" content="citation_title=Biochronology of the microtine rodents of the United States.; citation_author=CA Repenning; citation_number=28; citation_pages=236-268; citation_date=1987; citation_publisher=Cenozoic Mammals of North America: Geochronology and Biostratigraphy University of California Press; " />
      <meta name="citation_reference" content="citation_title=A refined age for the earliest opening of Bering Strait.; citation_author=AY Gladenkov; citation_author=AE Oleinik; citation_author=L Marincovich; citation_author=KB Barinov; citation_journal_title=Palaeogeography, Palaeoclimatology, Palaeoecology; citation_volume=183; citation_number=29; citation_pages=321-328; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Biogeographical history of the European beaver Castor fiber (Rodentia, Mammlia).; citation_author=G Veron; citation_journal_title=Mammalia; citation_volume=56; citation_number=30; citation_pages=87-108; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial phylogeography of the Eurasian beaver Castor fiber L.; citation_author=W Durka; citation_author=W Babik; citation_author=J-F Ducroz; citation_author=D Heidecke; citation_author=F Rosell; citation_journal_title=Molecular Ecology; citation_volume=14; citation_number=31; citation_pages=3843-3856; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Effect of Inappropriate Calibration: Three Case Studies in Molecular Ecology.; citation_author=SYW Ho; citation_author=U Saarma; citation_author=R Barnett; citation_author=J Haile; citation_author=B Shapiro; citation_journal_title=PLoS ONE; citation_volume=3; citation_number=32; citation_pages=e1615; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Evidence for Time Dependency of Molecular Rate Estimates.; citation_author=SYW Ho; citation_author=B Shapiro; citation_author=MJ Phillips; citation_author=A Cooper; citation_author=AJ Drummond; citation_journal_title=Systematic Biology; citation_volume=56; citation_number=33; citation_pages=515-522; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=First DNA sequences from Asian cave bear fossils reveal deep divergences and complex phylogeographic patterns.; citation_author=M Knapp; citation_author=N Rohland; citation_author=J Weinstock; citation_author=G Baryshnikov; citation_author=A Sher; citation_journal_title=Molecular Ecology; citation_volume=18; citation_number=34; citation_pages=1225-1238; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Mitogenetic structure of brown bears (Ursus arctos L.) in northeastern Europe and a new time frame for the formation of European brown bear lineages.; citation_author=U Saarma; citation_author=SYW Ho; citation_author=OG Pybus; citation_author=M Kaljuste; citation_author=IL Tumanov; citation_journal_title=Molecular Ecology; citation_volume=16; citation_number=35; citation_pages=401-413; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The genetic legacy of the Quaternary ice ages.; citation_author=G Hewitt; citation_journal_title=Nature; citation_volume=405; citation_number=36; citation_pages=907-913; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial whims: metabolic rate, longevity and the rate of molecular evolution.; citation_author=N Galtier; citation_author=R Jobson; citation_author=B Nabholz; citation_author=S GlÃ©min; citation_author=P Blier; citation_journal_title=Biological Letters; citation_volume=5; citation_number=37; citation_pages=413-416; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Strong Variations of Mitochondrial Mutation Rate across Mammals-the Longevity Hypothesis.; citation_author=B Nabholz; citation_author=S Glemin; citation_author=N Galtier; citation_journal_title=Molecular Biology and Evolution; citation_volume=25; citation_number=38; citation_pages=120-130; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=An Evolutionary Genome Scan for Longevity-Related Natural Selection in Mammals.; citation_author=RW Jobson; citation_author=B Nabholz; citation_author=N Galtier; citation_journal_title=Molecular Biology and Evolution; citation_volume=27; citation_number=39; citation_pages=840-847; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Guidelines for the Capture, Handling, and Care of Mammals as Approved by the American Society of Mammalogists.; citation_author=C Animal; citation_author=C Use; citation_journal_title=Journal of Mammalogy; citation_volume=79; citation_number=40; citation_pages=1416-1431; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Parallel tagged sequencing on the 454 platform.; citation_author=M Meyer; citation_author=U Stenzel; citation_author=M Hofreiter; citation_journal_title=Nat Protocols; citation_volume=3; citation_number=41; citation_pages=267-278; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Sequencing and analysis of Neanderthal genomic DNA.; citation_author=JP Noonan; citation_author=G Coop; citation_author=S Kudaravalli; citation_author=D Smith; citation_author=J Krause; citation_journal_title=Science; citation_volume=314; citation_number=42; citation_pages=1113-1118; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Direct multiplex sequencing (DMPS): A novel method for targeted high-throughput sequencing of ancient and highly degraded DNA.; citation_author=M Stiller; citation_author=M Knapp; citation_author=U Stenzel; citation_author=M Hofreiter; citation_author=M Meyer; citation_journal_title=Genome Research; citation_volume=19; citation_number=43; citation_pages=1843-1848; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=From micrograms to picograms: quantitative PCR reduces the material demands of high-throughput sequencing.; citation_author=M Meyer; citation_author=AW Briggs; citation_author=T Maricic; citation_author=B Hober; citation_author=B Hoffner; citation_journal_title=Nucl Acids Res; citation_volume=36; citation_number=44; citation_pages=e5; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT.; citation_author=TA Hall; citation_journal_title=Nucleic Acids Symposium Series; citation_volume=41; citation_number=45; citation_pages=95-98; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=An iterative, mapping assembly strategy for next-generation sequencing data enables assembly of otherwise intractable datasets.; citation_author=R Green; citation_number=46; citation_date=2009; citation_publisher=Cold Spring Harbor Laboratory meeting âThe Biology of Genomesâ; " />
      <meta name="citation_reference" content="citation_title=Recent developments in the MAFFT multiple sequence alignment program.; citation_author=K Katoh; citation_author=H Toh; citation_journal_title=Brief Bioinform; citation_volume=9; citation_number=47; citation_pages=286-298; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution.; citation_author=D Posada; citation_author=K Crandall; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=48; citation_pages=817-818; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=PAUP*. Phylogenetic Analysis Using Parsimony (*and Other Methods). 4 ed.; citation_author=DL Swofford; citation_number=49; citation_date=1998; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinform; citation_volume=5; citation_number=50; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=A Rambaut; citation_number=51; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=BEAST: Bayesian evolutionary analysis by sampling trees.; citation_author=A Drummond; citation_author=A Rambaut; citation_journal_title=BMC Evolutionary Biology; citation_volume=7; citation_number=52; citation_pages=214; citation_date=2007; " />
      <meta name="citation_reference" content="citation_author=A Rambaut; citation_author=A Drummond; citation_number=53; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Alarm Bells for the Molecular Clock? No Support for Ho et al.'s Model of Time-Dependent Molecular Rate Estimates.; citation_author=BC Emerson; citation_journal_title=Syst Biol; citation_volume=56; citation_number=54; citation_pages=337-345; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Population and distribution of European beavers (Castor fiber).; citation_author=DJ Halley; citation_author=F Rosell; citation_journal_title=Lutra; citation_volume=46; citation_number=55; citation_pages=91-101; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014622" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Mitochondrial Genomes Reveal Slow Rates of Molecular Evolution and the Timing of Speciation in Beavers (Castor), One of the Largest Rodent Species"/>
    <meta name="twitter:description" content="
Background
Beavers are one of the largest and ecologically most distinct rodent species. Little is known about their evolution and even their closest phylogenetic relatives have not yet been identified with certainty. Similarly, little is known about the timing of divergence events within the genus Castor.

Methodology/Principal Findings
We sequenced complete mitochondrial genomes from both extant beaver species and used these sequences to place beavers in the phylogenetic tree of rodents and date their divergence from other rodents as well as the divergence events within the genus Castor. Our analyses support the phylogenetic position of beavers as a sister lineage to the scaly tailed squirrel Anomalurus within the mouse related clade. Molecular dating places the divergence time of the lineages leading to beavers and Anomalurus as early as around 54 million years ago (mya). The living beaver species, Castor canadensis from North America and Castor fiber from Eurasia, although similar in appearance, appear to have diverged from a common ancestor more than seven mya. This result is consistent with the hypothesis that a migration of Castor from Eurasia to North America as early as 7.5 mya could have initiated their speciation. We date the common ancestor of the extant Eurasian beaver relict populations to around 210,000 years ago, much earlier than previously thought. Finally, the substitution rate of Castor mitochondrial DNA is considerably lower than that of other rodents. We found evidence that this is correlated with the longer life span of beavers compared to other rodents.

Conclusions/Significance
A phylogenetic analysis of mitochondrial genome sequences suggests a sister-group relationship between Castor and Anomalurus, and allows molecular dating of species divergence in congruence with paleontological data. The implementation of a relaxed molecular clock enabled us to estimate mitochondrial substitution rates and to evaluate the effect of life history traits on it.
"/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0014622.g005"/>

  <meta property="og:title" content="Mitochondrial Genomes Reveal Slow Rates of Molecular Evolution and the Timing of Speciation in Beavers (Castor), One of the Largest Rodent Species" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014622" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9538'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8625'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2138&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0014622">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0014622" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0014622&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Mitochondrial Genomes Reveal Slow Rates of Molecular Evolution and the Timing of Speciation in Beavers (<i>Castor</i>), One of the Largest Rodent Species
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Susanne Horn
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:horn@eva.mpg.de">horn@eva.mpg.de</a></p>

                <p>Affiliation:
                  Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Walter Durka, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department for Community Ecology, Helmholtz-Centre for Environmental Research-UFZ, Halle, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ronny Wolf, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department for Molecular Evolution and Animal Systematics, University of Leipzig, Leipzig, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Aslak Ermala, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Finnish Game and Fisheries Research Institute, Helsinki, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Annegret Stubbe, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Zoology, University of Halle-Wittenberg, Wittenberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Stubbe, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Zoology, University of Halle-Wittenberg, Wittenberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Hofreiter
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany
                </p>

                <p>
                    Current address: Department of Biology, University of York, York, United Kingdom
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 28, 2011</li>
    <li>DOI: 10.1371/journal.pone.0014622</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0014622" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0014622" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0014622" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0014622" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0014622-g001" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0014622-g002" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0014622-g003" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0014622-g004" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0014622-t001" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0014622-g005" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0014622">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0014622" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Beavers are one of the largest and ecologically most distinct rodent species. Little is known about their evolution and even their closest phylogenetic relatives have not yet been identified with certainty. Similarly, little is known about the timing of divergence events within the genus <em>Castor</em>.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We sequenced complete mitochondrial genomes from both extant beaver species and used these sequences to place beavers in the phylogenetic tree of rodents and date their divergence from other rodents as well as the divergence events within the genus <em>Castor</em>. Our analyses support the phylogenetic position of beavers as a sister lineage to the scaly tailed squirrel <em>Anomalurus</em> within the mouse related clade. Molecular dating places the divergence time of the lineages leading to beavers and <em>Anomalurus</em> as early as around 54 million years ago (mya). The living beaver species, <em>Castor canadensis</em> from North America and <em>Castor fiber</em> from Eurasia, although similar in appearance, appear to have diverged from a common ancestor more than seven mya. This result is consistent with the hypothesis that a migration of <em>Castor</em> from Eurasia to North America as early as 7.5 mya could have initiated their speciation. We date the common ancestor of the extant Eurasian beaver relict populations to around 210,000 years ago, much earlier than previously thought. Finally, the substitution rate of <em>Castor</em> mitochondrial DNA is considerably lower than that of other rodents. We found evidence that this is correlated with the longer life span of beavers compared to other rodents.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>A phylogenetic analysis of mitochondrial genome sequences suggests a sister-group relationship between <em>Castor</em> and <em>Anomalurus</em>, and allows molecular dating of species divergence in congruence with paleontological data. The implementation of a relaxed molecular clock enabled us to estimate mitochondrial substitution rates and to evaluate the effect of life history traits on it.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Horn S, Durka W, Wolf R, Ermala A, Stubbe A, et al.  (2011) Mitochondrial Genomes Reveal Slow Rates of Molecular Evolution and the Timing of Speciation in Beavers (<em>Castor</em>), One of the Largest Rodent Species. PLoS ONE 6(1):
          e14622.
            doi:10.1371/journal.pone.0014622</p><p><strong>Editor: </strong>William J. Murphy, Texas A&amp;M University, United States of America</p><p><strong>Received:</strong> July 13, 2010; <strong>Accepted:</strong> November 10, 2010; <strong>Published:</strong> January 28, 2011</p><p><strong>Copyright:</strong> Â© 2011 Horn et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Volkswagen Foundation and the Max Planck Society. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Dating back to approximately 40 million years, today the family of beavers, Castoridae, is represented by only two extant species, <em>Castor canadensis</em> in North America and <em>Castor fiber</em> in Eurasia. Both species are characterized by their large body size, being the second largest rodent, and their semi-aquatic lifestyle <a href="#pone.0014622-Baker1">[1]</a>, <a href="#pone.0014622-Djoshkin1">[2]</a>. However, several aspects of the early history and evolution of beavers remain unclear. Earlier attempts of their phylogenetic placement relative to other rodents were difficult because of a lack of fixed morphological differences, poor taxon sampling in many genetic studies, limited sequence data in previous studies, and contemporaneous radiations of multiple rodent lineages. More recent molecular data strongly support the placement of <em>Castor</em> within a âmouse-related clade,â containing several families including Pedetidae, Anomaluridae, Muridae, Dipodidae, Geomyidae, and Heteromyidae <a href="#pone.0014622-Huchon1">[3]</a>, <a href="#pone.0014622-Adkins1">[4]</a>, <a href="#pone.0014622-BlangaKanfi1">[5]</a>, <a href="#pone.0014622-Montgelard1">[6]</a>. Previous multigene studies have suggested Geomyoidea to be the closest relatives of beavers <a href="#pone.0014622-BlangaKanfi1">[5]</a>, <a href="#pone.0014622-Montgelard1">[6]</a>. However, the branches leading to both groups diverged very early in rodent evolution and transposon insertion analyses are inconclusive with regard to their monophyly <a href="#pone.0014622-Farwick1">[7]</a>, <a href="#pone.0014622-Churakov1">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Not only the phylogenetic placement of beavers within rodents is not completely understood, little is also known about the timing of the speciation event leading to the two extant beaver species. Both have a strikingly similar phenotype, making them almost indistinguishable in the field; therefore molecular methods are applied to differentiate them <a href="#pone.0014622-Baker1">[1]</a>, <a href="#pone.0014622-Sieber1">[9]</a>. Despite the morphological similarities, subtle morphological and biochemical features and different chromosome numbers support their distinction as different species <a href="#pone.0014622-Ward1">[10]</a>. The fossil record provides further information on the timing of <em>Castor</em> speciation. The appearance of the genus <em>Castor</em> in Eurasia and North America was estimated to the late Miocene and the Pliocene, between 9â4.9 mya <a href="#pone.0014622-Rybczynski1">[11]</a>, <a href="#pone.0014622-Flynn1">[12]</a>, <a href="#pone.0014622-Lindsay1">[13]</a>, <a href="#pone.0014622-Hugueney1">[14]</a>, <a href="#pone.0014622-Xu1">[15]</a>, <a href="#pone.0014622-Lychev1">[16]</a>. <em>Castor</em> is assumed to have emerged in Eurasia as a close relative of <em>Steneofiber</em> <a href="#pone.0014622-Rybczynski1">[11]</a>, <a href="#pone.0014622-Flynn1">[12]</a>, <a href="#pone.0014622-Hugueney1">[14]</a>, <a href="#pone.0014622-Xu1">[15]</a>, <a href="#pone.0014622-Lychev1">[16]</a>, and to have subsequently dispersed to North America via the Bering landbridge <a href="#pone.0014622-Flynn1">[12]</a>, <a href="#pone.0014622-Lindsay1">[13]</a>, <a href="#pone.0014622-Xu1">[15]</a>, <a href="#pone.0014622-Lychev1">[16]</a>, <a href="#pone.0014622-Lavrov1">[17]</a>. This dispersal event was estimated to 4.9â6.6 mya <a href="#pone.0014622-Lindsay1">[13]</a>, <a href="#pone.0014622-Xu1">[15]</a>. However, since the estimates for the earliest appearance of <em>Castor</em> on both continents overlap, it is not entirely clear in which direction the dispersal of <em>Castor</em> took place. Irrespective of dispersal direction, since the two extant species of <em>Castor</em> are native to either Eurasia or North America, it seems reasonable to assume that a migration across the Bering strait could have initiated the speciation of <em>C. canadensis</em> and <em>C. fiber</em> <a href="#pone.0014622-Ward1">[10]</a>, <a href="#pone.0014622-Hugueney1">[14]</a>, <a href="#pone.0014622-Xu1">[15]</a> (<a href="#pone-0014622-g001">Fig. 1</a>). However, no study to date attempted obtaining a sequence based molecular date for the timing of divergence between the two species. As mitochondrial genomes are available for a number of rodent taxa <a href="#pone.0014622-Horner1">[18]</a>, the addition of <em>Castor</em> mitochondrial genomes could shed light not only on the deep phylogeny of beavers, but also on the timing of divergence to other rodents and within the genus itself. Finally, mitochondrial genome data should allow for determination of the overall substitution rate in beaver mitochondrial DNA, which allows retrospectively investigating and dating population genetic processes.</p>
<div class="figure" id="pone-0014622-g001"><div class="img"><a name="pone-0014622-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0014622.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0014622.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Distribution of beavers and sampling sites.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Beavers live in North America (<em>C. canadensis</em>, striped areas) and Eurasia (<em>C. fiber</em>, grey areas). Black areas mark <em>C. fiber</em> relict populations from which the current populations developed. <em>C. canadensis</em> was introduced in Europe and Asia (striped areas, distribution not exactly known for Kamchatka). Numbers indicate beaver populations sampled for mitochondrial genome sequencing. <em>C. fiber</em> was sampled in the areas of the relict populations (1,3,4,5) or close to them (2) <a href="#pone.0014622-Durka1">[31]</a>. <em>C. canadensis</em> was sampled in a European introduced population (6). Arrow: migration via the Bering land bridge is suggested to have initiated <em>Castor</em> speciation around 8â7.6 million years ago (mya). Map redrawn from <a href="#pone.0014622-Baker1">[1]</a>, <a href="#pone.0014622-Durka1">[31]</a> and <a href="#pone.0014622-Halley1">[55]</a>.</p>
<span>doi:10.1371/journal.pone.0014622.g001</span></div><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here we present the first complete mitochondrial genomes of beavers. We use these DNA sequences to investigate the phylogenetic position of beavers within rodents, date evolutionary events within the extant members of the family Castoridae and explore the substitution rate of their mitochondrial DNA.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Phylogenetics</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We sequenced mitochondrial genomes of one <em>C. canadensis</em> and five <em>C. fiber</em> utilizing long range PCR, DNA hybridization capture and 454 sequencing (<a href="#pone.0014622.s001">Table S1</a> and and <a href="#pone.0014622.s009">Supporting References S1</a>). Assemblies derived from hybridization capture showed a considerably smoother read distribution than those derived from long range PCR (<a href="#pone-0014622-g002">Fig. 2</a>). The consensus sequences obtained were aligned with a variety of rodent and outgroup mitochondrial genomes for phylogenetic analyses and molecular dating (<a href="#pone.0014622.s002">Table S2</a>). We employed several different methods for investigating the phylogenetic relationships in our dataset. All methods, Bayesian approaches, maximum likelihood (ML), neighbor joining (NJ) and maximum parsimony (MP) recovered a well supported, monophyletic clade of the <em>Castor</em> individuals. In <a href="#pone-0014622-g003">Fig. 3</a> we present a maximum clade credibility tree from BEAST with support values from all phylogenetic methods. Highest support was obtained for the sister group relationship of <em>C. canadensis</em> and <em>C. fiber</em>, the latter represented by a monophyletic clade of five subspecies (<em>C. fiber</em> ssp. <em>albicus</em>, <em>belorussicus/orientoeuropaeus</em>, <em>birulai</em>, <em>tuvinicus</em>, and <em>pohlei</em>). A position of <em>Castor</em> within the mouse related clade of rodents, as sister to the scaly tailed squirrel <em>Anomalurus</em>, was supported with very high and maximum support, respectively, from Bayesian analyses as well as with bootstrap values higher than 70 in ML analyses. The squirrel related clade and Hystricognathi were recovered with high support values in all phylogenetic methods. Although the monophyly of rodents was not supported in NJ and MP, it received maximum support from Bayesian and a bootstrap value of 70 in ML analyses. However, the relationship between the three major rodent clades remains enigmatic, since their branching order could not be resolved. Since difficulties in resolving the root of the rodent tree have been attributed to the effect of long branch attraction, causing rodent sequences to cluster with outgroups <a href="#pone.0014622-Horner1">[18]</a>, <a href="#pone.0014622-Lin1">[19]</a>, we conducted additional ML analyses with a more representative sample of outgroup sequences (<a href="#pone.0014622.s002">Table S2</a>). These additional outgroup sequences could not improve the resolution of the early branchings, instead, the ML bootstrap support value for rodent monophyly decreased from 70 to 52 (<a href="#pone.0014622.s003">Table S3</a>). However, since the monophyly of rodents was supported in our Bayesian and ML trees based on the smaller set of taxa as well as other studies on nuclear DNA sequences and transposon insertions <a href="#pone.0014622-Farwick1">[7]</a>, <a href="#pone.0014622-Churakov1">[8]</a>, we nevertheless attempted estimating the time elapsed since a monophyletic origin of all rodents using a molecular dating approach implemented in BEAST <a href="#pone.0014622-Drummond1">[20]</a>.</p>
<div class="figure" id="pone-0014622-g002"><div class="img"><a name="pone-0014622-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0014622.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0014622.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Sequencing coverage plots of beaver mitochondrial genomes.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Plots for <em>C.f.</em> ssp. <em>birulai</em> amplified by long range PCR (grey line) and <em>C.f.</em> ssp. <em>pohlei</em> enriched by hybridization capture (black line). Coverage was more even when hybridization capture was used instead of long range PCR. Peaks of sequencing coverage are visible for <em>C.f.</em> ssp. <em>birulai</em> (grey) in proximity to priming sites (indicated on top of the plots).</p>
<span>doi:10.1371/journal.pone.0014622.g002</span></div><div class="figure" id="pone-0014622-g003"><div class="img"><a name="pone-0014622-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0014622.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0014622.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Timetree of rodents and outgroups.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Beavers share a common ancestor with Anomaluromorpha around 54 mya (CI: 44â64 mya); the only extant beaver genus <em>Castor</em> separated into two species from 8â7.6 mya onwards (CI: 3.7â13 mya). The common ancestor of the Eurasian relict populations was estimated to have lived around 210,000 ya (CI: 0.11â0.34 mya). The tree depicted is a maximum clade credibility tree from BEAST analyses based on a 16,352 bp alignment (including gaps). Bayesian posterior probability (&gt;0.6) and bootstrap support values (&gt;50) are shown at the branches and separated by slashes for MrBayes, maximum likelihood, neighbor joining and maximum parsimony, respectively. Diamonds indicate fossil calibrations (<a href="#pone.0014622.s004">Table S4</a>). Paleoc.: Paleocene; Oligoc.: Oligocene; Pl: Pliocene, PH: Pleistocene and Holocene. <em>Castor fiber</em> po, tu, bi, in and al indicate the subspecies sampled (see <a href="#pone.0014622.s001">Table S1</a>).</p>
<span>doi:10.1371/journal.pone.0014622.g003</span></div>

<h4>Divergence estimates</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We estimated that all rodents shared a common ancestor around 67 mya (CI: 57â76 mya) (<a href="#pone.0014622.s004">Table S4</a>). Around 54 mya (CI: 44â64 mya), the branch leading to beavers diverged from the common ancestor with the scaly tailed squirrel <em>Anomalurus</em>. The mitochondrial DNA of the two beaver species <em>C. fiber</em> and <em>C. canadensis</em> shared a most recent common ancestor between 8 mya (mean) and 7.6 mya (median) (CI: 3.7â13 mya; <a href="#pone.0014622.s004">Table S4</a>). Within Eurasian beavers, the individuals coming from different relict populations shared a common ancestor around 210,000 years ago (CI: 110,000-340,000 ya).</p>


<h4>Substitution rates</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>As beavers are the largest rodents for which a mitochondrial genome sequence is currently available, we explored the mitochondrial substitution rate in comparison to other rodents. There is some evidence that the substitution rate is linked to life expectancy and body mass <a href="#pone.0014622-Welch1">[21]</a> even though this connection is controversial <a href="#pone.0014622-Thomas1">[22]</a>. Animals like the beaver, with higher life expectancy and larger body mass are therefore expected to show decreased substitution rates. Molecular substitution rates for the taxa in our dataset were estimated in the BEAST analysis and are depicted on each branch in <a href="#pone-0014622-g004">Fig. 4</a> (see also <a href="#pone.0014622.s005">Table S5</a>). The only rodents with a lower mitochondrial rate were <em>Sciurus</em> and <em>Myoxus</em> in the squirrel-related clade. Within the mouse-related clade the mitochondrial tip rates of beavers were significantly lower than those of the other members of this clade (p-value  = 0.005). Also, when the distribution of rates for beavers was compared to the distribution of rates for all other rodents in the dataset, the rates of beavers were significantly lower (p-value  = 0.014, Mann-Whitney U-test, <a href="#pone-0014622-g004">Fig.4</a>, <a href="#pone-0014622-t001">Table 1</a>). This difference became even more evident when instead of tip rates average rates leading to the individual taxa were used (see methods and <a href="#pone.0014622.s001">Table S1</a>). If tested across all glires (rodents and lagomorphs), the tip and average rates of beavers did not differ significantly anymore (<a href="#pone-0014622-t001">Table 1</a>).</p>
<div class="figure" id="pone-0014622-g004"><div class="img"><a name="pone-0014622-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0014622.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0014622.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparison of mitochondrial substitution rates among glires.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Phylogenetic tree of glires (rodents and lagomorphs) with branch lengths from the BEAST analysis. Mitochondrial substitution rates are shown on the branches or next to each taxon in units of substitutions per million years. Beavers exhibit short branches and substitution rates for beavers are the lowest within the mouse-related clade, significantly lower than those of most other rodents (see also <a href="#pone-0014622-t001">Table 1</a>).</p>
<span>doi:10.1371/journal.pone.0014622.g004</span></div><div class="figure" id="pone-0014622-t001"><div class="img"><a name="pone-0014622-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0014622.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0014622.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Mitochondrial substitution rates of <em>Castor</em> differ significantly from those of other rodents.</span></strong></p><span>doi:10.1371/journal.pone.0014622.t001</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We related the inferred substitution rates to body mass and life history traits of rodents as had been done earlier <a href="#pone.0014622-Welch1">[21]</a> (<a href="#pone-0014622-g005">Fig. 5</a>, <a href="#pone.0014622.s006">Table S6</a>). In linear regressions, more than 30 percent of the variation in tip rates (R<sup>2</sup> = 0.34, p-value = 0.0225) and average rates (R<sup>2</sup> = 0.40, p-value = 0.0113) could be explained by maximum lifespan. Thus considerably more variation in lifespan was attributable to mitochondrial rate variation than had been estimated previously (below 20 percent in <a href="#pone.0014622-Welch1">[21]</a>). However, for body mass and age at sexual maturity, no significant correlations with substitution rates could be identified (p-values&gt;0.05, <a href="#pone.0014622.s006">Table S6</a>).</p>
<div class="figure" id="pone-0014622-g005"><div class="img"><a name="pone-0014622-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0014622" data-uri="info:doi/10.1371/journal.pone.0014622.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0014622.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0014622.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0014622.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0014622.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0014622.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Linear regression of average substitution rates and lifespan of rodents.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Rodents with higher mitochondrial substitution rates exhibit a shorter lifespan (R squared  = 0.4011, p-value  = 0.0113). See <a href="#pone.0014622.s006">Table S6</a> and <a href="#pone.0014622.s007">S7</a> for details.</p>
<span>doi:10.1371/journal.pone.0014622.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The phylogenetic relationships of extant rodent families have been difficult to resolve. However, several previous molecular studies recognized three major phylogenetic clades of rodents: the mouse-related clade, Ctenohystrica (relatives of the guinea pig) and the squirrel-related clade (<a href="#pone-0014622-g003">Fig. 3</a>). This branching was supported by nuclear DNA analyses <a href="#pone.0014622-Huchon1">[3]</a>, <a href="#pone.0014622-BlangaKanfi1">[5]</a>, <a href="#pone.0014622-Farwick1">[7]</a>, <a href="#pone.0014622-Churakov1">[8]</a> as well as some studies including mitochondrial DNA <a href="#pone.0014622-Montgelard1">[6]</a>, <a href="#pone.0014622-Kjer1">[23]</a>. We find this phylogeny to be also supported by complete mitochondrial nucleotide sequences. In accordance with studies on nuclear DNA <a href="#pone.0014622-Huchon1">[3]</a>, <a href="#pone.0014622-BlangaKanfi1">[5]</a>, <a href="#pone.0014622-Montgelard1">[6]</a>, <em>Anomalurus</em> is recovered as an early member of the mouse-related clade. In contrast, an earlier analysis of complete mitochondrial genome sequences did not recover <em>Anomalurus</em> within the mouse-related clade <a href="#pone.0014622-Horner1">[18]</a>, possibly due to reducing the sequences to the coding regions for some of the phylogenetic inferences.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The root of the rodent tree is much more difficult to resolve. Although we obtain high support for rodents as a monophyletic group, confirming earlier studies <a href="#pone.0014622-Huchon1">[3]</a>, <a href="#pone.0014622-BlangaKanfi1">[5]</a>, <a href="#pone.0014622-Churakov1">[8]</a>, <a href="#pone.0014622-Herv1">[24]</a>, <a href="#pone.0014622-Huchon2">[25]</a>, as in previous studies, we could not determine the branching order of the three major clades <a href="#pone.0014622-Huchon1">[3]</a>, <a href="#pone.0014622-BlangaKanfi1">[5]</a>, <a href="#pone.0014622-Horner1">[18]</a>. However, our analyses are based on mitochondrial DNA and therefore represent a single genomic locus only. Also, since many rodents exhibit high substitution rates, part of the lower resolution for the deeper nodes might result from mutational saturation of the sequences. Analyses of a larger number of nuclear loci should improve the resolution of the deeper phylogenetic nodes within rodents in the future.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We recovered the six beaver mitochondrial genomes in one monophyletic group within the mouse-related clade of rodents (<a href="#pone-0014622-g003">Fig. 3</a>). This phylogenetic position of beavers was indicated already in earlier analyses based on nuclear gene sequences and transposon insertions <a href="#pone.0014622-BlangaKanfi1">[5]</a>, <a href="#pone.0014622-Farwick1">[7]</a>, <a href="#pone.0014622-Churakov1">[8]</a>. Now, mitochondrial genomics adds another line of evidence to this relationship. In our dataset, the mitochondrial sequence of the scaly tailed squirrel <em>Anomalurus</em> was most closely related to those of the beavers. However, there is no mitochondrial genome sequence available for the Geomyoidea (gophers and kangaroo rats), which have been found to be most closely related to beavers in multigene studies <a href="#pone.0014622-BlangaKanfi1">[5]</a>, <a href="#pone.0014622-Montgelard1">[6]</a> and could attach to the evolutionary branch leading to beavers even later than <em>Anomalurus</em>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We estimated divergence times within rodents in BEAST using six fossil calibration points and a Bayesian relaxed molecular clock approach <a href="#pone.0014622-Drummond1">[20]</a> and found all rodents to share a common ancestor around 67 mya (CI: 57â76 mya) (<a href="#pone.0014622.s004">Table S4</a>). This timing is remarkably coincident with the mass extinction event that marked the transition between the Cretaceous and Tertiary (K-T boundary). At that time dinosaurs went extinct and a major faunal change took place all over the world. The origin of rodents has been dated both earlier and later in other studies: around 96 mya <a href="#pone.0014622-Adkins2">[26]</a>, 71â89 mya <a href="#pone.0014622-Montgelard1">[6]</a>, around 72 mya <a href="#pone.0014622-Huchon1">[3]</a> and around 62 mya <a href="#pone.0014622-Horner1">[18]</a>. Our estimate lies in between the range of earlier estimates, close to the K-T boundary and predating the explosive radiation of rodents in the early Eocene (dated to around 55mya) <a href="#pone.0014622-Wood1">[27]</a>. Thus, the changing biota at the K-T boundary may have been causal for a first rodent radiation.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Around 54 mya (CI: 44â64 mya), a phylogenetic lineage leading to beavers diverged from its common ancestor with Anomaluromorpha (<em>Anomalurus</em> and <em>Pedetes</em>). Thus, beavers probably have a very long evolutionary history, which might explain their ecological and morphological peculiarities. In a previous study using 5.5 kb of nuclear and mitochondrial DNA sequences <a href="#pone.0014622-Huchon1">[3]</a>, the divergence of beavers and Anomaluromorpha was estimated even older, at more than 65 mya. However, since Geomyoidea are potentially even more closely related to beavers, additional DNA sequences for Geomyoidea will be required to provide information on the beginning of beaver evolution.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Similar to the divergence of the family Castoridae, the divergence time of the two extant beaver species has not yet been estimated with much precision. The origin of the extant beaver genus <em>Castor</em> has been suggested to lie in Eurasia at some time between 9.7 and 5.2 mya based on the fossil record and similarities with <em>Steneofiber</em> <a href="#pone.0014622-Hugueney1">[14]</a>, <a href="#pone.0014622-Xu1">[15]</a>, <a href="#pone.0014622-Lychev1">[16]</a>. However, since there is overlap with the earliest appearance of <em>Castor</em> in North America (6.6 mya to 7.5 mya <a href="#pone.0014622-Flynn1">[12]</a>, <a href="#pone.0014622-Repenning1">[28]</a>), the geographical origin of <em>Castor</em> remains uncertain. Independent of that, a migration of ancient <em>Castor</em> via Beringia and the subsequent geographical isolation of the two populations in Eurasia and North America, respectively, most likely led to the divergence of <em>Castor</em> lineages that ultimately gave rise to the modern species <em>C. fiber</em> and <em>C. canadensis</em>. According to our molecular datings, the two beaver species shared a common ancestor around 7.6â8 (CI: 3.7â13) mya. This molecular estimate is intriguingly close to the earliest fossil beaver remains in North America possibly dating as early as 7.5 mya <a href="#pone.0014622-Flynn1">[12]</a>. The Beringian land bridge allowed faunal exchange between Eurasia and North America until its first flooding 5.4â5.5 mya and then again several times during the Pliocene and Pleistocene <a href="#pone.0014622-Gladenkov1">[29]</a>. Thus, a migration over the Beringian landbridge around 8â7.6 mya has likely been the starting point for the speciation between <em>C. fiber</em> and <em>C. canadensis</em>, each on a different continent (<a href="#pone-0014622-g001">Fig. 1</a>).</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Despite a successful evolutionary history across the Palaearctic for several millions of years, beaver populations decreased dramatically in size in more recent history. Extensive hunting by humans and habitat destruction affected both beaver species <a href="#pone.0014622-Baker1">[1]</a>, <a href="#pone.0014622-Veron1">[30]</a> and left Eurasia with a few, isolated relict populations of <em>C. fiber</em> at the end of the 19<sup>th</sup> century <a href="#pone.0014622-Veron1">[30]</a>. The diversification of these extant beaver populations was proposed to have happened during the last glacial period, from 115,000 ya onwards <a href="#pone.0014622-Durka1">[31]</a>. We dated the last common ancestor of the extant Eurasian beaver mitochondrial genomes to almost twice the age than estimated before, around 210,000 ya (CI: 110,000â340,00 ya). However, it has been pointed out that due to the problems in determining substitution rates precisely, such dates should be viewed with caution and correlating molecular divergence dates with changes in environmental conditions may be tentative at best <a href="#pone.0014622-Ho1">[32]</a>. It should be noted that when using fossil calibration points like in this study, the divergences towards the tips of the tree may be overestimated <a href="#pone.0014622-Ho2">[33]</a>. However, the substitution rate estimates for the tip branches leading to the individual subspecies are very similar to the overall estimate for the branch from the MRCA of all rodents to the tips of the beaver branches, suggesting that for this evolutionary lineage, time-dependent variation in substitution rates is negligible. Therefore, the diversification of the extant Eurasian beaver lineages started most likely substantially earlier than previously estimated. Also, geographically distinct lineages of other Eurasian, potentially forest dependant species showed similar divergence estimates, such as cave bears (173,000â414,000 ya <a href="#pone.0014622-Knapp1">[34]</a>) and brown bears (174,000â314,000 ya <a href="#pone.0014622-Saarma1">[35]</a>). The genetic diversity of these populations was probably shaped by climatic fluctuations during the glacial cycles since around 0.9 mya <a href="#pone.0014622-Hewitt1">[36]</a>.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>The analysis of substitution rates also shows that beavers display a lower substitution rate than most other rodents, a feature clearly visible in the branch length of the phylogenetic tree (<a href="#pone-0014622-g004">Fig. 4</a>). This difference was highly significant when beavers were tested against other members of the mouse-related clade. The rate difference was still significant when <em>Castor</em> was compared with all other rodents in the dataset, including the squirrel-related clade which also showed lower substitution rates. It has been argued before that certain morphological, physiological, and life history traits influence DNA substitution rates. For example, Welch et al. <a href="#pone.0014622-Welch1">[21]</a> showed that life history traits like body mass, lifespan and age at sexual maturity are negatively correlated with substitution rates over a wide range of mammals. Beavers are atypical among rodents with respect to several life history traits as they have a much larger body mass, longer lifespan and do not reach sexual maturity until 1.5â3 years of age (<a href="#pone.0014622.s007">Table S7</a>, <a href="#pone.0014622-Baker1">[1]</a>, <a href="#pone.0014622-Hugueney1">[14]</a>). Thus, a lower substitution rate can be expected for beavers compared to other rodents. In our analysis of mitochondrial genomes, body mass and age at sexual maturity did not correlate significantly with mitochondrial substitution rates, although body mass did so earlier <a href="#pone.0014622-Welch1">[21]</a>. However, we found a significant negative correlation between the substitution rate and maximum lifespan as has been previously reported for mitochondrial synonymous sites across mammals <a href="#pone.0014622-Welch1">[21]</a>. Despite the smaller sample size in our analysis, a larger fraction of the variation in substitution rate was explained by maximum lifespan, even though we included non-synonymous sites, which did not show a significant correlation earlier <a href="#pone.0014622-Welch1">[21]</a>. Correlates of substitution rates with lifespan are not fully explained to date <a href="#pone.0014622-Welch1">[21]</a>, but natural selection could have acted to reduce the mutation rate in mitochondria of long lived taxa such as the beaver <a href="#pone.0014622-Galtier1">[37]</a>, <a href="#pone.0014622-Nabholz1">[38]</a>. However, in a genome scan across 25 species, genes involved in DNA replication, repair or antioxidation did not show signatures of selection in long-lived taxa. Instead, the selected features were connected to cellular membrane and extracellular collagen composition and their functional relevance remains puzzling <a href="#pone.0014622-Jobson1">[39]</a>.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>In summary, our study showed that over the wide taxonomic range of glires, datasets comprising whole mitochondrial genome sequences facilitate the inference of substitution rates, phylogenetic analyses and divergence estimates.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Ethics statement</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The tissue sample of <em>Castor canadensis</em> was taken from a dead beaver, which was shot during the open hunting season in Finland. The sample of <em>Castor fiber</em> ssp. <em>albicus</em> came from a road kill, dissected with the approval from the environmental agency of Germany, administrative district of Leipzig. Samples <em>Castor fiber</em> ssp. <em>belorussicus</em>/<em>orientoeuropaeus</em>, <em>birulai</em>, <em>tuvinicus</em> and <em>pohlei</em> consisted of beaver tail skin or hair that was obtained from live animals captured with nets, live traps, and at night with a search light and netting from a boat. Animals were released afterward. Sampling procedures were consistent with guidelines of the American Society of Mammalogists for the capture and handling of mammals <a href="#pone.0014622-Animal1">[40]</a>. All catches were performed in cooperation with and under approval by, the Nature Reserve âMalaya Sosvaâ in Russia, the Zhitkov Russian Research Institute of Game Management and Fur Farming in Kirov, Russia, as well as the National University of Mongolia in Ulaan-Baatar, Mongolia.</p>


<h4>Samples, DNA extraction and long range PCR</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>DNA was extracted from five tissue samples of <em>C. fiber</em> and from one sample of <em>C. canadensis</em> (<a href="#pone.0014622.s001">Table S1</a>) using the DNeasy blood &amp; tissue kit (Quiagen). For well preserved DNA, the mitochondrial genome was amplified in two overlapping pieces around 11 and 6 kb in length, by long range PCR using the expand dNTPack (Roche) according to the manufacturer's instructions. Primer sequences are listed in <a href="#pone.0014622.s008">Table S8</a>. Long range PCR products were sheared using a Bioruptor UCD-200 (Diagenode). Barcoding adaptors with sample specific sequences and 454 sequencing adaptors were ligated to the sonicated products as described previously <a href="#pone.0014622-Meyer1">[41]</a>.</p>


<h4>DNA hybridization capture and sequencing</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>For less well preserved samples, for which long range PCR did not work, DNA hybridization capture was used to enrich for mitochondrial genomes (<a href="#pone.0014622.s001">Table S1</a>). For this process, biotinylated bait molecules are hybridized with a genomic library and later selectively captured on streptavidin beads <a href="#pone.0014622-Noonan1">[42]</a>. Barcoded genomic libraries were prepared from <em>Castor</em> DNA as described previously <a href="#pone.0014622-Meyer1">[41]</a>, <a href="#pone.0014622-Stiller1">[43]</a>, allowing the simultaneous sequencing of different samples on the same 454 lane. To produce bait molecules, long range amplicons of <em>C. fiber</em> ssp. <em>albicus</em> were used. The PCR products were sheared until they had a length of around 300 bp, and double stranded, biotinylated adapters were ligated onto the ends. The hybridization mixture was set up as follows: 1 Âµg of genomic library, 100 ng of bait and four blocking oligos (each 2 ÂµM) in 1x blocking Agent (Agilent) and 1x hybridization buffer (Agilent). Agilent reagents were from the aCGH Kit #5188-5220. Sequences for the blocking oligos are listed in <a href="#pone.0014622.s008">Table S8</a>. After denaturation of the mixture at 95Â°C for 5 min, the hybridization was carried out in 200 Âµl tubes (Eppendorf), rotating at 65Â°C for 48 hours in a conventional hybridization oven (SciGene).</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>After hybridization, biotinylated bait molecules were captured by incubation with 5 Âµl of magnetic streptavidin covered beads (Dynalbeads M270, Invitrogen) for 20 min at room temperature. The mixture was then placed into a magnetic rack (Beckman Coulter #A32782) allowing the separation of magnetic beads from the supernatant. The supernatant, containing non-target molecules was discarded and the remaining beads were washed five times using 1xBWT buffer (1 M NaCl, 10 mM Tris-Cl, 1 mM EDTA, 0.05% Tween-20 (Sigma), pH 8.0) and once in pre-warmed HW buffer (200 Âµl 10x AmpliTaq Gold buffer, 200 Âµl MgCl<sub>2</sub>, 1.6 ml H<sub>2</sub>O) at 50Â°C for 2 min. After one more wash with 1xBWT, the beads were transferred into a new tube with 100 Âµl of TE-buffer (containing 0.05% Tween-20). Finally, hybridized target molecules were separated from the bait molecules in 30 ul 1xTE by 5 min incubation at 95Â°C in a thermocycler. The eluate containing the sequencing library enriched for mitochondrial DNA, was directly used for quantification and sequencing.</p>


<h4>Sequence analyses and assembly</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>After 454 library preparation and quantification of the libraries by qPCR using emPCR priming sites <a href="#pone.0014622-Meyer2">[44]</a>, sequencing was carried out on the 454 FLX platform. <em>De novo</em> assembly of 454 reads of the mitochondrial genomes for <em>C. fiber</em> ssp. <em>albicus</em> and <em>C. canadensis</em> was done with runAssembly (454 Roche), separately for each of the two overlapping long range amplicons, and the overlaps joined by hand in BioEdit <a href="#pone.0014622-Hall1">[45]</a>. For the remaining <em>C. fiber</em> specimens, sequencing reads were mapped onto the assembled <em>C. fiber</em> mitochondrial genome using runMapping (454 software) and the iterative mapping tool IMA <a href="#pone.0014622-Green1">[46]</a>. The output of the mappings was viewed with clview (software available at <a href="http://compbio.dfci.harvard.edu/tgi/software/">http://compbio.dfci.harvard.edu/tgi/softâware/</a>) and the map aligner <a href="#pone.0014622-Green1">[46]</a>. Mapping of the sequencing reads from one PCR derived sequence showed a somewhat uneven coverage and peaks of read counts in the proximity to priming sites (<a href="#pone-0014622-g002">Fig. 2</a>), which could result from amplicons that were aborted during the PCR shortly after priming. The generally observed differences in coverage between both long range PCR products might reflect variation in quantification or pooling of these prior to sequencing. Coverage plots from hybridization capture were smoother than those derived from long range PCR (<a href="#pone-0014622-g002">Fig. 2</a>). The obtained mappings covered the complete mitochondrial genomes for all samples. One site each in beaver samples <em>C. canadensis</em> and <em>C. fiber albicus</em> had less than 3x coverage, for which the sequence was confirmed by PCR and Sanger sequencing (sample specific primers listed in <a href="#pone.0014622.s008">Table S8</a>). Nuclear mitochondrial insertions (numts) could be ruled out for long range PCR by joining the overlaps creating a circular sequence, and are not expected to be a problem for hybridization capture since nuclear DNA has a much lower copy number than mitochondrial DNA. The length of the repeat region between the control region and tRNA-Phe is a minimum estimate, since the 454 sequencing reads did not span the region completely. For between two and 21 positions per mitochondrial genome, corrections of the assembled DNA sequence were made by hand in BioEdit. These affected mainly protein coding regions, where the length of homopolymers was corrected so that the reading frame for amino acid sequences was retained. The obtained sequences were deposited in GenBank with accession numbers FR691684-FR691689.</p>


<h4>Phylogenetic analyses and divergence estimates</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Alignments of DNA sequences were done using mafft v6.708b <a href="#pone.0014622-Katoh1">[47]</a> for the taxa listed in <a href="#pone.0014622.s002">Table S2</a>. The complete alignment including gaps had 19,419 bp in length. 3067 bp of the alignment containing the control region between tRNA-Pro and tRNA-Phe were not well aligned due to high sequence divergence and were therefore removed, resulting in an alignment of 16,352 bp for phylogenetic analyses.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Modeltest was used to determine the optimal evolutionary model for the dataset <a href="#pone.0014622-Posada1">[48]</a>. The general time reversible (GTR) model with a proportion of invariant sites of 0.2559 and a gamma shape parameter of 0.4721 was determined to be most appropriate. Using this substitution model, phylogenetic trees were calculated with maximum likelihood (ML). One hundred bootstrap replicates were done in Paup <a href="#pone.0014622-Swofford1">[49]</a> version 4.0d105. The heuristic search used tree bisection reconnection (TBR) limited to 1000 rearrangements per bootstrap replicate due to computational constraints.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>In order to test if the removal of fast evolving sites from the sequences could improve phylogenetic inference <a href="#pone.0014622-Montgelard1">[6]</a>, two shorter versions of the original alignment were created and also subjected to ML inference (<a href="#pone.0014622.s003">Table S3</a>). One alignment contained all annotated loci with 15,865 bp (coding genes, l-rRNA, s-rRNA, replication origin, tRNAs) and thus excluded only 487 bp of non-annotated loci. The other alignment (containing coding genes, l-rRNA and s-rRNA) had 14,270 bp, excluding all tRNAs and the replication origin. Since these shortened alignments did not improve phylogenetic inference with ML (<a href="#pone.0014622.s003">Table S3</a>) and the longer alignment with 16,352 bp allowed the recognition of all major rodent clades, the latter was used for the following phylogenetic inferences and molecular dating with BEAST. The initial ML analyses on three alignments of different length were performed also on a larger taxon set in order to evaluate the topology of our phylogenetic tree, especially the monophyly of rodents. This larger taxon set comprised all 39 taxa listed in <a href="#pone.0014622.s002">Table S2</a>. Due to computational constraints, the taxon set was then reduced to 24 taxa, excluding some of the outgroup sequences for the following phylogenetic inferences and molecular dating with BEAST.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>For neighbor joining (NJ) the T3P model was used in Mega4 <a href="#pone.0014622-Kumar1">[50]</a>. 1,000 Bootstrap replicates were calculated with pairwise deletion and gamma distributed rates among sites (gamma shape parameter was 0.4729). For maximum parsimony (MP) 1,000 bootstrap replicates were done with close neighbor interchange (CNI, level 1) and initial trees for the CNI search by random addition trees (10 replications) in Mega4. Trees were searched in MrBayes v.3.1.2 with four parallel search chains for 10 million generations in triplicates. The search chains reached similar posterior probability levels in the triplicates as viewed in Tracer v1.4.1, final ESS were 339. The treefiles were combined in logcombiner and maximum clade credibility trees were created with TreeAnnotator implemented in BEAST v.1.5.3 and viewed in FigTree v.1.3.1 <a href="#pone.0014622-Rambaut1">[51]</a>.</p>
<a id="article1.body1.sec4.sec5.p5" name="article1.body1.sec4.sec5.p5"></a><p>Molecular dating was carried out with BEAST v.1.5.3, a coalescence based method for parameter estimation <a href="#pone.0014622-Drummond2">[52]</a>. Six fossil calibrations were taken into account as prior assumptions on divergence times (diamonds in <a href="#pone-0014622-g003">Fig. 3</a>, <a href="#pone.0014622.s004">Table S4</a>) and simultaneously optimized with other parameters describing the phylogeny. The relaxed molecular clock models implemented in BEAST assume independent rates on different branches of a phylogeny, thus allowing for calibrations in rather distantly related taxonomic groups. Thus, BEAST is specifically suited to accommodate the rate variation between lagomorphs, rodents and primates as well as that between rodents with slower and faster rates. The sampling priors for fossil calibrations were set to be normally distributed, incorporating knowledge on divergence estimates, such as the earliest appearance of members of a clade in the fossil record as well as the age of an assumed monophyletic origin and early relatives of the clade. Since there are uncertainties for determining the age of fossils and times that lineages actually diverged, we used rather wide ranges around these events for calibration (<a href="#pone.0014622.s004">Table S4</a>). Further, we used estimated base frequencies, an uncorrelated lognormal relaxed molecular clock, the Yule prior for speciation scenarios and UPGMA starting trees. BEAST xml-files were created in beauty (implemented in BEAST). Search chains were run in BEAST for 50 million generations and log files were written every 5,000 generations. The GTR substitution model was used and the first 10 percent of runs were discarded as burnin. Logcombiner from the BEAST package did not remove the burnin correctly, when combining logfiles. Therefore, logfiles of triplicates of the 50 million generation runs were combined into a single logfile by hand and the results were viewed in Tracer version v1.4.1 <a href="#pone.0014622-Rambaut2">[53]</a>. The posterior distributions for the fossil calibrations were visually inspected in Tracer v1.4.1 and verified to be symmetrically bell shaped, indicating a proper sampling from the prior distribution. A maximum clade credibility tree was created with treeannotator and viewed in FigTree.</p>


<h4>Comparison of evolutionary rates</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Substitution rates for the branches leading to different rodent taxa were determined in the BEAST analyses (combined from three 50 million generation runs) and shown as branch labels in Figtree. We tested if beavers differ in substitution rates from other rodents using a Mann-Whitney U-test, implemented in R 2.9.2 as Wilcoxon rank sum test with continuity correction. Since this test examines if two distributions differ significantly from each other, all beaver samples were included in the test.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>We showed that mitochondrial substitution rates of <em>Castor</em> differed significantly from those of the other rodents in the dataset. An increase of the observed rates towards the tips in a phylogeny (<a href="#pone-0014622-g004">Fig. 4</a>) is assumed to be a result of transient polymorphisms present in the more recent timescales of a phylogeny <a href="#pone.0014622-Ho2">[33]</a>. Although this hypothesis is controversial <a href="#pone.0014622-Emerson1">[54]</a>, in our BEAST phylogeny the estimated substitution rates also increase towards the tips. In order to explore this possible bias due to sampling on different taxonomic levels, we averaged the substitution rates along all branches leading to the common ancestor of all tested taxa (<a href="#pone.0014622.s005">Table S5</a>) and repeated the Mann-Whitney U-tests with those average rates (<a href="#pone-0014622-t001">Table 1</a>). In order to average the rates, for each of the three tests (<em>Castor</em> vs. mouse-related clade, <em>Castor</em> vs. rest of rodents, <em>Castor</em> vs. rest of glires, as indicated in <a href="#pone-0014622-t001">Table 1</a>) the arithmetic mean was calculated from the rates along the branches leading to the ancestral node connecting the tested taxa. The resulting p-values indicated an even more pronounced, highly significant difference between the lower rates of beavers and the higher rates of the other rodents in the dataset (<a href="#pone-0014622-t001">Table 1</a>). Further, we correlated the substitution rates of rodents with life history traits, such as body mass, lifespan and age at sexual maturity <a href="#pone.0014622-Welch1">[21]</a> using linear regressions in R. In order to avoid unbalanced data, we collapsed all sampled beavers into one data point for each regression. Measurements of substitution rates and life history traits for beavers were therefore averaged using the arithmetic mean. References for maximum lifespan, body mass and age at sexual maturity are given in <a href="#pone.0014622.s007">Table S7</a> and <a href="#pone.0014622.s009">Supporting References S1</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0014622"></div><a name="pone.0014622.s001" id="pone.0014622.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014622.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Samples and overview of processing for mitochondrial genome sequencing. Samples were obtained from different source populations of beavers [S1], enriched for mitochondrial DNA and barcoded before sequencing. LR PCR: long range PCR. Hyb: hybridization capture.</p>
<p class="siDoi">doi:10.1371/journal.pone.0014622.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0014622.s002" id="pone.0014622.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014622.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Taxa and accession numbers of the mitochondrial genome sequences used. Sequences marked by * are additional outgroup sequences to rodents and were used to evaluate the topology of our phylogenetic tree in the initial ML analysis but were excluded in the later ML, NJ, MP, and Bayesian analyses due to computational constraints.</p>
<p class="siDoi">doi:10.1371/journal.pone.0014622.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0014622.s003" id="pone.0014622.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014622.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Bootstrap support for monophyletic clades in maximum likelihood analyses of rodent mitochondrial genome alignments of different length. Support values for clades recovered in previous studies decline when using smaller fractions of the sequence data. Support values are given for alignments including 24 taxa and 39 taxa separated by a slash. Note that all alignments excluded a less well aligned region containing the control region.</p>
<p class="siDoi">doi:10.1371/journal.pone.0014622.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0014622.s004" id="pone.0014622.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014622.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Fossil calibrations and age estimates. Prior and posterior values for the time to the most recent common ancestor (tmrca) of monophyletic clades determined by BEAST analyses are given in million years ago (mya). L.: late. M.: middle. HPD: highest posterior density. Stderr: Standard error.</p>
<p class="siDoi">doi:10.1371/journal.pone.0014622.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0014622.s005" id="pone.0014622.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014622.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Mitochondrial substitution rates for glires. Tip rates estimated by BEAST and averaged rates given in substitutions per position per million years.</p>
<p class="siDoi">doi:10.1371/journal.pone.0014622.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0014622.s006" id="pone.0014622.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014622.s006">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">The relationships of body mass, life history traits and substitution rates were explored in linear regressions. A linear model correlated the estimated rates with lifespan and reached a significance level below 0.05 for tip as well as averaged rates. In contrast, linear models correlating rates with body mass and age at sexual maturity did not reach this significane level. Data on the body mass of the sequenced Anomalurus sp. individual were missing. Thus independent regressions were made for datasets containing Anomalurus with a body mass of 700 g and 2000 g. *: significant, &lt;0.05.</p>
<p class="siDoi">doi:10.1371/journal.pone.0014622.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0014622.s007" id="pone.0014622.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014622.s007">Table S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Life history traits of rodents and references. Since no data were available for M. kikuchii, data for M. oeconomus was used instead. Both taxa are closely related [S19]. Note that high values for lifespan could result from animals held in captivity.</p>
<p class="siDoi">doi:10.1371/journal.pone.0014622.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0014622.s008" id="pone.0014622.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014622.s008">Table S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Oligo sequences. Long range PCR primer and blocking oligos.</p>
<p class="siDoi">doi:10.1371/journal.pone.0014622.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0014622.s009" id="pone.0014622.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014622.s009">Supporting References S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Supporting References</p>
<p class="siDoi">doi:10.1371/journal.pone.0014622.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Alexander P. Saveljev for sample collection, Dietrich Heidecke and Natalia Rybczynski for fruitful discussions and Karl Andreas Nitsche for the drawing of a beaver in <a href="#pone-0014622-g004">Fig. 1</a>. We thank Matthias Meyer for help with tagging and sequencing protocols as well as Tomislav Maricic for help with the hybridization capture and two anonymous reviewers for helpful comments.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SH MH. Performed the experiments: SH. Analyzed the data: SH. Contributed reagents/materials/analysis tools: WD RW AE AS MS MH. Wrote the paper: SH WD MH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0014622-Baker1" id="pone.0014622-Baker1"></a>Baker BW, Hill EP (2003)  Beaver (<em>Castor canadensis</em>). In: Feldhamer GA, Thompson BC, Chapman JA, editors. Wild Mammals of North America: biology, management, and conservation Baltimore, MD The Johns Hopkins University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0014622-Djoshkin1" id="pone.0014622-Djoshkin1"></a>Djoshkin WW, Safonow WG (1972) Die Biber der alten und neuen Welt. Germany: Die Neue Brehm-BÃ¼cherei, A Ziemsen Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0014622-Huchon1" id="pone.0014622-Huchon1"></a>Huchon De, Chevret P, Jordan U, Kilpatrick CW, Ranwez V, et al.  (2007) Multiple molecular evidences for a living mammalian fossil. Proceedings of the National Academy of Sciences  104: 7495â7499.  <ul class="find" data-citedArticleID="2279250" data-doi="10.1073/pnas.0701289104"><li><a href="http://dx.doi.org/10.1073/pnas.0701289104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+molecular+evidences+for+a+living+mammalian+fossil." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+molecular+evidences+for+a+living+mammalian+fossil.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0014622-Adkins1" id="pone.0014622-Adkins1"></a>Adkins RM, Walton AH, Honeycutt RL (2003) Higher-level systematics of rodents and divergence time estimates based on two congruent nuclear genes. Molecular Phylogenetics and Evolution  26: 409â420.  <ul class="find" data-citedArticleID="2279296" data-doi="10.1016/s1055-7903(02)00304-4"><li><a href="http://dx.doi.org/10.1016/s1055-7903(02)00304-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Higher-level+systematics+of+rodents+and+divergence+time+estimates+based+on+two+congruent+nuclear+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Higher-level+systematics+of+rodents+and+divergence+time+estimates+based+on+two+congruent+nuclear+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0014622-BlangaKanfi1" id="pone.0014622-BlangaKanfi1"></a>Blanga-Kanfi S, Miranda H, Penn O, Pupko T, DeBry R, et al.  (2009) Rodent phylogeny revised: analysis of six nuclear genes from all major rodent clades. BMC Evolutionary Biology  9: 71.  <ul class="find" data-citedArticleID="2279308" data-doi="10.1186/1471-2148-9-71"><li><a href="http://dx.doi.org/10.1186/1471-2148-9-71" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rodent+phylogeny+revised%3A+analysis+of+six+nuclear+genes+from+all+major+rodent+clades." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rodent+phylogeny+revised%3A+analysis+of+six+nuclear+genes+from+all+major+rodent+clades.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0014622-Montgelard1" id="pone.0014622-Montgelard1"></a>Montgelard C, Forty E, Arnal V, Matthee C (2008) Suprafamilial relationships among Rodentia and the phylogenetic effect of removing fast-evolving nucleotides in mitochondrial, exon and intron fragments. BMC Evolutionary Biology  8: 321.  <ul class="find" data-citedArticleID="2279310" data-doi="10.1186/1471-2148-8-321"><li><a href="http://dx.doi.org/10.1186/1471-2148-8-321" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suprafamilial+relationships+among+Rodentia+and+the+phylogenetic+effect+of+removing+fast-evolving+nucleotides+in+mitochondrial%2C+exon+and+intron+fragments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suprafamilial+relationships+among+Rodentia+and+the+phylogenetic+effect+of+removing+fast-evolving+nucleotides+in+mitochondrial%2C+exon+and+intron+fragments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0014622-Farwick1" id="pone.0014622-Farwick1"></a>Farwick A, Jordan U, Fuellen G, Huchon D, Catzeflis F, et al.  (2006) Automated Scanning for Phylogenetically Informative Transposed Elements in Rodents. Syst Biol  55: 936â948.  <ul class="find" data-citedArticleID="2279260"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Automated+Scanning+for+Phylogenetically+Informative+Transposed+Elements+in+Rodents.&amp;auth=&amp;atitle=Automated+Scanning+for+Phylogenetically+Informative+Transposed+Elements+in+Rodents." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automated+Scanning+for+Phylogenetically+Informative+Transposed+Elements+in+Rodents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automated+Scanning+for+Phylogenetically+Informative+Transposed+Elements+in+Rodents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0014622-Churakov1" id="pone.0014622-Churakov1"></a>Churakov G, Sadasivuni MK, Rosenbloom KR, Huchon D, Brosius J, et al.  (2010) Rodent Evolution: Back to the Root. Mol Biol Evol  27: 1315â1326.  <ul class="find" data-citedArticleID="2279244" data-doi="10.1093/molbev/msq019"><li><a href="http://dx.doi.org/10.1093/molbev/msq019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rodent+Evolution%3A+Back+to+the+Root." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rodent+Evolution%3A+Back+to+the+Root.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0014622-Sieber1" id="pone.0014622-Sieber1"></a>Sieber J, Suchentrunk F, HARTL GB (1999)  A biochemical-genetic discrimination method for the two beaver species, <em>Castor fiber</em> and <em>Castor canadensis</em>, as a tool for conservation. In: Busher PE, Dziecidowski RM, editors. New York: Beaver Protection, Management, and Utilization in Europe and North America Kluver Academic/Plenum Publishers. pp. 61â65.  <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0014622-Ward1" id="pone.0014622-Ward1"></a>Ward OG, Graphodatsky AS, Wursterhill DH, Eremina VR, Park JP, et al.  (1991) Cytogenetics of beavers - a case of speciation by monobrachial centric fusions. Genome  34: 324â328.  <ul class="find" data-citedArticleID="2279268" data-doi="10.1139/g91-053"><li><a href="http://dx.doi.org/10.1139/g91-053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytogenetics+of+beavers+-+a+case+of+speciation+by+monobrachial+centric+fusions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytogenetics+of+beavers+-+a+case+of+speciation+by+monobrachial+centric+fusions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0014622-Rybczynski1" id="pone.0014622-Rybczynski1"></a>Rybczynski N (2007) Castorid Phylogenetics: Implications for the Evolution of Swimming and Tree-Exploitation in Beavers. Journal of Mammalian Evolution  14: 1â35.  <ul class="find" data-citedArticleID="2279292" data-doi="10.1007/s10914-006-9017-3"><li><a href="http://dx.doi.org/10.1007/s10914-006-9017-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Castorid+Phylogenetics%3A+Implications+for+the+Evolution+of+Swimming+and+Tree-Exploitation+in+Beavers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Castorid+Phylogenetics%3A+Implications+for+the+Evolution+of+Swimming+and+Tree-Exploitation+in+Beavers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0014622-Flynn1" id="pone.0014622-Flynn1"></a>Flynn LJ, Jacobs LL (2007) Castoroidea. In: Evolution of Tertiary Mammals of North America. Cambridge University Press  2:   <ul class="find" data-citedArticleID="2279264" data-doi="10.1017/cbo9780511541438.024"><li><a href="http://dx.doi.org/10.1017/cbo9780511541438.024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Castoroidea.+In%3A+Evolution+of+Tertiary+Mammals+of+North+America." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Castoroidea.+In%3A+Evolution+of+Tertiary+Mammals+of+North+America.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0014622-Lindsay1" id="pone.0014622-Lindsay1"></a>Lindsay EH, Opdyke ND, Johnson NM (1984) Blancan-Hemphillian Land Mammal Ages and Late Cenozoic Mammal Dispersal Events. Annual Review of Earth and Planetary Sciences  12: 445â488.  <ul class="find" data-citedArticleID="2279306" data-doi="10.1146/annurev.ea.12.050184.002305"><li><a href="http://dx.doi.org/10.1146/annurev.ea.12.050184.002305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Blancan-Hemphillian+Land+Mammal+Ages+and+Late+Cenozoic+Mammal+Dispersal+Events." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Blancan-Hemphillian+Land+Mammal+Ages+and+Late+Cenozoic+Mammal+Dispersal+Events.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0014622-Hugueney1" id="pone.0014622-Hugueney1"></a>Hugueney M, Escuillie F (1996) Fossil evidence for the origin of behavioral strategies in early Miocene Castoridae, and their role in the evolution of the family. Paleobiology  22: 507â513.  <ul class="find" data-citedArticleID="2279272"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fossil+evidence+for+the+origin+of+behavioral+strategies+in+early+Miocene+Castoridae%2C+and+their+role+in+the+evolution+of+the+family.&amp;auth=&amp;atitle=Fossil+evidence+for+the+origin+of+behavioral+strategies+in+early+Miocene+Castoridae%2C+and+their+role+in+the+evolution+of+the+family." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fossil+evidence+for+the+origin+of+behavioral+strategies+in+early+Miocene+Castoridae%2C+and+their+role+in+the+evolution+of+the+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fossil+evidence+for+the+origin+of+behavioral+strategies+in+early+Miocene+Castoridae%2C+and+their+role+in+the+evolution+of+the+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0014622-Xu1" id="pone.0014622-Xu1"></a>Xu X (1994) Evolution of Chinese Castoridae. National Science Museum monographs  8: 77â97.  <ul class="find" data-citedArticleID="2279286"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+of+Chinese+Castoridae.&amp;auth=&amp;atitle=Evolution+of+Chinese+Castoridae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+Chinese+Castoridae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+Chinese+Castoridae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0014622-Lychev1" id="pone.0014622-Lychev1"></a>Lychev G (1983) Glavne napravlenia evolutsi semeistva Castoridae. [Basic directions in the evolution of the family Castoridae]. [In:] Istoria i evolutsya sivremennoy fauny gryzunov SSSR, Nauka, Moskva (In Russian, German translation 179â203.  <ul class="find" data-citedArticleID="2279314"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Glavne+napravlenia+evolutsi+semeistva+Castoridae.+%5BBasic+directions+in+the+evolution+of+the+family+Castoridae%5D.&amp;auth=&amp;atitle=Glavne+napravlenia+evolutsi+semeistva+Castoridae.+%5BBasic+directions+in+the+evolution+of+the+family+Castoridae%5D." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glavne+napravlenia+evolutsi+semeistva+Castoridae.+%5BBasic+directions+in+the+evolution+of+the+family+Castoridae%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glavne+napravlenia+evolutsi+semeistva+Castoridae.+%5BBasic+directions+in+the+evolution+of+the+family+Castoridae%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0014622-Lavrov1" id="pone.0014622-Lavrov1"></a>Lavrov LS (1983) Evolutionary development of the genus Castor and taxonomy of the contemporary beavers in Eurasia. Acta Zool Fennica  174: 87â90.  <ul class="find" data-citedArticleID="2279304"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolutionary+development+of+the+genus+Castor+and+taxonomy+of+the+contemporary+beavers+in+Eurasia.&amp;auth=&amp;atitle=Evolutionary+development+of+the+genus+Castor+and+taxonomy+of+the+contemporary+beavers+in+Eurasia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+development+of+the+genus+Castor+and+taxonomy+of+the+contemporary+beavers+in+Eurasia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+development+of+the+genus+Castor+and+taxonomy+of+the+contemporary+beavers+in+Eurasia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0014622-Horner1" id="pone.0014622-Horner1"></a>Horner D, Lefkimmiatis K, Reyes A, Gissi C, Saccone C, et al.  (2007) Phylogenetic analyses of complete mitochondrial genome sequences suggest a basal divergence of the enigmatic rodent Anomalurus. BMC Evolutionary Biology  7: 16.  <ul class="find" data-citedArticleID="2279300"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phylogenetic+analyses+of+complete+mitochondrial+genome+sequences+suggest+a+basal+divergence+of+the+enigmatic+rodent+Anomalurus.&amp;auth=&amp;atitle=Phylogenetic+analyses+of+complete+mitochondrial+genome+sequences+suggest+a+basal+divergence+of+the+enigmatic+rodent+Anomalurus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analyses+of+complete+mitochondrial+genome+sequences+suggest+a+basal+divergence+of+the+enigmatic+rodent+Anomalurus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analyses+of+complete+mitochondrial+genome+sequences+suggest+a+basal+divergence+of+the+enigmatic+rodent+Anomalurus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0014622-Lin1" id="pone.0014622-Lin1"></a>Lin Y-H, Waddell PJ, Penny D (2002) Pika and vole mitochondrial genomes increase support for both rodent monophyly and glires. Gene  294: 119â129.  <ul class="find" data-citedArticleID="2279330" data-doi="10.1016/s0378-1119(02)00695-9"><li><a href="http://dx.doi.org/10.1016/s0378-1119(02)00695-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pika+and+vole+mitochondrial+genomes+increase+support+for+both+rodent+monophyly+and+glires." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pika+and+vole+mitochondrial+genomes+increase+support+for+both+rodent+monophyly+and+glires.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0014622-Drummond1" id="pone.0014622-Drummond1"></a>Drummond AJ, Ho SYW, Phillips MJ, Rambaut A (2006) Relaxed Phylogenetics and Dating with Confidence. PLoS Biology  4: e88.  <ul class="find" data-citedArticleID="2279248" data-doi="10.1371/journal.pbio.0040088"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relaxed+Phylogenetics+and+Dating+with+Confidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relaxed+Phylogenetics+and+Dating+with+Confidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0014622-Welch1" id="pone.0014622-Welch1"></a>Welch J, Bininda-Emonds O, Bromham L (2008) Correlates of substitution rate variation in mammalian protein-coding sequences. BMC Evolutionary Biology  8: 53.  <ul class="find" data-citedArticleID="2279328" data-doi="10.1186/1471-2148-8-53"><li><a href="http://dx.doi.org/10.1186/1471-2148-8-53" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlates+of+substitution+rate+variation+in+mammalian+protein-coding+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlates+of+substitution+rate+variation+in+mammalian+protein-coding+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0014622-Thomas1" id="pone.0014622-Thomas1"></a>Thomas JA, Welch JJ, Woolfit M, Bromham L (2006) There is no universal molecular clock for invertebrates, but rate variation does not scale with body size. Proc Natl Acad Sci USA  103: 7366â7371.  <ul class="find" data-citedArticleID="2279344" data-doi="10.1073/pnas.0510251103"><li><a href="http://dx.doi.org/10.1073/pnas.0510251103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=There+is+no+universal+molecular+clock+for+invertebrates%2C+but+rate+variation+does+not+scale+with+body+size." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22There+is+no+universal+molecular+clock+for+invertebrates%2C+but+rate+variation+does+not+scale+with+body+size.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0014622-Kjer1" id="pone.0014622-Kjer1"></a>Kjer K, Honeycutt R (2007) Site specific rates of mitochondrial genomes and the phylogeny of eutheria. BMC Evolutionary Biology  7: 8.  <ul class="find" data-citedArticleID="2279334"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Site+specific+rates+of+mitochondrial+genomes+and+the+phylogeny+of+eutheria.&amp;auth=&amp;atitle=Site+specific+rates+of+mitochondrial+genomes+and+the+phylogeny+of+eutheria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Site+specific+rates+of+mitochondrial+genomes+and+the+phylogeny+of+eutheria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Site+specific+rates+of+mitochondrial+genomes+and+the+phylogeny+of+eutheria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0014622-Herv1" id="pone.0014622-Herv1"></a>HervÃ© P (1997) Rodent Monophyly: Pitfalls of Molecular Phylogenies. Journal of Molecular Evolution  45: 712â715.  <ul class="find" data-citedArticleID="2279326"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rodent+Monophyly%3A+Pitfalls+of+Molecular+Phylogenies.&amp;auth=&amp;atitle=Rodent+Monophyly%3A+Pitfalls+of+Molecular+Phylogenies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rodent+Monophyly%3A+Pitfalls+of+Molecular+Phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rodent+Monophyly%3A+Pitfalls+of+Molecular+Phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0014622-Huchon2" id="pone.0014622-Huchon2"></a>Huchon D, Madsen O, Sibbald MJJ, Ament K, Stanhope MJ, et al.  (2002) Rodent phylogeny and a timescale for the evolution of Glires: evidence from an extensive taxon sampling using three nuclear genes. Mol Biol Evol  19: 1053â1065.  <ul class="find" data-citedArticleID="2279246" data-doi="10.1093/oxfordjournals.molbev.a004164"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004164" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rodent+phylogeny+and+a+timescale+for+the+evolution+of+Glires%3A+evidence+from+an+extensive+taxon+sampling+using+three+nuclear+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rodent+phylogeny+and+a+timescale+for+the+evolution+of+Glires%3A+evidence+from+an+extensive+taxon+sampling+using+three+nuclear+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0014622-Adkins2" id="pone.0014622-Adkins2"></a>Adkins RM, Walton AH, Honeycutt RL (2003) Higher-level systematics of rodents and divergence time estimates based on two congruent nuclear genes. Mol Phylogenet Evol  26: 409â420.  <ul class="find" data-citedArticleID="2279262" data-doi="10.1016/s1055-7903(02)00304-4"><li><a href="http://dx.doi.org/10.1016/s1055-7903(02)00304-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Higher-level+systematics+of+rodents+and+divergence+time+estimates+based+on+two+congruent+nuclear+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Higher-level+systematics+of+rodents+and+divergence+time+estimates+based+on+two+congruent+nuclear+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0014622-Wood1" id="pone.0014622-Wood1"></a>Wood AE (1959) Eocene Radiation and Phylogeny of the Rodents. Evolution  13: 354â361.  <ul class="find" data-citedArticleID="2279282" data-doi="10.2307/2406112"><li><a href="http://dx.doi.org/10.2307/2406112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eocene+Radiation+and+Phylogeny+of+the+Rodents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eocene+Radiation+and+Phylogeny+of+the+Rodents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0014622-Repenning1" id="pone.0014622-Repenning1"></a>Repenning CA (1987)  Biochronology of the microtine rodents of the United States. In: Woodburne MO, editor. Berkeley: Cenozoic Mammals of North America: Geochronology and Biostratigraphy University of California Press. pp. 236â268.  <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0014622-Gladenkov1" id="pone.0014622-Gladenkov1"></a>Gladenkov AY, Oleinik AE, Marincovich L, Barinov KB (2002) A refined age for the earliest opening of Bering Strait. Palaeogeography, Palaeoclimatology, Palaeoecology  183: 321â328.  <ul class="find" data-citedArticleID="2279312" data-doi="10.1016/s0031-0182(02)00249-3"><li><a href="http://dx.doi.org/10.1016/s0031-0182(02)00249-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+refined+age+for+the+earliest+opening+of+Bering+Strait." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+refined+age+for+the+earliest+opening+of+Bering+Strait.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0014622-Veron1" id="pone.0014622-Veron1"></a>Veron G (1992) Biogeographical history of the European beaver <em>Castor fiber</em> (Rodentia, Mammlia). Mammalia  56: 87â108.  <ul class="find" data-citedArticleID="2279346"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biogeographical+history+of+the+European+beaver+Castor+fiber+%28Rodentia%2C+Mammlia%29.&amp;auth=&amp;atitle=Biogeographical+history+of+the+European+beaver+Castor+fiber+%28Rodentia%2C+Mammlia%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biogeographical+history+of+the+European+beaver+Castor+fiber+%28Rodentia%2C+Mammlia%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biogeographical+history+of+the+European+beaver+Castor+fiber+%28Rodentia%2C+Mammlia%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0014622-Durka1" id="pone.0014622-Durka1"></a>Durka W, Babik W, Ducroz J-F, Heidecke D, Rosell F, et al.  (2005) Mitochondrial phylogeography of the Eurasian beaver <em>Castor fiber</em> L. Molecular Ecology  14: 3843â3856.  <ul class="find" data-citedArticleID="2279316" data-doi="10.1111/j.1365-294x.2005.02704.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2005.02704.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+phylogeography+of+the+Eurasian+beaver+Castor+fiber+L." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+phylogeography+of+the+Eurasian+beaver+Castor+fiber+L.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0014622-Ho1" id="pone.0014622-Ho1"></a>Ho SYW, Saarma U, Barnett R, Haile J, Shapiro B (2008) The Effect of Inappropriate Calibration: Three Case Studies in Molecular Ecology. PLoS ONE  3: e1615.  <ul class="find" data-citedArticleID="2279276" data-doi="10.1371/journal.pone.0001615"><li><a href="http://dx.doi.org/10.1371/journal.pone.0001615" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Effect+of+Inappropriate+Calibration%3A+Three+Case+Studies+in+Molecular+Ecology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Effect+of+Inappropriate+Calibration%3A+Three+Case+Studies+in+Molecular+Ecology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0014622-Ho2" id="pone.0014622-Ho2"></a>Ho SYW, Shapiro B, Phillips MJ, Cooper A, Drummond AJ (2007) Evidence for Time Dependency of Molecular Rate Estimates. Systematic Biology  56: 515â522.  <ul class="find" data-citedArticleID="2279270"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evidence+for+Time+Dependency+of+Molecular+Rate+Estimates.&amp;auth=&amp;atitle=Evidence+for+Time+Dependency+of+Molecular+Rate+Estimates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+Time+Dependency+of+Molecular+Rate+Estimates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+Time+Dependency+of+Molecular+Rate+Estimates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0014622-Knapp1" id="pone.0014622-Knapp1"></a>Knapp M, Rohland N, Weinstock J, Baryshnikov G, Sher A, et al.  (2009) First DNA sequences from Asian cave bear fossils reveal deep divergences and complex phylogeographic patterns. Molecular Ecology  18: 1225â1238.  <ul class="find" data-citedArticleID="2279338" data-doi="10.1111/j.1365-294x.2009.04088.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2009.04088.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=First+DNA+sequences+from+Asian+cave+bear+fossils+reveal+deep+divergences+and+complex+phylogeographic+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22First+DNA+sequences+from+Asian+cave+bear+fossils+reveal+deep+divergences+and+complex+phylogeographic+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0014622-Saarma1" id="pone.0014622-Saarma1"></a>Saarma U, Ho SYW, Pybus OG, Kaljuste M, Tumanov IL, et al.  (2007) Mitogenetic structure of brown bears (Ursus arctos L.) in northeastern Europe and a new time frame for the formation of European brown bear lineages. Molecular Ecology  16: 401â413.  <ul class="find" data-citedArticleID="2279252" data-doi="10.1111/j.1365-294x.2006.03130.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2006.03130.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitogenetic+structure+of+brown+bears+%28Ursus+arctos+L.%29+in+northeastern+Europe+and+a+new+time+frame+for+the+formation+of+European+brown+bear+lineages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitogenetic+structure+of+brown+bears+%28Ursus+arctos+L.%29+in+northeastern+Europe+and+a+new+time+frame+for+the+formation+of+European+brown+bear+lineages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0014622-Hewitt1" id="pone.0014622-Hewitt1"></a>Hewitt G (2000) The genetic legacy of the Quaternary ice ages. Nature  405: 907â913.  <ul class="find" data-citedArticleID="2279322" data-doi="10.1038/35016000"><li><a href="http://dx.doi.org/10.1038/35016000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+legacy+of+the+Quaternary+ice+ages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+legacy+of+the+Quaternary+ice+ages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0014622-Galtier1" id="pone.0014622-Galtier1"></a>Galtier N, Jobson R, Nabholz B, GlÃ©min S, Blier P (2009) Mitochondrial whims: metabolic rate, longevity and the rate of molecular evolution. Biological Letters  5: 413â416.  <ul class="find" data-citedArticleID="2279256" data-doi="10.1098/rsbl.2008.0662"><li><a href="http://dx.doi.org/10.1098/rsbl.2008.0662" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+whims%3A+metabolic+rate%2C+longevity+and+the+rate+of+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+whims%3A+metabolic+rate%2C+longevity+and+the+rate+of+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0014622-Nabholz1" id="pone.0014622-Nabholz1"></a>Nabholz B, Glemin S, Galtier N (2008) Strong Variations of Mitochondrial Mutation Rate across Mammals-the Longevity Hypothesis. Molecular Biology and Evolution  25: 120â130.  <ul class="find" data-citedArticleID="2279254" data-doi="10.1093/molbev/msm248"><li><a href="http://dx.doi.org/10.1093/molbev/msm248" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+Variations+of+Mitochondrial+Mutation+Rate+across+Mammals-the+Longevity+Hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+Variations+of+Mitochondrial+Mutation+Rate+across+Mammals-the+Longevity+Hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0014622-Jobson1" id="pone.0014622-Jobson1"></a>Jobson RW, Nabholz B, Galtier N (2010) An Evolutionary Genome Scan for Longevity-Related Natural Selection in Mammals. Molecular Biology and Evolution  27: 840â847.  <ul class="find" data-citedArticleID="2279284" data-doi="10.1093/molbev/msp293"><li><a href="http://dx.doi.org/10.1093/molbev/msp293" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Evolutionary+Genome+Scan+for+Longevity-Related+Natural+Selection+in+Mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Evolutionary+Genome+Scan+for+Longevity-Related+Natural+Selection+in+Mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0014622-Animal1" id="pone.0014622-Animal1"></a>Animal C, Use C (1998) Guidelines for the Capture, Handling, and Care of Mammals as Approved by the American Society of Mammalogists. Journal of Mammalogy  79: 1416â1431.  <ul class="find" data-citedArticleID="2279266" data-doi="10.2307/1383033"><li><a href="http://dx.doi.org/10.2307/1383033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Guidelines+for+the+Capture%2C+Handling%2C+and+Care+of+Mammals+as+Approved+by+the+American+Society+of+Mammalogists." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Guidelines+for+the+Capture%2C+Handling%2C+and+Care+of+Mammals+as+Approved+by+the+American+Society+of+Mammalogists.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0014622-Meyer1" id="pone.0014622-Meyer1"></a>Meyer M, Stenzel U, Hofreiter M (2008) Parallel tagged sequencing on the 454 platform. Nat Protocols  3: 267â278.  <ul class="find" data-citedArticleID="2279324" data-doi="10.1038/nprot.2007.520"><li><a href="http://dx.doi.org/10.1038/nprot.2007.520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+tagged+sequencing+on+the+454+platform." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+tagged+sequencing+on+the+454+platform.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0014622-Noonan1" id="pone.0014622-Noonan1"></a>Noonan JP, Coop G, Kudaravalli S, Smith D, Krause J, et al.  (2006) Sequencing and analysis of Neanderthal genomic DNA. Science  314: 1113â1118.  <ul class="find" data-citedArticleID="2279320" data-doi="10.1126/science.1131412"><li><a href="http://dx.doi.org/10.1126/science.1131412" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+and+analysis+of+Neanderthal+genomic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+and+analysis+of+Neanderthal+genomic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0014622-Stiller1" id="pone.0014622-Stiller1"></a>Stiller M, Knapp M, Stenzel U, Hofreiter M, Meyer M (2009) Direct multiplex sequencing (DMPS): A novel method for targeted high-throughput sequencing of ancient and highly degraded DNA. Genome Research  19: 1843â1848.  <ul class="find" data-citedArticleID="2279258" data-doi="10.1101/gr.095760.109"><li><a href="http://dx.doi.org/10.1101/gr.095760.109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+multiplex+sequencing+%28DMPS%29%3A+A+novel+method+for+targeted+high-throughput+sequencing+of+ancient+and+highly+degraded+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+multiplex+sequencing+%28DMPS%29%3A+A+novel+method+for+targeted+high-throughput+sequencing+of+ancient+and+highly+degraded+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0014622-Meyer2" id="pone.0014622-Meyer2"></a>Meyer M, Briggs AW, Maricic T, Hober B, Hoffner B, et al.  (2008) From micrograms to picograms: quantitative PCR reduces the material demands of high-throughput sequencing. Nucl Acids Res  36: e5.  <ul class="find" data-citedArticleID="2279290" data-doi="10.1093/nar/gkm1095"><li><a href="http://dx.doi.org/10.1093/nar/gkm1095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+micrograms+to+picograms%3A+quantitative+PCR+reduces+the+material+demands+of+high-throughput+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+micrograms+to+picograms%3A+quantitative+PCR+reduces+the+material+demands+of+high-throughput+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0014622-Hall1" id="pone.0014622-Hall1"></a>Hall TA (1999) BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT. Nucleic Acids Symposium Series  41: 95â98.  <ul class="find" data-citedArticleID="2279278"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=BioEdit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT.&amp;auth=&amp;atitle=BioEdit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BioEdit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BioEdit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0014622-Green1" id="pone.0014622-Green1"></a>Green R (2009) An iterative, mapping assembly strategy for next-generation sequencing data enables assembly of otherwise intractable datasets. Cold Spring Harbor Laboratory meeting âThe Biology of Genomesâ.   <ul class="find-nolinks"></ul></li><li><span class="label">47.
              </span><a name="pone.0014622-Katoh1" id="pone.0014622-Katoh1"></a>Katoh K, Toh H (2008) Recent developments in the MAFFT multiple sequence alignment program. Brief Bioinform  9: 286â298.  <ul class="find" data-citedArticleID="2279288" data-doi="10.1093/bib/bbn013"><li><a href="http://dx.doi.org/10.1093/bib/bbn013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+developments+in+the+MAFFT+multiple+sequence+alignment+program." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+developments+in+the+MAFFT+multiple+sequence+alignment+program.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0014622-Posada1" id="pone.0014622-Posada1"></a>Posada D, Crandall K (1998) MODELTEST: testing the model of DNA substitution. Bioinformatics  14: 817â818.  <ul class="find" data-citedArticleID="2279302" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MODELTEST%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MODELTEST%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0014622-Swofford1" id="pone.0014622-Swofford1"></a>Swofford DL (1998) PAUP*. Phylogenetic Analysis Using Parsimony (*and Other Methods). 4 ed. Sunderland: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0014622-Kumar1" id="pone.0014622-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment. Brief Bioinform  5: 150â163.  <ul class="find" data-citedArticleID="2279298" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0014622-Rambaut1" id="pone.0014622-Rambaut1"></a>Rambaut A (2006)   <a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtreâe/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">52.
              </span><a name="pone.0014622-Drummond2" id="pone.0014622-Drummond2"></a>Drummond A, Rambaut A (2007) BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evolutionary Biology  7: 214.  <ul class="find" data-citedArticleID="2279332" data-doi="10.1186/1471-2148-7-214"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-214" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BEAST%3A+Bayesian+evolutionary+analysis+by+sampling+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BEAST%3A+Bayesian+evolutionary+analysis+by+sampling+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0014622-Rambaut2" id="pone.0014622-Rambaut2"></a>Rambaut A, Drummond A (2007)   Tracer v1.4, Available from <a href="http://beast.bio.ed.ac.uk/Tracer">http://beast.bio.ed.ac.uk/Tracer</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">54.
              </span><a name="pone.0014622-Emerson1" id="pone.0014622-Emerson1"></a>Emerson BC (2007) Alarm Bells for the Molecular Clock? No Support for Ho et al.'s Model of Time-Dependent Molecular Rate Estimates. Syst Biol  56: 337â345.  <ul class="find" data-citedArticleID="2279280"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Alarm+Bells+for+the+Molecular+Clock%3F+No+Support+for+Ho+et+al.%27s+Model+of+Time-Dependent+Molecular+Rate+Estimates.&amp;auth=&amp;atitle=Alarm+Bells+for+the+Molecular+Clock%3F+No+Support+for+Ho+et+al.%27s+Model+of+Time-Dependent+Molecular+Rate+Estimates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alarm+Bells+for+the+Molecular+Clock%3F+No+Support+for+Ho+et+al.%27s+Model+of+Time-Dependent+Molecular+Rate+Estimates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alarm+Bells+for+the+Molecular+Clock%3F+No+Support+for+Ho+et+al.%27s+Model+of+Time-Dependent+Molecular+Rate+Estimates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0014622-Halley1" id="pone.0014622-Halley1"></a>Halley DJ, Rosell F (2003) Population and distribution of European beavers (Castor fiber). Lutra  46: 91â101.  <ul class="find" data-citedArticleID="2279318"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Population+and+distribution+of+European+beavers+%28Castor+fiber%29.&amp;auth=&amp;atitle=Population+and+distribution+of+European+beavers+%28Castor+fiber%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+and+distribution+of+European+beavers+%28Castor+fiber%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+and+distribution+of+European+beavers+%28Castor+fiber%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.XML" value="109622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.PDF" value="835515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g001.PNG_L" value="504301"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g001.PNG_M" value="119121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g001.PNG_S" value="8924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g001.TIF" value="685316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g001.PNG_I" value="17711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g002.PNG_L" value="159126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g002.PNG_M" value="175736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g002.PNG_S" value="10230"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g002.TIF" value="438958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g002.PNG_I" value="26816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g003.PNG_L" value="411599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g003.PNG_M" value="279015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g003.PNG_S" value="14053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g003.TIF" value="650984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g003.PNG_I" value="33711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g004.PNG_L" value="141101"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g004.PNG_M" value="101661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g004.PNG_S" value="8118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g004.TIF" value="3111216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g004.PNG_I" value="13645"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.t001.PNG_L" value="236856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.t001.PNG_M" value="58642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.t001.PNG_S" value="7731"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.t001.TIF" value="4008060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.t001.PNG_I" value="6224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g005.PNG_L" value="47250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g005.PNG_M" value="52340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g005.PNG_S" value="6218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g005.TIF" value="86148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.g005.PNG_I" value="9144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.s001.DOC" value="44032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.s002.DOC" value="63488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.s003.DOC" value="39424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.s004.DOC" value="52736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.s005.DOC" value="64000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.s006.DOC" value="43520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.s007.DOC" value="65024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.s008.DOC" value="49664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0014622.s009.DOC" value="33280"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0014622&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0014622" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0014622&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0014622&volume=&issue=&title=Mitochondrial Genomes Reveal Slow Rates of Molecular Evolution and the Timing of Speciation in Beavers (Castor), One of the Largest Rodent Species&author_name=Susanne%20Horn%2C%20Walter%20Durka%2C%20Ronny%20Wolf%2C%20Aslak%20Ermala%2C%20Annegret%20Stubbe%2C%20Michael%20Stubbe%2C%20Michael%20Hofreiter&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014622" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014622" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0014622" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014622&amp;t=Mitochondrial%20Genomes%20Reveal%20Slow%20Rates%20of%20Molecular%20Evolution%20and%20the%20Timing%20of%20Speciation%20in%20Beavers%20(Castor)%2C%20One%20of%20the%20Largest%20Rodent%20Species" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014622&title=Mitochondrial%20Genomes%20Reveal%20Slow%20Rates%20of%20Molecular%20Evolution%20and%20the%20Timing%20of%20Speciation%20in%20Beavers%20(Castor)%2C%20One%20of%20the%20Largest%20Rodent%20Species&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014622&amp;title=Mitochondrial%20Genomes%20Reveal%20Slow%20Rates%20of%20Molecular%20Evolution%20and%20the%20Timing%20of%20Speciation%20in%20Beavers%20(Castor)%2C%20One%20of%20the%20Largest%20Rodent%20Species" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014622" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0014622&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Mitochondrial Genomes Reveal Slow Rates of Molecular Evolution and the Timing of Speciation in Beavers (Castor), One of the Largest Rodent Species';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0014622';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Mitochondrial%20Genomes%20Reveal%20Slow%20Rates%20of%20Molecular%20Evolution%20and%20the%20Timing%20of%20Speciation%20in%20Beavers%20(Castor)%2C%20One%20of%20the%20Largest%20Rodent%20Species http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014622" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0014622" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Beavers%22" title="Search for articles in the subject area:'Beavers'"><div class="flagText">Beavers</div></a>
              <div data-categoryid="45311" data-articleid="52054"
                   data-categoryname="Beavers"
                   class="flagImage" title="Flag 'Beavers' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="52054"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="52054"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleogenetics%22" title="Search for articles in the subject area:'Paleogenetics'"><div class="flagText">Paleogenetics</div></a>
              <div data-categoryid="33815" data-articleid="52054"
                   data-categoryname="Paleogenetics"
                   class="flagImage" title="Flag 'Paleogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="52054"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="52054"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Rodents%22" title="Search for articles in the subject area:'Rodents'"><div class="flagText">Rodents</div></a>
              <div data-categoryid="42763" data-articleid="52054"
                   data-categoryname="Rodents"
                   class="flagImage" title="Flag 'Rodents' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="52054"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1099'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7092'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1018&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
