

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The Genome Sequence of a Widespread Apex Predator, the Golden Eagle (Aquila chrysaetos)</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0095599"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0095599" />

    <meta name="citation_title" content="The Genome Sequence of a Widespread Apex Predator, the Golden Eagle (Aquila chrysaetos)"/>
    <meta itemprop="name" content="The Genome Sequence of a Widespread Apex Predator, the Golden Eagle (Aquila chrysaetos)"/>

      <meta name="citation_author" content="Jacqueline M. Doyle"/>
            <meta name="citation_author_institution" content="Department of Forestry and Natural Resources, Purdue University, West Lafayette, Indiana, United States of America"/>
      <meta name="citation_author" content="Todd E. Katzner"/>
            <meta name="citation_author_institution" content="Division of Forestry and Natural Resources, West Virginia University, Morgantown, West Virginia, United States of America"/>
            <meta name="citation_author_institution" content="Northern Research Station, USDA Forest Service, Parsons, West Virginia, United States of America"/>
      <meta name="citation_author" content="Peter H. Bloom"/>
            <meta name="citation_author_institution" content="Western Foundation of Vertebrate Zoology, Camarillo, California, United States of America"/>
      <meta name="citation_author" content="Yanzhu Ji"/>
            <meta name="citation_author_institution" content="Department of Forestry and Natural Resources, Purdue University, West Lafayette, Indiana, United States of America"/>
      <meta name="citation_author" content="Bhagya K. Wijayawardena"/>
            <meta name="citation_author_institution" content="Department of Forestry and Natural Resources, Purdue University, West Lafayette, Indiana, United States of America"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Purdue University, West Lafayette, Indiana, United States of America"/>
      <meta name="citation_author" content="J. Andrew DeWoody"/>
            <meta name="citation_author_institution" content="Department of Forestry and Natural Resources, Purdue University, West Lafayette, Indiana, United States of America"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Purdue University, West Lafayette, Indiana, United States of America"/>

    <meta name="citation_date" content="2014/4/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0095599.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e95599"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="9"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_number=1; " />
      <meta name="citation_reference" content="citation_number=2; " />
      <meta name="citation_reference" content="citation_title=Direct and indirect effects: Interactions between Bald Eagles and Common Murres; citation_volume=11; citation_number=3; citation_pages=1858-1869; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=First documented predation of Sika deer (Cervus nippon) by Golden Eagle (Aquila chrysaetos) in Russian far east; citation_volume=47; citation_number=4; citation_pages=328-330; citation_date=2013; " />
      <meta name="citation_reference" content="citation_number=5; " />
      <meta name="citation_reference" content="citation_number=6; " />
      <meta name="citation_reference" content="citation_title=The food habits of North American Golden Eagles; citation_volume=95; citation_number=7; citation_pages=231-236; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=Golden eagles in the US and Canada: Status, trends, and conservation challenges; citation_volume=36; citation_number=8; citation_pages=32-40; citation_date=2002; " />
      <meta name="citation_reference" content="citation_number=9; " />
      <meta name="citation_reference" content="citation_number=10; " />
      <meta name="citation_reference" content="citation_title=The effects of disturbance on nesting Wedge-tailed eagles (Aquila audax fleayi) in Tasmania; citation_volume=3; citation_number=11; citation_pages=15-31; citation_date=1991; " />
      <meta name="citation_reference" content="citation_number=12; " />
      <meta name="citation_reference" content="citation_number=13; " />
      <meta name="citation_reference" content="citation_title=Eighteen polymorphic microsatellite markers for the highly endangered Spanish imperial eagle (Aquila adalberti) and related species; citation_volume=2; citation_number=14; citation_pages=323-326; citation_date=2002; " />
      <meta name="citation_reference" content="citation_number=15; " />
      <meta name="citation_reference" content="citation_number=16; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of eleven microsatellite loci in an endangered species, Mountain Hawk-Eagle (Spizaetus nipalensis); citation_volume=2; citation_number=17; citation_pages=113-115; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Complete mitochondrial genome of the Spilornis cheela (Falconiformes, Accipitridae): Comparison of S. cheela and Spizaetus alboniger; citation_volume=24; citation_number=18; citation_pages=255-261; citation_date=2013; " />
      <meta name="citation_reference" content="citation_number=19; " />
      <meta name="citation_reference" content="citation_number=20; " />
      <meta name="citation_reference" content="citation_number=21; " />
      <meta name="citation_reference" content="citation_title=Population trends of migratory raptors in Western North America, 1977–2001; citation_volume=105; citation_number=22; citation_pages=397-419; citation_date=2003; " />
      <meta name="citation_reference" content="citation_number=23; " />
      <meta name="citation_reference" content="citation_number=24; " />
      <meta name="citation_reference" content="citation_number=25; " />
      <meta name="citation_reference" content="citation_title=A simple and universal method for molecular sexing of non-ratite birds; citation_volume=30; citation_number=26; citation_pages=116-121; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics; citation_volume=40; citation_number=27; citation_pages=W622-W627; citation_date=2012; " />
      <meta name="citation_reference" content="citation_number=28; " />
      <meta name="citation_reference" content="citation_title=Assessing the gene space in draft genomes; citation_volume=37; citation_number=29; citation_pages=298-297; citation_date=2009; " />
      <meta name="citation_reference" content="citation_number=30; " />
      <meta name="citation_reference" content="citation_number=31; " />
      <meta name="citation_reference" content="citation_number=32; " />
      <meta name="citation_reference" content="citation_number=33; " />
      <meta name="citation_reference" content="citation_title=Automatic annotation of organellar genomes with DOGMA; citation_volume=20; citation_number=34; citation_pages=3252-3255; citation_date=2004; " />
      <meta name="citation_reference" content="citation_number=35; " />
      <meta name="citation_reference" content="citation_title=Peregrine and saker falcon genome sequences provide insights into evolution of a predatory lifestyle; citation_volume=45; citation_number=36; citation_pages=536-566; citation_date=2013; " />
      <meta name="citation_reference" content="citation_title=Trinity: reconstructing a full-length transcriptome without a genome from RNA-Seq data; citation_volume=29; citation_number=37; citation_pages=644-652; citation_date=2013; " />
      <meta name="citation_reference" content="citation_number=38; " />
      <meta name="citation_reference" content="citation_number=39; " />
      <meta name="citation_reference" content="citation_title=Gene finding in novel genomes; citation_volume=5; citation_number=40; citation_pages=59; citation_date=2004; " />
      <meta name="citation_reference" content="citation_number=41; " />
      <meta name="citation_reference" content="citation_number=42; " />
      <meta name="citation_reference" content="citation_title=Discovering and detecting transposable elements in genome sequences; citation_volume=8; citation_number=43; citation_pages=382-392; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Transposable elements and their identification; citation_volume=1; citation_number=44; citation_pages=337-359; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Identifying repeats and transposable elements in sequenced genomes: how to find your way through the dense forest of programs; citation_volume=104; citation_number=45; citation_pages=520-533; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Computation approaches and tools used in identification of dispersed repetitive DNA sequences; citation_volume=1; citation_number=46; citation_pages=85-96; citation_date=2008; " />
      <meta name="citation_reference" content="citation_number=47; " />
      <meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences; citation_volume=27; citation_number=48; citation_pages=573-580; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Levels of linkage disequilibrium in a wild bird population; citation_volume=2; citation_number=49; citation_pages=435-438; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic diversity and evolution of the head crest in the rock pigeon; citation_volume=339; citation_number=50; citation_pages=1063-1067; citation_date=2013; " />
      <meta name="citation_reference" content="citation_number=51; " />
      <meta name="citation_reference" content="citation_title=Apollo: a sequence annotation editor; citation_volume=3; citation_number=52; citation_pages=1-14; citation_date=2002; " />
      <meta name="citation_reference" content="citation_number=53; " />
      <meta name="citation_reference" content="citation_title=The sequence alignment/map (SAM) format and SAMtools; citation_volume=25; citation_number=54; citation_pages=2078-2079; citation_date=2009; " />
      <meta name="citation_reference" content="citation_number=55; " />
      <meta name="citation_reference" content="citation_number=56; " />
      <meta name="citation_reference" content="citation_number=57; " />
      <meta name="citation_reference" content="citation_number=58; " />
      <meta name="citation_reference" content="citation_title=On the evolution of genome size of birds; citation_volume=82; citation_number=59; citation_pages=363-368; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Size and structure of the bird genome–I. DNA content of 48 species of Neognathae; citation_volume=85; citation_number=60; citation_pages=61-65; citation_date=1986; " />
      <meta name="citation_reference" content="citation_number=61; " />
      <meta name="citation_reference" content="citation_number=62; " />
      <meta name="citation_reference" content="citation_number=63; " />
      <meta name="citation_reference" content="citation_title=Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution; citation_volume=432; citation_number=64; citation_pages=695-716; citation_date=2004; " />
      <meta name="citation_reference" content="citation_number=65; " />
      <meta name="citation_reference" content="citation_number=66; " />
      <meta name="citation_reference" content="citation_number=67; " />
      <meta name="citation_reference" content="citation_title=Applications of mitochondrial DNA analysis in conservation: a critical review; citation_volume=3; citation_number=68; citation_pages=401-411; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Implications of different species concepts for conserving biodiversity; citation_volume=153; citation_number=69; citation_pages=25-31; citation_date=2012; " />
      <meta name="citation_reference" content="citation_number=70; " />
      <meta name="citation_reference" content="citation_number=71; " />
      <meta name="citation_reference" content="citation_number=72; " />
      <meta name="citation_reference" content="citation_number=73; " />
      <meta name="citation_reference" content="citation_title=Avian visual pigments: characteristics, spectral tuning, and evolution; citation_volume=169; citation_number=74; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Attraction of kestrels to vole scent marks visible in ultraviolet light; citation_volume=373; citation_number=75; citation_pages=425-426; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Assessment of compositional heterogeneity within and between eukaryotic genomes; citation_volume=10; citation_number=76; citation_pages=1986-1995; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Microsatellite analyses across three diverse vertebrate transcriptomes (Acipenser fulvescens, Ambystoma tigrinum, and Dipodomys spectabilis); citation_volume=56; citation_number=77; citation_pages=1-8; citation_date=2013; " />
      <meta name="citation_reference" content="citation_title=Non-invasive sampling and DNA amplification for paternity exclusion, community structure, and phylogeography in wild chimpanzees; citation_volume=34; citation_number=78; citation_pages=347-356; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Wild gibbons’ parentage tested by non-invasive DNA sampling and PCR-amplified polymorphic microsatellites; citation_volume=42; citation_number=79; citation_pages=67-73; citation_date=2001; " />
      <meta name="citation_reference" content="citation_number=80; " />
      <meta name="citation_reference" content="citation_number=81; " />
      <meta name="citation_reference" content="citation_number=82; " />
      <meta name="citation_reference" content="citation_number=83; " />
      <meta name="citation_reference" content="citation_number=84; " />
      <meta name="citation_reference" content="citation_number=85; " />
      <meta name="citation_reference" content="citation_number=86; " />
      <meta name="citation_reference" content="citation_number=87; " />
      <meta name="citation_reference" content="citation_number=88; " />
      <meta name="citation_reference" content="citation_number=89; " />
      <meta name="citation_reference" content="citation_title=The relationship between synaptonemal complex length and genome size in four vertebrate classes; citation_volume=2; citation_number=90; citation_pages=153-162; citation_date=1994; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0095599" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The Genome Sequence of a Widespread Apex Predator, the Golden Eagle (Aquila chrysaetos)"/>
    <meta name="twitter:description" content="Biologists routinely use molecular markers to identify conservation units, to quantify genetic connectivity, to estimate population sizes, and to identify targets of selection. Many imperiled eagle populations require such efforts and would benefit from enhanced genomic resources. We sequenced, assembled, and annotated the first eagle genome using DNA from a male golden eagle (Aquila chrysaetos) captured in western North America. We constructed genomic libraries that were sequenced using Illumina technology and assembled the high-quality data to a depth of &sim;40x coverage. The genome assembly includes 2,552 scaffolds &gt;10 Kb and 415 scaffolds &gt;1.2 Mb. We annotated 16,571 genes that are involved in myriad biological processes, including such disparate traits as beak formation and color vision. We also identified repetitive regions spanning 92 Mb (&sim;6% of the assembly), including LINES, SINES, LTR-RTs and DNA transposons. The mitochondrial genome encompasses 17,332 bp and is &sim;91% identical to the Mountain Hawk-Eagle (Nisaetus nipalensis). Finally, the data reveal that several anonymous microsatellites commonly used for population studies are embedded within protein-coding genes and thus may not have evolved in a neutral fashion. Because the genome sequence includes &sim;800,000 novel polymorphisms, markers can now be chosen based on their proximity to functional genes involved in migration, carnivory, and other biological processes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0095599.g003"/>

  <meta property="og:title" content="The Genome Sequence of a Widespread Apex Predator, the Golden Eagle (Aquila chrysaetos)" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0095599" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1700'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2693'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7918&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0095599">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0095599" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0095599&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The Genome Sequence of a Widespread Apex Predator, the Golden Eagle (<i>Aquila chrysaetos</i>)
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jacqueline M. Doyle
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:jmdoyle@purdue.edu">jmdoyle@purdue.edu</a></p>

                <p>Affiliation:
                  Department of Forestry and Natural Resources, Purdue University, West Lafayette, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Todd E. Katzner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Division of Forestry and Natural Resources, West Virginia University, Morgantown, West Virginia, United States of America, 
                  Northern Research Station, USDA Forest Service, Parsons, West Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter H. Bloom, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Western Foundation of Vertebrate Zoology, Camarillo, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yanzhu Ji, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Forestry and Natural Resources, Purdue University, West Lafayette, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bhagya K. Wijayawardena, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Forestry and Natural Resources, Purdue University, West Lafayette, Indiana, United States of America, 
                  Department of Biological Sciences, Purdue University, West Lafayette, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            J. Andrew DeWoody
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Forestry and Natural Resources, Purdue University, West Lafayette, Indiana, United States of America, 
                  Department of Biological Sciences, Purdue University, West Lafayette, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 23, 2014</li>
    <li>DOI: 10.1371/journal.pone.0095599</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0095599" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0095599" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0095599" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0095599" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0095599-t001" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0095599-g001" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0095599-g002" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0095599-g003" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0095599-t002" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0095599-t003" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0095599">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0095599" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Biologists routinely use molecular markers to identify conservation units, to quantify genetic connectivity, to estimate population sizes, and to identify targets of selection. Many imperiled eagle populations require such efforts and would benefit from enhanced genomic resources. We sequenced, assembled, and annotated the first eagle genome using DNA from a male golden eagle (<em>Aquila chrysaetos</em>) captured in western North America. We constructed genomic libraries that were sequenced using Illumina technology and assembled the high-quality data to a depth of ~40x coverage. The genome assembly includes 2,552 scaffolds &gt;10 Kb and 415 scaffolds &gt;1.2 Mb. We annotated 16,571 genes that are involved in myriad biological processes, including such disparate traits as beak formation and color vision. We also identified repetitive regions spanning 92 Mb (~6% of the assembly), including LINES, SINES, LTR-RTs and DNA transposons. The mitochondrial genome encompasses 17,332 bp and is ~91% identical to the Mountain Hawk-Eagle (<em>Nisaetus nipalensis</em>). Finally, the data reveal that several anonymous microsatellites commonly used for population studies are embedded within protein-coding genes and thus may not have evolved in a neutral fashion. Because the genome sequence includes ~800,000 novel polymorphisms, markers can now be chosen based on their proximity to functional genes involved in migration, carnivory, and other biological processes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Doyle JM, Katzner TE, Bloom PH, Ji Y, Wijayawardena BK, et al.  (2014) The Genome Sequence of a Widespread Apex Predator, the Golden Eagle (<em>Aquila chrysaetos</em>). PLoS ONE 9(4):
          e95599.
            doi:10.1371/journal.pone.0095599</p><p><strong>Editor: </strong>Ludovic Orlando, Natural History Museum of Denmark, University of Copenhagen, Denmark</p><p><strong>Received:</strong> October 10, 2013; <strong>Accepted:</strong> March 28, 2014; <strong>Published:</strong> April 23, 2014</p><p><strong>Copyright:</strong> © 2014 Doyle et al. This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the U.S. Bureau of Land Management (award numbers L12AC20102, L11PX02237, and L12AC2010), the California Department of Fish and Wildlife (Agreement #P1182024), and the Provost’s Office at Purdue University (University Faculty Scholar program). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>For millennia, eagles have been cultural icons emblematic of nations, religions, and peoples around the world (<a href="#pone.0095599-Tingay1">[1]</a>, <a href="#pone.0095599-Watson1">[2]</a>; <a href="#pone.0095599.s001">Figure S1</a>). In ancient Egypt, eagle hieroglyphs were symbolic of the soul after death. In contemporary North America, native cultures incorporate eagle feathers into medicines and religious ceremonies. Eagles have long been trained for falconry in Central Asia and are still used to hunt prey as large as wolves in Mongolia <a href="#pone.0095599-Watson1">[2]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Eagles are also apex predators whose trophic impacts cascade through ecosystems, as their prey range in size from beetles to marine mammals and span a gamut that includes frugivores, herbivores, carnivores, omnivores, and planktivores (e.g., monkeys, deer, hawks, tortoises, fishes, etc.) <a href="#pone.0095599-Parrish1">[3]</a>–<a href="#pone.0095599-Olendorff1">[7]</a>. Unfortunately, many eagle species are of worldwide conservation concern due to direct threats to individuals (e.g., poaching and collisions with wind turbines) and indirect threats to populations (e.g., habitat loss and environmental toxins) <a href="#pone.0095599-Watson1">[2]</a>, <a href="#pone.0095599-Kochert1">[8]</a>–<a href="#pone.0095599-Mooney1">[11]</a>. Conservation efforts have often been hampered by the generally secretive nature and remote habitats of eagles, but recently described molecular markers have provided new tools for population monitoring <a href="#pone.0095599-Rudnick1">[12]</a>, <a href="#pone.0095599-Rudnick2">[13]</a>. Modest suites of microsatellite markers are now available for a few species (e.g., <em>Aquila adalberti</em>, <a href="#pone.0095599-MartinezCruz1">[14]</a>); <em>A. heliaca</em>, <a href="#pone.0095599-Busch1">[15]</a>; <em>Haliaeetus albicilla</em>, <a href="#pone.0095599-Hailer1">[16]</a>; <em>Nisaetus nipalensis</em>, <a href="#pone.0095599-Hirai1">[17]</a>), and complete mitochondrial genome sequences are available for three species (<em>Spilornis cheela</em> <a href="#pone.0095599-Qin1">[18]</a>, <em>N. nipalensis,</em> and <em>Spizaetus alboniger</em> <a href="#pone.0095599-Asai1">[19]</a>).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Avian genomics, however, still lags far behind mammalian genomics as scores of complete mammalian genomes have been sequenced, but only about a dozen avian genomes have been published (<a href="#pone-0095599-t001">Table 1</a>). With this in mind, we sequenced the genome of the golden eagle (<em>Aquila chrysaetos</em>) to facilitate comparative studies of avian genomics and to further the development of genetic tools for eagle research and conservation. Golden eagles are among the most widespread of avian species, with a distribution that spans the Paleartic and Nearctic and extends into the Afrotropic and Indomalaya ecozones <a href="#pone.0095599-Watson1">[2]</a>. They are often considered a mountain resident, but can thrive in an array of habitats including shrub-steppe communities, deserts, bogs, peatlands and tundra <a href="#pone.0095599-Watson1">[2]</a>. Nevertheless, the golden eagle is threatened throughout much of its range. Historical and ongoing population declines and a suite of persistent and novel threats have led to governmental protection of these birds in much of their range <a href="#pone.0095599-Watson1">[2]</a>, <a href="#pone.0095599-Katzner1">[10]</a>, <a href="#pone.0095599-Eaton1">[20]</a>–<a href="#pone.0095599-Hoffman1">[22]</a>.</p>
<div class="figure" id="pone-0095599-t001"><div class="img"><a name="pone-0095599-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0095599.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0095599.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Assembled avian nuclear genomes in NCBI as of 12 September 2013.</span></strong></p><span>doi:10.1371/journal.pone.0095599.t001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>A complete sequence of the golden eagle genome can facilitate the conservation of this species in a number of ways. For example, a major source of mortality to golden eagles is collision with wind turbines and other structures <a href="#pone.0095599-Watson1">[2]</a>, <a href="#pone.0095599-Katzner1">[10]</a>. Scientists have hypothesized that raptors might be better able to avoid these structures if they were coated with ultraviolet-reflective paint <a href="#pone.0095599-Young1">[23]</a>. The color vision system is undescribed in golden eagles, however. The golden eagle genome sequence can be used to determine whether the color vision system is violet-tuned or ultraviolet-tuned, shedding light on whether UV-reflective paint has potential merit. Furthermore, a complete sequence of the golden eagle genome will prove valuable for those interested in the evolution, ecology, and demography of this charismatic species by virtue of the molecular polymorphisms contained therein.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Here, we provide a broad overview of our methods. Further details are available in the Electronic Supplementary Materials (ESM) available online at the journal’s website.</p>

<h4>Sampling, Molecular Methods, and Quality Control</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A male golden eagle (subspecies <em>A. c. canadensis</em>) was captured 6 December 2012 in the California foothills of the southern Sierra Nevada, between the Central Valley and the Mojave Desert (N 35 18 29.2 W 118 38 05.7). The propositus was captured with a bow net following approved protocols (West Virginia University’s Animal Care and Use Committee, protocol #11-0304) and under federal and state bird banding permits (BBL#20431; Cal SCP #SC-221) <a href="#pone.0095599-Bloom1">[24]</a>. Three drops of blood (~2 ml) were collected via venipuncture of the brachial vein were preserved in 1 ml of lysis buffer (100 mM Tris-HCl, 100 mM EDTA, 10 mM NaCl, 2% SDS) and the eagle was outfitted with a GPS-GSM tracking device <a href="#pone.0095599-Bloom1">[24]</a> before release (<a href="#pone-0095599-g001">Figure 1</a>). Genomic DNA was subsequently extracted using a standard phenol chloroform protocol <a href="#pone.0095599-Sambrook1">[25]</a> and a standard PCR assay was used to confirm sex genetically <a href="#pone.0095599-Fridolfsson1">[26]</a>.</p>
<div class="figure" id="pone-0095599-g001"><div class="img"><a name="pone-0095599-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0095599.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0095599.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Movements of the captured male golden eagle.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Movements of the golden eagle (USFWS Band #0679-02608) whose genome sequence is presented herein. GPS data were collected by a CTT-11060 telemetry unit at 15-minute intervals from capture date (6 December 2012) through 07 March 2013. Home range size during this period was 1068 km<sup>2</sup> (95% KDE).</p>
<span>doi:10.1371/journal.pone.0095599.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In February and March 2013, we conducted one lane of paired-end sequencing and one lane of mate-paired sequencing using an Illumina HiSeq2000 that produced read lengths of 100 bp. Quality control included a) adaptor removal using Trimmomatic (<a href="#pone.0095599-Lohse1">[27]</a>, Table S1 in <a href="#pone.0095599.s003">File S1</a>); b) discarding short reads (&lt;30 bp); c) trimming poor quality bases (Illumina Q-value≤20) from both 5′ and 3′ ends of raw sequence reads; and d) removing all identical paired-end reads (i.e., PCR duplicates).</p>


<h4>Genome Assembly and Genome Size Estimation</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We used ABySS <a href="#pone.0095599-Simpson1">[28]</a> for <em>de novo</em> assembly of the <em>A. chrysaetos</em> nuclear genome. We used trimmed paired-end reads and mate-paired reads (as single-end reads) to create consensus sequences. Briefly, all possible K-mers were generated from sequence reads and a de Bruijn graph <a href="#pone.0095599-Simpson1">[28]</a> was created by joining overlapping K-mers. Subsequently, both paired-end and mate-paired data were used to resolve ambiguities among contigs and to link contigs into scaffolds. The completeness of the assembly was assessed by CEGMA, which assesses the proportion of proteins predicted from the <em>A. chrysaetos</em> genome relative to a conserved set of core eukaryotic proteins <a href="#pone.0095599-Parra1">[29]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We used the K-mer approach to estimate total genome size. Briefly, we used Jellyfish <a href="#pone.0095599-Marais1">[30]</a> to divide all paired-end sequenced reads into K-mers of 17 nucleotides and to plot the frequency of each K-mer so that the peak depth represented the mean K-mer coverage (M) of the genome (<a href="#pone.0095599.s002">Figure S2</a>). We then estimated the actual coverage of the genome (N) using the equation N = M/((L−K+1)/L), where L is the mean read length and K is the K-mer size <a href="#pone.0095599-Li1">[31]</a>. Sequence coverage was estimated by dividing total sequence data by genome size.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>For assembly of the <em>A. chrysaetos</em> mitochondrial DNA (mtDNA) genome, we first used the Mountain Hawk-Eagle (<em>Nisaetus nipalensis</em>; Asai et al., 2006) mtDNA genome as a reference to map our paired-end reads using Bowtie2 <a href="#pone.0095599-Langmead1">[32]</a>. We also used MITObim, which employs a baiting and iterative mapping approach <a href="#pone.0095599-Hahn1">[33]</a>.</p>


<h4>Gene Annotation</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The <em>A. chrysaetos</em> mtDNA genome was annotated using DOGMA <a href="#pone.0095599-Wyman1">[34]</a> and visualized with OGDraw <a href="#pone.0095599-Lohse2">[35]</a>. To help annotate the <em>A. chrysaetos</em> nuclear genome, we used EST and protein evidence from other avian species. We downloaded <em>Gallus gallus</em>, <em>Meleagris gallopavo, Taeniopygia guttata</em> and <em>Columba livia</em> protein sequences from the UniProtKB database (<a href="http://www.uniprot.org">www.uniprot.org</a>) and <em>Falco cherrug</em> RNAseq reads from the NCBI short read archive <a href="#pone.0095599-Zhan1">[36]</a>. The RNA-seq reads were assembled <em>de novo</em> into contigs using Trinity <a href="#pone.0095599-Grabherr1">[37]</a> after employing the quality control measures described earlier. We then used the pipeline MAKER <a href="#pone.0095599-Cantarel1">[38]</a>, which incorporates the following programs (among others): 1) RepeatMasker <a href="#pone.0095599-Smit1">[39]</a> which identified and masked stretches of repetitive DNA in the eagle genome; 2) BLAST, which aligned avian ESTs and proteins to the genome; and 3) SNAP <a href="#pone.0095599-Korf1">[40]</a> and AUGUSTUS <a href="#pone.0095599-Stanke1">[41]</a>, which produced <em>ab initio</em> gene predictions for <em>A. chrysaetos</em>. MAKER synthesized these data and produced final annotations with evidence-based quality values. MAKER was run in an iterative manner such that gene models from one run acted as inputs for subsequent runs. The initial evidence used in MAKER included the 415 <em>A. chrysaetos</em> genome sequences greater than 1.2 Mb in length (Table S2 in <a href="#pone.0095599.s003">File S1</a>) and the 2,385 protein sequences from <em>Gallus gallus</em>, <em>Meleagris gallopavo, Taeniopygia guttata</em> and <em>Columba livia</em>. The protein2genome setting in MAKER was used to produce gene annotations directly from protein evidence, and this output file was used to train SNAP. We then completed a second MAKER run using the same initial evidence, but the protein2genome setting was not used. The results were then used to train SNAP a second time. In the third iteration, we supplied MAKER with 1) 2,552 <em>A. chrysaetos</em> genome sequences greater than 10.0 Kb; 2) all 2,385 avian protein sequences; and 3) 234,818 ESTs (i.e., RNAseq contigs) from <em>Falco cherrug</em>. We ran AUGUSTUS with the “chicken” species setting and RepeatMasker with the “all” setting.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Given our heterospecific libraries of protein and EST evidence, we initiated a second pipeline to identify genes that remained unannotated. We collected all SNAP and AUGUSTUS <em>ab initio</em> gene predictions that were not supported by EST or protein evidence and used InterProScan to identify putative protein domains. Accordingly, gene predictions containing presumptive protein domains were promoted to gene annotations, and InterProScan was used to assign ontologies to each gene. In order to compare our results to those of other studies, we also used InterProScan to assign ontologies to saker and peregrine falcon genes <a href="#pone.0095599-Zhan2">[42]</a>.</p>


<h4>Xenobiotics and Repetitive Sequences</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>All of our sequences were derived from genomic libraries constructed from bird blood, but this does not mean that all sequences are of eagle origin. We delineated xenobiotic sequences to identify potential pathogens, parasites, and commensals of <em>A. chrysaetos.</em> First, all contigs longer than 200 bases were used as BLAST queries (BLASTN parameters; E value = 1E-6) against the chicken genome (ensembl database: Gallus_gallus.Galgal4.72.dna.toplevel.fa​)to identify known avian sequences. Subsequently, all remaining contigs (i.e., those very dissimilar to chicken) were extracted and used as BLAST queries (BLASTN parameters; E value = 1E-6) of the entire GenBank catalog. For each of these query sequences, up to 1000 hits were collected and the sequence was categorized as either vertebrate or invertebrate in origin. Contigs that matched no vertebrate taxa were identified as putative xenobiotics (Table S3 in <a href="#pone.0095599.s003">File S1</a>).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>After excluding the xenobiotic contigs, repetitive elements in the <em>A. chrysaetos</em> assembly were detected by a combination of methods, including homology-based and <em>de novo</em> approaches <a href="#pone.0095599-Bergmen1">[43]</a>–<a href="#pone.0095599-Saha1">[46]</a>. We used RepeatMasker <a href="#pone.0095599-Smit1">[39]</a>, RepeatProteinMask <a href="#pone.0095599-Smit1">[39]</a> and RepeatModeler <a href="#pone.0095599-Smit2">[47]</a> to identify interspersed repeats, then ran Tandem Repeats Finder <a href="#pone.0095599-Benson1">[48]</a>. Custom perl scripts (modified from L. Hu, personal communication) were used to remove overlapping regions and calculate overall repeat content.</p>


<h4>Linkage Disequilibrium and Molecular Markers</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The extent of linkage disequilibrium (LD) in avian species is known to vary between 0.5–400 Kb (<a href="#pone.0095599-Backstrom1">[49]</a> <a href="#pone.0095599-Shapiro1">[50]</a>). Bourke and Dawson <a href="#pone.0095599-Bourke1">[51]</a> described fifteen anonymous microsatellites from the <em>A. chrysaetos</em> nuclear genome. We used a custom perl script to identify their primer sequences in our scaffolds, then used the program Apollo <a href="#pone.0095599-Lewis1">[52]</a> to locate genes within 400 Kb in an effort to determine which of these 15 markers might be most heavily influenced by hitchhiking associated with selective sweeps.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>To extend the suite of <em>A. chrysaetos</em> molecular markers, we used the genome assembly to identify additional microsatellites using MISA <a href="#pone.0095599-Thiel1">[53]</a>. Single nucleotide polymorphisms (SNPs) were identified using Bowtie2 <a href="#pone.0095599-Hahn1">[33]</a> to align all filtered paired-end reads to contigs longer than 200 bases. Samtools <a href="#pone.0095599-Li2">[54]</a> was subsequently used to call SNPs with coverage greater than 10 reads and less than 60 reads, with a quality score of 20 or better, in order to compare our results to that of other studies (e.g., peregrine and saker genomes <a href="#pone.0095599-Zhan1">[36]</a>).</p>


<h4>Color Vision Determination</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Avian color vision can be categorized as violet or ultraviolet, and associated sensitivity can be determined from the SWS1 opsin protein sequence <a href="#pone.0095599-Thiel1">[53]</a>. We downloaded opsin sequences for three raptor species from NCBI (<em>Accipiter gentilis</em> AY227148; <em>Buteo buteo</em> AY227150; <em>Pandion haliaetus</em> AY227152 <a href="#pone.0095599-Odeen1">[55]</a>). We used blastn to identify a single scaffold in our assembly that contained the SWS1 opsin coding region and used ExPASy to translate the nucleotide sequence to amino acid sequence.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We generated 68.4 Gb of raw sequence data from <em>A. chrysaetos</em>, including 25.3 Gb from the paired-end library and 43.1 Gb from the mate-paired library (Table S4 in <a href="#pone.0095599.s003">File S1</a>). Quality control filtering yielded 24.5 Gb and 21.0 Gb from the paired-end and mate-paired libraries, respectively, so about one-third of the raw data fell to the cutting-room floor <a href="#pone.0095599-DeWoody1">[56]</a>. More reads were filtered from the mate-paired data than the paired-end data because the cluster density associated with mate-paired data was higher. As cluster density increases, so too does interference from nearby clusters and therefore more reads are discarded by the clipping/filtering program.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The MITObim assembly of the <em>A. chrysaetos</em> mtDNA genome produced a sequence of 17,332 bp (<a href="#pone-0095599-g002">Figure 2</a>), whereas the Bowtie2-produced genome was 17,647 bp. These assemblies were 97% identical to each other and, on average, were 92% identical to the <em>N. nipalensis</em> mtDNA genome. Given the strong concordance between the two approaches, hereafter we refer only to the MITObim assembly. The mtDNA genome is characterized by 13 protein-coding genes, two ribosomal subunit genes (rRNA), 23 transfer RNA genes (tRNA; Table S5 and Table S10 in <a href="#pone.0095599.s003">File S1</a>). Twenty-eight genes reside on the α-strand and 10 on the β-strand, and the putative control region is 1157 bp. As in most vertebrates, all protein-coding genes except NAD6 were found on the α-strand (<a href="#pone-0095599-g002">Figure 2</a>, Tables S5 and S10 in <a href="#pone.0095599.s003">File S1</a>).</p>
<div class="figure" id="pone-0095599-g002"><div class="img"><a name="pone-0095599-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0095599.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0095599.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span><em>A. chrysaetos</em> mitochondrial genome map.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Cox1, cox2 and cox3 indicate cytochrome oxidase subunits 1–3; cob indicates cytochrome b; atp6 and atp8 indicate ATPase subunits 6 and 8; nad1–nad6 indicate NADH dehydrogenase subunits 1–6. Transfer RNA genes are designated by single-letter amino acid codes.</p>
<span>doi:10.1371/journal.pone.0095599.g002</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We divided our total paired-end sequence data (24,385,716,189 bp) by N to estimate a genome size of 1.28 Gb (including the mtDNA genome) and overall genome coverage was estimated as 38.9X (<a href="#pone-0095599-g003">Figure 3</a>, Table S4 in <a href="#pone.0095599.s003">File S1</a>). Nuclear genome assembly with ABySS produced 42,926 scaffolds that contain 1,548 Mb. These scaffolds had an N50 of 1,746,960 bp and the longest scaffold was 11,517,212 bp in length (Table S2 in <a href="#pone.0095599.s003">File S1</a>). Table S6 in <a href="#pone.0095599.s003">File S1</a> indicates that approximately 90% of the core eukaryotic genes were identified in the <em>A. chrysaetos</em> genome.</p>
<div class="figure" id="pone-0095599-g003"><div class="img"><a name="pone-0095599-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0095599.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0095599.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Depth of sequencing of the <em>A. chrysaetos</em> genome.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>Sequencing depth is on the x-axis while the y-axis shows the percentage of total bases at a given depth. Reads were aligned to the genome using bowtie2.</p>
<span>doi:10.1371/journal.pone.0095599.g003</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>EST and protein evidence greatly facilitates genome annotation. The 2,385 <em>Gallus gallus</em>, <em>Meleagris gallopavo, Taeniopygia guttata</em> and <em>Columba livia</em> protein sequences we used corresponded to 1,125,485 bases in total and had a N50 of 603. Our <em>de novo</em> assembly of the <em>Falco cherrug</em> transcriptome from RNA-seq reads produced 234,818 contigs that spanned 162,920,697 nucleotides, and contig length ranged from 101–17,136 bp with a N50 of 2,306.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Our comprehensive annotation of the <em>A. chrysaetos</em> genome produced a total of 16,571 predicted nuclear genes. Mean gene length was 25,049 nucleotides and on average, 8.6 exons were predicted in each gene. Mean exon and intron lengths were 143 bp and 2,646 bp, respectively. Based on protein domains, 89% of the <em>A. chrysaetos</em> genes were assigned gene ontologies and the top 100 protein domains can be found in Table S7 in <a href="#pone.0095599.s003">File S1</a>. We assigned gene ontologies to 79% and 80% of the saker and peregrine falcon predicted genes, respectively.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The total repeat content of the <em>A. chrysaetos</em> genome was estimated to be 5.94% (<a href="#pone-0095599-t002">Table 2</a>). Golden eagle repetitive elements are primarily composed of long interspersed nuclear elements (LINEs), then long terminal repeat retrotransposons (LTR-RTs), followed by DNA transposons and short interspersed nuclear elements (SINEs, <a href="#pone-0095599-t002">Table 2</a>). The total repeat content of the <em>A. chrysaetos</em> genome is most similar to the 5.86% found in mallard ducks (<a href="#pone.0095599-Huang1">[57]</a>, Table S8 in <a href="#pone.0095599.s003">File S1</a>). Putative xenobiotic organisms represented in our sequence data are listed in Table S3 in <a href="#pone.0095599.s003">File S1</a>.</p>
<div class="figure" id="pone-0095599-t002"><div class="img"><a name="pone-0095599-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0095599.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0095599.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Repetitive elements in the <em>A. chrysaetos</em> genome. Numbers indicate repeat size in bp and percentage of genome assembly (in parenthesis).</span></strong></p><span>doi:10.1371/journal.pone.0095599.t002</span></div><a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Each of Bourke and Dawson’s 15 microsatellites <a href="#pone.0095599-Bourke1">[51]</a> were located in a genomic scaffold (Table S9 in <a href="#pone.0095599.s003">File S1</a>). Twelve were found within 400 kb of a gene, three within 20 kb of a gene, and two microsatellites were located in the noncoding regions of annotated genes (<a href="#pone-0095599-t003">Table 3</a>). Gene ontology terms associated with these genes ranged from metabolic process to tumor necrosis factor (<a href="#pone-0095599-t003">Table 3</a>).</p>
<div class="figure" id="pone-0095599-t003"><div class="img"><a name="pone-0095599-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0095599" data-uri="info:doi/10.1371/journal.pone.0095599.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0095599.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0095599.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0095599.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0095599.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0095599.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Proximity of anonymous microsatellites <a href="#pone.0095599-Bourke1">[51]</a> to annotated <em>A. chrysaetos</em> genes.</span></strong></p><span>doi:10.1371/journal.pone.0095599.t003</span></div><a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Our search for additional <em>A. chrysaetos</em> markers revealed 60,346 microsatellites (34,443 dinucleotides, 16,660 trinucleotides, 5,370 tetranucleotides, 3,389 pentanucleotides, and 484 hexanucleotides). We also identified 767,898 biallelic SNPs with read depths between 10–60x with quality scores greater than 20, which corresponds to 0.77 SNPs per Kbp.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>The putative <em>A. chrysaetos</em> SWS1 opsin gene aligned with 100% identity to that of <em>Buteo buteo</em> and <em>Pandion haliaetus</em>, and with 99% identity to <em>Accipiter gentilis</em> (see supplementary material). The translated amino acid sequence (FISCIFSVFTV) indicates a violet-tuned color vision system as opposed to ultraviolet <a href="#pone.0095599-Odeen1">[55]</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We have sequenced, assembled, and annotated the <em>A. chrysaetos</em> genome. Avian genomics is still in its infancy and thus meaningful comparisons of the eagle genome with other bird genomes are difficult. Extant birds are generally grouped into more than 200 families, yet complete genome sequences are currently restricted to 10 avian families and no other members of the family Accipitridae (<a href="#pone-0095599-t001">Table 1</a>). Avian genome assemblies range in size from 1.04 Gb in the Tibetan Ground-tit to 1.55 Gb in the Golden Eagle (<a href="#pone-0095599-t001">Table 1</a>). NCBI contains far more sequenced mammalian genomes (n&gt;50), the assemblies of which are larger (mean of 2.5 Gb) and more variable in size (range 2.00 Gb to 4.21 Gb) than avian genomes. The homogeneity in avian genome size relative to mammalian genome size is also reflected in flow cytometry data <a href="#pone.0095599-Gregory1">[58]</a>. <em>A. chrysaetos</em> gene lengths are similar to other birds but mean exon and intron lengths are somewhat shorter (<a href="#pone-0095599-t001">Table 1</a>), suggesting that promoters, 5′ UTRs, and 3′UTRs may be longer in eagles.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Golden eagle genome size estimates range from 1.28–1.48 Gb (<a href="#pone.0095599-Tiersch1">[59]</a>, <a href="#pone.0095599-Venturini1">[60]</a>, <a href="#pone-0095599-t001">Table 1</a>), indicating that our assembly is potentially 5–21% larger than the actual genome size. Bradnam et al. <a href="#pone.0095599-Bradnam1">[61]</a> argued that large assemblies may result from assembly errors, but can also occur when heterogeneous regions of the genome are legitimately resolved into independent scaffolds. This study also provided evidence that assemblies which are relatively larger or smaller than the estimated genome size can perform well in terms of other metrics, such as the number of correctly identified core eukaryotic genes. The “completeness” of our overall genome assembly is indeed evidenced by our identification of most all (90%) core eukaryotic genes (CEGs; <a href="#pone.0095599-Parra1">[29]</a>, Table S6 in <a href="#pone.0095599.s003">File S1</a>); as well as by our microsatellite mapping exercise (i.e., all 15 anonymous loci were identified in our scaffolds) and our recovery of the entire <em>A. chrysaetos</em> mtDNA genome sequence. These results are comparable to recently published, high-quality genomes (e.g., rock pigeon <a href="#pone.0095599-Shapiro1">[50]</a>) and indicate that our assembly includes the vast majority of <em>A. chrysaetos</em> genes.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Our xenobiotic analyses, whereby we parsed eagle (vertebrate) sequences from invertebrate sequences, revealed that blood from the propositus also contained DNA from other species. Thus, our deep sequencing identified previously uncharacterized organisms that may be important to the ecology and evolution of <em>A. chrysaetos</em>. For example, these xenobiotic sequences include hits to a number of avian retroviruses, viruses, and pathogenic bacteria (Table S3 in <a href="#pone.0095599.s003">File S1</a>).</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>The repertoire of repetitive DNA in <em>A. chrysaetos</em> is limited relative to mammals, but is generally similar to known avian genomes (Table S8 in <a href="#pone.0095599.s003">File S1</a>, <a href="#pone.0095599-Organ1">[62]</a>, <a href="#pone.0095599-Ellegren1">[63]</a>). The <em>A. chrysaetos</em> genome does not exhibit substantial variation in repeat content, either in the total proportion of repeats in the genome or in the relative proportions of different superfamilies and/or classes of repetitive elements. The <em>A. chrysaetos</em> genome appears to have fewer LINEs than the chicken genome <a href="#pone.0095599-Hillier1">[64]</a>, but this could also be attributable to technical factors such as enrichment of repetitive regions in unassembled portions of the genome and/or incomplete repeat libraries (see supplementary material). Overall, the lack of variation in repeat contents is consistent with the relative homogeneity of avian genome sizes compared to mammalian genomes <a href="#pone.0095599-Organ1">[62]</a>, <a href="#pone.0095599-Ellegren1">[63]</a>.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>We annotated 16,571 genes in the golden eagle genome, including orthologs, for example, to <em>Bmp4</em>, a gene implicated in raptor beak formation <a href="#pone.0095599-Zhan2">[42]</a>. These annotations are the first step to exploring unique golden eagle adaptations. For example, 57 predicted genes have ontologies associated with olfaction (e.g., olfactory receptors), a number similar to saker and peregrine falcons. Historically, birds were thought to rely primarily on magnetic or visual cues to hunt and navigate. As a result, only a few studies have addressed avian sensitivity to and navigation by odor <a href="#pone.0095599-Nevitt1">[65]</a>, <a href="#pone.0095599-Nevitt2">[66]</a> or the olfactory receptor (OR) genes that may underlie these abilities <a href="#pone.0095599-Zhan2">[42]</a>, <a href="#pone.0095599-Steiger1">[67]</a>. Our identification of OR genes may ultimately allow scientists to determine the molecular mechanisms underlying eagle olfaction, which may be important in locating carrion in forests or fish in the open sea.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Genome sequencing provides opportunities to develop new tools for species of conservation concern. MtDNA has been used to quantify genetic variation of threatened species, identify evolutionary distinct populations, and evolutionary significant units <a href="#pone.0095599-Moritz1">[68]</a>, <a href="#pone.0095599-Frankham1">[69]</a>. Molecular clock analyses based on the mtDNA genome sequence [see ESM] suggest the golden eagle diverged from the Mountain Hawk-Eagle roughly 2.1 MYA, and from the Peregrine Falcon roughly 4.6 years ago. These estimates are generally consistent with previously published molecular phylogenies <a href="#pone.0095599-Lerner1">[70]</a>. Our estimate of overall nucleotide variability (0.77 SNPs per Kbp), is remarkably similar to estimates of SNP density of the scarlet macaw, saker and peregrine falcons (0.86, 0.63, and 0.88 SNPs per Kbp; respectively) but considerably less than the 1.75 SNPs per Kbp of zebra finch <a href="#pone.0095599-Zhan1">[36]</a>, <a href="#pone.0095599-Seabury1">[71]</a>, <a href="#pone.0095599-Warren1">[72]</a>.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>Our SWS1 opsin gene analysis provides evidence only for a vision system biased toward violet (VS) vision, rather than ultraviolet (UVS). Avian species with a VS-tuned vision are particularly sensitive at wavelengths above 400 nm, while UVS-tuned birds are sensitive at wavelengths below 400 nm <a href="#pone.0095599-Lind1">[73]</a>, <a href="#pone.0095599-Hart1">[74]</a>. Although classic studies suggested that raptors hunt by following ultraviolet signals in the urine of prey <a href="#pone.0095599-Viitala1">[75]</a>, Odeen and Hastad <a href="#pone.0095599-Odeen1">[55]</a> determined that VS-tuned systems are predominant in raptors. They additionally hypothesized that UVS-tuned passerine prey may be able to communicate with one another using colors inconspicuous to raptors. Furthermore, Lind et al. <a href="#pone.0095599-Lind1">[73]</a> measured transmission properties of tissues (ocular media transmittance) in the common buzzard eye and argued that the chromatic contrast between vole urine and substrate would provide an unreliable cue to hunting raptors. Taken in total, these results provide little evidence that golden eagles are sensitive to ultraviolet light, and thus that UV-reflective paint likely would not increase the visibility of structures and prevent golden eagle collisions.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>Genome sequencing also provides geneticists with opportunities to investigate assumptions associated with previously-developed tools. For example, microsatellite markers are commonly used in studies of natural populations, but the vast majority of these markers are anonymous with respect to their position in the genome. Disequilibrium tests are often used to determine if microsatellites are inherited independently of one another, but such tests do not include genomic position. This may be important, as eukaryotic genomes are not homogenous and selection can vary greatly across the genome <a href="#pone.0095599-Nekrutenko1">[76]</a>. Microsatellites located in or near functional genes are likely to be more exposed to selection and selective sweeps than those occurring in gene deserts, and it is known that vertebrate microsatellites are often found in expressed genes <a href="#pone.0095599-Doyle1">[77]</a>.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>Of Bourke and Dawson’s <a href="#pone.0095599-Bourke1">[51]</a> 15 anonymous <em>A. chrysaetos</em> microsatellites, twelve were within 400 kb of an annotated gene and two were found in the intron or untranslated region of a gene. A published study <a href="#pone.0095599-Bourke1">[51]</a> of over a hundred Scottish golden eagles found no deviations from Hardy-Weinberg expectations (HWE) at these twelve loci, but unpublished data on North American golden eagles found that seven of these twelve loci deviated from HWE (Maria Wheeler, personal communication). Hitchhiking is often suspected as the culprit when only one or a few microsatellite loci deviate from HWE in a population study, but as genome sequences become more commonplace, investigators will increasingly have the genomic infrastructure necessary to tease out location effects associated with functional genes.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p>Non-invasive molecular methods have the capacity to profoundly influence our understanding of threatened and endangered species <a href="#pone.0095599-Rudnick1">[12]</a>, <a href="#pone.0095599-Rudnick2">[13]</a>, <a href="#pone.0095599-Morin1">[78]</a>–<a href="#pone.0095599-Ahlering1">[81]</a>. For example, DNA fingerprints associated with naturally shed feathers have provided estimates of population size, reproductive success, and demographic turnover in Imperial Eagles (<em>A. heliaca,</em> <a href="#pone.0095599-Rudnick1">[12]</a>, <a href="#pone.0095599-Rudnick2">[13]</a>). Genomic resources such as those reported herein will help extend studies based on anonymous genetic markers to those that include important functional genes. These might include avian genes associated with migratory tendencies, beak development, and olfaction <a href="#pone.0095599-Zhan1">[36]</a>, <a href="#pone.0095599-Seabury1">[71]</a>. Future study of these (and other) genes will no doubt reveal their functional, molecular contributions to the widespread distribution of <em>A. chrysaetos</em> and their trophic position as apex predators. Thus, we anticipate that the <em>A. chrysaetos</em> genome sequence will guide our understanding of avian adaptation, while providing additional molecular tools that facilitate the conservation of these charismatic organisms.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0095599"></div><a name="pone.0095599.s001" id="pone.0095599.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0095599.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI"><strong>The Mexican coat of arms contains a golden eagle.</strong></p>
<p class="siDoi">doi:10.1371/journal.pone.0095599.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0095599.s002" id="pone.0095599.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0095599.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI"><strong>17 bp-mer estimation of the genome size of </strong><strong><em>A. chrysaetos</em></strong><strong>.</strong> K-mer depth is on the x-axis, while the frequency of K-mer counts at a given sequencing depth is represented on the y-axis.</p>
<p class="siDoi">doi:10.1371/journal.pone.0095599.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(TIF)</p>
<a name="pone.0095599.s003" id="pone.0095599.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0095599.s003">File S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI"><strong>Table S1, Table S2, Table S3, Table S4, Table S5, Table S6, Table S7, Table S8, Table S9 and Table S10. Table S1.</strong> Software used to assemble, annotate, and describe the <em>A. chrysaetos</em> genome. ORF, open reading frame. <strong>Table S2.</strong> 70-mer statistics for the <em>Aquila chrysaetos</em> genome. <strong>Table S3.</strong> Summary of the BLASTN search against NCBI nucleotide database (BLASTN parameters: E 697 value = 1E-6, 1000 hits per each query). The contigs with only non-vertebrate hits are listed along with 698 the description of hits. When the BLASTN search resulted in &gt;3 hits from the same group (indicated by 699*), only the top 3 hits for each taxonomic group are listed. <strong>Table S4.</strong> <em>Aquila chrysaetos</em> genome data production. <strong>Table S5.</strong> Mitochondrial gene profile of <em>Aquila chrysaetos.</em> <strong>Table S6.</strong> Identification of CEGs (partial and complete) in the <em>Aquila chrysaetos</em> genome. <strong>Table S7.</strong> Top Pfam domain hits and their counts. <strong>Table S8.</strong> Repetitive elements expressed as percentages of avian genomes. Note that comparisons among assemblies are complicated by technical differences in genome assembly and databases employed. <strong>Table S9.</strong> Bourke and Dawson’s microsatellites, their reported sizes in <a href="#pone.0095599-Bourke1">[51]</a>, and observed size in the <em>A.</em> 500 <em>chrysaetos</em> genome assembly. <strong>Table S10.</strong> Genomic composition of avian mitochondrial DNA.</p>
<p class="siDoi">doi:10.1371/journal.pone.0095599.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(DOCX)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Jyothi Thimmapuram, Phillip San Miguel, Rick Westerman, Paul Parker, Melissa Braham, Michael Campbell, Christoph Hahn, Carson Holt, L. Hu, Rick Westerman, Doug Yatcilla, Scott Thomas, Cheryl Thomas, Michael Kuehn, Chris A. Niemela and members of the DeWoody lab for assistance and comments on the manuscript. Maria Wheeler provided unpublished microsatellite data. The Nature Conservancy provided permission to use their lands.</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p><strong>Data Management:</strong> The filtered sequencing reads are available from SRA (Experiment: SRX363774, Runs: SRR1017148 and SRR1016445). The assembly is available from NCBI (BioSample: SAMN02371870, BioProject: PRJNA222866, Genome: SUB356731). The mitochondrial genome is also available from NCBI (Accession: KF905228). Additional files are available from Dryad (eg., SNP and microsatellite analyses).</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JMD JAD TEK YJ BKW. Performed the experiments: JMD PHB YJ BKW. Analyzed the data: JMD YJ BKW. Contributed reagents/materials/analysis tools: TEK JAD PHB. Wrote the paper: JMD TEK JAD PHB BKW YJ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0095599-Tingay1" id="pone.0095599-Tingay1"></a>Tingay R, Katzner T, Bildstein K, Parry-Jones J (2010) The eagle watchers: Observing and conserving raptors around the world. Ithaca, NY: Cornell University Press. <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0095599-Watson1" id="pone.0095599-Watson1"></a>Watson J (2010) The Golden Eagle. Soho Square, London: A&amp;C Black Publishers Ltd. <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0095599-Parrish1" id="pone.0095599-Parrish1"></a>Parrish JK, Marvier M, Paine RT (2001) Direct and indirect effects: Interactions between Bald Eagles and Common Murres. Ecol Appl 11: 1858–1869.
        doi: 10.1890/1051-0761(2001)011[1858:daieib]2.0.co;2 <ul class="find" data-citedArticleID="11172815" data-doi="10.1890/1051-0761(2001)011[1858:daieib]2.0.co;2"><li><a href="http://dx.doi.org/10.1890/1051-0761(2001)011[1858:daieib]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parrish%5Bauthor%5D+AND+Direct+and+indirect+effects%3A+Interactions+between+Bald+Eagles+and+Common+Murres" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AParrish+%22Direct+and+indirect+effects%3A+Interactions+between+Bald+Eagles+and+Common+Murres%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0095599-Kerley1" id="pone.0095599-Kerley1"></a>Kerley L, Slaght J (2013) First documented predation of Sika deer (<em>Cervus nippon</em>) by Golden Eagle (<em>Aquila chrysaetos</em>) in Russian far east. J Raptor Res 47: 328–330.
        doi: 10.3356/jrr-12-00008.1 <ul class="find" data-citedArticleID="11172817" data-doi="10.3356/jrr-12-00008.1"><li><a href="http://dx.doi.org/10.3356/jrr-12-00008.1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kerley%5Bauthor%5D+AND+First+documented+predation+of+Sika+deer+%28Cervus+nippon%29+by+Golden+Eagle+%28Aquila+chrysaetos%29+in+Russian+far+east" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKerley+%22First+documented+predation+of+Sika+deer+%28Cervus+nippon%29+by+Golden+Eagle+%28Aquila+chrysaetos%29+in+Russian+far+east%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0095599-Suryan1" id="pone.0095599-Suryan1"></a>Suryan RM, Irons DB, Brown ED, Jodice PGR, Roby DD (2006) Site-specific effects on productivity of an upper trophic-level marine predator: Bottom-up, top-down, and mismatch effects on reproduction in a colonial seabird. Prog Oceanogr 68: 303–328. Available: <a href="http://linkinghub.elsevier.com/retrieve/pii/S0079661106000115">http://linkinghub.elsevier.com/retrieve/​pii/S0079661106000115</a>. Accessed 2013 Aug 11. <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0095599-Harvey1" id="pone.0095599-Harvey1"></a>Harvey CJ, Williams GD, Levin PS (2012) Food Web Structure and Trophic Control in Central Puget Sound. Estuaries and Coasts 35: 821–838. Available: <a href="http://link.springer.com/10.1007/s12237-012-9483-1">http://link.springer.com/10.1007/s12237-​012-9483-1</a>. Accessed 2013 Aug 28. <ul class="find-nolinks"></ul></li><li><span class="label">7.
              </span><a name="pone.0095599-Olendorff1" id="pone.0095599-Olendorff1"></a>Olendorff RR (1976) The food habits of North American Golden Eagles. Am Midl Nat 95: 231–236.
        doi: 10.2307/2424254 <ul class="find" data-citedArticleID="11172823" data-doi="10.2307/2424254"><li><a href="http://dx.doi.org/10.2307/2424254" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Olendorff%5Bauthor%5D+AND+The+food+habits+of+North+American+Golden+Eagles" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AOlendorff+%22The+food+habits+of+North+American+Golden+Eagles%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0095599-Kochert1" id="pone.0095599-Kochert1"></a>Kochert M, Steenhof K (2002) Golden eagles in the US and Canada: Status, trends, and conservation challenges. J Raptor Res 36: 32–40. <ul class="find" data-citedArticleID="11172825"><li><a href="http://www.crossref.org/guestquery/?auth2=Kochert&amp;atitle2=Golden+eagles+in+the+US+and+Canada%3A+Status%2C+trends%2C+and+conservation+challenges&amp;auth=Kochert&amp;atitle=Golden+eagles+in+the+US+and+Canada%3A+Status%2C+trends%2C+and+conservation+challenges" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kochert%5Bauthor%5D+AND+Golden+eagles+in+the+US+and+Canada%3A+Status%2C+trends%2C+and+conservation+challenges" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKochert+%22Golden+eagles+in+the+US+and+Canada%3A+Status%2C+trends%2C+and+conservation+challenges%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0095599-Nadjafzadeh1" id="pone.0095599-Nadjafzadeh1"></a>Nadjafzadeh M, Hofer H, Krone O (2013) The link between feeding ecology and lead poisoning in white-tailed eagles. J Wildl Manage 77: 48–57. Available: <a href="http://doi.wiley.com/10.1002/jwmg.440">http://doi.wiley.com/10.1002/jwmg.440</a>. Accessed 2013 Aug 29. <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0095599-Katzner1" id="pone.0095599-Katzner1"></a>Katzner T, Johnson JA, Evans DM, Garner TWJ, Gompper ME, et al.. (2013) Challenges and opportunities for animal conservation from renewable energy development. Anim Conserv 16: 367–369. Available: <a href="http://doi.wiley.com/10.1111/acv.12067">http://doi.wiley.com/10.1111/acv.12067</a>. Accessed 2013 Aug 12. <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0095599-Mooney1" id="pone.0095599-Mooney1"></a>Mooney N, Holdsworth M (1991) The effects of disturbance on nesting Wedge-tailed eagles (<em>Aquila audax fleayi</em>) in Tasmania. Tasforests 3: 15–31. <ul class="find" data-citedArticleID="11172831"><li><a href="http://www.crossref.org/guestquery/?auth2=Mooney&amp;atitle2=The+effects+of+disturbance+on+nesting+Wedge-tailed+eagles+%28Aquila+audax+fleayi%29+in+Tasmania&amp;auth=Mooney&amp;atitle=The+effects+of+disturbance+on+nesting+Wedge-tailed+eagles+%28Aquila+audax+fleayi%29+in+Tasmania" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mooney%5Bauthor%5D+AND+The+effects+of+disturbance+on+nesting+Wedge-tailed+eagles+%28Aquila+audax+fleayi%29+in+Tasmania" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMooney+%22The+effects+of+disturbance+on+nesting+Wedge-tailed+eagles+%28Aquila+audax+fleayi%29+in+Tasmania%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0095599-Rudnick1" id="pone.0095599-Rudnick1"></a>Rudnick JA, Katzner TE, Bragin EA, Rhodes OE, DeWoody JA (2005) Using naturally shed feathers for individual identification, genetic parentage analyses, and population monitoring in an endangered Eastern imperial eagle (<em>Aquila heliaca</em>) population from Kazakhstan. Mol Ecol 14: 2959–2967. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16101766">http://www.ncbi.nlm.nih.gov/pubmed/16101​766</a>. Accessed 2013 Aug 7. <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0095599-Rudnick2" id="pone.0095599-Rudnick2"></a>Rudnick JA, Katzner TE, Bragin EA, DeWoody JA (2007) A non-invasive genetic evaluation of population size, natal philopatry, and roosting behavior of non-breeding eastern imperial eagles (<em>Aquila heliaca</em>) in central Asia. Conserv Genet 9: 667–676. Available: <a href="http://link.springer.com/10.1007/s10592-007-9397-9">http://link.springer.com/10.1007/s10592-​007-9397-9</a>. Accessed 2013 Aug 7. <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0095599-MartinezCruz1" id="pone.0095599-MartinezCruz1"></a>Martinez-Cruz B, David V, Godoy J, Negro J, O’Brien S, et al. (2002) Eighteen polymorphic microsatellite markers for the highly endangered Spanish imperial eagle (<em>Aquila adalberti</em>) and related species. Mol Ecol Notes 2: 323–326.
        doi: 10.1046/j.1471-8286.2002.00231.x <ul class="find" data-citedArticleID="11172837" data-doi="10.1046/j.1471-8286.2002.00231.x"><li><a href="http://dx.doi.org/10.1046/j.1471-8286.2002.00231.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Martinez-Cruz%5Bauthor%5D+AND+Eighteen+polymorphic+microsatellite+markers+for+the+highly+endangered+Spanish+imperial+eagle+%28Aquila+adalberti%29+and+related+species" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMartinez-Cruz+%22Eighteen+polymorphic+microsatellite+markers+for+the+highly+endangered+Spanish+imperial+eagle+%28Aquila+adalberti%29+and+related+species%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0095599-Busch1" id="pone.0095599-Busch1"></a>Busch JD, Katzner TE, Bragin E, Keim P (2005) Tetranucleotide microsatellites for <em>aquila</em> and <em>haliaeetus</em> eagles. Mol Ecol Notes 5: 39–41. Available: <a href="http://doi.wiley.com/10.1111/j.1471-8286.2004.00823.x">http://doi.wiley.com/10.1111/j.1471-8286​.2004.00823.x</a>. Accessed 2013 Aug 29. <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0095599-Hailer1" id="pone.0095599-Hailer1"></a>Hailer F, Gautschi B, Helander B (2005) Development and multiplex PCR amplification of novel microsatellite markers in the White-tailed Sea Eagle, <em>Haliaeetus albicilla</em> (Aves: Falconiformes, Accipitridae). Mol Ecol Notes 5: 938–940. Available: <a href="http://doi.wiley.com/10.1111/j.1471-8286.2005.01122.x">http://doi.wiley.com/10.1111/j.1471-8286​.2005.01122.x</a>. Accessed 2013 Aug 29. <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0095599-Hirai1" id="pone.0095599-Hirai1"></a>Hirai M (2010) Isolation and characterization of eleven microsatellite loci in an endangered species, Mountain Hawk-Eagle (<em>Spizaetus nipalensis</em>). Conserv Genet Resour 2: 113–115.
        doi: 10.1007/s12686-010-9226-1 <ul class="find" data-citedArticleID="11172843" data-doi="10.1007/s12686-010-9226-1"><li><a href="http://dx.doi.org/10.1007/s12686-010-9226-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hirai%5Bauthor%5D+AND+Isolation+and+characterization+of+eleven+microsatellite+loci+in+an+endangered+species%2C+Mountain+Hawk-Eagle+%28Spizaetus+nipalensis%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHirai+%22Isolation+and+characterization+of+eleven+microsatellite+loci+in+an+endangered+species%2C+Mountain+Hawk-Eagle+%28Spizaetus+nipalensis%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0095599-Qin1" id="pone.0095599-Qin1"></a>Qin X-M, Guan Q-X, Shi J-P, Hou L-X, Qin P-S (2013) Complete mitochondrial genome of the <em>Spilornis cheela</em> (Falconiformes, Accipitridae): Comparison of <em>S. cheela</em> and <em>Spizaetus alboniger</em>. Mitochondrial DNA 24: 255–261.
        doi: 10.3109/19401736.2012.752484 <ul class="find" data-citedArticleID="11172845" data-doi="10.3109/19401736.2012.752484"><li><a href="http://dx.doi.org/10.3109/19401736.2012.752484" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Qin%5Bauthor%5D+AND+Complete+mitochondrial+genome+of+the+Spilornis+cheela+%28Falconiformes%2C+Accipitridae%29%3A+Comparison+of+S.+cheela+and+Spizaetus+alboniger" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AQin+%22Complete+mitochondrial+genome+of+the+Spilornis+cheela+%28Falconiformes%2C+Accipitridae%29%3A+Comparison+of+S.+cheela+and+Spizaetus+alboniger%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0095599-Asai1" id="pone.0095599-Asai1"></a>Asai S, Yamamoto Y, Yamagishi S (2006) Genetic diversity and extent of gene flow in the endangered Japanese population of Hodgson’s hawk-eagle, <em>Spizaetus nipalensis</em>. Bird Conserv Int 16: 113. Available: <a href="http://www.journals.cambridge.org/abstract_S0959270906000050">http://www.journals.cambridge.org/abstra​ct_S0959270906000050</a>. Accessed 2013 Aug 28. <ul class="find-nolinks"></ul></li><li><span class="label">20.
              </span><a name="pone.0095599-Eaton1" id="pone.0095599-Eaton1"></a>Eaton MA, Dillon IA, Stirling-Aird PK, Whitfield DP (2007) Status of Golden Eagle <em>Aquila chrysaetos</em> in Britain in 2003: Capsule The third complete survey of Golden Eagles in Britain found 442 pairs. Bird Study 54: 212–220. Available: <a href="http://www.tandfonline.com/doi/abs/10.1080/00063650709461477">http://www.tandfonline.com/doi/abs/10.10​80/00063650709461477</a>. Accessed 2013 Aug 27. <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0095599-Smith1" id="pone.0095599-Smith1"></a>Smith JP, Farmer CJ, Hoffman SW, Kaltenecker GS, Woodruff KZ, et al.. (2008) Trends in autumn counts of migratory raptors in Western North America. In: Bildstein KL, Smith JP, Ruelas Inzunza E, Veit RR, editors. State of North America’s birds of prey. Cambridge, MA and Washington, DC: Nuttall Ornithological Club and American Ornithologists. 217–251. <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0095599-Hoffman1" id="pone.0095599-Hoffman1"></a>Hoffman S, Smith J (2003) Population trends of migratory raptors in Western North America, 1977–2001. Condor 105: 397–419.
        doi: 10.1650/7146 <ul class="find" data-citedArticleID="11172853" data-doi="10.1650/7146"><li><a href="http://dx.doi.org/10.1650/7146" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hoffman%5Bauthor%5D+AND+Population+trends+of+migratory+raptors+in+Western+North+America%2C+1977%E2%80%932001" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHoffman+%22Population+trends+of+migratory+raptors+in+Western+North+America%2C+1977%E2%80%932001%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0095599-Young1" id="pone.0095599-Young1"></a>Young DP, Erickson WP, Strickland MD, Good RE, Sernka KJ (2003) Comparison of avian responses to UV-light-reflective paint on wind turbines. Golden, CO: National Renewable Energy Laboratory subcontract report. <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0095599-Bloom1" id="pone.0095599-Bloom1"></a>Bloom P, Clark W, Kidd J (2007) Capture techniques. In: Bird DM, Bildstein KL, Barber DR, Zimmerman A, editors. Raptor Research and Management Techniques. Blain, WA: Hancock House Publishers. 193–219. <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0095599-Sambrook1" id="pone.0095599-Sambrook1"></a>Sambrook J, Russell D (2001) Molecular Cloning: A Laboratory Manual, 3rd edition. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press. <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0095599-Fridolfsson1" id="pone.0095599-Fridolfsson1"></a>Fridolfsson A-K, Ellegren H (1999) A simple and universal method for molecular sexing of non-ratite birds. J Avian Biol 30: 116–121.
        doi: 10.2307/3677252 <ul class="find" data-citedArticleID="11172861" data-doi="10.2307/3677252"><li><a href="http://dx.doi.org/10.2307/3677252" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fridolfsson%5Bauthor%5D+AND+A+simple+and+universal+method+for+molecular+sexing+of+non-ratite+birds" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFridolfsson+%22A+simple+and+universal+method+for+molecular+sexing+of+non-ratite+birds%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0095599-Lohse1" id="pone.0095599-Lohse1"></a>Lohse M, Bolger A, Nagel A, Fernie A, Lunn J, et al. (2012) RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics. Nucleic Acids Res 40: W622–W627.
        doi: 10.1093/nar/gks540 <ul class="find" data-citedArticleID="11172863" data-doi="10.1093/nar/gks540"><li><a href="http://dx.doi.org/10.1093/nar/gks540" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lohse%5Bauthor%5D+AND+RobiNA%3A+a+user-friendly%2C+integrated+software+solution+for+RNA-Seq-based+transcriptomics" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALohse+%22RobiNA%3A+a+user-friendly%2C+integrated+software+solution+for+RNA-Seq-based+transcriptomics%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0095599-Simpson1" id="pone.0095599-Simpson1"></a>Simpson JT, Wong K, Jackman SD, Schein JE, Jones SJM, et al.. (2009) ABySS: a parallel assembler for short read sequence data. Genome Res 19: 1117–1123. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2694472&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=2694472&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2013 Aug 6. <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0095599-Parra1" id="pone.0095599-Parra1"></a>Parra G, Bradnam K, Ning Z, Keane T, Korf I (2009) Assessing the gene space in draft genomes. Nucleic Acids Res 37: 298–297.
        doi: 10.1093/nar/gkn916 <ul class="find" data-citedArticleID="11172867" data-doi="10.1093/nar/gkn916"><li><a href="http://dx.doi.org/10.1093/nar/gkn916" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parra%5Bauthor%5D+AND+Assessing+the+gene+space+in+draft+genomes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AParra+%22Assessing+the+gene+space+in+draft+genomes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0095599-Marais1" id="pone.0095599-Marais1"></a>Marçais G, Kingsford C (2011) A fast, lock-free approach for efficient parallel counting of occurrences of k-mers. Bioinformatics 27: 764–770. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3051319&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=3051319&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2013 Aug 7. <ul class="find-nolinks"></ul></li><li><span class="label">31.
              </span><a name="pone.0095599-Li1" id="pone.0095599-Li1"></a>Li R, Fan W, Tian G, Zhu H, He L, et al.. (2010) The sequence and de novo assembly of the giant panda genome. Nature 463: 311–317. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20010809">http://www.ncbi.nlm.nih.gov/pubmed/20010​809</a>. Accessed 2013 Aug 7. <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0095599-Langmead1" id="pone.0095599-Langmead1"></a>Langmead B, Salzberg SL (2012) Fast gapped-read alignment with Bowtie 2. Nat Methods 9: 357–359. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3322381&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=3322381&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2013 Aug 6. <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0095599-Hahn1" id="pone.0095599-Hahn1"></a>Hahn C, Bachmann L, Chevreux B (2013) Reconstructing mitochondrial genomes directly from genomic next-generation sequencing reads–a baiting and iterative mapping approach. Nucleic Acids Res 41: e129. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3711436&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=3711436&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2013 Aug 6. <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0095599-Wyman1" id="pone.0095599-Wyman1"></a>Wyman S, Jansen R, Boore J (2004) Automatic annotation of organellar genomes with DOGMA. Bioinformatics 20: 3252–3255.
        doi: 10.1093/bioinformatics/bth352 <ul class="find" data-citedArticleID="11172877" data-doi="10.1093/bioinformatics/bth352"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth352" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wyman%5Bauthor%5D+AND+Automatic+annotation+of+organellar+genomes+with+DOGMA" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWyman+%22Automatic+annotation+of+organellar+genomes+with+DOGMA%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0095599-Lohse2" id="pone.0095599-Lohse2"></a>Lohse M, Drechsel O, Kahlau S, Bock R (2013) OrganellarGenomeDRAW–a suite of tools for generating physical maps of plastid and mitochondrial genomes and visualizing expression data sets. Nucleic Acids Res. <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0095599-Zhan1" id="pone.0095599-Zhan1"></a>Zhan X, Pan S, Wang J, Dixon A, He J, et al. (2013) Peregrine and saker falcon genome sequences provide insights into evolution of a predatory lifestyle. Nat Genet 45: 536–566  doi:10.1038/ng.2588.  <ul class="find" data-citedArticleID="11172881" data-doi="10.1038/ng.2588"><li><a href="http://dx.doi.org/10.1038/ng.2588" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zhan%5Bauthor%5D+AND+Peregrine+and+saker+falcon+genome+sequences+provide+insights+into+evolution+of+a+predatory+lifestyle" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AZhan+%22Peregrine+and+saker+falcon+genome+sequences+provide+insights+into+evolution+of+a+predatory+lifestyle%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0095599-Grabherr1" id="pone.0095599-Grabherr1"></a>Grabherr M, Haas B, Yassour M, Levin J, Thompson D, et al. (2013) Trinity: reconstructing a full-length transcriptome without a genome from RNA-Seq data. Nat Biotechnol 29: 644–652  doi:10.1038/nbt.1883.Trinity.  <ul class="find" data-citedArticleID="11172883" data-doi="10.1038/nbt.1883"><li><a href="http://dx.doi.org/10.1038/nbt.1883" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Grabherr%5Bauthor%5D+AND+Trinity%3A+reconstructing+a+full-length+transcriptome+without+a+genome+from+RNA-Seq+data" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGrabherr+%22Trinity%3A+reconstructing+a+full-length+transcriptome+without+a+genome+from+RNA-Seq+data%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0095599-Cantarel1" id="pone.0095599-Cantarel1"></a>Cantarel BL, Korf I, Robb SMC, Parra G, Ross E, et al.. (2008) MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes. Genome Res 18: 188–196. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2134774&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=2134774&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2013 Aug 7. <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0095599-Smit1" id="pone.0095599-Smit1"></a>Smit A, Hubley R, Green P (n.d.) RepeatMasker Open-3.0: <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>. <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0095599-Korf1" id="pone.0095599-Korf1"></a>Korf I (2004) Gene finding in novel genomes. BMC Bioinformatics 5: 59 Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=421630&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=421630&amp;tool=pmcentrez&amp;​rendertype=abstract</a>. <ul class="find" data-citedArticleID="11172889" data-doi="10.1186/1471-2105-5-59"><li><a href="http://dx.doi.org/10.1186/1471-2105-5-59" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Korf%5Bauthor%5D+AND+Gene+finding+in+novel+genomes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKorf+%22Gene+finding+in+novel+genomes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0095599-Stanke1" id="pone.0095599-Stanke1"></a>Stanke M, Waack S (2003) Gene prediction with a hidden Markov model and a new intron submodel. Bioinformatics 19: ii215–ii225. Available: <a href="http://bioinformatics.oxfordjournals.org/cgi/doi/10.1093/bioinformatics/btg1080">http://bioinformatics.oxfordjournals.org​/cgi/doi/10.1093/bioinformatics/btg1080</a>. Accessed 2013 Sep 20. <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0095599-Zhan2" id="pone.0095599-Zhan2"></a>Zhan X, Pan S, Wang J, Dixon A, He J, et al.. (2013) Peregrine and saker falcon genome sequences provide insights into evolution of a predatory lifestyle. Nat Genet 45: 563–566. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23525076">http://www.ncbi.nlm.nih.gov/pubmed/23525​076</a>. Accessed 2013 Aug 22. <ul class="find-nolinks"></ul></li><li><span class="label">43.
              </span><a name="pone.0095599-Bergmen1" id="pone.0095599-Bergmen1"></a>Bergmen C, Quesneville H (2007) Discovering and detecting transposable elements in genome sequences. Brief Bioinform 8: 382–392.
        doi: 10.1093/bib/bbm048 <ul class="find" data-citedArticleID="11172895" data-doi="10.1093/bib/bbm048"><li><a href="http://dx.doi.org/10.1093/bib/bbm048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bergmen%5Bauthor%5D+AND+Discovering+and+detecting+transposable+elements+in+genome+sequences" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABergmen+%22Discovering+and+detecting+transposable+elements+in+genome+sequences%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0095599-Makalowski1" id="pone.0095599-Makalowski1"></a>Makalowski W, Pande A, Gotea V, Makalowski I (2012) Transposable elements and their identification. Evol genomics Stat Comput methods 1: 337–359.
        doi: 10.1007/978-1-61779-582-4_12 <ul class="find" data-citedArticleID="11172897" data-doi="10.1007/978-1-61779-582-4_12"><li><a href="http://dx.doi.org/10.1007/978-1-61779-582-4_12" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Makalowski%5Bauthor%5D+AND+Transposable+elements+and+their+identification" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMakalowski+%22Transposable+elements+and+their+identification%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0095599-Lerat1" id="pone.0095599-Lerat1"></a>Lerat E (2010) Identifying repeats and transposable elements in sequenced genomes: how to find your way through the dense forest of programs. Heredity (Edinb) 104: 520–533.
        doi: 10.1038/hdy.2009.165 <ul class="find" data-citedArticleID="11172899" data-doi="10.1038/hdy.2009.165"><li><a href="http://dx.doi.org/10.1038/hdy.2009.165" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lerat%5Bauthor%5D+AND+Identifying+repeats+and+transposable+elements+in+sequenced+genomes%3A+how+to+find+your+way+through+the+dense+forest+of+programs" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALerat+%22Identifying+repeats+and+transposable+elements+in+sequenced+genomes%3A+how+to+find+your+way+through+the+dense+forest+of+programs%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0095599-Saha1" id="pone.0095599-Saha1"></a>Saha S, Bridges S, Magbanua Z, Peterson D (2008) Computation approaches and tools used in identification of dispersed repetitive DNA sequences. Trop Plant Biol 1: 85–96.
        doi: 10.1007/s12042-007-9007-5 <ul class="find" data-citedArticleID="11172901" data-doi="10.1007/s12042-007-9007-5"><li><a href="http://dx.doi.org/10.1007/s12042-007-9007-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Saha%5Bauthor%5D+AND+Computation+approaches+and+tools+used+in+identification+of+dispersed+repetitive+DNA+sequences" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASaha+%22Computation+approaches+and+tools+used+in+identification+of+dispersed+repetitive+DNA+sequences%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0095599-Smit2" id="pone.0095599-Smit2"></a>Smit A, Hubley R (n.d.) RepeatModeler Open-1.0: <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>. <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0095599-Benson1" id="pone.0095599-Benson1"></a>Benson G (1999) Tandem repeats finder: a program to analyze DNA sequences. Nucleic Acids Res 27: 573–580.
        doi: 10.1093/nar/27.2.573 <ul class="find" data-citedArticleID="11172905" data-doi="10.1093/nar/27.2.573"><li><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Benson%5Bauthor%5D+AND+Tandem+repeats+finder%3A+a+program+to+analyze+DNA+sequences" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABenson+%22Tandem+repeats+finder%3A+a+program+to+analyze+DNA+sequences%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0095599-Backstrom1" id="pone.0095599-Backstrom1"></a>Backstrom N, Qvarnstrom A, Gustafsson L, Lundberg A (2006) Levels of linkage disequilibrium in a wild bird population. Biol Lett 2: 435–438.
        doi: 10.1098/rsbl.2006.0507 <ul class="find" data-citedArticleID="11172907" data-doi="10.1098/rsbl.2006.0507"><li><a href="http://dx.doi.org/10.1098/rsbl.2006.0507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Backstrom%5Bauthor%5D+AND+Levels+of+linkage+disequilibrium+in+a+wild+bird+population" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABackstrom+%22Levels+of+linkage+disequilibrium+in+a+wild+bird+population%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0095599-Shapiro1" id="pone.0095599-Shapiro1"></a>Shapiro MD, Kronenberg Z, Li C, Domyan ET, Pan H, et al. (2013) Genomic diversity and evolution of the head crest in the rock pigeon. Science 339: 1063–1067  doi:10.1126/science.1230422.  <ul class="find" data-citedArticleID="11172909" data-doi="10.1126/science.1230422"><li><a href="http://dx.doi.org/10.1126/science.1230422" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shapiro%5Bauthor%5D+AND+Genomic+diversity+and+evolution+of+the+head+crest+in+the+rock+pigeon" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AShapiro+%22Genomic+diversity+and+evolution+of+the+head+crest+in+the+rock+pigeon%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0095599-Bourke1" id="pone.0095599-Bourke1"></a>Bourke BP, Dawson DA (2006) Fifteen microsatellite loci characterized in the golden eagle <em>Aquila chrysaetos</em> (Accipitridae, Aves). Mol Ecol Notes 6: 1047–1050. Available: <a href="http://doi.wiley.com/10.1111/j.1471-8286.2006.01429.x">http://doi.wiley.com/10.1111/j.1471-8286​.2006.01429.x</a>. Accessed 2013 Aug 28. <ul class="find-nolinks"></ul></li><li><span class="label">52.
              </span><a name="pone.0095599-Lewis1" id="pone.0095599-Lewis1"></a>Lewis SE, Searle SMJ, Harris N, Gibson M, Lyer V, et al. (2002) Apollo: a sequence annotation editor. Genome Biol 3: 1–14 Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=151184&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=151184&amp;tool=pmcentrez&amp;​rendertype=abstract</a>. <ul class="find" data-citedArticleID="11172913" data-doi="10.1186/gb-2002-3-12-research0082"><li><a href="http://dx.doi.org/10.1186/gb-2002-3-12-research0082" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lewis%5Bauthor%5D+AND+Apollo%3A+a+sequence+annotation+editor" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALewis+%22Apollo%3A+a+sequence+annotation+editor%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0095599-Thiel1" id="pone.0095599-Thiel1"></a>Thiel T (2002) MISA - Microsatellite identification tool: <a href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>. <ul class="find-nolinks"></ul></li><li><span class="label">54.
              </span><a name="pone.0095599-Li2" id="pone.0095599-Li2"></a>Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, et al. (2009) The sequence alignment/map (SAM) format and SAMtools. Bioinformatics 25: 2078–2079.
        doi: 10.1093/bioinformatics/btp352 <ul class="find" data-citedArticleID="11172917" data-doi="10.1093/bioinformatics/btp352"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Li%5Bauthor%5D+AND+The+sequence+alignment%2Fmap+%28SAM%29+format+and+SAMtools" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALi+%22The+sequence+alignment%2Fmap+%28SAM%29+format+and+SAMtools%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0095599-Odeen1" id="pone.0095599-Odeen1"></a>Odeen A, Hastad O (2003) Complex distribution of avian color vision systems revealed by sequencing the SWS1 opsin from total DNA. Mol Biol Evol 20: 855–861. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12716987">http://www.ncbi.nlm.nih.gov/pubmed/12716​987</a>. Accessed 2013 Dec 12. <ul class="find-nolinks"></ul></li><li><span class="label">56.
              </span><a name="pone.0095599-DeWoody1" id="pone.0095599-DeWoody1"></a>DeWoody JA, Abts KC, Fahey AL, Ji Y, Kimble SJA, et al.. (2013) Of contigs and quagmires: next-generation sequencing pitfalls associated with transcriptomic studies. Mol Ecol Resour 13: 551–558. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23615313">http://www.ncbi.nlm.nih.gov/pubmed/23615​313</a>. Accessed 2013 Sep 16. <ul class="find-nolinks"></ul></li><li><span class="label">57.
              </span><a name="pone.0095599-Huang1" id="pone.0095599-Huang1"></a>Huang Y, Li Y, Burt DW, Chen H, Zhang Y, et al.. (2013) The duck genome and transcriptome provide insight into an avian influenza virus reservoir species. Nat Genet 45: 776–783. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23749191">http://www.ncbi.nlm.nih.gov/pubmed/23749​191</a>. Accessed 2013 Aug 18. <ul class="find-nolinks"></ul></li><li><span class="label">58.
              </span><a name="pone.0095599-Gregory1" id="pone.0095599-Gregory1"></a>Gregory T (2013) Animal genome size database: <a href="http://www.genomesize.com">http://www.genomesize.com</a>. <ul class="find-nolinks"></ul></li><li><span class="label">59.
              </span><a name="pone.0095599-Tiersch1" id="pone.0095599-Tiersch1"></a>Tiersch TR, Wachtel SS (1991) On the evolution of genome size of birds. J Hered 82: 363–368 Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1940280">http://www.ncbi.nlm.nih.gov/pubmed/19402​80</a>. <ul class="find" data-citedArticleID="11172927"><li><a href="http://www.crossref.org/guestquery/?auth2=Tiersch&amp;atitle2=On+the+evolution+of+genome+size+of+birds&amp;auth=Tiersch&amp;atitle=On+the+evolution+of+genome+size+of+birds" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tiersch%5Bauthor%5D+AND+On+the+evolution+of+genome+size+of+birds" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATiersch+%22On+the+evolution+of+genome+size+of+birds%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0095599-Venturini1" id="pone.0095599-Venturini1"></a>Venturini G, D’Ambrogi R, Capanna E (1986) Size and structure of the bird genome–I. DNA content of 48 species of Neognathae. Comp Biochem Physiol B 85: 61–65 Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3769460">http://www.ncbi.nlm.nih.gov/pubmed/37694​60</a>. <ul class="find" data-citedArticleID="11172929" data-doi="10.1016/0305-0491(86)90221-x"><li><a href="http://dx.doi.org/10.1016/0305-0491(86)90221-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Venturini%5Bauthor%5D+AND+Size+and+structure+of+the+bird+genome%E2%80%93I.+DNA+content+of+48+species+of+Neognathae" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AVenturini+%22Size+and+structure+of+the+bird+genome%E2%80%93I.+DNA+content+of+48+species+of+Neognathae%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0095599-Bradnam1" id="pone.0095599-Bradnam1"></a>Bradnam KR, Fass JN, Alexandrov A, Baranay P, Bechner M, et al.. (2013) Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species. Gigascience 2: 10. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3844414&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=3844414&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2014 Feb 24. <ul class="find-nolinks"></ul></li><li><span class="label">62.
              </span><a name="pone.0095599-Organ1" id="pone.0095599-Organ1"></a>Organ CL, Shedlock AM, Meade A, Pagel M, Edwards SV (2007) Origin of avian genome size and structure in non-avian dinosaurs. Nature 446: 180–184. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17344851">http://www.ncbi.nlm.nih.gov/pubmed/17344​851</a>. Accessed 2013 Sep 20. <ul class="find-nolinks"></ul></li><li><span class="label">63.
              </span><a name="pone.0095599-Ellegren1" id="pone.0095599-Ellegren1"></a>Ellegren H (2005) The avian genome uncovered. Trends Ecol Evol 20: 180–186. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16701366">http://www.ncbi.nlm.nih.gov/pubmed/16701​366</a>. Accessed 2013 Sep 21. <ul class="find-nolinks"></ul></li><li><span class="label">64.
              </span><a name="pone.0095599-Hillier1" id="pone.0095599-Hillier1"></a>Hillier LW, Miller W, Birney E, Warren W HR (2004) Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution. Nature 432: 695–716 Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15592404">http://www.ncbi.nlm.nih.gov/pubmed/15592​404</a>. <ul class="find" data-citedArticleID="11172937"><li><a href="http://www.crossref.org/guestquery/?auth2=Hillier&amp;atitle2=Sequence+and+comparative+analysis+of+the+chicken+genome+provide+unique+perspectives+on+vertebrate+evolution&amp;auth=Hillier&amp;atitle=Sequence+and+comparative+analysis+of+the+chicken+genome+provide+unique+perspectives+on+vertebrate+evolution" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hillier%5Bauthor%5D+AND+Sequence+and+comparative+analysis+of+the+chicken+genome+provide+unique+perspectives+on+vertebrate+evolution" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHillier+%22Sequence+and+comparative+analysis+of+the+chicken+genome+provide+unique+perspectives+on+vertebrate+evolution%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0095599-Nevitt1" id="pone.0095599-Nevitt1"></a>Nevitt GA, Bonadonna F (2005) Sensitivity to dimethyl sulphide suggests a mechanism for olfactory navigation by seabirds. Biol Lett 1: 303–305. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1617144&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=1617144&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2014 Jan 17. <ul class="find-nolinks"></ul></li><li><span class="label">66.
              </span><a name="pone.0095599-Nevitt2" id="pone.0095599-Nevitt2"></a>Nevitt G, Reid K, Trathan P (2004) Testing olfactory foraging strategies in an Antarctic seabird assemblage. J Exp Biol 207: 3537–3544. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15339950">http://www.ncbi.nlm.nih.gov/pubmed/15339​950</a>. Accessed 2014 Jan 17. <ul class="find-nolinks"></ul></li><li><span class="label">67.
              </span><a name="pone.0095599-Steiger1" id="pone.0095599-Steiger1"></a>Steiger SS, Fidler AE, Valcu M, Kempenaers B (2008) Avian olfactory receptor gene repertoires: evidence for a well-developed sense of smell in birds? Proc Biol Sci 275: 2309–2317. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2495045&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=2495045&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2014 Jan 15. <ul class="find-nolinks"></ul></li><li><span class="label">68.
              </span><a name="pone.0095599-Moritz1" id="pone.0095599-Moritz1"></a>Moritz C (1994) Applications of mitochondrial DNA analysis in conservation: a critical review. Mol Ecol 3: 401–411.
        doi: 10.1111/j.1365-294x.1994.tb00080.x <ul class="find" data-citedArticleID="11172945" data-doi="10.1111/j.1365-294x.1994.tb00080.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.1994.tb00080.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Moritz%5Bauthor%5D+AND+Applications+of+mitochondrial+DNA+analysis+in+conservation%3A+a+critical+review" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMoritz+%22Applications+of+mitochondrial+DNA+analysis+in+conservation%3A+a+critical+review%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0095599-Frankham1" id="pone.0095599-Frankham1"></a>Frankham R, Ballou J, Dudash M, Eldridge M, Fenster C, et al. (2012) Implications of different species concepts for conserving biodiversity. Biol Conserv 153: 25–31.
        doi: 10.1016/j.biocon.2012.04.034 <ul class="find" data-citedArticleID="11172947" data-doi="10.1016/j.biocon.2012.04.034"><li><a href="http://dx.doi.org/10.1016/j.biocon.2012.04.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frankham%5Bauthor%5D+AND+Implications+of+different+species+concepts+for+conserving+biodiversity" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFrankham+%22Implications+of+different+species+concepts+for+conserving+biodiversity%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0095599-Lerner1" id="pone.0095599-Lerner1"></a>Lerner HRL, Mindell DP (2005) Phylogeny of eagles, Old World vultures, and other Accipitridae based on nuclear and mitochondrial DNA. Mol Phylogenet Evol 37: 327–346. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15925523">http://www.ncbi.nlm.nih.gov/pubmed/15925​523</a>. Accessed 2013 Sep 23. <ul class="find-nolinks"></ul></li><li><span class="label">71.
              </span><a name="pone.0095599-Seabury1" id="pone.0095599-Seabury1"></a>Seabury CM, Dowd SE, Seabury PM, Raudsepp T, Brightsmith DJ, et al.. (2013) A multi-platform draft de novo genome assembly and comparative analysis for the Scarlet Macaw (<em>Ara macao</em>). PLoS One 8: e62415. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3648530&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=3648530&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2013 Aug 21. <ul class="find-nolinks"></ul></li><li><span class="label">72.
              </span><a name="pone.0095599-Warren1" id="pone.0095599-Warren1"></a>Warren WC, Clayton DF, Ellegren H, Arnold AP, Hillier LW, et al.. (2010) The genome of a songbird. Nature 464: 757–762. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3187626&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=3187626&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2013 Aug 12. <ul class="find-nolinks"></ul></li><li><span class="label">73.
              </span><a name="pone.0095599-Lind1" id="pone.0095599-Lind1"></a>Lind O, Mitkus M, Olsson P, Kelber A (2013) Ultraviolet sensitivity and colour vision in raptor foraging. J Exp Biol 216: 1819–1826. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23785106">http://www.ncbi.nlm.nih.gov/pubmed/23785​106</a>. Accessed 2014 Jan 7. <ul class="find-nolinks"></ul></li><li><span class="label">74.
              </span><a name="pone.0095599-Hart1" id="pone.0095599-Hart1"></a>Hart NS, Hunt DM (2007) Avian visual pigments: characteristics, spectral tuning, and evolution. Am Nat 169 Suppl: S7–26Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19426092">http://www.ncbi.nlm.nih.gov/pubmed/19426​092</a>. <ul class="find" data-citedArticleID="11172957" data-doi="10.1086/510141"><li><a href="http://dx.doi.org/10.1086/510141" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hart%5Bauthor%5D+AND+Avian+visual+pigments%3A+characteristics%2C+spectral+tuning%2C+and+evolution" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHart+%22Avian+visual+pigments%3A+characteristics%2C+spectral+tuning%2C+and+evolution%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0095599-Viitala1" id="pone.0095599-Viitala1"></a>Viitala J, Korpimaki E, Palokangas P, Kolvula M (1995) Attraction of kestrels to vole scent marks visible in ultraviolet light. Nature 373: 425–426.
        doi: 10.1038/373425a0 <ul class="find" data-citedArticleID="11172959" data-doi="10.1038/373425a0"><li><a href="http://dx.doi.org/10.1038/373425a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viitala%5Bauthor%5D+AND+Attraction+of+kestrels+to+vole+scent+marks+visible+in+ultraviolet+light" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AViitala+%22Attraction+of+kestrels+to+vole+scent+marks+visible+in+ultraviolet+light%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0095599-Nekrutenko1" id="pone.0095599-Nekrutenko1"></a>Nekrutenko A, Li W-H (2000) Assessment of compositional heterogeneity within and between eukaryotic genomes. Genome Res 10: 1986–1995.
        doi: 10.1101/gr.10.12.1986 <ul class="find" data-citedArticleID="11172961" data-doi="10.1101/gr.10.12.1986"><li><a href="http://dx.doi.org/10.1101/gr.10.12.1986" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nekrutenko%5Bauthor%5D+AND+Assessment+of+compositional+heterogeneity+within+and+between+eukaryotic+genomes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ANekrutenko+%22Assessment+of+compositional+heterogeneity+within+and+between+eukaryotic+genomes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0095599-Doyle1" id="pone.0095599-Doyle1"></a>Doyle J, Siegmund G, Ruhl J, Eo S-H, Hale M, et al. (2013) Microsatellite analyses across three diverse vertebrate transcriptomes (<em>Acipenser fulvescens</em>, <em>Ambystoma tigrinum</em>, and <em>Dipodomys spectabilis</em>). Genome 56: 1–8.
        doi: 10.1139/gen-2013-0056 <ul class="find" data-citedArticleID="11172963" data-doi="10.1139/gen-2013-0056"><li><a href="http://dx.doi.org/10.1139/gen-2013-0056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Doyle%5Bauthor%5D+AND+Microsatellite+analyses+across+three+diverse+vertebrate+transcriptomes+%28Acipenser+fulvescens%2C+Ambystoma+tigrinum%2C+and+Dipodomys+spectabilis%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ADoyle+%22Microsatellite+analyses+across+three+diverse+vertebrate+transcriptomes+%28Acipenser+fulvescens%2C+Ambystoma+tigrinum%2C+and+Dipodomys+spectabilis%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0095599-Morin1" id="pone.0095599-Morin1"></a>Morin A, Wallis J, Moore J, Chakraborty R, Woodruff D (1993) Non-invasive sampling and DNA amplification for paternity exclusion, community structure, and phylogeography in wild chimpanzees. Primates 34: 347–356.
        doi: 10.1007/bf02382630 <ul class="find" data-citedArticleID="11172965" data-doi="10.1007/bf02382630"><li><a href="http://dx.doi.org/10.1007/bf02382630" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Morin%5Bauthor%5D+AND+Non-invasive+sampling+and+DNA+amplification+for+paternity+exclusion%2C+community+structure%2C+and+phylogeography+in+wild+chimpanzees" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMorin+%22Non-invasive+sampling+and+DNA+amplification+for+paternity+exclusion%2C+community+structure%2C+and+phylogeography+in+wild+chimpanzees%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0095599-Oka1" id="pone.0095599-Oka1"></a>Oka T, Takenaka O (2001) Wild gibbons’ parentage tested by non-invasive DNA sampling and PCR-amplified polymorphic microsatellites. Primates 42: 67–73.
        doi: 10.1007/bf02640690 <ul class="find" data-citedArticleID="11172967" data-doi="10.1007/bf02640690"><li><a href="http://dx.doi.org/10.1007/bf02640690" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oka%5Bauthor%5D+AND+Wild+gibbons%E2%80%99+parentage+tested+by+non-invasive+DNA+sampling+and+PCR-amplified+polymorphic+microsatellites" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AOka+%22Wild+gibbons%E2%80%99+parentage+tested+by+non-invasive+DNA+sampling+and+PCR-amplified+polymorphic+microsatellites%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0095599-Heinsohn1" id="pone.0095599-Heinsohn1"></a>Heinsohn R, Ebert D, Legge S, Peakall R (2007) Genetic evidence for cooperative polyandry in reverse dichromatic <em>Eclectus</em> parrots. Anim Behav 74: 1047–1054. Available: <a href="http://linkinghub.elsevier.com/retrieve/pii/S0003347207002771">http://linkinghub.elsevier.com/retrieve/​pii/S0003347207002771</a>. Accessed 2013 Sep 17. <ul class="find-nolinks"></ul></li><li><span class="label">81.
              </span><a name="pone.0095599-Ahlering1" id="pone.0095599-Ahlering1"></a>Ahlering MA, Hedges S, Johnson A, Tyson M, Schuttler SG, et al.. (2010) Genetic diversity, social structure, and conservation value of the elephants of the Nakai Plateau, Lao PDR, based on non-invasive sampling. Conserv Genet 12: 413–422. Available: <a href="http://link.springer.com/10.1007/s10592-010-0148-y">http://link.springer.com/10.1007/s10592-​010-0148-y</a>. Accessed 2013 Sep 17. <ul class="find-nolinks"></ul></li><li><span class="label">82.
              </span><a name="pone.0095599-Oleksyk1" id="pone.0095599-Oleksyk1"></a>Oleksyk TK, Pombert J-F, Siu D, Mazo-Vargas A, Ramos B, et al.. (2012) A locally funded Puerto Rican parrot (<em>Amazona vittata</em>) genome sequencing project increases avian data and advances young researcher education. Gigascience 1: 14. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3626513&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=3626513&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2013 Sep 17. <ul class="find-nolinks"></ul></li><li><span class="label">83.
              </span><a name="pone.0095599-Rasch1" id="pone.0095599-Rasch1"></a>Rasch E (1985) DNA “standards” and the range of accurate DNA estimates by Feulgen absorption microspectrometry. In: Cowden R, Harrison S, editors. Advances in Microscopy. 137–166. <ul class="find-nolinks"></ul></li><li><span class="label">84.
              </span><a name="pone.0095599-Ellegren2" id="pone.0095599-Ellegren2"></a>Ellegren H, Smeds L, Burri R, Olason PI, Backström N, et al.. (2012) The genomic landscape of species divergence in <em>Ficedula</em> flycatchers. Nature 491: 756–760. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23103876">http://www.ncbi.nlm.nih.gov/pubmed/23103​876</a>. Accessed 2013 Sep 16. <ul class="find-nolinks"></ul></li><li><span class="label">85.
              </span><a name="pone.0095599-Rands1" id="pone.0095599-Rands1"></a>Rands CM, Darling A, Fujita M, Kong L, Webster MT, et al.. (2013) Insights into the evolution of Darwin’s finches from comparative analysis of the <em>Geospiza magnirostris</em> genome sequence. BMC Genomics 14: 95. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3575239&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=3575239&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2013 Sep 17. <ul class="find-nolinks"></ul></li><li><span class="label">86.
              </span><a name="pone.0095599-Dalloul1" id="pone.0095599-Dalloul1"></a>Dalloul RA, Long JA, Zimin AV, Aslam L, Beal K, et al.. (2010) Multi-platform next-generation sequencing of the domestic turkey (<em>Meleagris gallopavo</em>): genome assembly and analysis. PLoS Biol 8. Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2935454&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/article​render.fcgi?artid=2935454&amp;tool=pmcentrez​&amp;rendertype=abstract</a>. Accessed 2013 Aug 15. <ul class="find-nolinks"></ul></li><li><span class="label">87.
              </span><a name="pone.0095599-Krishan1" id="pone.0095599-Krishan1"></a>Krishan A, Dandekar P, Nathan N, Hamelik R, Miller C, et al.. (2005) DNA index, genome size, and electronic nuclear volume of vertebrates from the Miami Metro Zoo. Cytometry A 65: 26–34. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15779063">http://www.ncbi.nlm.nih.gov/pubmed/15779​063</a>. Accessed 2014 Jan 9. <ul class="find-nolinks"></ul></li><li><span class="label">88.
              </span><a name="pone.0095599-Andrews1" id="pone.0095599-Andrews1"></a>Andrews CB, Gregory TR (2009) Genome size is inversely correlated with relative brain size in parrots and cockatoos. Genome 52: 261–267. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19234554">http://www.ncbi.nlm.nih.gov/pubmed/19234​554</a>. Accessed 2014 Jan 9. <ul class="find-nolinks"></ul></li><li><span class="label">89.
              </span><a name="pone.0095599-Qu1" id="pone.0095599-Qu1"></a>Qu Y, Zhao H, Han N, Zhou G, Song G, et al.. (2013) Ground tit genome reveals avian adaptation to living at high altitudes in the Tibetan plateau. Nat Commun 4: 2071. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23817352">http://www.ncbi.nlm.nih.gov/pubmed/23817​352</a>. Accessed 2013 Sep17. <ul class="find-nolinks"></ul></li><li><span class="label">90.
              </span><a name="pone.0095599-Peterson1" id="pone.0095599-Peterson1"></a>Peterson DG, Stack SM, Healy JL, Donohoe BS, Anderson LK (1994) The relationship between synaptonemal complex length and genome size in four vertebrate classes. Chromosom Res 2: 153–162.
        doi: 10.1007/bf01553494 <ul class="find" data-citedArticleID="11172989" data-doi="10.1007/bf01553494"><li><a href="http://dx.doi.org/10.1007/bf01553494" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peterson%5Bauthor%5D+AND+The+relationship+between+synaptonemal+complex+length+and+genome+size+in+four+vertebrate+classes" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3APeterson+%22The+relationship+between+synaptonemal+complex+length+and+genome+size+in+four+vertebrate+classes%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t001.PNG_I" value="20084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g001.PNG_I" value="182240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g002.PNG_I" value="61882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g003.PNG_I" value="16332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t002.PNG_I" value="11810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t003.PNG_I" value="15056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.s001.TIF" value="1161770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.s002.TIF" value="23048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.s003.DOCX" value="79601"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.XML" value="126835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g001.PNG_S" value="22546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g001.PNG_M" value="574973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g001.PNG_L" value="7148534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g001.TIF" value="6051808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g002.PNG_S" value="18106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g002.PNG_M" value="130366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g002.PNG_L" value="443267"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g002.TIF" value="556388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g003.PNG_S" value="9817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g003.PNG_M" value="27163"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g003.PNG_L" value="78630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.g003.TIF" value="309522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.PDF" value="3209387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t001.PNG_S" value="7937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t001.PNG_M" value="39755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t001.PNG_L" value="253732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t001.TIF" value="667730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t002.PNG_S" value="2351"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t002.PNG_M" value="28600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t002.PNG_L" value="56501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t002.TIF" value="224630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t003.PNG_S" value="7134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t003.PNG_M" value="27418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t003.PNG_L" value="170506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0095599.t003.TIF" value="536168"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0095599&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0095599" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0095599&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0095599&volume=&issue=&title=The Genome Sequence of a Widespread Apex Predator, the Golden Eagle (Aquila chrysaetos)&author_name=Jacqueline%20M.%20Doyle%2C%20Todd%20E.%20Katzner%2C%20Peter%20H.%20Bloom%2C%20Yanzhu%20Ji%2C%20Bhagya%20K.%20Wijayawardena%2C%20J.%20Andrew%20DeWoody&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0095599" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0095599" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0095599" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0095599&amp;t=The%20Genome%20Sequence%20of%20a%20Widespread%20Apex%20Predator%2C%20the%20Golden%20Eagle%20(Aquila%20chrysaetos)" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0095599&title=The%20Genome%20Sequence%20of%20a%20Widespread%20Apex%20Predator%2C%20the%20Golden%20Eagle%20(Aquila%20chrysaetos)&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0095599&amp;title=The%20Genome%20Sequence%20of%20a%20Widespread%20Apex%20Predator%2C%20the%20Golden%20Eagle%20(Aquila%20chrysaetos)" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0095599" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0095599&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The Genome Sequence of a Widespread Apex Predator, the Golden Eagle (Aquila chrysaetos)';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0095599';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20Genome%20Sequence%20of%20a%20Widespread%20Apex%20Predator%2C%20the%20Golden%20Eagle%20(Aquila%20chrysaetos) http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0095599" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0095599" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bird+genomics%22" title="Search for articles in the subject area:'Bird genomics'"><div class="flagText">Bird genomics</div></a>
              <div data-categoryid="20529" data-articleid="239891"
                   data-categoryname="Bird genomics"
                   class="flagImage" title="Flag 'Bird genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Birds%22" title="Search for articles in the subject area:'Birds'"><div class="flagText">Birds</div></a>
              <div data-categoryid="42813" data-articleid="239891"
                   data-categoryname="Birds"
                   class="flagImage" title="Flag 'Birds' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Eagles%22" title="Search for articles in the subject area:'Eagles'"><div class="flagText">Eagles</div></a>
              <div data-categoryid="43619" data-articleid="239891"
                   data-categoryname="Eagles"
                   class="flagImage" title="Flag 'Eagles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="239891"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="239891"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+libraries%22" title="Search for articles in the subject area:'Genomic libraries'"><div class="flagText">Genomic libraries</div></a>
              <div data-categoryid="40315" data-articleid="239891"
                   data-categoryname="Genomic libraries"
                   class="flagImage" title="Flag 'Genomic libraries' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="239891"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="239891"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5712'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3899'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5056&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.17</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
